# Mathematical foundation of cell-type deconvolution of biological data {#methods}

In this chapter, we will discuss how mathematical models can be used to extract information about specific cell-types from 'bulk' data or how to unmix mixed sources.  It will introduce you to basic concepts of data analysis as well as most popular advanced solutions adapted for estimating presence and proportion of immune cells within cancer biopsies. 

*Note: Parts of this chapter with benchmark data might be a part of a future review*

Explain the principle 


```{r deconvolution-cartoon, fig.cap='(ref:deconvolution-cartoon-caption)', out.width='100%', fig.align='center', echo = FALSE}
if (is_pdf_output()) {
  knitr::include_graphics('figures-ext/deconv.pdf')
} else {
  knitr::include_graphics('figures-ext/deconv.png')
}
```
(ref:deconvolution-cartoon-caption) **Principle of the deconvolution applied to transcriptome** Graphical illustration of the deconvolution of mixed samples. Starting from the left, gene expression of genes A B C is a sum of expression of cell types 1, 2, 3, 4. After deconvolution, cell types are separated and gene expression of each cell type is estimated taking into account cell type proportions.


## Introduction to supervised  and unsupervised learning

* Explaining difference between supervised and non-supervised learning technically and in our context

## Blind source separation
 (ICA, NMF, CAM)

## Finding optimal number of components and over-decomposition of transcriptopmes

* Explain why this problem is important
* Explain shortly  my role in the paper

[@Ulykbek2017]

\includepdf[pages={1-}, scale=1]{pdf-ext/BMCMSTD.pdf}

## Cell-type deconvolution models 
(families of approaches/strategies) 

### basis matrix

* define what basis matrix is vs signatures vs marker genes
* explain the differences between basis matrices in published algorithms

### regression algorithm

* explain what it is for 
* explain how it works
* explain the different approaches

### Others

* explain possible normalisation 
* explain additional possible features 

## Short review of most popular cell-type deconvolution tools 

Tools will already be mentioned in the section above. However a comment on other than mentioned aspects are needed. 

Explain the success of CIBEROSRT

Explain originality of each tool

## Methodological dimension of the thesis

* expose state of art and compare existing tools 
* adjustment of existing methodology
* develop a new user-friendly tool
* discuss limitations of computational approaches in biology




