# (PART) Results {-}

# Determining the optimal number of independent components for reproducible transcriptomic data analysis {#mstd}

```{r smallChapMSTD, results="asis", echo = FALSE}
if(is_pdf_output()){
cat("\\chaptermark{Most Stable Transcriptomic Dimension}")
}
```
Ulykbek Kairov$^\star$, Laura Cantini$^\star$, Alessandro Greco, Askhat Molkenov, **Urszula Czerwinska**, Emmanuel Barillot and Andrei Zinovyev

 $^\star$ $^{_{contributed}}$ $^{_{equally}}$

*Published in BMC Genomics, 11 September 2017*

## Context

In the introduction to the computational cell-type deconvolution, I have introduced Matrix factorization *family* of approaches including, Independent Components Analysis (ICA). ICA decomposes transcriptome ($X$) into to matrices of sources ($S$) and mixing matrix ($A$).  I have mentioned that one of the most critical parameters to decompose transcriptome with matrix factorization methods is the parameter that I called $k$ (called $M$ through this publication), which is the number of sources (independent components). 

## Description

In this publication, @Kairov2017 developed a way to identify a Maximally Stable Transcriptomic Dimension (MSTD). This index helps to decompose transcriptomes into interpretable biological factors. We mention that different methods to find the right number of $k$ were developed in the previous works. However, none was conceived with biological interpretation clarity in mind. The MSTD index is computed based on the stability of the components over different $k$. The details of how MSTD is computed are explained in the publication.

The components coming from decompositions of within MSTD range have a higher probability to be found in other transcriptome datasets. In this way, reproducible signatures of cancer transcriptomes can be identified in diverse cancer datasets, published by different authors and in different platforms. 

The concept was tested on TCGA data and six independent breast cancer datasets (37 datasets in total).

In overall, we observed that average stability of computed components decreases when the number of components increase (Fig 1. [@Kairov2017]), while the top components keep their stability almost unchanged.

If one uses $k$ < MSTD, it makes the identification of biological signals difficult because different factors (sources) remain merged or mixed.

An unexpected observation while working on MSTD was that if the transcriptome is decomposed into a high number of $k$ $\gg$ MSTD, the signals existing in lower dimensions are robustly identifiable. Besides, an important observation was made in METABRIC dataset, where the *Immune* signal existing in MSTD dimension, in high $k$, splits into three signals identifiable as groups of immune cells (Fig. 2f [@Kairov2017]). 

A side-effect of $k$ $\gg$ MSTD were also components driven by a small number of genes. This phenomenon is described in details in the publication. 

## Impact on the further work

I have contributed to this publication by running numerous simulations and working on the final manuscript. 

This work was a significant breakthrough in my work on cell-type deconvolution. While the co-authors started to develop the MSTD index, I was working on finding the best $k$ for immune cells identification. I was trying mainly two different strategies:

1. decomposition of the transcriptome matrix with fastICA into a very stable dimension $k$$\approx$ 10, selection of the immune signal, selection of the $n$ top genes of these immune components from the transcriptome and running the fastICA again
2. trying different decompositions and interpreting results, with an objective to define best $k$ for the interpretability.

I have presented the first strategy (1), that was giving promising results on a breast carcinoma dataset, at [ISMB conference in 2016](https://www.iscb.org/ismb2016). I gave a short talk and presented an [award-winning poster](https://www.iscb.org/ismb2016general/ismb2016-awardwinners#f1000) presenting the strategy. However, this strategy was not easy to generalize and apply to multiple datasets. 

Thus, I started to experiment with the second strategy. However, it was not easy to evaluate the quality of decomposition, as I was employing gene enrichment methods (like GSEA or Fisher test described in the previous chapter) that are not free from false positives. 

Finally, participating in the work on this publication, I have found a third possibility:

3. decomposing transcriptome into a high number of components ($k \gg MSTD$) that allows direct identification of cell-type-specific components

The possibility to apply this strategy reproducibly remained unclear. The *unstable* components were not supposed to be found with high probability in other transcriptome data. 

In the next [chapter](#Lvaica), I describe the study where I test the third strategy in multiple cancer transcriptome datasets. 

```{r pdf, echo=FALSE, results='asis', eval=is_pdf_output()}
cat("\\includepdf[pages={1-}, scale=1]{pdf-ext/BMCMSTD.pdf}")
```





