# (PART) Discussion {-}

# Discussion{#discussiongenerale}

> *There are known knowns. These are things we know that we know. There are known unknowns. That is to say, there are things that we know we don't know. But there are also unknown unknowns. There are things we don't know we don't know.*
>                                                           Donald Rumsfeld


In this work, I aimed to combine biological and mathematical expertise in order to approach the better understanding of the TME. I approached the complexity of TME with mathematical tools applied to transcriptome data. Starting t his work, three years ago, little was know about the possibility of extracting immune signals from bulk tumor transcriptomes. **In this Ph.D. project I tested the limits of detection of the immune signal from bulk transcriptomes with unsupervised methods.** 

## Improving our knowledge of how best apply ICA to transcriptomic data

Through Chapters \@ref(mstd)-\@ref(nmfica), I tested parameters of Independent Components Analysis to optimize it for immune signal extraction. 

First publication (Chapter \@ref(mstd)) focused on developing and computing the MSTD index. This work led to a better understanding of our methods. It also resulted in helpful observations for my further deconvolution work. So far, many authors applied ICA to transcriptomic data for separation of biological, technical and environmental factors [@Teschendorff2007; @Biton2014; @Gorban2007; @Zinovyev2013]. However, mostly theoretical works were discussing the adequate number of dimensions proposing metrics based on purely mathematical/statistical arguments [@Cangelosi2007; @Kegl2003; @Bro2008]. In contrast, as mentioned in the publication (Chapter \@ref(mstd))[@Kairov2017], we asked a question: *which is the most informative number of components to use in the downstream analysis?*.

The work on MSTD index triggered a change in my protocol towards *overdecomposition* of transcriptomes. Most of previous works on unsupervised deconvolution of cell types, either impose the number of sources based ($k$) based on previous knowledge [@Repsilber2010; @Ogundijo2017; @Newberg2018] or estimate is based on statistical indexes, such as Akaike information criterion (AICc) [proposed initially by @Hurvich1989, applied to estimation of number of cell types by @Liebner2014], Minimum Description Length [proposed initially by @Wax1985, applied in CAM model by @Wang2016], leave-one-out cross-validation (LOOCV) [@Lahdesmaki2005] and Â symmetric Kullback-Leibler divergence (SKLD) [@Zuckerman2013] (when sources are known) or else @Moffitt2015 applied a custom method testing many possible dimensions. To my knowledge, no previous work employes the *overdecomposition* approach to solving the problem of the unknown number of cell types, the methods mentioned above of $k$ estimation propose a lower number of dimensions (in the range 5-20).

## Confronting ICA with other matrix factorization approaches

With my application of this protocol to six breast cancer transcriptomic datasets, I validated a hypothesis that ICA can extract reproducible signals from breast cancer transcriptomes and that some of those signals can be consequently labeled as cell-types (Chapter \@ref(lva)). In my additional work, I compared ICA and NMF algorithms showing that ICA breast cancer transcriptomes decomposition results in more interpretable readout (Chapter \@ref(nmfica)). NMF and ICA were systematically compared in other fields of application (i.e., image deconvolution [@Mirzal2017], web mining [@Chikhi2007]; physiological signal processing [@Tresch2006]; face recognition [@Rajapakse2003]) with the outcome depending on the context. On the topic of gene expression, little works compare NMF and ICA in a systematic study. Some authors claim the similar performance of both algorithms[@Kong2011; @Schachtner2008], some assume that NMF is better suited for biological data because of the non-negativity assumption [@Li2013], other proved ICA not to recover cell states in the specific context [@Sokol2015] other discussed the outperformance of ICA over PCA [@Teschendorff2007].  My findings are consistent with the more extensive comparison of @Cantini2018 from our group. Despite this result, no categorical statement on the superiority of ICA over NMF can be made. ICA and NMF differ in the mathematical approach, and its efficiency might be highly context-dependent. Some recent works on ICA indicate that the performance of an algorithm can be strikingly different when applied to simulated and real data [@Ablin2018]. Thus different *flavors* of NMF and ICA should also be tested on real experimental data taking into account user-friendliness, the speed, and interpretability.

## Deconvolution methods overall impact and utility

In Chapter \@ref(deconica), I implemented the methods used in the previous studies in a well-documented tool. It was an essential part of this work to make it reproducible and freely accessible to any researcher. I demonstrated that ICA is able to separate immune cell-types in blood transcriptomes with a competitive performance. 

To put it in the context, in the introduction of Chapter \@ref(methods), I presented a wide array of deconvolution tools (Table \@ref(tab:mytab)). It can be observed that the field evolves impressively fast, and competition is fierce. *It would be interesting to measure the direct impact of each of the tool on the immunobiology field and percentage of the progress in immunotherapies due to the deconvolution methods.* What can be observed, it is the number and type of citations (columns *pop.index* and *citations*  Tab. \@ref(tab:mytab)). For most of the methods published in theory-focused journals, they are rarely cited in biological works using the method. All in all, without doubts CIBERSORT [@Newman2015], met the most significant success, not only thanks to the solid scientific basis, extensive validation, and high impact journal but probably also because of the user-friendly web interface and simplicity (from the user perspective). Even though newer tools argue its accuracy for RNA-seq data [@Li2017; @Tamborero2018], it will probably remain the champion of the field for a long time. Which does not mean that subsequent efforts are pointless as (1) CIBERSORT is under MIT license which means any new tools based on CIBERSORT methods belong to MIT (2) there is more to explore in the TME than only cell-type abundance (3) validity of CIBERSORT (and other methods) cannot be confirmed without gold standard benchmark.

The success of CIBERSORT brings into the light another critical topic which is reproducibility crisis in research [@Fanelli2018]. It is still common to find publications about software without code or online access provided. Moreover, even if thanks to community and publishers pressure it is less and less frequent (as demonstrated with numbers in Chapter \@ref(methods)), still it is not trivial to understand and reproduce published tools. Often documentation is not provided or floppy. Scripts deposed on a public repository are not conformed to any standards and are not tested on different operating systems and software versions. One answer to this problem could be Docker image technology [@Boettiger2014] that allows sharing a *frozen* environment where the tool can run, and that is not affected by user informatics environment. However, this does not replace substantial documentation and examples provided with published tools. I put my best effort to make my tool easy to use providing a tutorial and an R package following good practice guidelines with a sincere hope that users will be able to reproduce my work and build on without my extensive assistance. 

As I stated in the Chapter \@ref(methods), there is a schematic validation framework that is followed by many authors of deconvolution tools [i.e., @Racle2017; @Newman2015; @Aran2017; @Becht2016]. This framework has one crucial problem: lack of gold standard validation datasets. Without a high-dimensional collection of bulk transcriptomic samples paired with an independent measure of cell-type proportions in different solid tumors, it is not possible to objectively assess the performance of published cell-type deconvolution tools. With such a benchmark it would be easier to make the field progress in the direction that can bring most benefits to immunobiology through trustful information. 

## Unsupervised deconvolution

I claimed that most of the tools published in the filed are using cell profiles available from the blood[@Newman2015; @Aran2017; @Becht2016] and some from cancer single cell studies [@Racle2017; @Schelker2017]. The few existing unsupervised tools [@Repsilber2010; @Wang2016; @Gaujoux2010] were not widely applied in the context of tumor transcriptomes. **This work shall be the first one applying an unsupervised deconvolution framework pan-cancer with focus on cell-type deconvolution**. 

A crucial part of an unsupervised analysis is the data interpretation. So far most of the unsupervised methods proposed deconvolution algorithm without a method to interpret of the resulting components [@Newberg2018; @Wang2016]. Among previous works, @Repsilber2010 proposed a *mapping strategy* extended by @Gaujoux2012 and included in *CellMix* package [@Gaujoux2013]. The mapping strategy consisted on computing contingency table between discovered markers (top genes of NMF metagenes) and the known lists of source markers. The proposed method is close to the Fisher exact test enrichment. The main difference is the iterative scheme that avoids repetition of labels (the same cell type attributed to many components). It would be interesting to test this approach with the signals extracted from tumor transcriptomes.

The data-driven nature of the unsupervised methods can bring unexpected results, contrasted with the more predictable behavior of supervised methods and discourage researches from experimenting with them. @Gaujoux2012 overcomes this issue using the semi-supervised method that guides NMF to find desired components. I followed the utterly unsupervised strategy to learn from the data possible limitations.

### Limitations of unsupervised deconvolution

It is worth mentioning that breast cancer transcriptome was mostly used to calibrate the ICA methods as it is one of the most frequent cancers and there are publicly available datasets with large cohorts. Later results (Chapter \@ref(results)) show that immune signal extracted through ICA deconvolution from breast cancer transcriptomes are the closest to the reference profiles for all three considered cell types (T cell, B cell, and Myeloid cells) and their *detectability* is quite high.  This suggests that similar studies could not be reproduced in some cancer types, the, i.e., the T-cell signal is much further from the reference profiles (or not detectable) in colorectal cancer. 

Through the application of DeconICA to various transcriptomic datasets, I observed variable detectability depending on tumor type and a number of samples. For the tumor types, some trends can be observed and interpreted. In a case of AML which immune cell-type signals were highly correlated with the reference profiles, the samples are liquid to contrast with solid tissues. Then the differences in the decomposition of solid tumors and possible biological consequences of deviation of the cell-type specific signal from the reference to be understood. The number of samples is more of technical matter. However, it is surprising that for some datasets even a low number of samples (50-90) results in meaningful signal extraction, while for some datasets with >100 samples, the decomposition can be unsuccessful. The general trend says more tumor samples, better chance to extract meaningful signals however it is not guaranteed.  As the DeconICA signal extraction is data-driven and the same reference as used for all datasets, we can learn that probably use of reference genes in some tumor types may be difficult. In xCell publication [@Aran2017], the authors applied their deconvolution tool to TCGA tumor types. Their sample space tSNE visualization (Fig. [4b](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs13059-017-1349-1/MediaObjects/13059_2017_1349_Fig4_HTML.gif)) show that some tumor types samples can be clearly distinguished from other (AML, KIRC, SKCM, BRCA) and for some, it is more difficult (STAD, COAD, BLCA). This fact is not directly commented on by the authors. This result partially confirms our hypothesis that tissue type can play a role in the immune signal detectability. 

**Deep deconvolution ** should also be discussed. @Shen-Orr2013 defines it as (in the context of PBMC):

> accurately estimating from whole blood or PBMC sample, the proportions and expression patterns of a higher number of cell subsets, going further down into the hematopoietic tree (T-regs, naive, memory, and effector cell subsets).


Many authors [@Newman2015; @He2018; @Aran2017] claimed to be able to distinguish cell subtypes from transcriptome. My results say that data-driven unsupervised deconvolution is not suited to accomplish this task. As shown in Chapter \@ref(map), I dissected the functional subgroups of the NK and Macrophages. Starting with single-cell resolution, I was able to detect groups of cells within studied cell types with distinct functional phenotype. Limited by the number of cells, the number of cell states was limited to two. 

In Chapter \@ref(results), I decomposed five **single-cell cancer transcriptomes**: Melanoma, CRC, Breast, Liver, and Head and Neck.  Applying ICA decomposition, I did not detect cell-subtypes. However, I identified cell types and common signals: cell-cycle, stress, etc. This shows the limits of the blind deconvolution methods for sub-types detection. For Liver T-cell, ICA did not detect T cell subsets. In the article describing the data [@Zheng2017] authors also did not detect a strong cell subtype effect but clusters that are shared by subtypes (Fig. 2A [@Zheng2017]). It is possible that gene expression profiles of immune cell sub-types are not strong enough to be detected in an unsupervised setup. ICA discovers cellular states that do not always overlap with established cell types or subtypes.  

## Data sources 

The overall diversity of resources should also be discussed. As mentioned in Chapter \@ref(intro),  TCGA is the most widely used resource in pan-cancer omics studies. Main reasons are: 

* accessibility
* multilevel data for the same patient
* many cancer types
* big number of patients

Studies integrating many data sources [@Kairov2017; @Biton2014] of different cancer types are less frequent as an additional effort is necessary, and even then the removal of technical biases is not guaranteed [@Battke2011]. Usually, data integration poses an essential challenge in transcriptomic studies[@Battke2011]. Different tools like Combat [@Johnson2007] were developed to overcome this issue. 

In hoped to overcome this issue including datasets generated with different platforms and by different research groups. In this work, I integrated data sets of published by different authors and from different technologies RNA-seq and Microarray.  TCGA data were analyzed together with a corpus of datasets shared with us by AurÃ©lien de ReynÃ¨s and publicly available data.  This integration was slightly less successful for the scRNA-seq as the extracted T-cell components were more correlated with each other than with any bulk-extracted components. It can be due to the different data distribution. Many tools were developed to correct single cell data for confounding effects [@Wang2017; @Kharchenko2014; @Yuan2017]. Thanks to ICA, we can get rid of most of the batch effects as they can be discovered as an independent factor. In [@Biton2014], the authors detected a particular batch effect thanks to one of the non-biological components.  Therefore, **ICA proposes a unique working framework that allowed me to compare signals from independently produced datasets in a single study without a need for re-normalization of the data**. It was possible thanks to the use of ICA that removes the significant batch effects and the fact that the extracted from different datasets ICA components are comparable.

## Missing time and space dimensions in transcriptomic studies

Despite the best research effort, the pan-cancer studies are missing some critical elements: time and space dimensions.

Most studies are based on omic data of tumor biopsies, which do not allow spatial localization of gathered information. In TCGA project pathology slides are available for a subset of patients. However, it is impossible to project the cell markers on them a posteriori or obtain gene expression from a selected area. Many studies, including [@Bindea2011; @Galon2016; @Dieu-Nosjean2014; @Sautes-Fridman2016] demonstrated that the presence of the immune cells at the invasive margin versus tumor core, or the adjacent tertiary lymphoid structures (TLS) could have a different impact on the tumor evolution and patients response to prognosis. 

It is possible to estimate the abundance of some immune cells based on pathological images and also relate the patterns of cells to the patients' survival [@Saltz2018]. This work can be done with an algorithm or by a pathologist.  However, we cannot learn anything about the nature of the cells, cell state or functional subtype from the pathological images.

Therefore, even though most of the immunophenotyping works [@Charoentong2017; @Angelova2015; @Thorsson2018] neglect this aspect, it is important to remember that even if we assume that we can correctly estimate cell type abundance in the sample, its impact can be confounded with spatial information. 

There is a hope that with an evolution of spatial transcriptomic [@Stahl2016; @Svensson2018; @Svensson2018b; @Moncada2018] this gap in our knowledge will be filled in a near future.

Another crucial missing in most studies dimension is time. **The immune system is a highly dynamic system**. Immune cells secrete various molecules depending on stimulants coming from the surrounding tissue (endothelial cells, blood, and lymphatic vessels), tumor cells and other immune cells. Different stimuli can have additive, suppressive of synergistic effects, the order of stimuli can also matter [@Touzot2014]. @Bergthaler2017 compare the network of immune interactions to a social network based on collaborations and communication features.

The animal studies can profit from time resolution data, but it is linked with a sacrifice of the animal. In human studies, this approach cannot be considered. Thus, most of the patients' samples are a *snapshot* at some time $t$. With sequential biopsies [@Dowlati2001; @Gomez-Roca2012; @Roh2017], it is possible to have several time points and observe a sort of dynamic nature of TME. Unfortunately, this kind of data is not widely available for many patients, and pan-cancer multi-omics cancer immunophenotyping remains wishful thinking. 

## The place of bulk transcriptomes in single cell era

Finally, I would like to discuss the place of bulk deconvolution in a single cell transcriptomic. As I mentioned before in the introduction (Chapter \@ref(intro)), scRNA-seq data are bringing single-cell resolution to transcriptomics, that do profit cancer immunology research [@Neu2017; @Giladi2018; @Papalexi2017]. This allows studying cell states and re-definition of cell types from gene expression perspective. With few cancer singe cell datasets, the scientific community learned a lot about patients and cell heterogeneity. The single cell signatures are already used in bulk deconvolution. In my work, I showed that immune cells signature extracted from a single cell are closely related to the signature extracted from the bulk transcriptomes. Due to technical challenges, scRNA-seq is not so far applied to large patient cohorts [@Seymour2017; @Levitin2018]. It was also observed that rare cell types are not trivial to capture even with single-cell technologies and new approaches are being developed for this purpose [@Torre2018]. Even though scRNA-seq may one day replace the bulk RNA-seq, it will not be immediate. Till then both technologies should be used to cross-confirm findings and advance the state of our knowledge.

## The field of computational cancer immunology

Despite continuous advances in research, our knowledge is limited on how tumor and immune cells interact with each other and how much this ecosystem is depending on intrinsic and extrinsic factors. The interest in the TME increased significantly over the last twenty years based on the percentage of publications dedicated to the TME. This was due mostly due to the vital breakthrough of immunotherapies. **Medical advances become a motivation for many projects, mine included, to be founded and perform fundamental or applied for work on the TME. **Many fundamental questions remain unanswered or controversial in the field [@Zhang2018]. For example, the cell-type definition is an open issue that leads to multiple interpretations [@EdiorialCellSystems2017]. Also, the role of different compartments of TME is now considered as context-dependent [@Janssen2017] which means that it is difficult to infer a clinical-level conclusion and prognosis with heterogeneous patients.  The putative predictive/explanatory still await large cohort studies followed by independent validation

As discussed briefly in the Chapter \@ref(intro), researchers produce more and more data [@Angerer2017; @Maia2017], on different scales: from molecule specific to system level which does not always directly leads to a generation of knowledge. **Biological scientists need to join their efforts with analysts (mathematicians, physicists, engineers and computer scientists) to better exploit the available data, to generate the data in a smart way which would improve our understanding of complex biological systems**. 

Thanks to a multi-level transversal analysis of available data a few recent classifications of cancers based on TME features were proposed [@Bindea2011; @Charoentong2017; @Thorsson2018; @Tamborero2018]. **It remains unclear how these classifications can be brought to clinical practice in a near future and what the real impact of those studies will be on patients' survival, diagnosis, and treatment selection.** The descriptive character of the immunophenotyping lack simplicity (combination of various machine-learning-derived scores and knowledge-based curation) and it is not guaranteed to be universal (applicable to different cohorts, technologies). 

Possible research directions will be discussed in perspectives.

# Conclusions  and perspectives {#conclusions}



## Conclusions

This thesis described methods and results of applying unsupervised deconvolution to bulk omic data to extract cell-type specific signals. 

The first contribution of this thesis is the review of deconvolution tools, including very recent ones that illustrate the diversity of the approaches to the bulk transcriptome deconvolution problems. 

The second contribution is the work on methodological aspects of ICA deconvolution. I participated in the definition of Most Stable Transcriptomic Dimension (MSTD) index, and I redefined the way to apply ICA to extract cell-type related signals (overdecomposition) best. I demonstrated that ICA-based signals are reproducible in breast cancer and that the interpretability of ICA is higher than *brunet* version of NMF. 

The third contribution is the DeconICA method for omic data deconvolution through immune components and the R package published online. DeconICA allows detection of immune cell-type signals from tumor bulk data and quantification of their abundance. The tool is not limited to ICA-based decomposition interpretation and can be easily used with different metagene generating methods. The R package has extensive documentation and tutorials that help the user to use the method autonomously. The performance of the DeconICA was evaluated in PBMC transcriptome and concluded to obtain better performance for extraction of some of the cell-type signals than the state-of-the-art published methods.

The fourth contribution is the pan-cancer DeconICA deconvolution study in which signals from 119 datasets of the bulk transcriptome, of 30 tumor types, with a total of 26561 samples and additional five scRNA-seq datasets from five different tumor types were analyzed. On the one hand, the ongoing analysis highlighted detection limits of immune cell signals in some tumor types. On the other hand, I focused on T-cell signal analysis which revealed that there is a heterogeneity of t-cell signals that was identified in T-cell signatures extracted from bulk transcriptome. Further analyses will show if this diversity has a link with patient survival or impacts other immune populations.

Finally, I contributed to a study of heterogeneity of NK and Macrophages based on scRNA-seq transcriptomic data illustrating distinct cell states of the mentioned cell types revealed thanks to a new resource: Innate immune map (of the tumor microenvironment).

## Perspectives

Hopefully, the achievements and findings of the thesis will not finish with the Ph.D. project itself. Many directions can be employed to continue presented work. 

In the first place, the DeconICA package can still be improved. Actual compatibility of the tool with other BSS/Matrix factorization methods should be illustrated with examples, and future adjustments can be integrated into the R package. The reference signatures (for cell types and biological processes) can be extended with new signatures, i.e., based on single-cell technology if proven to bring a better interpretation to bulk decompositions. A graphical web-based interface could be a real added value and should be realized in the near future.  The applicability of DeconICA to other data types, for instance, methylome is to be demonstrated.

There is a wide array of possibilities on how the analysis of ICA-based deconvoluted immune landscape can be continued. The ways I consider to be employed before journal publication of the results are:

* incorporation of clinical and survival data (when available): test for correlations between clinical features and immune cell infiltration, compare survival of patients with high and low infiltrate of different immune cell types
* better study signal reproducibility in different tumor types, better understand why in some tumor types extracted cell-type signals are closer to the reference than others. 
* analysis of the diversity of Myeloid cells, B cells, CAFs, mast cells
* study of the relationship of immune signatures and cell cycle using bulk and single cell data

In a long-term perspective, the possible biological findings resulting from this work, concerning a gene or a set of genes, that would novel in the cancer immunity context could be validated *in vitro* through our partnership with the team of Vassili Soumelis.

From the more general point of view, this work could be extended in a multi-omic manner.  Many groups proposed ways to combine multilevel data. Would the analysis of the immune infiltrates be more meaningful if other data types were used simultaneously?

The primary constraints for all algorithms applied to biological data are the amount of data (efficiency of the algorithm) and the course of dimensionality (large p, small n ) [@Angerer2017; @Maia2017].  Different data types can have specific difficulties (sparsity, missing values, drop out). Therefore in the multi-omics integration, one needs to cope with all the constraints of different data types simultaneously and the integration problem itself. 

One possibility is to employ the tensor decomposition that allows simultaneous decomposition of multidimensional matrices [@Teschendorff2018; @Taguchi2017] (called orders in tensors jargon). Also, joint NMF was proposed to factorize omic data [@Zhang2012]. *Late* integration can also be considered: applying algorithms to multi-omic data independently and integrate them *a posteriori*, through a consensus [@Bonnet2015]. Many methods were developed for multi-omics integration [@Huang2017], little literature is available on the profit of multi-omics integration on the extraction of immune-related signals from bulk cancer data. 

A significant constraint of unsupervised approaches is the need to use data including high variability and therefore, many samples. In theory, it could be possible to compute values for a new single sample given a space established by other samples. In practice, the values predicted for the new samples should be carefully verified for possible biases.

Finally, the blind deconvolution approaches can be applied to detect different signals from diverse tissues. For their interpretation adequate reference profiles or known signatures are necessary. Also for single-cell data, blind deconvolution can be a powerful tool to unveil new cell states.
