<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="summary.html">
<link rel="next" href="abbreviations.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li><a href="index.html#unsupervised-deconvolution-of-bulk-omics-profiles-methodology-and-application-to-characterize-the-immune-landscape-in-tumors"><strong>Unsupervised deconvolution of bulk omics profiles: methodology and application to characterize the immune landscape in tumors</strong></a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="tcga-study-abbreviations.html"><a href="tcga-study-abbreviations.html"><i class="fa fa-check"></i>TCGA Study Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-ph.d.---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary Ph.D. - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-ph.d.topic"><i class="fa fa-check"></i>The origins of the Ph.D. topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="mstd.html"><a href="mstd.html#context"><i class="fa fa-check"></i><b>3.1</b> Context</a></li>
<li class="chapter" data-level="3.2" data-path="mstd.html"><a href="mstd.html#description"><i class="fa fa-check"></i><b>3.2</b> Description</a></li>
<li class="chapter" data-level="3.3" data-path="mstd.html"><a href="mstd.html#impact-on-the-further-work"><i class="fa fa-check"></i><b>3.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a><ul>
<li class="chapter" data-level="4.1" data-path="lva.html"><a href="lva.html#context-1"><i class="fa fa-check"></i><b>4.1</b> Context</a></li>
<li class="chapter" data-level="4.2" data-path="lva.html"><a href="lva.html#description-1"><i class="fa fa-check"></i><b>4.2</b> Description</a></li>
<li class="chapter" data-level="4.3" data-path="lva.html"><a href="lva.html#impact-on-the-further-work-1"><i class="fa fa-check"></i><b>4.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="nmfica.html"><a href="nmfica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.1" data-path="nmfica.html"><a href="nmfica.html#comparing-metagenes-obtained-with-nmf-versus-ica"><i class="fa fa-check"></i><b>5.1</b> Comparing metagenes obtained with NMF versus ICA</a></li>
<li class="chapter" data-level="5.2" data-path="nmfica.html"><a href="nmfica.html#summary-4"><i class="fa fa-check"></i><b>5.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#unsupervised-deconvolution"><i class="fa fa-check"></i><b>6.2</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#fastica-overdecomposition-protocol"><i class="fa fa-check"></i><b>6.2.1</b> FastICA overdecomposition protocol</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="deconica.html"><a href="deconica.html#interpretation-of-the-components"><i class="fa fa-check"></i><b>6.3</b> Interpretation of the components</a></li>
<li class="chapter" data-level="6.4" data-path="deconica.html"><a href="deconica.html#computing-the-abundance-of-the-identified-cell-types"><i class="fa fa-check"></i><b>6.4</b> Computing the abundance of the identified cell-types</a></li>
<li class="chapter" data-level="6.5" data-path="deconica.html"><a href="deconica.html#validation-of-the-abundance-estimation-with-deconica"><i class="fa fa-check"></i><b>6.5</b> Validation of the abundance estimation with <em>DeconICA</em></a><ul>
<li class="chapter" data-level="6.5.1" data-path="deconica.html"><a href="deconica.html#in-silico"><i class="fa fa-check"></i><b>6.5.1</b> <em>In silico</em></a></li>
<li class="chapter" data-level="6.5.2" data-path="deconica.html"><a href="deconica.html#in-vitro"><i class="fa fa-check"></i><b>6.5.2</b> <em>In vitro</em></a></li>
<li class="chapter" data-level="6.5.3" data-path="deconica.html"><a href="deconica.html#pbmc-transcriptome"><i class="fa fa-check"></i><b>6.5.3</b> PBMC transcriptome</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="deconica.html"><a href="deconica.html#summary-5"><i class="fa fa-check"></i><b>6.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Landscape of tumor immunity, with focus on T-cells diversity, uncovered via unsupervised deconvolution applied to bulk transcriptomes across 30 tumor types</a><ul>
<li class="chapter" data-level="7.1" data-path="results.html"><a href="results.html#background"><i class="fa fa-check"></i><b>7.1</b> Background</a></li>
<li class="chapter" data-level="7.2" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>7.2</b> Results</a><ul>
<li class="chapter" data-level="7.2.1" data-path="results.html"><a href="results.html#unsupervised-deconvolution-of-119-datasets"><i class="fa fa-check"></i><b>7.2.1</b> Unsupervised deconvolution of 119 datasets</a></li>
<li class="chapter" data-level="7.2.2" data-path="results.html"><a href="results.html#interpretation-of-the-immune-components"><i class="fa fa-check"></i><b>7.2.2</b> Interpretation of the immune components</a></li>
<li class="chapter" data-level="7.2.3" data-path="results.html"><a href="results.html#quality-of-cell-type-signal-extraction-is-affected-by-tumor-type"><i class="fa fa-check"></i><b>7.2.3</b> Quality of cell-type signal extraction is affected by tumor type</a></li>
<li class="chapter" data-level="7.2.4" data-path="results.html"><a href="results.html#deconvoluted-bulk-transcriptome-t-cell-signatures-showcase-different-cell-states-in-cancers"><i class="fa fa-check"></i><b>7.2.4</b> Deconvoluted bulk transcriptome T cell signatures showcase different cell states in cancers</a></li>
<li class="chapter" data-level="7.2.5" data-path="results.html"><a href="results.html#single-cell-rnaseq-deconvolution-leads-to-cell-specific-metagenes"><i class="fa fa-check"></i><b>7.2.5</b> Single-cell RNAseq deconvolution leads to cell-specific metagenes</a></li>
<li class="chapter" data-level="7.2.6" data-path="results.html"><a href="results.html#landscape-of-patient-based-on-the-immune-infiltration-scores"><i class="fa fa-check"></i><b>7.2.6</b> Landscape of patient based on the immune infiltration scores</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="results.html"><a href="results.html#discussion"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
<li class="chapter" data-level="7.4" data-path="results.html"><a href="results.html#resconcl"><i class="fa fa-check"></i><b>7.4</b> Conclusions</a></li>
<li class="chapter" data-level="7.5" data-path="results.html"><a href="results.html#methods-1"><i class="fa fa-check"></i><b>7.5</b> Methods</a><ul>
<li class="chapter" data-level="7.5.1" data-path="results.html"><a href="results.html#data-sources"><i class="fa fa-check"></i><b>7.5.1</b> Data sources</a></li>
<li class="chapter" data-level="7.5.2" data-path="results.html"><a href="results.html#data-pre-processing"><i class="fa fa-check"></i><b>7.5.2</b> Data pre-processing</a></li>
<li class="chapter" data-level="7.5.3" data-path="results.html"><a href="results.html#data-analysis"><i class="fa fa-check"></i><b>7.5.3</b> Data analysis</a></li>
<li class="chapter" data-level="7.5.4" data-path="results.html"><a href="results.html#software"><i class="fa fa-check"></i><b>7.5.4</b> Software</a></li>
<li class="chapter" data-level="7.5.5" data-path="results.html"><a href="results.html#hardware"><i class="fa fa-check"></i><b>7.5.5</b> Hardware</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="results.html"><a href="results.html#supplementary"><i class="fa fa-check"></i><b>7.6</b> Supplementary</a><ul>
<li class="chapter" data-level="7.6.1" data-path="results.html"><a href="results.html#figures"><i class="fa fa-check"></i><b>7.6.1</b> Figures</a></li>
<li class="chapter" data-level="7.6.2" data-path="results.html"><a href="results.html#tables"><i class="fa fa-check"></i><b>7.6.2</b> Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> A multiscale signalling network map of innate immune response in cancer reveals signatures of cell heterogeneity and functional polarization</a><ul>
<li class="chapter" data-level="8.1" data-path="map.html"><a href="map.html#context-2"><i class="fa fa-check"></i><b>8.1</b> Context</a></li>
<li class="chapter" data-level="8.2" data-path="map.html"><a href="map.html#description-2"><i class="fa fa-check"></i><b>8.2</b> Description</a></li>
<li class="chapter" data-level="8.3" data-path="map.html"><a href="map.html#discussion-and-perspectives"><i class="fa fa-check"></i><b>8.3</b> Discussion and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussiongenerale.html"><a href="discussiongenerale.html"><i class="fa fa-check"></i><b>9</b> Discussion</a><ul>
<li class="chapter" data-level="9.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#improving-our-knowledge-of-how-best-apply-ica-to-transcriptomic-data"><i class="fa fa-check"></i><b>9.1</b> Improving our knowledge of how best apply ICA to transcriptomic data</a></li>
<li class="chapter" data-level="9.2" data-path="discussiongenerale.html"><a href="discussiongenerale.html#confronting-ica-with-other-matrix-factorization-approaches"><i class="fa fa-check"></i><b>9.2</b> Confronting ICA with other matrix factorization approaches</a></li>
<li class="chapter" data-level="9.3" data-path="discussiongenerale.html"><a href="discussiongenerale.html#deconvolution-methods-overall-impact-and-utility"><i class="fa fa-check"></i><b>9.3</b> Deconvolution methods overall impact and utility</a></li>
<li class="chapter" data-level="9.4" data-path="discussiongenerale.html"><a href="discussiongenerale.html#unsupervised-deconvolution-1"><i class="fa fa-check"></i><b>9.4</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="9.4.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#limitations-of-unsupervised-deconvolution"><i class="fa fa-check"></i><b>9.4.1</b> Limitations of unsupervised deconvolution</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="discussiongenerale.html"><a href="discussiongenerale.html#data-sources-1"><i class="fa fa-check"></i><b>9.5</b> Data sources</a></li>
<li class="chapter" data-level="9.6" data-path="discussiongenerale.html"><a href="discussiongenerale.html#missing-time-and-space-dimensions-in-transcriptomic-studies"><i class="fa fa-check"></i><b>9.6</b> Missing time and space dimensions in transcriptomic studies</a></li>
<li class="chapter" data-level="9.7" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-place-of-bulk-transcriptomes-in-single-cell-era"><i class="fa fa-check"></i><b>9.7</b> The place of bulk transcriptomes in single cell era</a></li>
<li class="chapter" data-level="9.8" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-field-of-computational-cancer-immunology"><i class="fa fa-check"></i><b>9.8</b> The field of computational cancer immunology</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a><ul>
<li class="chapter" data-level="10.1" data-path="conclusions.html"><a href="conclusions.html#conclusions-1"><i class="fa fa-check"></i><b>10.1</b> Conclusions</a></li>
<li class="chapter" data-level="10.2" data-path="conclusions.html"><a href="conclusions.html#perspectives"><i class="fa fa-check"></i><b>10.2</b> Perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#deconica-documentation"><i class="fa fa-check"></i>1 DeconICA documentation</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#introduction-to-deconica"><i class="fa fa-check"></i>1.1 Introduction to deconICA</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#running-fastica-with-icasso-stabilisation"><i class="fa fa-check"></i>1.2 Running fastICA with icasso stabilisation</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#reference-manual"><i class="fa fa-check"></i>1.3 Reference manual</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#publications-and-conferences"><i class="fa fa-check"></i>2 Publications and conferences</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#adjustment-of-dendritic-cells-to-the-breast-cancer-microenvironment-is-subset-specific"><i class="fa fa-check"></i>2.1 Adjustment of dendritic cells to the breast-cancer microenvironment is subset-specific</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#the-inconvenience-of-data-of-convenience-computational-research-beyond-post-mortem-analyses"><i class="fa fa-check"></i>2.2 The inconvenience of data of convenience: computational research beyond post-mortem analyses</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv-publications-conferences-courses"><i class="fa fa-check"></i>2.3 CV: publications, conferences, courses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a><ul>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#biological-terms"><i class="fa fa-check"></i>Biological terms</a></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#mathematical-terms"><i class="fa fa-check"></i>Mathematical terms</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="acknowledgements" class="section level1 unnumbered">
<h1>Acknowledgements</h1>
<p>I would like to thank my supervisors Andrei Zinovyev and Vassili Soumelis for guiding this project and enabling me to interact with their teams and sharing the resources.
I would also like to thank the U900 lab and his head Emmanuel Barillot to generously equip me with the professional environment, the place and the tools.</p>
<p>I address my gratitude to the TAC committee members Franck Pagès and Denis Thieffry for helping me organizing the jury and giving constructive comments along with my thesis, for being present, at least remotely despite severe weather conditions or travels.</p>
<p>I would also like to express gratitude towards the jury for taking the time to assess this work.</p>
<p>This is also the place to thank the ITMO Cancer - AVIESAN for funding my Ph.D. scholarship and the Pharmacology Faculty of Paris Descartes and specifically Prof. Chantal Guihenneuc for giving me the opportunity to teach in parallel of my Ph.D. as a part of her pedagogical team. I would also thank a lot Center of Interdisciplinary Research for equipping me with unusual skills through numerous courses and Bettencourt Foundation for financing part of the training and sponsoring travel expenses. Special thanks to FdV coordinators: Sofie Leon, David Manset, Elodie Kaslikowski and Maria Molina Calvita for their availability and dynamism. Also, I would express my gratitude for supporting my application for French nationality to François Taddei, director of the FdV Ph.D. school.</p>
<p>Thanks to all people I worked with in both teams: Arnau, Pauline, Gaelle, Paul, Cristobal, Luca, Laura, Laurence, Loredana, Jonas, Floriane, Maude, Philemon, Lilith, Paula, Mihaly, Louis, Caroline. To my FdV mates: Roberta, Juanma, Miza, Guillermo, Aamir and others. To other Ph.D. students of the unit I got along with: Peter, Jo, Hector and Benoît.</p>
<p>This work would never be possible without help and patience of my family, my partner Arnaud and his family. Especially, I would like to thank Arnaud, who managed to be with me on the daily basis, spent the endless hours correcting my writing and speaking, discussed about the code and good practices, was making me laugh when I was coming home tired, angry or unmotivated, and just for being him adorable self. This thesis is dedicated to his father that will not be able anymore to profit from any breakthrough in cancer research.</p>
<p>I am proud to finish the thesis and face new professional adventures. I learned a lot about myself during this three years. I would like to thank very much everyone whom I crossed on this path. I hope to meet you again one day.</p>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="summary.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="abbreviations.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/index.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
