<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="lva.html">
<link rel="next" href="deconica.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; min-height: 1.25em; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; }
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
@media screen {
a.sourceLine::before { text-decoration: underline; color: initial; }
}
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.bn { color: #40a070; } /* BaseN */
code span.fl { color: #40a070; } /* Float */
code span.ch { color: #4070a0; } /* Char */
code span.st { color: #4070a0; } /* String */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.ot { color: #007020; } /* Other */
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.fu { color: #06287e; } /* Function */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code span.cn { color: #880000; } /* Constant */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.ss { color: #bb6688; } /* SpecialString */
code span.im { } /* Import */
code span.va { color: #19177c; } /* Variable */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.op { color: #666666; } /* Operator */
code span.bu { } /* BuiltIn */
code span.ex { } /* Extension */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.at { color: #7d9029; } /* Attribute */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i><b>1</b> Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-phd---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary PhD - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-phd-topic"><i class="fa fa-check"></i>The origins of the PhD topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="2" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>2</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="2.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>2.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="2.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>2.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="2.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>2.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>2.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="2.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>2.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="2.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>2.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="2.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>2.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>2.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="2.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>2.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="2.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>2.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="2.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>2.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="2.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>2.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="2.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>2.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>2.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>3</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="3.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>3.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="3.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>3.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="3.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>3.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="3.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>3.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>3.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="3.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>3.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="3.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>3.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="3.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>3.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="3.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>3.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="3.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>3.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="3.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>3.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="3.3.6" data-path="methods.html"><a href="methods.html#matrix-factorisation-methods"><i class="fa fa-check"></i><b>3.3.6</b> Matrix factorisation methods</a></li>
<li class="chapter" data-level="3.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>3.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="3.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>3.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="3.3.9" data-path="methods.html"><a href="methods.html#summary-2"><i class="fa fa-check"></i><b>3.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>3.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="3.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>3.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="3.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>3.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>3.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="4" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>4</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a></li>
<li class="chapter" data-level="5" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>5</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a></li>
<li class="chapter" data-level="6" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html"><i class="fa fa-check"></i><b>6</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="6.0.1" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html#comparing-metagenes-obtained-with-nmf-vs-ica."><i class="fa fa-check"></i><b>6.0.1</b> Comparing metagenes obtained with NMF vs ICA.</a></li>
<li class="chapter" data-level="6.1" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html#impact-of-modification-of-signatures-list-on-result-for-signature-based-deconvolution-methods"><i class="fa fa-check"></i><b>6.1</b> ? Impact of modification of signatures list on result for signature-based deconvolution methods</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>7</b> Deconvolution of transcriptomes and methylomes</a><ul>
<li class="chapter" data-level="7.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>7.1</b> From blind deconvolution to cell-type quantification: general overview</a><ul>
<li class="chapter" data-level="7.1.1" data-path="deconica.html"><a href="deconica.html#the-ica-based-deconvolution-of-transcriptomes"><i class="fa fa-check"></i><b>7.1.1</b> The ICA-based deconvolution of Transcriptomes</a></li>
<li class="chapter" data-level="7.1.2" data-path="deconica.html"><a href="deconica.html#interpretation-of-independent-components"><i class="fa fa-check"></i><b>7.1.2</b> Interpretation of Independent components</a></li>
<li class="chapter" data-level="7.1.3" data-path="deconica.html"><a href="deconica.html#transforming-metagenes-into-signature-matrix"><i class="fa fa-check"></i><b>7.1.3</b> Transforming metagenes into signature matrix</a></li>
<li class="chapter" data-level="7.1.4" data-path="deconica.html"><a href="deconica.html#regression-based-estimation-of-cell-type-proportions-solving-system-of-equations"><i class="fa fa-check"></i><b>7.1.4</b> Regression-based estimation of cell-type proportions : solving system of equations</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="deconica.html"><a href="deconica.html#deconica-r-package-for-ica-based-deconvolution"><i class="fa fa-check"></i><b>7.2</b> <em>DeconICA</em> R package for ICA-based deconvolution</a><ul>
<li class="chapter" data-level="7.2.1" data-path="deconica.html"><a href="deconica.html#demo"><i class="fa fa-check"></i><b>7.2.1</b> <em>Demo</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>8</b> Comparative analysis of cancer immune infiltration</a></li>
<li class="chapter" data-level="9" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>9</b> Heterogeneity of immune cell types</a></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="10" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>10</b> Discussion</a></li>
<li class="chapter" data-level="11" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>11</b> Conclusions and perspectives</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#dc-subtypes"><i class="fa fa-check"></i>Dc subtypes</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#dreamidea-challenge"><i class="fa fa-check"></i>DreamIdea Challenge</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#full-list-of-publications"><i class="fa fa-check"></i>Full list of publications</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv"><i class="fa fa-check"></i>CV</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="comparison-of-reproducibility-between-nmf-and-ica" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Comparison of reproducibility between NMF and ICA</h1>
<p>NMF and ICA are both algorithms often applied to solve blind source deconvolution problem. NMF gained a popularity as a tool of transcriptomic analysis mainly thanks to the publications [publicaition_list]. However, the non-negativity contraint, an attractive concept in the case of non-negative transcriptome counts, may be a reason why the reusults of NMF decomposition are not the best candidate for our deconvolution task. We observed that NMF-based metagenes are less reprouctible between different transcriptomic datasets than ICA-based metagenes.</p>
<div id="comparing-metagenes-obtained-with-nmf-vs-ica." class="section level3">
<h3><span class="header-section-number">6.0.1</span> Comparing metagenes obtained with NMF vs ICA.</h3>
<p>We compared the reproducibility of NMF and ICA through decomposition of four breast cancer datasets (BRCATCGA, METABRIC, BEK, WAN)[ref]. Those datasets were selected because of their size (number of samples &gt; 50) and because they were available in not centred format necessary for NMF.</p>
<p>For NMF the procedure was following:</p>
<ul>
<li>data was transformed into log2</li>
<li>zero rows were removed</li>
<li>the algorithm assesing cophentic index was applied to chose optimal number of components</li>
<li>datasets were decomposed with matlab NMF implementation from Brunet et al. <span class="citation">(<span class="citeproc-not-found" data-reference-id="Brunet"><strong>???</strong></span>)</span> into (i) number of components suggested by cophenetic coefficient (ii) MSTD dimension (iii) 50 components (approaching overdecomposition)</li>
<li>the obtained metagenes were decorellated from the mean using a linear regression model</li>
</ul>
<p>For ICA, the procedure was following:</p>
<ul>
<li>data were transformed into log2</li>
<li>transformed data were mean-centered by gene</li>
<li>our implementation of MSTD (most stable transcriptomic dimension) from <span class="citation">(Kairov et al. <a href="#ref-Kairov2017">2017</a>)</span> was used to evaluate most stable dimension</li>
<li>datasets were demposed into (i) MSTD dimension and (ii) 50 components (approaching overdecomposition) with matlab implementantion of fastICA with icasso stabilisation</li>
</ul>
<p>We did not decompose ICA into low number of components as we consider it as strong underdecomposition and we suspect signals would not be the most reproducible. We limited the over decomposition higher than 50 with NMF as for our biggest dataset (METABRIC) NMF decomposition into 50 took 30245 minutes (3 weeks).</p>
<p>Then separately for NMF and ICA, we correlated all obtained metagens with each other and with known Biton et al. metagenes (obtain from previous ICA decompostion applied pan-cancer). We represented the results in a form of a correlation graph where nodes are metagenes from different datasets and decompostion levels and edge width corresponds peasron correaltion coefficients (Fig <a href="comparison-of-reproducibility-between-nmf-and-ica.html#fig:icavsnmf">6.1</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:icavsnmf"></span>
<img src="figures-ext/ICANMF.png" alt="Correlation graph of ICA and NMF multiple decompositions. In the upper part of the figure (A,B) we observe the correlation graph of all metagenes (ICA or NMF-based) disposed using edge-weighted bio layout. In the lower part of the figure (C,D) we applied &gt;0.4 thereshold in order to filter the edges. In the case of ICA (C), remaining nodes form pseudo-cliques, immune-related pseudo-clique is highlighted. In the case of NMF (D), components cluster by dataset. Edges’ width coressponds to Pearson correlation coefficient. Node colors correspond to dataset from which a metagene was obtained (see legend)." width="100%" />
<p class="caption">
Figure 6.1: <strong>Correlation graph of ICA and NMF multiple decompositions.</strong> In the upper part of the figure (A,B) we observe the correlation graph of all metagenes (ICA or NMF-based) disposed using edge-weighted bio layout. In the lower part of the figure (C,D) we applied &gt;0.4 thereshold in order to filter the edges. In the case of ICA (C), remaining nodes form pseudo-cliques, immune-related pseudo-clique is highlighted. In the case of NMF (D), components cluster by dataset. Edges’ width coressponds to Pearson correlation coefficient. Node colors correspond to dataset from which a metagene was obtained (see legend).
</p>
</div>

<p>We hoped to observe a subset of components from different datasets (no matter the decomposition level) correlate with each strongly and much less with other components in order to confirm that the signal is reproducible (can be found in several dataset) and specific. We used the Biton et al. componets here to help with eventual identification of signals (labelling). What we observe from ICA-decomposition that indeed, without applying any threshold some emerging clusters can be remarked and after application of &gt;0.4 threshold on the correlation coeffcient pseudo-cliques emerge. While metagenes from NMF-decpmpostion are more tighlty connected globally and when the threshold is applied, remaing metagenes do not form clear clusters but group by data set. In NMF decomposition if it hard to define different signals as the datasets seem to be all related to each other. We can see from (Fig <a href="comparison-of-reproducibility-between-nmf-and-ica.html#fig:icavsnmf">6.1</a>D) that the IMMUNE signal is correlated &gt;0.4 with a high number of NMF components that are also linked to some other components. In ICA (Fig <a href="comparison-of-reproducibility-between-nmf-and-ica.html#fig:icavsnmf">6.1</a>C) components related to the IMMUNE metagens form a pseudo-clique that is related with one link to INTERFERON metagene.</p>
<p>This simple analysis illustrates that NMF applied to cancer transcriptomes decomposes them to metagenes that are not highly reproductible between datasets. In practice, it will not always be possible to work with big cohorts and the same processing methods. Using ICA for decomposition gives mor credit that it will be possible to use the obtained metagenes as reference in which new data of similar type could be projected.</p>
<p><em>to do:</em></p>
<ul>
<li><p><em>quantify: with clustering coefficient?</em></p></li>
<li><p>Explain why ICA is more reproducible</p></li>
</ul>
</div>
<div id="impact-of-modification-of-signatures-list-on-result-for-signature-based-deconvolution-methods" class="section level2">
<h2><span class="header-section-number">6.1</span> ? Impact of modification of signatures list on result for signature-based deconvolution methods</h2>
<p>Carry on a “sensitivity study”:</p>
<ul>
<li>remove some % of genes from basis matrix or marker gene list</li>
<li>evaluate how it changes results</li>
</ul>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Kairov2017">
<p>Kairov, Ulykbek, Laura Cantini, Alessandro Greco, Askhat Molkenov, Urszula Czerwinska, Emmanuel Barillot, and Andrei Zinovyev. 2017. “Determining the optimal number of independent components for reproducible transcriptomic data analysis.” <em>BMC Genomics</em> 18 (1). BioMed Central: 712. doi:<a href="https://doi.org/10.1186/s12864-017-4112-9">10.1186/s12864-017-4112-9</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="lva.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="deconica.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/07-nmfica.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
