<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="discussiongenerale.html">
<link rel="next" href="annexes.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li><a href="index.html#unsupervised-deconvolution-of-bulk-omics-profiles-methodology-and-application-to-characterize-the-immune-landscape-in-tumors"><strong>Unsupervised deconvolution of bulk omics profiles: methodology and application to characterize the immune landscape in tumors</strong></a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="tcga-study-abbreviations.html"><a href="tcga-study-abbreviations.html"><i class="fa fa-check"></i>TCGA Study Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-ph.d.---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary Ph.D. - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-ph.d.topic"><i class="fa fa-check"></i>The origins of the Ph.D. topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="mstd.html"><a href="mstd.html#context"><i class="fa fa-check"></i><b>3.1</b> Context</a></li>
<li class="chapter" data-level="3.2" data-path="mstd.html"><a href="mstd.html#description"><i class="fa fa-check"></i><b>3.2</b> Description</a></li>
<li class="chapter" data-level="3.3" data-path="mstd.html"><a href="mstd.html#impact-on-the-further-work"><i class="fa fa-check"></i><b>3.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a><ul>
<li class="chapter" data-level="4.1" data-path="lva.html"><a href="lva.html#context-1"><i class="fa fa-check"></i><b>4.1</b> Context</a></li>
<li class="chapter" data-level="4.2" data-path="lva.html"><a href="lva.html#description-1"><i class="fa fa-check"></i><b>4.2</b> Description</a></li>
<li class="chapter" data-level="4.3" data-path="lva.html"><a href="lva.html#impact-on-the-further-work-1"><i class="fa fa-check"></i><b>4.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="nmfica.html"><a href="nmfica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.1" data-path="nmfica.html"><a href="nmfica.html#comparing-metagenes-obtained-with-nmf-versus-ica"><i class="fa fa-check"></i><b>5.1</b> Comparing metagenes obtained with NMF versus ICA</a></li>
<li class="chapter" data-level="5.2" data-path="nmfica.html"><a href="nmfica.html#summary-4"><i class="fa fa-check"></i><b>5.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#unsupervised-deconvolution"><i class="fa fa-check"></i><b>6.2</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#fastica-overdecomposition-protocol"><i class="fa fa-check"></i><b>6.2.1</b> FastICA overdecomposition protocol</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="deconica.html"><a href="deconica.html#interpretation-of-the-components"><i class="fa fa-check"></i><b>6.3</b> Interpretation of the components</a></li>
<li class="chapter" data-level="6.4" data-path="deconica.html"><a href="deconica.html#computing-the-abundance-of-the-identified-cell-types"><i class="fa fa-check"></i><b>6.4</b> Computing the abundance of the identified cell-types</a></li>
<li class="chapter" data-level="6.5" data-path="deconica.html"><a href="deconica.html#validation-of-the-abundance-estimation-with-deconica"><i class="fa fa-check"></i><b>6.5</b> Validation of the abundance estimation with <em>DeconICA</em></a><ul>
<li class="chapter" data-level="6.5.1" data-path="deconica.html"><a href="deconica.html#in-silico"><i class="fa fa-check"></i><b>6.5.1</b> <em>In silico</em></a></li>
<li class="chapter" data-level="6.5.2" data-path="deconica.html"><a href="deconica.html#in-vitro"><i class="fa fa-check"></i><b>6.5.2</b> <em>In vitro</em></a></li>
<li class="chapter" data-level="6.5.3" data-path="deconica.html"><a href="deconica.html#pbmc-transcriptome"><i class="fa fa-check"></i><b>6.5.3</b> PBMC transcriptome</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="deconica.html"><a href="deconica.html#summary-5"><i class="fa fa-check"></i><b>6.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Landscape of tumor immunity, with focus on T-cells diversity, uncovered via unsupervised deconvolution applied to bulk transcriptomes across 30 tumor types</a><ul>
<li class="chapter" data-level="7.1" data-path="results.html"><a href="results.html#background"><i class="fa fa-check"></i><b>7.1</b> Background</a></li>
<li class="chapter" data-level="7.2" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>7.2</b> Results</a><ul>
<li class="chapter" data-level="7.2.1" data-path="results.html"><a href="results.html#unsupervised-deconvolution-of-119-datasets"><i class="fa fa-check"></i><b>7.2.1</b> Unsupervised deconvolution of 119 datasets</a></li>
<li class="chapter" data-level="7.2.2" data-path="results.html"><a href="results.html#interpretation-of-the-immune-components"><i class="fa fa-check"></i><b>7.2.2</b> Interpretation of the immune components</a></li>
<li class="chapter" data-level="7.2.3" data-path="results.html"><a href="results.html#quality-of-cell-type-signal-extraction-is-affected-by-tumor-type"><i class="fa fa-check"></i><b>7.2.3</b> Quality of cell-type signal extraction is affected by tumor type</a></li>
<li class="chapter" data-level="7.2.4" data-path="results.html"><a href="results.html#deconvoluted-bulk-transcriptome-t-cell-signatures-showcase-different-cell-states-in-cancers"><i class="fa fa-check"></i><b>7.2.4</b> Deconvoluted bulk transcriptome T cell signatures showcase different cell states in cancers</a></li>
<li class="chapter" data-level="7.2.5" data-path="results.html"><a href="results.html#single-cell-rnaseq-deconvolution-leads-to-cell-specific-metagenes"><i class="fa fa-check"></i><b>7.2.5</b> Single-cell RNAseq deconvolution leads to cell-specific metagenes</a></li>
<li class="chapter" data-level="7.2.6" data-path="results.html"><a href="results.html#landscape-of-patient-based-on-the-immune-infiltration-scores"><i class="fa fa-check"></i><b>7.2.6</b> Landscape of patient based on the immune infiltration scores</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="results.html"><a href="results.html#discussion"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
<li class="chapter" data-level="7.4" data-path="results.html"><a href="results.html#resconcl"><i class="fa fa-check"></i><b>7.4</b> Conclusions</a></li>
<li class="chapter" data-level="7.5" data-path="results.html"><a href="results.html#methods-1"><i class="fa fa-check"></i><b>7.5</b> Methods</a><ul>
<li class="chapter" data-level="7.5.1" data-path="results.html"><a href="results.html#data-sources"><i class="fa fa-check"></i><b>7.5.1</b> Data sources</a></li>
<li class="chapter" data-level="7.5.2" data-path="results.html"><a href="results.html#data-pre-processing"><i class="fa fa-check"></i><b>7.5.2</b> Data pre-processing</a></li>
<li class="chapter" data-level="7.5.3" data-path="results.html"><a href="results.html#data-analysis"><i class="fa fa-check"></i><b>7.5.3</b> Data analysis</a></li>
<li class="chapter" data-level="7.5.4" data-path="results.html"><a href="results.html#software"><i class="fa fa-check"></i><b>7.5.4</b> Software</a></li>
<li class="chapter" data-level="7.5.5" data-path="results.html"><a href="results.html#hardware"><i class="fa fa-check"></i><b>7.5.5</b> Hardware</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="results.html"><a href="results.html#supplementary"><i class="fa fa-check"></i><b>7.6</b> Supplementary</a><ul>
<li class="chapter" data-level="7.6.1" data-path="results.html"><a href="results.html#figures"><i class="fa fa-check"></i><b>7.6.1</b> Figures</a></li>
<li class="chapter" data-level="7.6.2" data-path="results.html"><a href="results.html#tables"><i class="fa fa-check"></i><b>7.6.2</b> Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> A multiscale signalling network map of innate immune response in cancer reveals signatures of cell heterogeneity and functional polarization</a><ul>
<li class="chapter" data-level="8.1" data-path="map.html"><a href="map.html#context-2"><i class="fa fa-check"></i><b>8.1</b> Context</a></li>
<li class="chapter" data-level="8.2" data-path="map.html"><a href="map.html#description-2"><i class="fa fa-check"></i><b>8.2</b> Description</a></li>
<li class="chapter" data-level="8.3" data-path="map.html"><a href="map.html#discussion-and-perspectives"><i class="fa fa-check"></i><b>8.3</b> Discussion and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussiongenerale.html"><a href="discussiongenerale.html"><i class="fa fa-check"></i><b>9</b> Discussion</a><ul>
<li class="chapter" data-level="9.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#improving-our-knowledge-of-how-best-apply-ica-to-transcriptomic-data"><i class="fa fa-check"></i><b>9.1</b> Improving our knowledge of how best apply ICA to transcriptomic data</a></li>
<li class="chapter" data-level="9.2" data-path="discussiongenerale.html"><a href="discussiongenerale.html#confronting-ica-with-other-matrix-factorization-approaches"><i class="fa fa-check"></i><b>9.2</b> Confronting ICA with other matrix factorization approaches</a></li>
<li class="chapter" data-level="9.3" data-path="discussiongenerale.html"><a href="discussiongenerale.html#deconvolution-methods-overall-impact-and-utility"><i class="fa fa-check"></i><b>9.3</b> Deconvolution methods overall impact and utility</a></li>
<li class="chapter" data-level="9.4" data-path="discussiongenerale.html"><a href="discussiongenerale.html#unsupervised-deconvolution-1"><i class="fa fa-check"></i><b>9.4</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="9.4.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#limitations-of-unsupervised-deconvolution"><i class="fa fa-check"></i><b>9.4.1</b> Limitations of unsupervised deconvolution</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="discussiongenerale.html"><a href="discussiongenerale.html#data-sources-1"><i class="fa fa-check"></i><b>9.5</b> Data sources</a></li>
<li class="chapter" data-level="9.6" data-path="discussiongenerale.html"><a href="discussiongenerale.html#missing-time-and-space-dimensions-in-transcriptomic-studies"><i class="fa fa-check"></i><b>9.6</b> Missing time and space dimensions in transcriptomic studies</a></li>
<li class="chapter" data-level="9.7" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-place-of-bulk-transcriptomes-in-single-cell-era"><i class="fa fa-check"></i><b>9.7</b> The place of bulk transcriptomes in single cell era</a></li>
<li class="chapter" data-level="9.8" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-field-of-computational-cancer-immunology"><i class="fa fa-check"></i><b>9.8</b> The field of computational cancer immunology</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a><ul>
<li class="chapter" data-level="10.1" data-path="conclusions.html"><a href="conclusions.html#conclusions-1"><i class="fa fa-check"></i><b>10.1</b> Conclusions</a></li>
<li class="chapter" data-level="10.2" data-path="conclusions.html"><a href="conclusions.html#perspectives"><i class="fa fa-check"></i><b>10.2</b> Perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#deconica-documentation"><i class="fa fa-check"></i>1 DeconICA documentation</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#introduction-to-deconica"><i class="fa fa-check"></i>1.1 Introduction to deconICA</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#running-fastica-with-icasso-stabilisation"><i class="fa fa-check"></i>1.2 Running fastICA with icasso stabilisation</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#reference-manual"><i class="fa fa-check"></i>1.3 Reference manual</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#publications-and-conferences"><i class="fa fa-check"></i>2 Publications and conferences</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#adjustment-of-dendritic-cells-to-the-breast-cancer-microenvironment-is-subset-specific"><i class="fa fa-check"></i>2.1 Adjustment of dendritic cells to the breast-cancer microenvironment is subset-specific</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#the-inconvenience-of-data-of-convenience-computational-research-beyond-post-mortem-analyses"><i class="fa fa-check"></i>2.2 The inconvenience of data of convenience: computational research beyond post-mortem analyses</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv-publications-conferences-courses"><i class="fa fa-check"></i>2.3 CV: publications, conferences, courses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a><ul>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#biological-terms"><i class="fa fa-check"></i>Biological terms</a></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#mathematical-terms"><i class="fa fa-check"></i>Mathematical terms</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="conclusions" class="section level1">
<h1><span class="header-section-number">Chapter 10</span> Conclusions and perspectives</h1>
<div id="conclusions-1" class="section level2">
<h2><span class="header-section-number">10.1</span> Conclusions</h2>
<p>This thesis described methods and results of applying unsupervised deconvolution to bulk omic data to extract cell-type specific signals.</p>
<p>The first contribution of this thesis is the review of deconvolution tools, including very recent ones that illustrate the diversity of the approaches to the bulk transcriptome deconvolution problems.</p>
<p>The second contribution is the work on methodological aspects of ICA deconvolution. I participated in the definition of Most Stable Transcriptomic Dimension (MSTD) index, and I redefined the way to apply ICA to extract cell-type related signals (overdecomposition) best. I demonstrated that ICA-based signals are reproducible in breast cancer and that the interpretability of ICA is higher than <em>brunet</em> version of NMF.</p>
<p>The third contribution is the DeconICA method for omic data deconvolution through immune components and the R package published online. DeconICA allows detection of immune cell-type signals from tumor bulk data and quantification of their abundance. The tool is not limited to ICA-based decomposition interpretation and can be easily used with different metagene generating methods. The R package has extensive documentation and tutorials that help the user to use the method autonomously. The performance of the DeconICA was evaluated in PBMC transcriptome and concluded to obtain better performance for extraction of some of the cell-type signals than the state-of-the-art published methods.</p>
<p>The fourth contribution is the pan-cancer DeconICA deconvolution study in which signals from 119 datasets of the bulk transcriptome, of 30 tumor types, with a total of 26561 samples and additional five scRNA-seq datasets from five different tumor types were analyzed. On the one hand, the ongoing analysis highlighted detection limits of immune cell signals in some tumor types. On the other hand, I focused on T-cell signal analysis which revealed that there is a heterogeneity of t-cell signals that was identified in T-cell signatures extracted from bulk transcriptome. Further analyses will show if this diversity has a link with patient survival or impacts other immune populations.</p>
<p>Finally, I contributed to a study of heterogeneity of NK and Macrophages based on scRNA-seq transcriptomic data illustrating distinct cell states of the mentioned cell types revealed thanks to a new resource: Innate immune map (of the tumor microenvironment).</p>
</div>
<div id="perspectives" class="section level2">
<h2><span class="header-section-number">10.2</span> Perspectives</h2>
<p>Hopefully, the achievements and findings of the thesis will not finish with the Ph.D. project itself. Many directions can be employed to continue presented work.</p>
<p>In the first place, the DeconICA package can still be improved. Actual compatibility of the tool with other BSS/Matrix factorization methods should be illustrated with examples, and future adjustments can be integrated into the R package. The reference signatures (for cell types and biological processes) can be extended with new signatures, i.e., based on single-cell technology if proven to bring a better interpretation to bulk decompositions. A graphical web-based interface could be a real added value and should be realized in the near future. The applicability of DeconICA to other data types, for instance, methylome is to be demonstrated.</p>
<p>There is a wide array of possibilities on how the analysis of ICA-based deconvoluted immune landscape can be continued. The ways I consider to be employed before journal publication of the results are:</p>
<ul>
<li>incorporation of clinical and survival data (when available): test for correlations between clinical features and immune cell infiltration, compare survival of patients with high and low infiltrate of different immune cell types</li>
<li>better study signal reproducibility in different tumor types, better understand why in some tumor types extracted cell-type signals are closer to the reference than others.</li>
<li>analysis of the diversity of Myeloid cells, B cells, CAFs, mast cells</li>
<li>study of the relationship of immune signatures and cell cycle using bulk and single cell data</li>
</ul>
<p>In a long-term perspective, the possible biological findings resulting from this work, concerning a gene or a set of genes, that would novel in the cancer immunity context could be validated <em>in vitro</em> through our partnership with the team of Vassili Soumelis.</p>
<p>From the more general point of view, this work could be extended in a multi-omic manner. Many groups proposed ways to combine multilevel data. Would the analysis of the immune infiltrates be more meaningful if other data types were used simultaneously?</p>
<p>The primary constraints for all algorithms applied to biological data are the amount of data (efficiency of the algorithm) and the course of dimensionality (large p, small n ) <span class="citation">(Angerer et al. <a href="#ref-Angerer2017">2017</a>; Maia et al. <a href="#ref-Maia2017">2017</a>)</span>. Different data types can have specific difficulties (sparsity, missing values, drop out). Therefore in the multi-omics integration, one needs to cope with all the constraints of different data types simultaneously and the integration problem itself.</p>
<p>One possibility is to employ the tensor decomposition that allows simultaneous decomposition of multidimensional matrices <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2018">2018</a>; Taguchi <a href="#ref-Taguchi2017">2017</a>)</span> (called orders in tensors jargon). Also, joint NMF was proposed to factorize omic data <span class="citation">(Zhang et al. <a href="#ref-Zhang2012">2012</a>)</span>. <em>Late</em> integration can also be considered: applying algorithms to multi-omic data independently and integrate them <em>a posteriori</em>, through a consensus <span class="citation">(Bonnet, Calzone, and Michoel <a href="#ref-Bonnet2015">2015</a>)</span>. Many methods were developed for multi-omics integration <span class="citation">(Huang, Chaudhary, and Garmire <a href="#ref-Huang2017">2017</a>)</span>, little literature is available on the profit of multi-omics integration on the extraction of immune-related signals from bulk cancer data.</p>
<p>A significant constraint of unsupervised approaches is the need to use data including high variability and therefore, many samples. In theory, it could be possible to compute values for a new single sample given a space established by other samples. In practice, the values predicted for the new samples should be carefully verified for possible biases.</p>
<p>Finally, the blind deconvolution approaches can be applied to detect different signals from diverse tissues. For their interpretation adequate reference profiles or known signatures are necessary. Also for single-cell data, blind deconvolution can be a powerful tool to unveil new cell states.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Angerer2017">
<p>Angerer, Philipp, Lukas Simon, Sophie Tritschler, F. Alexander Wolf, David Fischer, and Fabian J. Theis. 2017. “Single cells make big data: New challenges and opportunities in transcriptomics.” <em>Curr. Opin. Syst. Biol.</em> 4. Elsevier: 85–91. doi:<a href="https://doi.org/10.1016/J.COISB.2017.07.004">10.1016/J.COISB.2017.07.004</a>.</p>
</div>
<div id="ref-Maia2017">
<p>Maia, Ana-Teresa, Stephen-John Sammut, Ana Jacinta-Fernandes, and Suet-Feung Chin. 2017. “Big data in cancer genomics.” <em>Curr. Opin. Syst. Biol.</em> 4. Elsevier: 78–84. doi:<a href="https://doi.org/10.1016/J.COISB.2017.07.007">10.1016/J.COISB.2017.07.007</a>.</p>
</div>
<div id="ref-Teschendorff2018">
<p>Teschendorff, Andrew E, Han Jing, Dirk S Paul, Joni Virta, and Klaus Nordhausen. 2018. “Tensorial blind source separation for improved analysis of multi-omic data.” <em>Genome Biol.</em> 19: 76. doi:<a href="https://doi.org/10.1186/s13059-018-1455-8">10.1186/s13059-018-1455-8</a>.</p>
</div>
<div id="ref-Taguchi2017">
<p>Taguchi, Y-h. 2017. “One-class Differential Expression Analysis using Tensor Decomposition-based Unsupervised Feature Extraction Applied to Integrated Analysis of Multiple Omics Data from 26 Lung Adenocarcinoma Cell Lines.” In <em>2017 Ieee 17th Int. Conf. Bioinforma. Bioeng.</em>, 131–38. IEEE. doi:<a href="https://doi.org/10.1109/BIBE.2017.00-66">10.1109/BIBE.2017.00-66</a>.</p>
</div>
<div id="ref-Zhang2012">
<p>Zhang, Shihua, Chun-Chi Liu, Wenyuan Li, Hui Shen, Peter W. Laird, and Xianghong Jasmine Zhou. 2012. “Discovery of multi-dimensional modules by integrative analysis of cancer genomic data.” <em>Nucleic Acids Res.</em> 40 (19). Oxford University Press: 9379–91. doi:<a href="https://doi.org/10.1093/nar/gks725">10.1093/nar/gks725</a>.</p>
</div>
<div id="ref-Bonnet2015">
<p>Bonnet, Eric, Laurence Calzone, and Tom Michoel. 2015. “Integrative Multi-omics Module Network Inference with Lemon-Tree.” Edited by Paul P Gardner. <em>PLOS Comput. Biol.</em> 11 (2): e1003983. doi:<a href="https://doi.org/10.1371/journal.pcbi.1003983">10.1371/journal.pcbi.1003983</a>.</p>
</div>
<div id="ref-Huang2017">
<p>Huang, Sijia, Kumardeep Chaudhary, and Lana X Garmire. 2017. “More Is Better: Recent Progress in Multi-Omics Data Integration Methods.” <em>Front. Genet.</em> 8. Frontiers Media SA: 84. doi:<a href="https://doi.org/10.3389/fgene.2017.00084">10.3389/fgene.2017.00084</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="discussiongenerale.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="annexes.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/11-Conclusions.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
