<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="nmfica.html">
<link rel="next" href="results.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li><a href="index.html#unsupervised-deconvolution-of-bulk-omics-profiles-methodology-and-application-to-characterize-the-immune-landscape-in-tumors"><strong>Unsupervised deconvolution of bulk omics profiles: methodology and application to characterize the immune landscape in tumors</strong></a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="tcga-study-abbreviations.html"><a href="tcga-study-abbreviations.html"><i class="fa fa-check"></i>TCGA Study Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-ph.d.---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary Ph.D. - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-ph.d.topic"><i class="fa fa-check"></i>The origins of the Ph.D. topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="mstd.html"><a href="mstd.html#context"><i class="fa fa-check"></i><b>3.1</b> Context</a></li>
<li class="chapter" data-level="3.2" data-path="mstd.html"><a href="mstd.html#description"><i class="fa fa-check"></i><b>3.2</b> Description</a></li>
<li class="chapter" data-level="3.3" data-path="mstd.html"><a href="mstd.html#impact-on-the-further-work"><i class="fa fa-check"></i><b>3.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a><ul>
<li class="chapter" data-level="4.1" data-path="lva.html"><a href="lva.html#context-1"><i class="fa fa-check"></i><b>4.1</b> Context</a></li>
<li class="chapter" data-level="4.2" data-path="lva.html"><a href="lva.html#description-1"><i class="fa fa-check"></i><b>4.2</b> Description</a></li>
<li class="chapter" data-level="4.3" data-path="lva.html"><a href="lva.html#impact-on-the-further-work-1"><i class="fa fa-check"></i><b>4.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="nmfica.html"><a href="nmfica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.1" data-path="nmfica.html"><a href="nmfica.html#comparing-metagenes-obtained-with-nmf-versus-ica"><i class="fa fa-check"></i><b>5.1</b> Comparing metagenes obtained with NMF versus ICA</a></li>
<li class="chapter" data-level="5.2" data-path="nmfica.html"><a href="nmfica.html#summary-4"><i class="fa fa-check"></i><b>5.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#unsupervised-deconvolution"><i class="fa fa-check"></i><b>6.2</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#fastica-overdecomposition-protocol"><i class="fa fa-check"></i><b>6.2.1</b> FastICA overdecomposition protocol</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="deconica.html"><a href="deconica.html#interpretation-of-the-components"><i class="fa fa-check"></i><b>6.3</b> Interpretation of the components</a></li>
<li class="chapter" data-level="6.4" data-path="deconica.html"><a href="deconica.html#computing-the-abundance-of-the-identified-cell-types"><i class="fa fa-check"></i><b>6.4</b> Computing the abundance of the identified cell-types</a></li>
<li class="chapter" data-level="6.5" data-path="deconica.html"><a href="deconica.html#validation-of-the-abundance-estimation-with-deconica"><i class="fa fa-check"></i><b>6.5</b> Validation of the abundance estimation with <em>DeconICA</em></a><ul>
<li class="chapter" data-level="6.5.1" data-path="deconica.html"><a href="deconica.html#in-silico"><i class="fa fa-check"></i><b>6.5.1</b> <em>In silico</em></a></li>
<li class="chapter" data-level="6.5.2" data-path="deconica.html"><a href="deconica.html#in-vitro"><i class="fa fa-check"></i><b>6.5.2</b> <em>In vitro</em></a></li>
<li class="chapter" data-level="6.5.3" data-path="deconica.html"><a href="deconica.html#pbmc-transcriptome"><i class="fa fa-check"></i><b>6.5.3</b> PBMC transcriptome</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="deconica.html"><a href="deconica.html#summary-5"><i class="fa fa-check"></i><b>6.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Landscape of tumor immunity, with focus on T-cells diversity, uncovered via unsupervised deconvolution applied to bulk transcriptomes across 30 tumor types</a><ul>
<li class="chapter" data-level="7.1" data-path="results.html"><a href="results.html#background"><i class="fa fa-check"></i><b>7.1</b> Background</a></li>
<li class="chapter" data-level="7.2" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>7.2</b> Results</a><ul>
<li class="chapter" data-level="7.2.1" data-path="results.html"><a href="results.html#unsupervised-deconvolution-of-119-datasets"><i class="fa fa-check"></i><b>7.2.1</b> Unsupervised deconvolution of 119 datasets</a></li>
<li class="chapter" data-level="7.2.2" data-path="results.html"><a href="results.html#interpretation-of-the-immune-components"><i class="fa fa-check"></i><b>7.2.2</b> Interpretation of the immune components</a></li>
<li class="chapter" data-level="7.2.3" data-path="results.html"><a href="results.html#quality-of-cell-type-signal-extraction-is-affected-by-tumor-type"><i class="fa fa-check"></i><b>7.2.3</b> Quality of cell-type signal extraction is affected by tumor type</a></li>
<li class="chapter" data-level="7.2.4" data-path="results.html"><a href="results.html#deconvoluted-bulk-transcriptome-t-cell-signatures-showcase-different-cell-states-in-cancers"><i class="fa fa-check"></i><b>7.2.4</b> Deconvoluted bulk transcriptome T cell signatures showcase different cell states in cancers</a></li>
<li class="chapter" data-level="7.2.5" data-path="results.html"><a href="results.html#single-cell-rnaseq-deconvolution-leads-to-cell-specific-metagenes"><i class="fa fa-check"></i><b>7.2.5</b> Single-cell RNAseq deconvolution leads to cell-specific metagenes</a></li>
<li class="chapter" data-level="7.2.6" data-path="results.html"><a href="results.html#landscape-of-patient-based-on-the-immune-infiltration-scores"><i class="fa fa-check"></i><b>7.2.6</b> Landscape of patient based on the immune infiltration scores</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="results.html"><a href="results.html#discussion"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
<li class="chapter" data-level="7.4" data-path="results.html"><a href="results.html#resconcl"><i class="fa fa-check"></i><b>7.4</b> Conclusions</a></li>
<li class="chapter" data-level="7.5" data-path="results.html"><a href="results.html#methods-1"><i class="fa fa-check"></i><b>7.5</b> Methods</a><ul>
<li class="chapter" data-level="7.5.1" data-path="results.html"><a href="results.html#data-sources"><i class="fa fa-check"></i><b>7.5.1</b> Data sources</a></li>
<li class="chapter" data-level="7.5.2" data-path="results.html"><a href="results.html#data-pre-processing"><i class="fa fa-check"></i><b>7.5.2</b> Data pre-processing</a></li>
<li class="chapter" data-level="7.5.3" data-path="results.html"><a href="results.html#data-analysis"><i class="fa fa-check"></i><b>7.5.3</b> Data analysis</a></li>
<li class="chapter" data-level="7.5.4" data-path="results.html"><a href="results.html#software"><i class="fa fa-check"></i><b>7.5.4</b> Software</a></li>
<li class="chapter" data-level="7.5.5" data-path="results.html"><a href="results.html#hardware"><i class="fa fa-check"></i><b>7.5.5</b> Hardware</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="results.html"><a href="results.html#supplementary"><i class="fa fa-check"></i><b>7.6</b> Supplementary</a><ul>
<li class="chapter" data-level="7.6.1" data-path="results.html"><a href="results.html#figures"><i class="fa fa-check"></i><b>7.6.1</b> Figures</a></li>
<li class="chapter" data-level="7.6.2" data-path="results.html"><a href="results.html#tables"><i class="fa fa-check"></i><b>7.6.2</b> Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> A multiscale signalling network map of innate immune response in cancer reveals signatures of cell heterogeneity and functional polarization</a><ul>
<li class="chapter" data-level="8.1" data-path="map.html"><a href="map.html#context-2"><i class="fa fa-check"></i><b>8.1</b> Context</a></li>
<li class="chapter" data-level="8.2" data-path="map.html"><a href="map.html#description-2"><i class="fa fa-check"></i><b>8.2</b> Description</a></li>
<li class="chapter" data-level="8.3" data-path="map.html"><a href="map.html#discussion-and-perspectives"><i class="fa fa-check"></i><b>8.3</b> Discussion and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussiongenerale.html"><a href="discussiongenerale.html"><i class="fa fa-check"></i><b>9</b> Discussion</a><ul>
<li class="chapter" data-level="9.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#improving-our-knowledge-of-how-best-apply-ica-to-transcriptomic-data"><i class="fa fa-check"></i><b>9.1</b> Improving our knowledge of how best apply ICA to transcriptomic data</a></li>
<li class="chapter" data-level="9.2" data-path="discussiongenerale.html"><a href="discussiongenerale.html#confronting-ica-with-other-matrix-factorization-approaches"><i class="fa fa-check"></i><b>9.2</b> Confronting ICA with other matrix factorization approaches</a></li>
<li class="chapter" data-level="9.3" data-path="discussiongenerale.html"><a href="discussiongenerale.html#deconvolution-methods-overall-impact-and-utility"><i class="fa fa-check"></i><b>9.3</b> Deconvolution methods overall impact and utility</a></li>
<li class="chapter" data-level="9.4" data-path="discussiongenerale.html"><a href="discussiongenerale.html#unsupervised-deconvolution-1"><i class="fa fa-check"></i><b>9.4</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="9.4.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#limitations-of-unsupervised-deconvolution"><i class="fa fa-check"></i><b>9.4.1</b> Limitations of unsupervised deconvolution</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="discussiongenerale.html"><a href="discussiongenerale.html#data-sources-1"><i class="fa fa-check"></i><b>9.5</b> Data sources</a></li>
<li class="chapter" data-level="9.6" data-path="discussiongenerale.html"><a href="discussiongenerale.html#missing-time-and-space-dimensions-in-transcriptomic-studies"><i class="fa fa-check"></i><b>9.6</b> Missing time and space dimensions in transcriptomic studies</a></li>
<li class="chapter" data-level="9.7" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-place-of-bulk-transcriptomes-in-single-cell-era"><i class="fa fa-check"></i><b>9.7</b> The place of bulk transcriptomes in single cell era</a></li>
<li class="chapter" data-level="9.8" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-field-of-computational-cancer-immunology"><i class="fa fa-check"></i><b>9.8</b> The field of computational cancer immunology</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a><ul>
<li class="chapter" data-level="10.1" data-path="conclusions.html"><a href="conclusions.html#conclusions-1"><i class="fa fa-check"></i><b>10.1</b> Conclusions</a></li>
<li class="chapter" data-level="10.2" data-path="conclusions.html"><a href="conclusions.html#perspectives"><i class="fa fa-check"></i><b>10.2</b> Perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#deconica-documentation"><i class="fa fa-check"></i>1 DeconICA documentation</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#introduction-to-deconica"><i class="fa fa-check"></i>1.1 Introduction to deconICA</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#running-fastica-with-icasso-stabilisation"><i class="fa fa-check"></i>1.2 Running fastICA with icasso stabilisation</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#reference-manual"><i class="fa fa-check"></i>1.3 Reference manual</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#publications-and-conferences"><i class="fa fa-check"></i>2 Publications and conferences</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#adjustment-of-dendritic-cells-to-the-breast-cancer-microenvironment-is-subset-specific"><i class="fa fa-check"></i>2.1 Adjustment of dendritic cells to the breast-cancer microenvironment is subset-specific</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#the-inconvenience-of-data-of-convenience-computational-research-beyond-post-mortem-analyses"><i class="fa fa-check"></i>2.2 The inconvenience of data of convenience: computational research beyond post-mortem analyses</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv-publications-conferences-courses"><i class="fa fa-check"></i>2.3 CV: publications, conferences, courses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a><ul>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#biological-terms"><i class="fa fa-check"></i>Biological terms</a></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#mathematical-terms"><i class="fa fa-check"></i>Mathematical terms</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="deconica" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</h1>
<p><em>Selected content from this chapter is a part of a publication in preparation</em></p>
<div id="from-blind-deconvolution-to-cell-type-quantification-general-overview" class="section level2">
<h2><span class="header-section-number">6.1</span> From blind deconvolution to cell-type quantification: general overview</h2>
<p>In the introduction chapters 1 and 2, I have presented why there is a need to extract knowledge about the immune system from the cancer transcriptomes and how it can be done. In the result chapters 3, 4 and 5 I have presented studies of ICA application to transcriptomes, ideas for finding a way to extract the cell-specific components through <em>overdecomposition</em> procedure.</p>
<p>However, I in the presented works, the proposed biological interpretation was not deepened. In order to enable standardized unsupervised deconvolution framework and an interpretation of obtained components as immune cell types and their quantification that is scalable, I introduce, in this chapter, a method named “DeconICA”: <strong>Decon</strong>volution of omic data through <strong>I</strong>mmune <strong>C</strong>omponents <strong>A</strong>nalysis. The method is published online on GitHub: <a href="https://github.com/UrszulaCzerwinska/DeconICA">UrszulaCzerwinska/DeconICA</a> in the form of an R package and has a <a href="https://zenodo.org/record/1250070">doi number (10.5281/zenodo.1250069)</a> for citations. In this chapter, I will describe my pipeline and the rationale behind my strategy of analysis and quantification of components. I will also briefly compare its performance in the quantifying abundance with previously published methods. The user guide for DeconICA R package is available in <a href="#deconicatut">Annexes</a> and online at <a href="https://urszulaczerwinska.github.io/DeconICA/DeconICA_introduction.html" class="uri">https://urszulaczerwinska.github.io/DeconICA/DeconICA_introduction.html</a>.</p>
</div>
<div id="unsupervised-deconvolution" class="section level2">
<h2><span class="header-section-number">6.2</span> Unsupervised deconvolution</h2>
<p>So far, I have focused on ICA-based decompositions of transcriptomes. In the <a href="nmfica.html#nmfica">chapter 5</a>, I have compared ICA and NMF decompositions concluding that ICA-based decompositions shall be more convenient. In my work, because of my team expertise, the proven computational efficiency, interpretability, and reproducibility, I will use mostly ICA components to deconvolve tumor data (analysis described in the next <a href="results.html#results">chapter 6</a>). However, the constructed interpretation pipeline can take as input any metagenes, i.e., NMF factors, convex-hull derived sources or attractor metagenes (methods described in <a href="methods.html#methods">chapter 2</a>).</p>
<p>In this section, I will explain the DeconICA pipeline (Fig. <a href="deconica.html#fig:deconicaschema">6.1</a>) based on the stabilized fastICA overdecomposition protocol as it is resulting from our expertise.</p>
<div class="figure" style="text-align: center"><span id="fig:deconicaschema"></span>
<img src="figures-ext/deconica_schema.png" alt="Flowchart of DeconICA method. In this flowchart steps of DeconICA are represented as boxes. Each operation corresponds to one or multiple functions in the R package. Input data X (red) is preprocessed and then decomposed into components. Interpretation of the components is performed with correlation or gene enrichment analysis (using reference materials - green). For components labeled as cell types or other important factors, abundance can be estimated using top genes of each component and computing average of counts in a non-log scale of those genes. Main outputs are ( in blue), the S component matrix, labeled components, and their abundance scores." width="3750" height="80%" />
<p class="caption">
Figure 6.1: <strong>Flowchart of DeconICA method</strong>. In this flowchart steps of DeconICA are represented as boxes. Each operation corresponds to one or multiple functions in the R package. Input data X (red) is preprocessed and then decomposed into components. Interpretation of the components is performed with correlation or gene enrichment analysis (using reference materials - green). For components labeled as cell types or other important factors, abundance can be estimated using top genes of each component and computing average of counts in a non-log scale of those genes. Main outputs are ( in blue), the S component matrix, labeled components, and their abundance scores.
</p>
</div>

<div id="fastica-overdecomposition-protocol" class="section level3">
<h3><span class="header-section-number">6.2.1</span> FastICA overdecomposition protocol</h3>
<p>The main inputs of unsupervised deconvolution methods are data matrix <span class="math inline">\(X\)</span> and number of output sources <span class="math inline">\(k\)</span>. I have developed a protocol that can define <span class="math inline">\(k\)</span> for fastICA algorithm applied to cancer transcriptome datasets and prepares the data for interpretation steps.</p>
<div id="data-transformation" class="section level4">
<h4><span class="header-section-number">6.2.1.1</span> Data transformation</h4>
<p>The input matrix of gene expression is transformed to <span class="math inline">\(log_{2}(x +1)\)</span> where is <span class="math inline">\(x\)</span> is a data point and then row (gene) centered: mean of each row is removed. This step is necessary for ICA algorithm.</p>
<p>If gene names or probes are provided, the duplicated genes are removed - the genes with higher variance are kept.</p>
</div>
<div id="determining-k-number-of-sources" class="section level4">
<h4><span class="header-section-number">6.2.1.2</span> Determining <span class="math inline">\(k\)</span> number of sources</h4>
<p>If the number of sources in the mixture is known, <span class="math inline">\(k\)</span> can be fixed by the user.</p>
<p>In case of complex mixtures of tumor transcriptomes, the input matrix of gene in rows (<span class="math inline">\(n\)</span>) and samples (<span class="math inline">\(m\)</span>) in columns is decomposed into <span class="math inline">\(k=100\)</span> components if <span class="math inline">\(m\)</span> &gt; 100. If m &lt;<span class="math inline">\(100\)</span>, then the <span class="math inline">\(k\)</span> number is equal to the number of PCA components necessary to explain 90% of data.</p>
<p>If a different data type is analyzed, one can also compute MSTD (see <a href="mstd.html#mstd">chapter 3</a>) of the data and redefine the <em>overdecompositon</em> dimension.</p>
</div>
<div id="fastica" class="section level4">
<h4><span class="header-section-number">6.2.1.3</span> FastICA</h4>
<p>The fastICA algorithm uses icasso stabilization (described in chapter 2 section 2.3.6.3) to obtain an average representation of components over the <span class="math inline">\(i\)</span> number of iterations (by default <span class="math inline">\(i=100\)</span>) in order to buffer the effect of local minima resulting from stochastic initializations.</p>
<p>The stabilized version of FastICA algorithm is, so far, only available in Matlab. A custom Matlab scripts “fastica++” that I am using have initially been distributed with <a href="https://github.com/LabBandSB/BIODICA">BIODICA software</a> by Zinovyev and Kairov. I have created an R interface to use it easily without any knowledge of Matlab. I also provided a detailed description on how and why use this version of fastICA as a part of <a href="https://urszulaczerwinska.github.io/DeconICA/Icasso.html">DeconICA tutorial</a> (available in Annex Z), even without Matlab software installed (through a Docker image). Although it is possible to use non-stabilized and slower R version of fastICA, it impacts the results significantly.</p>
<p>To remind, the input <span class="math inline">\(X_{n\times m}\)</span> matrix is in therefore decomposed to <span class="math inline">\(S_{k\times n}\)</span> source matrix and <span class="math inline">\(A_{m \times k}\)</span> mixing matrix, where <span class="math inline">\(k\)</span> is the number of components. Therefore, for example, having as an input transcriptome matrix of 20 000 genes and 150 samples, if <span class="math inline">\(k\)</span> = 100 (overdecomposition), then the <span class="math inline">\(S\)</span> will have dimensions 20 000 <span class="math inline">\(\times\)</span> 100 and <span class="math inline">\(A\)</span> matrix 100 <span class="math inline">\(\times\)</span> 150.</p>
</div>
<div id="orienting-the-components" class="section level4">
<h4><span class="header-section-number">6.2.1.4</span> Orienting the components</h4>
<p>The <span class="math inline">\(S\)</span> columns (components/sources) contain positive and negative values that the projections of data in the given dimension. The sign (positive or negative) cannot be directly interpreted. Therefore the values should be seen as an absolute value of the projection. The genes ranked top (by the absolute value), separately for positive and negative ends) are representative for the component. Usually, only one end leads to a biological interpretation. To define which end should be interpreted, I apply a simple statistical procedure (Fig. <a href="deconica.html#fig:orienting">6.2</a>).</p>
<p>I plot density distribution of a component, compute a standard deviation and count how many genes are above or below a threshold of <span class="math inline">\(t\)</span>. I define <span class="math inline">\(t =\)</span> 3 standard deviation (sd). If there are no points &gt;<span class="math inline">\(t\)</span>, the threshold is lowered to 2sd, etc. The end of the independent components with a higher number of genes over the threshold is decided to be the one representing the component that we call <em>heavy tail</em>.</p>
<p>If the <em>heavy tail</em> is negative the component weights are multiplied by -1 in order to reverse signs.</p>
<p>In this way, all the representative ends of <span class="math inline">\(S\)</span> are all positive which makes the further steps easier.</p>
<p>This procedure can fail if the initial sources are coming from, i.e., uniform distribution and the component <span class="math inline">\(S_i\)</span> is symmetric. In those cases, components can be oriented for examples based on correlation with known sources (to have only positive correlations). In practical terms, there is an option to skip this step.</p>
<p>Generally, the orientation procedure applies to microarray and RNA-seq transcriptomes that are overdecomposed.</p>
<div class="figure" style="text-align: center"><span id="fig:orienting"></span>
<img src="figures-ext/orienting.png" alt="Principle of components orienting. An independent component illustrated here has more values under the threshold of - 3 standard deviations (sd) than over 3 sd (64 vs. 4). Therefore the heavy tail is on the negative side. DeconICA orients the heavy tail towards the positive side. This procedure is applied to all independent components." width="70%" />
<p class="caption">
Figure 6.2: <strong>Principle of components orienting</strong>. An independent component illustrated here has more values under the threshold of - 3 standard deviations (sd) than over 3 sd (64 vs. 4). Therefore the heavy tail is on the negative side. DeconICA orients the heavy tail towards the positive side. This procedure is applied to all independent components.
</p>
</div>

</div>
</div>
</div>
<div id="interpretation-of-the-components" class="section level2">
<h2><span class="header-section-number">6.3</span> Interpretation of the components</h2>
<div id="identification-of-immune-cell-types-with-an-enrichment-test" class="section level4">
<h4><span class="header-section-number">6.3.0.1</span> Identification of immune cell types with an enrichment test</h4>
<p>Based on previous work of our team, a way to characterize obtained components is to select top genes and verify if they belong to a described biological pathway or a list of genes. There is a wide choice of websites (i.e., <a href="http://biogps.org/#goto=welcome">BioGPS</a>, <a href="https://toppgene.cchmc.org/">Toppgene</a> or <a href="http://amp.pharm.mssm.edu/Enrichr/">EnrichR</a>) that compares a selected list of genes with databases and computes, in different ways, an enrichment score. Often, proposed pathways or conditions are quite general, for instance, type of immune response and associated cell type would not be specified. Another way to compute enrichment is to use GSEA (described in section 2.3.3). It is possible that I will extend the package to facilitate an enrichment with GSEA (for example using the <em>fgsea</em> R package <span class="citation">(Sergushichev <a href="#ref-Sergushichev2016">2016</a>)</span>). However, the essence of the interpretation though enrichment is not the score itself but the pathways/processes identified to be associated with the analyzed component.</p>
<p>This is why, in DeconICA package, I implemented a simple Fisher exact test (described in section 2.3.3) that computes a significance of an overlap between a list of top genes from a component and a collection of known gene sets. An ensemble of known pathways is called the universe.</p>
<p>I set some default parameters for the analysis: the number of top genes, minimal and maximal length of the gene list in the universe, p-value correction. I have tested the enrichment of the components in a custom collection of cell-type specific signatures published in primary research articles or as a part of deconvolution tools.</p>
<p>Enrichment allows to link a list of component’s top genes to biological process, but the p-value depends strongly on the size of the universe and the size of gene sets. In most cases, only a small number of genes is known to be specific to a cell type. These specific genes are not always expressed in the analyzed dataset. Therefore, often false positives are found only due to technical dimension of the analysis. Depending on which compendium of signatures I used I would identify very different cell types with the enrichment test for the same component.</p>
<p>The enrichment methods were not conceived for small and close related gene sets. A more sophisticated solution like in xCell <span class="citation">(Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>)</span>, are necessary to overcome those limitations. Thus, I would advise using basic enrichment for exploratory analysis of the general factors impacting the transcriptome rather than to identify cell types.</p>
<p>Looking for more robust evaluation of immune cells identification I have proposed a correlation-based interpretation.</p>
</div>
<div id="correlation-based-identification" class="section level4">
<h4><span class="header-section-number">6.3.0.2</span> Correlation based identification</h4>
</div>
<div id="reciprocal-match" class="section level4">
<h4><span class="header-section-number">6.3.0.3</span> Reciprocal match</h4>
<p>Working with many datasets, I remarked that correlation with a reference metagene enables quickly and robustly label the components. The reference metagenes provided with DeconICA were published in the study of <span class="citation">Biton et al. (<a href="#ref-Biton2014">2014</a>)</span>. As these signals represent independent biological factors, one reference metagene should match one component. This is why I applied the rule of reciprocal matching to label components corresponding to reference metagenes.</p>
<p>Formally, the reciprocity rule is defined as follows. Given correlations between the set of reference metagenes <span class="math inline">\(M=\{M_{1},...,M_{m}\}\)</span> and <span class="math inline">\(S\)</span> source matrix <span class="math inline">\(S=\{IC_{1},...,IC_{N}\}\)</span>, if <span class="math inline">\(S_{i}=argmax_k (corr(M_{j},S_{k}))\)</span> and <span class="math inline">\(M_{j}=argmax_k (corr(S_{i},M_{k}))\)</span>, then <span class="math inline">\(S_{i}\)</span> and <span class="math inline">\(M_{j}\)</span> are reciprocal. This rule was already applied previous publications of our group <span class="citation">(Kairov et al. <a href="#ref-Kairov2017">2017</a>; Czerwinska <a href="#ref-Czerwinska2018">2018</a>; Cantini, Kairov, et al. <a href="#ref-Cantini2018">2018</a>)</span>. An important feature of this association is that it is not based on the strength of the correlation and hypothesise that even weak correlation can be meaningful if there is an exclusive reciprocal match between reference metagene and a component.</p>
</div>
<div id="maximal-correlation-match" class="section level4">
<h4><span class="header-section-number">6.3.0.4</span> Maximal correlation match</h4>
<p>As there are no established metagenes of immune cells, I used a signature matrix published in <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>)</span> LM22 containing 22 immune cells profiles reduced to 510 genes that should enable the differentiation between the cell types. I used specifically this reference as it had more genes than other similar matrices (i.e., the one from EPIC <span class="citation">(Racle et al. <a href="#ref-Racle2017">2017</a>)</span> contains <span class="math inline">\(\approx\)</span> 100 genes). The number of genes is critical because the correlation needs to be based on a minimal number of genes to be reliable. It may happen that genes present in the reference profiles are not present in studied gene matrix. Therefore, more genes in the reference set to increase the probability to interpret the data. Generally, the power of the correlation coefficient increases with the number of genes used to compute it (the overlap between the reference and the analyzed data). On the other hand, the full cell profiles (10000-20000 genes) would introduce a significant amount of noise.</p>
<p>To assign a reference cell type to a component, I adopted a strategy different from described in the previous section. It is known that cell type subtypes have quite similar expression profiles. Therefore, the match of one component to different subtypes of the same cell type should not be penalized. I have also remarked that sometimes closely related cell types cannot be discriminated and one component can be the most related to, i.e., T-cell and NK. Thus, the label is attributed through maximal correlation, for instance, which of all components has the highest correlation coefficient with given reference profile.</p>
<p>This results in the fact that each reference metagene has a component attributed. Then, the user needs to define manually is the association can be trusted from the value of the Pearson correlation coefficient. Usually, if there is one component with remarkably stronger associated to the reference cell type than others, the association can be trusted. A graphical representation (Fig. <a href="deconica.html#fig:corrEx">6.3</a>) can be help in decision making. This step should be automatized if possible in the future.</p>
<div class="figure" style="text-align: center"><span id="fig:corrEx"></span>
<img src="figures-ext/corrEx.png" alt="Example of successful and unsuccessful component matching to a refrence. Among all components there can be one component that matches the reference profile (A) or many components that weekly matches the reference profile (B). Only in the case, ‘A’ the most correlated component should be labeled as B-cell." width="70%" />
<p class="caption">
Figure 6.3: <strong>Example of successful and unsuccessful component matching to a refrence</strong>. Among all components there can be one component that matches the reference profile (A) or many components that weekly matches the reference profile (B). Only in the case, ‘A’ the most correlated component should be labeled as B-cell.
</p>
</div>

</div>
</div>
<div id="computing-the-abundance-of-the-identified-cell-types" class="section level2">
<h2><span class="header-section-number">6.4</span> Computing the abundance of the identified cell-types</h2>
<p>Once the components are labeled, their contribution in each sample can be estimated. For cell type related components this contribution can be interpreted as cell-type abundance (in arbitrary units).</p>
<p>Before deciding on the final way in which cell-type contribution can be evaluated, I have tested different possibilities.</p>
<p>In theory, the <span class="math inline">\(A\)</span> mixing matrix reflects contributions of each component. However, it reflects the contribution of both positive and negative ends of components. In my protocol, one end is selected to be representative of a biological signal. This is why <span class="math inline">\(A\)</span> matrix scores do not reflect well abundance of the information on cell types.</p>
<p>One idea was to use the components as “pure cell-type expression” in a regression model (testing different regression types: SVR, quadratic programming, simple linear regression, lasso regression). I tested this approach on blood transcriptome benchmark (described in more details below). The results were acceptable, but they got outperformed by the mean of top genes approach that is included in DeconICA.</p>
<p>Finally, I tested the approach inspired by <span class="citation">(Becht et al. <a href="#ref-Becht2016">2016</a>)</span>. Selecting the most specific genes and computing arithmetic mean of the genes in the original counts matrix. This method is based on the hypothesis that those particular markers are unique to one cell type. Therefore gene expression of those genes should be proportional to the abundance of the cell type.</p>
<div id="defining-markers" class="section level4">
<h4><span class="header-section-number">6.4.0.1</span> Defining markers</h4>
<p>In <span class="citation">(Becht et al. <a href="#ref-Becht2016">2016</a>)</span> cell-type specific markers are defined based on expert knowledge and validated in gene expression data. In this work, the specific markers are generated in the unsupervised deconvolution. I adopted a hypothesis that <span class="math inline">\(n_{top}\)</span> genes of a labeled component is a unique signature of this component. I defined the value of <span class="math inline">\(n_{top}\)</span> empirically to be in a range of 10 to 30 genes.</p>
</div>
<div id="computing-scores" class="section level4">
<h4><span class="header-section-number">6.4.0.2</span> Computing scores</h4>
<p>In order to compute the scores, a mean of the selected marker genes for each component is computed. The user has a choice between arithmetical, geometric, harmonic or weighted mean. So far, the arithmetic means to seem to give the best performance on the benchmark data.</p>
</div>
</div>
<div id="validation-of-the-abundance-estimation-with-deconica" class="section level2">
<h2><span class="header-section-number">6.5</span> Validation of the abundance estimation with <em>DeconICA</em></h2>
<p><em>Code used to produce the validation, and more extensive description of each step is available as a part of the <a href="https://urszulaczerwinska.github.io/DeconICA/DeconICA_introduction.html">online tutorial</a> and in Annexes</em></p>
<div id="in-silico" class="section level3">
<h3><span class="header-section-number">6.5.1</span> <em>In silico</em></h3>
<p>First, the performance of the DeconICA was tested on simulated data. I wrote a function that produces a linear mixture of randomly generated sources of selected distribution with known proportions and possibility to set the number of marker genes. Here I generated ten sources drawn from the negative binomial distribution (10000 <em>genes</em>) and mix them at known proportions, adding some noise resulting in 130 mixtures (<em>samples</em>). As the ten original sources are known, a component is attributed to an original source through reciprocal correlation. I demonstrate that for each original source a matching component can be identified and using 10 top markers I estimate abundance with correlation coefficient ranging from 0.95 to 0.99 (where 1.0 is a perfect correlation), equal to average <span class="math inline">\(R^2\)</span>= 0.96 (Fig <a href="deconica.html#fig:insilico">6.4</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:insilico"></span>
<img src="figures-ext/insilico.png" alt="Accuracy of estimation versus true proportions in an in silico mixture. The simulated matrix of mixtures (10000 \(\times\) 130) was decomposed, and obtained components were used to estimate proportions. The estimated proportions of each of 10 cell types were correlated with the known abundance values of the given cell type. The Pearson correlation coefficient values are reported in the correlation matrix." width="70%" />
<p class="caption">
Figure 6.4: <strong>Accuracy of estimation versus true proportions in an <em>in silico</em> mixture</strong>. The simulated matrix of mixtures (10000 <span class="math inline">\(\times\)</span> 130) was decomposed, and obtained components were used to estimate proportions. The estimated proportions of each of 10 cell types were correlated with the known abundance values of the given cell type. The Pearson correlation coefficient values are reported in the correlation matrix.
</p>
</div>

</div>
<div id="in-vitro" class="section level3">
<h3><span class="header-section-number">6.5.2</span> <em>In vitro</em></h3>
<p>Previously published in <span class="citation">(Becht et al. <a href="#ref-Becht2016">2016</a>)</span> <em>in vitro</em> immune cell types sorted from 3 healthy donors’ peripheral blood and mixed at different proportions resulting in 12 mixed samples. Following my pipeline, data is decomposed with stabilized fastICA, each of cell types finds its match and marker genes are defined as top 10 genes of each component. Then the correlation with abundance is computed. The Pearson correlation with true mixing proportions varies from 0.93 to 0.99, average <span class="math inline">\(R^2\)</span>= 0.94 (Fig. <a href="deconica.html#fig:invitro">6.5</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:invitro"></span>
<img src="figures-ext/invitro.png" alt="Accuracy of estimation versus true proportions in an in vitro mixture. Five different immune cell types from three donors were mixed at known proportions. The estimated proportions of each of 5 cell types were correlated with the known abundance values of the given cell type. The Pearson correlation coefficient values are reported in the correlation matrix." width="70%" />
<p class="caption">
Figure 6.5: <strong>Accuracy of estimation versus true proportions in an <em>in vitro</em> mixture</strong>. Five different immune cell types from three donors were mixed at known proportions. The estimated proportions of each of 5 cell types were correlated with the known abundance values of the given cell type. The Pearson correlation coefficient values are reported in the correlation matrix.
</p>
</div>

</div>
<div id="pbmc-transcriptome" class="section level3">
<h3><span class="header-section-number">6.5.3</span> PBMC transcriptome</h3>
<p>Finally, I applied DeconICA to PBMC expression data of 104 healthy patients, paired CyTOF proportion estimation for each sample <span class="citation">(Whiting et al. <a href="#ref-Whiting2015">2015</a>)</span>, processed data were shared kindly by <span class="citation">(Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>)</span>. I use MSTD to estimate the optimal dimension (39). From the correlation profiles, it can be seen that this task is remarkably more challenging than the previous tests (Fig @(fig:radarB)) and not all cell types can be perfectly matched to components. Based on maximal correlation a subset of components is labeled as immune cells and based on 10 top markers abundance is computed for B-cells, T cells CD8, T cells CD4, NK and Monocytes.</p>
<p>The correlation between CyTOF measured abundance and DeconICA estimated abundance varies from 0.31 to 0.77. Some cell types as T-regs, NKT, other T-cell, naive-B-cells subtypes could not be differentiated with individual components. I compared DeconICA performance with five other methods of immune cell-type deconvolution (Fig <a href="deconica.html#fig:comparison">6.7</a>). The strategical differences between compared methods were explained in <a href="methods.html#methods">Chapter 2</a>. All of them are recent supervised deconvolution approaches, state-of-art at the date. DeconICA performance is better or similar than the one of previously published methods for the cell types that we could identify and in average has slightly better <span class="math inline">\(R^2\)</span>.</p>
<p>It is necessary to mention that EPIC probably performs worse than expected because data were not TPM normalized. For MCPcounter CD8 T-cells were matched to “cytotoxic T-cells” and CD4 T-cells to T-cells. For CIBERSORT between naive B-cells and activated B-cells better correlation was reported as “B-cells”.</p>
<div class="figure" style="text-align: center"><span id="fig:radarB"></span>
<img src="figures-ext/radarBlood.png" alt="Correlation between independent components and reference immune cell-type metagenes. All correlation and reference cell types are illustrated. Surrounded by color squares are the most important panels for decision making and matching cell types measures as well with CyTOF, used for further comparison." width="100%" />
<p class="caption">
Figure 6.6: <strong>Correlation between independent components and reference immune cell-type metagenes</strong>. All correlation and reference cell types are illustrated. Surrounded by color squares are the most important panels for decision making and matching cell types measures as well with CyTOF, used for further comparison.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:comparison"></span>
<img src="figures-ext/comparisonR2.png" alt="Estimation of abundance of immune cell types in PBMC transcriptome of 104 healthy donors. Five different methods (xCell (Aran, Hu, and Butte 2017), CIBERSORT (Newman et al. 2015), EPIC (Racle et al. 2017), MCPcounter (Becht et al. 2016), DeconICA (Czerwinska 2018)), were applied to compare estimated proportions and CyTOF measured proportions of five cell types: B-cells, CD8 T-.cells, CD4 T-cells, Monocytes and NK as they were identified with DeconICA and measured proportions were available. In the case of a not exact match of cell types between the CyTOF and deconvolutions method, best correlation was reported. The average \(R^2\) is computed as an average of \(R^2\) for each cell types by the method. DeconICA slightly overperforms existing methods." width="70%" />
<p class="caption">
Figure 6.7: <strong>Estimation of abundance of immune cell types in PBMC transcriptome of 104 healthy donors</strong>. Five different methods (xCell <span class="citation">(Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>)</span>, CIBERSORT <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>)</span>, EPIC <span class="citation">(Racle et al. <a href="#ref-Racle2017">2017</a>)</span>, MCPcounter <span class="citation">(Becht et al. <a href="#ref-Becht2016">2016</a>)</span>, DeconICA <span class="citation">(Czerwinska <a href="#ref-Czerwinska2018">2018</a>)</span>), were applied to compare estimated proportions and CyTOF measured proportions of five cell types: B-cells, CD8 T-.cells, CD4 T-cells, Monocytes and NK as they were identified with DeconICA and measured proportions were available. In the case of a not exact match of cell types between the CyTOF and deconvolutions method, best correlation was reported. The average <span class="math inline">\(R^2\)</span> is computed as an average of <span class="math inline">\(R^2\)</span> for each cell types by the method. DeconICA slightly overperforms existing methods.
</p>
</div>

<p>Besides, the markers discovered from data by DeconICA, are not significantly overlapping with markers used by MCPcounter or xCell (Fig. <a href="deconica.html#fig:venn">6.8</a>) (for EPIC and CIBERSORT list of specific cell-markers is provided as they use a basis matrix).</p>
<div class="figure" style="text-align: center"><span id="fig:venn"></span>
<img src="figures-ext/venn.png" alt="Comparison of markers used by different deconvolution methods: markers of xCell, MCPcounter, and markers discovered from data by DeconICA are compared. Venn diagrams illustrate insignificant overlap between the specific markers list for each cell type." width="100%" />
<p class="caption">
Figure 6.8: <strong>Comparison of markers used by different deconvolution methods</strong>: markers of xCell, MCPcounter, and markers discovered from data by DeconICA are compared. Venn diagrams illustrate insignificant overlap between the specific markers list for each cell type.
</p>
</div>

</div>
</div>
<div id="summary-5" class="section level2">
<h2><span class="header-section-number">6.6</span> Summary</h2>
<p>I developed a DeconICA method and R package that allow decomposition of omic data into components. Based on stabilized fastICA decomposition I demonstrated that DeconICA could estimate cell proportions of immune cell types in PMBC transcriptome with better performance than previously published tools without an <em>a priori</em> use of cell-type signatures. The <em>marker</em> genes discovered with ICA, proven to evaluate cell abundance correctly, turned out to be different from the markers used by knowledge-based supervised deconvolution methods.</p>
<p>Even though the first release of DeconICA is published online <span class="citation">(Czerwinska <a href="#ref-Czerwinska2018">2018</a>)</span> and fully functional, some improvements can still be considered in the future. Examples using other unsupervised deconvolution methods can be included to demonstrate that the pipeline is not limited to ICA. An interactive online interface can be built with R shiny for instance. Also, more automatized label attribution and confidence of the match between a reference metagene and a component should be added. I would like also demonstrate the use of DeconICA with methylome data.</p>
<p>Main limitations of DeconICA are</p>
<ul>
<li>the need to work with many samples (&gt;100)</li>
<li>the interpretation that requires manual adjustments</li>
<li>the number of detected cell types usually lower than other tools</li>
<li>the abundance scores that cannot be directly interpreted as percentages of sample content</li>
<li>it is not guaranteed to find a source of the desired cell-type</li>
</ul>
<p>Main advantages of DeconICA are:</p>
<ul>
<li>the possibility to discover new markers from data</li>
<li>context independence (no a priori use of blood-derived cell-type signatures)</li>
<li>universality: DeconICA can identify not only cell types but also other factors governing cancer transcriptomes, can also be applied for different purposes with a little adjustment</li>
<li>sequencing/ microarray platform independence</li>
<li>data normalization independence</li>
<li>speed (a formal benchmark is to be provided, but the analysis of the biggest available transcriptomic dataset METABRIC (1980 samples) can be analyzed within less than 1 hour)</li>
<li>user-friendly form of R package, tutorials, and transparent, open-source code</li>
</ul>
<p>So far, the tool does not have many users from outside my research group. I hope it will change once the tool is published in a scientific journal.</p>
<p>An ability to discover possible new marker genes of cell states can be extremely interesting in cancer context where true cell-type context-dependent signatures are not known. If this hypothesis is correct, DeconICA should not only correctly evaluate cell-type abundances but also give insight to context-dependent cell type/cell state signatures.</p>
<p>In the next chapter, I will apply DeconICA to &gt;100 transcriptomic datasets of different cancer types to identify tumor-specific signatures of immune cell types, compare them and describe their features.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Sergushichev2016">
<p>Sergushichev, Alexey. 2016. “An algorithm for fast preranked gene set enrichment analysis using cumulative statistic calculation.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 060012. doi:<a href="https://doi.org/10.1101/060012">10.1101/060012</a>.</p>
</div>
<div id="ref-Aran2017">
<p>Aran, Dvir, Zicheng Hu, and Atul J. Butte. 2017. “xCell: digitally portraying the tissue cellular heterogeneity landscape.” <em>Genome Biol.</em> 18 (1). BioMed Central: 220. doi:<a href="https://doi.org/10.1186/s13059-017-1349-1">10.1186/s13059-017-1349-1</a>.</p>
</div>
<div id="ref-Biton2014">
<p>Biton, Anne, Isabelle Bernard-Pierrot, Yinjun Lou, Clémentine Krucker, Elodie Chapeaublanc, Carlota Rubio-Pérez, Nuria López-Bigas, et al. 2014. “Independent Component Analysis Uncovers the Landscape of the Bladder Tumor Transcriptome and Reveals Insights into Luminal and Basal Subtypes.” <em>Cell Rep.</em> 9 (4): 1235–45. doi:<a href="https://doi.org/10.1016/j.celrep.2014.10.035">10.1016/j.celrep.2014.10.035</a>.</p>
</div>
<div id="ref-Kairov2017">
<p>Kairov, Ulykbek, Laura Cantini, Alessandro Greco, Askhat Molkenov, Urszula Czerwinska, Emmanuel Barillot, and Andrei Zinovyev. 2017. “Determining the optimal number of independent components for reproducible transcriptomic data analysis.” <em>BMC Genomics</em> 18 (1). BioMed Central: 712. doi:<a href="https://doi.org/10.1186/s12864-017-4112-9">10.1186/s12864-017-4112-9</a>.</p>
</div>
<div id="ref-Czerwinska2018">
<p>Czerwinska, Urszula. 2018. “UrszulaCzerwinska/DeconICA: DeconICA first release.” doi:<a href="https://doi.org/10.5281/ZENODO.1250070">10.5281/ZENODO.1250070</a>.</p>
</div>
<div id="ref-Cantini2018">
<p>Cantini, Laura, Ulykbek Kairov, Aurelien de Reynies, Emmanuel Barillot, Francois Radvanyi, and Andrei Zinovyev. 2018. “Stabilized Independent Component Analysis outperforms other methods in finding reproducible signals in tumoral transcriptomes.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 318154. doi:<a href="https://doi.org/10.1101/318154">10.1101/318154</a>.</p>
</div>
<div id="ref-Newman2015">
<p>Newman, Aaron M, Chih Long Liu, Michael R Green, Andrew J Gentles, Weiguo Feng, Yue Xu, Chuong D Hoang, Maximilian Diehn, and Ash A Alizadeh. 2015. “Robust enumeration of cell subsets from tissue expression profiles.” <em>Nat. Methods</em> 12 (5). Nature Publishing Group: 453–57. doi:<a href="https://doi.org/10.1038/nmeth.3337">10.1038/nmeth.3337</a>.</p>
</div>
<div id="ref-Racle2017">
<p>Racle, Julien, Kaat de Jonge, Petra Baumgaertner, Daniel E Speiser, and David Gfeller. 2017. “Simultaneous enumeration of cancer and immune cell types from bulk tumor gene expression data.” <em>Elife</em> 6. eLife Sciences Publications Limited: e26476. doi:<a href="https://doi.org/10.7554/eLife.26476">10.7554/eLife.26476</a>.</p>
</div>
<div id="ref-Becht2016">
<p>Becht, Etienne, Aurélien de Reyniès, Nicolas A Giraldo, Camilla Pilati, Bénédicte Buttard, Laetitia Lacroix, Janick Selves, Catherine Sautès-Fridman, Pierre Laurent-Puig, and Wolf Herman Fridman. 2016. “Immune and Stromal Classification of Colorectal Cancer Is Associated with Molecular Subtypes and Relevant for Precision Immunotherapy.” <em>Clin. Cancer Res.</em> 22 (16). American Association for Cancer Research: 4057–66. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-15-2879">10.1158/1078-0432.CCR-15-2879</a>.</p>
</div>
<div id="ref-Whiting2015">
<p>Whiting, Chan C., Janet Siebert, Aaron M. Newman, Hong-wu Du, Ash A. Alizadeh, Jorg Goronzy, Cornelia M. Weyand, Eswar Krishnan, C. Garrison Fathman, and Holden T. Maecker. 2015. “Large-Scale and Comprehensive Immune Profiling and Functional Analysis of Normal Human Aging.” Edited by Derya Unutmaz. <em>PLoS One</em> 10 (7): e0133627. doi:<a href="https://doi.org/10.1371/journal.pone.0133627">10.1371/journal.pone.0133627</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="nmfica.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="results.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/08-DeconICA.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
