<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="comparison-of-reproducibility-between-nmf-and-ica.html">
<link rel="next" href="results.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; min-height: 1.25em; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; }
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
@media screen {
a.sourceLine::before { text-decoration: underline; color: initial; }
}
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.bn { color: #40a070; } /* BaseN */
code span.fl { color: #40a070; } /* Float */
code span.ch { color: #4070a0; } /* Char */
code span.st { color: #4070a0; } /* String */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.ot { color: #007020; } /* Other */
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.fu { color: #06287e; } /* Function */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code span.cn { color: #880000; } /* Constant */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.ss { color: #bb6688; } /* SpecialString */
code span.im { } /* Import */
code span.va { color: #19177c; } /* Variable */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.op { color: #666666; } /* Operator */
code span.bu { } /* BuiltIn */
code span.ex { } /* Extension */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.at { color: #7d9029; } /* Attribute */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i><b>1</b> Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-phd---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary PhD - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-phd-topic"><i class="fa fa-check"></i>The origins of the PhD topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="2" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>2</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="2.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>2.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="2.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>2.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="2.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>2.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>2.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="2.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>2.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="2.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>2.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="2.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>2.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>2.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="2.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>2.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="2.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>2.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="2.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>2.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="2.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>2.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="2.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>2.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>2.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>3</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="3.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>3.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="3.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>3.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="3.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>3.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="3.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>3.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>3.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="3.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>3.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="3.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>3.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="3.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>3.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="3.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>3.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="3.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>3.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="3.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>3.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="3.3.6" data-path="methods.html"><a href="methods.html#matrix-factorisation-methods"><i class="fa fa-check"></i><b>3.3.6</b> Matrix factorisation methods</a></li>
<li class="chapter" data-level="3.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>3.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="3.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>3.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="3.3.9" data-path="methods.html"><a href="methods.html#summary-2"><i class="fa fa-check"></i><b>3.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>3.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="3.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>3.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="3.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>3.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>3.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="4" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>4</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a></li>
<li class="chapter" data-level="5" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>5</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a></li>
<li class="chapter" data-level="6" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html"><i class="fa fa-check"></i><b>6</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="6.0.1" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html#comparing-metagenes-obtained-with-nmf-vs-ica."><i class="fa fa-check"></i><b>6.0.1</b> Comparing metagenes obtained with NMF vs ICA.</a></li>
<li class="chapter" data-level="6.1" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html#impact-of-modification-of-signatures-list-on-result-for-signature-based-deconvolution-methods"><i class="fa fa-check"></i><b>6.1</b> ? Impact of modification of signatures list on result for signature-based deconvolution methods</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>7</b> Deconvolution of transcriptomes and methylomes</a><ul>
<li class="chapter" data-level="7.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>7.1</b> From blind deconvolution to cell-type quantification: general overview</a><ul>
<li class="chapter" data-level="7.1.1" data-path="deconica.html"><a href="deconica.html#the-ica-based-deconvolution-of-transcriptomes"><i class="fa fa-check"></i><b>7.1.1</b> The ICA-based deconvolution of Transcriptomes</a></li>
<li class="chapter" data-level="7.1.2" data-path="deconica.html"><a href="deconica.html#interpretation-of-independent-components"><i class="fa fa-check"></i><b>7.1.2</b> Interpretation of Independent components</a></li>
<li class="chapter" data-level="7.1.3" data-path="deconica.html"><a href="deconica.html#transforming-metagenes-into-signature-matrix"><i class="fa fa-check"></i><b>7.1.3</b> Transforming metagenes into signature matrix</a></li>
<li class="chapter" data-level="7.1.4" data-path="deconica.html"><a href="deconica.html#regression-based-estimation-of-cell-type-proportions-solving-system-of-equations"><i class="fa fa-check"></i><b>7.1.4</b> Regression-based estimation of cell-type proportions : solving system of equations</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="deconica.html"><a href="deconica.html#deconica-r-package-for-ica-based-deconvolution"><i class="fa fa-check"></i><b>7.2</b> <em>DeconICA</em> R package for ICA-based deconvolution</a><ul>
<li class="chapter" data-level="7.2.1" data-path="deconica.html"><a href="deconica.html#demo"><i class="fa fa-check"></i><b>7.2.1</b> <em>Demo</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>8</b> Comparative analysis of cancer immune infiltration</a></li>
<li class="chapter" data-level="9" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>9</b> Heterogeneity of immune cell types</a></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="10" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>10</b> Discussion</a></li>
<li class="chapter" data-level="11" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>11</b> Conclusions and perspectives</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#dc-subtypes"><i class="fa fa-check"></i>Dc subtypes</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#dreamidea-challenge"><i class="fa fa-check"></i>DreamIdea Challenge</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#full-list-of-publications"><i class="fa fa-check"></i>Full list of publications</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv"><i class="fa fa-check"></i>CV</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="deconica" class="section level1">
<h1><span class="header-section-number">Chapter 7</span> Deconvolution of transcriptomes and methylomes</h1>
<p>We describe our methods in this chapter. The pre-eliminary pipeline and simple results are described in the manuscript submitted to Springer-Verlag’s Lecture Notes in Computer Science (<a href="http://www.springer.com/gb/computer-science/lncs">LNCS</a>) entitled <strong>Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific And Reproducible Immune-related Signals</strong> that is placed at the end of this chapter. In the final thesis final pipeline will be split into following structure</p>
<div id="from-blind-deconvolution-to-cell-type-quantification-general-overview" class="section level2">
<h2><span class="header-section-number">7.1</span> From blind deconvolution to cell-type quantification: general overview</h2>
<p>Few lines describing our idea</p>
<p>Figure?</p>
<div id="the-ica-based-deconvolution-of-transcriptomes" class="section level3">
<h3><span class="header-section-number">7.1.1</span> The ICA-based deconvolution of Transcriptomes</h3>
<ul>
<li>remind shortly ICA</li>
<li>describe stabilisation procedure <em>icasso</em></li>
<li>explain IC-metagene concept</li>
</ul>
<p>If completed add related section about two other ways of getting metagenes</p>
<ul>
<li>attractor metagenes</li>
<li>k-lines</li>
</ul>
</div>
<div id="interpretation-of-independent-components" class="section level3">
<h3><span class="header-section-number">7.1.2</span> Interpretation of Independent components</h3>
<div id="correlation-based-identification-of-confounding-factors" class="section level4">
<h4><span class="header-section-number">7.1.2.1</span> Correlation based identification of confounding factors</h4>
</div>
<div id="identification-of-immune-cell-types-with-enrichment-test-other" class="section level4">
<h4><span class="header-section-number">7.1.2.2</span> Identification of immune cell types with enrichment test / other</h4>
</div>
</div>
<div id="transforming-metagenes-into-signature-matrix" class="section level3">
<h3><span class="header-section-number">7.1.3</span> Transforming metagenes into signature matrix</h3>
</div>
<div id="regression-based-estimation-of-cell-type-proportions-solving-system-of-equations" class="section level3">
<h3><span class="header-section-number">7.1.4</span> Regression-based estimation of cell-type proportions : solving system of equations</h3>
</div>
</div>
<div id="deconica-r-package-for-ica-based-deconvolution" class="section level2">
<h2><span class="header-section-number">7.2</span> <em>DeconICA</em> R package for ICA-based deconvolution</h2>
<p>This part of the chapter will be adapted from package vignettes</p>
<p>It will contain</p>
<ul>
<li><p>technical package description</p></li>
<li><p>user guide</p></li>
<li><p>examples</p></li>
</ul>
<div id="demo" class="section level3">
<h3><span class="header-section-number">7.2.1</span> <em>Demo</em></h3>
<p>The package needs to installed and then imported.</p>
<pre class="sourceCode r" id="cb1"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="co">#import package</span></a>
<a class="sourceLine" id="cb1-2" data-line-number="2"><span class="kw">library</span>(deconica)</a></code></pre>
<p>Then we can perform our pipeline on sample data available in the package</p>
<pre class="sourceCode r" id="cb2"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="co">#import sample data</span></a>
<a class="sourceLine" id="cb2-2" data-line-number="2"><span class="kw">data</span>(BRCA)</a>
<a class="sourceLine" id="cb2-3" data-line-number="3"><span class="co">#decompose data</span></a>
<a class="sourceLine" id="cb2-4" data-line-number="4">fastica.res &lt;-<span class="st"> </span><span class="kw">run_fastica</span> (</a>
<a class="sourceLine" id="cb2-5" data-line-number="5">  BRCA,</a>
<a class="sourceLine" id="cb2-6" data-line-number="6">  <span class="dt">optimal =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb2-7" data-line-number="7">  <span class="dt">row.center =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb2-8" data-line-number="8">  <span class="dt">with.names =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb2-9" data-line-number="9">  <span class="dt">gene.names =</span> <span class="ot">NULL</span>,</a>
<a class="sourceLine" id="cb2-10" data-line-number="10">  <span class="dt">alg.typ =</span> <span class="st">&quot;parallel&quot;</span>,</a>
<a class="sourceLine" id="cb2-11" data-line-number="11">  <span class="dt">method =</span> <span class="st">&quot;C&quot;</span>,</a>
<a class="sourceLine" id="cb2-12" data-line-number="12">  <span class="dt">n.comp =</span> <span class="dv">100</span>,</a>
<a class="sourceLine" id="cb2-13" data-line-number="13">  <span class="dt">isLog =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb2-14" data-line-number="14">  <span class="dt">R =</span> <span class="ot">TRUE</span></a>
<a class="sourceLine" id="cb2-15" data-line-number="15">)</a>
<a class="sourceLine" id="cb2-16" data-line-number="16"><span class="co">#correlate obtained metagenes with Biton et al. </span></a>
<a class="sourceLine" id="cb2-17" data-line-number="17"><span class="co">#metagenes (by default)</span></a>
<a class="sourceLine" id="cb2-18" data-line-number="18">correlate.res &lt;-</a>
<a class="sourceLine" id="cb2-19" data-line-number="19"><span class="st">  </span><span class="kw">correlate_metagenes</span>(fastica.res<span class="op">$</span>S, fastica.res<span class="op">$</span>names)</a>
<a class="sourceLine" id="cb2-20" data-line-number="20"><span class="co">#assign reciprocal components</span></a>
<a class="sourceLine" id="cb2-21" data-line-number="21">assign.res &lt;-<span class="st"> </span><span class="kw">assign_metagenes</span>(correlate.res<span class="op">$</span>r)</a>
<a class="sourceLine" id="cb2-22" data-line-number="22"><span class="co">#identify components that are &gt;0.1 correlated with </span></a>
<a class="sourceLine" id="cb2-23" data-line-number="23"><span class="co">#immune and are not assigned to any other component</span></a>
<a class="sourceLine" id="cb2-24" data-line-number="24">identify.immune &lt;-</a>
<a class="sourceLine" id="cb2-25" data-line-number="25"><span class="st">  </span><span class="kw">identify_immune_ic</span>(correlate.res<span class="op">$</span>r[, <span class="st">&quot;M8_IMMUNE&quot;</span>], assign.res[, <span class="dv">2</span>])</a>
<a class="sourceLine" id="cb2-26" data-line-number="26"><span class="co">#test enrichment with fisher test in </span></a>
<a class="sourceLine" id="cb2-27" data-line-number="27"><span class="co">#Immgen signatures (by default)</span></a>
<a class="sourceLine" id="cb2-28" data-line-number="28">enrichment.res &lt;-<span class="st"> </span><span class="kw">gene.enrichment.test</span>(</a>
<a class="sourceLine" id="cb2-29" data-line-number="29">  fastica.res<span class="op">$</span>S,</a>
<a class="sourceLine" id="cb2-30" data-line-number="30">  fastica.res<span class="op">$</span>names,</a>
<a class="sourceLine" id="cb2-31" data-line-number="31">  <span class="kw">names</span>(identify.immune),</a>
<a class="sourceLine" id="cb2-32" data-line-number="32">  <span class="dt">gmt =</span> ImmgenHUGO,</a>
<a class="sourceLine" id="cb2-33" data-line-number="33">  <span class="dt">alternative =</span> <span class="st">&quot;greater&quot;</span>,</a>
<a class="sourceLine" id="cb2-34" data-line-number="34">  <span class="dt">p.adjust.method =</span> <span class="st">&quot;BH&quot;</span>,</a>
<a class="sourceLine" id="cb2-35" data-line-number="35">  <span class="dt">p.value.threshold =</span> <span class="fl">0.05</span></a>
<a class="sourceLine" id="cb2-36" data-line-number="36">)</a></code></pre>
The present state of the package is described in Fig <a href="deconica.html#fig:deconICAflow">7.1</a>.
<div class="figure" style="text-align: center"><span id="fig:deconICAflow"></span>
<img src="figures-ext/deconICApipeline.png" alt="State of the deconICA package in January 2018. The flow chart illustrates existing functions in the R package DeconICA. Squares represent functions, red are user-provided inputs, brown are inputs we provide but that can be replaced easily by user and in blu we marked outputs." width="100%" />
<p class="caption">
Figure 7.1: <strong>State of the deconICA package in January 2018.</strong> The flow chart illustrates existing functions in the R package <em>DeconICA</em>. Squares represent functions, red are user-provided inputs, brown are inputs we provide but that can be replaced easily by user and in blu we marked outputs.
</p>
</div>

<p>Next step will be:</p>
<ul>
<li>adding the metagenes selection and transformation into basis matrix for deconvolution</li>
<li>identifying confounding factors</li>
<li>estimating purity with an existing tool</li>
<li>running an equations solver (based on least squares or other type of regression) including basis matrix, confounding factors, purity</li>
<li>including regularisation factos</li>
<li>adding graphics</li>
<li>adding user interface</li>
<li>writing a demo (best interactive)</li>
</ul>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="comparison-of-reproducibility-between-nmf-and-ica.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="results.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/08-DeconICA.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
