<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="map.html">
<link rel="next" href="conclusions.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li><a href="index.html#unsupervised-deconvolution-of-bulk-omics-profiles-methodology-and-application-to-characterize-the-immune-landscape-in-tumors"><strong>Unsupervised deconvolution of bulk omics profiles: methodology and application to characterize the immune landscape in tumors</strong></a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="tcga-study-abbreviations.html"><a href="tcga-study-abbreviations.html"><i class="fa fa-check"></i>TCGA Study Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-ph.d.---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary Ph.D. - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-ph.d.topic"><i class="fa fa-check"></i>The origins of the Ph.D. topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="mstd.html"><a href="mstd.html#context"><i class="fa fa-check"></i><b>3.1</b> Context</a></li>
<li class="chapter" data-level="3.2" data-path="mstd.html"><a href="mstd.html#description"><i class="fa fa-check"></i><b>3.2</b> Description</a></li>
<li class="chapter" data-level="3.3" data-path="mstd.html"><a href="mstd.html#impact-on-the-further-work"><i class="fa fa-check"></i><b>3.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a><ul>
<li class="chapter" data-level="4.1" data-path="lva.html"><a href="lva.html#context-1"><i class="fa fa-check"></i><b>4.1</b> Context</a></li>
<li class="chapter" data-level="4.2" data-path="lva.html"><a href="lva.html#description-1"><i class="fa fa-check"></i><b>4.2</b> Description</a></li>
<li class="chapter" data-level="4.3" data-path="lva.html"><a href="lva.html#impact-on-the-further-work-1"><i class="fa fa-check"></i><b>4.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="nmfica.html"><a href="nmfica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.1" data-path="nmfica.html"><a href="nmfica.html#comparing-metagenes-obtained-with-nmf-versus-ica"><i class="fa fa-check"></i><b>5.1</b> Comparing metagenes obtained with NMF versus ICA</a></li>
<li class="chapter" data-level="5.2" data-path="nmfica.html"><a href="nmfica.html#summary-4"><i class="fa fa-check"></i><b>5.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#unsupervised-deconvolution"><i class="fa fa-check"></i><b>6.2</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#fastica-overdecomposition-protocol"><i class="fa fa-check"></i><b>6.2.1</b> FastICA overdecomposition protocol</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="deconica.html"><a href="deconica.html#interpretation-of-the-components"><i class="fa fa-check"></i><b>6.3</b> Interpretation of the components</a></li>
<li class="chapter" data-level="6.4" data-path="deconica.html"><a href="deconica.html#computing-the-abundance-of-the-identified-cell-types"><i class="fa fa-check"></i><b>6.4</b> Computing the abundance of the identified cell-types</a></li>
<li class="chapter" data-level="6.5" data-path="deconica.html"><a href="deconica.html#validation-of-the-abundance-estimation-with-deconica"><i class="fa fa-check"></i><b>6.5</b> Validation of the abundance estimation with <em>DeconICA</em></a><ul>
<li class="chapter" data-level="6.5.1" data-path="deconica.html"><a href="deconica.html#in-silico"><i class="fa fa-check"></i><b>6.5.1</b> <em>In silico</em></a></li>
<li class="chapter" data-level="6.5.2" data-path="deconica.html"><a href="deconica.html#in-vitro"><i class="fa fa-check"></i><b>6.5.2</b> <em>In vitro</em></a></li>
<li class="chapter" data-level="6.5.3" data-path="deconica.html"><a href="deconica.html#pbmc-transcriptome"><i class="fa fa-check"></i><b>6.5.3</b> PBMC transcriptome</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="deconica.html"><a href="deconica.html#summary-5"><i class="fa fa-check"></i><b>6.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Comparative analysis of cancer immune infiltration</a><ul>
<li class="chapter" data-level="7.1" data-path="results.html"><a href="results.html#background"><i class="fa fa-check"></i><b>7.1</b> Background</a></li>
<li class="chapter" data-level="7.2" data-path="results.html"><a href="results.html#methods-1"><i class="fa fa-check"></i><b>7.2</b> Methods</a><ul>
<li class="chapter" data-level="7.2.1" data-path="results.html"><a href="results.html#data-sources"><i class="fa fa-check"></i><b>7.2.1</b> Data sources</a></li>
<li class="chapter" data-level="7.2.2" data-path="results.html"><a href="results.html#the-deconica-pipeline-on-bulk"><i class="fa fa-check"></i><b>7.2.2</b> The DeconICA pipeline on bulk</a></li>
<li class="chapter" data-level="7.2.3" data-path="results.html"><a href="results.html#the-deconica-pipeline-on-single-cell"><i class="fa fa-check"></i><b>7.2.3</b> The DeconICA pipeline on single cell</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>7.3</b> Results</a></li>
<li class="chapter" data-level="7.4" data-path="results.html"><a href="results.html#discussion"><i class="fa fa-check"></i><b>7.4</b> Discussion</a></li>
<li class="chapter" data-level="7.5" data-path="results.html"><a href="results.html#conclussions"><i class="fa fa-check"></i><b>7.5</b> Conclussions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> A multiscale signalling network map of innate immune response in cancer reveals signatures of cell heterogeneity and functional polarization</a><ul>
<li class="chapter" data-level="8.1" data-path="map.html"><a href="map.html#context-2"><i class="fa fa-check"></i><b>8.1</b> Context</a></li>
<li class="chapter" data-level="8.2" data-path="map.html"><a href="map.html#description-2"><i class="fa fa-check"></i><b>8.2</b> Description</a></li>
<li class="chapter" data-level="8.3" data-path="map.html"><a href="map.html#discussion-and-perspectives"><i class="fa fa-check"></i><b>8.3</b> Discussion and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussiongenerale.html"><a href="discussiongenerale.html"><i class="fa fa-check"></i><b>9</b> Discussion</a></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a><ul>
<li class="chapter" data-level="10.1" data-path="conclusions.html"><a href="conclusions.html#conclusions-1"><i class="fa fa-check"></i><b>10.1</b> Conclusions</a></li>
<li class="chapter" data-level="10.2" data-path="conclusions.html"><a href="conclusions.html#perspectives"><i class="fa fa-check"></i><b>10.2</b> Perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#deconica-documentation"><i class="fa fa-check"></i>1 DeconICA documentation</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#introduction-to-deconica"><i class="fa fa-check"></i>1.1 Introduction to deconICA</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#running-fastica-with-icasso-stabilisation"><i class="fa fa-check"></i>1.2 Running fastICA with icasso stabilisation</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#reference-manual"><i class="fa fa-check"></i>1.3 Reference manual</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#publications-and-conferences"><i class="fa fa-check"></i>2 Publications and conferences</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#adjustment-of-dendritic-cells-to-the-breast-cancer-microenvironment-is-subset-specific"><i class="fa fa-check"></i>2.1 Adjustment of dendritic cells to the breast-cancer microenvironment is subset-specific</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#the-inconvenience-of-data-of-convenience-computational-research-beyond-post-mortem-analyses"><i class="fa fa-check"></i>2.2 The inconvenience of data of convenience: computational research beyond post-mortem analyses</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv-publications-conferences-courses"><i class="fa fa-check"></i>2.3 CV: publications, conferences, courses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a><ul>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#biological-terms"><i class="fa fa-check"></i>Biological terms</a></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#mathematical-terms"><i class="fa fa-check"></i>Mathematical terms</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="discussiongenerale" class="section level1">
<h1><span class="header-section-number">Chapter 9</span> Discussion</h1>
<blockquote>
<p><em>There are known knowns. These are things we know that we know. There are known unknowns. That is to say, there are things that we know we don’t know. But there are also unknown unknowns. There are things we don’t know we don’t know.</em>
Donald Rumsfeld</p>
</blockquote>
<p>In this work, I aimed to combine biological and mathematical expertise in order to approach the better understanding of the TME. I approached the complexity of TME with mathematical tools applied to transcriptome data. Starting this work, three years ago, little was know about the possibility of extracting immune signals from bulk tumor transcriptomes. <strong>In this Ph.D. project I tested the limits of detection of the immune signal from bulk transcriptomes with unsupervised methods.</strong></p>
<p>Through Chapters <a href="mstd.html#mstd">3</a>-<a href="nmfica.html#nmfica">5</a>, I tested parameters of Independent Components Analysis to optimize it for immune signal extraction. First publication (Chapter <a href="mstd.html#mstd">3</a>) focused on developing and computing the MSTD index. This work led to a better understanding of our methods. It also resulted in helpful observations for my further deconvolution work.</p>
<p>The work on MSTD index triggered a change in my protocol towards <em>overdecomposition</em> of transcriptomes. With my application of this protocol to six breast cancer transcriptomic datasets, I validated a hypothesis that ICA can extract reproducible signals from breast cancer transcriptomes and that some of those signals can be consequently labeled as cell-types (Chapter <a href="lva.html#lva">4</a>). In my additional work, I compared ICA and NMF algorithms showing that ICA breast cancer transcriptomes decomposition results in more interpretable readout (Chapter <a href="nmfica.html#nmfica">5</a>). Despite this result, no categorical statement on the superiority of ICA over NMF can be made. ICA and NMF differ in mathematical approach, and usually, a direct comparison (as I did) is avoided. Different <em>flavors</em> of NMF and ICA should also be tested with real data taking into account facility of use, the speed, and interpretability. Some recent works on ICA indicate that the performance of an algorithm can be strikingly different when applied to simulated and real data <span class="citation">(<span class="citeproc-not-found" data-reference-id="Ablin2018"><strong>???</strong></span>)</span>.</p>
<p>​It is worth mentioning that the tests were performed all in breast cancer as it is one of the most frequent cancers and there are publicly available datasets with large cohorts. Later results (Chapter <a href="results.html#results">7</a>) show that immune signal extracted through ICA deconvolution from breast cancer transcriptomes are the closest to the reference profiles for all three considered cell types (T cell, B cell, and Myeloid cells) and their <em>detectability</em> is quite high. This suggests that similar studies could not be reproduced in some cancer types, the, i.e., the T-cell signal is much further from the reference profiles (or not detectable) in colorectal cancer.</p>
<p>In Chapter <a href="deconica.html#deconica">6</a>, I implemented the methods used in the previous studies in a well-documented tool. It was an essential part of this work to make it reproducible and freely accessible to any researcher. I demonstrated that ICA is able to separate immune cell-types in blood transcriptomes with a competitive performance.</p>
<p>To put it in the context, in the introduction of Chapter <a href="methods.html#methods">2</a>, I presented a wide array of deconvolution tools. It can be observed that the field evolves impressively fast, and competition is fierce. <em>It would be interesting to measure the direct impact of each of the tool on the immunobiology field and percentage of the progress in immunotherapies due to the deconvolution methods.</em> What can be observed, it is the number and type of citations. For most of the methods published in theory-focused journals, they are rarely cited in biological works using the method. All in all, without doubts CIBERSORT met the most significant success, not only thanks to the solid scientific basis, extensive validation, and high impact journal but probably also because of the user-friendly web interface and simplicity (from the user perspective). Even though newer tools argue its accuracy for RNA-seq data, it will probably remain the champion of the field for a long time. Which does not mean that subsequent efforts are pointless as (1) CIBERSORT is under MIT license which means any new tools based on CIBERSORT methods belong to MIT (2) there is more to explore in the TME than only cell-type abundance (3) validity of CIBERSORT (and other methods) cannot be confirmed without gold standard benchmark.</p>
<p>The success of CIBERSORT brings into the light another critical topic which is reproducibility crisis in research <span class="citation">(<span class="citeproc-not-found" data-reference-id="Fanelli2018"><strong>???</strong></span>)</span>. It is still common to find publications about software without code or online access provided. Moreover, even if thanks to community and publishers pressure it is less and less frequent (as demonstrated with numbers in Chapter <a href="methods.html#methods">2</a>), still it is not trivial to understand and reproduce published tools. Often documentation is not provided or floppy. Scripts deposed on a public repository are not conformed to any standards and are not tested on different operating systems and software versions. One answer to this problem could be Docker image technology that allows sharing a <em>frozen</em> environment where the tool can run, and that is not affected by user informatics environment. However, this does not replace substantial documentation and examples provided with realized tools. I put my best effort to make my tool easy to use providing a tutorial and an R package following good practice guidelines with a sincere hope that users will be able to reproduce my work and build on without my extensive assistance.</p>
<p>As I stated in the Chapter <a href="methods.html#methods">2</a>, there is a schematic validation framework that is followed by many authors of deconvolution tools. This framework has one crucial problem: lack of gold standard validation datasets. Without a high-dimensional collection of bulk transcriptomic samples paired with an independent measure of cell-type proportions in different solid tumors, it is not possible to objectively assess the performance of published cell-type deconvolution tools. With such a benchmark it would be easier to make the field progress in the direction that can bring most benefits to immunobiology through trustful information.</p>
<p>I claimed that most of the tools published in the filed are using cell profiles available from the blood and some from cancer single cell studies. The few existing unsupervised tools were not widely applied in the context of tumor transcriptomes and, if tested, they were generally over-performed by supervised frameworks.</p>
<p>A crucial part of an unsupervised analysis is the data interpretation. So far most of the unsupervised methods proposed deconvolution algorithm without the facilitation of the interpretation of the resulting components. This may be a reason why supervised tools met great success. Another possibility is that the data-driven nature of the unsupervised methods, that can bring unexpected results, contrasted with the more predictable behavior of supervised methods discourage researches from experimenting with them.</p>
<p>This brings the discussion back to our awareness of the limits of deconvolution methods in the context of the tumor transcriptomes. Through the application of DeconICA to numerous transcriptomic datasets, I observed variable detectability depending on tumor type and a number of samples. For the tumor types, some trends can be observed and interpreted. In a case of AML which immune cell-type signals were highly correlated with the reference profiles, the samples are liquid to contrast with solid tissues. Then the differences in the decomposition of solid tumors and possible biological consequences of deviation of the cell-type specific signal from the reference to be understood. The number of samples is more of technical matter. However, it is surprising that for some datasets even a low number of samples (50-90) results in meaningful signal extraction, while for some datasets with &gt;100 samples, the decomposition can be unsuccessful. The general trend says more tumor samples, better chance to extract meaningful signals however it is not guaranteed. As the DeconICA signal extraction is data-driven and the same reference as used for all datasets, we can learn that probably use of reference genes in some tumor types may be difficult. In xCell publication <span class="citation">(Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>)</span>, the authors applied their deconvolution tool to TCGA tumor types. Their sample space tSNE visualization (Fig. <a href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs13059-017-1349-1/MediaObjects/13059_2017_1349_Fig4_HTML.gif">4b</a>) show that some tumor types samples can be clearly distinguished from other (AML, KIRC, SKCM, BRCA) and for some, it is more difficult (STAD, COAD, BLCA). This fact is not directly commented on by the authors. This result partially confirms our hypothesis that tissue type can play a role in the immune signal detectability.</p>
<p><strong>Deep deconvolution </strong> should also be discussed. Many authors <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>, <span class="citation">He et al. (<a href="#ref-He2018">2018</a>)</span>, <span class="citation">Aran, Hu, and Butte (<a href="#ref-Aran2017">2017</a>)</span>)</span> claimed to be able to distinguish cell subtypes from transcriptome. My results say that data-driven deconvolution is not suited to accomplish this task. As shown in Chapter <a href="#maps"><strong>??</strong></a>, I dissected the functional subgroups of the NK and Macrophages. Starting with single-cell resolution, I was able to detect groups of cells within studied cell types with distinct functional phenotype. Limited by the number of cells, the number of cell states was limited to two.</p>
<p>In Chapter <a href="results.html#results">7</a>, I decomposed five single-cell cancer transcriptomes: Melanoma, CRC, Breast, Liver, and Head and Neck. Applying ICA decomposition, I did not detect cell-subtypes. However, I identified cell types and common signals: cell-cycle, stress, etc. This shows the limits of the blind deconvolution methods for sub-types detection. Probably the gene expression profiles of immune cell sub-types are not strong enough to be detected in an unsupervised setup.</p>
<p>The overall diversity of resources should also be discussed. As mentioned in Chapter <a href="intro.html#intro">1</a>, TCGA is the most widely used resource in pan-cancer omics studies. Main reasons are:</p>
<ul>
<li>accessibility</li>
<li>multilevel data for the same patient</li>
<li>many cancer types</li>
<li>big number of patients</li>
</ul>
<p>Studies integrating many data sources for different cancer types are less frequent as an additional effort is necessary, and even then the removal of technical biases is not guaranteed.</p>
<p>In hoped to overcome this issue including datasets generated with different platforms and by different research groups. In this work, I integrated data sets of published by different authors and from different technologies RNA-seq and Microarray. TCGA data were analyzed together with a corpus of datasets shared with us by Aurélien de Reynès and publicly available data. Usually, data integration poses an essential challenge in transcriptomic studies. Different tools like Combat <span class="citation">(<span class="citeproc-not-found" data-reference-id="Johnson207"><strong>???</strong></span>)</span> were developed to overcome this issue. Thanks to ICA, we can get rid of most of the batch effects as they can be discovered as an independent factor. In <span class="citation">(Biton et al. <a href="#ref-Biton2014">2014</a>)</span>, the authors detected a particular batch effect thanks to one of the non-biological components. Therefore, ICA proposes a unique working framework that allowed me to compare signals from independently produced datasets in a single study without a need for re-normalization of the data. It was possible thanks to the use of ICA that removes the significant batch effects and the fact that the extracted from different datasets ICA components are comparable.</p>
<p>Despite the best research effort, the pan-cancer studies are missing some critical elements: time and space dimensions.</p>
<p>Most studies are based on omic data of tumor biopsies, which do not allow spatial localization of gathered information. In TCGA project pathology slides are available for a subset of patients. However, it is impossible to project the cell markers on them a posteriori or obtain gene expression from a selected area. Many studies, including <span class="citation">(<span class="citeproc-not-found" data-reference-id="Bindea2014"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Galon2018"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Dieu-Nosjean2014"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Sautes-Fridman2016"><strong>???</strong></span>)</span> demonstrated that the presence of the immune cells at the invasive margin versus tumor core, or the adjacent tertiary lymphoid structures (TLS) could have a different impact on the tumor evolution and patients response to prognosis.</p>
<p>It is possible to estimate the abundance of some immune cells based on pathological images and also relate the patterns of cells to the patients’ survival <span class="citation">(<span class="citeproc-not-found" data-reference-id="Saltz2018"><strong>???</strong></span>)</span>. This work can be done with an algorithm or by a pathologist. However, we cannot learn anything about the nature of the cells, cell state or functional subtype from the pathological images.</p>
<p>Therefore, even though most of the immunophenotyping works neglect this aspect, it is important to remember that even if we assume that we can correctly estimate cell type abundance in the sample, its impact can be confounded with spatial information.</p>
<p>There is a hope that with an evolution of spatial transcriptomic this gap in our knowledge will be filled in a near future.</p>
<p>Another crucial missing in most studies dimension is time. <strong>The immune system is a highly dynamic system</strong>. Immune cells secrete various molecules depending on stimulants coming from the surrounding tissue (endothelial cells, blood, and lymphatic vessels), tumor cells and other immune cells. Different stimuli can have additive, suppressive of synergistic effects, the order of stimuli can also matter <span class="citation">(<span class="citeproc-not-found" data-reference-id="Touzot2014"><strong>???</strong></span>)</span>.</p>
<p>The animal studies can profit from time resolution data, but it is linked with a sacrifice of the animal. In human studies, this approach cannot be considered. Thus, most of the patients’ samples are a <em>snapshot</em> at some time <span class="math inline">\(t\)</span>. With sequential biopsies, it is possible to have several time points and observe a sort of dynamic nature of TME. Unfortunately, this kind of data is not widely available for many patients, and pan-cancer multi-omics cancer immunophenotyping remains wishful thinking.</p>
<p>Finally, I would like to discuss the place of bulk deconvolution in a single cell transcriptomic. As I mentioned before in the introduction (Chapter <a href="intro.html#intro">1</a>), scRNA-seq data are bringing single-cell resolution to transcriptomics. This allows studying cell states and re-definition of cell types from gene expression perspective. With few cancer singe cell datasets, the scientific community learned a lot about patients and cell heterogeneity. The single cell signatures are already used in bulk deconvolution. In my work, I showed that immune cells signature extracted from a single cell are closely related to the signature extracted from the bulk transcriptomes. Due to technical challenges, scRNA-seq is not so far applied to large patient cohorts. It was also observed that rare cell types are not trivial to capture even with single-cell technologies. Even though scRNA-seq may one day replace the bulk RNA-seq, it will not be immediate. Till then both technologies should be used to cross-confirm findings and advance the state of our knowledge.</p>
<p>Despite continuous advances in research, our knowledge is limited on how tumor and immune cells interact with each other and how much this ecosystem is depending on intrinsic and extrinsic factors. The interest in the TME increased significantly over the last twenty years based on the percentage of publications dedicated to the TME. This was due mostly due to the vital breakthrough of immunotherapies. <strong>Medical advances become a motivation for many projects, mine included, to be founded and perform fundamental or applied for work on the TME. </strong>Many fundamental questions remain unanswered or controversial in the field. For example, the cell-type definition is an open issue that leads to multiple interpretations. Also, the role of different compartments of TME is now considered as context-dependent which means that it is difficult to infer a clinical-level conclusion and prognosis with heterogeneous patients. The putative predictive/explanatory still await large cohort studies followed by independent validation</p>
<p>As discussed briefly in the Chapter <a href="intro.html#intro">1</a>, researchers produce more and more data, on different scales: from molecule specific to system level which does not always directly leads to a generation of knowledge. <strong>Biological scientists need to join their efforts with analysts (mathematicians, physicists, engineers and computer scientists) to better exploit the available data, to generate the data in a smart way which would improve our understanding of complex biological systems</strong>.</p>
<p>Thanks to a multi-level transversal analysis of available data a few recent classifications of cancers based on TME features were proposed. It remains unclear how these classifications can be brought to clinical practice in a near future and what the real impact of those studies will be on patients’ survival, diagnosis, and treatment selection. The descriptive character of the immunophenotyping lack simplicity (combination of various machine-learning-derived scores and knowledge-based curation) and it is not guaranteed to be universal (applicable to different cohorts, technologies).</p>
<p>Possible research directions will be discussed in perspectives.</p>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Aran2017">
<p>Aran, Dvir, Zicheng Hu, and Atul J. Butte. 2017. “xCell: digitally portraying the tissue cellular heterogeneity landscape.” <em>Genome Biol.</em> 18 (1). BioMed Central: 220. doi:<a href="https://doi.org/10.1186/s13059-017-1349-1">10.1186/s13059-017-1349-1</a>.</p>
</div>
<div id="ref-Newman2015">
<p>Newman, Aaron M, Chih Long Liu, Michael R Green, Andrew J Gentles, Weiguo Feng, Yue Xu, Chuong D Hoang, Maximilian Diehn, and Ash A Alizadeh. 2015. “Robust enumeration of cell subsets from tissue expression profiles.” <em>Nat. Methods</em> 12 (5). Nature Publishing Group: 453–57. doi:<a href="https://doi.org/10.1038/nmeth.3337">10.1038/nmeth.3337</a>.</p>
</div>
<div id="ref-He2018">
<p>He, Jing, H. Carlo Maurer, Sam R. Holmstrom, Tao Su, Aqeel Ahmed, Hanina Hibshoosh, John A. Chabot, et al. 2018. “Transcriptional deconvolution reveals consistent functional subtypes of pancreatic cancer epithelium and stroma.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 288779. doi:<a href="https://doi.org/10.1101/288779">10.1101/288779</a>.</p>
</div>
<div id="ref-Biton2014">
<p>Biton, Anne, Isabelle Bernard-Pierrot, Yinjun Lou, Clémentine Krucker, Elodie Chapeaublanc, Carlota Rubio-Pérez, Nuria López-Bigas, et al. 2014. “Independent Component Analysis Uncovers the Landscape of the Bladder Tumor Transcriptome and Reveals Insights into Luminal and Basal Subtypes.” <em>Cell Rep.</em> 9 (4): 1235–45. doi:<a href="https://doi.org/10.1016/j.celrep.2014.10.035">10.1016/j.celrep.2014.10.035</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="map.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/11-Conclusions.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
