<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="map.html">
<link rel="next" href="conclusions.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li><a href="index.html#unsupervised-deconvolution-of-bulk-omics-profiles-methodology-and-application-to-characterize-the-immune-landscape-in-tumors"><strong>Unsupervised deconvolution of bulk omics profiles: methodology and application to characterize the immune landscape in tumors</strong></a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="tcga-study-abbreviations.html"><a href="tcga-study-abbreviations.html"><i class="fa fa-check"></i>TCGA Study Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-ph.d.---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary Ph.D. - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-ph.d.topic"><i class="fa fa-check"></i>The origins of the Ph.D. topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="mstd.html"><a href="mstd.html#context"><i class="fa fa-check"></i><b>3.1</b> Context</a></li>
<li class="chapter" data-level="3.2" data-path="mstd.html"><a href="mstd.html#description"><i class="fa fa-check"></i><b>3.2</b> Description</a></li>
<li class="chapter" data-level="3.3" data-path="mstd.html"><a href="mstd.html#impact-on-the-further-work"><i class="fa fa-check"></i><b>3.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a><ul>
<li class="chapter" data-level="4.1" data-path="lva.html"><a href="lva.html#context-1"><i class="fa fa-check"></i><b>4.1</b> Context</a></li>
<li class="chapter" data-level="4.2" data-path="lva.html"><a href="lva.html#description-1"><i class="fa fa-check"></i><b>4.2</b> Description</a></li>
<li class="chapter" data-level="4.3" data-path="lva.html"><a href="lva.html#impact-on-the-further-work-1"><i class="fa fa-check"></i><b>4.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="nmfica.html"><a href="nmfica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.1" data-path="nmfica.html"><a href="nmfica.html#comparing-metagenes-obtained-with-nmf-versus-ica"><i class="fa fa-check"></i><b>5.1</b> Comparing metagenes obtained with NMF versus ICA</a></li>
<li class="chapter" data-level="5.2" data-path="nmfica.html"><a href="nmfica.html#summary-4"><i class="fa fa-check"></i><b>5.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#unsupervised-deconvolution"><i class="fa fa-check"></i><b>6.2</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#fastica-overdecomposition-protocol"><i class="fa fa-check"></i><b>6.2.1</b> FastICA overdecomposition protocol</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="deconica.html"><a href="deconica.html#interpretation-of-the-components"><i class="fa fa-check"></i><b>6.3</b> Interpretation of the components</a></li>
<li class="chapter" data-level="6.4" data-path="deconica.html"><a href="deconica.html#computing-the-abundance-of-the-identified-cell-types"><i class="fa fa-check"></i><b>6.4</b> Computing the abundance of the identified cell-types</a></li>
<li class="chapter" data-level="6.5" data-path="deconica.html"><a href="deconica.html#validation-of-the-abundance-estimation-with-deconica"><i class="fa fa-check"></i><b>6.5</b> Validation of the abundance estimation with <em>DeconICA</em></a><ul>
<li class="chapter" data-level="6.5.1" data-path="deconica.html"><a href="deconica.html#in-silico"><i class="fa fa-check"></i><b>6.5.1</b> <em>In silico</em></a></li>
<li class="chapter" data-level="6.5.2" data-path="deconica.html"><a href="deconica.html#in-vitro"><i class="fa fa-check"></i><b>6.5.2</b> <em>In vitro</em></a></li>
<li class="chapter" data-level="6.5.3" data-path="deconica.html"><a href="deconica.html#pbmc-transcriptome"><i class="fa fa-check"></i><b>6.5.3</b> PBMC transcriptome</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="deconica.html"><a href="deconica.html#summary-5"><i class="fa fa-check"></i><b>6.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Landscape of tumor immunity, with focus on T-cells diversity, uncovered via unsupervised deconvolution applied to bulk transcriptomes across 30 tumor types</a><ul>
<li class="chapter" data-level="7.1" data-path="results.html"><a href="results.html#background"><i class="fa fa-check"></i><b>7.1</b> Background</a></li>
<li class="chapter" data-level="7.2" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>7.2</b> Results</a><ul>
<li class="chapter" data-level="7.2.1" data-path="results.html"><a href="results.html#unsupervised-deconvolution-of-119-datasets"><i class="fa fa-check"></i><b>7.2.1</b> Unsupervised deconvolution of 119 datasets</a></li>
<li class="chapter" data-level="7.2.2" data-path="results.html"><a href="results.html#interpretation-of-the-immune-components"><i class="fa fa-check"></i><b>7.2.2</b> Interpretation of the immune components</a></li>
<li class="chapter" data-level="7.2.3" data-path="results.html"><a href="results.html#quality-of-cell-type-signal-extraction-is-affected-by-tumor-type"><i class="fa fa-check"></i><b>7.2.3</b> Quality of cell-type signal extraction is affected by tumor type</a></li>
<li class="chapter" data-level="7.2.4" data-path="results.html"><a href="results.html#deconvoluted-bulk-transcriptome-t-cell-signatures-showcase-different-cell-states-in-cancers"><i class="fa fa-check"></i><b>7.2.4</b> Deconvoluted bulk transcriptome T cell signatures showcase different cell states in cancers</a></li>
<li class="chapter" data-level="7.2.5" data-path="results.html"><a href="results.html#single-cell-rnaseq-deconvolution-leads-to-cell-specific-metagenes"><i class="fa fa-check"></i><b>7.2.5</b> Single-cell RNAseq deconvolution leads to cell-specific metagenes</a></li>
<li class="chapter" data-level="7.2.6" data-path="results.html"><a href="results.html#landscape-of-patient-based-on-the-immune-infiltration-scores"><i class="fa fa-check"></i><b>7.2.6</b> Landscape of patient based on the immune infiltration scores</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="results.html"><a href="results.html#discussion"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
<li class="chapter" data-level="7.4" data-path="results.html"><a href="results.html#resconcl"><i class="fa fa-check"></i><b>7.4</b> Conclusions</a></li>
<li class="chapter" data-level="7.5" data-path="results.html"><a href="results.html#methods-1"><i class="fa fa-check"></i><b>7.5</b> Methods</a><ul>
<li class="chapter" data-level="7.5.1" data-path="results.html"><a href="results.html#data-sources"><i class="fa fa-check"></i><b>7.5.1</b> Data sources</a></li>
<li class="chapter" data-level="7.5.2" data-path="results.html"><a href="results.html#data-pre-processing"><i class="fa fa-check"></i><b>7.5.2</b> Data pre-processing</a></li>
<li class="chapter" data-level="7.5.3" data-path="results.html"><a href="results.html#data-analysis"><i class="fa fa-check"></i><b>7.5.3</b> Data analysis</a></li>
<li class="chapter" data-level="7.5.4" data-path="results.html"><a href="results.html#software"><i class="fa fa-check"></i><b>7.5.4</b> Software</a></li>
<li class="chapter" data-level="7.5.5" data-path="results.html"><a href="results.html#hardware"><i class="fa fa-check"></i><b>7.5.5</b> Hardware</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="results.html"><a href="results.html#supplementary"><i class="fa fa-check"></i><b>7.6</b> Supplementary</a><ul>
<li class="chapter" data-level="7.6.1" data-path="results.html"><a href="results.html#figures"><i class="fa fa-check"></i><b>7.6.1</b> Figures</a></li>
<li class="chapter" data-level="7.6.2" data-path="results.html"><a href="results.html#tables"><i class="fa fa-check"></i><b>7.6.2</b> Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> A multiscale signalling network map of innate immune response in cancer reveals signatures of cell heterogeneity and functional polarization</a><ul>
<li class="chapter" data-level="8.1" data-path="map.html"><a href="map.html#context-2"><i class="fa fa-check"></i><b>8.1</b> Context</a></li>
<li class="chapter" data-level="8.2" data-path="map.html"><a href="map.html#description-2"><i class="fa fa-check"></i><b>8.2</b> Description</a></li>
<li class="chapter" data-level="8.3" data-path="map.html"><a href="map.html#discussion-and-perspectives"><i class="fa fa-check"></i><b>8.3</b> Discussion and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussiongenerale.html"><a href="discussiongenerale.html"><i class="fa fa-check"></i><b>9</b> Discussion</a><ul>
<li class="chapter" data-level="9.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#improving-our-knowledge-of-how-best-apply-ica-to-transcriptomic-data"><i class="fa fa-check"></i><b>9.1</b> Improving our knowledge of how best apply ICA to transcriptomic data</a></li>
<li class="chapter" data-level="9.2" data-path="discussiongenerale.html"><a href="discussiongenerale.html#confronting-ica-with-other-matrix-factorization-approaches"><i class="fa fa-check"></i><b>9.2</b> Confronting ICA with other matrix factorization approaches</a></li>
<li class="chapter" data-level="9.3" data-path="discussiongenerale.html"><a href="discussiongenerale.html#deconvolution-methods-overall-impact-and-utility"><i class="fa fa-check"></i><b>9.3</b> Deconvolution methods overall impact and utility</a></li>
<li class="chapter" data-level="9.4" data-path="discussiongenerale.html"><a href="discussiongenerale.html#unsupervised-deconvolution-1"><i class="fa fa-check"></i><b>9.4</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="9.4.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#limitations-of-unsupervised-deconvolution"><i class="fa fa-check"></i><b>9.4.1</b> Limitations of unsupervised deconvolution</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="discussiongenerale.html"><a href="discussiongenerale.html#data-sources-1"><i class="fa fa-check"></i><b>9.5</b> Data sources</a></li>
<li class="chapter" data-level="9.6" data-path="discussiongenerale.html"><a href="discussiongenerale.html#missing-time-and-space-dimensions-in-transcriptomic-studies"><i class="fa fa-check"></i><b>9.6</b> Missing time and space dimensions in transcriptomic studies</a></li>
<li class="chapter" data-level="9.7" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-place-of-bulk-transcriptomes-in-single-cell-era"><i class="fa fa-check"></i><b>9.7</b> The place of bulk transcriptomes in single cell era</a></li>
<li class="chapter" data-level="9.8" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-field-of-computational-cancer-immunology"><i class="fa fa-check"></i><b>9.8</b> The field of computational cancer immunology</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a><ul>
<li class="chapter" data-level="10.1" data-path="conclusions.html"><a href="conclusions.html#conclusions-1"><i class="fa fa-check"></i><b>10.1</b> Conclusions</a></li>
<li class="chapter" data-level="10.2" data-path="conclusions.html"><a href="conclusions.html#perspectives"><i class="fa fa-check"></i><b>10.2</b> Perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#deconica-documentation"><i class="fa fa-check"></i>1 DeconICA documentation</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#introduction-to-deconica"><i class="fa fa-check"></i>1.1 Introduction to deconICA</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#running-fastica-with-icasso-stabilisation"><i class="fa fa-check"></i>1.2 Running fastICA with icasso stabilisation</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#reference-manual"><i class="fa fa-check"></i>1.3 Reference manual</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#publications-and-conferences"><i class="fa fa-check"></i>2 Publications and conferences</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#adjustment-of-dendritic-cells-to-the-breast-cancer-microenvironment-is-subset-specific"><i class="fa fa-check"></i>2.1 Adjustment of dendritic cells to the breast-cancer microenvironment is subset-specific</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#the-inconvenience-of-data-of-convenience-computational-research-beyond-post-mortem-analyses"><i class="fa fa-check"></i>2.2 The inconvenience of data of convenience: computational research beyond post-mortem analyses</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv-publications-conferences-courses"><i class="fa fa-check"></i>2.3 CV: publications, conferences, courses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a><ul>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#biological-terms"><i class="fa fa-check"></i>Biological terms</a></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#mathematical-terms"><i class="fa fa-check"></i>Mathematical terms</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="discussiongenerale" class="section level1">
<h1><span class="header-section-number">Chapter 9</span> Discussion</h1>
<blockquote>
<p><em>There are known knowns. These are things we know that we know. There are known unknowns. That is to say, there are things that we know we don’t know. But there are also unknown unknowns. There are things we don’t know we don’t know.</em>
Donald Rumsfeld</p>
</blockquote>
<p>In this work, I aimed to combine biological and mathematical expertise in order to approach the better understanding of the TME. I approached the complexity of TME with mathematical tools applied to transcriptome data. Starting t his work, three years ago, little was know about the possibility of extracting immune signals from bulk tumor transcriptomes. <strong>In this Ph.D. project I tested the limits of detection of the immune signal from bulk transcriptomes with unsupervised methods.</strong></p>
<div id="improving-our-knowledge-of-how-best-apply-ica-to-transcriptomic-data" class="section level2">
<h2><span class="header-section-number">9.1</span> Improving our knowledge of how best apply ICA to transcriptomic data</h2>
<p>Through Chapters <a href="mstd.html#mstd">3</a>-<a href="nmfica.html#nmfica">5</a>, I tested parameters of Independent Components Analysis to optimize it for immune signal extraction.</p>
<p>First publication (Chapter <a href="mstd.html#mstd">3</a>) focused on developing and computing the MSTD index. This work led to a better understanding of our methods. It also resulted in helpful observations for my further deconvolution work. So far, many authors applied ICA to transcriptomic data for separation of biological, technical and environmental factors <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2007">2007</a>; Biton et al. <a href="#ref-Biton2014">2014</a>; Gorban <a href="#ref-Gorban2007">2007</a>; Zinovyev et al. <a href="#ref-Zinovyev2013">2013</a>)</span>. However, mostly theoretical works were discussing the adequate number of dimensions proposing metrics based on purely mathematical/statistical arguments <span class="citation">(Cangelosi and Goriely <a href="#ref-Cangelosi2007">2007</a>; Kégl, n.d.; Bro et al. <a href="#ref-Bro2008">2008</a>)</span>. In contrast, as mentioned in the publication (Chapter <a href="mstd.html#mstd">3</a>)<span class="citation">(Kairov et al. <a href="#ref-Kairov2017">2017</a>)</span>, we asked a question: <em>which is the most informative number of components to use in the downstream analysis?</em>.</p>
<p>The work on MSTD index triggered a change in my protocol towards <em>overdecomposition</em> of transcriptomes. Most of previous works on unsupervised deconvolution of cell types, either impose the number of sources based (<span class="math inline">\(k\)</span>) based on previous knowledge <span class="citation">(Repsilber et al. <a href="#ref-Repsilber2010">2010</a>; Ogundijo and Wang <a href="#ref-Ogundijo2017">2017</a>; Newberg et al. <a href="#ref-Newberg2018">2018</a>)</span> or estimate is based on statistical indexes, such as Akaike information criterion (AICc) <span class="citation">(proposed initially by Hurvich <a href="#ref-Hurvich1989">1989</a>, applied to estimation of number of cell types by <span class="citation">Liebner, Huang, and Parvin (<a href="#ref-Liebner2014">2014</a>)</span>)</span>, Minimum Description Length <span class="citation">(proposed initially by Wax et al., n.d., applied in CAM model by <span class="citation">Wang et al. (<a href="#ref-Wang2016">2016</a>)</span>)</span>, leave-one-out cross-validation (LOOCV) <span class="citation">(Lähdesmäki et al. <a href="#ref-Lahdesmaki2005">2005</a>)</span> and  symmetric Kullback-Leibler divergence (SKLD) <span class="citation">(Zuckerman et al. <a href="#ref-Zuckerman2013">2013</a>)</span> (when sources are known) or else <span class="citation">Moffitt et al. (<a href="#ref-Moffitt2015">2015</a>)</span> applied a custom method testing many possible dimensions. To my knowledge, no previous work employes the <em>overdecomposition</em> approach to solving the problem of the unknown number of cell types, the methods mentioned above of <span class="math inline">\(k\)</span> estimation propose a lower number of dimensions (in the range 5-20).</p>
</div>
<div id="confronting-ica-with-other-matrix-factorization-approaches" class="section level2">
<h2><span class="header-section-number">9.2</span> Confronting ICA with other matrix factorization approaches</h2>
<p>With my application of this protocol to six breast cancer transcriptomic datasets, I validated a hypothesis that ICA can extract reproducible signals from breast cancer transcriptomes and that some of those signals can be consequently labeled as cell-types (Chapter <a href="lva.html#lva">4</a>). In my additional work, I compared ICA and NMF algorithms showing that ICA breast cancer transcriptomes decomposition results in more interpretable readout (Chapter <a href="nmfica.html#nmfica">5</a>). NMF and ICA were systematically compared in other fields of application (i.e., image deconvolution <span class="citation">(Mirzal <a href="#ref-Mirzal2017">2017</a>)</span>, web mining <span class="citation">(Chikhi, Rothenburger, and Aussenac-Gilles <a href="#ref-Chikhi2007">2007</a>)</span>; physiological signal processing <span class="citation">(Tresch, Cheung, and D’avella <a href="#ref-Tresch2006">2006</a>)</span>; face recognition <span class="citation">(Rajapakse and Wyse, n.d.)</span>) with the outcome depending on the context. On the topic of gene expression, little works compare NMF and ICA in a systematic study. Some authors claim the similar performance of both algorithms<span class="citation">(Kong, Mou, and Hu <a href="#ref-Kong2011">2011</a>; Schachtner et al. <a href="#ref-Schachtner2008">2008</a>)</span>, some assume that NMF is better suited for biological data because of the non-negativity assumption <span class="citation">(Li and Xie <a href="#ref-Li2013">2013</a>)</span>, other proved ICA not to recover cell states in the specific context <span class="citation">(Sokol et al. <a href="#ref-Sokol2015">2015</a>)</span> other discussed the outperformance of ICA over PCA <span class="citation">(Teschendorff et al. <a href="#ref-Teschendorff2007">2007</a>)</span>. My findings are consistent with the more extensive comparison of <span class="citation">Cantini, Kairov, et al. (<a href="#ref-Cantini2018">2018</a>)</span> from our group. Despite this result, no categorical statement on the superiority of ICA over NMF can be made. ICA and NMF differ in the mathematical approach, and its efficiency might be highly context-dependent. Some recent works on ICA indicate that the performance of an algorithm can be strikingly different when applied to simulated and real data <span class="citation">(Ablin, Cardoso, and Gramfort <a href="#ref-Ablin2018">2018</a>)</span>. Thus different <em>flavors</em> of NMF and ICA should also be tested on real experimental data taking into account user-friendliness, the speed, and interpretability.</p>
</div>
<div id="deconvolution-methods-overall-impact-and-utility" class="section level2">
<h2><span class="header-section-number">9.3</span> Deconvolution methods overall impact and utility</h2>
<p>In Chapter <a href="deconica.html#deconica">6</a>, I implemented the methods used in the previous studies in a well-documented tool. It was an essential part of this work to make it reproducible and freely accessible to any researcher. I demonstrated that ICA is able to separate immune cell-types in blood transcriptomes with a competitive performance.</p>
<p>To put it in the context, in the introduction of Chapter <a href="methods.html#methods">2</a>, I presented a wide array of deconvolution tools (Table <a href="methods.html#tab:mytab">2.1</a>). It can be observed that the field evolves impressively fast, and competition is fierce. <em>It would be interesting to measure the direct impact of each of the tool on the immunobiology field and percentage of the progress in immunotherapies due to the deconvolution methods.</em> What can be observed, it is the number and type of citations (columns <em>pop.index</em> and <em>citations</em> Tab. <a href="methods.html#tab:mytab">2.1</a>). For most of the methods published in theory-focused journals, they are rarely cited in biological works using the method. All in all, without doubts CIBERSORT <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>)</span>, met the most significant success, not only thanks to the solid scientific basis, extensive validation, and high impact journal but probably also because of the user-friendly web interface and simplicity (from the user perspective). Even though newer tools argue its accuracy for RNA-seq data <span class="citation">(Li, Liu, and Liu <a href="#ref-Li2017">2017</a>; Tamborero et al. <a href="#ref-Tamborero2018">2018</a>)</span>, it will probably remain the champion of the field for a long time. Which does not mean that subsequent efforts are pointless as (1) CIBERSORT is under MIT license which means any new tools based on CIBERSORT methods belong to MIT (2) there is more to explore in the TME than only cell-type abundance (3) validity of CIBERSORT (and other methods) cannot be confirmed without gold standard benchmark.</p>
<p>The success of CIBERSORT brings into the light another critical topic which is reproducibility crisis in research <span class="citation">(Fanelli <a href="#ref-Fanelli2018">2018</a>)</span>. It is still common to find publications about software without code or online access provided. Moreover, even if thanks to community and publishers pressure it is less and less frequent (as demonstrated with numbers in Chapter <a href="methods.html#methods">2</a>), still it is not trivial to understand and reproduce published tools. Often documentation is not provided or floppy. Scripts deposed on a public repository are not conformed to any standards and are not tested on different operating systems and software versions. One answer to this problem could be Docker image technology <span class="citation">(Boettiger <a href="#ref-Boettiger2014">2014</a>)</span> that allows sharing a <em>frozen</em> environment where the tool can run, and that is not affected by user informatics environment. However, this does not replace substantial documentation and examples provided with published tools. I put my best effort to make my tool easy to use providing a tutorial and an R package following good practice guidelines with a sincere hope that users will be able to reproduce my work and build on without my extensive assistance.</p>
<p>As I stated in the Chapter <a href="methods.html#methods">2</a>, there is a schematic validation framework that is followed by many authors of deconvolution tools <span class="citation">(i.e., Racle et al. <a href="#ref-Racle2017">2017</a>; Newman et al. <a href="#ref-Newman2015">2015</a>; Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>; Becht et al. <a href="#ref-Becht2016">2016</a>)</span>. This framework has one crucial problem: lack of gold standard validation datasets. Without a high-dimensional collection of bulk transcriptomic samples paired with an independent measure of cell-type proportions in different solid tumors, it is not possible to objectively assess the performance of published cell-type deconvolution tools. With such a benchmark it would be easier to make the field progress in the direction that can bring most benefits to immunobiology through trustful information.</p>
</div>
<div id="unsupervised-deconvolution-1" class="section level2">
<h2><span class="header-section-number">9.4</span> Unsupervised deconvolution</h2>
<p>I claimed that most of the tools published in the filed are using cell profiles available from the blood<span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>; Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>; Becht et al. <a href="#ref-Becht2016">2016</a>)</span> and some from cancer single cell studies <span class="citation">(Racle et al. <a href="#ref-Racle2017">2017</a>; Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>. The few existing unsupervised tools <span class="citation">(Repsilber et al. <a href="#ref-Repsilber2010">2010</a>; Wang et al. <a href="#ref-Wang2016">2016</a>; Gaujoux and Seoighe <a href="#ref-Gaujoux2010">2010</a>)</span> were not widely applied in the context of tumor transcriptomes. <strong>This work shall be the first one applying an unsupervised deconvolution framework pan-cancer with focus on cell-type deconvolution</strong>.</p>
<p>A crucial part of an unsupervised analysis is the data interpretation. So far most of the unsupervised methods proposed deconvolution algorithm without a method to interpret of the resulting components <span class="citation">(Newberg et al. <a href="#ref-Newberg2018">2018</a>; Wang et al. <a href="#ref-Wang2016">2016</a>)</span>. Among previous works, <span class="citation">Repsilber et al. (<a href="#ref-Repsilber2010">2010</a>)</span> proposed a <em>mapping strategy</em> extended by <span class="citation">Gaujoux and Seoighe (<a href="#ref-Gaujoux2012">2012</a>)</span> and included in <em>CellMix</em> package <span class="citation">(Gaujoux and Seoighe <a href="#ref-Gaujoux2013">2013</a>)</span>. The mapping strategy consisted on computing contingency table between discovered markers (top genes of NMF metagenes) and the known lists of source markers. The proposed method is close to the Fisher exact test enrichment. The main difference is the iterative scheme that avoids repetition of labels (the same cell type attributed to many components). It would be interesting to test this approach with the signals extracted from tumor transcriptomes.</p>
<p>The data-driven nature of the unsupervised methods can bring unexpected results, contrasted with the more predictable behavior of supervised methods and discourage researches from experimenting with them. <span class="citation">Gaujoux and Seoighe (<a href="#ref-Gaujoux2012">2012</a>)</span> overcomes this issue using the semi-supervised method that guides NMF to find desired components. I followed the utterly unsupervised strategy to learn from the data possible limitations.</p>
<div id="limitations-of-unsupervised-deconvolution" class="section level3">
<h3><span class="header-section-number">9.4.1</span> Limitations of unsupervised deconvolution</h3>
<p>It is worth mentioning that breast cancer transcriptome was mostly used to calibrate the ICA methods as it is one of the most frequent cancers and there are publicly available datasets with large cohorts. Later results (Chapter <a href="results.html#results">7</a>) show that immune signal extracted through ICA deconvolution from breast cancer transcriptomes are the closest to the reference profiles for all three considered cell types (T cell, B cell, and Myeloid cells) and their <em>detectability</em> is quite high. This suggests that similar studies could not be reproduced in some cancer types, the, i.e., the T-cell signal is much further from the reference profiles (or not detectable) in colorectal cancer.</p>
<p>Through the application of DeconICA to various transcriptomic datasets, I observed variable detectability depending on tumor type and a number of samples. For the tumor types, some trends can be observed and interpreted. In a case of AML which immune cell-type signals were highly correlated with the reference profiles, the samples are liquid to contrast with solid tissues. Then the differences in the decomposition of solid tumors and possible biological consequences of deviation of the cell-type specific signal from the reference to be understood. The number of samples is more of technical matter. However, it is surprising that for some datasets even a low number of samples (50-90) results in meaningful signal extraction, while for some datasets with &gt;100 samples, the decomposition can be unsuccessful. The general trend says more tumor samples, better chance to extract meaningful signals however it is not guaranteed. As the DeconICA signal extraction is data-driven and the same reference as used for all datasets, we can learn that probably use of reference genes in some tumor types may be difficult. In xCell publication <span class="citation">(Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>)</span>, the authors applied their deconvolution tool to TCGA tumor types. Their sample space tSNE visualization (Fig. <a href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs13059-017-1349-1/MediaObjects/13059_2017_1349_Fig4_HTML.gif">4b</a>) show that some tumor types samples can be clearly distinguished from other (AML, KIRC, SKCM, BRCA) and for some, it is more difficult (STAD, COAD, BLCA). This fact is not directly commented on by the authors. This result partially confirms our hypothesis that tissue type can play a role in the immune signal detectability.</p>
<p><strong>Deep deconvolution </strong> should also be discussed. <span class="citation">Shen-Orr and Gaujoux (<a href="#ref-Shen-Orr2013">2013</a>)</span> defines it as (in the context of PBMC):</p>
<blockquote>
<p>accurately estimating from whole blood or PBMC sample, the proportions and expression patterns of a higher number of cell subsets, going further down into the hematopoietic tree (T-regs, naive, memory, and effector cell subsets).</p>
</blockquote>
<p>Many authors <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>; He et al. <a href="#ref-He2018">2018</a>; Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>)</span> claimed to be able to distinguish cell subtypes from transcriptome. My results say that data-driven unsupervised deconvolution is not suited to accomplish this task. As shown in Chapter <a href="map.html#map">8</a>, I dissected the functional subgroups of the NK and Macrophages. Starting with single-cell resolution, I was able to detect groups of cells within studied cell types with distinct functional phenotype. Limited by the number of cells, the number of cell states was limited to two.</p>
<p>In Chapter <a href="results.html#results">7</a>, I decomposed five <strong>single-cell cancer transcriptomes</strong>: Melanoma, CRC, Breast, Liver, and Head and Neck. Applying ICA decomposition, I did not detect cell-subtypes. However, I identified cell types and common signals: cell-cycle, stress, etc. This shows the limits of the blind deconvolution methods for sub-types detection. For Liver T-cell, ICA did not detect T cell subsets. In the article describing the data <span class="citation">(Zheng et al. <a href="#ref-Zheng2017">2017</a>)</span> authors also did not detect a strong cell subtype effect but clusters that are shared by subtypes (Fig. 2A <span class="citation">(Zheng et al. <a href="#ref-Zheng2017">2017</a>)</span>). It is possible that gene expression profiles of immune cell sub-types are not strong enough to be detected in an unsupervised setup. ICA discovers cellular states that do not always overlap with established cell types or subtypes.</p>
</div>
</div>
<div id="data-sources-1" class="section level2">
<h2><span class="header-section-number">9.5</span> Data sources</h2>
<p>The overall diversity of resources should also be discussed. As mentioned in Chapter <a href="intro.html#intro">1</a>, TCGA is the most widely used resource in pan-cancer omics studies. Main reasons are:</p>
<ul>
<li>accessibility</li>
<li>multilevel data for the same patient</li>
<li>many cancer types</li>
<li>big number of patients</li>
</ul>
<p>Studies integrating many data sources <span class="citation">(Kairov et al. <a href="#ref-Kairov2017">2017</a>; Biton et al. <a href="#ref-Biton2014">2014</a>)</span> of different cancer types are less frequent as an additional effort is necessary, and even then the removal of technical biases is not guaranteed <span class="citation">(Battke and Nieselt <a href="#ref-Battke2011">2011</a>)</span>. Usually, data integration poses an essential challenge in transcriptomic studies<span class="citation">(Battke and Nieselt <a href="#ref-Battke2011">2011</a>)</span>. Different tools like Combat <span class="citation">(Johnson, Li, and Rabinovic <a href="#ref-Johnson2007">2007</a>)</span> were developed to overcome this issue.</p>
<p>In hoped to overcome this issue including datasets generated with different platforms and by different research groups. In this work, I integrated data sets of published by different authors and from different technologies RNA-seq and Microarray. TCGA data were analyzed together with a corpus of datasets shared with us by Aurélien de Reynès and publicly available data. This integration was slightly less successful for the scRNA-seq as the extracted T-cell components were more correlated with each other than with any bulk-extracted components. It can be due to the different data distribution. Many tools were developed to correct single cell data for confounding effects <span class="citation">(Wang et al. <a href="#ref-Wang2017">2017</a>; Kharchenko, Silberstein, and Scadden <a href="#ref-Kharchenko2014">2014</a>; Yuan et al. <a href="#ref-Yuan2017">2017</a>)</span>. Thanks to ICA, we can get rid of most of the batch effects as they can be discovered as an independent factor. In <span class="citation">(Biton et al. <a href="#ref-Biton2014">2014</a>)</span>, the authors detected a particular batch effect thanks to one of the non-biological components. Therefore, <strong>ICA proposes a unique working framework that allowed me to compare signals from independently produced datasets in a single study without a need for re-normalization of the data</strong>. It was possible thanks to the use of ICA that removes the significant batch effects and the fact that the extracted from different datasets ICA components are comparable.</p>
</div>
<div id="missing-time-and-space-dimensions-in-transcriptomic-studies" class="section level2">
<h2><span class="header-section-number">9.6</span> Missing time and space dimensions in transcriptomic studies</h2>
<p>Despite the best research effort, the pan-cancer studies are missing some critical elements: time and space dimensions.</p>
<p>Most studies are based on omic data of tumor biopsies, which do not allow spatial localization of gathered information. In TCGA project pathology slides are available for a subset of patients. However, it is impossible to project the cell markers on them a posteriori or obtain gene expression from a selected area. Many studies, including <span class="citation">(Bindea et al. <a href="#ref-Bindea2011">2011</a>; Galon et al. <a href="#ref-Galon2016">2016</a>; Dieu-Nosjean et al. <a href="#ref-Dieu-Nosjean2014">2014</a>; Sautès-Fridman et al. <a href="#ref-Sautes-Fridman2016">2016</a>)</span> demonstrated that the presence of the immune cells at the invasive margin versus tumor core, or the adjacent tertiary lymphoid structures (TLS) could have a different impact on the tumor evolution and patients response to prognosis.</p>
<p>It is possible to estimate the abundance of some immune cells based on pathological images and also relate the patterns of cells to the patients’ survival <span class="citation">(Saltz et al. <a href="#ref-Saltz2018">2018</a>)</span>. This work can be done with an algorithm or by a pathologist. However, we cannot learn anything about the nature of the cells, cell state or functional subtype from the pathological images.</p>
<p>Therefore, even though most of the immunophenotyping works <span class="citation">(Charoentong et al. <a href="#ref-Charoentong2017">2017</a>; Angelova et al. <a href="#ref-Angelova2015">2015</a>; Thorsson et al. <a href="#ref-Thorsson2018">2018</a>)</span> neglect this aspect, it is important to remember that even if we assume that we can correctly estimate cell type abundance in the sample, its impact can be confounded with spatial information.</p>
<p>There is a hope that with an evolution of spatial transcriptomic <span class="citation">(Ståhl et al. <a href="#ref-Stahl2016">2016</a>; Svensson, Teichmann, and Stegle <a href="#ref-Svensson2018">2018</a>; Svensson, Vento-Tormo, and Teichmann <a href="#ref-Svensson2018b">2018</a>; Moncada et al. <a href="#ref-Moncada2018">2018</a>)</span> this gap in our knowledge will be filled in a near future.</p>
<p>Another crucial missing in most studies dimension is time. <strong>The immune system is a highly dynamic system</strong>. Immune cells secrete various molecules depending on stimulants coming from the surrounding tissue (endothelial cells, blood, and lymphatic vessels), tumor cells and other immune cells. Different stimuli can have additive, suppressive of synergistic effects, the order of stimuli can also matter <span class="citation">(Touzot et al. <a href="#ref-Touzot2014">2014</a>)</span>. <span class="citation">Bergthaler and Menche (<a href="#ref-Bergthaler2017">2017</a>)</span> compare the network of immune interactions to a social network based on collaborations and communication features.</p>
<p>The animal studies can profit from time resolution data, but it is linked with a sacrifice of the animal. In human studies, this approach cannot be considered. Thus, most of the patients’ samples are a <em>snapshot</em> at some time <span class="math inline">\(t\)</span>. With sequential biopsies <span class="citation">(Dowlati et al. <a href="#ref-Dowlati2001">2001</a>; Gomez-Roca et al. <a href="#ref-Gomez-Roca2012">2012</a>; Roh et al. <a href="#ref-Roh2017">2017</a>)</span>, it is possible to have several time points and observe a sort of dynamic nature of TME. Unfortunately, this kind of data is not widely available for many patients, and pan-cancer multi-omics cancer immunophenotyping remains wishful thinking.</p>
</div>
<div id="the-place-of-bulk-transcriptomes-in-single-cell-era" class="section level2">
<h2><span class="header-section-number">9.7</span> The place of bulk transcriptomes in single cell era</h2>
<p>Finally, I would like to discuss the place of bulk deconvolution in a single cell transcriptomic. As I mentioned before in the introduction (Chapter <a href="intro.html#intro">1</a>), scRNA-seq data are bringing single-cell resolution to transcriptomics, that do profit cancer immunology research <span class="citation">(Neu et al. <a href="#ref-Neu2017">2017</a>; Giladi and Amit <a href="#ref-Giladi2018">2018</a>; Papalexi and Satija <a href="#ref-Papalexi2017">2017</a>)</span>. This allows studying cell states and re-definition of cell types from gene expression perspective. With few cancer singe cell datasets, the scientific community learned a lot about patients and cell heterogeneity. The single cell signatures are already used in bulk deconvolution. In my work, I showed that immune cells signature extracted from a single cell are closely related to the signature extracted from the bulk transcriptomes. Due to technical challenges, scRNA-seq is not so far applied to large patient cohorts <span class="citation">(Seymour et al., n.d.; Levitin, Yuan, and Sims <a href="#ref-Levitin2018">2018</a>)</span>. It was also observed that rare cell types are not trivial to capture even with single-cell technologies and new approaches are being developed for this purpose <span class="citation">(Torre et al. <a href="#ref-Torre2018">2018</a>)</span>. Even though scRNA-seq may one day replace the bulk RNA-seq, it will not be immediate. Till then both technologies should be used to cross-confirm findings and advance the state of our knowledge.</p>
</div>
<div id="the-field-of-computational-cancer-immunology" class="section level2">
<h2><span class="header-section-number">9.8</span> The field of computational cancer immunology</h2>
<p>Despite continuous advances in research, our knowledge is limited on how tumor and immune cells interact with each other and how much this ecosystem is depending on intrinsic and extrinsic factors. The interest in the TME increased significantly over the last twenty years based on the percentage of publications dedicated to the TME. This was due mostly due to the vital breakthrough of immunotherapies. <strong>Medical advances become a motivation for many projects, mine included, to be founded and perform fundamental or applied for work on the TME. </strong>Many fundamental questions remain unanswered or controversial in the field <span class="citation">(Zhang and Chen <a href="#ref-Zhang2018">2018</a>)</span>. For example, the cell-type definition is an open issue that leads to multiple interpretations <span class="citation">(Ediorial Cell Systems <a href="#ref-EdiorialCellSystems2017">2017</a>)</span>. Also, the role of different compartments of TME is now considered as context-dependent <span class="citation">(Janssen et al. <a href="#ref-Janssen2017">2017</a>)</span> which means that it is difficult to infer a clinical-level conclusion and prognosis with heterogeneous patients. The putative predictive/explanatory still await large cohort studies followed by independent validation</p>
<p>As discussed briefly in the Chapter <a href="intro.html#intro">1</a>, researchers produce more and more data <span class="citation">(Angerer et al. <a href="#ref-Angerer2017">2017</a>; Maia et al. <a href="#ref-Maia2017">2017</a>)</span>, on different scales: from molecule specific to system level which does not always directly leads to a generation of knowledge. <strong>Biological scientists need to join their efforts with analysts (mathematicians, physicists, engineers and computer scientists) to better exploit the available data, to generate the data in a smart way which would improve our understanding of complex biological systems</strong>.</p>
<p>Thanks to a multi-level transversal analysis of available data a few recent classifications of cancers based on TME features were proposed <span class="citation">(Bindea et al. <a href="#ref-Bindea2011">2011</a>; Charoentong et al. <a href="#ref-Charoentong2017">2017</a>; Thorsson et al. <a href="#ref-Thorsson2018">2018</a>; Tamborero et al. <a href="#ref-Tamborero2018">2018</a>)</span>. <strong>It remains unclear how these classifications can be brought to clinical practice in a near future and what the real impact of those studies will be on patients’ survival, diagnosis, and treatment selection.</strong> The descriptive character of the immunophenotyping lack simplicity (combination of various machine-learning-derived scores and knowledge-based curation) and it is not guaranteed to be universal (applicable to different cohorts, technologies).</p>
<p>Possible research directions will be discussed in perspectives.</p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Teschendorff2007">
<p>Teschendorff, Andrew E., Michel Journée, Pierre A. Absil, Rodolphe Sepulchre, and Carlos Caldas. 2007. “Elucidating the Altered Transcriptional Programs in Breast Cancer using Independent Component Analysis.” <em>PLoS Comput. Biol.</em> 3 (8). Public Library of Science: e161. doi:<a href="https://doi.org/10.1371/journal.pcbi.0030161">10.1371/journal.pcbi.0030161</a>.</p>
</div>
<div id="ref-Biton2014">
<p>Biton, Anne, Isabelle Bernard-Pierrot, Yinjun Lou, Clémentine Krucker, Elodie Chapeaublanc, Carlota Rubio-Pérez, Nuria López-Bigas, et al. 2014. “Independent Component Analysis Uncovers the Landscape of the Bladder Tumor Transcriptome and Reveals Insights into Luminal and Basal Subtypes.” <em>Cell Rep.</em> 9 (4): 1235–45. doi:<a href="https://doi.org/10.1016/j.celrep.2014.10.035">10.1016/j.celrep.2014.10.035</a>.</p>
</div>
<div id="ref-Gorban2007">
<p>Gorban, A. N. (Aleksandr Nikolaevich). 2007. <em>Principal manifolds for data visualization and dimension reduction</em>. Springer.</p>
</div>
<div id="ref-Zinovyev2013">
<p>Zinovyev, Andrei, Ulykbek Kairov, Tatyana Karpenyuk, and Erlan Ramanculov. 2013. “Blind source separation methods for deconvolution of complex signals in cancer biology.” <em>Biochem. Biophys. Res. Commun.</em> 430 (3). Academic Press: 1182–7. doi:<a href="https://doi.org/10.1016/J.BBRC.2012.12.043">10.1016/J.BBRC.2012.12.043</a>.</p>
</div>
<div id="ref-Cangelosi2007">
<p>Cangelosi, Richard, and Alain Goriely. 2007. “Component retention in principal component analysis with application to cDNA microarray data.” <em>Biol. Direct</em> 2. BioMed Central: 2. doi:<a href="https://doi.org/10.1186/1745-6150-2-2">10.1186/1745-6150-2-2</a>.</p>
</div>
<div id="ref-Bro2008">
<p>Bro, R., K. Kjeldahl, A. K. Smilde, and H. A. L. Kiers. 2008. “Cross-validation of component models: A critical look at current methods.” <em>Anal. Bioanal. Chem.</em> 390 (5): 1241–51. doi:<a href="https://doi.org/10.1007/s00216-007-1790-1">10.1007/s00216-007-1790-1</a>.</p>
</div>
<div id="ref-Kairov2017">
<p>Kairov, Ulykbek, Laura Cantini, Alessandro Greco, Askhat Molkenov, Urszula Czerwinska, Emmanuel Barillot, and Andrei Zinovyev. 2017. “Determining the optimal number of independent components for reproducible transcriptomic data analysis.” <em>BMC Genomics</em> 18 (1). BioMed Central: 712. doi:<a href="https://doi.org/10.1186/s12864-017-4112-9">10.1186/s12864-017-4112-9</a>.</p>
</div>
<div id="ref-Repsilber2010">
<p>Repsilber, Dirk, Sabine Kern, Anna Telaar, Gerhard Walzl, Gillian F Black, Joachim Selbig, Shreemanta K Parida, Stefan HE Kaufmann, and Marc Jacobsen. 2010. “Biomarker discovery in heterogeneous tissue samples -taking the in-silico deconfounding approach.” <em>BMC Bioinformatics</em> 11 (1). BioMed Central: 27. doi:<a href="https://doi.org/10.1186/1471-2105-11-27">10.1186/1471-2105-11-27</a>.</p>
</div>
<div id="ref-Ogundijo2017">
<p>Ogundijo, Oyetunji E., and Xiaodong Wang. 2017. “A sequential Monte Carlo approach to gene expression deconvolution.” Edited by Lars Kaderali. <em>PLoS One</em> 12 (10). Public Library of Science: e0186167. doi:<a href="https://doi.org/10.1371/journal.pone.0186167">10.1371/journal.pone.0186167</a>.</p>
</div>
<div id="ref-Newberg2018">
<p>Newberg, Lee A., Xiaowei Chen, Chinnappa D. Kodira, and Maria I. Zavodszky. 2018. “Computational de novo discovery of distinguishing genes for biological processes and cell types in complex tissues.” Edited by Paolo Provero. <em>PLoS One</em> 13 (3). Public Library of Science: e0193067. doi:<a href="https://doi.org/10.1371/journal.pone.0193067">10.1371/journal.pone.0193067</a>.</p>
</div>
<div id="ref-Hurvich1989">
<p>Hurvich, Clifford M. 1989. “Regression and time series model selection in small samples.” <em>Biometrika</em> 76 (2): 297–307. <a href="https://www.stat.berkeley.edu/{~}binyu/summer08/Hurvich.AICc.pdf" class="uri">https://www.stat.berkeley.edu/{~}binyu/summer08/Hurvich.AICc.pdf</a>.</p>
</div>
<div id="ref-Liebner2014">
<p>Liebner, David A., Kun Huang, and Jeffrey D. Parvin. 2014. “MMAD: microarray microdissection with analysis of differences is a computational tool for deconvoluting cell type-specific contributions from tissue samples.” <em>Bioinformatics</em> 30 (5). Oxford University Press: 682–89. doi:<a href="https://doi.org/10.1093/bioinformatics/btt566">10.1093/bioinformatics/btt566</a>.</p>
</div>
<div id="ref-Wang2016">
<p>Wang, Niya, Eric P. Hoffman, Lulu Chen, Li Chen, Zhen Zhang, Chunyu Liu, Guoqiang Yu, David M. Herrington, Robert Clarke, and Yue Wang. 2016. “Mathematical modelling of transcriptional heterogeneity identifies novel markers and subpopulations in complex tissues.” <em>Sci. Rep.</em> 6 (1). Nature Publishing Group: 18909. doi:<a href="https://doi.org/10.1038/srep18909">10.1038/srep18909</a>.</p>
</div>
<div id="ref-Lahdesmaki2005">
<p>Lähdesmäki, Harri, llya Shmulevich, Valerie Dunmire, Olli Yli-Harja, and Wei Zhang. 2005. “In silico microdissection of microarray data from heterogeneous cell populations.” <em>BMC Bioinformatics</em> 6 (1). BioMed Central: 54. doi:<a href="https://doi.org/10.1186/1471-2105-6-54">10.1186/1471-2105-6-54</a>.</p>
</div>
<div id="ref-Zuckerman2013">
<p>Zuckerman, Neta S., Yair Noam, Andrea J. Goldsmith, and Peter P. Lee. 2013. “A Self-Directed Method for Cell-Type Identification and Separation of Gene Expression Microarrays.” Edited by Teresa M. Przytycka. <em>PLoS Comput. Biol.</em> 9 (8). Public Library of Science: e1003189. doi:<a href="https://doi.org/10.1371/journal.pcbi.1003189">10.1371/journal.pcbi.1003189</a>.</p>
</div>
<div id="ref-Moffitt2015">
<p>Moffitt, Richard A, Raoud Marayati, Elizabeth L Flate, Keith E Volmar, S Gabriela Herrera Loeza, Katherine A Hoadley, Naim U Rashid, et al. 2015. “Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma.” <em>Nat. Genet.</em> 47 (10). Nature Publishing Group: 1168–78. doi:<a href="https://doi.org/10.1038/ng.3398">10.1038/ng.3398</a>.</p>
</div>
<div id="ref-Mirzal2017">
<p>Mirzal, Andri. 2017. “NMF versus ICA for blind source separation.” <em>Adv. Data Anal. Classif.</em> 11 (1): 25–48. doi:<a href="https://doi.org/10.1007/s11634-014-0192-4">10.1007/s11634-014-0192-4</a>.</p>
</div>
<div id="ref-Chikhi2007">
<p>Chikhi, Nacim Fateh, Bernard Rothenburger, and Nathalie Aussenac-Gilles. 2007. “A Comparison of Dimensionality Reduction Techniques for Web Structure Mining.” In <em>IEEE/Wic/Acm Int. Conf. Web Intell.</em>, 116–19. IEEE. doi:<a href="https://doi.org/10.1109/WI.2007.86">10.1109/WI.2007.86</a>.</p>
</div>
<div id="ref-Tresch2006">
<p>Tresch, Matthew C, Vincent C K Cheung, and Andrea D’avella. 2006. “Simulated and Experimental Data Sets Identification of Muscle Synergies: Evaluation on Matrix Factorization Algorithms for the Downloaded from.” <em>J. Neurophysiol.</em>, 2199–2212. doi:<a href="https://doi.org/10.1152/jn.00222.2005">10.1152/jn.00222.2005</a>.</p>
</div>
<div id="ref-Kong2011">
<p>Kong, Wei, Xiaoyang Mou, and Xiaohua Hu. 2011. “Exploring matrix factorization techniques for significant genes identification of Alzheimer’s disease microarray gene expression data.” <em>BMC Bioinformatics</em> 12 Suppl 5 (Suppl 5). BioMed Central: S7. doi:<a href="https://doi.org/10.1186/1471-2105-12-S5-S7">10.1186/1471-2105-12-S5-S7</a>.</p>
</div>
<div id="ref-Schachtner2008">
<p>Schachtner, R., D. Lutter, P. Knollmüller, A. M. Tomé, F. J. Theis, G. Schmitz, M. Stetter, P. Gómez Vilda, and E. W. Lang. 2008. “Knowledge-based gene expression classification via matrix factorization.” <em>Bioinformatics</em> 24 (15). Oxford University Press: 1688–97. doi:<a href="https://doi.org/10.1093/bioinformatics/btn245">10.1093/bioinformatics/btn245</a>.</p>
</div>
<div id="ref-Li2013">
<p>Li, Yi, and Xiaohui Xie. 2013. “A mixture model for expression deconvolution from RNA-seq in heterogeneous tissues.” <em>BMC Bioinforma. 2013 145</em> 14 (5). BioMed Central: S11. doi:<a href="https://doi.org/10.1186/1471-2105-14-s5-s11">10.1186/1471-2105-14-s5-s11</a>.</p>
</div>
<div id="ref-Sokol2015">
<p>Sokol, Ethan S., Sandhya Sanduja, Dexter X. Jin, Daniel H. Miller, Robert A. Mathis, and Piyush B. Gupta. 2015. “Perturbation-Expression Analysis Identifies RUNX1 as a Regulator of Human Mammary Stem Cell Differentiation.” Edited by Sheng Zhong. <em>PLOS Comput. Biol.</em> 11 (4). Public Library of Science: e1004161. doi:<a href="https://doi.org/10.1371/journal.pcbi.1004161">10.1371/journal.pcbi.1004161</a>.</p>
</div>
<div id="ref-Cantini2018">
<p>Cantini, Laura, Ulykbek Kairov, Aurelien de Reynies, Emmanuel Barillot, Francois Radvanyi, and Andrei Zinovyev. 2018. “Stabilized Independent Component Analysis outperforms other methods in finding reproducible signals in tumoral transcriptomes.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 318154. doi:<a href="https://doi.org/10.1101/318154">10.1101/318154</a>.</p>
</div>
<div id="ref-Ablin2018">
<p>Ablin, Pierre, Jean-François Cardoso, and Alexandre Gramfort. 2018. “Accelerating Likelihood Optimization for ICA on Real Signals.” In, 151–60. Springer, Cham. doi:<a href="https://doi.org/10.1007/978-3-319-93764-9_15">10.1007/978-3-319-93764-9_15</a>.</p>
</div>
<div id="ref-Newman2015">
<p>Newman, Aaron M, Chih Long Liu, Michael R Green, Andrew J Gentles, Weiguo Feng, Yue Xu, Chuong D Hoang, Maximilian Diehn, and Ash A Alizadeh. 2015. “Robust enumeration of cell subsets from tissue expression profiles.” <em>Nat. Methods</em> 12 (5). Nature Publishing Group: 453–57. doi:<a href="https://doi.org/10.1038/nmeth.3337">10.1038/nmeth.3337</a>.</p>
</div>
<div id="ref-Li2017">
<p>Li, Bo, Jun S. Liu, and X. Shirley Liu. 2017. “Revisit linear regression-based deconvolution methods for tumor gene expression data.” <em>Genome Biol.</em> 18 (1). BioMed Central: 127. doi:<a href="https://doi.org/10.1186/s13059-017-1256-5">10.1186/s13059-017-1256-5</a>.</p>
</div>
<div id="ref-Tamborero2018">
<p>Tamborero, David, Carlota Rubio-Perez, Ferran Muinos, Radhakrishnan Sabarinathan, Josep Maria Piulats, Aura Muntasell, Rodrigo Dienstmann, Nuria Lopez-Bigas, and Abel Gonzalez-Perez. 2018. “A pan-cancer landscape of interactions between solid tumors and infiltrating immune cell populations.” <em>Clin. Cancer Res.</em> American Association for Cancer Research, clincanres.3509.2017. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-17-3509">10.1158/1078-0432.CCR-17-3509</a>.</p>
</div>
<div id="ref-Fanelli2018">
<p>Fanelli, Daniele. 2018. “Opinion: Is science really facing a reproducibility crisis, and do we need it to?” <em>Proc. Natl. Acad. Sci.</em> 115 (11): 2628–31. doi:<a href="https://doi.org/10.1073/pnas.1708272114">10.1073/pnas.1708272114</a>.</p>
</div>
<div id="ref-Boettiger2014">
<p>Boettiger, Carl. 2014. “An introduction to Docker for reproducible research, with examples from the R environment.” <a href="https://arxiv.org/pdf/1410.0846.pdf" class="uri">https://arxiv.org/pdf/1410.0846.pdf</a>.</p>
</div>
<div id="ref-Racle2017">
<p>Racle, Julien, Kaat de Jonge, Petra Baumgaertner, Daniel E Speiser, and David Gfeller. 2017. “Simultaneous enumeration of cancer and immune cell types from bulk tumor gene expression data.” <em>Elife</em> 6. eLife Sciences Publications Limited: e26476. doi:<a href="https://doi.org/10.7554/eLife.26476">10.7554/eLife.26476</a>.</p>
</div>
<div id="ref-Aran2017">
<p>Aran, Dvir, Zicheng Hu, and Atul J. Butte. 2017. “xCell: digitally portraying the tissue cellular heterogeneity landscape.” <em>Genome Biol.</em> 18 (1). BioMed Central: 220. doi:<a href="https://doi.org/10.1186/s13059-017-1349-1">10.1186/s13059-017-1349-1</a>.</p>
</div>
<div id="ref-Becht2016">
<p>Becht, Etienne, Aurélien de Reyniès, Nicolas A Giraldo, Camilla Pilati, Bénédicte Buttard, Laetitia Lacroix, Janick Selves, Catherine Sautès-Fridman, Pierre Laurent-Puig, and Wolf Herman Fridman. 2016. “Immune and Stromal Classification of Colorectal Cancer Is Associated with Molecular Subtypes and Relevant for Precision Immunotherapy.” <em>Clin. Cancer Res.</em> 22 (16). American Association for Cancer Research: 4057–66. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-15-2879">10.1158/1078-0432.CCR-15-2879</a>.</p>
</div>
<div id="ref-Schelker2017">
<p>Schelker, Max, Sonia Feau, Jinyan Du, Nav Ranu, Edda Klipp, Birgit Schoeberl, Gavin MacBeath, et al. 2017. “Estimation of immune cell content in tumour tissue using single-cell RNA-seq data.” <em>Nat. Commun.</em> 8 (1). Nature Publishing Group: 2032. doi:<a href="https://doi.org/10.1038/s41467-017-02289-3">10.1038/s41467-017-02289-3</a>.</p>
</div>
<div id="ref-Gaujoux2010">
<p>Gaujoux, Renaud, and Cathal Seoighe. 2010. “A flexible R package for nonnegative matrix factorization.” <em>BMC Bioinformatics</em> 11 (1). BioMed Central: 367. doi:<a href="https://doi.org/10.1186/1471-2105-11-367">10.1186/1471-2105-11-367</a>.</p>
</div>
<div id="ref-Gaujoux2012">
<p>Gaujoux, Renaud, and Cathal Seoighe. 2012. “Semi-supervised Nonnegative Matrix Factorization for gene expression deconvolution: A case study.” <em>Infect. Genet. Evol.</em> 12 (5). Elsevier: 913–21. doi:<a href="https://doi.org/10.1016/J.MEEGID.2011.08.014">10.1016/J.MEEGID.2011.08.014</a>.</p>
</div>
<div id="ref-Gaujoux2013">
<p>Gaujoux, R., and C. Seoighe. 2013. “CellMix: a comprehensive toolbox for gene expression deconvolution.” <em>Bioinformatics</em> 29 (17): 2211–2. doi:<a href="https://doi.org/10.1093/bioinformatics/btt351">10.1093/bioinformatics/btt351</a>.</p>
</div>
<div id="ref-Shen-Orr2013">
<p>Shen-Orr, Shai S, and Renaud Gaujoux. 2013. “Computational deconvolution: extracting cell type-specific information from heterogeneous samples.” <em>Curr. Opin. Immunol.</em> 25 (5): 571–78. doi:<a href="https://doi.org/10.1016/j.coi.2013.09.015">10.1016/j.coi.2013.09.015</a>.</p>
</div>
<div id="ref-He2018">
<p>He, Jing, H. Carlo Maurer, Sam R. Holmstrom, Tao Su, Aqeel Ahmed, Hanina Hibshoosh, John A. Chabot, et al. 2018. “Transcriptional deconvolution reveals consistent functional subtypes of pancreatic cancer epithelium and stroma.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 288779. doi:<a href="https://doi.org/10.1101/288779">10.1101/288779</a>.</p>
</div>
<div id="ref-Zheng2017">
<p>Zheng, Chunhong, Liangtao Zheng, Jae-Kwang Yoo, Huahu Guo, Yuanyuan Zhang, Xinyi Guo, Boxi Kang, et al. 2017. “Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing.” <em>Cell</em> 169 (7). Cell Press: 1342–1356.e16. doi:<a href="https://doi.org/10.1016/J.CELL.2017.05.035">10.1016/J.CELL.2017.05.035</a>.</p>
</div>
<div id="ref-Battke2011">
<p>Battke, Florian, and Kay Nieselt. 2011. “Mayday SeaSight: Combined Analysis of Deep Sequencing and Microarray Data.” Edited by Vincent Laudet. <em>PLoS One</em> 6 (1). Public Library of Science: e16345. doi:<a href="https://doi.org/10.1371/journal.pone.0016345">10.1371/journal.pone.0016345</a>.</p>
</div>
<div id="ref-Johnson2007">
<p>Johnson, W. E., C. Li, and A. Rabinovic. 2007. “Adjusting batch effects in microarray expression data using empirical Bayes methods.” <em>Biostatistics</em> 8 (1): 118–27. doi:<a href="https://doi.org/10.1093/biostatistics/kxj037">10.1093/biostatistics/kxj037</a>.</p>
</div>
<div id="ref-Wang2017">
<p>Wang, Zeya, Shaolong Cao, Jeffrey S. Morris, Jaeil Ahn, Rongjie Liu, Svitlana Tyekucheva, Fan Gao, et al. 2017. “Transcriptome Deconvolution of Heterogeneous Tumor Samples with Immune Infiltration.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 146795. doi:<a href="https://doi.org/10.1101/146795">10.1101/146795</a>.</p>
</div>
<div id="ref-Kharchenko2014">
<p>Kharchenko, Peter V, Lev Silberstein, and David T Scadden. 2014. “Bayesian approach to single-cell differential expression analysis.” <em>Nat. Methods</em> 11 (7). Nature Publishing Group: 740–42. doi:<a href="https://doi.org/10.1038/nmeth.2967">10.1038/nmeth.2967</a>.</p>
</div>
<div id="ref-Yuan2017">
<p>Yuan, Guo-Cheng, Long Cai, Michael Elowitz, Tariq Enver, Guoping Fan, Guoji Guo, Rafael Irizarry, et al. 2017. “Challenges and emerging directions in single-cell analysis.” <em>Genome Biol.</em> 18 (1). BioMed Central: 84. doi:<a href="https://doi.org/10.1186/s13059-017-1218-y">10.1186/s13059-017-1218-y</a>.</p>
</div>
<div id="ref-Bindea2011">
<p>Bindea, Gabriela, Bernhard Mlecnik, Wolf-Herman Fridman, and Jérôme Galon. 2011. “The prognostic impact of anti-cancer immune response: a novel classification of cancer patients.” <em>Semin. Immunopathol.</em> 33 (4). Springer-Verlag: 335–40. doi:<a href="https://doi.org/10.1007/s00281-011-0264-x">10.1007/s00281-011-0264-x</a>.</p>
</div>
<div id="ref-Galon2016">
<p>Galon, J., B. A. Fox, C. B. Bifulco, G. Masucci, T. Rau, G. Botti, F. M. Marincola, et al. 2016. “Immunoscore and Immunoprofiling in cancer: An update from the melanoma and immunotherapy bridge 2015.” <em>J. Transl. Med.</em> 14 (1). doi:<a href="https://doi.org/10.1186/s12967-016-1029-z">10.1186/s12967-016-1029-z</a>.</p>
</div>
<div id="ref-Dieu-Nosjean2014">
<p>Dieu-Nosjean, Marie-Caroline, Jérémy Goc, Nicolas A. Giraldo, Catherine Sautès-Fridman, and Wolf Herman Fridman. 2014. “Tertiary lymphoid structures in cancer and beyond.” <em>Trends Immunol.</em> 35 (11): 571–80. doi:<a href="https://doi.org/10.1016/j.it.2014.09.006">10.1016/j.it.2014.09.006</a>.</p>
</div>
<div id="ref-Sautes-Fridman2016">
<p>Sautès-Fridman, Catherine, Myriam Lawand, Nicolas A. Giraldo, Hélène Kaplon, Claire Germain, Wolf Herman Fridman, and Marie-Caroline Dieu-Nosjean. 2016. “Tertiary Lymphoid Structures in Cancers: Prognostic Value, Regulation, and Manipulation for Therapeutic Intervention.” <em>Front. Immunol.</em> 7. Frontiers: 407. doi:<a href="https://doi.org/10.3389/fimmu.2016.00407">10.3389/fimmu.2016.00407</a>.</p>
</div>
<div id="ref-Saltz2018">
<p>Saltz, Joel, Rajarsi Gupta, Le Hou, Alexander J Lazar, Ashish Sharma, Tahsin Kurc, Pankaj Singh, et al. 2018. “Spatial Organization and Molecular Correlation of Tumor-Infiltrating Lymphocytes Using Deep Learning on Pathology Images Cell Reports Resource Spatial Organization and Molecular Correlation of Tumor-Infiltrating Lymphocytes Using Deep Learning on Pathology Images.” <em>Cell Rep.</em> 23: 181–200. doi:<a href="https://doi.org/10.1016/j.celrep.2018.03.086">10.1016/j.celrep.2018.03.086</a>.</p>
</div>
<div id="ref-Charoentong2017">
<p>Charoentong, Pornpimol, Francesca Finotello, Mihaela Angelova, Clemens Mayer, Mirjana Efremova, Dietmar Rieder, Hubert Hackl, and Zlatko Trajanoski. 2017. “Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade.” <em>Cell Rep.</em> 18 (1). Elsevier: 248–62. doi:<a href="https://doi.org/10.1016/j.celrep.2016.12.019">10.1016/j.celrep.2016.12.019</a>.</p>
</div>
<div id="ref-Angelova2015">
<p>Angelova, Mihaela, Pornpimol Charoentong, Hubert Hackl, Maria L Fischer, Rene Snajder, Anne M Krogsdam, Maximilian J Waldner, et al. 2015. “Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.” <em>Genome Biol.</em> 16 (1). BioMed Central: 64. doi:<a href="https://doi.org/10.1186/s13059-015-0620-6">10.1186/s13059-015-0620-6</a>.</p>
</div>
<div id="ref-Thorsson2018">
<p>Thorsson, Vésteinn, David L Gibbs, Scott D Brown, Denise Wolf, Dante S Bortone, Tai-Hsien Ou Yang, Eduard Porta-Pardo, et al. 2018. “The Immune Landscape of Cancer.” <em>Immunity</em> 48 (4). Elsevier: 812–830.e14. doi:<a href="https://doi.org/10.1016/j.immuni.2018.03.023">10.1016/j.immuni.2018.03.023</a>.</p>
</div>
<div id="ref-Stahl2016">
<p>Ståhl, Patrik L, Fredrik Salmén, Sanja Vickovic, Anna Lundmark, José Fernández Navarro, Jens Magnusson, Stefania Giacomello, et al. 2016. “Visualization and analysis of gene expression in tissue sections by spatial transcriptomics.” <em>Science</em> 353 (6294). American Association for the Advancement of Science: 78–82. doi:<a href="https://doi.org/10.1126/science.aaf2403">10.1126/science.aaf2403</a>.</p>
</div>
<div id="ref-Svensson2018">
<p>Svensson, Valentine, Sarah A Teichmann, and Oliver Stegle. 2018. “SpatialDE: identification of spatially variable genes.” <em>Nat. Methods</em> 15 (5). Nature Publishing Group: 343–46. doi:<a href="https://doi.org/10.1038/nmeth.4636">10.1038/nmeth.4636</a>.</p>
</div>
<div id="ref-Svensson2018b">
<p>Svensson, Valentine, Roser Vento-Tormo, and Sarah A Teichmann. 2018. “Exponential scaling of single-cell RNA-seq in the past decade.” <em>Nat. Protoc.</em> 13 (4). Nature Publishing Group: 599–604. doi:<a href="https://doi.org/10.1038/nprot.2017.149">10.1038/nprot.2017.149</a>.</p>
</div>
<div id="ref-Moncada2018">
<p>Moncada, Reuben, Florian Wagner, Marta Chiodin, Joseph C. Devlin, Maayan Baron, Cristina H. Hajdu, Diane Simeone, and Itai Yanai. 2018. “Building a tumor atlas: integrating single-cell RNA-Seq data with spatial transcriptomics in pancreatic ductal adenocarcinoma.” <em>bioRxiv</em>, March. Cold Spring Harbor Laboratory, 254375. doi:<a href="https://doi.org/10.1101/254375">10.1101/254375</a>.</p>
</div>
<div id="ref-Touzot2014">
<p>Touzot, Maxime, Maximilien Grandclaudon, Antonio Cappuccio, Takeshi Satoh, Carolina Martinez-Cingolani, Nicolas Servant, Nicolas Manel, and Vassili Soumelis. 2014. “Combinatorial flexibility of cytokine function during human T helper cell differentiation.” <em>Nat. Commun.</em> 5 (1). Nature Publishing Group: 3987. doi:<a href="https://doi.org/10.1038/ncomms4987">10.1038/ncomms4987</a>.</p>
</div>
<div id="ref-Bergthaler2017">
<p>Bergthaler, Andreas, and Jörg Menche. 2017. “The immune system as a social network.” <em>Nat. Immunol.</em> 18 (5): 481–82. doi:<a href="https://doi.org/10.1038/ni.3727">10.1038/ni.3727</a>.</p>
</div>
<div id="ref-Dowlati2001">
<p>Dowlati, A, J Haaga, S C Remick, T P Spiro, S L Gerson, L Liu, S J Berger, N A Berger, and J K Willson. 2001. “Sequential tumor biopsies in early phase clinical trials of anticancer agents for pharmacodynamic evaluation.” <em>Clin. Cancer Res.</em> 7 (10): 2971–6. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11595684" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/11595684</a>.</p>
</div>
<div id="ref-Gomez-Roca2012">
<p>Gomez-Roca, C. A., L. Lacroix, C. Massard, T. De Baere, F. Deschamps, R. Pramod, R. Bahleda, et al. 2012. “Sequential research-related biopsies in phase I trials: acceptance, feasibility and safety.” <em>Ann. Oncol.</em> 23 (5): 1301–6. doi:<a href="https://doi.org/10.1093/annonc/mdr383">10.1093/annonc/mdr383</a>.</p>
</div>
<div id="ref-Roh2017">
<p>Roh, Whijae, Pei-Ling Chen, Alexandre Reuben, Christine N Spencer, Peter A Prieto, John P Miller, Vancheswaran Gopalakrishnan, et al. 2017. “Integrated molecular analysis of tumor biopsies on sequential CTLA-4 and PD-1 blockade reveals markers of response and resistance.” <em>Sci. Transl. Med.</em> 9 (379). American Association for the Advancement of Science: eaah3560. doi:<a href="https://doi.org/10.1126/scitranslmed.aah3560">10.1126/scitranslmed.aah3560</a>.</p>
</div>
<div id="ref-Neu2017">
<p>Neu, Karlynn E, Qingming Tang, Patrick C Wilson, and Aly A Khan. 2017. “Single-Cell Genomics: Approaches and Utility in Immunology.” <em>Trends Immunol.</em> 38 (2). Elsevier: 140–49. doi:<a href="https://doi.org/10.1016/j.it.2016.12.001">10.1016/j.it.2016.12.001</a>.</p>
</div>
<div id="ref-Giladi2018">
<p>Giladi, Amir, and Ido Amit. 2018. “Single-Cell Genomics: A Stepping Stone for Future Immunology Discoveries.” <em>Cell</em> 172 (1-2). Elsevier: 14–21. doi:<a href="https://doi.org/10.1016/j.cell.2017.11.011">10.1016/j.cell.2017.11.011</a>.</p>
</div>
<div id="ref-Papalexi2017">
<p>Papalexi, Efthymia, and Rahul Satija. 2017. “Single-cell RNA sequencing to explore immune cell heterogeneity.” <em>Nat. Rev. Immunol.</em> 18 (1): 35–45. doi:<a href="https://doi.org/10.1038/nri.2017.76">10.1038/nri.2017.76</a>.</p>
</div>
<div id="ref-Levitin2018">
<p>Levitin, Hanna Mendes, Jinzhou Yuan, and Peter A Sims. 2018. “Single-Cell Transcriptomic Analysis of Tumor Heterogeneity.” <em>Trends in Cancer</em> 4 (4). Elsevier: 264–68. doi:<a href="https://doi.org/10.1016/j.trecan.2018.02.003">10.1016/j.trecan.2018.02.003</a>.</p>
</div>
<div id="ref-Torre2018">
<p>Torre, Eduardo, Hannah Dueck, Sydney Shaffer, Janko Gospocic, Rohit Gupte, Roberto Bonasio, Junhyong Kim, John Murray, and Arjun Raj. 2018. “Rare Cell Detection by Single-Cell RNA Sequencing as Guided by Single-Molecule RNA FISH.” <em>Cell Syst.</em> 6 (2). Elsevier: 171–179.e5. doi:<a href="https://doi.org/10.1016/j.cels.2018.01.014">10.1016/j.cels.2018.01.014</a>.</p>
</div>
<div id="ref-Zhang2018">
<p>Zhang, Hongming, and Jibei Chen. 2018. “Current status and future directions of cancer immunotherapy.” <em>J. Cancer</em> 9 (10): 1773–81. doi:<a href="https://doi.org/10.7150/jca.24577">10.7150/jca.24577</a>.</p>
</div>
<div id="ref-EdiorialCellSystems2017">
<p>Ediorial Cell Systems. 2017. “What Is Your Conceptual Definition of ‘Cell Type’ in the Context of a Mature Organism?” <em>Cell Syst.</em> 4 (3). Elsevier: 255–59. doi:<a href="https://doi.org/10.1016/j.cels.2017.03.006">10.1016/j.cels.2017.03.006</a>.</p>
</div>
<div id="ref-Janssen2017">
<p>Janssen, Louise M.E., Emma E. Ramsay, Craig D. Logsdon, and Willem W. Overwijk. 2017. “The immune system in cancer metastasis: friend or foe?” <em>J. Immunother. Cancer</em> 5 (1). BioMed Central: 79. doi:<a href="https://doi.org/10.1186/s40425-017-0283-9">10.1186/s40425-017-0283-9</a>.</p>
</div>
<div id="ref-Angerer2017">
<p>Angerer, Philipp, Lukas Simon, Sophie Tritschler, F. Alexander Wolf, David Fischer, and Fabian J. Theis. 2017. “Single cells make big data: New challenges and opportunities in transcriptomics.” <em>Curr. Opin. Syst. Biol.</em> 4. Elsevier: 85–91. doi:<a href="https://doi.org/10.1016/J.COISB.2017.07.004">10.1016/J.COISB.2017.07.004</a>.</p>
</div>
<div id="ref-Maia2017">
<p>Maia, Ana-Teresa, Stephen-John Sammut, Ana Jacinta-Fernandes, and Suet-Feung Chin. 2017. “Big data in cancer genomics.” <em>Curr. Opin. Syst. Biol.</em> 4. Elsevier: 78–84. doi:<a href="https://doi.org/10.1016/J.COISB.2017.07.007">10.1016/J.COISB.2017.07.007</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="map.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/11-Conclusions.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
