<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="annexes.html">
<link rel="next" href="post-scriptum-thesis-writing.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li><a href="index.html#unsupervised-deconvolution-of-bulk-omics-profiles-methodology-and-application-to-characterize-the-immune-landscape-in-tumors"><strong>Unsupervised deconvolution of bulk omics profiles: methodology and application to characterize the immune landscape in tumors</strong></a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="tcga-study-abbreviations.html"><a href="tcga-study-abbreviations.html"><i class="fa fa-check"></i>TCGA Study Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-ph.d.---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary Ph.D. - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-ph.d.topic"><i class="fa fa-check"></i>The origins of the Ph.D. topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="mstd.html"><a href="mstd.html#context"><i class="fa fa-check"></i><b>3.1</b> Context</a></li>
<li class="chapter" data-level="3.2" data-path="mstd.html"><a href="mstd.html#description"><i class="fa fa-check"></i><b>3.2</b> Description</a></li>
<li class="chapter" data-level="3.3" data-path="mstd.html"><a href="mstd.html#impact-on-the-further-work"><i class="fa fa-check"></i><b>3.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a><ul>
<li class="chapter" data-level="4.1" data-path="lva.html"><a href="lva.html#context-1"><i class="fa fa-check"></i><b>4.1</b> Context</a></li>
<li class="chapter" data-level="4.2" data-path="lva.html"><a href="lva.html#description-1"><i class="fa fa-check"></i><b>4.2</b> Description</a></li>
<li class="chapter" data-level="4.3" data-path="lva.html"><a href="lva.html#impact-on-the-further-work-1"><i class="fa fa-check"></i><b>4.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="nmfica.html"><a href="nmfica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.1" data-path="nmfica.html"><a href="nmfica.html#comparing-metagenes-obtained-with-nmf-versus-ica"><i class="fa fa-check"></i><b>5.1</b> Comparing metagenes obtained with NMF versus ICA</a></li>
<li class="chapter" data-level="5.2" data-path="nmfica.html"><a href="nmfica.html#summary-4"><i class="fa fa-check"></i><b>5.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#unsupervised-deconvolution"><i class="fa fa-check"></i><b>6.2</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#fastica-overdecomposition-protocol"><i class="fa fa-check"></i><b>6.2.1</b> FastICA overdecomposition protocol</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="deconica.html"><a href="deconica.html#interpretation-of-the-components"><i class="fa fa-check"></i><b>6.3</b> Interpretation of the components</a></li>
<li class="chapter" data-level="6.4" data-path="deconica.html"><a href="deconica.html#computing-the-abundance-of-the-identified-cell-types"><i class="fa fa-check"></i><b>6.4</b> Computing the abundance of the identified cell-types</a></li>
<li class="chapter" data-level="6.5" data-path="deconica.html"><a href="deconica.html#validation-of-the-abundance-estimation-with-deconica"><i class="fa fa-check"></i><b>6.5</b> Validation of the abundance estimation with <em>DeconICA</em></a><ul>
<li class="chapter" data-level="6.5.1" data-path="deconica.html"><a href="deconica.html#in-silico"><i class="fa fa-check"></i><b>6.5.1</b> <em>In silico</em></a></li>
<li class="chapter" data-level="6.5.2" data-path="deconica.html"><a href="deconica.html#in-vitro"><i class="fa fa-check"></i><b>6.5.2</b> <em>In vitro</em></a></li>
<li class="chapter" data-level="6.5.3" data-path="deconica.html"><a href="deconica.html#pbmc-transcriptome"><i class="fa fa-check"></i><b>6.5.3</b> PBMC transcriptome</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="deconica.html"><a href="deconica.html#summary-5"><i class="fa fa-check"></i><b>6.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Landscape of tumor immunity, with focus on T-cells diversity, uncovered via unsupervised deconvolution applied to bulk transcriptomes across 30 tumor types</a><ul>
<li class="chapter" data-level="7.1" data-path="results.html"><a href="results.html#background"><i class="fa fa-check"></i><b>7.1</b> Background</a></li>
<li class="chapter" data-level="7.2" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>7.2</b> Results</a><ul>
<li class="chapter" data-level="7.2.1" data-path="results.html"><a href="results.html#unsupervised-deconvolution-of-119-datasets"><i class="fa fa-check"></i><b>7.2.1</b> Unsupervised deconvolution of 119 datasets</a></li>
<li class="chapter" data-level="7.2.2" data-path="results.html"><a href="results.html#interpretation-of-the-immune-components"><i class="fa fa-check"></i><b>7.2.2</b> Interpretation of the immune components</a></li>
<li class="chapter" data-level="7.2.3" data-path="results.html"><a href="results.html#quality-of-cell-type-signal-extraction-is-affected-by-tumor-type"><i class="fa fa-check"></i><b>7.2.3</b> Quality of cell-type signal extraction is affected by tumor type</a></li>
<li class="chapter" data-level="7.2.4" data-path="results.html"><a href="results.html#deconvoluted-bulk-transcriptome-t-cell-signatures-showcase-different-cell-states-in-cancers"><i class="fa fa-check"></i><b>7.2.4</b> Deconvoluted bulk transcriptome T cell signatures showcase different cell states in cancers</a></li>
<li class="chapter" data-level="7.2.5" data-path="results.html"><a href="results.html#single-cell-rnaseq-deconvolution-leads-to-cell-specific-metagenes"><i class="fa fa-check"></i><b>7.2.5</b> Single-cell RNAseq deconvolution leads to cell-specific metagenes</a></li>
<li class="chapter" data-level="7.2.6" data-path="results.html"><a href="results.html#landscape-of-patient-based-on-the-immune-infiltration-scores"><i class="fa fa-check"></i><b>7.2.6</b> Landscape of patient based on the immune infiltration scores</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="results.html"><a href="results.html#discussion"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
<li class="chapter" data-level="7.4" data-path="results.html"><a href="results.html#resconcl"><i class="fa fa-check"></i><b>7.4</b> Conclusions</a></li>
<li class="chapter" data-level="7.5" data-path="results.html"><a href="results.html#methods-1"><i class="fa fa-check"></i><b>7.5</b> Methods</a><ul>
<li class="chapter" data-level="7.5.1" data-path="results.html"><a href="results.html#data-sources"><i class="fa fa-check"></i><b>7.5.1</b> Data sources</a></li>
<li class="chapter" data-level="7.5.2" data-path="results.html"><a href="results.html#data-pre-processing"><i class="fa fa-check"></i><b>7.5.2</b> Data pre-processing</a></li>
<li class="chapter" data-level="7.5.3" data-path="results.html"><a href="results.html#data-analysis"><i class="fa fa-check"></i><b>7.5.3</b> Data analysis</a></li>
<li class="chapter" data-level="7.5.4" data-path="results.html"><a href="results.html#software"><i class="fa fa-check"></i><b>7.5.4</b> Software</a></li>
<li class="chapter" data-level="7.5.5" data-path="results.html"><a href="results.html#hardware"><i class="fa fa-check"></i><b>7.5.5</b> Hardware</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="results.html"><a href="results.html#supplementary"><i class="fa fa-check"></i><b>7.6</b> Supplementary</a><ul>
<li class="chapter" data-level="7.6.1" data-path="results.html"><a href="results.html#figures"><i class="fa fa-check"></i><b>7.6.1</b> Figures</a></li>
<li class="chapter" data-level="7.6.2" data-path="results.html"><a href="results.html#tables"><i class="fa fa-check"></i><b>7.6.2</b> Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> A multiscale signalling network map of innate immune response in cancer reveals signatures of cell heterogeneity and functional polarization</a><ul>
<li class="chapter" data-level="8.1" data-path="map.html"><a href="map.html#context-2"><i class="fa fa-check"></i><b>8.1</b> Context</a></li>
<li class="chapter" data-level="8.2" data-path="map.html"><a href="map.html#description-2"><i class="fa fa-check"></i><b>8.2</b> Description</a></li>
<li class="chapter" data-level="8.3" data-path="map.html"><a href="map.html#discussion-and-perspectives"><i class="fa fa-check"></i><b>8.3</b> Discussion and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussiongenerale.html"><a href="discussiongenerale.html"><i class="fa fa-check"></i><b>9</b> Discussion</a><ul>
<li class="chapter" data-level="9.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#improving-our-knowledge-of-how-best-apply-ica-to-transcriptomic-data"><i class="fa fa-check"></i><b>9.1</b> Improving our knowledge of how best apply ICA to transcriptomic data</a></li>
<li class="chapter" data-level="9.2" data-path="discussiongenerale.html"><a href="discussiongenerale.html#confronting-ica-with-other-matrix-factorization-approaches"><i class="fa fa-check"></i><b>9.2</b> Confronting ICA with other matrix factorization approaches</a></li>
<li class="chapter" data-level="9.3" data-path="discussiongenerale.html"><a href="discussiongenerale.html#deconvolution-methods-overall-impact-and-utility"><i class="fa fa-check"></i><b>9.3</b> Deconvolution methods overall impact and utility</a></li>
<li class="chapter" data-level="9.4" data-path="discussiongenerale.html"><a href="discussiongenerale.html#unsupervised-deconvolution-1"><i class="fa fa-check"></i><b>9.4</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="9.4.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#limitations-of-unsupervised-deconvolution"><i class="fa fa-check"></i><b>9.4.1</b> Limitations of unsupervised deconvolution</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="discussiongenerale.html"><a href="discussiongenerale.html#data-sources-1"><i class="fa fa-check"></i><b>9.5</b> Data sources</a></li>
<li class="chapter" data-level="9.6" data-path="discussiongenerale.html"><a href="discussiongenerale.html#missing-time-and-space-dimensions-in-transcriptomic-studies"><i class="fa fa-check"></i><b>9.6</b> Missing time and space dimensions in transcriptomic studies</a></li>
<li class="chapter" data-level="9.7" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-place-of-bulk-transcriptomes-in-single-cell-era"><i class="fa fa-check"></i><b>9.7</b> The place of bulk transcriptomes in single cell era</a></li>
<li class="chapter" data-level="9.8" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-field-of-computational-cancer-immunology"><i class="fa fa-check"></i><b>9.8</b> The field of computational cancer immunology</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a><ul>
<li class="chapter" data-level="10.1" data-path="conclusions.html"><a href="conclusions.html#conclusions-1"><i class="fa fa-check"></i><b>10.1</b> Conclusions</a></li>
<li class="chapter" data-level="10.2" data-path="conclusions.html"><a href="conclusions.html#perspectives"><i class="fa fa-check"></i><b>10.2</b> Perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#deconica-documentation"><i class="fa fa-check"></i>1 DeconICA documentation</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#introduction-to-deconica"><i class="fa fa-check"></i>1.1 Introduction to deconICA</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#running-fastica-with-icasso-stabilisation"><i class="fa fa-check"></i>1.2 Running fastICA with icasso stabilisation</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#reference-manual"><i class="fa fa-check"></i>1.3 Reference manual</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#publications-and-conferences"><i class="fa fa-check"></i>2 Publications and conferences</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#adjustment-of-dendritic-cells-to-the-breast-cancer-microenvironment-is-subset-specific"><i class="fa fa-check"></i>2.1 Adjustment of dendritic cells to the breast-cancer microenvironment is subset-specific</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#the-inconvenience-of-data-of-convenience-computational-research-beyond-post-mortem-analyses"><i class="fa fa-check"></i>2.2 The inconvenience of data of convenience: computational research beyond post-mortem analyses</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv-publications-conferences-courses"><i class="fa fa-check"></i>2.3 CV: publications, conferences, courses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a><ul>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#biological-terms"><i class="fa fa-check"></i>Biological terms</a></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#mathematical-terms"><i class="fa fa-check"></i>Mathematical terms</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="glossary" class="section level1 unnumbered">
<h1>Glossary</h1>
<div id="biological-terms" class="section level2 unnumbered">
<h2>Biological terms</h2>
<p><strong>bulk data</strong> - a pooled assay using a weighted average of a bulk cell sample from a particular tissue (i.e., a large population of cells), obscuring cell-to-cell variation</p>
<p><strong>cancer biopsy</strong> - the removal of cells or tissues for examination by a pathologist from a cancer patient. The most common types include: (1) incisional biopsy, in which only a sample of tissue is removed; (2) excisional biopsy, in which an entire lump or suspicious area is removed; and (3) needle biopsy, in which a sample of tissue or fluid is removed with a needle. <span class="citation">(“Definition of biopsy - NCI Dictionary of Cancer Terms - National Cancer Institute” <a href="#ref-biopsy">2018</a>)</span></p>
<p><strong>cancer subtype</strong> - describes the smaller groups that a type of cancer can be divided into, based on certain characteristics of the cancer cells. These characteristics include how the cancer cells look under a microscope and whether there are certain substances in or on the cells or certain changes to the DNA of the cells. It is important to know the subtype of a cancer in order to plan treatment and determine prognosis. <span class="citation">(“Definition of cancer subtype - NCI Dictionary of Cancer Terms - National Cancer Institute” <a href="#ref-cansub">2018</a>)</span></p>
<p><strong>cytotoxic</strong> - toxic to cells, cell-toxic, cell-killing. Any agent or process that kills cells. Chemotherapy and radiotherapy are forms of cytotoxic therapy. They kill cells. <span class="citation">(“Definition of Cytotoxic” <a href="#ref-cytotoxic">2018</a>)</span></p>
<p><strong>effectors molecule</strong> - small molecule that selectively binds to a protein and regulates its biological activity. In this manner, effector molecules act as ligands that can increase or decrease enzyme activity, gene expression, or cell signaling. <span class="citation">(Cambronne and Roy <a href="#ref-Cambronne2006">2006</a>)</span></p>
<p><strong>gene enrichment analysis</strong> - a means to characterize biological attributes in a given <em>gene</em> set <span class="citation">(“Analysis of DNA Chips and Gene Networks c,” n.d.)</span></p>
<p><strong>gene expression</strong> - the generation of a functional gene product from the information encoded by a gene, through the processes of transcription and translation. Gene products are often proteins, however non-protein coding genes can encode functional RNA, including ribosomal RNA (rRNA), transfer RNA (tRNA) or small nuclear RNA (snRNA). <span class="citation">(“Gene expression definition” <a href="#ref-geneexprsNature">2018</a>)</span></p>
<p><strong>genome</strong> - the full genetic complement of an organism, encoded in either DNA or, in many viruses, RNA. It includes both the genes and non-coding sequences. <span class="citation">(“Genome” <a href="#ref-genomeNature">2018</a>)</span></p>
<p><strong>immunosuppression</strong> - suppression of the immune system and its ability to fight infection. Immunosuppression may result from certain diseases, such as AIDS or lymphoma, or from certain drugs, such as some of those used to treat cancer. Immunosuppression may also be deliberately induced with drugs, as in preparation for bone marrow or other organ transplantation, to prevent the rejection of a transplant. Also known as immunodepression. <span class="citation">(“Definition of Immunosuppression” <a href="#ref-immunosupp">2018</a>)</span></p>
<p><strong>inducers</strong> - a molecule, usually a substrate of a specific enzyme pathway, that combines with and deactivates an active repressor(produced by a regulator gene); this allows an operator gene previously repressed to activate the structural genes controlled by it to result in enzyme production; a homeostatic mechanism for regulating enzyme production in an inducible enzyme system. <span class="citation">(“Inducer (biology) | definition of Inducer (biology) by Medical dictionary” <a href="#ref-inducer">2018</a>)</span></p>
<p><strong>ligand</strong> - a molecule, as an antibody, hormone, or drug, that binds to a receptor. <span class="citation">(“Ligand | Define Ligand at Dictionary.com” <a href="#ref-ligand">2018</a>)</span></p>
<p><strong>liquid tissue</strong> - a collection of similar cell that perform a particular function; liquid such as blood and lymph, carried food, waste, products, and hormones through the body <span class="citation">(“Tissue &amp; Organ Flashcards” <a href="#ref-tissue">2018</a>)</span></p>
<p><strong>marker gene</strong> - a detectable genetic trait or segment of DNA that can be identified and tracked. A marker gene can serve as a flag for another gene, sometimes called the target gene. A marker gene must be on the same chromosome as the target gene and near enough to it so that the two genes (the marker gene and the target gene) are genetically linked and are usually inherited together. <span class="citation">(“Definition of Marker gene” <a href="#ref-marker">2018</a>)</span></p>
<p><strong>molecular biology</strong>- the field of biology that studies the composition, structure and interactions of cellular molecules  such as nucleic acids and proteins  that carry out the biological processes essential for the cells functions and maintenance <span class="citation">(“Molecular biology definition” <a href="#ref-biomolNature">2018</a>)</span></p>
<p><strong>omics</strong> - informally refers to a field of study in biology ending in -omics, such as genomics, proteomics or metabolomics. <span class="citation">(“Omics definition” <a href="#ref-omics">2018</a>)</span></p>
<p><strong>phenotyping</strong> - attribution a phenotype; the visible or observable expression of the results of genes, combined with the environmental influence on an organism’s appearance or behavior. <span class="citation">(“Examples of Genotype &amp; Phenotype” <a href="#ref-pheno">2018</a>)</span></p>
<p><strong>receptor</strong> - in cell biology, a structure on the surface of a cell (or inside a cell) that selectively receives and binds a specific substance. There are many receptors. There is a receptor for (<a href="https://www.medicinenet.com/script/main/art.asp?articlekey=3989">insulin</a>; there is a receptor for low-density <a href="https://www.medicinenet.com/script/main/art.asp?articlekey=15389">lipoproteins</a> (<a href="https://www.medicinenet.com/script/main/art.asp?articlekey=6232">LDL</a>); etc. T <span class="citation">(“Definition of Receptor” <a href="#ref-receptor">2018</a>)</span></p>
<p><strong>subtyping</strong> - describes the smaller groups that a type of cancer can be divided into, based on certain characteristics of the cancer cells. These characteristics include how the cancer cells look under a microscope and whether there are certain substances in or on the cells or certain changes to the DNA of the cells. It is important to know the subtype of a cancer in order to plan treatment and determine prognosis. <span class="citation">(“Definition of cancer subtype - NCI Dictionary of Cancer Terms - National Cancer Institute” <a href="#ref-cansub">2018</a>)</span></p>
<p><strong>tumor purity</strong> - the proportion of cancer cells in the admixture <span class="citation">(Aran, Sirota, and Butte <a href="#ref-Aran2015">2015</a>)</span></p>

</div>
<div id="mathematical-terms" class="section level2 unnumbered">
<h2>Mathematical terms</h2>
<p><strong>basis matrix</strong> - in cell-type deconvolution, the characteristic expression profiles for each of the cell types to be estimated used in the regression-based deconvolution</p>
<p><strong>Bayes’ theorem</strong> - is a formula that describes how to update the probabilities of hypotheses when given evidence <span class="citation">(“Bayes’ Theorem and Conditional Probability | Brilliant Math &amp; Science Wiki” <a href="#ref-bayes">2018</a>)</span></p>
<p><strong>condition number</strong> - a function with respect to an argument measures how much the output value of the function can change for a small change in the input argument. A problem with a low condition number is said to be <strong>well-conditioned</strong>, while a problem with a high condition number is said to be <strong>ill-conditioned</strong>. In linear regression the condition number can be used as a diagnostic for multicollinearity <span class="citation">(Belsley, Kuh, and Welsch <a href="#ref-Belsley1980">1980</a>)</span>.</p>
<p><strong>correlation</strong> - a quantity measuring the extent of the interdependence of variable quantities; a mutual relationship or connection between two or more things <span class="citation">(“correlation - Google Search” <a href="#ref-corr">2018</a>)</span></p>
<p><strong>covariance</strong> - of two variables x and y in a data set measures how the two are linearly related. A positive covariance would indicate a positive linear relationship between the variables, and a negative covariance would indicate the opposite. <span class="citation">(“Covariance | R Tutorial” <a href="#ref-cov">2018</a>)</span></p>
<p><strong>deconvolution</strong> - <em>Math</em>:the resolution of a convolution function into the functions from which it was formed in order to separate their effects; <em>Common</em>: a process of resolving something into its constituent elements or removing complication <span class="citation">(“deconvolution | Definition of deconvolution in English by Oxford Dictionaries” <a href="#ref-deconvolution">2018</a>)</span></p>
<p><strong>diagonal matrix</strong> - a matrix having non-zero elements only in the diagonal running from the upper left to the lower right. <span class="citation">(“diagonal matrix - Google Search” <a href="#ref-diag">2018</a>)</span></p>
<p><strong>dimension reduction </strong>- the process of reducing the number of random variables under consideration, by obtaining a set of principal variables. It can be divided into feature selection and feature extraction. <span class="citation">(“Introduction to Dimensionality Reduction - GeeksforGeeks” <a href="#ref-dimred">2018</a>)</span></p>
<p><strong>eigenvalue</strong>- a special set of scalars associated with a linear system of equations (i.e., a matrix equation) that are sometimes also known as characteristic roots, characteristic values , proper values, or latent roots. <span class="citation">(Weisstein, n.d.)</span></p>
<p><strong>eigenvector</strong>- are a special set of vectors associated with a linear system of equations(i.e., a matrix equation) that are sometimes also known as characteristic vectors, proper vectors, or latent vectors. Each eigenvector is paired with a corresponding eigenvalue. <span class="citation">(Weisstein, n.d.)</span></p>
<p><strong>matrix</strong> - a rectangular array of quantities or expressions in rows and columns that is treated as a single entity and manipulated according to particular rules. <span class="citation">(“matrix | Definition of matrix in English by Oxford Dictionaries” <a href="#ref-matrix">2018</a>)</span></p>
<p><strong>matrix factorisation</strong> - to factorize a matrix, i.e. to find out two (or more) matrices such that when you multiply them you will get back the original matrix. <span class="citation">(“Matrix Factorization: A Simple Tutorial and Implementation in Python | Albert Au Yeung” <a href="#ref-factMat">2018</a>)</span></p>
<p><strong>metagene</strong> - an aggregate pattern of gene expression <span class="citation">(“Metagene dictionary definition | metagene defined” <a href="#ref-metag">2018</a>)</span></p>
<p><strong>metasample </strong>- an aggregation of values for individual samples</p>
<p><strong>monte carlo simulation</strong> - a simulation that takes into account the variability of the inputs; mathematical technique that generates random variables for modelling risk or uncertainty of a certain system. <span class="citation">(“Definition of Monte Carlo Simulation | What is Monte Carlo Simulation ? Monte Carlo Simulation Meaning - The Economic Times” <a href="#ref-monte">2018</a>)</span></p>
<p><strong>multicollinearity</strong> - a state of very high intercorrelations or inter-associations among the independent variables. It is therefore a type of disturbance in the data, and if present in the data the statistical inferences made about the data may not be reliable. <span class="citation">(“Multicollinearity - Statistics Solutions” <a href="#ref-multi">2018</a>)</span></p>
<p><strong>p-value</strong> -  the probability of getting a sample similar observation, or more extreme than the tested one if the null hypothesis is true. So, we assume the null hypothesis is true and then determine how different the tested sample really is. If it is not that different (a large p-value) then the null hypothesis not rejected. As the p-value gets smaller, probability that null hypothesis is false increases (and reject the null hypothesis). <span class="citation">(“What is a p-value? - MathBootCamps” <a href="#ref-pv">2018</a>)</span></p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-biopsy">
<p>“Definition of biopsy - NCI Dictionary of Cancer Terms - National Cancer Institute.” 2018. Accessed July 23. <a href="https://www.cancer.gov/publications/dictionaries/cancer-terms/def/biopsy" class="uri">https://www.cancer.gov/publications/dictionaries/cancer-terms/def/biopsy</a>.</p>
</div>
<div id="ref-cansub">
<p>“Definition of cancer subtype - NCI Dictionary of Cancer Terms - National Cancer Institute.” 2018. Accessed July 23. <a href="https://www.cancer.gov/publications/dictionaries/cancer-terms/def/cancer-subtype" class="uri">https://www.cancer.gov/publications/dictionaries/cancer-terms/def/cancer-subtype</a>.</p>
</div>
<div id="ref-cytotoxic">
<p>“Definition of Cytotoxic.” 2018. Accessed July 23. <a href="https://www.medicinenet.com/script/main/art.asp?articlekey=19883" class="uri">https://www.medicinenet.com/script/main/art.asp?articlekey=19883</a>.</p>
</div>
<div id="ref-Cambronne2006">
<p>Cambronne, Eric D, and Craig R Roy. 2006. “Recognition and delivery of effector proteins into eukaryotic cells by bacterial secretion systems.” <em>Traffic</em> 7 (8): 929–39. doi:<a href="https://doi.org/10.1111/j.1600-0854.2006.00446.x">10.1111/j.1600-0854.2006.00446.x</a>.</p>
</div>
<div id="ref-geneexprsNature">
<p>“Gene expression definition.” 2018. Accessed July 23. <a href="https://www.nature.com/subjects/gene-expression" class="uri">https://www.nature.com/subjects/gene-expression</a>.</p>
</div>
<div id="ref-genomeNature">
<p>“Genome.” 2018. Accessed July 23. <a href="https://www.nature.com/subjects/genome?WT.ac=search{\_}subjects{\_}latestres{\_}genome{\#}research-and-reviews" class="uri">https://www.nature.com/subjects/genome?WT.ac=search{\_}subjects{\_}latestres{\_}genome{\#}research-and-reviews</a>.</p>
</div>
<div id="ref-immunosupp">
<p>“Definition of Immunosuppression.” 2018. Accessed July 23. <a href="https://www.medicinenet.com/script/main/art.asp?articlekey=3942" class="uri">https://www.medicinenet.com/script/main/art.asp?articlekey=3942</a>.</p>
</div>
<div id="ref-inducer">
<p>“Inducer (biology) | definition of Inducer (biology) by Medical dictionary.” 2018. Accessed July 23. <a href="https://medical-dictionary.thefreedictionary.com/Inducer+(biology)" class="uri">https://medical-dictionary.thefreedictionary.com/Inducer+(biology)</a>.</p>
</div>
<div id="ref-ligand">
<p>“Ligand | Define Ligand at Dictionary.com.” 2018. Accessed July 23. <a href="http://www.dictionary.com/browse/ligand" class="uri">http://www.dictionary.com/browse/ligand</a>.</p>
</div>
<div id="ref-tissue">
<p>“Tissue &amp; Organ Flashcards.” 2018. Accessed July 23. <a href="https://www.flashcardmachine.com/tissue-organ.html" class="uri">https://www.flashcardmachine.com/tissue-organ.html</a>.</p>
</div>
<div id="ref-marker">
<p>“Definition of Marker gene.” 2018. Accessed July 23. <a href="https://www.medicinenet.com/script/main/art.asp?articlekey=6731" class="uri">https://www.medicinenet.com/script/main/art.asp?articlekey=6731</a>.</p>
</div>
<div id="ref-biomolNature">
<p>“Molecular biology definition.” 2018. Accessed July 23. <a href="https://www.nature.com/subjects/molecular-biology" class="uri">https://www.nature.com/subjects/molecular-biology</a>.</p>
</div>
<div id="ref-omics">
<p>“Omics definition.” 2018. Accessed July 23. <a href="https://en.wikipedia.org/wiki/Omics" class="uri">https://en.wikipedia.org/wiki/Omics</a>.</p>
</div>
<div id="ref-pheno">
<p>“Examples of Genotype &amp; Phenotype.” 2018. Accessed July 23. <a href="http://examples.yourdictionary.com/examples-of-genotype-phenotype.html" class="uri">http://examples.yourdictionary.com/examples-of-genotype-phenotype.html</a>.</p>
</div>
<div id="ref-receptor">
<p>“Definition of Receptor.” 2018. Accessed July 23. <a href="https://www.medicinenet.com/script/main/art.asp?articlekey=5236" class="uri">https://www.medicinenet.com/script/main/art.asp?articlekey=5236</a>.</p>
</div>
<div id="ref-Aran2015">
<p>Aran, Dvir, Marina Sirota, and Atul J Butte. 2015. “Systematic pan-cancer analysis of tumour purity.” <em>Nat. Commun.</em> 6. Nature Publishing Group: 8971. doi:<a href="https://doi.org/10.1038/ncomms9971">10.1038/ncomms9971</a>.</p>
</div>
<div id="ref-bayes">
<p>“Bayes’ Theorem and Conditional Probability | Brilliant Math &amp; Science Wiki.” 2018. Accessed July 23. <a href="https://brilliant.org/wiki/bayes-theorem/" class="uri">https://brilliant.org/wiki/bayes-theorem/</a>.</p>
</div>
<div id="ref-Belsley1980">
<p>Belsley, David A., Edwin. Kuh, and Roy E. Welsch. 1980. <em>Regression diagnostics : identifying influential data and sources of collinearity</em>. Wiley Series in Probability and Statistics. Wiley. <a href="http://doi.wiley.com/10.1002/0471725153" class="uri">http://doi.wiley.com/10.1002/0471725153</a>.</p>
</div>
<div id="ref-corr">
<p>“correlation - Google Search.” 2018. Accessed July 23. <a href="https://www.google.fr/search?q=correlation{\&amp;}oq=correlation{\&amp;}aqs=chrome..69i57j0j69i60j69i61l2j0.1398j1j4{\&amp;}sourceid=chrome{\&amp;}ie=UTF-8" class="uri">https://www.google.fr/search?q=correlation{\&amp;}oq=correlation{\&amp;}aqs=chrome..69i57j0j69i60j69i61l2j0.1398j1j4{\&amp;}sourceid=chrome{\&amp;}ie=UTF-8</a>.</p>
</div>
<div id="ref-cov">
<p>“Covariance | R Tutorial.” 2018. Accessed July 23. <a href="http://www.r-tutor.com/elementary-statistics/numerical-measures/covariance" class="uri">http://www.r-tutor.com/elementary-statistics/numerical-measures/covariance</a>.</p>
</div>
<div id="ref-deconvolution">
<p>“deconvolution | Definition of deconvolution in English by Oxford Dictionaries.” 2018. Accessed July 3. <a href="https://en.oxforddictionaries.com/definition/deconvolution" class="uri">https://en.oxforddictionaries.com/definition/deconvolution</a>.</p>
</div>
<div id="ref-diag">
<p>“diagonal matrix - Google Search.” 2018. Accessed July 23. <a href="https://www.google.fr/search?q=diagonal+matrix{\&amp;}oq=diagonal+matrix{\&amp;}aqs=chrome..69i57.362j0j4{\&amp;}sourceid=chrome{\&amp;}ie=UTF-8" class="uri">https://www.google.fr/search?q=diagonal+matrix{\&amp;}oq=diagonal+matrix{\&amp;}aqs=chrome..69i57.362j0j4{\&amp;}sourceid=chrome{\&amp;}ie=UTF-8</a>.</p>
</div>
<div id="ref-dimred">
<p>“Introduction to Dimensionality Reduction - GeeksforGeeks.” 2018. Accessed July 23. <a href="https://www.geeksforgeeks.org/dimensionality-reduction/" class="uri">https://www.geeksforgeeks.org/dimensionality-reduction/</a>.</p>
</div>
<div id="ref-matrix">
<p>“matrix | Definition of matrix in English by Oxford Dictionaries.” 2018. Accessed July 23. <a href="https://en.oxforddictionaries.com/definition/matrix" class="uri">https://en.oxforddictionaries.com/definition/matrix</a>.</p>
</div>
<div id="ref-factMat">
<p>“Matrix Factorization: A Simple Tutorial and Implementation in Python | Albert Au Yeung.” 2018. Accessed July 23. <a href="http://www.albertauyeung.com/post/python-matrix-factorization/" class="uri">http://www.albertauyeung.com/post/python-matrix-factorization/</a>.</p>
</div>
<div id="ref-metag">
<p>“Metagene dictionary definition | metagene defined.” 2018. Accessed July 23. <a href="http://www.yourdictionary.com/metagene" class="uri">http://www.yourdictionary.com/metagene</a>.</p>
</div>
<div id="ref-monte">
<p>“Definition of Monte Carlo Simulation | What is Monte Carlo Simulation ? Monte Carlo Simulation Meaning - The Economic Times.” 2018. Accessed July 23. <a href="https://economictimes.indiatimes.com/definition/monte-carlo-simulation" class="uri">https://economictimes.indiatimes.com/definition/monte-carlo-simulation</a>.</p>
</div>
<div id="ref-multi">
<p>“Multicollinearity - Statistics Solutions.” 2018. Accessed July 23. <a href="http://www.statisticssolutions.com/multicollinearity/" class="uri">http://www.statisticssolutions.com/multicollinearity/</a>.</p>
</div>
<div id="ref-pv">
<p>“What is a p-value? - MathBootCamps.” 2018. Accessed July 23. <a href="https://www.mathbootcamps.com/what-is-a-p-value/" class="uri">https://www.mathbootcamps.com/what-is-a-p-value/</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="annexes.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="post-scriptum-thesis-writing.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/12-Annexes.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
