<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="organisation-of-the-dissertation.html">
<link rel="next" href="methods.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li><a href="index.html#unsupervised-deconvolution-of-bulk-omics-profiles-methodology-and-application-to-characterize-the-immune-landscape-in-tumors"><strong>Unsupervised deconvolution of bulk omics profiles: methodology and application to characterize the immune landscape in tumors</strong></a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="tcga-study-abbreviations.html"><a href="tcga-study-abbreviations.html"><i class="fa fa-check"></i>TCGA Study Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-ph.d.---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary Ph.D. - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-ph.d.topic"><i class="fa fa-check"></i>The origins of the Ph.D. topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="mstd.html"><a href="mstd.html#context"><i class="fa fa-check"></i><b>3.1</b> Context</a></li>
<li class="chapter" data-level="3.2" data-path="mstd.html"><a href="mstd.html#description"><i class="fa fa-check"></i><b>3.2</b> Description</a></li>
<li class="chapter" data-level="3.3" data-path="mstd.html"><a href="mstd.html#impact-on-the-further-work"><i class="fa fa-check"></i><b>3.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a><ul>
<li class="chapter" data-level="4.1" data-path="lva.html"><a href="lva.html#context-1"><i class="fa fa-check"></i><b>4.1</b> Context</a></li>
<li class="chapter" data-level="4.2" data-path="lva.html"><a href="lva.html#description-1"><i class="fa fa-check"></i><b>4.2</b> Description</a></li>
<li class="chapter" data-level="4.3" data-path="lva.html"><a href="lva.html#impact-on-the-further-work-1"><i class="fa fa-check"></i><b>4.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="nmfica.html"><a href="nmfica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.1" data-path="nmfica.html"><a href="nmfica.html#comparing-metagenes-obtained-with-nmf-versus-ica"><i class="fa fa-check"></i><b>5.1</b> Comparing metagenes obtained with NMF versus ICA</a></li>
<li class="chapter" data-level="5.2" data-path="nmfica.html"><a href="nmfica.html#summary-4"><i class="fa fa-check"></i><b>5.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#unsupervised-deconvolution"><i class="fa fa-check"></i><b>6.2</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#fastica-overdecomposition-protocol"><i class="fa fa-check"></i><b>6.2.1</b> FastICA overdecomposition protocol</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="deconica.html"><a href="deconica.html#interpretation-of-the-components"><i class="fa fa-check"></i><b>6.3</b> Interpretation of the components</a></li>
<li class="chapter" data-level="6.4" data-path="deconica.html"><a href="deconica.html#computing-the-abundance-of-the-identified-cell-types"><i class="fa fa-check"></i><b>6.4</b> Computing the abundance of the identified cell-types</a></li>
<li class="chapter" data-level="6.5" data-path="deconica.html"><a href="deconica.html#validation-of-the-abundance-estimation-with-deconica"><i class="fa fa-check"></i><b>6.5</b> Validation of the abundance estimation with <em>DeconICA</em></a><ul>
<li class="chapter" data-level="6.5.1" data-path="deconica.html"><a href="deconica.html#in-silico"><i class="fa fa-check"></i><b>6.5.1</b> <em>In silico</em></a></li>
<li class="chapter" data-level="6.5.2" data-path="deconica.html"><a href="deconica.html#in-vitro"><i class="fa fa-check"></i><b>6.5.2</b> <em>In vitro</em></a></li>
<li class="chapter" data-level="6.5.3" data-path="deconica.html"><a href="deconica.html#pbmc-transcriptome"><i class="fa fa-check"></i><b>6.5.3</b> PBMC transcriptome</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="deconica.html"><a href="deconica.html#summary-5"><i class="fa fa-check"></i><b>6.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Landscape of tumor immunity, with focus on T-cells diversity, uncovered via unsupervised deconvolution applied to bulk transcriptomes across 30 tumor types</a><ul>
<li class="chapter" data-level="7.1" data-path="results.html"><a href="results.html#background"><i class="fa fa-check"></i><b>7.1</b> Background</a></li>
<li class="chapter" data-level="7.2" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>7.2</b> Results</a><ul>
<li class="chapter" data-level="7.2.1" data-path="results.html"><a href="results.html#unsupervised-deconvolution-of-119-datasets"><i class="fa fa-check"></i><b>7.2.1</b> Unsupervised deconvolution of 119 datasets</a></li>
<li class="chapter" data-level="7.2.2" data-path="results.html"><a href="results.html#interpretation-of-the-immune-components"><i class="fa fa-check"></i><b>7.2.2</b> Interpretation of the immune components</a></li>
<li class="chapter" data-level="7.2.3" data-path="results.html"><a href="results.html#quality-of-cell-type-signal-extraction-is-affected-by-tumor-type"><i class="fa fa-check"></i><b>7.2.3</b> Quality of cell-type signal extraction is affected by tumor type</a></li>
<li class="chapter" data-level="7.2.4" data-path="results.html"><a href="results.html#deconvoluted-bulk-transcriptome-t-cell-signatures-showcase-different-cell-states-in-cancers"><i class="fa fa-check"></i><b>7.2.4</b> Deconvoluted bulk transcriptome T cell signatures showcase different cell states in cancers</a></li>
<li class="chapter" data-level="7.2.5" data-path="results.html"><a href="results.html#single-cell-rnaseq-deconvolution-leads-to-cell-specific-metagenes"><i class="fa fa-check"></i><b>7.2.5</b> Single-cell RNAseq deconvolution leads to cell-specific metagenes</a></li>
<li class="chapter" data-level="7.2.6" data-path="results.html"><a href="results.html#landscape-of-patient-based-on-the-immune-infiltration-scores"><i class="fa fa-check"></i><b>7.2.6</b> Landscape of patient based on the immune infiltration scores</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="results.html"><a href="results.html#discussion"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
<li class="chapter" data-level="7.4" data-path="results.html"><a href="results.html#resconcl"><i class="fa fa-check"></i><b>7.4</b> Conclusions</a></li>
<li class="chapter" data-level="7.5" data-path="results.html"><a href="results.html#methods-1"><i class="fa fa-check"></i><b>7.5</b> Methods</a><ul>
<li class="chapter" data-level="7.5.1" data-path="results.html"><a href="results.html#data-sources"><i class="fa fa-check"></i><b>7.5.1</b> Data sources</a></li>
<li class="chapter" data-level="7.5.2" data-path="results.html"><a href="results.html#data-pre-processing"><i class="fa fa-check"></i><b>7.5.2</b> Data pre-processing</a></li>
<li class="chapter" data-level="7.5.3" data-path="results.html"><a href="results.html#data-analysis"><i class="fa fa-check"></i><b>7.5.3</b> Data analysis</a></li>
<li class="chapter" data-level="7.5.4" data-path="results.html"><a href="results.html#software"><i class="fa fa-check"></i><b>7.5.4</b> Software</a></li>
<li class="chapter" data-level="7.5.5" data-path="results.html"><a href="results.html#hardware"><i class="fa fa-check"></i><b>7.5.5</b> Hardware</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="results.html"><a href="results.html#supplementary"><i class="fa fa-check"></i><b>7.6</b> Supplementary</a><ul>
<li class="chapter" data-level="7.6.1" data-path="results.html"><a href="results.html#figures"><i class="fa fa-check"></i><b>7.6.1</b> Figures</a></li>
<li class="chapter" data-level="7.6.2" data-path="results.html"><a href="results.html#tables"><i class="fa fa-check"></i><b>7.6.2</b> Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> A multiscale signalling network map of innate immune response in cancer reveals signatures of cell heterogeneity and functional polarization</a><ul>
<li class="chapter" data-level="8.1" data-path="map.html"><a href="map.html#context-2"><i class="fa fa-check"></i><b>8.1</b> Context</a></li>
<li class="chapter" data-level="8.2" data-path="map.html"><a href="map.html#description-2"><i class="fa fa-check"></i><b>8.2</b> Description</a></li>
<li class="chapter" data-level="8.3" data-path="map.html"><a href="map.html#discussion-and-perspectives"><i class="fa fa-check"></i><b>8.3</b> Discussion and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussiongenerale.html"><a href="discussiongenerale.html"><i class="fa fa-check"></i><b>9</b> Discussion</a><ul>
<li class="chapter" data-level="9.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#improving-our-knowledge-of-how-best-apply-ica-to-transcriptomic-data"><i class="fa fa-check"></i><b>9.1</b> Improving our knowledge of how best apply ICA to transcriptomic data</a></li>
<li class="chapter" data-level="9.2" data-path="discussiongenerale.html"><a href="discussiongenerale.html#confronting-ica-with-other-matrix-factorization-approaches"><i class="fa fa-check"></i><b>9.2</b> Confronting ICA with other matrix factorization approaches</a></li>
<li class="chapter" data-level="9.3" data-path="discussiongenerale.html"><a href="discussiongenerale.html#deconvolution-methods-overall-impact-and-utility"><i class="fa fa-check"></i><b>9.3</b> Deconvolution methods overall impact and utility</a></li>
<li class="chapter" data-level="9.4" data-path="discussiongenerale.html"><a href="discussiongenerale.html#unsupervised-deconvolution-1"><i class="fa fa-check"></i><b>9.4</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="9.4.1" data-path="discussiongenerale.html"><a href="discussiongenerale.html#limitations-of-unsupervised-deconvolution"><i class="fa fa-check"></i><b>9.4.1</b> Limitations of unsupervised deconvolution</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="discussiongenerale.html"><a href="discussiongenerale.html#data-sources-1"><i class="fa fa-check"></i><b>9.5</b> Data sources</a></li>
<li class="chapter" data-level="9.6" data-path="discussiongenerale.html"><a href="discussiongenerale.html#missing-time-and-space-dimensions-in-transcriptomic-studies"><i class="fa fa-check"></i><b>9.6</b> Missing time and space dimensions in transcriptomic studies</a></li>
<li class="chapter" data-level="9.7" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-place-of-bulk-transcriptomes-in-single-cell-era"><i class="fa fa-check"></i><b>9.7</b> The place of bulk transcriptomes in single cell era</a></li>
<li class="chapter" data-level="9.8" data-path="discussiongenerale.html"><a href="discussiongenerale.html#the-field-of-computational-cancer-immunology"><i class="fa fa-check"></i><b>9.8</b> The field of computational cancer immunology</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a><ul>
<li class="chapter" data-level="10.1" data-path="conclusions.html"><a href="conclusions.html#conclusions-1"><i class="fa fa-check"></i><b>10.1</b> Conclusions</a></li>
<li class="chapter" data-level="10.2" data-path="conclusions.html"><a href="conclusions.html#perspectives"><i class="fa fa-check"></i><b>10.2</b> Perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#deconica-documentation"><i class="fa fa-check"></i>1 DeconICA documentation</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#introduction-to-deconica"><i class="fa fa-check"></i>1.1 Introduction to deconICA</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#running-fastica-with-icasso-stabilisation"><i class="fa fa-check"></i>1.2 Running fastICA with icasso stabilisation</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#reference-manual"><i class="fa fa-check"></i>1.3 Reference manual</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#publications-and-conferences"><i class="fa fa-check"></i>2 Publications and conferences</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#adjustment-of-dendritic-cells-to-the-breast-cancer-microenvironment-is-subset-specific"><i class="fa fa-check"></i>2.1 Adjustment of dendritic cells to the breast-cancer microenvironment is subset-specific</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#the-inconvenience-of-data-of-convenience-computational-research-beyond-post-mortem-analyses"><i class="fa fa-check"></i>2.2 The inconvenience of data of convenience: computational research beyond post-mortem analyses</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv-publications-conferences-courses"><i class="fa fa-check"></i>2.3 CV: publications, conferences, courses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a><ul>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#biological-terms"><i class="fa fa-check"></i>Biological terms</a></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#mathematical-terms"><i class="fa fa-check"></i>Mathematical terms</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="intro" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Immuno-biology of cancer</h1>
<p>This chapter will first introduce a short history of cancer with a focus on discoveries linking cancer and its environment. It will also describe the participation of TME in cancer development, progression and response to treatment. Most important types of data used to study cancer microenvironment will be discussed. I also introduce a link between tumor immune-biology and cancer phenotyping for development of immunotherapies.</p>
<div id="cancer-disease" class="section level2">
<h2><span class="header-section-number">1.1</span> Cancer disease</h2>
<p>According to <a href="http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx">GLOBOCAN study</a> <span class="citation">(Research in Cancer <a href="#ref-GLOBOCAN">2018</a>)</span>, 14.1 million cancer cases were estimated to happen around the world in 2012. It touched 7.4 million men and 6.7 million women. It is estimated that the cancer cases will increase almost two-fold to 24 million by 2035.</p>
<p>In France only, in 2012 there were 349426 cases of cancer, of which leading is Prostate cancer (16,3%) followed by Breast (14%) and Lung (11,5%).</p>
<p>For a long time studying tumor was focused on tumor cells, their reprogramming, mutations. Cancer was seen as a disease of uncontrolled cells by the mainstream research. At the same time, the idea of the importance of the impact of other cells and structures on cancer cells was present but often not believed. A recent success of immunotherapies moved research focus to tumor cells in their context: tumor microenvironment. We will describe here what is the composition and role of the TME in tumor progression, diagnosis and response to treatment.</p>
<div id="hist" class="section level3">
<h3><span class="header-section-number">1.1.1</span> Historical understanding of cancer</h3>
<p>Cancer was historically described by a physician Hippocrates (460–370 B.C) <span class="citation">(Sudhakar <a href="#ref-Sudhakar2009">2009</a>)</span>. Even though there exist even earlier evidence of the disease. Hippocrates stated that the body contained 4 humors (body fluids): blood, phlegm, yellow bile and black bile. Any imbalance of these fluids will result in disease. Particularly the excess of black bile in an organ was meant to provoke cancer. For years, it was not known what factors cause cancer and it was easily confounded with other diseases. In the middle ages in the Renaissance Period, it was believed cancer is a punishment for the sins they committed against their god, that they deserved it to some extent.</p>
<p>Until the 18th century, it was believed that cancer is contagious and is spread by parasites.</p>
<p>In the 19th century, tumor cells started to be analyzed by pathologists. They were strike with their ability to proliferate uncontrollably, ability to spread and destroy the original tissue <span class="citation">(NPR <a href="#ref-NPR2010">2010</a><a href="#ref-NPR2010">b</a>)</span>. Around the same time, leukocytes from the blood were first described by Gabriel Andra and William Addison. Just a few years later, in 1845 Bennett and Virchow described blood cells in leukemia (Fig. <a href="intro.html#fig:Virchow-cell">1.1</a>). Virchow is also a father of Chronic irritation theory (nowadays called chronic inflammation) that says that cancer is caused by local “irritation” and, incorrectly, that cancer cells spread like liquid resulting in metastasis.</p>
<div class="figure" style="text-align: center"><span id="fig:Virchow-cell"></span>
<img src="figures-ext/01-Virchow-cell.jpg" alt="Illustration of Virchow’s cell theory. Virchow depicted different cells transformation due to irritation. (Virchow Rudolf 1847)" width="100%" />
<p class="caption">
Figure 1.1: <strong>Illustration of Virchow’s cell theory</strong>. Virchow depicted different cells transformation due to irritation. <span class="citation">(Virchow Rudolf <a href="#ref-VirchowRudolf1847">1847</a>)</span>
</p>
</div>

<p>In 1889, Stephen Paget introduced <em>soil and seed</em> hypothesis of metastases <span class="citation">(Paget <a href="#ref-Paget1889">1889</a>)</span>. He formulates it as follows</p>
<blockquote>
<p><em>When a plant goes to seed, its seeds are carried in all directions, but they can only live and grow if they fall on congenial soil.</em></p>
</blockquote>
<p>Which is parallel to cancer cells disseminated by body fluids, and they can grow only tissues - “soil” that is predisposed to host the cancer cell - “the seed”. He focused on the importance of tissue characteristics that favorize tumor development as opposed to most researchers of his time that were focusing on the “seed” itself.</p>
<p>In the 20th century, molecular causes started to be investigated. It was discovered that cancer could be caused by environmental factors, i.e. chemicals (carcinogens), radiation, viruses and also inherited from ancestors. Those factors would damage but contrary to a healthy condition they would not die.</p>
<p>Also in 1909, Paul Ehrlich, called one of fathers of immunology and Nobel Prize laureate, indicated a link between immune system and tumor suppression <span class="citation">(Ehrlich <a href="#ref-Ehrlich1909">1909</a>)</span>. One of the remarkable first immunotherapy attempts can be attributed to William Coley, that practiced injecting streptococcus bacteria directly into patients after cancer surgery in 1891, later called “Colley vaccine”. However, the impact of this procedure on patients recovery was judged by scientific community as “unclear”.</p>
<p>In 1968, Melvin Greenblatt and Philippe Shubik showed that tumor transplants secrete a substance stimulating the growth of blood vessels <span class="citation">(Greenblatt and Shubi <a href="#ref-Greenblatt1968">1968</a>)</span>, later identified as “tumor angiogenic factor (TAF)” by Judah Folkman in 1971 <span class="citation">(Folkman et al. <a href="#ref-Folkman1971">1971</a>)</span>. Folkman also suggested that TAF can be a target of a therapy itself. This was a revolutionary idea, at the time, as it did not target the tumor cells directly acted on their environment.</p>
<p>During the 1970s, oncogenes and tumor suppressor genes were discovered. Oncogenes are genes that allow a cell to become a cancer cell, while the tumor suppressor genes would repair DNA or execute cell death of a damaged cell. A new dimension to cancer studies was added in the 1980s, epigenetic changes were proven to occur to both oncogenes and tumor suppressors <span class="citation">(Feinberg and Vogelstein <a href="#ref-Feinberg1983">1983</a>; Greger et al. <a href="#ref-Greger1989">1989</a>)</span>, which are presently known as epigenetic markers used for diagnostics and therapeutic targets for cancer.</p>
<p>In 1982, Aline van Pel and Thierry Boon <span class="citation">(Van Pel and Boon <a href="#ref-VanPel1982">1982</a>)</span> discovered that a specific immunity to spontaneous tumor cells could be induced by vaccinating mice with mutagenized tumor cells. This raised an inspiration for many years of immune therapy development.</p>
<p>In Napoleone Ferrara and colleagues identified the gene encoding vascular endothelial growth factor (VEGF) that was shown to stimulate the growth of endothelial cells proliferation <em>in vitro</em> and angiogenesis (blood vessels formation) <em>in vivo</em> <span class="citation">(Leung et al. <a href="#ref-Leung1989">1989</a>)</span>.</p>
<p>In 1999 for the first time, gene-expression was used to study cancer (leukemia) by Todd Golub, Donna Slonim and colleagues <span class="citation">(Golub et al. <a href="#ref-Golub1999">1999</a>)</span>.</p>
<p>Since the end of the 20th century, cancer screens are developed along with multiple strategies to fight the tumor. Most classical ones are based on the idea of removing tumor cells (surgery), killing tumor cells with DNA-blocking drugs (chemotherapy), radiation, inhibit cancer growth (hormonal therapy, adjuvant therapy and immunotherapy). As none of those methods is fully efficient, often a combination of treatments is proposed. Nowadays, science is aiming in the direction of targeted therapies and personalized treatment.</p>
<p>The recent success of immunotherapies (discussed in <a href="intro.html#immunotherapies">Immunotherapies section</a> attracted the attention the scientific community again to the context in which tumor cells are found. This context called Tumor Microenvironment, as well as the communication that happens within it between different agents nowadays studied differently with available knowledge of molecular biology, have become a popular scientific topic of the 21st century (Fig. <a href="intro.html#fig:pubmedTME">1.2</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:pubmedTME"></span>
<div id="17c02f24f897" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="17c02f24f897">{"x":{"data":[{"orientation":"v","width":[0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1950,1951,1952,1953,1954,1955,1956,1957,1958,1959,1960,1961,1962,1963,1964,1965,1966,1967,1968,1969,1970,1971,1972,1973,1974,1975,1976,1977,1978,1979,1980,1981,1982,1983,1984,1985,1986,1987,1988,1989,1990,1991,1992,1993,1994,1995,1996,1997,1998,1999,2000,2001,2002,2003,2004,2005,2006,2007,2008,2009,2010,2011,2012,2013,2014,2015,2016],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4.01411362350023e-06,7.84713775650332e-06,0,1.46888713438481e-05,1.0646452601993e-05,1.42687446715157e-05,2.1182476443321e-05,1.35460514953148e-05,1.6175000566125e-05,3.46051240750994e-05,4.77814476584104e-05,4.86892565723336e-05,7.07502244959047e-05,6.72625438176668e-05,9.19871715187828e-05,9.74981962833688e-05,5.57414951807842e-05,9.34225718515396e-05,0.000135886118061608,0.000189523246282033,0.000205119058235976,0.000224835848002235,0.000234806175182962,0.000276840886228962,0.000245495861061516,0.000252070173326461,0.000390067025601582,0.00041818465333201,0.000476993893466869,0.000509121897881113,0.000629368028676296,0.000716625030975615,0.000856136023706814,0.0009827909827431,0.00111680424979417,0.00137703697796209,0.00187891461005892,0.00216715357778359,0.0024373280322394,0.00272225123846604,0.00309914470308773,0.00348693807766557],"text":["~Year: 1950<br />~percentage: 0.000000e+00","~Year: 1951<br />~percentage: 0.000000e+00","~Year: 1952<br />~percentage: 0.000000e+00","~Year: 1953<br />~percentage: 0.000000e+00","~Year: 1954<br />~percentage: 0.000000e+00","~Year: 1955<br />~percentage: 0.000000e+00","~Year: 1956<br />~percentage: 0.000000e+00","~Year: 1957<br />~percentage: 0.000000e+00","~Year: 1958<br />~percentage: 0.000000e+00","~Year: 1959<br />~percentage: 0.000000e+00","~Year: 1960<br />~percentage: 0.000000e+00","~Year: 1961<br />~percentage: 0.000000e+00","~Year: 1962<br />~percentage: 0.000000e+00","~Year: 1963<br />~percentage: 0.000000e+00","~Year: 1964<br />~percentage: 0.000000e+00","~Year: 1965<br />~percentage: 0.000000e+00","~Year: 1966<br />~percentage: 0.000000e+00","~Year: 1967<br />~percentage: 0.000000e+00","~Year: 1968<br />~percentage: 0.000000e+00","~Year: 1969<br />~percentage: 0.000000e+00","~Year: 1970<br />~percentage: 0.000000e+00","~Year: 1971<br />~percentage: 0.000000e+00","~Year: 1972<br />~percentage: 0.000000e+00","~Year: 1973<br />~percentage: 0.000000e+00","~Year: 1974<br />~percentage: 0.000000e+00","~Year: 1975<br />~percentage: 4.014114e-06","~Year: 1976<br />~percentage: 7.847138e-06","~Year: 1977<br />~percentage: 0.000000e+00","~Year: 1978<br />~percentage: 1.468887e-05","~Year: 1979<br />~percentage: 1.064645e-05","~Year: 1980<br />~percentage: 1.426874e-05","~Year: 1981<br />~percentage: 2.118248e-05","~Year: 1982<br />~percentage: 1.354605e-05","~Year: 1983<br />~percentage: 1.617500e-05","~Year: 1984<br />~percentage: 3.460512e-05","~Year: 1985<br />~percentage: 4.778145e-05","~Year: 1986<br />~percentage: 4.868926e-05","~Year: 1987<br />~percentage: 7.075022e-05","~Year: 1988<br />~percentage: 6.726254e-05","~Year: 1989<br />~percentage: 9.198717e-05","~Year: 1990<br />~percentage: 9.749820e-05","~Year: 1991<br />~percentage: 5.574150e-05","~Year: 1992<br />~percentage: 9.342257e-05","~Year: 1993<br />~percentage: 1.358861e-04","~Year: 1994<br />~percentage: 1.895232e-04","~Year: 1995<br />~percentage: 2.051191e-04","~Year: 1996<br />~percentage: 2.248358e-04","~Year: 1997<br />~percentage: 2.348062e-04","~Year: 1998<br />~percentage: 2.768409e-04","~Year: 1999<br />~percentage: 2.454959e-04","~Year: 2000<br />~percentage: 2.520702e-04","~Year: 2001<br />~percentage: 3.900670e-04","~Year: 2002<br />~percentage: 4.181847e-04","~Year: 2003<br />~percentage: 4.769939e-04","~Year: 2004<br />~percentage: 5.091219e-04","~Year: 2005<br />~percentage: 6.293680e-04","~Year: 2006<br />~percentage: 7.166250e-04","~Year: 2007<br />~percentage: 8.561360e-04","~Year: 2008<br />~percentage: 9.827910e-04","~Year: 2009<br />~percentage: 1.116804e-03","~Year: 2010<br />~percentage: 1.377037e-03","~Year: 2011<br />~percentage: 1.878915e-03","~Year: 2012<br />~percentage: 2.167154e-03","~Year: 2013<br />~percentage: 2.437328e-03","~Year: 2014<br />~percentage: 2.722251e-03","~Year: 2015<br />~percentage: 3.099145e-03","~Year: 2016<br />~percentage: 3.486938e-03"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(70,130,180,1)","line":{"width":1.88976377952756,"color":"transparent"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":40.1826484018265,"l":54.7945205479452},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"type":"linear","autorange":false,"range":[1946.205,2019.795],"tickmode":"array","ticktext":["1950","1955","1960","1965","1970","1975","1980","1985","1990","1995","2000","2005","2010","2015"],"tickvals":[1950,1955,1960,1965,1970,1975,1980,1985,1990,1995,2000,2005,2010,2015],"categoryorder":"array","categoryarray":["1950","1955","1960","1965","1970","1975","1980","1985","1990","1995","2000","2005","2010","2015"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"Year","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"type":"linear","autorange":false,"range":[-0.000174346903883279,0.00366128498154885],"tickmode":"array","ticktext":["0.000","0.001","0.002","0.003"],"tickvals":[0,0.001,0.002,0.003],"categoryorder":"array","categoryarray":["0.000","0.001","0.002","0.003"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Percentage of all publications [%]","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"source":"A","attrs":{"17c01d41607a":{"x":{},"y":{},"fill":{},"type":"bar"}},"cur_data":"17c01d41607a","visdat":{"17c01d41607a":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p class="caption">
Figure 1.2: <strong>Percentage of publications containing the phrase “tumor immunotherapy” is growing</strong>, numbers retrieved on 17.01.2018 from <a href="http://dan.corlan.net/medline-trend.html">Medline Trends</a> <span class="citation">(Corlan <a href="#ref-Corlan2004">2004</a>)</span>
</p>
</div>

</div>
<div id="tumor-microenvironment-as-a-complex-system" class="section level3">
<h3><span class="header-section-number">1.1.2</span> Tumor Microenvironment as a complex system</h3>
<p>Tumor Microenvironment is a complex tissue that surrounds tumor cells. It is composed of different compartments (in solid tumors):</p>
<ul>
<li>Stroma: blood and lymphatics vessels, epithelial cells, mesenchymal stem cells, fibroblasts, adipocytes supported by extracellular matrix (EM)</li>
<li>Immune cells: T cells, B cells, NK cells, Dendritic cells, Macrophages, Monocytes etc.</li>
</ul>
<p>Their proportion and specific roles vary significantly with tumor type and stage. Communication between the environmental cells and the tumor is critical for tumor development and has an impact on patient’s response to treatment. This communication between different compartments is bidirectional and all the players can influence each other. Depending on the nature and prevailing direction of those interactions different destiny is possible for each of the compartments, i.e. immune cells can be recruited to protect tumor cells or they can kill them directly. Many of the signals can be contradictory, many can suppress each other. Then is it possible to tilt this complex ecosystem into patients’ favor? Can we decipher the most important factors of this molecular knot and manipulate it?</p>
<p>Next section describes different scenarios of interaction within TME in order to illustrate the complexity of TME and possible targets for cancer therapies.</p>
<p>We recommend watching this video in order to visualize the TME and cancer thanks to 3D animations</p>
<iframe width="560" height="315" src="https://www.youtube.com/embed/K09xzIQ8zsg" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen>
</iframe>
<div id="interactions-between-tme-and-tumor" class="section level4">
<h4><span class="header-section-number">1.1.2.1</span> Interactions between TME and Tumor</h4>
<p>Three scenarios can be considered to describe the relationship between TME and tumor cells:</p>
<ol style="list-style-type: decimal">
<li>TME stimulates tumor growth and/or progression and/or impact negatively the response to treatment</li>
<li>TME has no influence on tumor cells and disease development</li>
<li>TME has a tumor-suppressive role and impact positively the response to treatment</li>
</ol>
<p>As it is presented in <a href="intro.html#hist">Section 1.1.1</a> these three hypotheses were gaining and losing popularity in the scientific and medical community over the decades.</p>
<div id="tme-as-a-foe-inflammation" class="section level5">
<h5><span class="header-section-number">1.1.2.1.1</span> TME as a foe: inflammation</h5>
<p>In 1863 Rudolf Virchow observed a link between chronic inflammation and tumorigenesis. According to Virchov theory, the genetic damage would be the “match that lights the fire” of cancer, and the inflammation or cytokines produced by immune cells should be the “fuel that feeds the flames” <span class="citation">(Balkwill and Mantovani <a href="#ref-Balkwill2001">2001</a>)</span>. Therefore lymphocyte infiltration was confirmed by subsequent studies as a hallmark o cancer. The question one may ask is why our immune system is not enough to defend the organism from tumor cells as it does efficiently in a range of bacterial and viral infections? It is mainly because of the ability of tumor cells to inhibit immune response through activation of negative regulatory pathways (so-called immune checkpoints).</p>
<p>It is worth mentioning, that the immune system can be already disabled and therefore cancer has more facility to develop. The immune system can be less efficient for example because of drugs given to patients after transplants of because of diseases like HIV/AIDS. These people have higher probability to develop cancers caused by infectious agents (viruses). These cancers are non-Hodgkin lymphoma (NHL) (caused by <a href="https://www.cancer.gov/Common/PopUps/popDefinition.aspx?id=CDR0000045684&amp;version=Patient&amp;language=English">Epstein-Barr virus</a> (EBV) infection), lung (no identified specific infectious agent), kidney (no identified specific infectious agent) and liver (caused by chronic infection with the <a href="https://www.cancer.gov/Common/PopUps/popDefinition.aspx?id=CDR0000046146&amp;version=Patient&amp;language=English">hepatitis B</a> (HBV) and <a href="https://www.cancer.gov/Common/PopUps/popDefinition.aspx?id=CDR0000044139&amp;version=Patient&amp;language=English">hepatitis C</a> (HCV) viruses) cancers. Human papillomavirus (HPV), can cause cervical, anal, oropharyngeal, and other cancers.</p>
<p>In the case of non-infectious cancers in patients with no history of immunosuppressive drugs intake or diseases, the question how tumor manages to break natural defence remains even more interesting. Many examples can be cited on how TME facilitates tumor development (Fig. <a href="intro.html#fig:met-dis">1.3</a>). For instance, in the early stages of tumorigenesis, some macrophage phenotypes support tumor growth and mobility through TGF-beta signaling. Also, it was shown that NK cells and myeloid-derived suppressor cells (MDSCs) have an ability to suppress immune defence i.e. immunosurveillance by dendritic cells (DCs), T cell activation and macrophage polarisation and they promote tumor vascularization as well. <span class="citation">(Talmadge and Gabrilovich <a href="#ref-Talmadge2013">2013</a>; Gabrilovich, Ostrand-Rosenberg, and Bronte <a href="#ref-Gabrilovich2012">2012</a>)</span> They create so-called niches that facilitate tumor colonization. T-regs and myeloid-derived suppressor cells can negatively impact natural immune defense and by these means allow growth and invasion of tumor cells <span class="citation">(Taube et al. <a href="#ref-Taube2017a">2017</a>)</span>. Another cell type, a part of EM, fibroblast, or more precisely Cancer-Associated Fibroblasts (CAFs) have proven pro-tumor functions in breast cancer where they enhance metastasis <span class="citation">(Dumont et al. <a href="#ref-Dumont2013">2013</a>)</span>. The blood and lymphatic vessels maintain tumor growth providing necessary nutritive compound to malignant cells.</p>
<div class="figure" style="text-align: center"><span id="fig:met-dis"></span>
<img src="figures-ext/massive-dissemination.png" alt="The microenvironment supports metastatic dissemination and colonization at secondary sites. Different tumor sites can communicate through exosomes realized by tumor cells and also immune and stromal cells such as NK cells, CAFs and DCs. Reprinted by permission from Springer Nature (Quail and Joyce 2013) © 2013 Nature America, Inc. All rights reserved." width="100%" />
<p class="caption">
Figure 1.3: <strong>The microenvironment supports metastatic dissemination and colonization at secondary sites.</strong> Different tumor sites can communicate through exosomes realized by tumor cells and also immune and stromal cells such as NK cells, CAFs and DCs. Reprinted by permission from Springer Nature <span class="citation">(Quail and Joyce <a href="#ref-Quail2013">2013</a>)</span> © 2013 Nature America, Inc. All rights reserved.
</p>
</div>

<p>According to <span class="citation">(Hanahan and Coussens <a href="#ref-Hanahan2012">2012</a>)</span> immune and stroma cells participate in almost all of Cancer Hallmarks <span class="citation">(Hanahan and Weinberg <a href="#ref-Hanahan2000">2000</a>; Hanahan and Coussens <a href="#ref-Hanahan2012">2012</a>)</span>. Most of the hallmarks of cancer are enabled and sustained to varying degrees through contributions from repertoires of stromal cell types and distinctive subcell types.</p>
</div>
<div id="tme-seen-as-neutral" class="section level5">
<h5><span class="header-section-number">1.1.2.1.2</span> TME seen as neutral</h5>
<p>In front of lack of definitive proof that TME can positively or negatively impact on tumor development, many scientists, in a long time, ignored the importance of this factor. Until the early-mid eighties, the TME research was mostly limited to angiogenesis and immune environment and most areas that are now driving the field were not represented.</p>
<p>From the early 70s until the end of the 90s. the most accepted statement was that genetic alterations in oncogenes and tumor suppressor genes are both necessary and sufficient to initiate tumorigenesis and drive tumor progression. Therefore TME was not seen as an important element of the puzzle.</p>
<p>The cancer geneticists, at the time, had a lot of influence on scientific community diminishing the work made on TME which were considered as “uninteresting” and definitely not “mainstream”.</p>
<p>After the 90s, with the discovery of signaling molecules involved in the communication of TME like VEGF general opinion started to change. Furthermore, discoveries made by developmental biology field supported the hypothesis that microenvironment plays an important role in development which was later shown for tumorigenesis. Additionally, the success of immune vaccines starting with the tuberculosis vaccine Bacille Calmette-Guérin (BCG) in 1976 and finishing, at the moment with checkpoint inhibitors did not leave the scientific community indifferent.</p>
</div>
<div id="tme-as-a-friend-immunosurveillance" class="section level5">
<h5><span class="header-section-number">1.1.2.1.3</span> TME as a friend: immunosurveillance</h5>
<p>As mentioned in <a href="intro.html#hist">Section 1.1.1</a> Paget proposed a hypothesis of “seed and soil” where the TME in a certain tissue (the soil) can either stimulate or suppress the metastasis (the seed). William Coley tested a possibility to trigger tumor-suppressive effect via stimulation of the immune system with bacteria. In the 1960s, the immune surveillance theory hypothesized “the ability to identify and destroy nascent tumors as a central asset of the immune system” <span class="citation">(Sebeok <a href="#ref-Sebeok1976">1976</a>; Burnet <a href="#ref-Burnet1970">1970</a>)</span>. Thus, the hypothesis that TME can have a positive role in tumor prognosis is not new.</p>
<p>In modern immuno-oncology, the term <em>immune-editing</em> was introduced by <span class="citation">Dunn et al. (<a href="#ref-Dunn2002">2002</a>)</span> in 2002, to describe the relationship between the tumor cells and the immune system. The immunosurveillance through immune-editing can be summarized in three processes: elimination, equilibrium, and escape <span class="citation">(Dunn et al. <a href="#ref-Dunn2002">2002</a>)</span>.</p>
<p>The elimination is the direct killing of cancer cells or growth inhibition by the immune system. The adoptive T cells and NK are actively involved in tumor killing and stimulate other immune cells. The CD8 + cytotoxic lymphocytes (CTLs) directly recognize tumor cells. Employing perforin- and granzyme-dependent mechanisms they can lyse tumor cells. The CD4 + T cells release factors to induce proliferation of B cells and to promote their differentiation to the antibody (Ab)-secreting plasma cells, activate macrophages. Macrophages use phagocytosis to eliminate cancer cells <span class="citation">(Vesely et al. <a href="#ref-Vesely2011">2011</a>)</span>.</p>
<p>The tumor-infiltrating lymphocytes (TILs) have been associated with an overall good prognosis and better survival in different cancer studies. Moreover, abundance of CD3 + and CD8 + T cells, NK cells, and <span class="math inline">\(\gamma\delta\)</span>T cells correlate with improved outcomes in epithelial ovarian cancers <span class="citation">(Marquez-Medina, Salla-Fortuny, and Salud-Salvia <a href="#ref-Marquez-Medina2012">2012</a>)</span>. Several studies report that the presence of the abundant immune infiltrates is correlated with a good prognosis or better survival <span class="citation">(Kornstein, Brooks, and Elder <a href="#ref-Kornstein1983">1983</a>; Baxevanis et al. <a href="#ref-Baxevanis1994">1994</a>; Naito et al. <a href="#ref-Naito1998">1998</a>; Pagès et al. <a href="#ref-Pages2005">2005</a>)</span>. Spontaneous regression of human tumors has been reported in cutaneous melanoma, retinoblastoma, osteosarcoma, etc. <span class="citation">(Aris, Barrio, and Mordoh <a href="#ref-Aris2012">2012</a>)</span>.</p>
<p>The equilibrium is the phase when cancer and immune cells coexist and their crosstalk is preventing metastasis.</p>
<p>T cells are the main actor in maintaining the equilibrium. Progressively, the tumor cells become more immunogenic as they are not edited by the immune system <span class="citation">(Bhatia and Kumar <a href="#ref-Bhatia2011">2011</a>)</span>. The state of tumor cells is then identified as “dormant” and active scientific reports investigate the possible molecular pathways that maintain dormancy or lead to escape <span class="citation">(Teng et al. <a href="#ref-Teng2008">2008</a>)</span>.</p>
<p>The immune escape is the final process when tumor cells impair the immune response.</p>
</div>
</div>
<div id="two-faced-nature-of-immune-cells-context-dependent-functional-plasticity" class="section level4">
<h4><span class="header-section-number">1.1.2.2</span> Two-faced nature of immune cells: context-dependent functional plasticity</h4>
<p>A modern vision of TME-tumor interactions assumes that tumor can be directed to several molecular pathways. This direction is decided by signals that are native of tumor cell and/or coming from the microenvironment.</p>
<p>Recent studies unveil ambivalent nature of immune cells in TME. While some as cytotoxic T cells, B cells and macrophages can manage to eliminate tumor cells. Treg cells role is to regulate expansion and activation of T and B cells. Depending on cancer type, they can be either pro- or anti-tumor. For example, as it has been shown for T-regs, that are usually associated with bad prognosis, they can be equally associated with improved survival (i.e. in colorectal cancer <span class="citation">(Frey et al. <a href="#ref-Frey2010">2010</a>)</span>). For innate immunity, there are widely accepted M1 (anti-tumor) and M2 (pro-tumor) extreme macrophages phenotypes in TME <span class="citation">(Qian and Pollard <a href="#ref-Qian2010">2010</a>)</span>. Most of the statements seem to be context dependent and not valid universally across all cancer types. We already mentioned Macrophages phenotypic plasticity as well as the different behavior of EMC depending on tumor stage.</p>
<p>From a more general point of view, it has been observed that immunodeficiency can correlate with high cancer incidence. Results of analysis based on observations of 25,914 female immunosuppressed organ transplant recipients, the tumor incidence was higher than predicted for multiple cancers. However, the number of breast cancer cases decreased which can be really disturbing if we need to decide on the role of immune defense in tumor progression <span class="citation">(Stewart et al. <a href="#ref-Stewart1995">1995</a>)</span>. This indicates that immune microenvironment can be cancer stimulating or inhibiting depending on the type of cancer and/or other factors.</p>
</div>
<div id="immune-cell-subtypes-in-tme" class="section level4">
<h4><span class="header-section-number">1.1.2.3</span> Immune cell (sub)types in TME</h4>
<p>We are taught that a cell is the basic structural, functional, and biological unit of all known living organisms. A human body contains around <span class="math inline">\(10^{14}\)</span> which is three orders of magnitude more than the number of stars in the Milky Way. This ensemble of cells is traditionally classified into cell types based on their phenotypical variety.</p>
<blockquote>
<p><em>for their immense number, the variety of cells is much smaller: only about 200 different cell types are represented in the collection of about <span class="math inline">\(10^{14}\)</span> cells that make up our bodies. These cells have diverse capabilities and, superficially, have remarkably different shapes….</em> <span class="citation">Boal (<a href="#ref-Boal2002">2002</a>)</span></p>
</blockquote>
<p>In the description of TME, I have referred to cell types of immune cells as well-established entities of the immune system. However, the definition of cell types remains controversial and there is no consensus among researchers on how exactly a cell type should be defined. The notion of the cell-subtypes is even vaguer. The problem does not only concerns immune cells, most of the cell types of our organism, classified initially according to their morphology, seem to fulfill multiple functions. One can also relate cell-type problem to species problem where scientist also debates about where to draw the borders between species. This problem is widely generalized as “theory of types” <span class="citation">(Slater <a href="#ref-Slater2013">2013</a>)</span> in many disciplines as philosophy, linguistics, mathematics.</p>
<p>In this chapter, I will limit the description to immune cell types.</p>
<p>An immune cell can be described nowadays along many axes:</p>
<ul>
<li>Phenotype /surface markers</li>
<li>Morphology (expressed proteins)</li>
<li>Ultrastructure (electron microscopy)</li>
<li>Molecular data (gene expression, genotype, epigenome)</li>
<li>Cell fate</li>
<li>Cell of origin</li>
<li>Function</li>
</ul>
<p>Depending on how well a cell is different from all other cells along with those axes, it will (or not) be defined as a distinct cell type. Each of these axes contains a pièce of information that can agree with other axis or not. These features can be independent or can overlap depending on cell types in question. Historically, there were given different importance based on the technologies and general tendencies. Thus, there is no available general recipe applicable to discriminate all cell types from each other. Moreover, usually, it is not possible to measure all these axes simultaneously (because of the experimental, money or other constraints). Therefore, depending on the scientific question, different researches will give different weights to these axes and a combination of 2-3 <em>most important</em> features will be used to discriminate cell types in a given study.</p>
<p>Besides, the discrimination of the cell types comes with more or less subjective threshold on where the cells become <em>significantly different</em>. These thresholds can be established computationally or by an expert. The usual practice is a mix of both methods.</p>
<p>Since the beginning of immunology, there was disagreement between pre-defined cell types and cell functions.</p>
<blockquote>
<p><em>Cette espèce de leucocytes a une grande ressemblance avec certains éléments fixes du tissu conjonctif, ainsi qu’avec des cellules endothéliales et des cellules de la pulpe splénique. On est donc souvent embarrassé, surtout lorsqu’on trouve ces leucocytes mononucléaires en dehors des vaisseaux, pour les distinguer des autres espèces de cellules mentionnées.</em>
— Elie Metchnikoff, Leçons sur la pathologie comparée de l’inflammation, 1891</p>
</blockquote>
<p>The definition of cell types and subtypes is widely discussed today with the arrival of single cell technologies that allow a change of paradigm in cell classifications. Up to now, the top-down approach was mostly used. A pre-defined set of parameters describing a cell was fixed in order to select cells and then other parameters were measured. Now, it is possible to practice bottom-up approach where all (or some) parameters are measured for a single cell and then, depending on its distance from other cells, cell types are defined <span class="citation">(Satija and Shalek <a href="#ref-Satija2014">2014</a>)</span>.</p>
<blockquote>
<p><em>The concept of “cell type” is poorly defined and incredibly useful</em><br />
— Allon Klein, Harvard Medical School</p>
</blockquote>
<p>Researchers recognize that the concept of cell type is artificial and a continuum of cell types is closer to the reality. According to Susanne Rafelski,</p>
<blockquote>
<p><em>A useful way to classify cells might thus be a multiscale and multi-parameter cell-type space that includes vectors for key intracellular organizational, dynamic, and functional features as well as tissue location, gene expression etc.</em></p>
</blockquote>
<p>Some, as Allon Klein, propose to introduce a concept of <em>cell states</em> which would better describe a cell depending on its context and function. However, an emerging challenge would be to connect <em>cell states</em> with historical <em>cell types.</em> <span class="citation">(Ediorial Cell Systems <a href="#ref-EdiorialCellSystems2017">2017</a>)</span> .</p>
<p>Another aspect of cells, that I am not approaching in this thesis, is time. Cells are shaped by their environment, intrinsic and extrinsic events and can change states, functions etc. Can one cell belong to different cell types depending on its trajectory? How to include the dynamic aspect of the cells into the classification?</p>
<p>Thus, most scientists agree that used convention of cell types is not ideal and it is more matter of convenience than biological reality. This leaves a room to study cells and challenge existing classification. Describing cell types or cell states in the tumor microenvironment is extremely interesting as still little is know about the diversity of cell infiltrated in solid tissues.</p>
</div>
<div id="summary-1" class="section level4">
<h4><span class="header-section-number">1.1.2.4</span> Summary</h4>
<p>Cancer is a disease concerning milliards of people with a long history. Scientific community recognizes the role of the environment where the tumor cells find themselves as an important factor influencing tumor development, prognosis and response to treatment. TME is a complex environment that constantly interacts with tumor cells, where both tumor and TME influence and shape each other.</p>
<p>Over the years, many interactions are being discovered and cell types re-defined and described in their context. However, lots of mechanisms and interactions of TME remains unknown due to very heterogeneous nature of this microenvironment. This leaves room for a more extensive investigation of TME.</p>
<p>A therapeutic goal is target interactions that would be able to pivot the essential processes in tumorigenesis or tumor escape in order to put the cells “back on track” and facilitate anti-tumor therapies.</p>
<p>These goals can be met thanks to the improvement of investigation methods, data quality and abundance. I will discuss the most important data types used in this project to investigate the TME.</p>
</div>
</div>
</div>
<div id="quantifying-and-qualifying-immune-infiltration-data" class="section level2">
<h2><span class="header-section-number">1.2</span> Quantifying and qualifying immune infiltration (data)</h2>
<p>Nowadays, more and more biological data is produced. However, this proliferation of accessible resources is not proportional to generated insights and wisdom. In this thesis, I aim to generate <em>Knowledge</em> and <em>Insights</em> and we hope to generate some <em>Wisdom</em> (Fig. <a href="intro.html#fig:information-power">1.4</a>). In this section, we will introduce the foundation of our analysis: different data types that will be further discussed and explored in chapters that follow.</p>
<div class="figure" style="text-align: center"><span id="fig:information-power"></span>
<img src="figures-ext/01-Information_power.jpg" alt="From Data to Wisdom. Illustration of different steps that it takes to go from Data to generating Wisdom. It highlights that generating data is not equal to understanding it and additional efforts are needed to generate value. Image authored by Clifford Stoll and Gary Schubert published by Portland Press Limited on behalf of the Biochemical Society and the Royal Society of Biology and distributed under the Creative Commons Attribution License 4.0 (CC-BY) in (Ponting 2017)." width="80%" />
<p class="caption">
Figure 1.4: <strong>From <em>Data</em> to <em>Wisdom</em></strong>. Illustration of different steps that it takes to go from <em>Data</em> to generating <em>Wisdom</em>. It highlights that generating data is not equal to understanding it and additional efforts are needed to generate value. Image authored by Clifford Stoll and Gary Schubert published by Portland Press Limited on behalf of the Biochemical Society and the Royal Society of Biology and distributed under the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License 4.0 (CC-BY)</a> in <span class="citation">(Ponting <a href="#ref-Ponting2017">2017</a>)</span>.
</p>
</div>

<p>As discussed with the previous section cell-types, but also the whole systems can be described at different levels (along different axes). These different levels demand distinct technologies and strategies to be developed to enable the measurements. For instance, a phenotypic distinction between cells can be reached using FACS technology, for molecular profiles omic methods were developed and for ultrastructure microscopic methods. We need to approach biological systems from different angles as no one of these axis provide a complete picture of the studied system.</p>
<p>I will introduce most relevant data types that are used to study immune infiltration of tumors.</p>
<div id="facs" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Cell sorting</h3>
<div id="flow-cytometry" class="section level4">
<h4><span class="header-section-number">1.2.1.1</span> Flow cytometry</h4>
<p>Flow cytometry is a laser-based technology. It uses marker genes: cell surface proteins to sort cells in different compartments. Nowadays, it permits quantification of the abundance of up to 17 cell surface proteins using fluorescently labeled antibodies <span class="citation">(Papalexi and Satija <a href="#ref-Papalexi2017">2017</a>)</span>. However this techniques is not free from bias, our knowledge about cell markers is limited and several markers may not be relevant in some context. Moreover, the scientific community did not clearly agree on the marker choice even for popular and well-studied cell types which introduced additional heterogeneity when independent studies are compared. Also, the quality of antibodies may influence the results of the FACs analysis. Besides those limitations FACs remains quite a popular method for analyzing cells in complex tissues. It was among first methods that allowed molecular phenotyping of immune cells, a discovery of numerous subsets and their further functional interpretation.</p>
</div>
<div id="mass-cytometry" class="section level4">
<h4><span class="header-section-number">1.2.1.2</span> Mass cytometry</h4>
<p>Mass cytometry (also known as CyTOF allows for the quantification of cellular protein levels by using isotopes. It allows to quantify up to 40 proteins per cell <span class="citation">(Papalexi and Satija <a href="#ref-Papalexi2017">2017</a>)</span>. It also demands lower starting number of cells (1000 - 1000000), a realistic number that can be extracted from patient biopsy <span class="citation">(Lyons et al. <a href="#ref-Lyons2017">2017</a>)</span>.</p>
</div>
</div>
<div id="staining" class="section level3">
<h3><span class="header-section-number">1.2.2</span> Microscope Staining</h3>
<p>Using microscope technics, histopathological cuts are analyzed. The number of cells per a unit of area (i.e. mm<span class="math inline">\(^2\)</span>) is defined either manually by a human or through diverse image analysis algorithms.</p>
<p>Current pathology practice utilizes chromogenic immunohistochemistry (IHC) <span class="citation">(NPR <a href="#ref-RamosVara2010">2010</a><a href="#ref-RamosVara2010">a</a>)</span>. Multiplexed approaches allow identifying multiple markers in the same histopathology cut. Modern techniques like imaging mass cytometry using FFPE tissue samples uses fluorescence and mass cytometry to identify and quantify marker proteins <span class="citation">(Giesen et al. <a href="#ref-Giesen2014">2014</a>)</span>.</p>
<p>The main advantage of aforementioned technics the number of cells that can be analyzed and the information about the spatial distribution of the different cell types. The limiting factor, as for <a href="intro.html#facs">cell sorting methods</a>, is the number of markers (~10-100) and consequently a number of cell types that can be identified <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>.</p>
<p>The <a href="intro.html#facs">cell sorting methods</a> and <a href="intro.html#staining">microscope staining</a> are usually considered as a gold standard for multidimensional data techniques. The reason why they are not applied at large scale is the cost but also quite laborious and time-consuming sample preparation demanding a fresh sample. In contrast, the <a href="intro.html#omics">omics methods</a> propose a more scalable way to measure tumor microenvironment.</p>
<div id="tissue-microarrays" class="section level4">
<h4><span class="header-section-number">1.2.2.1</span> Tissue Microarrays</h4>
<p>Tissue Microarrays aim to automatize “staining” techniques. A large number of small tissue segments can be organized in a single paraffin block where 100 tissue samples can be easily examined on one slide. A variety of molecular or microscopic method can be then applied to FFPE tissue including immunohistochemistry, FISH, and in situ hybridization <span class="citation">(Wilczynski <a href="#ref-Wilczynski2009">2009</a>)</span>. It is a technique in between traditional imaging and omic high-throughput.</p>
</div>
</div>
<div id="omics" class="section level3">
<h3><span class="header-section-number">1.2.3</span> omics</h3>
<p>In biological systems information is coded in the form of DNA that do not vary a lot between different individuals of the same species. To trigger a function in an organism, a part of the DNA is transcribed to RNA, depending on the intrinsic and extrinsic factors, and after additional modification messenger RNA (mRNA) is translated into a protein (i.e., digestive enzyme) that fulfill a role in the organism. The mRNA information (also called transcriptome) can be captured with experimental methods at high throughput (transcriptomics) and provides an approximation of the state of the studied system (i.e., a tissue). There is also information, not coded on the DNA sequence but in a pattern of chemical species that can regulate the state transition of DNA information. These additional regulators are called epigenome collectively and some of them, like methylation, can also be measured at high-throughput.</p>
<div id="transcriptome" class="section level4">
<h4><span class="header-section-number">1.2.3.1</span> Transcriptome</h4>
<p>Transcriptomics measures the number of counts of mRNA molecules using high-throughput techniques. mRNA is the part of the genetic information that should be translated into proteins. It reflects the activity of ongoing processes in a cell. In contrast to DNA, mRNA concentration can be highly variable <span class="citation">(Velculescu et al. <a href="#ref-Velculescu1997">1997</a>)</span>. This variability can be either “intrinsic” that reflect the stochastic process of cell machinery or “extrinsic” reflecting impact of factors upstream to mRNA synthesis <span class="citation">(Satija and Shalek <a href="#ref-Satija2014">2014</a>)</span>.</p>
<p>Transcriptome can be measured by microarrays or RNA-seq NGS technology. Microarrays remain cost-efficient and popular technique designed in 90. There exist two and one fluorescent color probes, both representing different challenges in experimental design for batch effect removal. RNA-seq, in contrast, uses sequenced RNA to quantify the expression. As not only selected genes (probes) are quantified, it can be used to study unknown parts of the genome. RNA-seq is also characterized by lower background noise than microarrays.</p>
<p>Bulk transcriptome data are quite accessible nowadays. They can be obtained from either flash-frozen or formalin-fixed, paraffin-embedded (FFPE) tissue samples, including both surgically resected material and core needle biopsies <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>.</p>
<p>The main flaw of transcriptomic data is that the reproducibility between different platforms is limited. As a result, direct comparison (direct merging, statistical difference tests) between two datasets produced by different platforms is not advised. There are 12 thousand genes that are matching between four sequencing platforms. Through gene names conversions much information is lost, and bias is introduced.</p>
<p>In bulk transcriptome samples of heterogeneous tissues, different cells cannot be distinguished (Fig. <a href="intro.html#fig:screvi">1.6</a>A).</p>
<p>Different strategies can be adapted to analyze bulk transcriptome.</p>
<p><span class="citation">Cieślik and Chinnaiyan (<a href="#ref-Cieslik2017">2017</a>)</span> describes five groups of most popular approaches that can be applied to study transcriptome (Fig. <a href="intro.html#fig:transcriptome-methods">1.5</a>). Despite a diversity of bioinformatic and statistical tools, the most popular differential approaches, mainly differential gene expression (DGE) based on the difference between two experimental conditions.</p>
<div class="figure" style="text-align: center"><span id="fig:transcriptome-methods"></span>
<img src="figures-ext/transcriptome-methods.png" alt="Five categories of RNA-seq data analysis. Differential analyses: comparing two (or more) conditions, Relative analyses: comparing to an internal reference (average, base level), Compositional analyses: inferring cell types or groups of cell types (i.e., tumor purity), Global analyses: pan-tissue and pan-cancer analyses and Integrative analyses: compiling heterogeneous data types. Reprinted by permission from Springer Nature (Cieślik and Chinnaiyan 2017) © 2018 Macmillian Publishers Limited, part of Springer Nature. All rights reserved." width="100%" />
<p class="caption">
Figure 1.5: <strong>Five categories of RNA-seq data analysis</strong>. Differential analyses: comparing two (or more) conditions, Relative analyses: comparing to an internal reference (average, base level), Compositional analyses: inferring cell types or groups of cell types (i.e., tumor purity), Global analyses: pan-tissue and pan-cancer analyses and Integrative analyses: compiling heterogeneous data types. Reprinted by permission from Springer Nature <span class="citation">(Cieślik and Chinnaiyan <a href="#ref-Cieslik2017">2017</a>)</span> © 2018 Macmillian Publishers Limited, part of Springer Nature. All rights reserved.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:screvi"></span>
<img src="figures-ext/sing_cell_reiview.png" alt="Overview of approaches to identify cell types with different technologies: bulk transcriptome, scRNA-seq and spatial transcriptomics. A - Bulk analysis: cell types are mixed in a sample and can be found in the mixture thanks to known refrence cell type profiles, B - In single-cell analysis, tissues are first dissociated to a single-cell suspension and then profiled individually by single-cell RNA sequencing (scRNA-seq). Cell types, including previously unknown cell types, can be identified but no positional information is retained. C - In spatial analysis, sections are cut from tissue blocks and the position of cell types within the tissue can be identified using marker genes. This technique generates two-dimensional (2D) information but often for only a few genes. Once spatial data are available, scRNA-seq data can be mapped back onto the tissue and it is possible to reconstruct a 2D image of the tissue. Reprinted by permission from Annu. Rev. Biomed. Data Sci. (Chen, Teichmann, and Meyer 2018) © Copyright 2018. All rights reserved." width="100%" />
<p class="caption">
Figure 1.6: <strong>Overview of approaches to identify cell types with different technologies: bulk transcriptome, scRNA-seq and spatial transcriptomics</strong>. A - Bulk analysis: cell types are mixed in a sample and can be found in the mixture thanks to known refrence cell type profiles, B - In single-cell analysis, tissues are first dissociated to a single-cell suspension and then profiled individually by single-cell RNA sequencing (scRNA-seq). Cell types, including previously unknown cell types, can be identified but no positional information is retained. C - In spatial analysis, sections are cut from tissue blocks and the position of cell types within the tissue can be identified using marker genes. This technique generates two-dimensional (2D) information but often for only a few genes. Once spatial data are available, scRNA-seq data can be mapped back onto the tissue and it is possible to reconstruct a 2D image of the tissue. Reprinted by permission from Annu. Rev. Biomed. Data Sci. <span class="citation">(Chen, Teichmann, and Meyer <a href="#ref-Chen2018b">2018</a>)</span> © Copyright 2018. All rights reserved.
</p>
</div>

<p>RNA-seq data was proven to be a useful indicator for clinical applications <span class="citation">(Mody et al. <a href="#ref-Mody2015">2015</a>; Oberg et al. <a href="#ref-Oberg2016">2016</a>; Robinson et al. <a href="#ref-Robinson2017">2017</a>)</span>. Its utility for immune profiling was demonstrated in many studies through the use of transcriptomic signatures to predict immunotherapy response or survival <span class="citation">(Chen et al. <a href="#ref-Chen2016">2016</a>)</span>.</p>
<p>In this work transcriptome data analysis falls into multiple categories: Compositional, Relative and aims to construct Global-level conclusions.</p>
</div>
<div id="single-cell-rna-seq" class="section level4">
<h4><span class="header-section-number">1.2.3.2</span> Single cell RNA-seq</h4>
<p>Described above methods of process DNA from hundreds of thousands of cells simultaneously and report averaged gene expression of all cells. In contrast, scRNA-seq technology allows getting results for each cell individually. This is tremendous step forward enhancement of our understanding of cell heterogeneity and opens new avenues of research questions.</p>
<p>Continuous discovery of new immune subtypes has proven that cell surface markers that are used for phenotyping by techniques like FACS and immunohistochemistry cannot capture the full complexity. ScRNA-seq methods allow clustering known cell types in subpopulations based on their genetic features (Fig. <a href="intro.html#fig:screvi">1.6</a>B). ScRNA-seq is also able to capture particularly rare cell types as it requires much less of RNA material (1 ng isolated from 100-1000 cells) compared to ‘bulk’ RNA-seq ( ~ 1 μg of total mRNA transcripts ). It also allows studying cells at high resolution capturing the phenotypes in much more refined scale than previously <span class="citation">(Papalexi and Satija <a href="#ref-Papalexi2017">2017</a>)</span>.</p>
<p>This new data type also brings into the field new challenges related to data processing due to the volume, distribution, noise, and biases. Experts highlight as the most “batch effect”, “noise” and “dropout effect” <span class="citation">(Perkel <a href="#ref-Perkel2017">2017</a>)</span>. So far, there are no official standards that can be applied which makes data comparison and post-processing even more challenging. Up to date, there are around 70 reported tools and resources for single cell data processing <span class="citation">(Davis <a href="#ref-Davis2016">2016</a>)</span>. A limited number of single-cell datasets of tumors are made publicly available, and more are to come.</p>
<p>One can ask why then developing computational deconvolution of bulk transcriptome if we can learn relevant information from single-cell data. Firstly, that single cell data do not provide a straightforward answer to the estimation of cell proportions. The coverage is not full and sequenced single cells are not entirely representative of the actual population. For instance, neutrophils are not found in scRNA-seq data because of they are “difficult to isolate, highly labile ex vivo and therefore difficult to preserve with current single-cell methods” <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>. Besides, a number of patients included in published studies of range &lt;100 cannot be compared to thousand people cohorts sequenced with bulk transcriptome methods. This is mostly because single cell experiments are challenging to perform, especially in a clinical setting as fresh samples are needed <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>. Today, single cell technology brings very interesting “zoom in” perspective, but it would be incautious to make fundings from a restricted group of individuals universal to the whole population. Primary brake to the use of single cell technology more broadly might be as well the price that is nearly 10x higher for single cell sample compared to bulk <span class="citation">(Core <a href="#ref-Cedar2018">2016</a>)</span>.</p>
<p>In this work, we are using single cell data in two ways. Firstly, in Chapter 5 we compare immune cell profiles defined by scRNA-seq, blood and blind deconvolution (problem introduced in Immune signatures section). Secondly, in Chapter 6 we use single call data of Metastatic melanoma generated by <span class="citation">Tirosh et al. (<a href="#ref-Tirosh2016">2016</a>)</span> to demonstrate heterogeneity of subpopulations of Macrophages and NK cells.</p>
</div>
<div id="epi" class="section level4">
<h4><span class="header-section-number">1.2.3.3</span> Epigenome</h4>
<p>An epigenome can be defined as a record of the chemical changes to the <strong>DNA and histone proteins</strong> of an organism. Changes to the epigenome can provoke changes to the structure of chromatin and changes to the function of the genome <span class="citation">(Bernstein, Meissner, and Lander <a href="#ref-Bernstein2007">2007</a>)</span>. Epigenome data usually contains information about methylation <strong>CpG island changes</strong>. In cancer, global genomic hypomethylation, CpG island promoter hypermethylation of tumor suppressor genes, an altered histone code for critical genes, a global loss of monoacetylated and trimethylated histone H4 were observed. Methylome profiles can also be used as a molecular signature of disease and potential diagnostic or predictive biomarker <span class="citation">(Jeschke et al. <a href="#ref-Jeschke2017">2017</a>)</span>.</p>
</div>
<div id="copy-number-variation-cnv-and-copy-number-aberration-cna" class="section level4">
<h4><span class="header-section-number">1.2.3.4</span> Copy number variation (CNV) and Copy number aberration (CNA)</h4>
<p>The differences between human genome come in the majority from <strong>Copy Number Variation </strong><span class="citation">(McCarroll and Altshuler <a href="#ref-McCarroll2007">2007</a>)</span>. CNV regions constitute  4.8–9.7%  of the whole human genome <span class="citation">(Zarrei et al. <a href="#ref-Zarrei2015">2015</a>)</span>. They can be reflected in structural variation that is duplication or deletion of DNA bases. CNV can affect a lot of base pairs of DNA code (deletion of more than 100 genes) and result in a phenotype change.</p>
<p>In addition, there can be distinguished, <strong>Copy number alterations/aberrations (CNAs)</strong> that are changes in copy number that have arisen in <strong>somatic</strong> tissue (for example, just in a tumor), in contrast to CNV that originated from changes in copy number in <strong>germline</strong> cells (and are thus in all cells of the organism) <span class="citation">(McCarroll and Altshuler <a href="#ref-McCarroll2007">2007</a>)</span>. CNV and CNA profiles can be associated with diseases or cancer subtypes.</p>
<p>There exist disease-related exome panels that focus on regions with high copy variation, or the full exome can be sequenced using whole-exome sequencing (WES) <span class="citation">(Yamamoto et al. <a href="#ref-Yamamoto2016">2016</a>)</span>.</p>
</div>
<div id="spatial-transcriptomics" class="section level4">
<h4><span class="header-section-number">1.2.3.5</span> Spatial transcriptomics</h4>
<blockquote>
<p>Spatial transcriptomics provides quantitative gene expression data and visualization of the distribution of mRNAs within tissue sections and enables novel types of bioinformatics analyses, valuable in research and diagnostics <span class="citation">(Ståhl et al. <a href="#ref-Stahl2016">2016</a>)</span></p>
</blockquote>
<p>It combines RNA-seq technology with spatial labeling which allows having a bulk gene expression of 10-20 cells with given space coordinates within the sample (Fig. <a href="intro.html#fig:screvi">1.6</a>). It allows to localize regions of highest gene expression and perform <em>Spatially Variable Genes</em> (<span class="citation">Svensson, Teichmann, and Stegle (<a href="#ref-Svensson2018">2018</a>)</span>). Some attempts were already made to combine Spatial Transcriptomics and scRNA-seq <span class="citation">(Moncada et al. <a href="#ref-Moncada2018">2018</a>)</span>. It remains an early-stage technique, and so far it is not widely used, but it might be a future of omics to add spatial information as it can be essential for many research problems.</p>
</div>
</div>
</div>
<div id="immunotherapies" class="section level2">
<h2><span class="header-section-number">1.3</span> From cancer phenotyping to immune therapies</h2>
<p>This section outlines different methods of cancer immune phenotyping and progress in cancer therapies with a focus on immune therapies. It will link the ongoing research on TME with therapeutical potential.</p>
<div id="cancer-immune-phenotypes" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Cancer immune phenotypes</h3>
<p>Since 20s century physicians decided on common nomenclature that classifies tumors into distinct groups that are relatively homogenous or that share common characteristic important for treatment and prognosis. Tumor typing should help to predict prognosis better, to adopt a therapy to the clinical situation, to enable therapeutic studies which are essential in proving any therapeutic progress.</p>
<p>Most of the classifications are based on clinical data. Most common factors taken into account are the degree of local invasion, the degree of remote invasion, histological types of cancer with specific grading for each type of cancer, possibly various tumor markers, general status of the patient.</p>
<p>However, cancers with similar morphological and histopathological features reveal very distinct patterns of progression and response to therapy <span class="citation">(Galon et al. <a href="#ref-Galon2014">2014</a>)</span>. In the era of gene sequencing, gene and protein expression, as well as epigenome, can provide valuable complementary information. Therefore gene markers or proteomic abnormalities can be integrated into classification panel. One famous example is a gene signature <em>PAM50</em> <span class="citation">(Parker et al. <a href="#ref-Parker2009">2009</a>)</span> used for prediction of patients’ prognosis in breast cancer, patented as a tumor profiling test.</p>
<p>Since the increase of importance of the immunotherapies, researches proposed several ways to classify tumors based on their microenvironment. Given different parameters describing TME, cancers can be sorted into groups that show similar characteristics. We will discuss most common frameworks that allow for phenotype cancers based on the TME.</p>
<p>The localization of the immune cells can be an indicator of the state and response to the therapy <span class="citation">(Bindea et al. <a href="#ref-Bindea2013">2013</a>)</span>.</p>
<p>The most standard approach is to convey an analysis of histopathological cuts to asses the number of infiltrating lymphocytes (TILs). Two typical patterns are usually identified: “hot” - immune inflamed and “cold” - no active immune response <span class="citation">(Berghoff, Kather, and Jäger <a href="#ref-Berghoff2018">2018</a>)</span>.</p>
<p><span class="citation">Chen and Mellman (<a href="#ref-Chen2017">2017</a>)</span> describe classification into inflamed and non-inflamed tumors, where non-inflamed phenotypes: can be further split into the immune-desert phenotype and the immune-excluded phenotype (Fig. <a href="intro.html#fig:immune-phenotypes">1.7</a>). The inflamed phenotype is characterized by the abundant presence of immune cells: T cells, myeloid cells, monocytes in tumor margin. Along with the immune cells, due to their communication, a high expression of cytokines is characteristic for this phenotype. According to <span class="citation">Chen and Mellman (<a href="#ref-Chen2017">2017</a>)</span>, this is a mark that an anti-tumor response was arrested by the tumor. The inflamed phenotype has shown to be most responsive to immunotherapies. In the immune-excluded phenotype, the immune cells are present as well but located in the stroma <span class="citation">(Herbst et al. <a href="#ref-Herbst2014">2014</a>)</span>, sometimes penetrating inside the tumor. However, when exposed to checkpoint immunotherapy, T cells do not gain the ability to infiltrate the tumor; therefore the treatment is inefficient. The immune-desert main features are little or no presence of immune cells, especially T cells. Surprisingly, these tumors have been proven to respond rarely to the checkpoint therapy <span class="citation">(Herbst et al. <a href="#ref-Herbst2014">2014</a>)</span>. In non-inflamed tumors, cytokines associated with immune suppression or tolerance are expressed.</p>
<div class="figure" style="text-align: center"><span id="fig:immune-phenotypes"></span>
<img src="figures-ext/immune-phenotypes.png" alt="Cancer-immune phenotypes: the immune-desert phenotype (brown), the immune-excluded phenotype (blue) and the inflamed phenotype (red). The immune-desert phenotype is characterized by a paucity of immune cells and cytokines. In the immune-excluded phenotypes, the T cells are often present but trapped in the stroma, enabled to migrate to the tumor site. The immune-inflamed phenotype is rich in immune cells and the most responsive to the immune checkpoint therapies. Reprinted by permission from Springer Nature (Chen and Mellman 2017) © 2017 Macmillian Publishers Limited, part of Springer Nature. All rights reserved." width="100%" />
<p class="caption">
Figure 1.7: <strong>Cancer-immune phenotypes: the immune-desert phenotype (brown), the immune-excluded phenotype (blue) and the inflamed phenotype (red).</strong> The immune-desert phenotype is characterized by a paucity of immune cells and cytokines. In the immune-excluded phenotypes, the T cells are often present but trapped in the stroma, enabled to migrate to the tumor site. The immune-inflamed phenotype is rich in immune cells and the most responsive to the immune checkpoint therapies. Reprinted by permission from Springer Nature <span class="citation">(Chen and Mellman <a href="#ref-Chen2017">2017</a>)</span> © 2017 Macmillian Publishers Limited, part of Springer Nature. All rights reserved.
</p>
</div>

<p>A presence of immune phenotypes was confirmed by for example by <span class="citation">Becht et al. (<a href="#ref-Becht2016">2016</a>)</span> in colorectal cancer, where after deconvolution of bulk tumor profiles, patterns of immune and stromal cells abundance was matching four cancer subtypes. The good prognosis was related to cytotoxic response and bad prognosis to lymphocytes and cells of monocytic origin.</p>
<p>According to <span class="citation">Gajewski et al. (<a href="#ref-Gajewski2006">2006</a>)</span>, the immunogenicity of the tumors can be explained by tumor-intrinsic factors and tumor-extrinsic factors. Tumor-intrinsic factors are the neoantigen load and frequency, the mutational load, the expression of immunoinhibitors and immunostimulators (e.i. PD-L1), and alteration of HLA class I molecules. Tumor-extrinsic factors include chemokines regulating T cell trafficking, infiltration of effector TILs and immunosuppressive TILs, and soluble immunomodulatory factors (cytokines).</p>
</div>
<div id="scoring-the-immune-infiltration" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Scoring the immune infiltration</h3>
<p>Experimental techniques and computational tools enabled us to characterize and classify TME with multi-omics data. Here I present <strong>a short list of most recent and influencing</strong> analysis aiming to redefine tumor phenotypes based on the immune infiltration, with a focus on computational techniques.</p>
<div id="immunoscore" class="section level4">
<h4><span class="header-section-number">1.3.2.1</span> Immunoscore</h4>
<p>Jerôme Galon lab in Paris authors one of the most recognized scoring method, based on fluorescent images and names <a href="http://www.haliodx.com/clinical-research-services/immunoscorer/">Immunoscore</a>. The Immunoscore ranges from 0 to 4 and it is based on the density of lymphocyte populations CD3/CD45RO, CD3/CD8, or CD8/ CD45RO. It also takes into account the spacial position of the cells: the tumor core and margins <span class="citation">(Galon et al. <a href="#ref-Galon2012">2012</a>)</span>. It was successfully applied to colorectal cancer to predict patients’ survival <span class="citation">(Anitei et al. <a href="#ref-Anitei2014">2014</a>)</span>. Since then, it resulted in numerous application to many cancer types. Immunoscore has been recently validated in a large cohort international independent study (14 centers in 13 countries) as a relevant prognostic score of time to recurrence, defined as the time from surgery to disease recurrence <span class="citation">(Pagès et al. <a href="#ref-Pages2018">2018</a>)</span>.</p>
<p>The immunoscore is an interesting indicator, especially in the scope of clinical applications, although it does not tell us a lot about underlying biology. It is also limited to a few cell types while it may be that in some cancer types or patients, the system requires more detailed or rich analysis of a larger panel of cells.</p>
</div>
<div id="spatiotemporal-dynamics-of-intratumoral-immune-cells-of-colorectal-cancer" class="section level4">
<h4><span class="header-section-number">1.3.2.2</span> Spatiotemporal dynamics of Intratumoral Immune Cells of Colorectal Cancer</h4>
<p><span class="citation">Bindea et al. (<a href="#ref-Bindea2013">2013</a>)</span> published a quite complete, and supported with strong experimental evidence, immune landscape of colorectal cancer. Authors introduced <em>the immunome compendium</em> containing 577 cell-type-specific genes, derived from analysis of a significant corpus of publicly available data. They used it to analyze CRC large transcriptomic data (105 patients). Using qPCR (more sensitive technique than microarray) expression of 81 “representative” genes from the compendium was investigated in 153 CRC patients. This study validated correlation of markers of the same type and also revealed the correlation of different cell-type markers (i.e., T-cells and NK or Th and macrophages). The data matrix was grouped into 3 clusters which were corresponding to 1) tumor 2) adaptive 3) innate immune responses. Besides, spatial positioning of markers was visualized thanks to Tissue Microarray technology in samples from 107 CRC patients distinguishing marker densities in tumor center and tumor margin areas. This was followed by an in-depth study of chemokines expression and genomic alterations. Also, authors validated potential prognostic biomarkers in murine orthotopic CRC models.</p>
<p>In summary, using marker genes measured and visualized with different data types of CRC, a high inter-patient heterogeneity was observed. Adaptive immunity cells were associated with the core of the tumor and the innate ones with the tumor margin. A mechanism involving CXCL13, Tfh cells, B cells and IL-21 was identified as associated with good prognosis.</p>
<p>Authors suggest a dynamic dimension of the study which is in practice comparison between tumor stages. It can be argued that used time scale is too discrete and true dynamics cannot be reflected only along tumor stages in different patients. It is extremely challenging to access truly dynamic data for human tumor biopsies, but some efforts are made in the direction of inclusion of sequential biopsies <span class="citation">(“Sequential liquid biopsies reveal dynamic alterations of EGFR driver mutations and indicate EGFR amplification as a new mechanism of resistance to osimertinib in NSCLC.” <a href="#ref-Knebel2017">2017</a>)</span> that allow better time resolution. In brief, the field is still waiting for the landscape of TME truly dynamic in space and time.</p>
</div>
<div id="immunophenoscore" class="section level4">
<h4><span class="header-section-number">1.3.2.3</span> Immunophenoscore</h4>
<p>Different approaches, sub-typing oriented, are based principally on gene expression patterns. Most commonly, machine learning supervised algorithms are trained to match known phenotype (established with microscopy or with clinical features) to genetic patterns, or an unsupervised clustering is used to discover new classification.</p>
<p>An example of well-formulated classification framework is Immunophenoscore <span class="citation">(Charoentong et al. <a href="#ref-Charoentong2017">2017</a>)</span>, based on the publication of <span class="citation">Angelova et al. (<a href="#ref-Angelova2015">2015</a>)</span>, where methylome, transcriptome and mutation of TCGA CRC dataset (n = 598) was used to describe <em>immunophenotypes</em>. Later on, it was reduced to gene expression indicator and summarised in the form of a score. This scoring scheme is based on the data of 20 solid tumors, using the expression of marker genes selected by a machine learning algorithm (random forest) for best prediction in each cancer. These indicators can be grouped into four categories:</p>
<ul>
<li>MHC molecules (MHC)</li>
<li>Immunomodulators (CP)</li>
<li>Effector cells (EC)</li>
<li>Suppressor cells (SC)</li>
</ul>
<p>The immunophenscore (IPS) is calculated on a 0-10 scale based on the expression of genes in each category. Stimulatory factors (cell types) impact the score positively and inhibitory factors (cell types) negatively. Z-scores <span class="math inline">\(\geq\)</span> three were designated as IPS10 and z-scores <span class="math inline">\(\leq\)</span> 0 are designated as IPS0. A similar conceptual framework called <em>cancer immunogram</em> was proposed by <span class="citation">Blank et al. (<a href="#ref-Blank2016">2016</a>)</span> included seven parameters: tumor foreignness (Mutational load), general immune status (Lymphocyte count), immune cell infiltration (Intratumoral T cells), absence of checkpoints (PD-L1), absence of soluble inhibitors (IL–6, CRP), absence of inhibitory tumor metabolism (LDH, glucose utilisation), tumor sensitivity to immune effectors (MHC expression, IFN-<span class="math inline">\(\gamma\)</span> sensitivity). <span class="citation">Charoentong et al. (<a href="#ref-Charoentong2017">2017</a>)</span> claim that the immunophenoscore can predict response to CTLA-4 and anti-PD-1.</p>
<p>Nonetheless, the details of the use of <em>cancer immunogram</em> in practice remain unclear and the result could be sensitive to patients’ and data heterogeneity as no standardization was proposed. It should also be validated in a systematic, independent study.</p>
</div>
<div id="the-immune-landscape-of-cancer" class="section level4">
<h4><span class="header-section-number">1.3.2.4</span> The immune landscape of cancer</h4>
<table class="table table-striped table-bordered" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:C6">Table 1.1: </span><strong>Six immunological subtypes of cancer</strong>. The general characteristic of subtypes generated by <span class="citation">Thorsson et al. (<a href="#ref-Thorsson2018">2018</a>)</span> as described in the original publication.
</caption>
<thead>
<tr>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Cluster
</th>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Features
</th>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Macrophage..lymphocyte
</th>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Th1.Th2
</th>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Proliferation
</th>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Intratumoral.heterogeneity
</th>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Other
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;">
C1
</td>
<td style="text-align:center;">
Wound healing
</td>
<td style="text-align:center;">
Balanced
</td>
<td style="text-align:center;">
Low
</td>
<td style="text-align:center;">
High
</td>
<td style="text-align:center;">
High
</td>
<td style="text-align:center;">
</td>
</tr>
<tr>
<td style="text-align:center;">
C2
</td>
<td style="text-align:center;">
IFN-γ dominant
</td>
<td style="text-align:center;">
Lowest
</td>
<td style="text-align:center;">
Lowest
</td>
<td style="text-align:center;">
High
</td>
<td style="text-align:center;">
Highest
</td>
<td style="text-align:center;">
Highest M1 and CD8 T cells
</td>
</tr>
<tr>
<td style="text-align:center;">
C3
</td>
<td style="text-align:center;">
Inflammatory
</td>
<td style="text-align:center;">
Balanced
</td>
<td style="text-align:center;">
High
</td>
<td style="text-align:center;">
Low
</td>
<td style="text-align:center;">
Lowest
</td>
<td style="text-align:center;">
Highest Th17
</td>
</tr>
<tr>
<td style="text-align:center;">
C4
</td>
<td style="text-align:center;">
Lymphocyte depleted
</td>
<td style="text-align:center;">
High
</td>
<td style="text-align:center;">
Minimal Th
</td>
<td style="text-align:center;">
Moderate
</td>
<td style="text-align:center;">
Moderate
</td>
<td style="text-align:center;">
</td>
</tr>
<tr>
<td style="text-align:center;">
C5
</td>
<td style="text-align:center;">
Immunologically quiet
</td>
<td style="text-align:center;">
Highest
</td>
<td style="text-align:center;">
Minimal Th
</td>
<td style="text-align:center;">
Low
</td>
<td style="text-align:center;">
Low
</td>
<td style="text-align:center;">
Highest M2
</td>
</tr>
<tr>
<td style="text-align:center;">
C6
</td>
<td style="text-align:center;">
TGF-β dominant
</td>
<td style="text-align:center;">
High
</td>
<td style="text-align:center;">
Balanced
</td>
<td style="text-align:center;">
Moderate
</td>
<td style="text-align:center;">
Moderate
</td>
<td style="text-align:center;">
Highest TGF-β signature
</td>
</tr>
</tbody>
</table>
<p><span class="citation">Thorsson et al. (<a href="#ref-Thorsson2018">2018</a>)</span> performed a multi-omic analysis of TCGA datasets that allowed them to define six subtypes that are valid across cancer types (see Tab. <a href="intro.html#tab:C6">1.1</a> ).</p>

<p>Authors selected eight indicators to define these six phenotypes:</p>
<ol style="list-style-type: decimal">
<li>differences in macrophage or lymphocyte signatures</li>
<li>Th1:Th2 cell ratio</li>
<li>extent of intratumoral heterogeneity</li>
<li>aneuploidy</li>
<li>extent of neoantigen load</li>
<li>overall cell proliferation</li>
<li>expression of immunomodulatory genes</li>
<li>prognosis</li>
</ol>
<p>These indicators were selected among many other indicators though machine learning (elastic net regression) for the best predictive power of survival.</p>
<p>All the data and computed parameters can be accessed at <a href="https://isb-cgc.shinyapps.io/shiny-iatlas/">CRI iAtlas Portal</a>. Among the six phenotypes C3 (Inflammatory) has the best-associated prognosis while C1 (wound healing) and C2 (IFN-<span class="math inline">\(\gamma\)</span> dominant), much less favorable outcome. This again illustrates the ambivalent nature of the immune system as the best, and the worst prognosis is associated with immunologically active tumors. C4 (lymphocyte depleted) and C6 (TGF-<span class="math inline">\(\beta\)</span> dominant) subtypes had the worst prognosis. The content of immune cells was determined using different tools and data types (expression, DNA methylation, images, etc.) We can learn a lot from the study. However, it seems difficult to integrate the methods into an ordinary practice because different data levels are necessary for the same samples to compute all the indicators.</p>
</div>
<div id="a-pan-cancer-landscape-of-immune-cancer-interactions-in-solid-tumors" class="section level4">
<h4><span class="header-section-number">1.3.2.5</span> A pan-cancer landscape of immune-cancer interactions in solid tumors</h4>
<p>A different classification was proposed by <span class="citation">Tamborero et al. (<a href="#ref-Tamborero2018">2018</a>)</span>, also using TCGA data. They distinguished 17 immune infiltration patterns based on the immune cell proportions and 6 different clusters based on cytotoxicity measure across all cancer types (named immune-phenotypes) that were finally summarized in three groups: cytotoxic immune infiltrate, infiltrate with more immune-suppressive component and poor immune infiltrate. According to the analysis, one of the most critical factors is cytotoxicity. Tumors with high cytotoxicity were characterized by low clonal heterogeneity, with gene alterations regulating epigenetic, antigen presentation and cell-cell communication. The medium-level cytotoxic tumors had activated invasion and remodeling of adjacent tissue, probably favorable to immune-suppressive cells. The low cytotoxicity subgroup of tumors had altered: cell-cycle, hedgehog, <span class="math inline">\(\beta\)</span>- catenin and TGF-<span class="math inline">\(\beta\)</span> pathways. This result roughly overlaps with the one of <span class="citation">Thorsson et al. (<a href="#ref-Thorsson2018">2018</a>)</span>. The survival analysis based on the six immune-phenotypes revealed that for most cancer types, high cytotoxic tumors are associated with better survival. To evaluate tumor environment cells, authors used gene set variation analysis <span class="citation">(Hänzelmann, Castelo, and Guinney <a href="#ref-Hanzelmann2013">2013</a>)</span> with a set of pre-defined cell-type markers. Another important conclusion of <span class="citation">Tamborero et al. (<a href="#ref-Tamborero2018">2018</a>)</span> is that tissue of origin is not the only important factor shaping cell-type patterns in tumors. However, the least infiltrated tumors were lung, uterine and bladder cancers, while the most infiltrated were pancreatic, kidney, skin cancers and glioblastoma. They also analyzed cancer cell pathways after computational purification of tumor samples (subtraction of the immune signal) to better understand cancer signaling.</p>
<p>A different approach is to characterize tumors based on signaling pathways organized in functional modules.</p>
</div>
<div id="immune-maps" class="section level4">
<h4><span class="header-section-number">1.3.2.6</span> Immune maps</h4>
<p>Another way to summarize tumor phenotype can be through the use of molecular maps. <a href="https://acsn.curie.fr/">Atlas of Cancer Signaling Network (ACSN)</a> <span class="citation">(Kuperstein et al. <a href="#ref-Kuperstein2013">2013</a>; Kuperstein et al. <a href="#ref-Kuperstein2015">2015</a>)</span> is a pathway database that contains a collection of interconnected cancer-related signaling network maps. An additional feature is ACSN web-based Google-maps-like visualization of the database. User data can be projected on the molecular map (for example gene/protein expression from user data can be paired with entities on the map ). ACSN 2.0 contains Cancer cell map and TME map (at the time: angiogenesis, innate immune map, T-cell signaling maps). All separate maps are available in <a href="https://navicell.curie.fr/pages/maps.html">Navicell website</a>. Through projection of the data on the innate immunity map, one can see if a tumor sample is characterized by pro- or anti-tumor activated pathways due to the organization of the map layout. Also, different CAF subtypes were characterized by the CAF specific map in <span class="citation">(Costa et al. <a href="#ref-Costa2018">2018</a>)</span>. Kondratova and colleagues (including myself) used the innate immune map to characterize NK and Macrophages subtypes (<a href="map.html#map">see Chapter Z</a>).</p>
</div>
<div id="summary-2" class="section level4">
<h4><span class="header-section-number">1.3.2.7</span> Summary</h4>
<p>Despite all scientific efforts, the gene expression-based classifications are not yet used in clinics. The measured multi-panel mRNA expression, which can be included into the category of In Vitro Diagnostic Multivariate Index Assay (IVDMIA) <span class="citation">(Győrffy et al. <a href="#ref-Gyorffy2015">2015</a>; Ross et al. <a href="#ref-Ross2008">2008</a>)</span>, may be a future of TME-based cancer classification, diagnosis and treatment recommendation <span class="citation">(Gnjatic et al. <a href="#ref-Gnjatic2017">2017</a>)</span>. For this best tools need to be used to evaluate the state of TME and tumor-stroma-immune cells communication properly.</p>
</div>
</div>
<div id="immune-signatures" class="section level3">
<h3><span class="header-section-number">1.3.3</span> Immune signatures - biological perspective</h3>
<p>A gene signature is</p>
<blockquote>
<p><em>a single or combined group of genes in with a uniquely characteristic pattern of gene expression that occurs as a result of an altered or unaltered biological process or pathogenic medical condition</em> <span class="citation">(Itadani, Mizuarai, and Kotani <a href="#ref-Itadani2008">2008</a>; Liu et al. <a href="#ref-Liu2008">2008</a>)</span>.</p>
</blockquote>
<p>They can be classified based on their form:</p>
<ul>
<li>metagene</li>
<li>gene list</li>
<li>weighted gene list</li>
</ul>
<p>A term <strong>metagene</strong> or <em>eigen gene</em> describes an aggregated pattern of gene expression. The aggregation can correspond to simple mean of samples or can be obtained though matrix factorisation or source separation techniques, clustering. A metagene usually provides values for all measured genes (all probes) in contrast to a wighted gene list where weights are associated with selected genes.</p>
<p><strong>Gene lists</strong> are simple enumeration of transcripts names or gene identifiers. Application of gene list is often limited to gene enrichment analysis tools or gene selection from the data.</p>
<p>An alternative is a <strong>weighted gene list</strong> or ranked gene list, where genes are ranked according to their importance. Often the ranks is obtained though comparison between two conditions or test/control. They can be also based on absolute gene expression values<span class="citation">(Lyons et al. <a href="#ref-Lyons2017">2017</a>)</span>. One possible problem with this weighted gene list can be platform dependence.</p>
<p>There exist a big choice of databases storing collections of signatures. They contain gene expression and other genomic data such as genotype, DNA methylation, and protein expression data attributed to some condition of reference. A big collection of immune signatures are regrouped by <a href="https://www.immgen.org/">Immunological Genome Project (IGP, ImmGen)</a> <span class="citation">(Heng et al. <a href="#ref-Heng2008">2008</a>)</span>. Gene expression of protein coding genes measure in mice immune cells, ex vivo, in different conditions (drug treatment, perturbations) were regrouped in this ressource. A different ressource <a href="https://sysimm.ifrec.osaka-u.ac.jp/immuno-navigator/">Immuno-navigator</a> <span class="citation">(Vandenbon et al. <a href="#ref-Vandenbon2016">2016</a>)</span> that stores information about human and murine immune genes and co-expression networks. <a href="http://software.broadinstitute.org/gsea/msigdb/collections.jsp">ImmuneSigDB</a> is a collection of gene-sets that describe immunity and inflammation in transcriptomic data <span class="citation">(Godec et al. <a href="#ref-Godec2016">2016</a>)</span> and a part of popular MSigDB ressource used commonly for gene set enrichment analysis (GSEA) <span class="citation">(Subramanian et al. <a href="#ref-Subramanian2005">2005</a>)</span>.</p>
<p>They can also be classified based on their use:</p>
<ul>
<li>prognostic signatures</li>
<li>predictive signatures</li>
<li>diagnostic signatures</li>
<li>specific signatures</li>
</ul>
<p>The <em>prognostic</em> signatures can distinguish between patients with a good or from patients with bad prognosis when deciding to assign a patient to a therapy.</p>
<p>The <em>predictive</em> signatures are able to predict treatment benefit between experimental and/or nontraditional treatment groups vs. control, i.e. in clinical trials <span class="citation">(Michiels, Ternès, and Rotolo <a href="#ref-Michiels2016">2016</a>)</span>.</p>
<p>The <em>diagnostic</em> signature, also called <em>biomarkers</em> can be used for detection of a disease in a patient, like for example in blood tests.</p>
<p>The <em>specific</em> signatures should describe with robustness and reproducibility the same group of cells, or patients, or condition with respect to other considered groups. For instance, in the context of cell-types, among studied cell-types a specific signature will distinguish only one cell type. In the context of cancer subtypes, it will indicate clearly one subtype among others.</p>
<p>Examples of predictive and prognostic gene signatures, used in clinical practice are Oncotype DX, EndoPredict, PAM50, and Breast Cancer Index for breast cancer <span class="citation">(Harris et al. <a href="#ref-Harris2016">2016</a>)</span>.</p>
<p>Studies discussed in this Chapter showed plausible importance of immune-related signals in cancer therapy. However, there is no no immune-related gene signatures used in clinical practice currently. This can be because of the lack of consistency of genes, both within the same tumor type and among different tumors that can be found in the signatures <span class="citation">(Chifman et al. <a href="#ref-Chifman2016">2016</a>)</span>. Difference in gene expression of different cell populations were found even intra- and interlabs. This difference can be due to confounding factors like stress or to contamination <span class="citation">(Heng et al. <a href="#ref-Heng2008">2008</a>)</span>.</p>
<p>In many studies <em>specifc</em> signatures of cell types are used. They seem to be good in discriminating between broad lineages of cell type, such as lymphoid and myeloid. Although thier capacity to describe cell states and cell subtypes is more discutable <span class="citation">(Chifman et al. <a href="#ref-Chifman2016">2016</a>)</span>. Another matter is that cell type signatures are often obtained in model organisms or extracted from different tissue (i.e. blood-derived signatures vs cancer-derived signatures).</p>
<blockquote>
<p><em>the gene expression profiles of tumour-associated immune cells differ considerably from those of blood derived immune cells</em> <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span></p>
</blockquote>
<p>With emergence of single-cell signatures, there are new horizons of gene signatures to be discovered. Especially signatures of rare cell types in solid tissues. Yet, it is up to researches to cross validate single cell signatures with different types of data as scRNA-seq is not free of platform and post-processing bias.</p>
<p>Immune signatures will be also discussed as a part of deconvolution pipeline in the <a href="methods.html#methods">Chapter 2</a> under the section about <em>basis matrix</em> in mathematical terms.</p>
</div>
<div id="cancer_Therapies" class="section level3">
<h3><span class="header-section-number">1.3.4</span> Cancer therapies</h3>
<p>Cancer is a complex disease. Up to date, no uniform and fully effective treatment were proposed, and usually different strategies are tested to kill tumor cells. <strong>Surgery</strong> is one of the oldest methods. The cancer is removed from the patient body. There are different ways, more or less invasive, that it can be performed. It is usually applied for solid tumor contained in a small area. <strong>Radiation Therapy</strong> uses high doses of radiation to eliminate tumor cells and shrink tumor mass. It can be applied externally or internally. <strong>Chemotherapy</strong> uses a drug (or a combination of drugs) that kill cancer cells, usually altering cell proliferation and growth. The drawback of radiotherapy and chemotherapy are substantial side effects. <strong>Hormone therapy </strong> modulate hormone levels in the body in order to inhibit tumor growth in breast and prostate cancers. In leukemia and lymphoma, can be applied <strong>stem cell transplants</strong> that restore blood-forming stem cells destroyed by the very high doses of chemotherapy or radiation therapy that are used to treat certain cancers.</p>
<p>Alternatively, <strong>targeted therapies</strong> represent a more focused strategy that aims to be more efficient and cause fewer side effects than systematic therapies. Two main types of targeted therapies are small-molecule drugs and monoclonal antibodies. Targeted therapies usually aim to stimulate/inhibit a selected molecular function. Particular types of targeted therapies are <strong>Immunotherapies</strong>. Through activation/inhibition of immune regulatory pathways, it stimulates the immune system to destroy malignant cells. A continuation of targeted therapies is <strong>precision medicine approach</strong>. It is based on genetic information to specify patient’s profile and find a suitable treatment. A number of innovative treatments targeting a specific change in tumor ecosystem are being tested presently in precision medicine clinical trials <span class="citation">(Institute <a href="#ref-NCI2018">2017</a>)</span>.</p>
</div>
<div id="recent-progress-in-immuno-therapies" class="section level3">
<h3><span class="header-section-number">1.3.5</span> Recent progress in immuno-therapies</h3>
<p>The immunotherapies, in contrast with other types of cancers therapies discussed in <a href="intro.html#cancer_Therapies">the previous section</a>, aim to trigger or restart the immune system to defend the organism and attack the malignant cells without provoking persisting inflammation state <span class="citation">(Predina et al. <a href="#ref-Predina2013">2013</a>)</span></p>
<p>The idea of stimulating the immune system to fight malignant cell was not born recently. For a long time, a possibility of development of an anti-cancer vaccine has been investigated. Unfortunately, this idea faced two essential limitations 1) lack of knowledge of antigens that should be used in a vaccine to stimulate cytotoxic T cells successfully) the ability of cancer to block the immune response also called <em>immunostat</em>. Despite those impediments works on anti-tumor vaccines do not cease <span class="citation">(Palucka and Banchereau <a href="#ref-Palucka2013">2013</a>)</span>. A very recent promising an in-situ anti-tumor vaccine was proposed by Sagiv-Barfi et al. <span class="citation">(Sagiv-Barfi et al. <a href="#ref-Sagiv-Barfi2018">2018</a>)</span>. The therapy tested in mice would be based on local injections of the combination of “unmethylated CG-enriched oligodeoxynucleotide (Cp-G) - a Toll-like receptor 9 (TLR9) ligand and anti-OX40 antibody. Low doses of CpG injected into a tumor induce the expression of OX40 on CD4+ T cells in the microenvironment in mouse or human tumors. An agonistic anti-OX40 antibody can then trigger a T cell immune response, which is specific to the antigens of the injected tumor”. Sagiv-Barfi et al. claim this therapy could be applied to all tumor types, as long as they are leucocyte-infiltrated. As a local therapy, in situ vaccination should have fewer side-effects than systematic administration. It is now undergoing clinical trials to test its efficiency in human patients.</p>
<p>Another idea involving using the immune system as a weapon to fight cancer would be the use of genetically modified patient’s T-cells, carrying <em>CARs</em> (chimeric antigen receptors) <span class="citation">(Jackson, Rafiq, and Brentjens <a href="#ref-Jackson2016">2016</a>)</span>. After an extended period of small unsuccessful trials, recently in 2017, two CAR T-cell therapies were accepted, one to “treat adults with certain type of large B-cell lymphoma” <span class="citation">(Health and Services <a href="#ref-FDACARTadult">2017</a><a href="#ref-FDACARTadult">b</a>)</span>, other to treat “children with acute lymphoblastic leukemia (ALL)” <span class="citation">(Health and Services <a href="#ref-FDACARTALL">2017</a><a href="#ref-FDACARTALL">a</a>)</span>, which are, at the same time, the first two gene therapies accepted by FDA.</p>
<p>However, the two most promising immuno-related strategies with proven clinical efficiency are based on blocking so-called immune checkpoint inhibitors: cytotoxic T-lymphocyte protein 4 (CTLA4) and programmed cell death protein 1 (PD-1). The anti-CLTA4 antibodies blocks repressive action of CLTA4 on T-cells and they become therefore activated. It was shown efficient in melanoma patients and accepted by FDA in 2015 as adjuvant therapy for stage III metastatic melanoma patients <span class="citation">(Health and Services <a href="#ref-FDACTLA4">2015</a>)</span>. PD-1 is a cell surface receptor of T cells, that binds to PD-L1/PD-L2. After binding, an immunosuppressive pathway is activated and T cells activity is dampened. An action of an anti-PD-L1 antibody is to prevent this immune exhaustion <span class="citation">(Chen and Mellman <a href="#ref-Chen2017">2017</a>)</span>. A stepping stone for anti-PD-L1 therapies was approval of Tecentriq (atezolizumab) for Bladder cancer <span class="citation">(Health and Services <a href="#ref-FDAPDL1Bladder">2016</a><a href="#ref-FDAPDL1Bladder">a</a>)</span> and anti-PD1 Keytruda (pembrolizumab) initially accepted for NSCLC and further extended to head and neck cancer, Hodgkin’s lymphoma, gastric cancer and microsatellite instability-high cancer <span class="citation">(Health and Services <a href="#ref-FDAPDL1NSCLC">2016</a><a href="#ref-FDAPDL1NSCLC">b</a>)</span>. Since that breakthrough, other anti-PD-L1 or anti-PD1 antibodies were accepted or entered advanced stages of clinical trials <span class="citation">(Wolchok <a href="#ref-Wolchok2015">2015</a>)</span>. A short history of immunotherapy FDA-accepted treatments can be found in Fig. <a href="intro.html#fig:timeline-immunotherapies">1.8</a></p>
<div class="figure" style="text-align: center"><span id="fig:timeline-immunotherapies"></span>
<img src="figures-ext/02-timeline-immunotherapies.png" alt="This timeline describes short history of FDA approval of checkpoint blocking immunotherapies up to 2017. Reprinted by permission from Springer Nature (Taube et al. 2017) Macmillan Publishers Limited, part of Springer Nature. All Rights Reserved." width="100%" />
<p class="caption">
Figure 1.8: <strong>This timeline describes short history of FDA approval of checkpoint blocking immunotherapies up to 2017.</strong> Reprinted by permission from Springer Nature <span class="citation">(Taube et al. <a href="#ref-Taube2017a">2017</a>)</span> Macmillan Publishers Limited, part of Springer Nature. All Rights Reserved.
</p>
</div>

<p>The main drawback of immunotherapies is heterogeneity of response rate, which can vary, i.e., from 10–40% in case of PD-L1blocking <span class="citation">(Zou, Wolchok, and Chen <a href="#ref-Zou2016">2016</a>)</span>, suggesting that some patients can have more chances than others to respond to immune therapy. So far, it has been shown that anti PD-L1 therapies work more effectively in T cell infiltrated tumors with the exclusion of Tregs because of lack of difference in expression of FOXP3 in responding and the non-responding group of patients <span class="citation">(Herbst et al. <a href="#ref-Herbst2014">2014</a>)</span>. Also, some light has been shade by <span class="citation">Rizvi et al. (<a href="#ref-Rizvi2015">2015</a>)</span> who connected mutational rate of cancer cells to the chances of response to immunotherapy.</p>
<p>Despite those findings, the precise qualifications of patients that should be sensitive to immunotherapy are not defined <span class="citation">(Pitt et al. <a href="#ref-Pitt2016">2016</a>)</span>. As most patients do not answer to immunotherapies, it stimulates researches to look for better biomarkers and patient stratifications, and pharmaceutical industries to discover new immune checkpoints based therapies.</p>
</div>
</div>
<div id="summary-of-the-chapter" class="section level2">
<h2><span class="header-section-number">1.4</span> Summary of the chapter</h2>
<p>Cancer remains a critical health problem of our era that touches many people. Tumor cells are interacting with their microenvironment (called Tumor Microenvironment (TME)) including normal cell, stromal cells and a variety of immune cells. These cells can have a role in disease progression and response to treatment. A modern approach to modulate TME was proposed through an application of immune therapies.</p>
<p>A new way to classify cancers based on their TME is called immunophenotyping. Widely used TCGA data contains many different data types, but not all of possible data types. Before entering a clinical practice, the immunophenotyping approaches will need to face important challenges. Would addition of a new data type (i.e., FACS) change the patients classification? Can the similar results be found in non-American patient cohorts? What if different technologies are used, if data are not normalized uniformly, would it change the conclusions? What if not all data types available in TCGA are not produced for other patients? How these classification can be reproduced for smaller cohorts? Can these complex classification schemes be reduced to a few easy measurable indices? It is important to acknowledge the authors for their remarkable work. However it is also crucial to remember we are biased by the piece of the truth (type of data) we use, that can be on the top biased with technical and experimental design.</p>
<p>To produce a very detailed system-level view of the TME with traditional experimental techniques an uncountable amount of work and resources would be necessary. Using omic techniques system approach is possible to reduce the time and resources. However, to embrace fully the data complexity, computational tools are indispensable. From the data generation to the analysis, different statistical and mathematical challenges need to be faced before arriving at valid biological results and interpretations.</p>
<p>As I will present in the next chapter, in order to solve the problem of extraction of cell-type heterogeneity from cancer bulk omic data, a number of approaches were developed.</p>

<!-- ```{r setup, include=FALSE} -->
<!--  is_pdf_output = function() { -->
<!--   knitr::opts_knit$get("rmarkdown.pandoc.to") =="latex" -->
<!--  } -->
<!-- ``` -->
<!-- ```{r setup2, include=FALSE} -->
<!--  is_html_output = function() { -->
<!--   knitr::opts_knit$get("rmarkdown.pandoc.to") =="html4" -->
<!--  } -->
<!-- size.fig = "100%" -->
<!-- ``` -->
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-GLOBOCAN">
<p>Research in Cancer, International Agency for. 2018. “GLOBOCAN - Facts Sheets by Cancer.” <a href="http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx" class="uri">http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx</a>.</p>
</div>
<div id="ref-Sudhakar2009">
<p>Sudhakar, Akulapalli. 2009. “History of Cancer, Ancient and Modern Treatment Methods.” <em>J. Cancer Sci. Ther.</em> 1 (2). NIH Public Access: 1–4. doi:<a href="https://doi.org/10.4172/1948-5956.100000e2">10.4172/1948-5956.100000e2</a>.</p>
</div>
<div id="ref-NPR2010">
<p>NPR. 2010b. “Science Diction: The Origin of the Word ’Cancer’ : NPR.” <a href="https://www.npr.org/templates/story/story.php?storyId=130754101" class="uri">https://www.npr.org/templates/story/story.php?storyId=130754101</a>.</p>
</div>
<div id="ref-VirchowRudolf1847">
<p>Virchow Rudolf. 1847. “Archiv für pathologische Anatomie und Physiologie und für klinische Medizin.” Berlin ; New York: Springer. <a href="file://catalog.hathitrust.org/Record/000493993 http://hdl.handle.net/2027/njp.32101076036878 (Bd.64 (1875)) http://hdl.handle.net/2027/uc1.31175008645106 (v.170 {\&amp;}index v.161-170) http://hdl.handle.net/2027/hvd.32044093331494 (Bd.111 (1888)) http://hdl.han" class="uri">file://catalog.hathitrust.org/Record/000493993 http://hdl.handle.net/2027/njp.32101076036878 (Bd.64 (1875)) http://hdl.handle.net/2027/uc1.31175008645106 (v.170 {\&amp;}index v.161-170) http://hdl.handle.net/2027/hvd.32044093331494 (Bd.111 (1888)) http://hdl.han</a>.</p>
</div>
<div id="ref-Paget1889">
<p>Paget, Stephen. 1889. “THE DISTRIBUTION OF SECONDARY GROWTHS IN CANCER OF THE BREAST.” <em>Lancet</em> 133 (3421). Elsevier: 571–73. doi:<a href="https://doi.org/10.1016/S0140-6736(00)49915-0">10.1016/S0140-6736(00)49915-0</a>.</p>
</div>
<div id="ref-Ehrlich1909">
<p>Ehrlich, P. 1909. “Über den jetzigen Stand der Chemotherapie.” <em>Berichte Der Dtsch. Chem. Gesellschaft</em> 42 (1). Wiley-Blackwell: 17–47. doi:<a href="https://doi.org/10.1002/cber.19090420105">10.1002/cber.19090420105</a>.</p>
</div>
<div id="ref-Greenblatt1968">
<p>Greenblatt, M, and P Shubi. 1968. “Tumor angiogenesis: transfilter diffusion studies in the hamster by the transparent chamber technique.” <em>J. Natl. Cancer Inst.</em> 41 (1): 111–24. <a href="http://www.ncbi.nlm.nih.gov/pubmed/5662020" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/5662020</a>.</p>
</div>
<div id="ref-Folkman1971">
<p>Folkman, J, E Merler, C Abernathy, and G Williams. 1971. “Isolation of a tumor factor responsible for angiogenesis.” <em>J. Exp. Med.</em> 133 (2). Rockefeller University Press: 275–88. doi:<a href="https://doi.org/10.1084/JEM.133.2.275">10.1084/JEM.133.2.275</a>.</p>
</div>
<div id="ref-Feinberg1983">
<p>Feinberg, Andrew P., and Bert Vogelstein. 1983. “Hypomethylation distinguishes genes of some human cancers from their normal counterparts.” <em>Nature</em> 301 (5895). Nature Publishing Group: 89–92. doi:<a href="https://doi.org/10.1038/301089a0">10.1038/301089a0</a>.</p>
</div>
<div id="ref-Greger1989">
<p>Greger, Valerie, Eberhard Passarge, Wolfgang H�pping, Elmar Messmer, and Bernhard Horsthemke. 1989. “Epigenetic changes may contribute to the formation and spontaneous regression of retinoblastoma.” <em>Hum. Genet.</em> 83 (2). Springer-Verlag: 155–58. doi:<a href="https://doi.org/10.1007/BF00286709">10.1007/BF00286709</a>.</p>
</div>
<div id="ref-VanPel1982">
<p>Van Pel, A, and T Boon. 1982. “Protection against a nonimmunogenic mouse leukemia by an immunogenic variant obtained by mutagenesis.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 79 (15): 4718–22. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6981814 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC346748" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/6981814 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC346748</a>.</p>
</div>
<div id="ref-Leung1989">
<p>Leung, D., G Cachianes, W. Kuang, D. Goeddel, and N Ferrara. 1989. “Vascular endothelial growth factor is a secreted angiogenic mitogen.” <em>Science (80-. ).</em> 246 (4935): 1306–9. doi:<a href="https://doi.org/10.1126/science.2479986">10.1126/science.2479986</a>.</p>
</div>
<div id="ref-Golub1999">
<p>Golub, T R, D K Slonim, P Tamayo, C Huard, M Gaasenbeek, J P Mesirov, H Coller, et al. 1999. “Molecular classification of cancer: class discovery and class prediction by gene expression monitoring.” <em>Science</em> 286 (5439). American Association for the Advancement of Science: 531–7. doi:<a href="https://doi.org/10.1126/SCIENCE.286.5439.531">10.1126/SCIENCE.286.5439.531</a>.</p>
</div>
<div id="ref-Corlan2004">
<p>Corlan, Alexandru Dan. 2004. “Medline Trend: Automated Yearly Statistics of Pubmed Results for Any Query.” <a href="http://dan.corlan.net/medline-trend.html" class="uri">http://dan.corlan.net/medline-trend.html</a>.</p>
</div>
<div id="ref-Balkwill2001">
<p>Balkwill, Fran, and Alberto Mantovani. 2001. “Inflammation and cancer: Back to Virchow?” doi:<a href="https://doi.org/10.1016/S0140-6736(00)04046-0">10.1016/S0140-6736(00)04046-0</a>.</p>
</div>
<div id="ref-Talmadge2013">
<p>Talmadge, J E, and D I Gabrilovich. 2013. “History of myeloid-derived suppressor cells.” <em>Nat Rev Cancer</em> 13 (10): 739–52. doi:<a href="https://doi.org/10.1038/nrc3581">10.1038/nrc3581</a>.</p>
</div>
<div id="ref-Gabrilovich2012">
<p>Gabrilovich, Dmitry I., Suzanne Ostrand-Rosenberg, and Vincenzo Bronte. 2012. “Coordinated regulation of myeloid cells by tumours.” doi:<a href="https://doi.org/10.1038/nri3175">10.1038/nri3175</a>.</p>
</div>
<div id="ref-Taube2017a">
<p>Taube, Janis M, Jérôme Galon, Lynette M Sholl, Scott J Rodig, Tricia R Cottrell, Nicolas A Giraldo, Alexander S Baras, et al. 2017. “Implications of the tumor immune microenvironment for staging and therapeutics.” <em>Mod. Pathol.</em> doi:<a href="https://doi.org/10.1038/modpathol.2017.156">10.1038/modpathol.2017.156</a>.</p>
</div>
<div id="ref-Dumont2013">
<p>Dumont, Nancy, Bob Liu, Rosa Anna DeFilippis, Hang Chang, Joseph T. Rabban, Anthony N. Karnezis, Judy A. Tjoe, James Marx, Bahram Parvin, and Thea D. Tlsty. 2013. “Breast Fibroblasts Modulate Early Dissemination, Tumorigenesis, and Metastasis through Alteration of Extracellular Matrix Characteristics.” <em>Neoplasia</em> 15 (3): 249–IN7. doi:<a href="https://doi.org/10.1593/neo.121950">10.1593/neo.121950</a>.</p>
</div>
<div id="ref-Quail2013">
<p>Quail, D F, and J A Joyce. 2013. “Microenvironmental regulation of tumor progression and metastasis.” <em>Nat Med</em> 19 (11): 1423–37. doi:<a href="https://doi.org/10.1038/nm.3394">10.1038/nm.3394</a>.</p>
</div>
<div id="ref-Hanahan2012">
<p>Hanahan, Douglas, and Lisa M. Coussens. 2012. “Accessories to the Crime: Functions of Cells Recruited to the Tumor Microenvironment.” <em>Cancer Cell</em> 21 (3): 309–22. doi:<a href="https://doi.org/10.1016/j.ccr.2012.02.022">10.1016/j.ccr.2012.02.022</a>.</p>
</div>
<div id="ref-Hanahan2000">
<p>Hanahan, D, and R A Weinberg. 2000. “The hallmarks of cancer.” <em>Cell</em> 100 (1). Elsevier: 57–70. doi:<a href="https://doi.org/10.1016/S0092-8674(00)81683-9">10.1016/S0092-8674(00)81683-9</a>.</p>
</div>
<div id="ref-Sebeok1976">
<p>Sebeok, Thomas A. 1976. “DISCUSSION.” <em>Ann. N. Y. Acad. Sci.</em> 280 (1 Origins and E). Blackwell Publishing Ltd: 481. doi:<a href="https://doi.org/10.1111/j.1749-6632.1976.tb25511.x">10.1111/j.1749-6632.1976.tb25511.x</a>.</p>
</div>
<div id="ref-Burnet1970">
<p>Burnet, F M. 1970. “The concept of immunological surveillance.” <em>Prog. Exp. Tumor Res.</em> 13: 1–27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4921480" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/4921480</a>.</p>
</div>
<div id="ref-Dunn2002">
<p>Dunn, Gavin P., Allen T. Bruce, Hiroaki Ikeda, Lloyd J. Old, and Robert D. Schreiber. 2002. “Cancer immunoediting: from immunosurveillance to tumor escape.” <em>Nat. Immunol.</em> 3 (11). Nature Publishing Group: 991–98. doi:<a href="https://doi.org/10.1038/ni1102-991">10.1038/ni1102-991</a>.</p>
</div>
<div id="ref-Vesely2011">
<p>Vesely, Matthew D., Michael H. Kershaw, Robert D. Schreiber, and Mark J. Smyth. 2011. “Natural Innate and Adaptive Immunity to Cancer.” <em>Annu. Rev. Immunol.</em> 29 (1): 235–71. doi:<a href="https://doi.org/10.1146/annurev-immunol-031210-101324">10.1146/annurev-immunol-031210-101324</a>.</p>
</div>
<div id="ref-Marquez-Medina2012">
<p>Marquez-Medina, Diego, Joel Salla-Fortuny, and Antonieta Salud-Salvia. 2012. “Role of gamma-delta T-cells in cancer. Another opening door to immunotherapy.” <em>Clin. Transl. Oncol.</em> 14 (12): 891–95. doi:<a href="https://doi.org/10.1007/s12094-012-0935-7">10.1007/s12094-012-0935-7</a>.</p>
</div>
<div id="ref-Kornstein1983">
<p>Kornstein, M J, J S Brooks, and D E Elder. 1983. “Immunoperoxidase localization of lymphocyte subsets in the host response to melanoma and nevi.” <em>Cancer Res.</em> 43 (6): 2749–53. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6342758" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/6342758</a>.</p>
</div>
<div id="ref-Baxevanis1994">
<p>Baxevanis, C N, G V Dedoussis, N G Papadopoulos, I Missitzis, G P Stathopoulos, and M Papamichail. 1994. “Tumor specific cytolysis by tumor infiltrating lymphocytes in breast cancer.” <em>Cancer</em> 74 (4): 1275–82. <a href="http://www.ncbi.nlm.nih.gov/pubmed/7914469" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/7914469</a>.</p>
</div>
<div id="ref-Naito1998">
<p>Naito, Y, K Saito, K Shiiba, A Ohuchi, K Saigenji, H Nagura, and H Ohtani. 1998. “CD8+ T cells infiltrated within cancer cell nests as a prognostic factor in human colorectal cancer.” <em>Cancer Res.</em> 58 (16): 3491–4. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9721846" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/9721846</a>.</p>
</div>
<div id="ref-Pages2005">
<p>Pagès, Franck, Anne Berger, Matthieu Camus, Fatima Sanchez-Cabo, Anne Costes, Robert Molidor, Bernhard Mlecnik, et al. 2005. “Effector Memory T Cells, Early Metastasis, and Survival in Colorectal Cancer.” <em>N. Engl. J. Med.</em> 353 (25): 2654–66. doi:<a href="https://doi.org/10.1056/NEJMoa051424">10.1056/NEJMoa051424</a>.</p>
</div>
<div id="ref-Aris2012">
<p>Aris, Mariana, María Marcela Barrio, and José Mordoh. 2012. “Lessons from Cancer Immunoediting in Cutaneous Melanoma.” <em>Clin. Dev. Immunol.</em> 2012: 1–14. doi:<a href="https://doi.org/10.1155/2012/192719">10.1155/2012/192719</a>.</p>
</div>
<div id="ref-Bhatia2011">
<p>Bhatia, Alka, and Yashwant Kumar. 2011. “Cancer-immune equilibrium: questions unanswered.” <em>Cancer Microenviron.</em> 4 (2). Springer: 209–17. doi:<a href="https://doi.org/10.1007/s12307-011-0065-8">10.1007/s12307-011-0065-8</a>.</p>
</div>
<div id="ref-Teng2008">
<p>Teng, Michele W. L., Jeremy B. Swann, Catherine M. Koebel, Robert D. Schreiber, and Mark J. Smyth. 2008. “Immune-mediated dormancy: an equilibrium with cancer.” <em>J. Leukoc. Biol.</em> 84 (4): 988–93. doi:<a href="https://doi.org/10.1189/jlb.1107774">10.1189/jlb.1107774</a>.</p>
</div>
<div id="ref-Frey2010">
<p>Frey, Daniel M., Raoul A. Droeser, Carsten T. Viehl, Inti Zlobec, Alessandro Lugli, Urs Zingg, Daniel Oertli, Christoph Kettelhack, Luigi Terracciano, and Luigi Tornillo. 2010. “High frequency of tumor-infiltrating FOXP3+ regulatory T cells predicts improved survival in mismatch repair-proficient colorectal cancer patients.” <em>Int. J. Cancer</em> 126 (11): 2635–43. doi:<a href="https://doi.org/10.1002/ijc.24989">10.1002/ijc.24989</a>.</p>
</div>
<div id="ref-Qian2010">
<p>Qian, Bin Z., and Jeffrey W. Pollard. 2010. “Macrophage Diversity Enhances Tumor Progression and Metastasis.” doi:<a href="https://doi.org/10.1016/j.cell.2010.03.014">10.1016/j.cell.2010.03.014</a>.</p>
</div>
<div id="ref-Stewart1995">
<p>Stewart, T., S. CJ Tsai, H. Grayson, R. Henderson, and G. Opelz. 1995. “Incidence of de-novo breast cancer in women chronically immunosuppressed after organ transplantation.” <em>Lancet</em> 346 (8978): 796–98. doi:<a href="https://doi.org/10.1016/S0140-6736(95)91618-0">10.1016/S0140-6736(95)91618-0</a>.</p>
</div>
<div id="ref-Boal2002">
<p>Boal, David H. 2002. <em>Mechanics of the cell</em>. Cambridge University Press.</p>
</div>
<div id="ref-Slater2013">
<p>Slater, Matthew H. 2013. “Cell Types as Natural Kinds.” <em>Biol. Theory</em> 7 (2). Springer Netherlands: 170–79. doi:<a href="https://doi.org/10.1007/s13752-012-0084-9">10.1007/s13752-012-0084-9</a>.</p>
</div>
<div id="ref-Satija2014">
<p>Satija, Rahul, and Alex K. Shalek. 2014. “Heterogeneity in immune responses: from populations to single cells.” <em>Trends Immunol.</em> 35 (5): 219–29. doi:<a href="https://doi.org/10.1016/j.it.2014.03.004">10.1016/j.it.2014.03.004</a>.</p>
</div>
<div id="ref-EdiorialCellSystems2017">
<p>Ediorial Cell Systems. 2017. “What Is Your Conceptual Definition of ‘Cell Type’ in the Context of a Mature Organism?” <em>Cell Syst.</em> 4 (3). Elsevier: 255–59. doi:<a href="https://doi.org/10.1016/j.cels.2017.03.006">10.1016/j.cels.2017.03.006</a>.</p>
</div>
<div id="ref-Ponting2017">
<p>Ponting, Chris P. 2017. “Big knowledge from big data in functional genomics.” <em>Emerg. Top. Life Sci.</em> 1 (3). Portland Press Journals portal: 245–48. doi:<a href="https://doi.org/10.1042/ETLS20170129">10.1042/ETLS20170129</a>.</p>
</div>
<div id="ref-Papalexi2017">
<p>Papalexi, Efthymia, and Rahul Satija. 2017. “Single-cell RNA sequencing to explore immune cell heterogeneity.” <em>Nat. Rev. Immunol.</em> 18 (1): 35–45. doi:<a href="https://doi.org/10.1038/nri.2017.76">10.1038/nri.2017.76</a>.</p>
</div>
<div id="ref-Lyons2017">
<p>Lyons, Yasmin A, Sherry Y Wu, Willem W Overwijk, Keith A Baggerly, and Anil K Sood. 2017. “Immune cell profiling in cancer: molecular approaches to cell- specific identification.” <em>Npj Precis. Oncol.</em> 1. doi:<a href="https://doi.org/10.1038/s41698-017-0031-0">10.1038/s41698-017-0031-0</a>.</p>
</div>
<div id="ref-RamosVara2010">
<p>NPR. 2010a. “Science Diction: The Origin of the Word ’Cancer’.” <a href="https://www.npr.org/templates/story/story.php?storyId=130754101" class="uri">https://www.npr.org/templates/story/story.php?storyId=130754101</a>.</p>
</div>
<div id="ref-Giesen2014">
<p>Giesen, Charlotte, Hao A O Wang, Denis Schapiro, Nevena Zivanovic, Andrea Jacobs, Bodo Hattendorf, Peter J Schüffler, et al. 2014. “Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry.” <em>Nat. Methods</em> 11 (4). Nature Publishing Group: 417–22. doi:<a href="https://doi.org/10.1038/nmeth.2869">10.1038/nmeth.2869</a>.</p>
</div>
<div id="ref-Schelker2017">
<p>Schelker, Max, Sonia Feau, Jinyan Du, Nav Ranu, Edda Klipp, Birgit Schoeberl, Gavin MacBeath, et al. 2017. “Estimation of immune cell content in tumour tissue using single-cell RNA-seq data.” <em>Nat. Commun.</em> 8 (1). Nature Publishing Group: 2032. doi:<a href="https://doi.org/10.1038/s41467-017-02289-3">10.1038/s41467-017-02289-3</a>.</p>
</div>
<div id="ref-Wilczynski2009">
<p>Wilczynski, Sharon P. 2009. “Molecular Biology.” In <em>Mod. Surg. Pathol.</em>, 85–120. Elsevier. doi:<a href="https://doi.org/10.1016/B978-1-4160-3966-2.00006-0">10.1016/B978-1-4160-3966-2.00006-0</a>.</p>
</div>
<div id="ref-Velculescu1997">
<p>Velculescu, V E, L Zhang, W Zhou, J Vogelstein, M A Basrai, D E Bassett, P Hieter, B Vogelstein, and K W Kinzler. 1997. “Characterization of the yeast transcriptome.” <em>Cell</em> 88 (2): 243–51. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9008165" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/9008165</a>.</p>
</div>
<div id="ref-Cieslik2017">
<p>Cieślik, Marcin, and Arul M. Chinnaiyan. 2017. “Cancer transcriptome profiling at the juncture of clinical translation.” <em>Nat. Rev. Genet.</em> 19 (2). Nature Publishing Group: 93–109. doi:<a href="https://doi.org/10.1038/nrg.2017.96">10.1038/nrg.2017.96</a>.</p>
</div>
<div id="ref-Chen2018b">
<p>Chen, Xi, Sarah A. Teichmann, and Kerstin B. Meyer. 2018. “From Tissues to Cell Types and Back: Single-Cell Gene Expression Analysis of Tissue Architecture.” <em>Annu. Rev. Biomed. Data Sci.</em> 1 (1). Annual Reviews: 29–51. doi:<a href="https://doi.org/10.1146/annurev-biodatasci-080917-013452">10.1146/annurev-biodatasci-080917-013452</a>.</p>
</div>
<div id="ref-Mody2015">
<p>Mody, Rajen J., Yi-Mi Wu, Robert J. Lonigro, Xuhong Cao, Sameek Roychowdhury, Pankaj Vats, Kevin M. Frank, et al. 2015. “Integrative Clinical Sequencing in the Management of Refractory or Relapsed Cancer in Youth.” <em>JAMA</em> 314 (9). American Medical Association: 913. doi:<a href="https://doi.org/10.1001/jama.2015.10080">10.1001/jama.2015.10080</a>.</p>
</div>
<div id="ref-Oberg2016">
<p>Oberg, Jennifer A., Julia L. Glade Bender, Maria Luisa Sulis, Danielle Pendrick, Anthony N. Sireci, Susan J. Hsiao, Andrew T. Turk, et al. 2016. “Implementation of next generation sequencing into pediatric hematology-oncology practice: moving beyond actionable alterations.” <em>Genome Med.</em> 8 (1). BioMed Central: 133. doi:<a href="https://doi.org/10.1186/s13073-016-0389-6">10.1186/s13073-016-0389-6</a>.</p>
</div>
<div id="ref-Robinson2017">
<p>Robinson, Dan R., Yi-Mi Wu, Robert J. Lonigro, Pankaj Vats, Erin Cobain, Jessica Everett, Xuhong Cao, et al. 2017. “Integrative clinical genomics of metastatic cancer.” <em>Nature</em> 548 (7667). Nature Publishing Group: 297–303. doi:<a href="https://doi.org/10.1038/nature23306">10.1038/nature23306</a>.</p>
</div>
<div id="ref-Chen2016">
<p>Chen, Pei Ling, Whijae Roh, Alexandre Reuben, Zachary A. Cooper, Christine N. Spencer, Peter A. Prieto, John P. Miller, et al. 2016. “Analysis of immune signatures in longitudinal tumor samples yields insight into biomarkers of response and mechanisms of resistance to immune checkpoint blockade.” <em>Cancer Discov.</em> 6 (8): 827–37. doi:<a href="https://doi.org/10.1158/2159-8290.CD-15-1545">10.1158/2159-8290.CD-15-1545</a>.</p>
</div>
<div id="ref-Perkel2017">
<p>Perkel, Jeffrey M. 2017. “Single-cell sequencing made simple.” <em>Nature</em> 547 (7661): 125–26. doi:<a href="https://doi.org/10.1038/547125a">10.1038/547125a</a>.</p>
</div>
<div id="ref-Davis2016">
<p>Davis, Sean. 2016. “Github: Awesome-Single-Cell.” <a href="https://github.com/seandavi/awesome-single-cell" class="uri">https://github.com/seandavi/awesome-single-cell</a>.</p>
</div>
<div id="ref-Cedar2018">
<p>Core, Cedar-Sinals Genomics. 2016. “Single Cell Genomics.” <a href="https://www.cedars-sinai.edu/Research/Research-Cores/Genomics-Core/Documents/website_pricing_jan_2018_final.pdf" class="uri">https://www.cedars-sinai.edu/Research/Research-Cores/Genomics-Core/Documents/website_pricing_jan_2018_final.pdf</a>.</p>
</div>
<div id="ref-Tirosh2016">
<p>Tirosh, Itay, Benjamin Izar, Sanjay M. Prakadan, Marc H. Wadsworth, Daniel Treacy, John J. Trombetta, Asaf Rotem, et al. 2016. “Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq.” <em>Science (80-. ).</em> 352 (6282): 189–96. doi:<a href="https://doi.org/10.1126/science.aad0501">10.1126/science.aad0501</a>.</p>
</div>
<div id="ref-Bernstein2007">
<p>Bernstein, Bradley E, Alexander Meissner, and Eric S Lander. 2007. “The mammalian epigenome.” <em>Cell</em> 128 (4). Elsevier: 669–81. doi:<a href="https://doi.org/10.1016/j.cell.2007.01.033">10.1016/j.cell.2007.01.033</a>.</p>
</div>
<div id="ref-Jeschke2017">
<p>Jeschke, Jana, Martin Bizet, Christine Desmedt, Emilie Calonne, Sarah Dedeurwaerder, Soizic Garaud, Alexander Koch, et al. 2017. “DNA methylation–based immune response signature improves patient diagnosis in multiple cancers.” <em>J. Clin. Invest.</em> 127 (8): 3090–3102. doi:<a href="https://doi.org/10.1172/JCI91095">10.1172/JCI91095</a>.</p>
</div>
<div id="ref-McCarroll2007">
<p>McCarroll, Steven A, and David M Altshuler. 2007. “Copy-number variation and association studies of human disease.” <em>Nat. Genet.</em> 39 (7s): S37–S42. doi:<a href="https://doi.org/10.1038/ng2080">10.1038/ng2080</a>.</p>
</div>
<div id="ref-Zarrei2015">
<p>Zarrei, Mehdi, Jeffrey R. MacDonald, Daniele Merico, and Stephen W. Scherer. 2015. “A copy number variation map of the human genome.” <em>Nat. Rev. Genet.</em> 16 (3): 172–83. doi:<a href="https://doi.org/10.1038/nrg3871">10.1038/nrg3871</a>.</p>
</div>
<div id="ref-Yamamoto2016">
<p>Yamamoto, Toshiyuki, Keiko Shimojima, Yumiko Ondo, Katsumi Imai, Pin Fee Chong, Ryutaro Kira, Mitsuhiro Amemiya, Akira Saito, and Nobuhiko Okamoto. 2016. “Challenges in detecting genomic copy number aberrations using next-generation sequencing data and the eXome Hidden Markov Model: a clinical exome-first diagnostic approach.” <em>Hum. Genome Var.</em> 3 (1). Nature Publishing Group: 16025. doi:<a href="https://doi.org/10.1038/hgv.2016.25">10.1038/hgv.2016.25</a>.</p>
</div>
<div id="ref-Stahl2016">
<p>Ståhl, Patrik L, Fredrik Salmén, Sanja Vickovic, Anna Lundmark, José Fernández Navarro, Jens Magnusson, Stefania Giacomello, et al. 2016. “Visualization and analysis of gene expression in tissue sections by spatial transcriptomics.” <em>Science</em> 353 (6294). American Association for the Advancement of Science: 78–82. doi:<a href="https://doi.org/10.1126/science.aaf2403">10.1126/science.aaf2403</a>.</p>
</div>
<div id="ref-Svensson2018">
<p>Svensson, Valentine, Sarah A Teichmann, and Oliver Stegle. 2018. “SpatialDE: identification of spatially variable genes.” <em>Nat. Methods</em> 15 (5). Nature Publishing Group: 343–46. doi:<a href="https://doi.org/10.1038/nmeth.4636">10.1038/nmeth.4636</a>.</p>
</div>
<div id="ref-Moncada2018">
<p>Moncada, Reuben, Florian Wagner, Marta Chiodin, Joseph C. Devlin, Maayan Baron, Cristina H. Hajdu, Diane Simeone, and Itai Yanai. 2018. “Building a tumor atlas: integrating single-cell RNA-Seq data with spatial transcriptomics in pancreatic ductal adenocarcinoma.” <em>bioRxiv</em>, March. Cold Spring Harbor Laboratory, 254375. doi:<a href="https://doi.org/10.1101/254375">10.1101/254375</a>.</p>
</div>
<div id="ref-Galon2014">
<p>Galon, Jérôme, Bernhard Mlecnik, Gabriela Bindea, Helen K Angell, Anne Berger, Christine Lagorce, Alessandro Lugli, et al. 2014. “Towards the introduction of the ‘Immunoscore’ in the classification of malignant tumours.” <em>J. Pathol.</em> 232 (2): 199–209. doi:<a href="https://doi.org/10.1002/path.4287">10.1002/path.4287</a>.</p>
</div>
<div id="ref-Parker2009">
<p>Parker, Joel S, Michael Mullins, Maggie C U Cheang, Samuel Leung, David Voduc, Tammi Vickery, Sherri Davies, et al. 2009. “Supervised risk predictor of breast cancer based on intrinsic subtypes.” <em>J. Clin. Oncol.</em> 27 (8). American Society of Clinical Oncology: 1160–7. doi:<a href="https://doi.org/10.1200/JCO.2008.18.1370">10.1200/JCO.2008.18.1370</a>.</p>
</div>
<div id="ref-Bindea2013">
<p>Bindea, Gabriela, Bernhard Mlecnik, Marie Tosolini, Amos Kirilovsky, Maximilian Waldner, Anna C. Obenauf, Helen Angell, et al. 2013. “Spatiotemporal Dynamics of Intratumoral Immune Cells Reveal the Immune Landscape in Human Cancer.” <em>Immunity</em> 39 (4): 782–95. doi:<a href="https://doi.org/10.1016/j.immuni.2013.10.003">10.1016/j.immuni.2013.10.003</a>.</p>
</div>
<div id="ref-Berghoff2018">
<p>Berghoff, Anna, Jakob Kather, and Dirk Jäger. 2018. “Genetics and Immunology: Tumor-Specific Genetic Alterations as a Target for Immune Modulating Therapies.” In <em>Oncoimmunology : A Practical Guide for Cancer Immunotherapy</em>, edited by Laurence Zitvogel and Guido Kroemer, 231–46.</p>
</div>
<div id="ref-Chen2017">
<p>Chen, Daniel S., and Ira Mellman. 2017. “Elements of cancer immunity and the cancer-immune set point.” doi:<a href="https://doi.org/10.1038/nature21349">10.1038/nature21349</a>.</p>
</div>
<div id="ref-Herbst2014">
<p>Herbst, Roy S., Jean-Charles Soria, Marcin Kowanetz, Gregg D. Fine, Omid Hamid, Michael S. Gordon, Jeffery A. Sosman, et al. 2014. “Predictive correlates of response to the anti-PD-L1 antibody MPDL3280A in cancer patients.” <em>Nature</em> 515 (7528): 563–67. doi:<a href="https://doi.org/10.1038/nature14011">10.1038/nature14011</a>.</p>
</div>
<div id="ref-Becht2016">
<p>Becht, Etienne, Aurélien de Reyniès, Nicolas A Giraldo, Camilla Pilati, Bénédicte Buttard, Laetitia Lacroix, Janick Selves, Catherine Sautès-Fridman, Pierre Laurent-Puig, and Wolf Herman Fridman. 2016. “Immune and Stromal Classification of Colorectal Cancer Is Associated with Molecular Subtypes and Relevant for Precision Immunotherapy.” <em>Clin. Cancer Res.</em> 22 (16). American Association for Cancer Research: 4057–66. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-15-2879">10.1158/1078-0432.CCR-15-2879</a>.</p>
</div>
<div id="ref-Gajewski2006">
<p>Gajewski, Thomas F., Yuru Meng, Christian Blank, Ian Brown, Aalok Kacha, Justin Kline, and Helena Harlin. 2006. “Immune resistance orchestrated by the tumor microenvironment.” <em>Immunol. Rev.</em> 213 (1). Blackwell Publishing Ltd: 131–45. doi:<a href="https://doi.org/10.1111/J.1600-065X.2006.00442.X">10.1111/J.1600-065X.2006.00442.X</a>.</p>
</div>
<div id="ref-Galon2012">
<p>Galon, Jérôme, Franck Pagès, Francesco M Marincola, Magdalena Thurin, Giorgio Trinchieri, Bernard A Fox, Thomas F Gajewski, and Paolo A Ascierto. 2012. “The immune score as a new possible approach for the classification of cancer.” <em>J. Transl. Med.</em> 10 (1). BioMed Central: 1. doi:<a href="https://doi.org/10.1186/1479-5876-10-1">10.1186/1479-5876-10-1</a>.</p>
</div>
<div id="ref-Anitei2014">
<p>Anitei, M G, G Zeitoun, B Mlecnik, F Marliot, N Haicheur, A M Todosi, A Kirilovsky, et al. 2014. “Prognostic and predictive values of the immunoscore in patients with rectal cancer.” <em>Clin Cancer Res</em> 20 (7): 1891–9. doi:<a href="https://doi.org/10.1158/1078-0432.ccr-13-2830">10.1158/1078-0432.ccr-13-2830</a>.</p>
</div>
<div id="ref-Pages2018">
<p>Pagès, Franck, Bernhard Mlecnik, Florence Marliot, Gabriela Bindea, Fang-Shu Ou, Carlo Bifulco, Alessandro Lugli, et al. 2018. “International validation of the consensus Immunoscore for the classification of colon cancer: a prognostic and accuracy study.” <em>Lancet (London, England)</em> 391 (10135). Elsevier: 2128–39. doi:<a href="https://doi.org/10.1016/S0140-6736(18)30789-X">10.1016/S0140-6736(18)30789-X</a>.</p>
</div>
<div id="ref-Knebel2017">
<p>“Sequential liquid biopsies reveal dynamic alterations of EGFR driver mutations and indicate EGFR amplification as a new mechanism of resistance to osimertinib in NSCLC.” 2017. <em>Lung Cancer</em> 108. Elsevier: 238–41. doi:<a href="https://doi.org/10.1016/j.lungcan.2017.04.004">10.1016/j.lungcan.2017.04.004</a>.</p>
</div>
<div id="ref-Charoentong2017">
<p>Charoentong, Pornpimol, Francesca Finotello, Mihaela Angelova, Clemens Mayer, Mirjana Efremova, Dietmar Rieder, Hubert Hackl, and Zlatko Trajanoski. 2017. “Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade.” <em>Cell Rep.</em> 18 (1). Elsevier: 248–62. doi:<a href="https://doi.org/10.1016/j.celrep.2016.12.019">10.1016/j.celrep.2016.12.019</a>.</p>
</div>
<div id="ref-Angelova2015">
<p>Angelova, Mihaela, Pornpimol Charoentong, Hubert Hackl, Maria L Fischer, Rene Snajder, Anne M Krogsdam, Maximilian J Waldner, et al. 2015. “Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.” <em>Genome Biol.</em> 16 (1). BioMed Central: 64. doi:<a href="https://doi.org/10.1186/s13059-015-0620-6">10.1186/s13059-015-0620-6</a>.</p>
</div>
<div id="ref-Blank2016">
<p>Blank, Christian U, John B Haanen, Antoni Ribas, and Ton N Schumacher. 2016. “CANCER IMMUNOLOGY. the ‘Cancer Immunogram’.” <em>Science</em> 352 (6286). American Association for the Advancement of Science: 658–60. doi:<a href="https://doi.org/10.1126/science.aaf2834">10.1126/science.aaf2834</a>.</p>
</div>
<div id="ref-Thorsson2018">
<p>Thorsson, Vésteinn, David L Gibbs, Scott D Brown, Denise Wolf, Dante S Bortone, Tai-Hsien Ou Yang, Eduard Porta-Pardo, et al. 2018. “The Immune Landscape of Cancer.” <em>Immunity</em> 48 (4). Elsevier: 812–830.e14. doi:<a href="https://doi.org/10.1016/j.immuni.2018.03.023">10.1016/j.immuni.2018.03.023</a>.</p>
</div>
<div id="ref-Tamborero2018">
<p>Tamborero, David, Carlota Rubio-Perez, Ferran Muinos, Radhakrishnan Sabarinathan, Josep Maria Piulats, Aura Muntasell, Rodrigo Dienstmann, Nuria Lopez-Bigas, and Abel Gonzalez-Perez. 2018. “A pan-cancer landscape of interactions between solid tumors and infiltrating immune cell populations.” <em>Clin. Cancer Res.</em> American Association for Cancer Research, clincanres.3509.2017. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-17-3509">10.1158/1078-0432.CCR-17-3509</a>.</p>
</div>
<div id="ref-Hanzelmann2013">
<p>Hänzelmann, Sonja, Robert Castelo, and Justin Guinney. 2013. “GSVA: gene set variation analysis for microarray and RNA-Seq data.” <em>BMC Bioinformatics</em> 14 (1). BioMed Central: 7. doi:<a href="https://doi.org/10.1186/1471-2105-14-7">10.1186/1471-2105-14-7</a>.</p>
</div>
<div id="ref-Kuperstein2013">
<p>Kuperstein, Inna, David PA Cohen, Stuart Pook, Eric Viara, Laurence Calzone, Emmanuel Barillot, and Andrei Zinovyev. 2013. “NaviCell: a web-based environment for navigation, curation and maintenance of large molecular interaction maps.” <em>BMC Syst. Biol.</em> 7 (1). BioMed Central: 100. doi:<a href="https://doi.org/10.1186/1752-0509-7-100">10.1186/1752-0509-7-100</a>.</p>
</div>
<div id="ref-Kuperstein2015">
<p>Kuperstein, I, E Bonnet, H-A Nguyen, D Cohen, E Viara, L Grieco, S Fourquet, et al. 2015. “Atlas of Cancer Signalling Network: a systems biology resource for integrative analysis of cancer data with Google Maps.” <em>Oncogenesis</em> 4 (7): e160–e160. doi:<a href="https://doi.org/10.1038/oncsis.2015.19">10.1038/oncsis.2015.19</a>.</p>
</div>
<div id="ref-Costa2018">
<p>Costa, Ana, Yann Kieffer, Alix Scholer-Dahirel, Floriane Pelon, Brigitte Bourachot, Melissa Cardon, Philemon Sirven, et al. 2018. “Fibroblast Heterogeneity and Immunosuppressive Environment in Human Breast Cancer.” <em>Cancer Cell</em> 33 (3). Elsevier: 463–479.e10. doi:<a href="https://doi.org/10.1016/j.ccell.2018.01.011">10.1016/j.ccell.2018.01.011</a>.</p>
</div>
<div id="ref-Gyorffy2015">
<p>Győrffy, Balázs, Christos Hatzis, Tara Sanft, Erin Hofstatter, Bilge Aktas, and Lajos Pusztai. 2015. “Multigene prognostic tests in breast cancer: past, present, future.” <em>Breast Cancer Res.</em> 17 (1). BioMed Central: 11. doi:<a href="https://doi.org/10.1186/s13058-015-0514-2">10.1186/s13058-015-0514-2</a>.</p>
</div>
<div id="ref-Ross2008">
<p>Ross, J. S., C. Hatzis, W. F. Symmans, L. Pusztai, and G. N. Hortobagyi. 2008. “Commercialized Multigene Predictors of Clinical Outcome for Breast Cancer.” <em>Oncologist</em> 13 (5): 477–93. doi:<a href="https://doi.org/10.1634/theoncologist.2007-0248">10.1634/theoncologist.2007-0248</a>.</p>
</div>
<div id="ref-Gnjatic2017">
<p>Gnjatic, Sacha, Vincenzo Bronte, Laura Rosa Brunet, Marcus O Butler, Mary L Disis, Jérôme Galon, Leif G Hakansson, et al. 2017. “Identifying baseline immune-related biomarkers to predict clinical outcome of immunotherapy.” <em>J. Immunother. Cancer</em> 5. BioMed Central: 44. doi:<a href="https://doi.org/10.1186/s40425-017-0243-4">10.1186/s40425-017-0243-4</a>.</p>
</div>
<div id="ref-Itadani2008">
<p>Itadani, Hiraku, Shinji Mizuarai, and Hidehito Kotani. 2008. “Can Systems Biology Understand Pathway Activation? Gene Expression Signatures as Surrogate Markers for Understanding the Complexity of Pathway Activation.” <em>Curr. Genomics</em> 9 (5): 349–60. doi:<a href="https://doi.org/10.2174/138920208785133235">10.2174/138920208785133235</a>.</p>
</div>
<div id="ref-Liu2008">
<p>Liu, Jiangang, Andrew Campen, Shuguang Huang, Sheng-Bin Peng, Xiang Ye, Mathew Palakal, A Keith Dunker, Yuni Xia, and Shuyu Li. 2008. “Identification of a gene signature in cell cycle pathway for breast cancer prognosis using gene expression profiling data.” <em>BMC Med. Genomics</em> 1 (1): 39. doi:<a href="https://doi.org/10.1186/1755-8794-1-39">10.1186/1755-8794-1-39</a>.</p>
</div>
<div id="ref-Heng2008">
<p>Heng, Tracy S P, Michio W Painter, Kutlu Elpek, Veronika Lukacs-Kornek, Nora Mauermann, Shannon J Turley, Daphne Koller, et al. 2008. “The Immunological Genome Project: networks of gene expression in immune cells.” <em>Nat. Immunol.</em> 9 (10): 1091–4. doi:<a href="https://doi.org/10.1038/ni1008-1091">10.1038/ni1008-1091</a>.</p>
</div>
<div id="ref-Vandenbon2016">
<p>Vandenbon, Alexis, Viet H Dinh, Norihisa Mikami, Yohko Kitagawa, Shunsuke Teraguchi, Naganari Ohkura, and Shimon Sakaguchi. 2016. “Immuno-Navigator, a batch-corrected coexpression database, reveals cell type-specific gene networks in the immune system.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 113 (17). National Academy of Sciences: E2393–402. doi:<a href="https://doi.org/10.1073/pnas.1604351113">10.1073/pnas.1604351113</a>.</p>
</div>
<div id="ref-Godec2016">
<p>Godec, Jernej, Yan Tan, Arthur Liberzon, Pablo Tamayo, Sanchita Bhattacharya, Atul J. Butte, Jill P. Mesirov, and W. Nicholas Haining. 2016. “Compendium of Immune Signatures Identifies Conserved and Species-Specific Biology in Response to Inflammation.” <em>Immunity</em> 44 (1): 194–206. doi:<a href="https://doi.org/10.1016/j.immuni.2015.12.006">10.1016/j.immuni.2015.12.006</a>.</p>
</div>
<div id="ref-Subramanian2005">
<p>Subramanian, Aravind, Pablo Tamayo, Vamsi K Mootha, Sayan Mukherjee, Benjamin L Ebert, Michael A Gillette, Amanda Paulovich, et al. 2005. “Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 102 (43). National Academy of Sciences: 15545–50. doi:<a href="https://doi.org/10.1073/pnas.0506580102">10.1073/pnas.0506580102</a>.</p>
</div>
<div id="ref-Michiels2016">
<p>Michiels, S., N. Ternès, and F. Rotolo. 2016. “Statistical controversies in clinical research: prognostic gene signatures are not (yet) useful in clinical practice.” <em>Ann. Oncol.</em> 27 (12): 2160–7. doi:<a href="https://doi.org/10.1093/annonc/mdw307">10.1093/annonc/mdw307</a>.</p>
</div>
<div id="ref-Harris2016">
<p>Harris, Lyndsay N., Nofisat Ismaila, Lisa M. McShane, Fabrice Andre, Deborah E. Collyar, Ana M. Gonzalez-Angulo, Elizabeth H. Hammond, et al. 2016. “Use of Biomarkers to Guide Decisions on Adjuvant Systemic Therapy for Women With Early-Stage Invasive Breast Cancer: American Society of Clinical Oncology Clinical Practice Guideline.” <em>J. Clin. Oncol.</em> 34 (10): 1134–50. doi:<a href="https://doi.org/10.1200/JCO.2015.65.2289">10.1200/JCO.2015.65.2289</a>.</p>
</div>
<div id="ref-Chifman2016">
<p>Chifman, Julia, Ashok Pullikuth, Jeff W Chou, Davide Bedognetti, and Lance D Miller. 2016. “Conservation of immune gene signatures in solid tumors and prognostic implications.” <em>BMC Cancer</em> 16 (1): 911. doi:<a href="https://doi.org/10.1186/s12885-016-2948-z">10.1186/s12885-016-2948-z</a>.</p>
</div>
<div id="ref-NCI2018">
<p>Institute, National Cancer. 2017. “Types of Cancer Treatment.” <a href="https://www.cancer.gov/about-cancer/treatment/types" class="uri">https://www.cancer.gov/about-cancer/treatment/types</a>.</p>
</div>
<div id="ref-Predina2013">
<p>Predina, J., E. Eruslanov, B. Judy, V. Kapoor, G. Cheng, L.-C. Wang, J. Sun, et al. 2013. “Changes in the local tumor microenvironment in recurrent cancers may explain the failure of vaccines after surgery.” <em>Proc. Natl. Acad. Sci.</em> 110 (5): E415–E424. doi:<a href="https://doi.org/10.1073/pnas.1211850110">10.1073/pnas.1211850110</a>.</p>
</div>
<div id="ref-Palucka2013">
<p>Palucka, Karolina, and Jacques Banchereau. 2013. “Dendritic-Cell-Based Therapeutic Cancer Vaccines.” doi:<a href="https://doi.org/10.1016/j.immuni.2013.07.004">10.1016/j.immuni.2013.07.004</a>.</p>
</div>
<div id="ref-Sagiv-Barfi2018">
<p>Sagiv-Barfi, Idit, Debra K. Czerwinski, Shoshana Levy, Israt S. Alam, Aaron T. Mayer, Sanjiv S. Gambhir, and Ronald Levy. 2018. “Eradication of spontaneous malignancy by local immunotherapy.” <em>Sci. Transl. Med.</em> 10 (426). American Association for the Advancement of Science: eaan4488. doi:<a href="https://doi.org/10.1126/SCITRANSLMED.AAN4488">10.1126/SCITRANSLMED.AAN4488</a>.</p>
</div>
<div id="ref-Jackson2016">
<p>Jackson, Hollie J., Sarwish Rafiq, and Renier J. Brentjens. 2016. “Driving CAR T-cells forward.” doi:<a href="https://doi.org/10.1038/nrclinonc.2016.36">10.1038/nrclinonc.2016.36</a>.</p>
</div>
<div id="ref-FDACARTadult">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p> 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p> 2016b. “Pembrolizumab (Keytruda) Checkpoint Inhibitor.” <a href="https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm" class="uri">https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm</a>.</p> 2017a. “FDA Approval Brings First Gene Therapy to the United States.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm</a>.</p> 2017b. “FDA Approves Car-T Cell Therapy to Treat Adults with Certain Types of Large B-Cell Lymphoma.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm581216.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm581216.htm</a>.</p>
</div>
<div id="ref-FDACARTALL">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p> 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p> 2016b. “Pembrolizumab (Keytruda) Checkpoint Inhibitor.” <a href="https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm" class="uri">https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm</a>.</p> 2017a. “FDA Approval Brings First Gene Therapy to the United States.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm</a>.</p>
</div>
<div id="ref-FDACTLA4">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p>
</div>
<div id="ref-FDAPDL1Bladder">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p> 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p>
</div>
<div id="ref-FDAPDL1NSCLC">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p> 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p> 2016b. “Pembrolizumab (Keytruda) Checkpoint Inhibitor.” <a href="https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm" class="uri">https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm</a>.</p>
</div>
<div id="ref-Wolchok2015">
<p>Wolchok, Jedd D. 2015. “PD-1 Blockers.” doi:<a href="https://doi.org/10.1016/j.cell.2015.07.045">10.1016/j.cell.2015.07.045</a>.</p>
</div>
<div id="ref-Zou2016">
<p>Zou, Weiping, Jedd D. Wolchok, and Lieping Chen. 2016. “PD-L1 (B7-H1) and PD-1 pathway blockade for cancer therapy: Mechanisms, response biomarkers, and combinations.” <em>Sci. Transl. Med.</em> 8 (328). doi:<a href="https://doi.org/10.1126/scitranslmed.aad7118">10.1126/scitranslmed.aad7118</a>.</p>
</div>
<div id="ref-Rizvi2015">
<p>Rizvi, Naiyer A., Matthew D. Hellmann, Alexandra Snyder, Pia Kvistborg, Vladimir Makarov, Jonathan J. Havel, William Lee, et al. 2015. “Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer.” <em>Science (80-. ).</em> 348 (6230): 124–28. doi:<a href="https://doi.org/10.1126/science.aaa1348">10.1126/science.aaa1348</a>.</p>
</div>
<div id="ref-Pitt2016">
<p>Pitt, Jonathan M, Marie Vétizou, Romain Daillère, María Paula Roberti, Takahiro Yamazaki, Bertrand Routy, Patricia Lepage, et al. 2016. “Resistance Mechanisms to Immune-Checkpoint Blockade in Cancer: Tumor-Intrinsic and -Extrinsic Factors.” <em>Immunity</em> 44 (6). Elsevier: 1255–69. doi:<a href="https://doi.org/10.1016/j.immuni.2016.06.001">10.1016/j.immuni.2016.06.001</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="organisation-of-the-dissertation.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="methods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/02-IntroductionBio.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
