<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="preamble-about-interdisciplinary-research.html">
<link rel="next" href="methods.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; min-height: 1.25em; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; }
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
@media screen {
a.sourceLine::before { text-decoration: underline; color: initial; }
}
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.bn { color: #40a070; } /* BaseN */
code span.fl { color: #40a070; } /* Float */
code span.ch { color: #4070a0; } /* Char */
code span.st { color: #4070a0; } /* String */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.ot { color: #007020; } /* Other */
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.fu { color: #06287e; } /* Function */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code span.cn { color: #880000; } /* Constant */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.ss { color: #bb6688; } /* SpecialString */
code span.im { } /* Import */
code span.va { color: #19177c; } /* Variable */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.op { color: #666666; } /* Operator */
code span.bu { } /* BuiltIn */
code span.ex { } /* Extension */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.at { color: #7d9029; } /* Attribute */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-phd---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary PhD - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#general-context-of-the-thesis"><i class="fa fa-check"></i>General context of the thesis</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#organisation-of-the-dissertation"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-micro-environment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor micro environment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.2</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.3</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.4</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#biological-dimension-of-the-thesis"><i class="fa fa-check"></i><b>1.4</b> Biological dimension of the thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#blind-source-separation"><i class="fa fa-check"></i><b>2.2</b> Blind source separation</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#finding-optimal-number-of-components-and-over-decomposition-of-transcriptopmes"><i class="fa fa-check"></i><b>2.3</b> Finding optimal number of components and over-decomposition of transcriptopmes</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#cell-type-deconvolution-models"><i class="fa fa-check"></i><b>2.4</b> Cell-type deconvolution models</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#basis-matrix"><i class="fa fa-check"></i><b>2.4.1</b> basis matrix</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#regression-algorithm"><i class="fa fa-check"></i><b>2.4.2</b> regression algorithm</a></li>
<li class="chapter" data-level="2.4.3" data-path="methods.html"><a href="methods.html#others"><i class="fa fa-check"></i><b>2.4.3</b> Others</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#short-review-of-most-popular-cell-type-deconvolution-tools"><i class="fa fa-check"></i><b>2.5</b> Short review of most popular cell-type deconvolution tools</a></li>
<li class="chapter" data-level="2.6" data-path="methods.html"><a href="methods.html#methodological-dimension-of-the-thesis"><i class="fa fa-check"></i><b>2.6</b> Methodological dimension of the thesis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="sens.html"><a href="sens.html"><i class="fa fa-check"></i><b>3</b> Study of sensitivity and reproducibility of known methods</a><ul>
<li class="chapter" data-level="3.1" data-path="sens.html"><a href="sens.html#reproducibility-of-nmf-versus-ica-vs-cam"><i class="fa fa-check"></i><b>3.1</b> Reproducibility of NMF versus ICA (vs CAM?)</a><ul>
<li class="chapter" data-level="3.1.1" data-path="sens.html"><a href="sens.html#comparing-metagenes-obtained-with-nmf-vs-ica."><i class="fa fa-check"></i><b>3.1.1</b> Comparing metagenes obtained with NMF vs ICA.</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="sens.html"><a href="sens.html#impact-of-modification-of-signatures-list-on-result-for-signature-based-deconvolution-methods"><i class="fa fa-check"></i><b>3.2</b> Impact of modification of signatures list on result for signature-based deconvolution methods</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>4</b> Deconvolution of transcriptomes and methylomes</a><ul>
<li class="chapter" data-level="4.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>4.1</b> From blind deconvolution to cell-type quantification: general overview</a><ul>
<li class="chapter" data-level="4.1.1" data-path="deconica.html"><a href="deconica.html#the-ica-based-deconvolution-of-transcriptomes"><i class="fa fa-check"></i><b>4.1.1</b> The ICA-based deconvolution of Transcriptomes</a></li>
<li class="chapter" data-level="4.1.2" data-path="deconica.html"><a href="deconica.html#interpretation-of-independent-components"><i class="fa fa-check"></i><b>4.1.2</b> Interpretation of Independent components</a></li>
<li class="chapter" data-level="4.1.3" data-path="deconica.html"><a href="deconica.html#transforming-metagenes-into-signature-matrix"><i class="fa fa-check"></i><b>4.1.3</b> Transforming metagenes into signature matrix</a></li>
<li class="chapter" data-level="4.1.4" data-path="deconica.html"><a href="deconica.html#regression-based-estimation-of-cell-type-proportions-solving-system-of-equations"><i class="fa fa-check"></i><b>4.1.4</b> Regression-based estimation of cell-type proportions : solving system of equations</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="deconica.html"><a href="deconica.html#deconica-r-package-for-ica-based-deconvolution"><i class="fa fa-check"></i><b>4.2</b> <em>DeconICA</em> R package for ICA-based deconvolution</a><ul>
<li class="chapter" data-level="4.2.1" data-path="deconica.html"><a href="deconica.html#demo"><i class="fa fa-check"></i><b>4.2.1</b> <em>Demo</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>5</b> Comparative analysis of cancer immune infiltration</a></li>
<li class="chapter" data-level="6" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>6</b> Heterogeneity of immune cell types</a></li>
<li class="chapter" data-level="7" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>7</b> Conclusions and perspectives</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#phd-timeline-for-defence-before-the-end-of-october-2018"><i class="fa fa-check"></i>PhD timeline for defence before the end of October 2018</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#thesis-writing"><i class="fa fa-check"></i>Thesis writing</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#activity-report-2017"><i class="fa fa-check"></i>Activity Report 2017</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="intro" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Immuno-biology of cancer</h1>
<p>This chapter will first introduce a short history of cancer with a focus on discoveries linking cancer and its environment. It will also describe participation of TME in cancer development, progression and response to treatment. Most important types of data used to study cancer microenvironment will be discussed. A link between tumor immune-biology and cancer phenotyping for development of immunotherapies will be introduced as well.</p>
<div id="cancer-disease" class="section level2">
<h2><span class="header-section-number">1.1</span> Cancer disease</h2>
<p>According to <a href="http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx">GLOBOCAN study</a> <span class="citation">(Research in Cancer <a href="#ref-GLOBOCAN">2018</a>)</span>, 14.1 million cancer cases was estimated to happen around the world in 2012. It touched 7.4 million men and 6.7 million women. It is estimated that the cancer cases will increase almost two-fold to 24 million by 2035.</p>
<p>In France only, in 2012 there were 194552 cases of cancer, of which leading is Prostate cancer (29,2%) followed by Lung (14,4%) and Colorectal cancers (11,1%).</p>
<p>For a long time studying tumor was focused on tumor cells, their reprogramming, mutations. Cancer was seen as disease of uncontrolled cells by the mainstream research. At the same time, the idea of importance of the impact of other cells and structures on cancer cells was present but often not believed. Recent success of immunotherapies moved research focus to tumor cells in their context: tumor microenvironment. We will describe here what is the composition and role of the TME in tumor progression, diagnosis and response to treatment.</p>
<div id="hist" class="section level3">
<h3><span class="header-section-number">1.1.1</span> Historical understanding of cancer</h3>
<p>Cancer was historically described by a physician Hippocrates (460–370 B.C) <span class="citation">(Sudhakar <a href="#ref-Sudhakar2009">2009</a>)</span>. Even though there exist even earlier evidence of the disease. Hippocrates stated that the body contained 4 humors (body fluids) : blood, phlegm, yellow bile and black bile. Any imbalance of these fluids will result in disease. Particularly the excess of black bile in an organ was meant to provoke cancer. For years, it was not known what factors cause cancer and it was easily confounded with other diseases. In the middle ages in the Renaissance Period it was believed cancer is a punishment for the sins they committed against their god, that they deserved it to some extend</p>
<p>Until 18th century it was believed that cancer is contagious and is spread by parasites.</p>
<p>In the 19th century, tumor cells started to be analysed by pathologists. They were strike with their ability to proliferate uncontrollably, ability to spread and destroy the original tissue <span class="citation">(NPR <a href="#ref-NPR2010">2010</a><a href="#ref-NPR2010">b</a>)</span>. Around the same time leukocytes from the blood was first described by Gabriel Andra and William Addison. Just a few years later, in 1845 Bennett and Virchow described blood cells in leukaemia (Fig. <a href="intro.html#fig:Virchow-cell">1.1</a>). Virchow is also a father of Chronic irritation theory (which we would call chronic inflammation) that says that cancer is caused by local “irritation” and, incorrectly, that cancer cells spread like liquid resulting in metastasis.</p>
<div class="figure" style="text-align: center"><span id="fig:Virchow-cell"></span>
<img src="figures-ext/01-Virchow-cell.jpg" alt="Illustration of Virchow’s cell theory. Virchow depicted different cells transformation due to irritation. (???)" width="100%" />
<p class="caption">
Figure 1.1: <strong>Illustration of Virchow’s cell theory</strong>. Virchow depicted different cells transformation due to irritation. <span class="citation">(<span class="citeproc-not-found" data-reference-id="Virchow1847"><strong>???</strong></span>)</span>
</p>
</div>

<p>In 1889, Stephen Paget introduced <em>soil and seed</em> hypothesis of metastases <span class="citation">(<span class="citeproc-not-found" data-reference-id="Paget1889"><strong>???</strong></span>)</span>. He formulates it as follows</p>
<blockquote>
<p><em>When a plant goes to seed, its seeds are carried in all directions, but they can only live and grow if they fall on congenial soil.</em></p>
</blockquote>
<p>Which is a parallel to cancer cells disseminated by body fluids, and they can grow only tissues - “soil” that is predisposed to host the cancer cell - “the seed”. He focused on the importance of tissue characteristics that favorise tumor development as opposed to most researchers of his time that were focusing on the “seed” itself.</p>
<p>In the 20th century, molecular causes started to be investigated. It was discovered that cancer could be caused by environmental factors, i.e. chemicals (carcinogens), radiation, viruses and also inherited from ancestors. Those factors would damage but contrary to a healthy condition they would not die.</p>
<p>Also in 1909, Paul Ehrlich, called one of fathers of immunology and Nobel Prize laureate, indicated a link between immune system and tumor suppression <span class="citation">(<span class="citeproc-not-found" data-reference-id="Ehrlich1909"><strong>???</strong></span>)</span>. One of remarkable first immontherapy attempts can be attributed to William Coley, that practiced injecting streptococcus bacteria directly into patients after cancer surgery in 1891, later called “Colley vaccine”. However, the impact of this procedure on patients recovery was judged by scientific community as “unclear”.</p>
<p>In 1968, Melvin Greenblatt and Philippe Shubik showed that tumour transplants secete a substance stimulating the growth of blood vessels <span class="citation">(<span class="citeproc-not-found" data-reference-id="Greenblatt1968"><strong>???</strong></span>)</span>, later identified as “tumour angiogenic factor (TAF)” by Judah Folkman in 1971 <span class="citation">(<span class="citeproc-not-found" data-reference-id="Folkman1971"><strong>???</strong></span>)</span>. Folkman also suggested that TAF can target of therapy itself which was a revolutional idea, as it did not target the tumor cells but their environment.</p>
<p>During the 1970s, oncogenes and tumor suppressor genes were discovered. Oncogenes are genes that allow a cell to become cancer cell, while the tumor suppressor genes would repair DNA or execute cell death of a damaged cell. A new dimension to cancer studies was added in the 1980s, epigenetic changes was proven to occur to both oncogenes and tumour suppressors <span class="citation">(<span class="citeproc-not-found" data-reference-id="Feinberg1983"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Greger1989"><strong>???</strong></span>)</span>, which are presently known as epigenetic markers used for diagnostics and therapeutic targets for cancer.</p>
<p>In 1982, Aline van Pel and Thierry Boon <span class="citation">(<span class="citeproc-not-found" data-reference-id="VanPel1982"><strong>???</strong></span>)</span> discovered that a specific immunity to spontanious tumor cells could be induced by vaccinating mice with mutagenized tumour cells. This araised an inspiration for many years of immune therapy developement.</p>
<p>In Napoleone Ferrara and colleagues identified gene encoding vascular endothelial growth factor (VEGF) that was shown to stimulate growth of enothelial cells proliferation <em>in vitro</em> and angiogenesis (blood vessels formation) <em>in vivo</em> <span class="citation">(<span class="citeproc-not-found" data-reference-id="Leung1989"><strong>???</strong></span>)</span>.</p>
<p>In 1999 for the first time, gene-expression was used to study cancer (leukeamia) by Todd Golub, Donna Slonim and colleagues <span class="citation">(<span class="citeproc-not-found" data-reference-id="Golub1999"><strong>???</strong></span>)</span>.</p>
<p>Since the end of the 20th century, cancer screens are developed along with multiple strategies to fight tumor. Most classical ones are based on the idea of removing tumor cells (surgery), killing tumor cells with DNA-blocking drugs (chemotherapy), radiation, inhibit cancer growth (hormonal therapy, adjuvant therapy and immunotherapy). As non of those methods is fully efficient, often a combination of treatments is proposed. Nowadays, science is aming in the direction of trageted therapies and personalized treatment.</p>
<p>The recent success of immunotherapies (discussed in <a href="intro.html#immunotherapies">Immunotherapies section</a> attracted the attention the scientific community again to the context in which tumor cells are found. This context called Tumor Microenvironment, as well as the communication that happens within it between different agents, nowadays studied differently with available knowledge of molecular biology, have become a popular scientific topics of 21st century (Fig. <a href="intro.html#fig:pubmedTME">1.2</a>).</p>
<div class="figure" style="text-align: center"><span id="fig:pubmedTME"></span>
<div id="21d4708581e1" style="width:70%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="21d4708581e1">{"x":{"data":[{"orientation":"v","width":[0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091,0.900000000000091],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1950,1951,1952,1953,1954,1955,1956,1957,1958,1959,1960,1961,1962,1963,1964,1965,1966,1967,1968,1969,1970,1971,1972,1973,1974,1975,1976,1977,1978,1979,1980,1981,1982,1983,1984,1985,1986,1987,1988,1989,1990,1991,1992,1993,1994,1995,1996,1997,1998,1999,2000,2001,2002,2003,2004,2005,2006,2007,2008,2009,2010,2011,2012,2013,2014,2015,2016],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4.01411362350023e-06,7.84713775650332e-06,0,1.46888713438481e-05,1.0646452601993e-05,1.42687446715157e-05,2.1182476443321e-05,1.35460514953148e-05,1.6175000566125e-05,3.46051240750994e-05,4.77814476584104e-05,4.86892565723336e-05,7.07502244959047e-05,6.72625438176668e-05,9.19871715187828e-05,9.74981962833688e-05,5.57414951807842e-05,9.34225718515396e-05,0.000135886118061608,0.000189523246282033,0.000205119058235976,0.000224835848002235,0.000234806175182962,0.000276840886228962,0.000245495861061516,0.000252070173326461,0.000390067025601582,0.00041818465333201,0.000476993893466869,0.000509121897881113,0.000629368028676296,0.000716625030975615,0.000856136023706814,0.0009827909827431,0.00111680424979417,0.00137703697796209,0.00187891461005892,0.00216715357778359,0.0024373280322394,0.00272225123846604,0.00309914470308773,0.00348693807766557],"text":["~Year: 1950<br />~percentage: 0.000000e+00","~Year: 1951<br />~percentage: 0.000000e+00","~Year: 1952<br />~percentage: 0.000000e+00","~Year: 1953<br />~percentage: 0.000000e+00","~Year: 1954<br />~percentage: 0.000000e+00","~Year: 1955<br />~percentage: 0.000000e+00","~Year: 1956<br />~percentage: 0.000000e+00","~Year: 1957<br />~percentage: 0.000000e+00","~Year: 1958<br />~percentage: 0.000000e+00","~Year: 1959<br />~percentage: 0.000000e+00","~Year: 1960<br />~percentage: 0.000000e+00","~Year: 1961<br />~percentage: 0.000000e+00","~Year: 1962<br />~percentage: 0.000000e+00","~Year: 1963<br />~percentage: 0.000000e+00","~Year: 1964<br />~percentage: 0.000000e+00","~Year: 1965<br />~percentage: 0.000000e+00","~Year: 1966<br />~percentage: 0.000000e+00","~Year: 1967<br />~percentage: 0.000000e+00","~Year: 1968<br />~percentage: 0.000000e+00","~Year: 1969<br />~percentage: 0.000000e+00","~Year: 1970<br />~percentage: 0.000000e+00","~Year: 1971<br />~percentage: 0.000000e+00","~Year: 1972<br />~percentage: 0.000000e+00","~Year: 1973<br />~percentage: 0.000000e+00","~Year: 1974<br />~percentage: 0.000000e+00","~Year: 1975<br />~percentage: 4.014114e-06","~Year: 1976<br />~percentage: 7.847138e-06","~Year: 1977<br />~percentage: 0.000000e+00","~Year: 1978<br />~percentage: 1.468887e-05","~Year: 1979<br />~percentage: 1.064645e-05","~Year: 1980<br />~percentage: 1.426874e-05","~Year: 1981<br />~percentage: 2.118248e-05","~Year: 1982<br />~percentage: 1.354605e-05","~Year: 1983<br />~percentage: 1.617500e-05","~Year: 1984<br />~percentage: 3.460512e-05","~Year: 1985<br />~percentage: 4.778145e-05","~Year: 1986<br />~percentage: 4.868926e-05","~Year: 1987<br />~percentage: 7.075022e-05","~Year: 1988<br />~percentage: 6.726254e-05","~Year: 1989<br />~percentage: 9.198717e-05","~Year: 1990<br />~percentage: 9.749820e-05","~Year: 1991<br />~percentage: 5.574150e-05","~Year: 1992<br />~percentage: 9.342257e-05","~Year: 1993<br />~percentage: 1.358861e-04","~Year: 1994<br />~percentage: 1.895232e-04","~Year: 1995<br />~percentage: 2.051191e-04","~Year: 1996<br />~percentage: 2.248358e-04","~Year: 1997<br />~percentage: 2.348062e-04","~Year: 1998<br />~percentage: 2.768409e-04","~Year: 1999<br />~percentage: 2.454959e-04","~Year: 2000<br />~percentage: 2.520702e-04","~Year: 2001<br />~percentage: 3.900670e-04","~Year: 2002<br />~percentage: 4.181847e-04","~Year: 2003<br />~percentage: 4.769939e-04","~Year: 2004<br />~percentage: 5.091219e-04","~Year: 2005<br />~percentage: 6.293680e-04","~Year: 2006<br />~percentage: 7.166250e-04","~Year: 2007<br />~percentage: 8.561360e-04","~Year: 2008<br />~percentage: 9.827910e-04","~Year: 2009<br />~percentage: 1.116804e-03","~Year: 2010<br />~percentage: 1.377037e-03","~Year: 2011<br />~percentage: 1.878915e-03","~Year: 2012<br />~percentage: 2.167154e-03","~Year: 2013<br />~percentage: 2.437328e-03","~Year: 2014<br />~percentage: 2.722251e-03","~Year: 2015<br />~percentage: 3.099145e-03","~Year: 2016<br />~percentage: 3.486938e-03"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(70,130,180,1)","line":{"width":1.88976377952756,"color":"transparent"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":40.1826484018265,"l":54.7945205479452},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"type":"linear","autorange":false,"range":[1946.205,2019.795],"tickmode":"array","ticktext":["1950","1955","1960","1965","1970","1975","1980","1985","1990","1995","2000","2005","2010","2015"],"tickvals":[1950,1955,1960,1965,1970,1975,1980,1985,1990,1995,2000,2005,2010,2015],"categoryorder":"array","categoryarray":["1950","1955","1960","1965","1970","1975","1980","1985","1990","1995","2000","2005","2010","2015"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"Year","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"type":"linear","autorange":false,"range":[-0.000174346903883279,0.00366128498154885],"tickmode":"array","ticktext":["0.000","0.001","0.002","0.003"],"tickvals":[0,0.001,0.002,0.003],"categoryorder":"array","categoryarray":["0.000","0.001","0.002","0.003"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"Percentage of all publications [%]","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"source":"A","attrs":{"21d482bc0e7":{"x":{},"y":{},"fill":{},"type":"bar"}},"cur_data":"21d482bc0e7","visdat":{"21d482bc0e7":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p class="caption">
Figure 1.2: <strong>Percentage of publications containg phrase “tumor immunotherapy” is growing</strong>, numbers retreived on 17.01.2018 from <a href="http://dan.corlan.net/medline-trend.html">Medline Trends</a> <span class="citation">(Corlan <a href="#ref-Corlan2004">2004</a>)</span>
</p>
</div>

</div>
<div id="tumor-micro-environment-as-a-complex-system" class="section level3">
<h3><span class="header-section-number">1.1.2</span> Tumor micro environment as a complex system</h3>
<p>Tumor Microenvironment is a complex tissue that surrounds tumor cells. It is composed of different compartments (in solid tumors):</p>
<ul>
<li>Stroma: blood and lymphatics vessels, epithelial cells, mesenchymal stem cells, fibroblast, adipocytes supported by extracellular matrix (EM)</li>
<li>Immune cells: T cells, B cells, NK cells, Dendritic cells, Macorphages, Monocytes etc.</li>
</ul>
<p>Their proportion and specific roles vary significantly with tumor type and stage. Communication between the environmental cells and the tumor is critical for tumor development and its impact on patient’s response to treatment. These communication between different compartments is bidirectional and all the players can influence each other. Depending on the nature and prevailing direction of those interactions different destiny is possible for each of the compartments, i.e. immune cells can be recruited to protect tumor cells or they can kill them directly. Many of the signals can be contradictory, many can suppress each other. Then is it possible to tilt this complex ecosystem into patients’ favour? Can we decipher the most important factors of this molecular knot and manipulate it?</p>
<p>This chapter describes different scenarios of interaction within TME in order to illustrate the complexity of TME and possible targets for cancer therapies.</p>
<p>We recommend watching this video in order to visualise the TME and cancer thanks to 3D animations</p>
<iframe width="560" height="315" src="https://www.youtube.com/embed/K09xzIQ8zsg" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen>
</iframe>
<div id="interactions-between-tme-and-tumor" class="section level4">
<h4><span class="header-section-number">1.1.2.1</span> Interactions between TME and Tumor</h4>
<p>Three scenarios can be considered to describe the relationship between TME and tumor cells:</p>
<ol style="list-style-type: decimal">
<li>TME stimulates tumor growth and/or progression and/or impact negatively the response to treatment</li>
<li>TME has no impact on tumor cells and disease development</li>
<li>TME has a tumor suppressive role and impact positively the response to treatment</li>
</ol>
<p>As can be seen partly in <a href="intro.html#hist">Historical understanding of cancer</a> these three hypothesis were gaining and loosing popularity in scientific and medical community over the decades.</p>
<div id="tme-as-a-foe-inflammation" class="section level5">
<h5><span class="header-section-number">1.1.2.1.1</span> TME as a foe: inflammation</h5>
<p>In 1863 Rudolf Virchow observed a link between chronic inflammation and tumorigenesis. According to Virchov theory genetic damage would be the “match that lights the fire” of cancer, and the inflammation or cytokines produced by immune cells should be the “fuel that feeds the flames” <span class="citation">(Balkwill and Mantovani <a href="#ref-Balkwill2001">2001</a>)</span>. Therefore lymphocyte infiltration was confirmed by subsequent studies as a hallmark o cancer. The question one may ask is why our immune system does not defend the organism from tumor cells as it does in a range of bacterial and viral infections? It is mainly because of the ability of tumor cells to inhibit immune response through activation of negative regulatory pathways (so called immune checkpoints).</p>
<p>Many examples can be cited on how TME facilitates tumor development (Fig. <a href="intro.html#fig:met-dis">1.3</a>). For instance, in the early stages of tumorigenesis some macrophage phenotypes support tumor growth and mobility through TGF-beta signaling. Also, it was shown that NK cells and myeloid-derived suppressor cells (MDSCs) have an ability to suppress immune defence i.e. immunosurveillance by dendritic cells (DCs), T cell activation and macrophage polarisation and they promote tumor vascularisation as well. <span class="citation">(Talmadge and Gabrilovich <a href="#ref-Talmadge2013">2013</a>; Gabrilovich, Ostrand-Rosenberg, and Bronte <a href="#ref-Gabrilovich2012">2012</a>)</span> They create so-called niches that facilitates tumor colonization. Tregs and myeloid-derived suppressor cells can negatively impact natural immune defence and by these means allow growth and invasion of tumor cells <span class="citation">(Taube et al. <a href="#ref-Taube2017a">2017</a>)</span>. Another cell type, a part of ECM, fibroblast, or more precisely Cancer Associated Fibroblasts (CAFs) have proven pro-tumor functions in breast cancer where they enhance metastasis <span class="citation">(Dumont et al. <a href="#ref-Dumont2013">2013</a>)</span>. The blood and lymphatic vessels maintain tumor growth providing necessary nutritive compound to malignant cells.</p>
<div class="figure" style="text-align: center"><span id="fig:met-dis"></span>
<img src="figures-ext/massive-dissemination.png" alt="The microenvironment supports metastatic dissemination and colonization at secondary sites. Different tumor sites can communicate through exosomes realized by tumor cells and also immune and stromal cells such as NK cells, CAFs and DCs. Reprinted by permission from Springer Nature (Quail and Joyce 2013) © 2013 Nature America, Inc. All rights reserved." width="100%" />
<p class="caption">
Figure 1.3: <strong>The microenvironment supports metastatic dissemination and colonization at secondary sites.</strong> Different tumor sites can communicate through exosomes realized by tumor cells and also immune and stromal cells such as NK cells, CAFs and DCs. Reprinted by permission from Springer Nature <span class="citation">(Quail and Joyce <a href="#ref-Quail2013">2013</a>)</span> © 2013 Nature America, Inc. All rights reserved.
</p>
</div>

<p>According to <span class="citation">(Hanahan and Coussens <a href="#ref-Hanahan2012">2012</a>)</span> immune and stroma cells participate in almost all of Cancer Hallmarks <span class="citation">(Hanahan and Weinberg <a href="#ref-Hanahan2000">2000</a>; Hanahan and Coussens <a href="#ref-Hanahan2012">2012</a>)</span>. Most of the hallmarks of cancer are enabled and sustained to varying degrees through contributions from repertoires of stromal cell types and distinctive subcell types.</p>
</div>
<div id="tme-seen-as-neutral" class="section level5">
<h5><span class="header-section-number">1.1.2.1.2</span> TME seen as neutral</h5>
<p>In front of lack of definitive proof that TME can positively or negatively impact on tumor development, many scientist, in a long time, ignored the importance of this factor. Until the early-mid eighties, the TME research was mostly limited to angiogenesis and immune environment and most areas that are now driving the field were not represented.</p>
<p>From early 70. until the end of the 90. the most accepted statement was that genetic alterations in oncogenes and tumor suppressor genes are both necessary and sufficient to initiate tumorigenesis and drive tumor progression. Therefore TME was not seen as an important element of the puzzle.</p>
<p>The cancer geneticists, at the time had a lot of influence on scientific community diminishing the work of made on TME which were considered as “uninteresting” and definitely not “mainstream”.</p>
<p>After 90. with discovery of singling molecules involved in communication of TME like VEGF. Also discoveries made by developmental biology field supported the hypothesis that microenvironment plays an important role in development which was later shown for tumorigenesis. Also success of immune vaccines starting with the tuberculosis vaccine Bacille Calmette-Guérin (BCG) in 1976 and finishing, at the moment with checkpoint inhibitors did not leave the scientific community indifferent.</p>
</div>
<div id="tme-as-a-friend-immunosurveillance" class="section level5">
<h5><span class="header-section-number">1.1.2.1.3</span> TME as a friend: immunosurveillance</h5>
<p>As mentioned in <a href="intro.html#hist">Historical understanding of cancer</a> Paget proposed a hypothesis of “seed and soil” where the TME in a certain tissue (the soil) can either stimulate or suppress the metastasis (the seed). William Coley tested a possibility to trigger tumorsuppressive effect via stimulation of the immune system with bacteria. In the 1960s, the immune surveillance theory hypothesized “the ability to identify and destroy nascent tumors as a central asset of the immune system” <span class="citation">(Sebeok <a href="#ref-Sebeok1976">1976</a>; Burnet <a href="#ref-Burnet1970">1970</a>)</span>, but it was highly criticized in consequence of no increase in tumor incidence in athymic nude mice <span class="citation">(Stutman <a href="#ref-Stutman1974">1974</a>; <span class="citeproc-not-found" data-reference-id="Rygaard1976"><strong>???</strong></span>)</span>. Later it was shown, that this mice model was not adequate <span class="citation">(Cavallo et al. <a href="#ref-Cavallo2011">2011</a>)</span> . Thus, the hypothesis that TME can have a positive role in tumor prognosis is not new.</p>
<p>The immunosurveillance through immune-editing can be summarized in three processes: elimination, equilibrium, and escape <span class="citation">(<span class="citeproc-not-found" data-reference-id="Dunn2002"><strong>???</strong></span>)</span>.</p>
<p>The elimination is direct killing of cancer cells or growth inhibition by immune system. The adoptive T cells and NK are actively involved in tumor killing and stimulate other immune cells. The CD8 + cytotoxic lymphocytes (CTLs) directly recognize tumor cells. Employing perforin- and granzyme-dependent mechanisms they can lyze tumor cells. The CD4 + T cells release factors to induce proliferation of B cells and to promote their differentiation to antibody (Ab)-secreting plasma cells, activate macrophages. Macrophages use phagocytosis to eliminate cancer cells <span class="citation">(<span class="citeproc-not-found" data-reference-id="Vesely2011"><strong>???</strong></span>)</span>.</p>
<p>The tumor-infiltrating lymphocytes (TILs) have been associated with an overall good prognosis and better survival in different cancer studies. Also, abundance of CD3 + and CD8 + T cells, NK cells, and <span class="math inline">\(\gamma\delta\)</span>T cells correlates with improved outcomes in epithelial ovarian cancers <span class="citation">(<span class="citeproc-not-found" data-reference-id="Marquez-Medina2012"><strong>???</strong></span>)</span>. Several studies report that the presence of the abundant immune infiltrate is correlated with good prognosis or better survival <span class="citation">(<span class="citeproc-not-found" data-reference-id="Kornstein1983"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Baxevanis1994"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Naito1998"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Pages2005"><strong>???</strong></span>)</span>. Spontaneous regression of human tumors has been reported in cutaneous melanoma, retinoblastoma, osteosarcoma, etc. <span class="citation">(<span class="citeproc-not-found" data-reference-id="Aris2012"><strong>???</strong></span>)</span>.</p>
<p>The equilibrium is the phase when cancer and immune cells coexist and their crosstalk is preventing metastasis.</p>
<p>T cells are the main actor maintaining the equilibrium. Progressively, the tumor cells become more immunogenic as they are not edited by the immune system <span class="citation">(<span class="citeproc-not-found" data-reference-id="Bahatia2011"><strong>???</strong></span>)</span>. The state of tumor cells is then identified as “dormant” and active scientific reports investigate the possible molecular pathways that maintain dormancy or lead to escape <span class="citation">(<span class="citeproc-not-found" data-reference-id="Teng2008"><strong>???</strong></span>)</span>.</p>
<p>The immune escape is the final process when tumor cells impair the immune response.</p>
</div>
</div>
<div id="two-faced-nature-of-immune-cells-context-dependent-functional-plasticity" class="section level4">
<h4><span class="header-section-number">1.1.2.2</span> Two-faced nature of immune cells: context-dependent functional plasticity</h4>
<p>Modern vision of TME-tumor interactions assumes that tumor can be directed to several molecular pathways. This direction is decided by signals that are native of tumor cell and/or coming from the microenvironment.</p>
<p>Recent studies unveil ambivalent nature of immune cells in TME. While some as cytotoxic T cells, B cells and macrophages can manage to eliminate tumor cells. Treg cells role is to regulate expansion and activation of T and B cells. Depending on cancer type, they can be either pro- or anti-tumor. For example, as it has been shown for T-regs, usually associated with bad prognosis, they can be associated with improved survival (i.e. in colorectal cancer <span class="citation">(Frey et al. <a href="#ref-Frey2010">2010</a>)</span>). For innate immunity, there are widely accepted M1 (ani-tumor) and M2 (pro-tumor) extreme macrophages phenotypes in TME <span class="citation">(Qian and Pollard <a href="#ref-Qian2010">2010</a>)</span>. Most of the statements seem to be context dependent and not valid universally across all cancer types. We already mentioned Macrophages phenotypic plasticity as well as different behaviour of EMC depending on tumor stage.</p>
<p>From more general point of view, it has been observed that immunodeficiency can correlate with high cancer incidence. Results of analysis based on observations of 25,914 female immunosuppressed organ transplant recipients, the tumor incidence was higher than predicted for multiple cancers. However, the number of breast cancer cases decreased which can be really disturbing if we need to decide on the role of immune defence in tumor progression <span class="citation">(Stewart et al. <a href="#ref-Stewart1995">1995</a>)</span>. This indicates that immune microenvironment can be cancer stimulating or inhibiting depending on the type of cancer and/or other factors.</p>
</div>
<div id="immune-cell-subtypes-in-tme" class="section level4">
<h4><span class="header-section-number">1.1.2.3</span> Immune cell (sub)types in TME</h4>
<blockquote>
<p>Cette espèce de leucocytes a une grande ressemblance avec certains éléments fixes du tissu conjonctif, ainsi qu’avec des cellules endothéliales et des cellules de la pulpe splénique. On est donc souvent embarrassé, surtout lorsqu’on trouve ces leucocytes mononucléaires en dehors des vaisseaux, pour les distinguer des autres espèces de cellules mentionnées.</p>
<p>​ — Elie Metchnikoff, Leçons sur la pathologie comparée de l’inflammation, 1891</p>
</blockquote>
<p>Pathological characterisation</p>
<p>FACS molecular charaterisation</p>
<p>Necessary for the chapter <a href="map.html#map">Heterogeneity of immune cell types</a></p>
</div>
<div id="summary" class="section level4">
<h4><span class="header-section-number">1.1.2.4</span> Summary</h4>
<p>Cancer is a disease concerning milliards of people with a long history. Scientific community recognises role of the environment where the tumor cells find themselves as an important factor influencing tumor development, prognosis and response to treatment. TME is a complex environment that constantly interacts with tumor cells, where both tumor and TME influence and shape each other.</p>
<p>Over the years, many interactions are being discovered and cell types re-defined and described in their context. However, lots of mechanisms and interactions of TME remains unknown due to very heterogeneous nature of this micro environment. This leaves room to more extensive investigation of TME.</p>
<p>A therapeutic goal are target interactions that would be able to pivot the essential processes in tumorigenesis or tumor escape in order to put the cells “back on track” and facilitate anti-tumor therapies.</p>
<p>These goals can be meet thanks to improvement of investigating techniques and data quality and abundance. We will discuss the most important data types used in this project to investigate the TME.</p>
</div>
</div>
</div>
<div id="quantifying-and-qualifying-immune-infiltration-data" class="section level2">
<h2><span class="header-section-number">1.2</span> Quantifying and qualifying immune infiltration (data)</h2>
<p>Nowadays, more and more biological data is produced. However, this proliferation of accessible resources is not proportional to generated insights and wisdom. In this thesis, we wok mostly generate <em>Knowledge</em> and <em>Insights</em> and we hope to generate some <em>Wisdom</em> (Fig. <a href="intro.html#fig:information-power">1.4</a>). In this section, we will introduce the foundation of our analysis: different data types that will be further discussed and explored in chapters that follow.</p>
<div class="figure" style="text-align: center"><span id="fig:information-power"></span>
<img src="figures-ext/01-Information_power.jpg" alt="From Data to Wisdom. Illustration of different steps that it takes to go from Data to generating Wisdom. It highlights that generating data is not equal to understanding it and additional efforts are needed to generate value. Image authored by Clifford Stoll and Gary Schubert published by Portland Press Limited on behalf of the Biochemical Society and the Royal Society of Biology and distributed under the Creative Commons Attribution License 4.0 (CC-BY) in (Ponting 2017)." width="80%" />
<p class="caption">
Figure 1.4: <strong>From <em>Data</em> to <em>Wisdom</em></strong>. Illustration of different steps that it takes to go from <em>Data</em> to generating <em>Wisdom</em>. It highlights that generating data is not equal to understanding it and additional efforts are needed to generate value. Image authored by Clifford Stoll and Gary Schubert published by Portland Press Limited on behalf of the Biochemical Society and the Royal Society of Biology and distributed under the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License 4.0 (CC-BY)</a> in <span class="citation">(Ponting <a href="#ref-Ponting2017">2017</a>)</span>.
</p>
</div>

<p>We will introduce most relevant data types that are used to study immune infiltration of tumors.</p>
<div id="facs" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Cell sorting</h3>
<div id="flow-cytometry" class="section level4">
<h4><span class="header-section-number">1.2.1.1</span> Flow cytometry</h4>
<p>Flow cytometry is a laser-based technology. It uses marker genes: cell surface proteins to sort cells in different compartments. Nowadays, it permits quantification of the abundance of up to 17 cell surface proteins using fluorescently labelled antibodies <span class="citation">(Papalexi and Satija <a href="#ref-Papalexi2017">2017</a>)</span>. However this techniques is not free from bias, our knowledge about cell markers is limited and several markers may not be relevant in some context. Moreover, the scientific community did not clearly agree on the marker choice even for popular and well studied cell types which introduced additional heterogeneity when independent studies are compared. Also the quality of antibodies may influence the results of the FACs analysis. Besides those limitations FACs remains quite popular method for analysing cells in complex tissues. It was among first methods that allowed molecular phenotyping of immune cells, a discovery of numerous subsets and thier further functional interpretation.</p>
</div>
<div id="mass-cytometry" class="section level4">
<h4><span class="header-section-number">1.2.1.2</span> Mass cytometry</h4>
<p>Mass cytometry (also known as CyTOF allows for the quantification of cellular protein levels by using isotopes. It allows to quantify up to 40 proteins per cell <span class="citation">(Papalexi and Satija <a href="#ref-Papalexi2017">2017</a>)</span>. It also demands lower starting number of cells (1000 - 1000000), a realistic number that can be extracted from patient biopsy <span class="citation">(<span class="citeproc-not-found" data-reference-id="Lyons2017"><strong>???</strong></span>)</span>.</p>
</div>
</div>
<div id="staining" class="section level3">
<h3><span class="header-section-number">1.2.2</span> Microscope Staining</h3>
<p>Using microscope technics, histopathological cuts are analysed. The number of cells per a unit of area (i.e. mm<span class="math inline">\(^2\)</span>) is defined either manually by human or though diverse image analysis algorithms. Current pathology practice utilises chromogenic immunohistochemistry (IHC) <span class="citation">(NPR <a href="#ref-RamosVara2010">2010</a><a href="#ref-RamosVara2010">a</a>)</span>. Multiplexed approaches allow to identify multiple markers in the same histopathology cut. Modern techniques as imaging mass cytometry using FFPE tissue samples uses fluorescence and mass cytometry to identify and quantify marker proteins <span class="citation">(Giesen et al. <a href="#ref-Giesen2014">2014</a>)</span>.</p>
<p>The main advantage of aforementioned technics the number of cells that can be analysed and the information about spatial distribution of the different cell types. The limiting factor, as for <a href="intro.html#facs">cell sorting methods</a>, is the number of markers (~10-100) and consequently number of cell types that can be identified <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>.</p>
<p>The <a href="intro.html#facs">cell sorting methods</a> and <a href="intro.html#staining">microscope staining</a> are usually considered as a gold standard for multidimensional data techniques. The reason why they are not applied at large scale is the cost but also quite laborious and time consuming sample preparation demanding a fresh sample. In contrast, the <a href="intro.html#omics">-omics methods</a> propose more scalable way to measure tumor micro environment.</p>
</div>
<div id="omics" class="section level3">
<h3><span class="header-section-number">1.2.3</span> omics</h3>
<ul>
<li>Some kind of sequencing explanation needed for non-biologists</li>
</ul>
<div id="transcriptome" class="section level4">
<h4><span class="header-section-number">1.2.3.1</span> Transcriptome</h4>
<p>Transcriptomics measures the number of counts of mRNA molecules using high-throughput techniques. mRNA is the part of genetic information that should be translated to proteins. It reflects the activity of ongoing processes in a cell. In contrast to DNA, mRNA is highly variant <span class="citation">(Velculescu et al. <a href="#ref-Velculescu1997">1997</a>)</span>. This variability can be either “intrinsic” that reflect the stochastic process or “extrinsic” reflecting impact of factors upstream to mRNA synthesis <span class="citation">(<span class="citeproc-not-found" data-reference-id="Satija2014"><strong>???</strong></span>)</span>.</p>
<p>In addition, many genetic and epigenetic events can be either directly observed or indirectly inferred from transcriptomic data. Transcriptome can be measure with microarrays or RNA-seq NGS technology.</p>
<p>Bulk transcriptome data are quite accessible nowadays. They can be obtained from either flash-frozen or formalin-fixed, paraffin-embedded (FFPE) tissue samples, including both surgically resected material and core needle biopsies <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>.</p>
<p>The main flaw of transcriptomic data is that the reproducibility between different platforms is limited. As a result, direct comparison between two datasets produced by different platforms is not advised.</p>
<p>Different strategies can be adapted to analyse bulk transcriptome.</p>
<p><span class="citation">Cieślik and Chinnaiyan (<a href="#ref-Cieslik2017">2017</a>)</span> describes five groups of most popular approaches that can be applied to study transcriptome (Fig. <a href="intro.html#fig:transcriptome-methods">1.5</a>). Despite a diversity of bioinformatic and statistical tools, the most popular differential approaches, mainly differential gene expression (DGE) based on difference between two experimental conditions.</p>
<div class="figure" style="text-align: center"><span id="fig:transcriptome-methods"></span>
<img src="figures-ext/transcriptome-methods.png" alt="Five categories of RNA-seq data analysis. Differential analyses: comparing two (or more) conditions, Relative analyses: comparing to an internal refernce (average, base level), Compositional analyses: inferring cell types or groups of cell types (i.e. tumor purity), Global analyses: pan-tissue and pan-cancer analyses and Integrative analyses: compiling heterogneious data types. Reprinted by permission from Springer Nature (Cieślik and Chinnaiyan 2017) © 2018 Macmillian Publishers Limited, part of Springler Nature. All rights reserved." width="100%" />
<p class="caption">
Figure 1.5: <strong>Five categories of RNA-seq data analysis.</strong> Differential analyses: comparing two (or more) conditions, Relative analyses: comparing to an internal refernce (average, base level), Compositional analyses: inferring cell types or groups of cell types (i.e. tumor purity), Global analyses: pan-tissue and pan-cancer analyses and Integrative analyses: compiling heterogneious data types. Reprinted by permission from Springer Nature <span class="citation">(Cieślik and Chinnaiyan <a href="#ref-Cieslik2017">2017</a>)</span> © 2018 Macmillian Publishers Limited, part of Springler Nature. All rights reserved.
</p>
</div>

<p>RNA-seq data was proven to be a useful indicator for clinical applications <span class="citation">(Mody et al. <a href="#ref-Mody2015">2015</a>; Oberg et al. <a href="#ref-Oberg2016">2016</a>; Robinson et al. <a href="#ref-Robinson2017">2017</a>)</span>. Its utility for immune profiling was demonstrated in may studies through a use of transcriptomic signatures to predict immunotherapy response or survival <span class="citation">(Chen et al. <a href="#ref-Chen2016">2016</a>)</span>.</p>
<p>In this work transcriptome data analysis fails into multiple categories: Compositional, Relative and aims to construct a Global-level conclusions.</p>
</div>
<div id="epigenome" class="section level4">
<h4><span class="header-section-number">1.2.3.2</span> Epigenome</h4>
<p>An epigenome can be defined as a record of the chemical changes to the DNA and histone proteins of an organism. Changes to the epigenome can provoke changes to the structure of chromatin and changes to the function of the genome <span class="citation">(Bernstein, Meissner, and Lander <a href="#ref-Bernstein2007">2007</a>)</span>. Epigenome data usually contains information about methylation CpG island changes. In cancer, global genomic hypomethylation, CpG island promoter hypermethylation of tumor suppressor genes, an altered histone code for critical genes, a global loss of monoacetylated and trimethylated histone H4 were observed. Methylome profiles can be also use as molecular signature <span class="citation">(<span class="citeproc-not-found" data-reference-id="Jeschke2017"><strong>???</strong></span>)</span>.</p>
</div>
<div id="single-cell-rna-seq" class="section level4">
<h4><span class="header-section-number">1.2.3.3</span> Single cell RNA-seq</h4>
<p>Described above methods of process DNA from hundreds of thousands of cells simultaneously and report averaged gene expression of all cells. In contrast, scRNA-seq technology allows getting results for each cell individually. This is tremendous step forward enhancement of our understanding of cell heterogeneity and opens new avenues of research questions.</p>
<p>Continuous discovery of new immune subtypes has proven that cell surface markers that are used for phenotyping by techniques like <a href="intro.html#facs">FACS</a> and <a href="intro.html#staining">immunohistochelistry</a> cannot capture the full complexity. ScRNA-seq methods allow to cluster known cell types in subpopulations based on their genetic features. ScRNA-seq is also able to capture particularly rare cell types as it requires much less of RNA material (1 ng isolated from 100-1000 cells) compared to ‘bulk’ RNA-seq ( ~ 1 μg of total mRNA transcripts ). It also allows to study cells at high resolution capturing the phenotypes in much more refined scale than previously <span class="citation">(Papalexi and Satija <a href="#ref-Papalexi2017">2017</a>)</span>.</p>
<p>This new data type also brings into the field new challenges related to data processing due to the volume, distribution, noise, and biases. Experts highlight as the most “batch effect”, “noise” and “dropout effect” <span class="citation">(Perkel <a href="#ref-Perkel2017">2017</a>)</span>. So far, there are no official standards that can be applied which makes data comparison and post-processing even more challenging. Up to date, there are around 70 reported tools and resources for single cell data processing <span class="citation">(Davis <a href="#ref-Davis2016">2016</a>)</span> .</p>
<p>A limited number of single-cell datasets of tumors are made publicly available (@ TABLE ?).</p>
<p>One can ask why then developing computational deconvolution of bulk transcriptome if we can learn relevant information from single-cell data. Firstly, that single cell data do not provide a straightforward answer to the estimation of cell proportions. The coverage is not full and sequenced single cells are not fully representative of the true population. For instance, neutrophiles are not found in scRNA-seq data because of they are “difficult to isolate, highly labile <em>ex vivo</em> and therefore difficult to preserve with current single-cell methods” <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>. In addition, a number of patients included in published studies of range &lt;100 cannot be compared to thousand people cohorts sequenced with bulk transcriptome methods. This is mostly because single cell experiments are challenging to perform, especially in clinical setting as fresh samples are needed <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span>. Today, single cell technology brings very interesting “zoom in” perspective, but it would be incautious to make fundings from a restricted group of individuals universal to the whole population. Major brake to the use of single cell technology more broadly might be as well the price that is nearly 10x higher for single cell sample compared to bulk <span class="citation">(Core <a href="#ref-Cedar2018">2016</a>)</span>.</p>
<table>
<thead>
<tr class="header">
<th align="center">Technology</th>
<th align="center">Price</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">scRNA-seq</td>
<td align="center">3000$ / sample</td>
</tr>
<tr class="even">
<td align="center">RNA-seq</td>
<td align="center">200 $ / sample</td>
</tr>
<tr class="odd">
<td align="center">FACS</td>
<td align="center">0.05$ / cell</td>
</tr>
<tr class="even">
<td align="center">CyTOF</td>
<td align="center">35$/cell</td>
</tr>
</tbody>
</table>
<p>In this work, we are using single cell data in two ways. Firstly, in <a href="results.html#results">Chapter 5</a> we compare immune cell profiles defined by scRNA-seq, blood and blind deconvolution (problem introduced in <a href="intro.html#immune-signatures">Immune signatures section</a>). Secondly, in <a href="map.html#map">Chapter 6</a> we use single call data of Metastatic melanoma generated by <span class="citation">Tirosh et al. (<a href="#ref-Tirosh2016">2016</a>)</span> to demonstrate heterogeneity of subpopulations of Macrophages and NK cells.</p>
</div>
</div>
</div>
<div id="immunotherapies" class="section level2">
<h2><span class="header-section-number">1.3</span> From cancer phenotyping to immune therapies</h2>
<p>This section outlines different methods of cancer immune phenotyping and progress in cancer therapies with a focus on immune therapies. It will link the ongoing research on TME with therapeutical potential.</p>
<div id="cancer-immune-phenotypes" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Cancer immune phenotypes</h3>
<p>Since 20. century physicians decided on common nomenclature that classify tumors into distinct groups that are relatively homogenous or that share common characteristic important for treatment and prognosis. Tumor typing should help to better asses predicting prognosis, to adapt a therapy to the clinical situation, to enable therapeutic studies which are essential in proving any therapeutic progress.</p>
<p>Most of the classifications are based on clinical data. Most common factors taken into account are: the degree of local invasion, the degree of remote invasion, histological types of cancer with specific grading for each type of cancer, possibly various tumour markers, general status of the patient.</p>
<p>However, cancers with similar morphological and histopathological features reveal very distinct patterns of progression and response to therapy <span class="citation">(<span class="citeproc-not-found" data-reference-id="Galon2014"><strong>???</strong></span>)</span>. In the era of gene sequencing, gene and protein expression as well as epigenome can provide an important complementary information. Therefore gene markers or proteomic abnormalities can integrated into classification panel. One popular example is a gene signature <em>PAM50</em> <span class="citation">(<span class="citeproc-not-found" data-reference-id="Parker2009"><strong>???</strong></span>)</span> used for prediction of patients’ prognosis in breast cancer, patented as a tumor profiling test.</p>
<p>Since the increase of importance of the immunotherapies, researches proposed several ways to classify tumors based on their microenvironment. Given different parameters describing TME, cancers can be sorted into groups that show similar characteristics. We will discuss most common frameworks that allow to phenotype cancers based on the TME.</p>
<p>The localisation of the immune cells can be an indicator of the state and response to the therapy <span class="citation">(Bindea et al. <a href="#ref-Bindea2013">2013</a>)</span>.</p>
<p>The most standard approach is to convey an analysis of histopathological cuts to asses the number of infiltrating lymphocytes (TILs). Two typical patterns are usually identified: “hot” - immune inflamed and “cold” - no active immune response <span class="citation">(Berghoff, Kather, and Jäger <a href="#ref-Berghoff2018">2018</a>)</span>.</p>
<p><span class="citation">Chen and Mellman (<a href="#ref-Chen2017">2017</a>)</span> describes classification into inflamed and non-inflamed tumors, where non-inlamed phenotypes: can be further split into the immune-desert phenotype and the immune–excluded phenotype (Fig. <a href="intro.html#fig:immune-phenotypes">1.6</a>). The inflamed phenotype is characterised by rich presence of immune cells : T cells, myeloid cells, monocytes in tumor margin. Along with the immune cels, due to their communication, a high expression of cytokines is characteristic for this phenotype. According to Chen2017, this is a mark of anti-tumor response that was arrested by tumor. The inflamed phenotype has shown to be most responsive to immunotherapies. In the immune-excluded phenotype, the immune cells are present as well but located in the stroma <span class="citation">(Herbst et al. <a href="#ref-Herbst2014">2014</a>)</span>, sometimes penetrating inside tumor. However, when exposed to check point immuotherapy, T cells does not gain the ability to infiltrate the tumor, therefore the treatment is inefficient. The immune-desert main features is little or no presence of immune cells, especially T cells. Surprisingly, this tumors have been to proven to rarely respond to the checkpoint therapy <span class="citation">(Herbst et al. <a href="#ref-Herbst2014">2014</a>)</span>. In non-inflamed tumours cytokines associated with immune suppression or tolerance are expressed.</p>
<div class="figure" style="text-align: center"><span id="fig:immune-phenotypes"></span>
<img src="figures-ext/immune-phenotypes.png" alt="Cancer-immune phenotypes: the immune-desert phenotype (brown), the immune–excluded phenotype (blue) and the inflamed phenotype (red). The immune–desert phenotype is characterised by paucity of immune cells and cytokines. In the immune-excluded phenotypes the T cells are often present but trapped in stroma, enabled to migrate to the tumor site. The immune-inflamed phenotype is rich in immune cells and the most responsive to the immune check point therapies. Reprinted by permission from Springer Nature (Chen and Mellman 2017) © 2017 Macmillian Publishers Limited, part of Springler Nature. All rights reserved." width="100%" />
<p class="caption">
Figure 1.6: <strong>Cancer-immune phenotypes: the immune-desert phenotype (brown), the immune–excluded phenotype (blue) and the inflamed phenotype (red).</strong> The immune–desert phenotype is characterised by paucity of immune cells and cytokines. In the immune-excluded phenotypes the T cells are often present but trapped in stroma, enabled to migrate to the tumor site. The immune-inflamed phenotype is rich in immune cells and the most responsive to the immune check point therapies. Reprinted by permission from Springer Nature <span class="citation">(Chen and Mellman <a href="#ref-Chen2017">2017</a>)</span> © 2017 Macmillian Publishers Limited, part of Springler Nature. All rights reserved.
</p>
</div>

<p><span class="citation">(<span class="citeproc-not-found" data-reference-id="Thorsson2018"><strong>???</strong></span>)</span> performed a multi-omic analysis of TCGA datasets that allowed them to define 6 subtypes that are valid across cancer types:</p>
<ol style="list-style-type: decimal">
<li>wound healing</li>
<li>IFN-<span class="math inline">\(\gamma\)</span> dominant</li>
<li>inflammatory</li>
<li>lymphocyte depleted</li>
<li>immunologically quiet</li>
<li>TGF-<span class="math inline">\(\beta\)</span> dominant</li>
</ol>
<p>Different indicators were used to define these six phenotypes:</p>
<p>The immunogenicity of the tumors can be explained by tumor-intrinsic factors and tumor-extrinsic factors <span class="citation">(Gajewski et al. <a href="#ref-Gajewski2006">2006</a>)</span>. Tumor-intrinsic factors are, the neoantigen load and frequency, the mutational load, the expression of immunoinhibitors and immunostimulators (e.i. PD-L1), and alteration of HLA class I molecules. Tumor-extrinsic factors include chemokines regulating T cell trafficking, infiltration of effector TILs and immunosuppressive TILs, and soluble immunomodulatory factors (cytokines).</p>
<p>A plethora of different computational methods have been developed in order to further characterize tumor samples using different factors. We mention here mainstream methods that cover different approaches of scoring the TME-cancer phenotype.</p>
<div id="immunoscore" class="section level4">
<h4><span class="header-section-number">1.3.1.1</span> Immunoscore</h4>
<p>One of the most recognised scoring method, based on fluorescent images is authored by Jerôme Galon lab in Paris and names <a href="http://www.haliodx.com/clinical-research-services/immunoscorer/">Immunoscore</a>. The Immunoscore ranges from 0 to 4 and it is based on the density of lymphocyte populations CD3/CD45RO, CD3/CD8, or CD8/ CD45RO. It also takes into account the spacial position of the cells: the tumor core and margins <span class="citation">(Galon et al. <a href="#ref-Galon2012">2012</a>)</span>. It was successfully applied to colorectal cancer to predict patients’ survival <span class="citation">(<span class="citeproc-not-found" data-reference-id="Anitei2014"><strong>???</strong></span>)</span> . Since it resulted in numerous application to many cancer types. It was also linked to time-to-recurrence in an international study. There is great potential to use immunoscore as a predictive marker if validated in prospective studies <span class="citation">(<span class="citeproc-not-found" data-reference-id="Galon2016"><strong>???</strong></span>)</span>.</p>
<p>The immunoscore is an interesting indicator, especially in the scope of clinical applications, although it does not tell us a lot about underlying biology. It is also limited to a few cell types while it may be that in some cancer types or patients, the system requires more detailed or rich analysis of larger panel of cells.</p>
</div>
<div id="immunophenoscore" class="section level4">
<h4><span class="header-section-number">1.3.1.2</span> Immunophenoscore</h4>
<p>Different approaches are based on gene expression patterns. Most commonly, machine learning supervised algorithms are trained to match known phenotype (established with microscopy or with clinical features) to genetic patterns or an unsupervised clustering is used to discover new classification.</p>
<p>An example of well-formulated classification framework is Immunophenoscore <span class="citation">(Charoentong et al. <a href="#ref-Charoentong2017">2017</a>)</span>, based on publication of <span class="citation">Angelova et al. (<a href="#ref-Angelova2015">2015</a>)</span>, where methylome, transcriptome and mutation of TCGA CRC dataset (n = 598) was used to describe <em>immunophenotypes</em>. Later on, it was reduced to gene expression indicator and summarised in a form of a score. In This scoring scheme is based on the data of 20 solid tumors, using expression of marker genes selected by a Machine Learning algorithm (random forest) for best prediction in each cancer. These indicators can be grouped into four categories:</p>
<ul>
<li>MHC molecules (MHC)</li>
<li>Immunomodulators (CP)</li>
<li>Effector cells (EC)</li>
<li>Suppressor cells (SC)</li>
</ul>
<p>The immunophenscore (IPS) is calculated on a 0-10 scale based on the expression of genes in each category. Stimulatory factors (cell types) impact the score positively and inhibitory factors (cell types) negatively. Z-scores ≥ 3 were designated as IPS10 and z-scores ≤ 0 are designated as IPS0. A similar conceptual framework called <em>cancer immunogram</em> was proposed by <span class="citation">Blank et al. (<a href="#ref-Blank2016">2016</a>)</span> included seven parameters: tumor foreignness (Mutational load), general immune status (Lymphocyte count), immune cell infiltration (Intratumoral T cells), absence of checkpoints (PD-L1), absence of soluble inhibitors (IL–6, CRP), absence of inhibitory tumor metabolism (LDH, glucose utilisation), tumor sensitivity to immune effectors (MHC expression, IFN-γ sensitivity). <span class="citation">Charoentong et al. (<a href="#ref-Charoentong2017">2017</a>)</span> claim that the immunophenoscore can predict response to CTLA-4 and anti-PD-1.</p>
<p>Nonetheless, the details of <em>cancer immunogram</em> use in practice remain unclear and result could be sensitive to patients’ and data heterogeneity as no standardisation was proposed. It should be also validated in a systematic independent study.</p>
</div>
<div id="immune-maps" class="section level4">
<h4><span class="header-section-number">1.3.1.3</span> Immune maps</h4>
<p>Another way to summarize tumor phenotype can be though use of molecular maps. <a href="https://acsn.curie.fr/">Atlas of Cancer Signaling Network (ACSN)</a> <span class="citation">(<span class="citeproc-not-found" data-reference-id="Kuperstein2013"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Kuperstein2015"><strong>???</strong></span>)</span> is primarily a pathway database that contains a collection of interconnected cancer-related signalling network maps. An additional feature is ACSN web-based goole-maps like visualisation of the database. User data can be projected on the molecular map (for example gene/protein expression from user data can be paired with entities on the map ). Currently it contains <a href="https://navicell.curie.fr/">11 maps</a> covering signalling processes involved in apoptosis, cell cycle, DNA repair, EMT, cell motility, Ewing sarcoma, Dendritic cells, Macrophage cells, Natural killer cells, CAF and integrated innate immunity map. Through projection of the data on the innate immunity map, one can see if the patient or a sample is characterised by pro- or anti-tumor activated pathways due to the organisation of the map layout. Also, different CAF subtypes were characterised with the CAF specific map in <span class="citation">(<span class="citeproc-not-found" data-reference-id="Costa2018"><strong>???</strong></span>)</span>. Kondratova and colleagues (including myself) used innate immune map to characterize NK and Macrophages subtypes and in another publication <span class="citation">(<span class="citeproc-not-found" data-reference-id="Kondratova2018"><strong>???</strong></span>)</span> different patient profiles using scRNAseq data of Metastatic melanoma.</p>
</div>
<div id="summary-1" class="section level4">
<h4><span class="header-section-number">1.3.1.4</span> Summary</h4>
<p>Despite those facts, the gene expression based classifications are not yet used in clinics. The measured multi-panel mRNA expression, that can be included into category of In Vitro Diagnostic Multivariate Index Assay (IVDMIA) <span class="citation">(Győrffy et al. <a href="#ref-Gyorffy2015">2015</a>; Ross et al. <a href="#ref-Ross2008">2008</a>)</span>, may be a future of TME-based cancer classification, diagnosis and treatment recommendation <span class="citation">(Gnjatic et al. <a href="#ref-Gnjatic2017">2017</a>)</span>. For this best tools need to be used to properly evaluate the state of TME and tumor-stroma-immune cells communication.</p>
</div>
</div>
<div id="immune-signatures" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Immune signatures - biological perspective</h3>
<ul>
<li>definition of signature: marker genes, list of genes, weighted list, metagenes</li>
<li>the general immune signature of signature of immune infiltration and stroma vs immune signature of a specific cell type of functional subpopulation</li>
<li>purpose of signatures</li>
<li>availability of immune signatures</li>
<li>the problem of not consistency of immune signatures</li>
<li>origin of signatures</li>
</ul>
<blockquote>
<p><em>the gene expression profiles of tumour-associated immune cells differ considerably from those of blood derived immune cells</em> <span class="citation">(Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span></p>
</blockquote>
<p>Immune signatures will be also discussed as a part of deconvolution pipeline in the <a href="methods.html#methods">Chapter 2</a> under the section about <em>basis matrix</em>.</p>
<p>———</p>
<p>Gene expression signatures</p>
<p>Lack of immune-specific signatures. Gene expression signatures are used as tools to predict which patients will benefit from certain treatments and to guide decision-making in the clinic.46</p>
<p>Gene expression signatures are considered “prognostic” when they can differentiate between patients with a good or bad prognosis in the context of traditional therapy or no treatment at all.47 Gene signatures are considered “predictive” when they are able to predict treatment benefit between experimental and/or nontraditional treatment groups vs. control, typically in the setting of a randomized controlled clinical trial.47 In breast cancer, the predictive and prognostic assays Oncotype DX, EndoPredict, PAM50, and Breast Cancer Index are now recommended as adjuncts for clinical decision-making for patients with specific subtypes of breast cancer.46 These signatures, however, are not specifically immune-related and share little overlap in their selected genes.47, 48 Because tumor cells and infiltrating immune cells both have prognostic value, evaluating tumors and the surrounding stroma with use of the methods described above, in order to generate immune-related signatures, can provide prognostic and perhaps predictive information associated with patient outcome. Although there are no immune-related gene signatures used in clinical practice currently, several studies have shown the validity and reproducibility of using immune-related signatures to predict outcome and response to therapy in patient cancer samples.</p>
<p>Solution: in silico, pan-cancer studies. One reason immune-related gene signatures have not been widely used in clinical settings is the lack of consistency of genes, both within the same tumor type and among different tumors.49 Efforts to address this problem are under way, with the development of conserved immune gene signatures representing multiple tumor types.49 Another problem with immune-related gene signatures is the difficulty in decipher- ing whether the genes represent specific immune cell popula- tions, antigen-presenting machinery, cytokines, or other immune- related molecules. In silico studies have validated pan-cancer immune-related gene signatures that represent many immune cell types with known prognostic significance including T cells, B cells, NK cells, macrophages, as well as various cytokine populations.49, 50 Immune-related gene signatures have proven to be prognostic of clinical outcome across multiple cancer types and are associated with response to immune therapy in specific cancers.49, 50 Recent findings on immune-related gene signatures have shown that gene signatures are able to distinguish between broad lineages of cell type, such as lymphoid and myeloid but are less adept at distinguishing between more differentiated cells such as CD8+ and CD4+ T cells.49</p>
<p>Ranked gene lists</p>
<p>Oversimplification of interpretation. Currently, given the multi-</p>
<p>tude of methods with which to analyze genomic data, the</p>
<p>problem no longer lies in obtaining gene expression profiles but</p>
<p>rather in interpreting the data. Analysis of gene expression data</p>
<p>typically involves a comparison of two groups and the creation of</p>
<p>a ranked gene list based on expression. One method of</p>
<p>interpretation involves looking at each end of the ranked list to</p>
<p>7, 51</p>
<p>Quantity of data. The NCBI Gene Expression Omnibus database contains more than 30,000 series and 1 million samples of array- based expression data, many of which involve the immune system.52 Because of the large amount of expression data relating to the immune system and the inability to compare information between the various datasets, the Immunological Genome Project (IGP) was created to provide a comprehensive compendium of the expression of protein-coding genes for all immune cell popula- tions in the mouse immune system.53 This project was developed to provide information on primary immune cells isolated ex vivo from the mouse immune system and then integrate that information into networks, while taking into account variation via genetic polymorphisms, knock-out of genes, knockdown via RNA interference, and drug treatment.53 Researchers can use these data to determine gene expression profiles that serve as biomarkers for predicting response to treatment and for measur- ing residual disease after immune therapy.</p>
<p>Solution 1: added-value/specialty databases. Primary databases contain gene expression and other genomic data such as genotype, DNA methylation, and protein expression data. These</p>
<p>evaluate genes showing the largest difference in expression. Problems with this method include small relevant biological differences between genes leading to no statistical significance, a long list of genes that are statistically significant but lack biologic relevance, missed effects on pathways by analyzing single genes, and variation in gene lists for the same biologic system based on two different groups’ data.51 This again underscores the challenge faced when trying to create a clinically useful immune-related gene signature from a high-throughput panel of genes.</p>
<p><span class="citation">(<span class="citeproc-not-found" data-reference-id="Lyons2017"><strong>???</strong></span>)</span></p>
<p>——-</p>
</div>
<div id="cancer_Therapies" class="section level3">
<h3><span class="header-section-number">1.3.3</span> Cancer therapies</h3>
<p>Cancer is a complex disease. Up to date, no uniform and fully effective treatment was proposed and usually different strategies are tested to kill tumor cells. <strong>Surgery</strong> is one of the oldest methods. The cancer is removed from the patient body. There are different ways, more or less invasive, that it can be performed. it is usually applied for solid tumor contained in a small area. <strong>Radiation Therapy</strong> uses high doses of radiation to eliminate tumor cells and shrink tumor mass. It can be applied externally or internally. <strong>Chemotherapy</strong> uses a drug (or a combination of drugs) that kill cancer cells, usually altering cell proliferation and growth. The drawback of radiotherapy and chemotherapy are strong side effects. <strong>Hormone therapy </strong> modulate hormone levels in the body in order to inhibit tumor growth in breast and prostate cancers. In leukemia and lymphoma, can be applied <strong>stem cell transplants</strong> that restore blood-forming stem cells destroyed by the very high doses of chemotherapy or radiation therapy that are used to treat certain cancers.</p>
<p>Alternatively, <strong>targeted therapies</strong> represent more focused strategy that aims to be more effective and cause less side effects than systematic therapies. Two main types of targeted therapies are small-molecule drugs and monoclonal antibodies. Targeted therapies usually aim to stimulate/inhibit a selected molecular function. A special types of targeted therapies are <strong>Immunotherapies</strong>. Through adtivation/inhibition of immune regulatory pathways, it stimulates immune system to destroy malignant cells. A continuation of targeted therapies is <strong>precision medecine approach</strong>. It is based on genetic information to specify patient’s profile and find adapted treatment. A number of innovative treatments targeting a specific change in tumor ecosystem are being tested presently in precision medicine clinical trials <span class="citation">(Institute <a href="#ref-NCI2018">2017</a>)</span>.</p>
</div>
<div id="recent-progress-in-immuno-therapies" class="section level3">
<h3><span class="header-section-number">1.3.4</span> Recent progress in immuno-therapies</h3>
<p>The immunotherapies, in contrast with other types of cancers therapies discussed in <a href="intro.html#cancer_Therapies">the previous chapter</a>, aim to trigger or restart the immune system to defend the organism and attack the malignant cells. All this, however without provoking persisting inflammation state <span class="citation">(Predina et al. <a href="#ref-Predina2013">2013</a>)</span></p>
<p>The idea of stimulating immune system to fight malignant cell was not born recently. Since a long time a possibility of development of an anti-cancer vaccine has been investigated. Unfortunately, this idea faced two important limitations 1) lack of knowledge of antigens that should be used in vaccine to successfully stimulate cytotoxic T cells 2) the ability of cancer to block the immune response also called <em>immunostat</em>. Despite those impediments works on anti-tumor vaccines do not cease <span class="citation">(Palucka and Banchereau <a href="#ref-Palucka2013">2013</a>)</span>. A very recent promising an in-situ ani-tumor vaccine was proposed by Sagiv-Barfi et al. <span class="citation">(<span class="citeproc-not-found" data-reference-id="Sagiv-Barfi2018"><strong>???</strong></span>)</span>. The therapy tested in mice, would be based on local injections of the combination of “unmethylated CG-enriched oligodeoxynucleotide (Cp-G) - a Toll-like receptor 9 (TLR9) ligand and anti-OX40 antibody. Low doses of CpG injected into a tumor induce the expression of OX40 on CD4+ T cells in the microenvironment in mouse or human tumors. An agonistic anti-OX40 antibody can then trigger a T cell immune response, which is specific to the antigens of the injected tumor”. Sagiv-Barfi et al. claim this therapy could be applied to all tumor types, as long as they are leucocyte-infiltrated. As a local therapy, in situ vaccination should have less side-effects than systematic administration. It is now undergoing clinical trials to test its efficiency in human patients.</p>
<p>Another idea involving using immune system as a weapon to fight cancer, would be the use of genetically modified patient’s T-cells, carrying <em>CARs</em> (chimeric antigen receptors) <span class="citation">(Jackson, Rafiq, and Brentjens <a href="#ref-Jackson2016">2016</a>)</span>. After a long period of small unsuccessful trials, recently in 2017, two CAR T-cell therapies were accepted, one to “treat adults with certain type of large B-cell lymphoma” <span class="citation">(Health and Services <a href="#ref-FDACARTadult">2017</a><a href="#ref-FDACARTadult">b</a>)</span>, other to treat “children with acute lymphoblastic leukemia (ALL)” <span class="citation">(Health and Services <a href="#ref-FDACARTALL">2017</a><a href="#ref-FDACARTALL">a</a>)</span> , which are, at the same time, the first two gene therapies accepted by FDA.</p>
<p>However, the two most promising immuno-related strategies with proven clinical efficiency are based on blocking so called immune check point inhibitors: cytotoxic T-lymphocyte protein 4 (CTLA4) and programmed cell death protein 1 (PD-1). The anti-CLTA4 antibodies blocks repressive action of CLTA4 on T-cells and they become therefore activated. It was shown efficient in melanoma patients and accepted by FDA in 2015 as adjuvant therapy for stage III metastatic melanoma patients <span class="citation">(Health and Services <a href="#ref-FDACTLA4">2015</a>)</span>. PD-1 is a cell surface receptor of T cells, that binds to PD-L1/PD-L2. After binding, an immunosuppressive pathway is activated and T cells activity is dampened. An action of an anti-PD-L1 antibody is to prevent this immune exhaustion <span class="citation">(Chen and Mellman <a href="#ref-Chen2017">2017</a>)</span>. A stepping stone for anti-PD-L1 therapies was approval of Tecentriq (atezolizumab) for Bladder cancer <span class="citation">(Health and Services <a href="#ref-FDAPDL1Bladder">2016</a><a href="#ref-FDAPDL1Bladder">a</a>)</span> and anit-PD1 Keytruda (pembrolizumab) initially accepted for NSCLC and further extended to head and neck cancer, Hodgkin’s lymphoma, gastric cancer and microsatellite instability-high cancer <span class="citation">(Health and Services <a href="#ref-FDAPDL1NSCLC">2016</a><a href="#ref-FDAPDL1NSCLC">b</a>)</span>. Since other anti-PD-L1 or anti-PD1 antibodies were accepted or entered advanced stages of clinical trials <span class="citation">(Wolchok <a href="#ref-Wolchok2015">2015</a>)</span>. A short history of immunotherapy FDA-accepted treatments can be found in Fig. <a href="intro.html#fig:timeline-immunotherapies">1.7</a></p>
<div class="figure" style="text-align: center"><span id="fig:timeline-immunotherapies"></span>
<img src="figures-ext/02-timeline-immunotherapies.png" alt="This timeline describes short history of FDA approval of checkpoint blocking immunotherapies up to 2017. Reprinted by permission from Springer Nature (Taube et al. 2017) Macmillan Publishers Limited, part of Springer Nature. All Rights Reserved." width="100%" />
<p class="caption">
Figure 1.7: <strong>This timeline describes short history of FDA approval of checkpoint blocking immunotherapies up to 2017.</strong> Reprinted by permission from Springer Nature <span class="citation">(Taube et al. <a href="#ref-Taube2017a">2017</a>)</span> Macmillan Publishers Limited, part of Springer Nature. All Rights Reserved.
</p>
</div>

<p>The main drawback of immunotherapies is a heterogeneity of response rate, which can vary i.e. from 10–40% in case of PD-L1blocking <span class="citation">(Zou, Wolchok, and Chen <a href="#ref-Zou2016">2016</a>)</span>, suggesting that some patient can have more chances than others to respond to an immune therapy. So far, it has been shown that anti PD-L1 therapies works more effectively in T cell infiltrated tumors with exclusion of Tregs because of lack of difference in expression of FOXP3 in responding and non-responding group of patients <span class="citation">(Herbst et al. <a href="#ref-Herbst2014">2014</a>)</span>. Also some light has been shade by <span class="citation">Rizvi et al. (<a href="#ref-Rizvi2015">2015</a>)</span> who connected mutational rate of cancer cells to the chances of response to an immunotherapy.</p>
<p>Despite those fundings, the precise qualifications of patients that should be sensitive to an immunotherapy are not defined <span class="citation">(Pitt et al. <a href="#ref-Pitt2016">2016</a>)</span>. As most patients do not answer to immunotherapies, it stimulates researches to look for better biomarkers and patient stratifications, and pharmaceutical industries to discover new immune checkpoints based therapies.</p>
</div>
</div>
<div id="biological-dimension-of-the-thesis" class="section level2">
<h2><span class="header-section-number">1.4</span> Biological dimension of the thesis</h2>
<p>Explaining the biological dimension of the project</p>
<ul>
<li>help to better understand tumor immunophenotypes (cell types present and their biology, interactions)</li>
<li>propose predictive biomarkers - precision medecine</li>
<li>better explore available data</li>
<li>integrate the data (transcriptome &amp; methylome, validate transcriptome with FACS etc.)</li>
</ul>
<!-- You can label chapter and section titles using `{#label}` after them, e.g., we can reference Chapter \@ref(intro). If you do not manually label them, there will be automatic labels anyway, e.g., Chapter \@ref(methods).

Figures and tables with captions will be placed in `figure` and `table` environments, respectively.


```r
par(mar = c(4, 4, .1, .1))
plot(pressure, type = 'b', pch = 19)
```

<div class="figure" style="text-align: center">
<img src="UCzPhDThesis_files/figure-html/nice-fig-1.png" alt="Here is a nice figure!" width="80%" />
<p class="caption">(\#fig:nice-fig)Here is a nice figure!</p>
</div>

Reference a figure by its code chunk label with the `fig:` prefix, e.g., see Figure \@ref(fig:nice-fig). Similarly, you can reference tables generated from `knitr::kable()`, e.g., see Table \@ref(tab:nice-tab).


```r
knitr::kable(
 head(iris, 20), caption = 'Here is a nice table!',
 booktabs = TRUE
)
```

<table>
<caption>(\#tab:nice-tab)Here is a nice table!</caption>
 <thead>
  <tr>
   <th style="text-align:right;"> Sepal.Length </th>
   <th style="text-align:right;"> Sepal.Width </th>
   <th style="text-align:right;"> Petal.Length </th>
   <th style="text-align:right;"> Petal.Width </th>
   <th style="text-align:left;"> Species </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:right;"> 5.1 </td>
   <td style="text-align:right;"> 3.5 </td>
   <td style="text-align:right;"> 1.4 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.9 </td>
   <td style="text-align:right;"> 3.0 </td>
   <td style="text-align:right;"> 1.4 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.7 </td>
   <td style="text-align:right;"> 3.2 </td>
   <td style="text-align:right;"> 1.3 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.6 </td>
   <td style="text-align:right;"> 3.1 </td>
   <td style="text-align:right;"> 1.5 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.0 </td>
   <td style="text-align:right;"> 3.6 </td>
   <td style="text-align:right;"> 1.4 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.4 </td>
   <td style="text-align:right;"> 3.9 </td>
   <td style="text-align:right;"> 1.7 </td>
   <td style="text-align:right;"> 0.4 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.6 </td>
   <td style="text-align:right;"> 3.4 </td>
   <td style="text-align:right;"> 1.4 </td>
   <td style="text-align:right;"> 0.3 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.0 </td>
   <td style="text-align:right;"> 3.4 </td>
   <td style="text-align:right;"> 1.5 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.4 </td>
   <td style="text-align:right;"> 2.9 </td>
   <td style="text-align:right;"> 1.4 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.9 </td>
   <td style="text-align:right;"> 3.1 </td>
   <td style="text-align:right;"> 1.5 </td>
   <td style="text-align:right;"> 0.1 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.4 </td>
   <td style="text-align:right;"> 3.7 </td>
   <td style="text-align:right;"> 1.5 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.8 </td>
   <td style="text-align:right;"> 3.4 </td>
   <td style="text-align:right;"> 1.6 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.8 </td>
   <td style="text-align:right;"> 3.0 </td>
   <td style="text-align:right;"> 1.4 </td>
   <td style="text-align:right;"> 0.1 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 4.3 </td>
   <td style="text-align:right;"> 3.0 </td>
   <td style="text-align:right;"> 1.1 </td>
   <td style="text-align:right;"> 0.1 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.8 </td>
   <td style="text-align:right;"> 4.0 </td>
   <td style="text-align:right;"> 1.2 </td>
   <td style="text-align:right;"> 0.2 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.7 </td>
   <td style="text-align:right;"> 4.4 </td>
   <td style="text-align:right;"> 1.5 </td>
   <td style="text-align:right;"> 0.4 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.4 </td>
   <td style="text-align:right;"> 3.9 </td>
   <td style="text-align:right;"> 1.3 </td>
   <td style="text-align:right;"> 0.4 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.1 </td>
   <td style="text-align:right;"> 3.5 </td>
   <td style="text-align:right;"> 1.4 </td>
   <td style="text-align:right;"> 0.3 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.7 </td>
   <td style="text-align:right;"> 3.8 </td>
   <td style="text-align:right;"> 1.7 </td>
   <td style="text-align:right;"> 0.3 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
  <tr>
   <td style="text-align:right;"> 5.1 </td>
   <td style="text-align:right;"> 3.8 </td>
   <td style="text-align:right;"> 1.5 </td>
   <td style="text-align:right;"> 0.3 </td>
   <td style="text-align:left;"> setosa </td>
  </tr>
</tbody>
</table>

You can write citations, too. For example, we are using the **bookdown** package [@R-bookdown] in this sample book, which was built on top of R Markdown and **knitr** [@xie2015].
 -->

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-GLOBOCAN">
<p>Research in Cancer, International Agency for. 2018. “GLOBOCAN - Facts Sheets by Cancer.” <a href="http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx" class="uri">http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx</a>.</p>
</div>
<div id="ref-Sudhakar2009">
<p>Sudhakar, Akulapalli. 2009. “History of Cancer, Ancient and Modern Treatment Methods.” <em>Journal of Cancer Science &amp; Therapy</em> 1 (2). NIH Public Access: 1–4. doi:<a href="https://doi.org/10.4172/1948-5956.100000e2">10.4172/1948-5956.100000e2</a>.</p>
</div>
<div id="ref-NPR2010">
<p>NPR. 2010b. “Science Diction: The Origin of the Word ’Cancer’ : NPR.” <a href="https://www.npr.org/templates/story/story.php?storyId=130754101" class="uri">https://www.npr.org/templates/story/story.php?storyId=130754101</a>.</p>
</div>
<div id="ref-Corlan2004">
<p>Corlan, Alexandru Dan. 2004. “Medline Trend: Automated Yearly Statistics of Pubmed Results for Any Query.” <a href="http://dan.corlan.net/medline-trend.html" class="uri">http://dan.corlan.net/medline-trend.html</a>.</p>
</div>
<div id="ref-Balkwill2001">
<p>Balkwill, Fran, and Alberto Mantovani. 2001. “Inflammation and cancer: Back to Virchow?” doi:<a href="https://doi.org/10.1016/S0140-6736(00)04046-0">10.1016/S0140-6736(00)04046-0</a>.</p>
</div>
<div id="ref-Talmadge2013">
<p>Talmadge, J E, and D I Gabrilovich. 2013. “History of myeloid-derived suppressor cells.” <em>Nat Rev Cancer</em> 13 (10): 739–52. doi:<a href="https://doi.org/10.1038/nrc3581">10.1038/nrc3581</a>.</p>
</div>
<div id="ref-Gabrilovich2012">
<p>Gabrilovich, Dmitry I., Suzanne Ostrand-Rosenberg, and Vincenzo Bronte. 2012. “Coordinated regulation of myeloid cells by tumours.” doi:<a href="https://doi.org/10.1038/nri3175">10.1038/nri3175</a>.</p>
</div>
<div id="ref-Taube2017a">
<p>Taube, Janis M, Jérôme Galon, Lynette M Sholl, Scott J Rodig, Tricia R Cottrell, Nicolas A Giraldo, Alexander S Baras, et al. 2017. “Implications of the tumor immune microenvironment for staging and therapeutics.” <em>Modern Pathology</em>. doi:<a href="https://doi.org/10.1038/modpathol.2017.156">10.1038/modpathol.2017.156</a>.</p>
</div>
<div id="ref-Dumont2013">
<p>Dumont, Nancy, Bob Liu, Rosa Anna DeFilippis, Hang Chang, Joseph T. Rabban, Anthony N. Karnezis, Judy A. Tjoe, James Marx, Bahram Parvin, and Thea D. Tlsty. 2013. “Breast Fibroblasts Modulate Early Dissemination, Tumorigenesis, and Metastasis through Alteration of Extracellular Matrix Characteristics.” <em>Neoplasia</em> 15 (3): 249–IN7. doi:<a href="https://doi.org/10.1593/neo.121950">10.1593/neo.121950</a>.</p>
</div>
<div id="ref-Quail2013">
<p>Quail, D F, and J A Joyce. 2013. “Microenvironmental regulation of tumor progression and metastasis.” <em>Nat Med</em> 19 (11): 1423–37. doi:<a href="https://doi.org/10.1038/nm.3394">10.1038/nm.3394</a>.</p>
</div>
<div id="ref-Hanahan2012">
<p>Hanahan, Douglas, and Lisa M. Coussens. 2012. “Accessories to the Crime: Functions of Cells Recruited to the Tumor Microenvironment.” <em>Cancer Cell</em> 21 (3): 309–22. doi:<a href="https://doi.org/10.1016/j.ccr.2012.02.022">10.1016/j.ccr.2012.02.022</a>.</p>
</div>
<div id="ref-Hanahan2000">
<p>Hanahan, D, and R A Weinberg. 2000. “The hallmarks of cancer.” <em>Cell</em> 100 (1). Elsevier: 57–70. doi:<a href="https://doi.org/10.1016/S0092-8674(00)81683-9">10.1016/S0092-8674(00)81683-9</a>.</p>
</div>
<div id="ref-Sebeok1976">
<p>Sebeok, Thomas A. 1976. “DISCUSSION.” <em>Annals of the New York Academy of Sciences</em> 280 (1 Origins and E). Blackwell Publishing Ltd: 481. doi:<a href="https://doi.org/10.1111/j.1749-6632.1976.tb25511.x">10.1111/j.1749-6632.1976.tb25511.x</a>.</p>
</div>
<div id="ref-Burnet1970">
<p>Burnet, F M. 1970. “The concept of immunological surveillance.” <em>Progress in Experimental Tumor Research</em> 13: 1–27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4921480" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/4921480</a>.</p>
</div>
<div id="ref-Stutman1974">
<p>Stutman, O. 1974. “Tumor development after 3-methylcholanthrene in immunologically deficient athymic-nude mice.” <em>Science (New York, N.Y.)</em> 183 (4124): 534–6. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4588620" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/4588620</a>.</p>
</div>
<div id="ref-Cavallo2011">
<p>Cavallo, Federica, Carla De Giovanni, Patrizia Nanni, Guido Forni, and Pier-Luigi Lollini. 2011. “No Title” 60 (3). Springer-Verlag. doi:<a href="https://doi.org/10.1007/s00262-010-0968-0">10.1007/s00262-010-0968-0</a>.</p>
</div>
<div id="ref-Frey2010">
<p>Frey, Daniel M., Raoul A. Droeser, Carsten T. Viehl, Inti Zlobec, Alessandro Lugli, Urs Zingg, Daniel Oertli, Christoph Kettelhack, Luigi Terracciano, and Luigi Tornillo. 2010. “High frequency of tumor-infiltrating FOXP3+ regulatory T cells predicts improved survival in mismatch repair-proficient colorectal cancer patients.” <em>International Journal of Cancer</em> 126 (11): 2635–43. doi:<a href="https://doi.org/10.1002/ijc.24989">10.1002/ijc.24989</a>.</p>
</div>
<div id="ref-Qian2010">
<p>Qian, Bin Z., and Jeffrey W. Pollard. 2010. “Macrophage Diversity Enhances Tumor Progression and Metastasis.” doi:<a href="https://doi.org/10.1016/j.cell.2010.03.014">10.1016/j.cell.2010.03.014</a>.</p>
</div>
<div id="ref-Stewart1995">
<p>Stewart, T., S. CJ Tsai, H. Grayson, R. Henderson, and G. Opelz. 1995. “Incidence of de-novo breast cancer in women chronically immunosuppressed after organ transplantation.” <em>The Lancet</em> 346 (8978): 796–98. doi:<a href="https://doi.org/10.1016/S0140-6736(95)91618-0">10.1016/S0140-6736(95)91618-0</a>.</p>
</div>
<div id="ref-Ponting2017">
<p>Ponting, Chris P. 2017. “Big knowledge from big data in functional genomics.” <em>Emerging Topics in Life Sciences</em> 1 (3). Portland Press Journals portal: 245–48. doi:<a href="https://doi.org/10.1042/ETLS20170129">10.1042/ETLS20170129</a>.</p>
</div>
<div id="ref-Papalexi2017">
<p>Papalexi, Efthymia, and Rahul Satija. 2017. “Single-cell RNA sequencing to explore immune cell heterogeneity.” <em>Nature Reviews Immunology</em> 18 (1): 35–45. doi:<a href="https://doi.org/10.1038/nri.2017.76">10.1038/nri.2017.76</a>.</p>
</div>
<div id="ref-RamosVara2010">
<p>NPR. 2010a. “Science Diction: The Origin of the Word ’Cancer’.” <a href="https://www.npr.org/templates/story/story.php?storyId=130754101" class="uri">https://www.npr.org/templates/story/story.php?storyId=130754101</a>.</p>
</div>
<div id="ref-Giesen2014">
<p>Giesen, Charlotte, Hao A O Wang, Denis Schapiro, Nevena Zivanovic, Andrea Jacobs, Bodo Hattendorf, Peter J Schüffler, et al. 2014. “Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry.” <em>Nature Methods</em> 11 (4). Nature Publishing Group: 417–22. doi:<a href="https://doi.org/10.1038/nmeth.2869">10.1038/nmeth.2869</a>.</p>
</div>
<div id="ref-Schelker2017">
<p>Schelker, Max, Sonia Feau, Jinyan Du, Nav Ranu, Edda Klipp, Birgit Schoeberl, Gavin MacBeath, et al. 2017. “Estimation of immune cell content in tumour tissue using single-cell RNA-seq data.” <em>Nature Communications</em> 8 (1). Nature Publishing Group: 2032. doi:<a href="https://doi.org/10.1038/s41467-017-02289-3">10.1038/s41467-017-02289-3</a>.</p>
</div>
<div id="ref-Velculescu1997">
<p>Velculescu, V E, L Zhang, W Zhou, J Vogelstein, M A Basrai, D E Bassett, P Hieter, B Vogelstein, and K W Kinzler. 1997. “Characterization of the yeast transcriptome.” <em>Cell</em> 88 (2): 243–51. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9008165" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/9008165</a>.</p>
</div>
<div id="ref-Cieslik2017">
<p>Cieślik, Marcin, and Arul M. Chinnaiyan. 2017. “Cancer transcriptome profiling at the juncture of clinical translation.” <em>Nature Reviews Genetics</em> 19 (2). Nature Publishing Group: 93–109. doi:<a href="https://doi.org/10.1038/nrg.2017.96">10.1038/nrg.2017.96</a>.</p>
</div>
<div id="ref-Mody2015">
<p>Mody, Rajen J., Yi-Mi Wu, Robert J. Lonigro, Xuhong Cao, Sameek Roychowdhury, Pankaj Vats, Kevin M. Frank, et al. 2015. “Integrative Clinical Sequencing in the Management of Refractory or Relapsed Cancer in Youth.” <em>JAMA</em> 314 (9). American Medical Association: 913. doi:<a href="https://doi.org/10.1001/jama.2015.10080">10.1001/jama.2015.10080</a>.</p>
</div>
<div id="ref-Oberg2016">
<p>Oberg, Jennifer A., Julia L. Glade Bender, Maria Luisa Sulis, Danielle Pendrick, Anthony N. Sireci, Susan J. Hsiao, Andrew T. Turk, et al. 2016. “Implementation of next generation sequencing into pediatric hematology-oncology practice: moving beyond actionable alterations.” <em>Genome Medicine</em> 8 (1). BioMed Central: 133. doi:<a href="https://doi.org/10.1186/s13073-016-0389-6">10.1186/s13073-016-0389-6</a>.</p>
</div>
<div id="ref-Robinson2017">
<p>Robinson, Dan R., Yi-Mi Wu, Robert J. Lonigro, Pankaj Vats, Erin Cobain, Jessica Everett, Xuhong Cao, et al. 2017. “Integrative clinical genomics of metastatic cancer.” <em>Nature</em> 548 (7667). Nature Publishing Group: 297–303. doi:<a href="https://doi.org/10.1038/nature23306">10.1038/nature23306</a>.</p>
</div>
<div id="ref-Chen2016">
<p>Chen, Pei Ling, Whijae Roh, Alexandre Reuben, Zachary A. Cooper, Christine N. Spencer, Peter A. Prieto, John P. Miller, et al. 2016. “Analysis of immune signatures in longitudinal tumor samples yields insight into biomarkers of response and mechanisms of resistance to immune checkpoint blockade.” <em>Cancer Discovery</em> 6 (8): 827–37. doi:<a href="https://doi.org/10.1158/2159-8290.CD-15-1545">10.1158/2159-8290.CD-15-1545</a>.</p>
</div>
<div id="ref-Bernstein2007">
<p>Bernstein, Bradley E, Alexander Meissner, and Eric S Lander. 2007. “The mammalian epigenome.” <em>Cell</em> 128 (4). Elsevier: 669–81. doi:<a href="https://doi.org/10.1016/j.cell.2007.01.033">10.1016/j.cell.2007.01.033</a>.</p>
</div>
<div id="ref-Perkel2017">
<p>Perkel, Jeffrey M. 2017. “Single-cell sequencing made simple.” <em>Nature</em> 547 (7661): 125–26. doi:<a href="https://doi.org/10.1038/547125a">10.1038/547125a</a>.</p>
</div>
<div id="ref-Davis2016">
<p>Davis, Sean. 2016. “Github: Awesome-Single-Cell.” <a href="https://github.com/seandavi/awesome-single-cell" class="uri">https://github.com/seandavi/awesome-single-cell</a>.</p>
</div>
<div id="ref-Cedar2018">
<p>Core, Cedar-Sinals Genomics. 2016. “Single Cell Genomics.” <a href="https://www.cedars-sinai.edu/Research/Research-Cores/Genomics-Core/Documents/website_pricing_jan_2018_final.pdf" class="uri">https://www.cedars-sinai.edu/Research/Research-Cores/Genomics-Core/Documents/website_pricing_jan_2018_final.pdf</a>.</p>
</div>
<div id="ref-Tirosh2016">
<p>Tirosh, Itay, Benjamin Izar, Sanjay M. Prakadan, Marc H. Wadsworth, Daniel Treacy, John J. Trombetta, Asaf Rotem, et al. 2016. “Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq.” <em>Science</em> 352 (6282): 189–96. doi:<a href="https://doi.org/10.1126/science.aad0501">10.1126/science.aad0501</a>.</p>
</div>
<div id="ref-Bindea2013">
<p>Bindea, Gabriela, Bernhard Mlecnik, Marie Tosolini, Amos Kirilovsky, Maximilian Waldner, Anna C. Obenauf, Helen Angell, et al. 2013. “Spatiotemporal Dynamics of Intratumoral Immune Cells Reveal the Immune Landscape in Human Cancer.” <em>Immunity</em> 39 (4): 782–95. doi:<a href="https://doi.org/10.1016/j.immuni.2013.10.003">10.1016/j.immuni.2013.10.003</a>.</p>
</div>
<div id="ref-Berghoff2018">
<p>Berghoff, Anna, Jakob Kather, and Dirk Jäger. 2018. “Genetics and Immunology: Tumor-Specific Genetic Alterations as a Target for Immune Modulating Therapies.” In <em>Oncoimmunology : A Practical Guide for Cancer Immunotherapy</em>, edited by Laurence Zitvogel and Guido Kroemer, 231–46.</p>
</div>
<div id="ref-Chen2017">
<p>Chen, Daniel S., and Ira Mellman. 2017. “Elements of cancer immunity and the cancer-immune set point.” doi:<a href="https://doi.org/10.1038/nature21349">10.1038/nature21349</a>.</p>
</div>
<div id="ref-Herbst2014">
<p>Herbst, Roy S., Jean-Charles Soria, Marcin Kowanetz, Gregg D. Fine, Omid Hamid, Michael S. Gordon, Jeffery A. Sosman, et al. 2014. “Predictive correlates of response to the anti-PD-L1 antibody MPDL3280A in cancer patients.” <em>Nature</em> 515 (7528): 563–67. doi:<a href="https://doi.org/10.1038/nature14011">10.1038/nature14011</a>.</p>
</div>
<div id="ref-Gajewski2006">
<p>Gajewski, Thomas F., Yuru Meng, Christian Blank, Ian Brown, Aalok Kacha, Justin Kline, and Helena Harlin. 2006. “Immune resistance orchestrated by the tumor microenvironment.” <em>Immunological Reviews</em> 213 (1). Blackwell Publishing Ltd: 131–45. doi:<a href="https://doi.org/10.1111/J.1600-065X.2006.00442.X">10.1111/J.1600-065X.2006.00442.X</a>.</p>
</div>
<div id="ref-Galon2012">
<p>Galon, Jérôme, Franck Pagès, Francesco M Marincola, Magdalena Thurin, Giorgio Trinchieri, Bernard A Fox, Thomas F Gajewski, and Paolo A Ascierto. 2012. “The immune score as a new possible approach for the classification of cancer.” <em>Journal of Translational Medicine</em> 10 (1). BioMed Central: 1. doi:<a href="https://doi.org/10.1186/1479-5876-10-1">10.1186/1479-5876-10-1</a>.</p>
</div>
<div id="ref-Charoentong2017">
<p>Charoentong, Pornpimol, Francesca Finotello, Mihaela Angelova, Clemens Mayer, Mirjana Efremova, Dietmar Rieder, Hubert Hackl, and Zlatko Trajanoski. 2017. “Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade.” <em>Cell Reports</em> 18 (1). Elsevier: 248–62. doi:<a href="https://doi.org/10.1016/j.celrep.2016.12.019">10.1016/j.celrep.2016.12.019</a>.</p>
</div>
<div id="ref-Angelova2015">
<p>Angelova, Mihaela, Pornpimol Charoentong, Hubert Hackl, Maria L Fischer, Rene Snajder, Anne M Krogsdam, Maximilian J Waldner, et al. 2015. “Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.” <em>Genome Biology</em> 16 (1). BioMed Central: 64. doi:<a href="https://doi.org/10.1186/s13059-015-0620-6">10.1186/s13059-015-0620-6</a>.</p>
</div>
<div id="ref-Blank2016">
<p>Blank, Christian U, John B Haanen, Antoni Ribas, and Ton N Schumacher. 2016. “CANCER IMMUNOLOGY. the ‘Cancer Immunogram’.” <em>Science (New York, N.Y.)</em> 352 (6286). American Association for the Advancement of Science: 658–60. doi:<a href="https://doi.org/10.1126/science.aaf2834">10.1126/science.aaf2834</a>.</p>
</div>
<div id="ref-Gyorffy2015">
<p>Győrffy, Balázs, Christos Hatzis, Tara Sanft, Erin Hofstatter, Bilge Aktas, and Lajos Pusztai. 2015. “Multigene prognostic tests in breast cancer: past, present, future.” <em>Breast Cancer Research : BCR</em> 17 (1). BioMed Central: 11. doi:<a href="https://doi.org/10.1186/s13058-015-0514-2">10.1186/s13058-015-0514-2</a>.</p>
</div>
<div id="ref-Ross2008">
<p>Ross, J. S., C. Hatzis, W. F. Symmans, L. Pusztai, and G. N. Hortobagyi. 2008. “Commercialized Multigene Predictors of Clinical Outcome for Breast Cancer.” <em>The Oncologist</em> 13 (5): 477–93. doi:<a href="https://doi.org/10.1634/theoncologist.2007-0248">10.1634/theoncologist.2007-0248</a>.</p>
</div>
<div id="ref-Gnjatic2017">
<p>Gnjatic, Sacha, Vincenzo Bronte, Laura Rosa Brunet, Marcus O Butler, Mary L Disis, Jérôme Galon, Leif G Hakansson, et al. 2017. “Identifying baseline immune-related biomarkers to predict clinical outcome of immunotherapy.” <em>Journal for Immunotherapy of Cancer</em> 5. BioMed Central: 44. doi:<a href="https://doi.org/10.1186/s40425-017-0243-4">10.1186/s40425-017-0243-4</a>.</p>
</div>
<div id="ref-NCI2018">
<p>Institute, National Cancer. 2017. “Types of Cancer Treatment.” <a href="https://www.cancer.gov/about-cancer/treatment/types" class="uri">https://www.cancer.gov/about-cancer/treatment/types</a>.</p>
</div>
<div id="ref-Predina2013">
<p>Predina, J., E. Eruslanov, B. Judy, V. Kapoor, G. Cheng, L.-C. Wang, J. Sun, et al. 2013. “Changes in the local tumor microenvironment in recurrent cancers may explain the failure of vaccines after surgery.” <em>Proceedings of the National Academy of Sciences</em> 110 (5): E415–E424. doi:<a href="https://doi.org/10.1073/pnas.1211850110">10.1073/pnas.1211850110</a>.</p>
</div>
<div id="ref-Palucka2013">
<p>Palucka, Karolina, and Jacques Banchereau. 2013. “Dendritic-Cell-Based Therapeutic Cancer Vaccines.” doi:<a href="https://doi.org/10.1016/j.immuni.2013.07.004">10.1016/j.immuni.2013.07.004</a>.</p>
</div>
<div id="ref-Jackson2016">
<p>Jackson, Hollie J., Sarwish Rafiq, and Renier J. Brentjens. 2016. “Driving CAR T-cells forward.” doi:<a href="https://doi.org/10.1038/nrclinonc.2016.36">10.1038/nrclinonc.2016.36</a>.</p>
</div>
<div id="ref-FDACARTadult">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p> 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p> 2016b. “Pembrolizumab (Keytruda) Checkpoint Inhibitor.” <a href="https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm" class="uri">https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm</a>.</p> 2017a. “FDA Approval Brings First Gene Therapy to the United States.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm</a>.</p> 2017b. “FDA Approves Car-T Cell Therapy to Treat Adults with Certain Types of Large B-Cell Lymphoma.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm581216.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm581216.htm</a>.</p>
</div>
<div id="ref-FDACARTALL">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p> 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p> 2016b. “Pembrolizumab (Keytruda) Checkpoint Inhibitor.” <a href="https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm" class="uri">https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm</a>.</p> 2017a. “FDA Approval Brings First Gene Therapy to the United States.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm</a>.</p>
</div>
<div id="ref-FDACTLA4">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p>
</div>
<div id="ref-FDAPDL1Bladder">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p> 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p>
</div>
<div id="ref-FDAPDL1NSCLC">
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p> 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p> 2016b. “Pembrolizumab (Keytruda) Checkpoint Inhibitor.” <a href="https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm" class="uri">https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm</a>.</p>
</div>
<div id="ref-Wolchok2015">
<p>Wolchok, Jedd D. 2015. “PD-1 Blockers.” doi:<a href="https://doi.org/10.1016/j.cell.2015.07.045">10.1016/j.cell.2015.07.045</a>.</p>
</div>
<div id="ref-Zou2016">
<p>Zou, Weiping, Jedd D. Wolchok, and Lieping Chen. 2016. “PD-L1 (B7-H1) and PD-1 pathway blockade for cancer therapy: Mechanisms, response biomarkers, and combinations.” <em>Science Translational Medicine</em> 8 (328). doi:<a href="https://doi.org/10.1126/scitranslmed.aad7118">10.1126/scitranslmed.aad7118</a>.</p>
</div>
<div id="ref-Rizvi2015">
<p>Rizvi, Naiyer A., Matthew D. Hellmann, Alexandra Snyder, Pia Kvistborg, Vladimir Makarov, Jonathan J. Havel, William Lee, et al. 2015. “Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer.” <em>Science</em> 348 (6230): 124–28. doi:<a href="https://doi.org/10.1126/science.aaa1348">10.1126/science.aaa1348</a>.</p>
</div>
<div id="ref-Pitt2016">
<p>Pitt, Jonathan M, Marie Vétizou, Romain Daillère, María Paula Roberti, Takahiro Yamazaki, Bertrand Routy, Patricia Lepage, et al. 2016. “Resistance Mechanisms to Immune-Checkpoint Blockade in Cancer: Tumor-Intrinsic and -Extrinsic Factors.” <em>Immunity</em> 44 (6). Elsevier: 1255–69. doi:<a href="https://doi.org/10.1016/j.immuni.2016.06.001">10.1016/j.immuni.2016.06.001</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="preamble-about-interdisciplinary-research.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="methods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/02-IntroductionBio.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
