<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="intro.html">
<link rel="next" href="objectives.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li><a href="index.html#unsupervised-deconvolution-of-bulk-omics-profiles-methodology-and-application-to-characterize-the-immune-landscape-in-tumors"><strong>Unsupervised deconvolution of bulk omics profiles: methodology and application to characterize the immune landscape in tumors</strong></a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="tcga-study-abbreviations.html"><a href="tcga-study-abbreviations.html"><i class="fa fa-check"></i>TCGA Study Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-ph.d.---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary Ph.D. - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-ph.d.topic"><i class="fa fa-check"></i>The origins of the Ph.D. topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a><ul>
<li class="chapter" data-level="3.1" data-path="mstd.html"><a href="mstd.html#context"><i class="fa fa-check"></i><b>3.1</b> Context</a></li>
<li class="chapter" data-level="3.2" data-path="mstd.html"><a href="mstd.html#description"><i class="fa fa-check"></i><b>3.2</b> Description</a></li>
<li class="chapter" data-level="3.3" data-path="mstd.html"><a href="mstd.html#impact-on-the-further-work"><i class="fa fa-check"></i><b>3.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a><ul>
<li class="chapter" data-level="4.1" data-path="lva.html"><a href="lva.html#context-1"><i class="fa fa-check"></i><b>4.1</b> Context</a></li>
<li class="chapter" data-level="4.2" data-path="lva.html"><a href="lva.html#description-1"><i class="fa fa-check"></i><b>4.2</b> Description</a></li>
<li class="chapter" data-level="4.3" data-path="lva.html"><a href="lva.html#impact-on-the-further-work-1"><i class="fa fa-check"></i><b>4.3</b> Impact on the further work</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="nmfica.html"><a href="nmfica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.1" data-path="nmfica.html"><a href="nmfica.html#comparing-metagenes-obtained-with-nmf-versus-ica"><i class="fa fa-check"></i><b>5.1</b> Comparing metagenes obtained with NMF versus ICA</a></li>
<li class="chapter" data-level="5.2" data-path="nmfica.html"><a href="nmfica.html#summary-4"><i class="fa fa-check"></i><b>5.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> DeconICA: an R package for Deconvolution of omic data through Immune Components Analysis</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#unsupervised-deconvolution"><i class="fa fa-check"></i><b>6.2</b> Unsupervised deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#fastica-overdecomposition-protocol"><i class="fa fa-check"></i><b>6.2.1</b> FastICA overdecomposition protocol</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="deconica.html"><a href="deconica.html#interpretation-of-the-components"><i class="fa fa-check"></i><b>6.3</b> Interpretation of the components</a></li>
<li class="chapter" data-level="6.4" data-path="deconica.html"><a href="deconica.html#computing-the-abundance-of-the-identified-cell-types"><i class="fa fa-check"></i><b>6.4</b> Computing the abundance of the identified cell-types</a></li>
<li class="chapter" data-level="6.5" data-path="deconica.html"><a href="deconica.html#validation-of-the-abundance-estimation-with-deconica"><i class="fa fa-check"></i><b>6.5</b> Validation of the abundance estimation with <em>DeconICA</em></a><ul>
<li class="chapter" data-level="6.5.1" data-path="deconica.html"><a href="deconica.html#in-silico"><i class="fa fa-check"></i><b>6.5.1</b> <em>In silico</em></a></li>
<li class="chapter" data-level="6.5.2" data-path="deconica.html"><a href="deconica.html#in-vitro"><i class="fa fa-check"></i><b>6.5.2</b> <em>In vitro</em></a></li>
<li class="chapter" data-level="6.5.3" data-path="deconica.html"><a href="deconica.html#pbmc-transcriptome"><i class="fa fa-check"></i><b>6.5.3</b> PBMC transcriptome</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="deconica.html"><a href="deconica.html#summary-5"><i class="fa fa-check"></i><b>6.6</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Comparative analysis of cancer immune infiltration</a><ul>
<li class="chapter" data-level="7.1" data-path="results.html"><a href="results.html#background"><i class="fa fa-check"></i><b>7.1</b> Background</a></li>
<li class="chapter" data-level="7.2" data-path="results.html"><a href="results.html#methods-1"><i class="fa fa-check"></i><b>7.2</b> Methods</a><ul>
<li class="chapter" data-level="7.2.1" data-path="results.html"><a href="results.html#data-sources"><i class="fa fa-check"></i><b>7.2.1</b> Data sources</a></li>
<li class="chapter" data-level="7.2.2" data-path="results.html"><a href="results.html#the-deconica-pipeline-on-bulk"><i class="fa fa-check"></i><b>7.2.2</b> The DeconICA pipeline on bulk</a></li>
<li class="chapter" data-level="7.2.3" data-path="results.html"><a href="results.html#the-deconica-pipeline-on-single-cell"><i class="fa fa-check"></i><b>7.2.3</b> The DeconICA pipeline on single cell</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="results.html"><a href="results.html#results-1"><i class="fa fa-check"></i><b>7.3</b> Results</a></li>
<li class="chapter" data-level="7.4" data-path="results.html"><a href="results.html#discussion"><i class="fa fa-check"></i><b>7.4</b> Discussion</a></li>
<li class="chapter" data-level="7.5" data-path="results.html"><a href="results.html#conclussions"><i class="fa fa-check"></i><b>7.5</b> Conclussions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> A multiscale signalling network map of innate immune response in cancer reveals signatures of cell heterogeneity and functional polarization</a><ul>
<li class="chapter" data-level="8.1" data-path="map.html"><a href="map.html#context-2"><i class="fa fa-check"></i><b>8.1</b> Context</a></li>
<li class="chapter" data-level="8.2" data-path="map.html"><a href="map.html#description-2"><i class="fa fa-check"></i><b>8.2</b> Description</a></li>
<li class="chapter" data-level="8.3" data-path="map.html"><a href="map.html#discussion-and-perspectives"><i class="fa fa-check"></i><b>8.3</b> Discussion and perspectives</a></li>
</ul></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussiongenerale.html"><a href="discussiongenerale.html"><i class="fa fa-check"></i><b>9</b> Discussion</a></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a><ul>
<li class="chapter" data-level="10.1" data-path="conclusions.html"><a href="conclusions.html#conclusions-1"><i class="fa fa-check"></i><b>10.1</b> Conclusions</a></li>
<li class="chapter" data-level="10.2" data-path="conclusions.html"><a href="conclusions.html#perspectives"><i class="fa fa-check"></i><b>10.2</b> Perspectives</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#deconica-documentation"><i class="fa fa-check"></i>1 DeconICA documentation</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#introduction-to-deconica"><i class="fa fa-check"></i>1.1 Introduction to deconICA</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#running-fastica-with-icasso-stabilisation"><i class="fa fa-check"></i>1.2 Running fastICA with icasso stabilisation</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#reference-manual"><i class="fa fa-check"></i>1.3 Reference manual</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#publications-and-conferences"><i class="fa fa-check"></i>2 Publications and conferences</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#adjustment-of-dendritic-cells-to-the-breast-cancer-microenvironment-is-subset-specific"><i class="fa fa-check"></i>2.1 Adjustment of dendritic cells to the breast-cancer microenvironment is subset-specific</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#the-inconvenience-of-data-of-convenience-computational-research-beyond-post-mortem-analyses"><i class="fa fa-check"></i>2.2 The inconvenience of data of convenience: computational research beyond post-mortem analyses</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv-publications-conferences-courses"><i class="fa fa-check"></i>2.3 CV: publications, conferences, courses</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a><ul>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#biological-terms"><i class="fa fa-check"></i>Biological terms</a></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html#mathematical-terms"><i class="fa fa-check"></i>Mathematical terms</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="methods" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Mathematical foundation of cell-type deconvolution of biological data</h1>
<p>In the previous chapter, I presented state-of-art of the current immuno-oncology research that has to embrace the vast complexity of cancer disease and the immune system. One part of this complexity can be explained by the presence and quantities of tumor-infiltrating immune cells, their interactions with each other and the tumor.</p>
<p>In this chapter, I will discuss how mathematical models can be used to extract information about different cell-types from ‘bulk’ omics data or how to de-mix mixed sources composing the bulk samples. To start with, I will introduce you to basic concepts of machine learning. Then I will focus on approaches adapted for cell-type deconvolution. In a literature overview, I will depict the evolution of the field as well as discuss the particularities of different tools for estimating presence and proportion of immune cells within cancer bulk omic data.</p>
<div id="introduction-to-supervised-and-unsupervised-learning" class="section level2">
<h2><span class="header-section-number">2.1</span> Introduction to supervised and unsupervised learning</h2>
<p>Machine learning (ML) is a field of computer science where a system can learn and improve given an objective function and the data.</p>
<p>Mitchell gave a popular definition of machine learning in 1997:</p>
<blockquote>
<p>Machine learning: <em>A computer program is said to learn from experience E with respect to some class of tasks T and performance measure P if its performance at tasks in T, as measured by P, improves with experience E</em>.</p>
<p>— Mitchell in 1997 <span class="citation">(Mitchell <a href="#ref-Mitchell1997">1997</a>)</span></p>
</blockquote>
<p>Term <em>Artificial intelligence</em> (AI) is often used by the media or the general public to describe machine learning. Indeed ML can be considered as a branch of AI, together with computer vision and deep neural networks applications. However, commonly ML and AI are used interchangeably by the broad public.</p>
<p>ML is applied commonly in many fields of science and industry. I will not discuss here subtle differences between machine learning, statistical learning, computational statistics and mathematical optimization.</p>
<p>In general, algorithms can be divided into groups given the application:</p>
<ul>
<li>classification - aims to assign observations to a group (discrete variable)</li>
<li>regression - aims to predict a continuous response of an input (continuous variable)</li>
<li>clustering - aims to divide data into groups that are related to each other based on a distance</li>
</ul>
<p>Another critical distinction can be made given the inputs to the algorithm. Here, I present the differences between supervised and unsupervised learning.</p>
<div id="supervised-learning" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Supervised learning</h3>
<p>Supervised learning can be described as “the analysis of data via a focused structure” <span class="citation">(Piegorsch, n.d.)</span>. The primary task is to predict an output given the inputs. In the statistical language, the inputs are often called the predictors or the independent variables. In the pattern recognition literature, the term features are preferred. The outputs are called the responses, or the dependent variables. <span class="citation">(Hastie, Tibshirani, and Friedman <a href="#ref-Hastie2009">2009</a>)</span></p>
<p>The initial data is divided into two sets: training and test. First, the model is trained with correct answers on the training data (learning to minimize the error), and then its performance is evaluated on the test data.</p>
<p>Among widely used classifiers there are Support Vector Machines (SVM), partition trees (and their extension random forests), and neural networks. For regression, it is common to encounter linear regression, boosted trees regression,</p>
</div>
<div id="unsup" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Unsupervised learning</h3>
<p>In Unsupervised learning is given the data and is asked to segment the data given a particular constraint. However, the true segments of the data are not known. Therefore an unsupervised algorithm aims to unveil the “hidden structure” of the data or latent variables.</p>
<p>One group of unsupervised learning are descriptive statistic methods, such as principal components, multidimensional scaling, self-organizing maps, and principal curves. These methods aim to represent to the data most adequately in low-dimensional space <span class="citation">(Hastie, Tibshirani, and Friedman <a href="#ref-Hastie2009">2009</a>)</span>.</p>
<p>Another group is clustering algorithms. Clustering is the way to create groups (multiple convex regions) based on the intrinsic architecture of the data. These groups are not necessarily known beforehand but can be validated with the domain knowledge. Popular clustering algorithms are k-means, hierarchical clustering, mixture density-based clustering <span class="citation">(<span class="citeproc-not-found" data-reference-id="Xu2008"><strong>???</strong></span>)</span>.</p>
<p>In both descriptive statistics and clustering, one important parameter (i.e. denoted <span class="math inline">\(k\)</span>) is the number (number of factors, variables, clusters) to which the data should be decomposed. Different algorithms and applications can propose an automatic choice of <span class="math inline">\(k\)</span> based on formal indexes or previous knowledge, in others, the user needs to provide the <span class="math inline">\(k\)</span>.</p>
</div>
<div id="low-dimensional-embeddingfor-visualization" class="section level3">
<h3><span class="header-section-number">2.1.3</span> Low-dimensional embedding for visualization</h3>
<p>There is a common confusion, often seen in computational biology, between dimension reduction and clustering. This confusion is highly pronounced with, a popular in biology, algorithm: T-distributed Stochastic Neighbor Embedding (t-SNE) <span class="citation">(Van Der Maaten and Hinton <a href="#ref-VanDerMaaten2008">2008</a>)</span>. t-SNE works in 2 main steps: (1) a probability distribution over pairs of high-dimensional objects is computed in such a way that similar objects have a high probability of being picked, whilst dissimilar points have an extremely small probability of being picked, (2) t-SNE defines a similar probability distribution over the points in the low-dimensional map, and it minimizes the Kullback–Leibler divergence between the two distributions with respect to the locations of the points in the map. It is not reliable to use t-SNE for clustering as it does not preserve distances. It can also easily overfit the data and uncover ‘fake’ or ‘forced’ patterns. Therefore, a clustering should not be applied to t-sne reduced data. An alternative to the t-SNE method is recently published Uniform Manifold Approximation and Projection for Dimension Reduction (UMAP) <span class="citation">(Mcinnes and Healy <a href="#ref-Mcinnes2018">2018</a>)</span>– that is based on Laplacian eigenmaps, highly scalable, reproducible and recently applied to biological data <span class="citation">(Becht et al. <a href="#ref-Becht2018">2018</a>)</span>. Older used alternatives are ISOMAPS (non-linear dimension reduction) or PCA (Principal components analysis). For any non-linear dimension reduction method, it is not recommended to use clustering <em>a posteriori</em>. Clusters should be computed on original data, and then the cluster labels can be visualized in low-dimensional embedding.</p>
<!--Impact of machine learning on Healthcare-->
<!--Machine learning may seem a discipline far from biology or healthcare. However with the expansion of data ML-based methods are participating more and more in medecine, especially in diagnostics, and will probably grow expotentially.-->
<!--https://qz.com/1251502/the-fda-just-opened-the-door-to-let-ai-make-medical-decisions-on-its-own/-->
<!--https://www.healthcatalyst.com/clinical-applications-of-machine-learning-in-healthcare-->
<!--http://collections.plos.org/mlforhealth-->
</div>
</div>
<div id="types-of-deconvolution" class="section level2">
<h2><span class="header-section-number">2.2</span> Types of deconvolution</h2>
<p>One specific application of mathematical/statistical tools is deconvolution of mixed signals.</p>
<p>According to a mathematical definition:</p>
<blockquote>
<p>Deconvolution : <em>the resolution of a convolution function into the functions from which it was formed in order to separate their effects.</em> <span class="citation">(<span class="citeproc-not-found" data-reference-id="deconvolution"><strong>???</strong></span>)</span></p>
</blockquote>
<p>Alternatively, in plain English:</p>
<blockquote>
<p><em>a process of resolving something into its constituent elements or removing complication</em> <span class="citation">(<span class="citeproc-not-found" data-reference-id="deconvolution"><strong>???</strong></span>)</span></p>
</blockquote>
<p>The similar problem of mixed sources can be encountered in other fields, i.e., signal processing, also known under the name of “<strong>cocktail party problem</strong>.” In the cocktail party problem, at a party with many people and music, sound in recorded with several microphones. Through blind source separation, it is possible to separate the voices of different people and the musical background (Fig. <a href="methods.html#fig:cocktailparty">2.1</a>) <span class="citation">(Cherry <a href="#ref-Cherry1953">1953</a>)</span>.</p>
<div class="figure" style="text-align: center"><span id="fig:cocktailparty"></span>
<img src="figures-ext/cocktailparty.jpg" alt="Illustration of the cocktail party problem. During a cocktail party voices of participants can be recorded with a set of microphones and then recovered through blind source separation. The illustration purposes only four sources are mixed with three microphones, in reality, the analysis can be performed with many sources. However, a number of samples (microphones) should be higher than the number of sources (contrary to the illustration)." width="100%" />
<p class="caption">
Figure 2.1: <strong>Illustration of the cocktail party problem</strong>. During a cocktail party voices of participants can be recorded with a set of microphones and then recovered through blind source separation. The illustration purposes only four sources are mixed with three microphones, in reality, the analysis can be performed with many sources. However, a number of samples (microphones) should be higher than the number of sources (contrary to the illustration).
</p>
</div>

<p>The same concept can be transposed to the bulk omic data, each biological species (like gene) is a cocktail party where each sample is a microphone that gathers mixed signals of different nature. The signals that form the mixtures can be different depending on the data type, and the scientific question asked.</p>
<p>In general, the total bulk data can be affected by three abundance components <span class="citation">(Shen-Orr and Gaujoux <a href="#ref-Shen-Orr2013">2013</a>)</span>:</p>
<ol style="list-style-type: decimal">
<li>sample characteristic (disease, clinical features)</li>
<li>individual variation, genotype-specific or technical variation</li>
<li>presence and abundance of different cell types expressing a set of characteristic genes</li>
</ol>
<p>Many scientists invested their efforts in order to dissect the bulk omic data into interpretable biological components.</p>
<p>In scientific literature, there can be encountered three main understanding of tumor deconvolution:</p>
<ul>
<li><strong>estimating clonality</strong>: using genomic data is it possible to trace tumor phylogeny raised from mutations and aberrations in tumor cells; therefore it is dissecting <em>intra</em>-tumor heterogeneity (i.e., using transcriptomic data <span class="citation">(Schwartz and Shackney <a href="#ref-Schwartz2010">2010</a>)</span>, or more often CNA data (see <a href="methods.html#otherDecon">Section 2.4.2</a>)</li>
<li><strong>estimating purity</strong>: deconvolution into the tumor and immune/stroma compartments, often aiming to “remove” not-tumor signal from the expression data, can be performed with different data types, the most reliable estimations are usually obtained from CNA data (see <a href="methods.html#otherDecon">Section 2.4</a>)</li>
<li><strong>estimating cell-type</strong> proportions and/or profiles from bulk omics data, most of works were performed on transcriptome data (see <a href="methods.html#cellTypeTrans">Section 2.3</a>) and some on the methylome data (see <a href="methods.html#otherDecon">Section 2.4.1</a>)</li>
</ul>
<p>These three types of deconvolution can be performed on the bulk omics data. Here we will focus on cell-type deconvolution models using bulk transcriptome. I will also briefly introduce deconvolution models applied to other data types (methylome and CNA).</p>
</div>
<div id="cellTypeTrans" class="section level2">
<h2><span class="header-section-number">2.3</span> Cell-type deconvolution of bulk transcriptomes</h2>
<p>The idea of un-mixing the bulk omic profiles is documented to first appear in an article of <span class="citation">Venet et al. (<a href="#ref-Venet2001">2001</a>)</span> as a way to</p>
<blockquote>
<p><em>infer the gene expression profile of the various cell types (…) directly from the measurements taken on the whole sample</em></p>
</blockquote>
<p>In the primary hypothesis <span class="citation">(Abbas et al. <a href="#ref-Abbas2009">2009</a>)</span>, a mixture of signals from TME in transcriptomic samples can be described as a linear mixture.</p>
<p><span class="math display" id="eq:linear">\[
X = SA  \tag{2.1}
\]</span></p>
<p>Where in Equation <a href="methods.html#eq:linear">(2.1)</a> <span class="math inline">\(X\)</span> is microarray data matrix of one biological sample, <span class="math inline">\(A\)</span> are mixing proportions, and <span class="math inline">\(S\)</span> is the matrix of expression of genes in each cell type.</p>
<p>Algebraically the same problem can be formalized as the latent variable model:</p>
<p><span class="math display" id="eq:algebraic">\[
\forall i \in \{1,M\},  \forall  j \in \{1,N\} \\
x_{ij}= \sum_{k=1}^K a_{kj} *s_{ik}+ e_{ij} \tag{2.2}
\]</span></p>
<p>Where <span class="math inline">\(x_{ij}\)</span> is expression of gene <span class="math inline">\(i\)</span> in sample <span class="math inline">\(j\)</span>, <span class="math inline">\(a_{kj}\)</span> is the proportion of cell type <span class="math inline">\(k\)</span> in sample <span class="math inline">\(j\)</span> and <span class="math inline">\(s_{ik}\)</span> is the expression of the gene <span class="math inline">\(i\)</span> in the cell type <span class="math inline">\(k\)</span>, <span class="math inline">\(K\)</span> total number of cell types, <span class="math inline">\(N\)</span> total number of samples, <span class="math inline">\(M\)</span> total number of genes. The error term <span class="math inline">\(e_{ij}\)</span> cannot be directly measured.</p>
<p>The goal of deconvolution is to reverse these equations and starting from the mixture infer the <span class="math inline">\(A\)</span> (or <span class="math inline">\(a_{kj}\)</span>) and <span class="math inline">\(S\)</span> (or <span class="math inline">\(s_{ik}\)</span>).</p>
<p>Graphically the deconvolution of bulk gene expression can be depicted as in Fig. <a href="methods.html#fig:deconvolution-cartoon">2.2</a>.</p>
<div class="figure" style="text-align: center"><span id="fig:deconvolution-cartoon"></span>
<img src="figures-ext/deconv.png" alt="Principle of the deconvolution applied to transcriptome Graphical illustration of the deconvolution of mixed samples. Starting from the left, gene expression of genes A B C is a sum of expression of cell types 1, 2, 3, 4. After deconvolution, cell types are separated, and gene expression of each cell type is estimated taking into account cell type proportions." width="100%" />
<p class="caption">
Figure 2.2: <strong>Principle of the deconvolution applied to transcriptome</strong> Graphical illustration of the deconvolution of mixed samples. Starting from the left, gene expression of genes A B C is a sum of expression of cell types 1, 2, 3, 4. After deconvolution, cell types are separated, and gene expression of each cell type is estimated taking into account cell type proportions.
</p>
</div>

<p>However, in this model, either the mixing proportions, number of mixing sources or an array of specific genes need to be known. While, in the real-life case, only <span class="math inline">\(X\)</span> is truly known. Therefore, developed models proposed various manners for estimating the number of mixing sources and their proportions, or the specific cell type expression.</p>
<p>Why there is a need for cell-type deconvolution approaches?</p>
<ul>
<li>for differential gene expression analysis, to avoid confusion between a studied condition (i.e. disease impact) and cell-type abundance (change in gene expression due to the change of cell proportions)</li>
<li>difference in gene expression in one cell type can be blurred by the presence of other cells expressing the gene</li>
<li>to obtain information about a fraction of given component in the sample</li>
<li>to study potential interactions between cell types in the studied context</li>
<li>to infer context-specific profile or signature</li>
</ul>
<div id="literature-overview" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Literature overview</h3>
<p>In order to answer general and specific need for cell-type deconvolution of bulk transcriptomes researches produced a large collection of tools. I have collected all (to my knowledge) articles published in journals or as a pre-print (up to May 2018) that propose original models/tools of <strong>cell-type deconvolution of bulk transcriptomes</strong> (Tab. <a href="methods.html#tab:mytab">2.1</a>). Therefore clonal deconvolution methods are not included in this overview. The transcriptome-based purity estimation methods are included as many of them proposed an initial 2-sources model that could be, at least in theory, extended to multiple sources model. Also, I did not include cell-type deconvolution methods of other data types (such as methylome). A separate <a href="methods.html#otherDecon">section 2.4</a> is dedicated to non-transcriptome methods.</p>
<p>####Growth of the field</p>
<p>The Table <a href="methods.html#tab:mytab">2.1</a> contains 64 (including mine) deconvolution methods. It can be observed (Fig. <a href="methods.html#fig:pubyear">2.3</a>) that since the begging of my thesis (2015) the number of publications has doubled (64 publications in 2018 vs. 33 in 2014). Also, from 2014 on, more methods are published every year. In Fig. <a href="methods.html#fig:pubyear">2.3</a> <em>hallmark</em> publications are indicated in red above their year of publication. The three most popular methods (based on number of citations/number of years since publication) are CIBERSORT <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>)</span> (2015, total number of citations: 343 and 88.75 citations per year), ESTIMATE <span class="citation">(Yoshihara et al. <a href="#ref-Yoshihara2013">2013</a>)</span> (2013, total number of citations: 266 and 44.33 citations per year), and csSAM <span class="citation">(Shen-Orr et al. <a href="#ref-ShenOrr2010">2010</a>)</span> (2010, total number of citations: 286 and 31.77 citations per year). It can be noticed that the high impact of the journal plays a role, the top 3 cited methods were published in <em>Nature Methods</em> and <em>Nature Communications</em> followed by Virtual Microdissection method <span class="citation">(Moffitt et al. <a href="#ref-Moffitt2015">2015</a>)</span> (2015) published in <em>Nature Genetics</em>. However, the fifth most cited publication <span class="citation">Abbas et al. (<a href="#ref-Abbas2009">2009</a>)</span> (2009, a total of 207 citations) appeared in <em>PLOS ONE</em>. As the index is a bit penalizing for recent publications, among commonly cited tools after 2015 are MCPcounter with 42 citations (2016, 32 without self-citations) and xCell with 14 citations (2017, 11 without self-citations). A big number of publications with a low number of citations were published in <em>Oxford Bioinformatics</em> or <em>BMC Bioinformatics</em> which underlines the importance of publishing a computational tool along with an important biological message rather than in a technical journal in order to increase a chance to be used by other researchers.</p>
<div id="availability" class="section level4">
<h4><span class="header-section-number">2.3.1.1</span> Availability</h4>
<p>Another essential aspect is the availability of the tool. One-third (in total 21) methods do not provide source code or a user-interface tool to reproduce their results. Among those articles, 13 was published before 2015. Therefore, it can be concluded that the pressure of publishers and research community on reproducibility and accessibility of bioinformatic tools gives positive results. <span class="citation">Shen-Orr and Gaujoux (<a href="#ref-Shen-Orr2013">2013</a>)</span>, authors of semi-supervised NMF method <span class="citation">(Gaujoux and Seoighe <a href="#ref-Gaujoux2012">2012</a>)</span>, published <em>CellMIx: a comprehensive toolbox for gene expression deconvolution</em> where he implements most of previously published tools in R language and group them in the same R-package. This work tremendously increased the usability of previously published deconvolution methods. The CellMix package is one of the state-of-the-art work on deconvolution that regroups algorithms, signatures and benchmark datasets up to 2013.</p>
<table>
<caption>
<span id="tab:mytab">Table 2.1: </span><b>Summary of methods for cell-type deconvolution of bulk transcriptome</b>
</caption>
</table>
<div id="htmlwidget-de10812a57f35338d397" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-de10812a57f35338d397">{"x":{"filter":"top","filterHTML":"<tr>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;MA&quot;,&quot;MA + RNA-seq&quot;,&quot;MA, GWAS&quot;,&quot;RNA-seq&quot;,&quot;RNA-seq + Images&quot;,&quot;RNAseq&quot;,&quot;scRNA-seq&quot;,&quot;undefined&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;BOTH&quot;,&quot;semi-supervised&quot;,&quot;supervised&quot;,&quot;unsupervised&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none; position: absolute; width: 200px;\">\n      <div data-min=\"2001\" data-max=\"2018\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;FALSE&quot;,&quot;TRUE&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;FALSE&quot;,&quot;FLASE&quot;,&quot;TRUE&quot;,&quot;TRUE*&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;blood&quot;,&quot;Blood&quot;,&quot;Blood gene expression&quot;,&quot;Blood, solid tissue, disease&quot;,&quot;Brain tissue&quot;,&quot;Cancer purity&quot;,&quot;Cancer transcriptome&quot;,&quot;Cancer xenografts&quot;,&quot;cander and normal tissue&quot;,&quot;Chronic kidney disease (Cell lineages)&quot;,&quot;detection of cancer and stroma in PDAC (TCGA)&quot;,&quot;Gene expression&quot;,&quot;Human tissues&quot;,&quot;in vitro cell mixtures&quot;,&quot;In vitro tissue mixtures&quot;,&quot;in vitro tissue mlxtures&quot;,&quot;Infected lung tissue&quot;,&quot;Infected macrophages&quot;,&quot;Lung tissue&quot;,&quot;Mice blood under flu infection&quot;,&quot;Mice diseased tissues&quot;,&quot;Murine mammary gland&quot;,&quot;Skin tuberculosis&quot;,&quot;Synovial tissue (cell types in silico)&quot;,&quot;Tissue mixtures&quot;,&quot;Tumor purity&quot;,&quot;Udefined&quot;,&quot;yeast cell cycle&quot;,&quot;Yeast cell cycle&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;FALSE&quot;,&quot;TRUE&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;FALSE&quot;,&quot;TRUE&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;convex hull&quot;,&quot;enrichment&quot;,&quot;matrix factorisation&quot;,&quot;probabilistic&quot;,&quot;regression&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;C, R&quot;,&quot;Java 2&quot;,&quot;matlab&quot;,&quot;matlab, R&quot;,&quot;octave&quot;,&quot;python&quot;,&quot;R&quot;,&quot;R- C-C++- Fortran&quot;,&quot;R-java&quot;,&quot;R, matlab&quot;,&quot;R, web tool&quot;,&quot;Stan&quot;,&quot;unknown&quot;,&quot;web tool&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none; position: absolute; width: 200px;\">\n      <div data-min=\"0\" data-max=\"343\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none; position: absolute; width: 200px;\">\n      <div data-min=\"0\" data-max=\"85.75\" data-scale=\"15\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;arXiv&quot;,&quot;bioRxiv&quot;,&quot;journal&quot;,&quot;NA&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n  <td data-type=\"factor\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"width: 100%; display: none;\">\n      <select multiple=\"multiple\" style=\"width: 100%;\" data-options=\"[&quot;FALSE&quot;,&quot;TRUE&quot;]\"><\/select>\n    <\/div>\n  <\/td>\n<\/tr>","data":[["CIBERSORT","ESTIMATE","csSAM","Virtual Microdissection","Abbas regression","MCPcounter","PSEA","Quadratic programming","Semi-supervised Nonnegative Matrix Factorization","ssGSEA applied to renal cell\r\ncarcinoma","DeconRNASeq","DSA","DECONVOLUTE","ISOpure","xCell","CellCODE","DCQ","DeMix","Statistical expression deconvolution","DSection","Nanodissection","TIMER","Direct method","SPEC","PERT","deconf","CTen","Mixture models","CAM","ISOLATE","UNDO","In silico microdissection","BioQC","Self-directed Method for Cell-Type Identification","Computational expression deconvolution","Electronical substraction","EPIC","MMAD","Immune  Quant","VoCAL","CellMapper","Estimation of immune cell content","contamDE","GSVA scores","Enumerateblood","CoD","ImmunoStates","quanTIseq","SMC","Modular discrimination index","Robust Computational Reconstitution","Statical mechanics approach","MHMM","MySort","ADVOCATE","DTD","CellDistinguisher ","dtangle","DeconICA","DemixT","Post‐modified non‐negative matrix factorization","Infino","ImSig","TEMT"],["22","2","user-defined","14","17","10","user-defined","6","user-defined","30","undefiend","user-defined","user-defined","2","64","user-defined","undefiend","2","2","user-defined","undefined","6","SM","user-defined","11","100","undefined","2","10","undefined","2","3","150","undefined","undefined","2","8","unknown","user-defined","user-defined","user-defined","8","2+","6","6","207","20","11","user-defined","5","SM","2","user-defined","22","2","user-defined","user-defined","user-defined","100","3","user-defined","user-defined","11","2"],["MA","MA + RNA-seq","MA","MA","MA","MA","MA","MA","MA","RNA-seq","RNA-seq","MA","MA","MA","MA + RNA-seq","MA","RNA-seq","MA","MA","MA","MA","MA + RNA-seq","MA","MA","MA","MA","MA","MA","MA","MA","MA","MA","MA + RNA-seq","MA","MA","MA","RNA-seq","MA","undefined","MA, GWAS","MA","scRNA-seq","RNA-seq","RNA-seq","MA","RNA-seq","MA","RNA-seq + Images","MA","MA + RNA-seq","MA","undefined","MA","MA","RNA-seq","scRNA-seq","MA + RNA-seq","MA + RNA-seq","MA + RNA-seq","MA + RNA-seq","RNA-seq","RNA-seq","RNA-seq","RNAseq"],["supervised","supervised","supervised","unsupervised","supervised","supervised","supervised","supervised","semi-supervised","supervised","supervised","supervised","supervised","supervised","supervised","semi-supervised","supervised","supervised","supervised","supervised","supervised","supervised","unsupervised","supervised","supervised","unsupervised","supervised","supervised","unsupervised","supervised","unsupervised","unsupervised","supervised","unsupervised","supervised","supervised","supervised","BOTH","supervised","supervised","semi-supervised","supervised","supervised","supervised","supervised","supervised","supervised","supervised","unsupervised","supervised","supervised","unsupervised","unsupervised","supervised","supervised","supervised","unsupervised","supervised","unsupervised","supervised","unsupervised","supervised","supervised","supervised"],["Supported vector regression","ssGSEA","least-squares regression","multiplicative update NMF","linear least squares regression","means of marker genes","linear regression","linear latent variable model solved with quadratic programming","NMF minimizing the Kullback-Leibler divergence on pre-selected genes and with pure proportions","ssGSEA","linear latent variable model solved with quadratic programming","linear model solved with quadratic programming.","system of linear equations","maximum a posteriori(MAP) estimation of multinomial distribution","ssGSEA+ spillover compensation","robust latent variable"," regularized regression model ","Maximul likelihood estimate","linear equations","Bayesian MCMC-based model","iterative SVM","constrained linear regression","non negative least squares and decorellation","enrichment score","Non-negative maximum likelihood model  with adjustement for perturbations","Least squares non-negative matrix factorization algorithm","GSEA","methods of moments procedures and the expectation–maximization algorithm","convex analysis of mixtures ","Latent Dirichlet Allocation (LDA)","matrix inversion","Gaussian Mixture Model","Wilcoxon-Mann-Whitney test","non-negative least squares, Kullback-Leibler divergence","linear equations with simulated annealing","system of linear equations","weighted constrained least square optimization","constrained regression with corrected AIC parameter fit","DCQ algorithm adapted to human","linear regression","SVD","CIBERSORT using scRNA-seq basis matrix","empirical Bayes estimate of the negative binomial dispersion","GSVA"," multi-response Gaussian model trained","DCQ algorithm + random forest classifier","regression (previously published)","constrained least squares regression","Bayesian inference with sequential monte carlo samplers","correlation-based score","trimmed least modulus (L1) regression.","Bayesian model with MCMC sampling assuming Gaussian distribution","Hidden state markov model"," v -Support Vector Regression","trained Gaussian-mixture mode","optimising loss-function of  penalized least-squares regression","topic modelling + onvex hull / NMF","linear regression","ICA + post identification: enrichment","Maximal likelihood estimate","NMF using alternating least square","Bayesian inference with a generative model","correlation-based score","generative mixture model "],["<a href=\"https://doi.org/10.1038/nmeth.3337\">https://doi.org/10.1038/nmeth.3337<\/a>","<a href=\"https://doi.org/10.1038/ncomms3612\">https://doi.org/10.1038/ncomms3612<\/a>","<a href=\"https://doi.org/10.1038/nmeth.1439\">https://doi.org/10.1038/nmeth.1439<\/a>","<a href=\"https://doi.org/10.1038/ng.3398\">https://doi.org/10.1038/ng.3398<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0006098\">https://doi.org/10.1371/journal.pone.0006098<\/a>","<a href=\"https://doi.org/10.1186/s13059-016-1070-5\">https://doi.org/10.1186/s13059-016-1070-5<\/a>","<a href=\"https://doi.org/10.1038/nmeth.1710\">https://doi.org/10.1038/nmeth.1710<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0027156\">https://doi.org/10.1371/journal.pone.0027156<\/a>","<a href=\"https://doi.org/10.1016/j.meegid.2011.08.014\">https://doi.org/10.1016/j.meegid.2011.08.014<\/a>","<a href=\"https://doi.org/10.1186/s13059-016-1092-z\">https://doi.org/10.1186/s13059-016-1092-z<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btt090\">https://doi.org/10.1093/bioinformatics/btt090<\/a>","<a href=\"https://dx.doi.org/10.1186%2F1471-2105-14-89\">https://dx.doi.org/10.1186%2F1471-2105-14-89<\/a>","<a href=\"https://doi.org/10.1073/pnas.1832361100\">https://doi.org/10.1073/pnas.1832361100<\/a>","<a href=\"https://doi.org/10.1186/gm433\">https://doi.org/10.1186/gm433<\/a>","<a href=\"https://dx.doi.org/10.1186%2Fs13059-017-1349-1\">https://dx.doi.org/10.1186%2Fs13059-017-1349-1<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btv015\">https://doi.org/10.1093/bioinformatics/btv015<\/a>","<a href=\"https://dx.doi.org/10.1002%2Fmsb.134947\">https://dx.doi.org/10.1002%2Fmsb.134947<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btt301\">https://doi.org/10.1093/bioinformatics/btt301<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btq097\">https://doi.org/10.1093/bioinformatics/btq097<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btq406\">https://doi.org/10.1093/bioinformatics/btq406<\/a>","<a href=\"https://doi.org/10.1101/gr.155697.113\">https://doi.org/10.1101/gr.155697.113<\/a>","<a href=\"https://doi.org/10.1186/s13059-016-1028-7\">https://doi.org/10.1186/s13059-016-1028-7<\/a>","<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11473019\">https://www.ncbi.nlm.nih.gov/pubmed/11473019<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-12-258\">https://doi.org/10.1186/1471-2105-12-258<\/a>","<a href=\"https://doi.org/10.1371/journal.pcbi.1002838\">https://doi.org/10.1371/journal.pcbi.1002838<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-11-27\">https://doi.org/10.1186/1471-2105-11-27<\/a>","<a href=\"https://doi.org/10.1186/1471-2164-13-460\">https://doi.org/10.1186/1471-2164-13-460<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/bth139\">https://doi.org/10.1093/bioinformatics/bth139<\/a>","<a href=\"https://doi.org/10.1038/srep18909\">https://doi.org/10.1038/srep18909<\/a>","<a href=\"https://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp378\">https://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp378<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btu607\">https://doi.org/10.1093/bioinformatics/btu607<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-6-54\">https://doi.org/10.1186/1471-2105-6-54<\/a>","<a href=\"https://doi.org/10.1186/s12864-017-3661-2\">https://doi.org/10.1186/s12864-017-3661-2<\/a>","<a href=\"https://doi.org/10.1371/journal.pcbi.1003189\">https://doi.org/10.1371/journal.pcbi.1003189<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-7-328\">https://doi.org/10.1186/1471-2105-7-328<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btm508\">https://doi.org/10.1093/bioinformatics/btm508<\/a>","<a href=\"https://dx.doi.org/10.7554%2FeLife.26476\">https://dx.doi.org/10.7554%2FeLife.26476<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btt566\">https://doi.org/10.1093/bioinformatics/btt566<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btw535\">https://doi.org/10.1093/bioinformatics/btw535<\/a>","<a href=\"https://doi.org/10.1371/journal.pcbi.1004856\">https://doi.org/10.1371/journal.pcbi.1004856<\/a>","<a href=\"https://doi.org/10.1186/s13059-016-1062-5\">https://doi.org/10.1186/s13059-016-1062-5<\/a>","<a href=\"https://doi.org/10.1038/s41467-017-02289-3\">https://doi.org/10.1038/s41467-017-02289-3<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btv657\">https://doi.org/10.1093/bioinformatics/btv657<\/a>","<a href=\"https://doi.org/10.1158/1078-0432.CCR-17-3509\">https://doi.org/10.1158/1078-0432.CCR-17-3509<\/a>","<a href=\"https://doi.org/10.1186/s12864-016-3460-1\">https://doi.org/10.1186/s12864-016-3460-1<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btv498\">https://doi.org/10.1093/bioinformatics/btv498<\/a>","<a href=\"https://doi.org/10.1101/206466\">https://doi.org/10.1101/206466<\/a>","<a href=\"https://doi.org/10.1101/223180\">https://doi.org/10.1101/223180<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0186167\">https://doi.org/10.1371/journal.pone.0186167<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0169271\">https://doi.org/10.1371/journal.pone.0169271<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-7-369\">https://doi.org/10.1186/1471-2105-7-369<\/a>","<a href=\"https://arxiv.org/abs/1210.7508v1\">https://arxiv.org/abs/1210.7508v1<\/a>","<a href=\"https://doi.org/10.1089/cmb.2006.13.1749\">https://doi.org/10.1089/cmb.2006.13.1749<\/a>","<a href=\"https://doi.org/10.1186/s12859-018-2069-6\">https://doi.org/10.1186/s12859-018-2069-6<\/a>","<a href=\"https://doi.org/10.1101/288779\">https://doi.org/10.1101/288779<\/a>","<a href=\"https://arxiv.org/abs/1801.08447v1\">https://arxiv.org/abs/1801.08447v1<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0193067\">https://doi.org/10.1371/journal.pone.0193067<\/a>","<a href=\"https://doi.org/10.1101/290262\">https://doi.org/10.1101/290262<\/a>","<a href=\"https://doi.org/10.5281/zenodo.1250069\">https://doi.org/10.5281/zenodo.1250069<\/a>","<a href=\"https://doi.org/10.1101/146795\">https://doi.org/10.1101/146795<\/a>","<a href=\"https://doi.org/10.1002/cem.2929\">https://doi.org/10.1002/cem.2929<\/a>","<a href=\"https://doi.org/10.1101/221671\">https://doi.org/10.1101/221671<\/a>","<a href=\"https://doi.org/10.1101/077487\">https://doi.org/10.1101/077487<\/a>","<a href=\"https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S11\">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S11<\/a>"],["Aaron M Newman","Kosuke Yoshihara","Shai S Shen-Orr","Richard A Moffitt","Alexander R. Abbas","Etienne Becht","Alexandre Kuhn","Ting Gong","Renaud Gaujoux","Yasin Şenbabaoğlu","Ting Gong","Yi Zhong","Peng Lu","Gerald Quon","Dvir Aran","Maria Chikina","Zeev Altboum","Jaeil Ahn","Jennifer Clarke","Timo Erkkilä","Wenjun Ju","Bo Li","David Venet","Christopher R Bolen","Wenlian Qiao","Dirk Repsilber","Jason E Shoemaker","Debashis Ghosh","Niya Wang","Gerald Quon","Niya Wang","Harri Lähdesmäki","Jitao David Zhang","Neta S. Zuckerman","Min Wang","Mark M. Gosink","Julien Racle","David A. Liebner","Amit Frishberg","Yael Steuerman","Bradlee D. Nelms","Max Schelker","Qi Shen","David Tamborero","Casey P. Shannon","Amit Frishberg","Francesco Vallania","Francesca Finotello","Oyetunji E. Ogundijo","Gabriele Pollara","Martin Hoffmann","Nico Riedel","Sushmita Roy","Shu-Hwa Chen","Jing He","Franziska Görtler","Lee A. Newberg","Gregory J. Hunt","Urszula Czerwinska","Zeya Wang","Yuan Liu","Maxim E Zaslavsky","Ajit Johnson Nirmal","Yi Li "],[2015,2013,2010,2015,2009,2016,2011,2011,2012,2016,2013,2013,2003,2013,2017,2015,2014,2013,2010,2010,2013,2013,2001,2011,2012,2010,2012,2004,2016,2009,2014,2005,2017,2013,2006,2007,2017,2013,2016,2016,2016,2017,2016,2018,2017,2015,2017,2017,2017,2017,2006,2013,2006,2018,2018,2018,2018,2018,2018,2017,2017,2017,2016,2013],["FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","FALSE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE"],["FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE*","TRUE","TRUE","FALSE","TRUE*","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","FALSE","TRUE*","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE*","FALSE","TRUE","TRUE","TRUE","TRUE*","FALSE","TRUE","FALSE","FALSE","FLASE","FALSE","FALSE","FLASE","FALSE","TRUE","FALSE","FLASE","FALSE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE"],["Cancer transcriptome","Cancer transcriptome","Blood","detection of cancer and stroma in PDAC (TCGA)","Blood","Cancer transcriptome","Brain tissue","Blood","Blood","Cancer transcriptome","Tissue mixtures","Cancer transcriptome","yeast cell cycle","Cancer transcriptome","Cancer transcriptome","Blood","Mice blood under flu infection","Cancer purity","Cancer xenografts","Tissue mixtures","Chronic kidney disease (Cell lineages)","Cancer transcriptome","cander and normal tissue","Blood","blood","Blood","Infected lung tissue","Cancer transcriptome","yeast cell cycle","Cancer transcriptome","Cancer transcriptome","In vitro tissue mixtures","Gene expression","Cancer transcriptome","Murine mammary gland","Infected macrophages","Cancer transcriptome","in vitro tissue mlxtures","Human tissues","Lung tissue","Brain tissue","Cancer transcriptome","Tumor purity","Cancer transcriptome","Blood gene expression","Mice diseased tissues","Blood, solid tissue, disease","Cancer transcriptome","Tissue mixtures","Skin tuberculosis","Synovial tissue (cell types in silico)","Udefined","Yeast cell cycle","Blood","Cancer transcriptome","Cancer transcriptome","yeast cell cycle","Blood","Cancer transcriptome","Cancer transcriptome","Cancer transcriptome","Cancer transcriptome","Cancer transcriptome","in vitro cell mixtures"],["<a href=\"http://cibersort.stanford.edu/\">http://cibersort.stanford.edu/<\/a>","<a href=\"https://sourceforge.net/projects/estimateproject/\">https://sourceforge.net/projects/estimateproject/<\/a>","<a href=\"https://github.com/shenorrLab/csSAM\">https://github.com/shenorrLab/csSAM<\/a>","NA","NA","<a href=\"https://github.com/ebecht/MCPcounter\">https://github.com/ebecht/MCPcounter<\/a>","<a href=\"https://bioconductor.org/packages/release/bioc/html/PSEA.html\">https://bioconductor.org/packages/release/bioc/html/PSEA.html<\/a>","NA","<a href=\"https://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix/\">https://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix/<\/a>","NA","<a href=\"https://www.bioconductor.org/packages/release/bioc/html/DeconRNASeq.html\">https://www.bioconductor.org/packages/release/bioc/html/DeconRNASeq.html<\/a>","<a href=\"https://github.com/zhandong/DSA\">https://github.com/zhandong/DSA<\/a>","broken link","<a href=\"https://qlab.faculty.ucdavis.edu/isopure/\">https://qlab.faculty.ucdavis.edu/isopure/<\/a>","<a href=\"http://xcell.ucsf.edu/; https://github.com/dviraran/xCell\">http://xcell.ucsf.edu/; https://github.com/dviraran/xCell<\/a>","<a href=\"http://www.pitt.edu/~mchikina/CellCODE/\">http://www.pitt.edu/~mchikina/CellCODE/<\/a>","<a href=\"http://www.dcq.tau.ac.il/\">http://www.dcq.tau.ac.il/<\/a>","<a href=\"http://odin.mdacc.tmc.edu/∼wwang7/DeMix.html.\">http://odin.mdacc.tmc.edu/∼wwang7/DeMix.html.<\/a>","NA","<a href=\"http://informatics.systemsbiology.net/DSection\">http://informatics.systemsbiology.net/DSection<\/a>","<a href=\"http://nano.princeton.edu/\">http://nano.princeton.edu/<\/a>","<a href=\"http://cistrome.org/TIMER/\">http://cistrome.org/TIMER/<\/a>","NA","<a href=\"http://clip.med.yale.edu/SPEC/\">http://clip.med.yale.edu/SPEC/<\/a>","<a href=\"https://github.com/gquon/PERT\">https://github.com/gquon/PERT<\/a>","<a href=\"https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-11-27/MediaObjects/12859_2009_3484_MOESM1_ESM.ZIP\">https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-11-27/MediaObjects/12859_2009_3484_MOESM1_ESM.ZIP<\/a>","<a href=\"http://www.influenza-x.org/~jshoemaker/cten/\">http://www.influenza-x.org/~jshoemaker/cten/<\/a>","broken link","<a href=\"http://mloss.org/software/view/437,\">http://mloss.org/software/view/437,<\/a>","<a href=\"https://qlab.faculty.ucdavis.edu/isolate/\">https://qlab.faculty.ucdavis.edu/isolate/<\/a>","<a href=\"https://www.bioconductor.org/packages/release/bioc/html/UNDO.html\">https://www.bioconductor.org/packages/release/bioc/html/UNDO.html<\/a>","NA","<a href=\"https://www.bioconductor.org/packages/release/bioc/html/BioQC.html\">https://www.bioconductor.org/packages/release/bioc/html/BioQC.html<\/a>","NA","NA","NA","<a href=\"https://github.com/GfellerLab/EPIC\">https://github.com/GfellerLab/EPIC<\/a>","<a href=\" http://sourceforge.net/projects/mmad/\"> http://sourceforge.net/projects/mmad/<\/a>","<a href=\"http://csgi.tau.ac.il/ImmQuant/\">http://csgi.tau.ac.il/ImmQuant/<\/a>","<a href=\"https://cran.r-project.org/web/packages/ComICS/index.html\">https://cran.r-project.org/web/packages/ComICS/index.html<\/a>","<a href=\"http://bioconductor.org/packages/release/bioc/html/CellMapper.html\">http://bioconductor.org/packages/release/bioc/html/CellMapper.html<\/a>","NA","<a href=\"https://github.com/zhanghfd/contamDE/\">https://github.com/zhanghfd/contamDE/<\/a>","NA","<a href=\"https://github.com/cashoes/enumerateblood\">https://github.com/cashoes/enumerateblood<\/a>","<a href=\"http://www.csgi.tau.ac.il/CoD/\">http://www.csgi.tau.ac.il/CoD/<\/a>","NA","<a href=\"http://icbi.at/software/quantiseq/doc/index.html\">http://icbi.at/software/quantiseq/doc/index.html<\/a>","<a href=\"https://github.com/moyanre/smcgenedeconv\">https://github.com/moyanre/smcgenedeconv<\/a>","<a href=\"https://github.com/MJMurray1/MDIScoring\">https://github.com/MJMurray1/MDIScoring<\/a>","NA","NA","NA","<a href=\"https://testtoolshed.g2.bx.psu.edu/repository?repository_id=6e9a9ab163e578e0&changeset_revision=e3afe097e80a\">https://testtoolshed.g2.bx.psu.edu/repository?repository_id=6e9a9ab163e578e0&changeset_revision=e3afe097e80a<\/a>","NA","NA","<a href=\"https:// github.com/GeneralElectric/CcellDdistinguisher\">https:// github.com/GeneralElectric/CcellDdistinguisher<\/a>","<a href=\"https://cran.r-project.org/package=dtangle\">https://cran.r-project.org/package=dtangle<\/a>","<a href=\"https://urszulaczerwinska.github.io/DeconICA/\">https://urszulaczerwinska.github.io/DeconICA/<\/a>","<a href=\"https://github.com/wwylab/DeMixT\">https://github.com/wwylab/DeMixT<\/a>","NA","<a href=\"https://github.com/hammerlab/infino\">https://github.com/hammerlab/infino<\/a>","NA","<a href=\"https://github.com/uci-cbcl/TEMT\">https://github.com/uci-cbcl/TEMT<\/a>"],["FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","FALSE","TRUE","FALSE","FALSE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","FALSE","FALSE"],["TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE"],[null,"purity estimation","computes DEG",null,null,null,null,null,null,"used Bindea et al. Signatures, validated with FACS, CNV and methymome",null,null,null,"purity estimation","deep deconvolution","goal improve DEG, plugs marker-genes based covariated to the SVD","comparing conditions (normal vs infection)","on log transformed data",null,null,"identifies genes","made for TCGA, other application need adaptation",null,null,null,null,"first enrichment based","improve DEG","partial profiles","designed for detection of site of origin",null,null,"heterogeneity detection",null,null,"pays attention to rare cell types","absolute quantities, requires TPM normalized data",null,null,"retruns association with eQTLS","ranks genes, computes p-value of specificity",null,"computes differential gene expression staistics",null,"speciffic to Affymetrix Gene ST, pre-trained","distinguish cell types important in a disease",null,"absolute quantities from images, strating with raw data, returns cell densities","DEG analysis","optimisation of signature genes",null,"theoretical solution","deaing with missign values and time dependency","Galaxy platform pluggable tool",null,null,"partial profiles",null,"metagene profiles",null,"DEG analysis","specalised in deep deconvolution",null,"use reads"],["regression","enrichment","regression","matrix factorisation","regression","enrichment","regression","regression","matrix factorisation","enrichment","regression","regression","regression","probabilistic","enrichment","matrix factorisation","regression","probabilistic","regression","probabilistic","regression","regression","matrix factorisation","enrichment","probabilistic","matrix factorisation","enrichment","probabilistic","convex hull","matrix factorisation","matrix factorisation","probabilistic","enrichment","matrix factorisation","regression","regression","regression","regression","regression","regression","matrix factorisation","regression","probabilistic","enrichment","probabilistic","regression","regression","regression","probabilistic","enrichment","regression","probabilistic","probabilistic","regression","probabilistic","regression","convex hull","regression","matrix factorisation","probabilistic","matrix factorisation","probabilistic","enrichment","probabilistic"],["R, web tool","R","R","matlab","R","R","R","unknown","R","R","R","R","Java 2","matlab, R","R, web tool","R- C-C++- Fortran","web tool","C, R","unknown","matlab","web tool","web tool","unknown","R","octave","R","web tool","R","R-java","matlab","R","unknown","R","matlab","unknown","unknown","R","matlab","web tool","R","R","unknown","R","unknown","R","web tool","R","web tool","matlab","R","unknown","unknown","unknown","R, web tool","R","unknown","R","R","R, matlab","R","matlab","Stan","unknown","python"],[343,266,286,86,207,42,96,76,61,40,52,52,135,44,15,28,32,38,53,52,33,33,96,39,33,41,31,66,12,37,18,45,6,18,36,30,4,11,5,5,5,3,4,1,2,4,1,1,1,1,6,2,4,0,0,0,0,0,0,0,0,0,0,0],[85.75,44.3333333333333,31.7777777777778,21.5,20.7,14,12,9.5,8.71428571428571,13.3333333333333,8.66666666666667,8.66666666666667,8.4375,7.33333333333333,7.5,7,6.4,6.33333333333333,5.88888888888889,5.77777777777778,5.5,5.5,5.33333333333333,4.875,4.71428571428571,4.55555555555556,4.42857142857143,4.4,4,3.7,3.6,3.21428571428571,3,3,2.76923076923077,2.5,2,1.83333333333333,1.66666666666667,1.66666666666667,1.66666666666667,1.5,1.33333333333333,1,1,1,0.5,0.5,0.5,0.5,0.461538461538462,0.333333333333333,0.307692307692308,0,0,0,0,0,0,0,0,0,0,0],["journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","bioRxiv","bioRxiv","journal","journal","journal","arXiv","journal","journal","bioRxiv","bioRxiv","journal","bioRxiv","NA","bioRxiv","journal","bioRxiv","bioRxiv","journal"],["TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","TRUE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>name<\/th>\n      <th>number of sources<\/th>\n      <th>data<\/th>\n      <th>type<\/th>\n      <th>method<\/th>\n      <th>doi<\/th>\n      <th>author<\/th>\n      <th>year<\/th>\n      <th>proportions.in<\/th>\n      <th>profiles.in<\/th>\n      <th>application<\/th>\n      <th>availability<\/th>\n      <th>out.profiles<\/th>\n      <th>out.proportions<\/th>\n      <th>comments<\/th>\n      <th>category<\/th>\n      <th>language<\/th>\n      <th>citations<\/th>\n      <th>pop.index<\/th>\n      <th>published<\/th>\n      <th>previously.covered<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"width":"10em","targets":[1,2,14]},{"width":"30em","targets":[4]},{"width":"5em","targets":[5,7,8,9]},{"className":"dt-right","targets":[7,17,18]}],"pageLength":20,"autoWidth":true,"scrollX":true,"order":[],"orderClasses":false,"orderCellsTop":true,"lengthMenu":[10,20,25,50,100]}},"evals":[],"jsHooks":[]}</script>

<p>The most popular language of implementation of published methods is R (49.2 %), followed by Matlab (11.11%), only one tool so far was published in Python.</p>
</div>
<div id="data-type" class="section level4">
<h4><span class="header-section-number">2.3.1.2</span> Data type</h4>
<p>Also, most of the methods were designed to work with microarray data. There is a high chance that some of them are adaptable to RNA-seq. However, a little number of older methods was tested in a different setup.
For some method, as CIBERSORT, demonstrated to work with microarray and applied commonly to RNA-seq by other researchers, the validity of results remains unclear as some studies claim that CIBERSORT performs accurately applied to RNA-seq <span class="citation">(Thorsson et al. <a href="#ref-Thorsson2018">2018</a>)</span> and other opt against it <span class="citation">(Li, Liu, and Liu <a href="#ref-Li2017">2017</a>; Tamborero et al. <a href="#ref-Tamborero2018">2018</a>)</span>.
Most of newer methods (i.e. EPIC <span class="citation">(Racle et al. <a href="#ref-Racle2017">2017</a>)</span>, quanTIseq <span class="citation">(Finotello et al. <a href="#ref-Finotello2017">2017</a>)</span> or Infino <span class="citation">(Zaslavsky et al. <a href="#ref-Zaslavsky2017">2017</a>)</span>) are specifically designed for RNA-seq TPM-normalized data. Some methods, mostly enrichment-based methods, are applicable to both technologies (i.e. xCell <span class="citation">(Aran, Hu, and Butte <a href="#ref-Aran2017">2017</a>)</span>).</p>
</div>
<div id="objectives-of-the-cell-type-deconvolution" class="section level4">
<h4><span class="header-section-number">2.3.1.3</span> Objectives of the cell-type deconvolution</h4>
<p>It is remarkable that the general aim of the cell-type deconvolution changed with time. The earlier methods aimed to improve the power of differential expression analysis though <em>purification</em> of the gene expression. For example, to compare differentially expressed genes (DEG) in T-cell from the blood under two conditions. However, the obtained purified profiles from complex mixtures were often uncertain <span class="citation">(Onuchic et al. <a href="#ref-Onuchic2016">2016</a>)</span>. Recently, the most mentioned goal of deconvolution is a quantification of proportions of different cell types, especially in the context of cancer transcriptomes motivated by redefinition of immunophenotypes discussed in the previous chapter. The most popular tissue of interest for deconvolution algorithms are cancer tissues and blood. Other applications are cell-cycle time-dependent fluctuations of yeast, brain cells, and glands .</p>
</div>
<div id="differences-between-approaches" class="section level4">
<h4><span class="header-section-number">2.3.1.4</span> Differences between approaches</h4>
<p>Mathematically speaking, I have divided methods into four categories: probabilistic, regression, matrix factorisation and convex hull depending on the nature of the approach. Most of the methods (48 - 74.6%) are working within a supervised framework, and only 20% (14) are unsupervised. The approaches will be described in detail in the following section.</p>
<p>There are numerous practical differences between the methods. <span class="citation">Shen-Orr and Gaujoux (<a href="#ref-Shen-Orr2013">2013</a>)</span> in their review of deconvolution tools grouped the tools depending on their inputs and outputs. Given the type of outputs, deconvolution can be considered as complete (proportions and cell profiles) or partial (one of those). Moreover, the inputs of the algorithms can be important to evaluate how practical the tool is. The most popular tools and the most recent tools ask for minimal input from the user: the bulk gene expression matrix, or even raw sequencing data <span class="citation">(Finotello et al. <a href="#ref-Finotello2017">2017</a>)</span>. Older methods usually request either at least approximative proportions of mixed cells or purified profiles to be provided. The newer methods include the reference profiles in the tool if necessary. Some tools, including most of purity estimation tools, demand an additional data input as normal samples or another data type such as CNA data (Timer <span class="citation">(Li and Xie <a href="#ref-Li2013">2013</a>)</span>, VoCAL <span class="citation">(Steuerman and Gat-Viks <a href="#ref-Steuerman2016">2016</a>)</span>) or image data (quanTIseq <span class="citation">(Finotello et al. <a href="#ref-Finotello2017">2017</a>)</span>). An important parameter is also a number of sources (<span class="math inline">\(k\)</span>) to which the algorithm deconvolutes the mixture. In many methods, it should be provided by the user, which can be difficult in a case of complex mixtures of human tissues. Besides, type of method can also limit the number of sources, for example, a probabilistic framework privilege lower number of sources (2-3) due to the theoretical constraints. In regression depending on provided reference the output number of estimated sources is imposed. Because of the problem of collinearity and similarity of immune cell profiles, it is hard to distinguish between cell sub-types, deconvolution into fine-grain cell subtypes is often called often deep deconvolution. Some methods (i.e., CIBERSORT, Infino, xCell) give specific attention to deconvolution of cell-subtypes. An absolute presence of a cell type in the mixture can also be an essential factor. If it is too low it can reach a detection limit, Electronic subtraction <span class="citation">(Gosink, Petrie, and Tsinoremas <a href="#ref-Gosink2007">2007</a>)</span> discuss specifically the detection of rare cel-types.</p>
</div>
<div id="computational-efficiency" class="section level4">
<h4><span class="header-section-number">2.3.1.5</span> Computational efficiency</h4>
<p>Running time and the necessary infrastructure are another way to characterize the methods. Although it is hard to compare the running time objectively simultaneously of all the tools because of the heterogeneity of methods and different datasets analysed, some tendencies can be observed. If one thinks about applying deconvolution methods to big cohorts, regression and enrichment-based methods should be well suited. As far as matrix factorisation is concerned, it depends on the implementation (i.e. R vs Matlab) and if the number of sources needs to be estimated (multiple runs for different <span class="math inline">\(k\)</span> parameter) or if a stabilisation needs to applied (multiple runs for the same <span class="math inline">\(k\)</span> parameter). Finally, probabilistic tools seem to be challenging to scale, i.e. authors of Infino admit that their pipeline is not yet applicable at high-throughput.</p>
<p>In order to let user better understand the differences between different mathematical approaches, I will introduce shortly the types of approaches used for cell-type deconvolution of transcriptomes as well as their strong and weak points.</p>
<div class="figure" style="text-align: center"><span id="fig:pubyear"></span>
<div id="5800297aecdf" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="5800297aecdf">{"x":{"data":[{"orientation":"v","width":[0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16],"y":[1,1,1,1,3,1,2,4,3,3,11,2,4,8,12,7],"text":["~year: 2001<br />~`nb. of papers`:  1","~year: 2003<br />~`nb. of papers`:  1","~year: 2004<br />~`nb. of papers`:  1","~year: 2005<br />~`nb. of papers`:  1","~year: 2006<br />~`nb. of papers`:  3","~year: 2007<br />~`nb. of papers`:  1","~year: 2009<br />~`nb. of papers`:  2","~year: 2010<br />~`nb. of papers`:  4","~year: 2011<br />~`nb. of papers`:  3","~year: 2012<br />~`nb. of papers`:  3","~year: 2013<br />~`nb. of papers`: 11","~year: 2014<br />~`nb. of papers`:  2","~year: 2015<br />~`nb. of papers`:  4","~year: 2016<br />~`nb. of papers`:  8","~year: 2017<br />~`nb. of papers`: 12","~year: 2018<br />~`nb. of papers`:  7"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(190,190,190,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[13],"y":[4.3],"text":"CIBERSORT","hovertext":"~x: 13<br />~y: 4.3","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[11],"y":[11.6],"text":"ESTIMATE<br /> CellMix toolbox","hovertext":"~x: 11<br />~y: 11.6","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[14],"y":[7.3],"text":"MCPcounter","hovertext":"~x: 14<br />~y: 7.3","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[15],"y":[12.6],"text":"xCell<br /> EPIC","hovertext":"~x: 15<br />~y: 12.6","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[7],"y":[2.3],"text":"Abbas et al.","hovertext":"~x: 7<br />~y: 2.3","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[8],"y":[4.3],"text":"csSAM","hovertext":"~x: 8<br />~y: 4.3","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":40.1826484018265,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"type":"linear","autorange":false,"range":[0.4,16.6],"tickmode":"array","ticktext":["2001","2003","2004","2005","2006","2007","2009","2010","2011","2012","2013","2014","2015","2016","2017","2018"],"tickvals":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16],"categoryorder":"array","categoryarray":["2001","2003","2004","2005","2006","2007","2009","2010","2011","2012","2013","2014","2015","2016","2017","2018"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"year","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"type":"linear","autorange":false,"range":[-0.63,13.23],"tickmode":"array","ticktext":["0","5","10"],"tickvals":[0,5,10],"categoryorder":"array","categoryarray":["0","5","10"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"nb. of papers","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"source":"A","attrs":{"580035a60bad":{"x":{},"y":{},"width":{},"type":"bar"},"58002848a65f":{"x":{},"y":{}},"580039a2bba2":{"x":{},"y":{}},"58006acca63d":{"x":{},"y":{}},"580021b62592":{"x":{},"y":{}},"58003d5ca588":{"x":{},"y":{}},"5800d77a331":{"x":{},"y":{}}},"cur_data":"580035a60bad","visdat":{"580035a60bad":["function (y) ","x"],"58002848a65f":["function (y) ","x"],"580039a2bba2":["function (y) ","x"],"58006acca63d":["function (y) ","x"],"580021b62592":["function (y) ","x"],"58003d5ca588":["function (y) ","x"],"5800d77a331":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p class="caption">
Figure 2.3: <strong>Distribution of publications of cell-type deconvolution of bulk transcriptome over the years</strong>. In red: hallmark publications. Data gathered based on PubMed and google scholar search in May 2018.
</p>
</div>

<div id="5800100e10b2" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="5800100e10b2">{"x":{"visdat":{"580018881c9b":["function () ","plotlyVisDat"]},"cur_data":"580018881c9b","attrs":{"580018881c9b":{"labels":{},"values":{},"textposition":"inside","textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"colors":["function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}"],"line":{"color":"#FFFFFF","width":1}},"alpha":1,"sizes":[10,100],"type":"pie"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"title":"a) Percentage of used approach type","xaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"yaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"data":[{"labels":["convex hull","enrichment","matrix factorisation","probabilistic","regression"],"values":[2,10,11,15,26],"textposition":["inside","inside","inside","inside","inside"],"textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","colors":["structure(function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}, apiSrc = TRUE)"],"line":{"color":"#FFFFFF","width":1}},"type":"pie","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<div id="580064e48ed0" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="580064e48ed0">{"x":{"visdat":{"580037b36da9":["function () ","plotlyVisDat"]},"cur_data":"580037b36da9","attrs":{"580037b36da9":{"labels":{},"values":{},"textposition":"inside","textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"colors":["function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}"],"line":{"color":"#FFFFFF","width":1}},"alpha":1,"sizes":[10,100],"type":"pie"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"title":"b) Percentage of supervised/unsupervised tools","xaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"yaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"data":[{"labels":["BOTH","semi-supervised","supervised","unsupervised"],"values":[1,3,47,13],"textposition":["inside","inside","inside","inside"],"textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","colors":["structure(function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}, apiSrc = TRUE)"],"line":{"color":"#FFFFFF","width":1}},"type":"pie","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<div class="figure" style="text-align: center"><span id="fig:languagesC"></span>
<div id="5800232418e7" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="5800232418e7">{"x":{"visdat":{"580069b7d1a2":["function () ","plotlyVisDat"]},"cur_data":"580069b7d1a2","attrs":{"580069b7d1a2":{"labels":{},"values":{},"textposition":"inside","textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"colors":["function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}"],"line":{"color":"#FFFFFF","width":1}},"alpha":1,"sizes":[10,100],"type":"pie"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"title":"c) Percentage of the programming languages of implementation","xaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"yaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"data":[{"labels":["Java 2","matlab","octave","python","R","Stan","unknown","web tool"],"values":[1,7,1,1,33,1,13,7],"textposition":["inside","inside","inside","inside","inside","inside","inside","inside"],"textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","colors":["structure(function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}, apiSrc = TRUE)"],"line":{"color":"#FFFFFF","width":1}},"type":"pie","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p class="caption">
Figure 2.4: <strong>Simple statistics illustrating characteristics of published cell-type deconvolution tools</strong>: <strong>a)</strong> Percentage of used approach type, <strong>b)</strong> Percentage of supervised/ unsupervised tools, <strong>c)</strong> Percentage of the programming languages of implementation. Data gathered based on pubmed and google scholar search in May 2018.
</p>
</div>

</div>
</div>
<div id="regression-based-methods" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Regression-based methods</h3>
<p>Regression models are the most popular methods for bulk gene expression deconvolution. They use estimated pure cell profiles as depending variables (or selected signature genes) that should explain the mixed profiles choosing best <span class="math inline">\(\beta\)</span> parameters (Eq. <a href="methods.html#eq:regLin">(2.3)</a>) that can be interpreted as cell proportions.</p>
<p>A standard type of regression is called linear regression. It reflects linear dependence between independent and dependent variables. The linear regression was developed in the <em>precomputer age of statistics</em> <span class="citation">(Hastie, Tibshirani, and Friedman <a href="#ref-Hastie2009">2009</a>)</span>.</p>
<p>In linear regression, we want to predict a real-valued output <span class="math inline">\(Y\)</span>, given a vector <span class="math inline">\(X^T = (X_1,X_2,… ,X_p)\)</span>. The linear regression model has the form:</p>
<span class="math display" id="eq:regLin">\[\begin{equation}
f(X) = \beta_0 + \sum_{j=1}^{p} X_j\beta_j \tag{2.3}
\end{equation}\]</span>
<p>Where the <span class="math inline">\(\beta_j\)</span>s are unknown parameters or coefficients, and <span class="math inline">\(X_j\)</span>s are the explaining variables. Given pairs of (<span class="math inline">\(x_1\)</span>,<span class="math inline">\(y_1\)</span>)…(<span class="math inline">\(x_N\)</span> ,<span class="math inline">\(y_N\)</span> ), one can estimate coefficients <span class="math inline">\(\beta\)</span> with an optimization of an objective function (also called cost function).</p>
<p>The most popular estimation method is <strong>least squares</strong>, the coefficients <span class="math inline">\(\beta = (\beta_0, \beta_1, ..., \beta_n)\)</span> are computed to minimize the residual sum of squares (RSS):</p>
<span class="math display" id="eq:rss">\[\begin{equation}
RSS(\beta) = \sum_{i = 1}^{N}(y_i - f(x_i))^2 = \sum_{i = 1}^{N}(y_i - \beta_0 - \sum_{j=1}^p x_i\beta_j)^2 \tag{2.4}
\end{equation}\]</span>
<p><strong>Ordinary least squares regression</strong> is using Eq.<a href="methods.html#eq:rss">(2.4)</a> to compute <span class="math inline">\(\beta\)</span>.</p>
<p><strong>Ridge regression</strong> (Eq.<a href="methods.html#eq:ridge">(2.5)</a>) (aka Tikhonov regularization) adds a regularizer (called <span class="math inline">\(L2\)</span> norm) to shrink the coefficients (<span class="math inline">\(\lambda \geq 1\)</span>) through imposing a penalty on their size.</p>
<span class="math display" id="eq:ridge">\[\begin{equation}
\hat{\beta}^{ridge} = \underset{\beta}{\text{argmin}}\{\sum_{i = 1}^{N}(y_i - f(x_i))^2 + \lambda\sum_{j=1}^{p}\beta^2_j\}\tag{2.5}
\end{equation}\]</span>
<p>Similarly <strong>Lasso regression</strong> (Equation <a href="methods.html#eq:lasso">(2.6)</a>) adds a regularization term to RSS (called <span class="math inline">\(L1\)</span> norm), it may set coefficients to 0 and therefore perform feature selection.</p>
<span class="math display" id="eq:lasso">\[\begin{equation}
\hat{\beta}^{ridge} = \underset{\beta}{\text{argmin}}\{\sum_{i = 1}^{N}(y_i - f(x_i))^2 + \lambda\sum_{j=1}^{p}\lvert\beta_j\rvert\}\tag{2.6}
\end{equation}\]</span>
<p>In <strong>Elastic net regression</strong> both penalties are applied.</p>
<p><strong>Support Vector Regression (SVR)</strong> is regression using <strong>Supported Vector Machines (SVM)</strong>. In SVR <span class="math inline">\(\beta\)</span> can be estimated as follows:</p>
<span class="math display" id="eq:svr1">\[\begin{equation}
H(\beta,\beta_0) = \sum_{i=1}^{N} V (y_i − f(x_i)) +\frac{λ}2\lVert\beta\rVert^2 \tag{2.7}
\end{equation}\]</span>
<p>where error is measured as follows:</p>
<span class="math display" id="eq:svr2">\[\begin{equation}
V_\epsilon(r) = \begin{cases}
    0, &amp; \text{if $\lvert r \rvert &lt; \epsilon$,}\\
    \rvert r\lvert - \epsilon, &amp; \text{otherwise}.
  \end{cases}  \tag{2.8}
\end{equation}\]</span>
<p>with <span class="math inline">\(\epsilon\)</span> being the limit of error measure, meaning errors of size less than <span class="math inline">\(\epsilon\)</span> are ignored.</p>
<p>In the SVM vocabulary, a subset of the input data that determine hyperplane boundaries are called the <strong>support vectors</strong> (Fig.<a href="methods.html#fig:svr">2.5</a>). SVR discovers a hyperplane that fits the maximal possible number of points within a constant distance, <span class="math inline">\(\epsilon\)</span>, thus performing a regression.</p>
<p>In brief, in SVR, RSS is replaced by a linear <span class="math inline">\(\epsilon\)</span>-insensitive loss function and uses <em>L</em>2-norm penalty function. There exist variants of SVR algorithm, i.e. <span class="math inline">\(\epsilon\)</span>-SVR <span class="citation">(Drucker et al. <a href="#ref-drucker1997">1997</a>)</span> and <span class="math inline">\(\nu\)</span>-SVR <span class="citation">(Schölkopf et al. <a href="#ref-Scholkopf2000">2000</a>)</span>. <span class="math inline">\(\epsilon\)</span>-SVR allows to control the error; this favors more complex models. In the <span class="math inline">\(\nu\)</span>-SVR the distance of the <span class="math inline">\(\epsilon\)</span> margin can be controlled and therefore the number of data points used for regression can be controlled. <span class="citation">Ju et al. (<a href="#ref-Ju2013">2013</a>)</span> used an SVM-based method to define cell type-specific genes. A model using <span class="math inline">\(\nu\)</span>-SVR with linear kernel was used by <span class="citation">Newman et al. (<a href="#ref-Newman2015">2015</a>)</span> in CIBERSORT.</p>
<div class="figure" style="text-align: center"><span id="fig:svr"></span>
<img src="figures-ext/vSVR.jpg" alt="Principle of the SVR regression. In SVR regression \(\epsilon\) represents the limit of error measure, input data points higher than \(+\epsilon\) or lower than \(-\epsilon\) are called support vectors. The \(\nu\) parameter in \(\nu\)-SVR regression controls the distance of training error bonds: left - lower \(\nu\) value larger bound, right - higher \(v\) margin, smaller bound. Reprinted by permission from Springer Nature (Newman et al. 2015) © 2018 Macmillan Publishers Limited, part of Springer Nature. All rights reserved." width="100%" />
<p class="caption">
Figure 2.5: <strong>Principle of the SVR regression</strong>. In SVR regression <span class="math inline">\(\epsilon\)</span> represents the limit of error measure, input data points higher than <span class="math inline">\(+\epsilon\)</span> or lower than <span class="math inline">\(-\epsilon\)</span> are called support vectors. The <span class="math inline">\(\nu\)</span> parameter in <span class="math inline">\(\nu\)</span>-SVR regression controls the distance of training error bonds: left - lower <span class="math inline">\(\nu\)</span> value larger bound, right - higher <span class="math inline">\(v\)</span> margin, smaller bound. Reprinted by permission from Springer Nature <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>)</span> © 2018 Macmillan Publishers Limited, part of Springer Nature. All rights reserved.
</p>
</div>

<p>As unconstrained optimization of the objective function can result in negative coefficients, in the context of cell-type deconvolution, authors often aim to avoid as it complicates the interpretation. Therefore, different constraints can be imposed on the <span class="math inline">\(\beta\)</span> coefficients. The most common conditions are <span class="math inline">\(\beta_0 +\beta_1+ ...+\beta_n =1\)</span> and <span class="math inline">\(\forall\beta_i \geq 0\)</span>. Solution respecting the non-negativity condition is also called non-negative least squares (NNLS) to contrast with ordinary least squares (OLS). NNLS was adopted by many authors <span class="citation">(Venet et al. <a href="#ref-Venet2001">2001</a>; Abbas et al. <a href="#ref-Abbas2009">2009</a>; Repsilber et al. <a href="#ref-Repsilber2010">2010</a>; Zuckerman et al. <a href="#ref-Zuckerman2013">2013</a>; Wang et al. <a href="#ref-Wang2016">2016</a>)</span>.</p>
<p>The task can also be solved differently from the computational perspective. <span class="citation">Lu, Nakorchevskiy, and Marcotte (<a href="#ref-Lu2003">2003</a>)</span> and <span class="citation">Wang, Master, and Chodosh (<a href="#ref-Wang2006">2006</a>)</span> propose to use simulated annealing to minimize the cost function. <span class="citation">Gong et al. (<a href="#ref-Gong2011">2011</a>)</span> proposed to solve the task using quadratic programming.</p>
<p>An extensive review on optimization of the objective function for regression methods in cell-type deconvolution was published by <span class="citation">Mohammadi et al. (<a href="#ref-Mohammadi2017">2017</a>)</span>. Authors carefully consider different possibilities of parameter choice in the loss and regularization formulations and its performance. They present as well recommendations for construction of basis matrix and data pre- and post-processing. Digital tissue deconvolution (DTD) <span class="citation">(Görtler et al. <a href="#ref-Gortler2018">2018</a>)</span> aims to train the loss function with <em>in silico</em> mixtures of single cell profiles resulting in improved performance of rare cell types (present in a small proportion). However, the training is computationally heavy, and the proper training data for bulk transcriptomes are not available.</p>
<p>Since the publication of CIBERSORT <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>)</span> some authors <span class="citation">(Chen et al. <a href="#ref-Chen2018">2018</a>; Schelker et al. <a href="#ref-Schelker2017">2017</a>)</span> used the <span class="citation">Newman et al. (<a href="#ref-Newman2015">2015</a>)</span> implementation directly with pre/post modifications or with different basis matrix or they re-implemented the SVR regression in their tools <span class="citation">(Chen et al. <a href="#ref-Chen2018">2018</a>)</span>.</p>
<p>Another recent method EPIC <span class="citation">(Racle et al. <a href="#ref-Racle2017">2017</a>)</span> introduced weights related to gene variability. In their constrained regression, they add it explicitly in the cost function modifying RSS (Eq.<a href="methods.html#eq:rss">(2.4)</a>):</p>
<span class="math display" id="eq:epic">\[\begin{equation}
RSS^{weighted} (\beta) = \sum_{i = 1}^{N}(y_i - \beta_0 - w_i \sum_{j=1}^p x_i\beta_j)^2  \tag{2.9}
\end{equation}\]</span>
<p>with the non-negativity and the sum constraints, we discussed above. The <span class="math inline">\(w_i\)</span> weights are corresponding to the variance of the given gene measure in the same cell type. It aims to give less importance to the genes variant between different measurements of the same cell-type. EPIC also allows a cell type that is not a referenced in the signature matrix with an assumption that the non-referenced cell type is equal to 1- a sum of proportions of other cell types (Eq.<a href="methods.html#eq:tumorcoeff">(2.10)</a>). Authors interpret this non-referenced cell type as the tumor fraction:</p>
<span class="math display" id="eq:tumorcoeff">\[\begin{equation}
\beta_m = 1 - \sum_{j=1}^{m-1}\beta_j \tag{2.10}
\end{equation}\]</span>
<p>An additional feature of EPIC is advanced data normalization and estimation of mRNA produced by each cell to adjust cell proportions, which was previously proposed by <span class="citation">Liebner, Huang, and Parvin (<a href="#ref-Liebner2014">2014</a>)</span> in the context of microarray data:</p>
<span class="math display" id="eq:norm">\[\begin{equation}
p_j=\alpha \frac{\beta_j}{r_j} \tag{2.11}
\end{equation}\]</span>
<p>where <span class="math inline">\(p_j\)</span> are actual cell proportions that are ‘normalized’ with empirically derived coefficient <span class="math inline">\(\alpha\)</span> and measured <span class="math inline">\(r_j\)</span> is the number of RNA nucleotides in cell type <span class="math inline">\(j\)</span>.</p>
<p>Recently CIBERSORT proposed an <em>absolute mode</em> where the proportions are not relative to the leucocyte infiltration but to the sample. It can be obtained with an assumption that the estimation of the proportion of all genes in CIBERSORT matrix is corresponding to sample purity. This functionality was not yet officially published, and it is still in experimental phase <span class="citation">(Newman, Liu, and Alizadeh <a href="#ref-Newman">2018</a>)</span>.</p>
<p>Regression methods combined with pre- and post-processing of data can result in estimation of proportions that can be interpreted directly as a percentage of cells in a mixed sample. It is an important feature hard to achieve with other methods. Some methods provide relative proportions of the immune infiltrate <span class="citation">(Newman et al. <a href="#ref-Newman2015">2015</a>)</span> and another aim to provide absolute abundance <span class="citation">(Racle et al. <a href="#ref-Racle2017">2017</a>)</span>. The absolute proportions are easily comparable between data sets and cell types. Regression-based methods are usually quite fast and can process large transcriptomic cohorts. However, as I will discuss in <a href="methods.html#Validation">Validation</a> section, they pose on the hypothesis that the reference profiles available in some context (i.e., blood) are valid in a different one (i.e., tumor) or that profiles extracted from one data type (scRNA-seq) are adapted to deconvolute bulk RNAseq. Most of the recent regression methods focused on estimating proportions and do not estimate context-specific profiles and can process as little as one sample.</p>
</div>
<div id="enrichment-based-methods" class="section level3">
<h3><span class="header-section-number">2.3.3</span> Enrichment-based methods</h3>
<table class="table table-striped table-bordered" style="font-size: 10px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:contangency">Table 2.2: </span><strong>Contingency table</strong> is the count of overlap of genes present in a certain condition (Y) vs. not present (Y-Z) and association to a pathway X (in X or not in X). The contingency table is used in the frequency based test as Fisher exact test.
</caption>
<thead>
<tr>
<th style="text-align:left;font-weight: bold;color: white;background-color: black;">
</th>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Y
</th>
<th style="text-align:center;font-weight: bold;color: white;background-color: black;">
Z-Y
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
in X
</td>
<td style="text-align:center;">
a
</td>
<td style="text-align:center;">
b
</td>
</tr>
<tr>
<td style="text-align:left;">
not in X
</td>
<td style="text-align:center;">
c
</td>
<td style="text-align:center;">
d
</td>
</tr>
</tbody>
</table>

<p>Enrichment-based methods aim to evaluate an amount of activity of a given list of genes within the data. This can be obtained by calculating a score based on gene expression. Traditionally enrichment methods were used to analyze set of DEG. Different statistical approaches were adapted: like Fisher exact test giving a p-value that estimated the chance a given list of genes is over/under present in the input list of DEGs and therefore characterize the condition vs. control expressed genes.</p>
<p>Let’s take an example; if one wants to compute enrichment in pathway X of the list of DEG genes Y with the total number of tested genes Z, a contingency table need to be constructed (Tab. <a href="methods.html#tab:contangency">2.2</a>).</p>
<p>In the Fisher exact test formula (Eq. <a href="methods.html#eq:fisher">(2.12)</a>) the <span class="math inline">\(a\)</span>, <span class="math inline">\(b\)</span>,<span class="math inline">\(c\)</span> and <span class="math inline">\(d\)</span> are the individual frequencies, i.e. number of genes in of the 2X2 contingency table, and <span class="math inline">\(N\)</span> is the total frequency (<span class="math inline">\(a + b + c + d\)</span>).</p>
<span class="math display" id="eq:fisher">\[\begin{equation}
p= \frac{( ( a + b ) ! ( c + d ) ! ( a + c ) ! ( b + d ) ! )}{a ! b ! c ! d ! N ! } \tag{2.12}
\end{equation}\]</span>
<p>Another important (&gt;14000 citations) algorithm computing such a score (enrichment score ES) is named gene set enrichment analysis (GSEA) <span class="citation">(Subramanian et al. <a href="#ref-Subramanian2005">2005</a>)</span> uses sum-statistics. The list of genes user wants to test for enrichment is usually ranked by fold change odd or p-value of DGE analysis.</p>
<p>The high score indicated high activity of genes included in the list. GSEA can also indicate an anti-activity of correlation. A variant of GSEA, single sample GSEA (ssGSEA) <span class="citation">(Barbie et al. <a href="#ref-Barbie2009">2009</a>)</span> was used by <span class="citation">Şenbabaoğlu et al. (<a href="#ref-Senbabaoglu2016">2016</a>)</span>, <span class="citation">Yoshihara et al. (<a href="#ref-Yoshihara2013">2013</a>)</span> and <span class="citation">Aran, Hu, and Butte (<a href="#ref-Aran2017">2017</a>)</span> to compute infiltration scores. In the ssGSEA genes are ranked by their absolute expression. A variance-based variant of GSEA - GSVA <span class="citation">(Hänzelmann, Castelo, and Guinney <a href="#ref-Hanzelmann2013">2013</a>)</span> was used by <span class="citation">Tamborero et al. (<a href="#ref-Tamborero2018">2018</a>)</span> for the same purpose. MCPcounter <span class="citation">(Becht et al. <a href="#ref-Becht2016">2016</a>)</span> uses an arithmetic mean of gene expression of highly specific signature genes to compute a score.</p>
<p>In this way obtained scores, are not comparable between different cell types and datasets. Therefore some authors propose normalization procedures that make the score more comparable. For instance, xCell uses a platform-specific transformation of enrichment scores. Similarly, Estimate transforms scores for TCGA though an empirically derived formula. MCPcounter authors use z-scoring to minimize platform-related differences. Unfortunately, the normalization is not directly included in the R package</p>
<p>Even though enrichment methods do not try to fit the linear model and derived scores are not mathematically conditioned to represent cell proportions; usually there can be observed a strong linear dependence. An advantage of the enrichment-based methods is the speed and possibility to include distinct signatures that can characterize cell-types and cell-states of different pathways.</p>
</div>
<div id="probabilistic-methods" class="section level3">
<h3><span class="header-section-number">2.3.4</span> Probabilistic methods</h3>
<p>The probabilistic methods share a common denominator: they aim to minimise a likelihood function of Bayes’ theorem:</p>
<span class="math display" id="eq:Bayes">\[\begin{equation}
p(y|\theta) = \frac{p(\theta|y )* p(y)}{p(\theta)} \tag{2.13}
\end{equation}\]</span>
<p>In Eq.<a href="methods.html#eq:Bayes">(2.13)</a> <span class="math inline">\(y\)</span> is our data, <span class="math inline">\(\theta\)</span> a parameter, <span class="math inline">\(p(y|\theta)\)</span> <em>posterior</em>, <span class="math inline">\(p(\theta|y )\)</span> <em>likelihood</em> and <span class="math inline">\(p(\theta)\)</span> <em>prior</em>. Prior distribution is what we know about the data before it was generated and combined with a probability distribution of the observed data is called posterior distribution. The likelihood describes how likely it is to observe the data (<span class="math inline">\(y\)</span>) given the parameter <span class="math inline">\(\theta\)</span> (probability of <span class="math inline">\(y\)</span> given <span class="math inline">\(\theta\)</span> - <span class="math inline">\(p(y|\theta)\)</span>). A parameter is characteristic of a chosen model and a hyperparameter is a parameter of prior distribution.</p>
<p>In the literature, there are mainly different types of probabilistic models, one that assumes some type of distribution of mixed sources (i.e., Gaussian or Poisson), others that learn the distribution parameters empirically from a training set, another that try to find the parameters of the distribution given the number of given sources. Then in each case, there are different ways of constructing different priors and posteriors functions. Among used techniques are Markov Chain Monte Carlo or Expectation-Maximisation, which themselves can be implemented in different ways <span class="citation">(Erkkilä et al. <a href="#ref-Erkkila2010">2010</a>; Ghosh <a href="#ref-Ghosh2004">2004</a>; Lähdesmäki et al. <a href="#ref-Lahdesmaki2005">2005</a>; Li and Xie <a href="#ref-Li2013">2013</a>; Roy et al. <a href="#ref-Roy2006">2006</a>, <span class="citation">Zaslavsky et al. (<a href="#ref-Zaslavsky2017">2017</a>)</span>)</span>.</p>
<p>The probabilistic approaches are the most popular for purity estimation (2 components models), that seems to be possible to extend to 3-components model <span class="citation">(Wang et al. <a href="#ref-Wang2017">2017</a>)</span>. As far as cell-type decomposition into a number of cells is concerned, a method published on BioRxiv <em>Infino</em> uses Bayesian inference with a generative model, trained on cell type pure profiles. Authors claim their method is notably suited for deep deconvolution that is able to build cell type similarities and estimate the confidence of the estimated proportions which help to interpret the results better.</p>
<p>A probabilistic framework is an attractive approach with solid statistical bases. It can be suited to many specific cases. The pitfalls are (1) the need of prior profiles or correct hypothesis on the distribution parameters (2) reduced performance when applied to high dimensional datasets due to extensive parameters search.</p>
</div>
<div id="convex-hull-based-methods" class="section level3">
<h3><span class="header-section-number">2.3.5</span> Convex-hull based methods</h3>
<p>An emerging family of BSS methods is convex geometry (CG)-based methods. Here, the <em>sources</em> are found by searching the facets of the convex hull spanned by the mapped observations solving a classical convex optimization problem <span class="citation">(Yang et al. <a href="#ref-Yang2015">2015</a>)</span>. It can be implemented in many ways <span class="citation">(Preparata and Shamos <a href="#ref-Preparata1985">1985</a>)</span>.</p>
<p><strong>Convex hull</strong> can be defined as follows <span class="citation">(Erickson, Jeff <a href="#ref-Erickson2018">2018</a>)</span>:</p>
<blockquote>
<p><em>We are given a set <span class="math inline">\(P\)</span> of <span class="math inline">\(n\)</span> points in the plane. We want to compute something called the <strong>convex hull</strong> of <span class="math inline">\(P\)</span>. Intuitively, the convex hull is what you get by driving a nail into the plane at each point and then wrapping a piece of string around the nails. More formally, the convex hull is the smallest convex polygon containing the points:</em></p>
</blockquote>
<blockquote>
<ul>
<li><p><em><strong>polygon</strong></em>: <em>A region of the plane bounded by a cycle of line segments, called <strong>edges</strong>, joined end-to-end in a cycle. Points, where two successive edges meet, are called <strong>vertices</strong>.</em></p></li>
<li><p><em><strong>convex</strong>: For any two points <span class="math inline">\(p\)</span>, <span class="math inline">\(q\)</span> inside the polygon, the line segment <span class="math inline">\(pq\)</span> is completely inside the polygon.</em></p></li>
<li><p><em><strong>smallest</strong></em>: <em>Any convex proper subset of the convex hull excludes at least one point in <span class="math inline">\(P\)</span>. This implies that every vertex of the convex hull is a point in <span class="math inline">\(P\)</span>.</em></p></li>
</ul>
</blockquote>
<div class="figure" style="text-align: center"><span id="fig:convexhull"></span>
<img src="figures-ext/convexhull.png" alt="Convex hull illustration. A set of points and its convex hull (line). Convex hull vertices are black, and interior points are white. Image reproduced after Erickson, Jeff (2018)." width="50%" />
<p class="caption">
Figure 2.6: <strong>Convex hull illustration</strong>. A set of points and its convex hull (line). Convex hull vertices are black, and interior points are white. Image reproduced after <span class="citation">Erickson, Jeff (<a href="#ref-Erickson2018">2018</a>)</span>.
</p>
</div>

<p>Convex hull methods have been used in many fields, from economics and engineering, I will discuss it with a focus on biological context to link tightly to cell-type deconvolution.</p>
<p>The central assumptions of Convex hull optimization are that the gene expression of pure cell types is non-negative and that cell type proportions are linearly independent.</p>
<p>The shapes can be fitted to a cloud of points in many ways in order to respond to given optimality criteria. A popular method introduced by <span class="citation">Shoval et al. (<a href="#ref-Shoval2012">2012</a>)</span> and applied to gene expression and morphological phenotypes of biological species employ the <strong>Pareto front</strong> concept which aims to find a set of designs that are the best trade-offs between different requirements.</p>
<p>Visually Pareto front correspond to the edge of the convex hull.</p>
<p><span class="citation">Wang et al. (<a href="#ref-Wang2013">2013</a>)</span> proposed Complex Analysis of Mixtures (CAM) method to find the Pareto front (the vertices of <span class="math inline">\(X\)</span> mixed matrix (a convex set)). In the context of the cell-type deconvolution, it can be said that “the scatter simplex of pure subpopulation expressions is compressed and rotated to form the scatter simplex of mixed expressions whose vertices coincide with cell proportions”<span class="citation">(Wang et al. <a href="#ref-Wang2016">2016</a>)</span>. In respect to the assumptions, under a noise-free scenario, novel <em>marker genes</em> can be blindly identified by locating the <em>vertices</em> of the mixed expression scatter simplex <span class="citation">(Fa-Yu Wang et al. <a href="#ref-Wang2010">2010</a>)</span>. In the figure (Fig. <a href="methods.html#fig:cam">2.7</a>), the <span class="math inline">\(a_i\)</span>’s are cell-type proportions of <span class="math inline">\(k\)</span> cell types, <span class="math inline">\(s_i\)</span> pure cell type expression and <span class="math inline">\(x_j\)</span> mixed expression in sample <span class="math inline">\(j\)</span>. Therefore the vertices correspond to the column vectors of the matrix <span class="math inline">\(A\)</span> (Eq. <a href="methods.html#eq:linear">(2.1)</a>). The genes placed in a distance <span class="math inline">\(d\)</span> from the vertices can be interpreted as marker genes.</p>
<div class="figure" style="text-align: center"><span id="fig:cam"></span>
<img src="figures-ext/cam.png" alt="Fitting gene expression data of mixed populations to a convex hull shape. The geometry of the mixing operation in scatter space that produces a compressed and rotated scatter simplex whose vertices host subpopulation-specific marker genes and corresponding to mixing proportions." width="50%" />
<p class="caption">
Figure 2.7: <strong>Fitting gene expression data of mixed populations to a convex hull shape</strong>. The geometry of the mixing operation in scatter space that produces a compressed and rotated scatter simplex whose vertices host subpopulation-specific marker genes and corresponding to mixing proportions.
</p>
</div>

<p>In the procedure suggested by <span class="citation">Wang et al. (<a href="#ref-Wang2013">2013</a>)</span>, before performing CAM, clustering (more precisely affinity propagation clustering (APC) ) is applied to the matrix in order to select genes representing clusters, called cluster centers <span class="math inline">\(g_m\)</span> and dimension reduction(PCA) is applied to the sample space. Then in order to fit a convex set, a margin-of-error should be minimized. The Eq. <a href="methods.html#eq:camErr">(2.14)</a> explains the computation of the error which computes <span class="math inline">\(L2\)</span> norm of the difference between <span class="math inline">\(g_m\)</span> possible vertices and remaining exterior clusters. All possibilities of combinations drew from <span class="math inline">\(C^M_K\)</span>, <span class="math inline">\(M\)</span> number of clusters and <span class="math inline">\(K\)</span> true vertices, are tested.</p>
<p><span class="math display" id="eq:camErr">\[
\text{given }\alpha_k \geq 0, \sum^K_{k=1}\alpha_k=1 \\
\delta_{m, \{1,...,K\} \in C^M_K }= \underset{\alpha_k}{min} \sqrt{{g_m} - \sum^K_{k=1}\alpha_kg_k} \tag{2.14}
\]</span></p>
<p>Once optimal configuration is found, the proportions are computed using standardised averaging:
<span class="math display">\[\begin{equation}
\hat{\alpha_k} = \frac{1}{n_{markers}} \sum_{i \in markers} \frac {x(i)}{\rVert x(i)\lVert}
\end{equation}\]</span>
where <span class="math inline">\(\hat{\alpha_k}\)</span> is proportion of cell type <span class="math inline">\(k\)</span>, <span class="math inline">\(n_{markers}\)</span> number of marker genes (obtained from CAM), and <span class="math inline">\(\rVert x(i)\lVert\)</span> is the <span class="math inline">\(L1\)</span> or <span class="math inline">\(L2\)</span> norm of a given marker gene <span class="math inline">\(x_i\)</span>.</p>
<p>Then the cell-type specific profiles are obtained with linear regression. Authors of CAM also propose a minimum description length (MDL) index that determines the number of sources in the mixture. It selects the <span class="math inline">\(K\)</span> minimizing the total description code length.</p>
<p>So far, the published R-Java package <em>CAM</em> does not allow to extract gene specific signatures, and it is not scalable to large cohorts (many samples). In the article, authors apply essential pre-processing steps that are not trivial to reproduce and which are not included in their tool. Authors apply CAM and validate on rather simple mixtures (tissue <em>in vitro</em> mixtures and yeast cell cycle).</p>
<p>A slightly different approach was proposed by <span class="citation">Newberg et al. (<a href="#ref-Newberg2018">2018</a>)</span>. It does not require initial dimension reduction steps or clustering before fitting the convex hull, and it is based on a probabilistic framework. The toll <em>CellDistinguisher</em> was inspired by topic modeling algorithm <span class="citation">(Arora et al. <a href="#ref-Arora2013">2013</a>)</span>. It first computes <span class="math inline">\(Q\)</span> matrix (Eq. <a href="methods.html#eq:q">(2.15)</a>). Then each row vector of <span class="math inline">\(Q\)</span> is normalized to 1 giving <span class="math inline">\(\overline{Q}\)</span> matrix. Every row of <span class="math inline">\(\overline{Q}\)</span> lies in the convex hull of the rows indexed by the cell-type specific genes. Then <span class="math inline">\(L_2\)</span> norm of each row is computes. Genes which rows have the highest norm can be used as <em>distinguishers</em> or <em>marker</em> genes. Then other runs of selections are applied after recentering the matrix to find more markers.</p>
<span class="math display" id="eq:q">\[\begin{equation}
Q=XX^T \tag{2.15}
\end{equation}\]</span>
<p>Once the set of possible distinguishers is defined, proportions and cell profiles are computed using a Bayesian framework to fit the convex hull. Authors provide a <a href="https://github.com/GeneralElectric/CellDistinguisher">user-friendly R package <em>CellDistinguisher</em></a>. Unfortunately, they do not provide any method for estimation of some sources, which is critical for source separation of complex tissues. Additionally, quantitative weights are provided only for signature genes which number can vary for different sources, and can be as small as one gene. Authors do not apply their algorithm to complex mixtures as tumor transcriptome; they establish a proof of concept with <em>in vitro</em> mixtures of tissues.</p>
<p>The convex hull-based method does not require the independence of cell types assumption, nor the non-correlation assumption which can be interesting in the setup of closely related cell types. In theory, they also allow <span class="math inline">\(k&gt;j\)</span> (more sources than samples). So far, the existing tools are not directly applicable to tumor transcriptomes.</p>
</div>
<div id="matrix-factorization-methods" class="section level3">
<h3><span class="header-section-number">2.3.6</span> Matrix factorization methods</h3>
<p>Matrix factorization is a general problem not specific to cell types deconvolution. It has been extensively used for signal processing <span class="citation">(Zinovyev et al. <a href="#ref-Zinovyev2013">2013</a>)</span>and extraction of features from images <span class="citation">(Hastie, Tibshirani, and Friedman <a href="#ref-Hastie2009">2009</a>)</span>. Matrix factorization can also be called BSS or dimension reduction. Despite quite simple statistical bases they have been proven to be able to solve quite complex problems. Many matrix factorization methods can solve the problem of Eq. <a href="methods.html#eq:linear">(2.1)</a>. They can solve it in different ways and concern different hypotheses.</p>
<p>Naturally, matrix factorization methods estimate simultaneously <span class="math inline">\(A\)</span> and <span class="math inline">\(S\)</span> matrices (cell proportions and profiles) given <span class="math inline">\(X\)</span> rectangular matrix (genes <span class="math inline">\(\times\)</span> samples) without any additional input.</p>
<div class="figure" style="text-align: center"><span id="fig:fact"></span>
<img src="figures-ext/factor.png" alt="Principle of matrix factorisation of gene expression. The gene expression matrix \(X\) is decomposed into a set of metagenes \(S\) matrix and metasamples \(A\). Number of components C is defined with parametre \(k\)." width="100%" />
<p class="caption">
Figure 2.8: <strong>Principle of matrix factorisation of gene expression</strong>. The gene expression matrix <span class="math inline">\(X\)</span> is decomposed into a set of <em>metagenes</em> <span class="math inline">\(S\)</span> matrix and <em>metasamples</em> <span class="math inline">\(A\)</span>. Number of components <em>C</em> is defined with parametre <span class="math inline">\(k\)</span>.
</p>
</div>

<div id="principal-components-analysis" class="section level4">
<h4><span class="header-section-number">2.3.6.1</span> Principal Components Analysis</h4>
<p>One of the most popular methods, <strong>Principal Components Analysis</strong> (PCA) computes projections of the variables onto the space of the eigenvectors of the empirical covariance matrix, the projections are mutually uncorrelated and ordered in variance. The principal components provide a sequence of best linear approximations to that data.</p>
<p>PCA can be computed through eigen decomposition of the covariance matrix. Covariance matrix is computed as follows:</p>
<span class="math display" id="eq:covPCA">\[\begin{equation}
\Sigma = \frac{1}{n-1}((X-\bar{x})^T(X-\bar{x})) \tag{2.16}
\end{equation}\]</span>
<p>where <span class="math inline">\(\bar{x}\)</span> is mean vector of the feature column in the data <span class="math inline">\(X\)</span>.</p>
<p>Then the matrix is decomposed to eigenvalues:
<span class="math display" id="eq:eigenvecPCA">\[\begin{equation}
\mathbf{V^{-1}\Sigma V=D} \tag{2.17}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(\mathbf{V}\)</span> is the matrix of eigenvectors and the <span class="math inline">\(\mathbf{D}\)</span> diagonal matrix of eigenvalues.</p>
<p>It can be also computed using <strong>singular value decomposition</strong> (SVD) (computationally more efficient way):</p>
<span class="math display" id="eq:svd">\[\begin{equation}
X= UDV^T \tag{2.18}
\end{equation}\]</span>
<p>Here <span class="math inline">\(U\)</span> is an <span class="math inline">\(N \times p\)</span> orthogonal matrix (<span class="math inline">\(U^T U = I_p\)</span>) whose columns <span class="math inline">\(u_j\)</span> are called the left singular vectors; <span class="math inline">\(V\)</span> is a <span class="math inline">\(p \times p\)</span> orthogonal matrix (<span class="math inline">\(V^T V = I_p\)</span>) with columns <span class="math inline">\(v_j\)</span> called the right singular vectors, and <span class="math inline">\(D\)</span> is a <span class="math inline">\(p \times p\)</span> diagonal matrix, with diagonal elements <span class="math inline">\(d_1 \geq d_2 \geq ... \geq d_p \geq 0\)</span> known as the singular values. The columns of <span class="math inline">\(UD\)</span> are called the projections of principal components of <span class="math inline">\(X\)</span> on axes.</p>
<p>PCA finds directions in which the samples are dispersed to define Principal Components. This dispersion is measured with variance, and resulting PCA components are variance-ordered.</p>
<p>As nicely described in <span class="citation">(Rutledge and Jouan-Rimbaud Bouveresse <a href="#ref-Rutledge2013">2013</a>)</span></p>
<blockquote>
<p><em>The first PC is the vector describing the direction of maximum sample dispersion. Each following PC describes the maximal remaining variability, with the additional constraint that it must be orthogonal to all the earlier PCs to avoid it contains any of the information already extracted from the data matrix. In other words, each PC extracts as much remaining variance from the data as possible. The calculated PCs are weighted sums of the original variables, the weights being elements of a so-called loadings vector. Inspection of these loadings vectors may help determine which original variables contribute most to this PC direction. However, PCs being mathematical constructs describing the directions of greatest dispersion of the samples, there is no reason for the loadings vectors to corresponding to underlying signals in the dataset. Most of the time, PCs are combinations of pure source signals and do not describe physical reality. For this reason, their interpretation can be fraught with danger.</em></p>
</blockquote>
<p>Especially in the context of the cell-type deconvolution, it can imagine that different cell-types contribute to the variance, but one PC could explain the joint variance of many cell types.</p>
<p><span class="citation">Wang et al. (<a href="#ref-Wang2015">2015</a>)</span> used SVD to compute matrix inversion in order to separate tumor from the stroma. The method was applied to tumor transcriptomes and gives purity estimation quite different from other popular enrichment-based method ESTIMATE <span class="citation">(Yoshihara et al. <a href="#ref-Yoshihara2013">2013</a>)</span>.</p>
<p><span class="citation">Nelms et al. (<a href="#ref-Nelms2016">2016</a>)</span> in CellMapper uses a semi-supervised approach based on SVD decomposition to dissect human brain bulk transcriptome. Authors define a query gene (a specific known gene), and then they decompose transcriptome into components (eigenvectors) and multiply by weights that are higher for the components correlated with the query gene. Then the matrix is transformed back to gene <span class="math inline">\(\times\)</span> samples matrix, but query signal is amplified. The point is to find marker genes that characterize the same cell-type as the query gene. Authors did not aim at the identification of cell-type proportions or cell types profiles but identification of cell-type specific markers. They underline applicability of the method to rare cell types where many markers are not available. This approach was proposed by authors to be used to prioritize candidate genes in disease susceptibility loci identified by GWAS.</p>
</div>
<div id="non-negative-matrix-factorisation" class="section level4">
<h4><span class="header-section-number">2.3.6.2</span> Non-negative matrix factorisation</h4>
<p>Non-negative matrix factorization <span class="citation">(Seung and Lee <a href="#ref-Seung1999">1999</a>)</span> is an alternative approach to principal components analysis. It requires data are non-negative and it estimates components that are non-negative as well. It finds its application in image analysis and gene expression analysis where analyzed data are non-negative. The <span class="math inline">\(N \times p\)</span> data matrix <span class="math inline">\(X\)</span> is approximated by</p>
<p><span class="math display">\[X \approx WH \]</span></p>
<p>where <span class="math inline">\(W\)</span> is <span class="math inline">\(N \times r\)</span> and <span class="math inline">\(H\)</span> is <span class="math inline">\(r \times p\)</span>, <span class="math inline">\(r ≤ max(N,p)\)</span>. We assume that <span class="math inline">\(x_{ij}\)</span> ,<span class="math inline">\(w_{ik}\)</span>, <span class="math inline">\(h_{kj}\)</span><span class="math inline">\(\geq 0\)</span>.</p>
<p>Which is a special case of Eq. <a href="methods.html#eq:linear">(2.1)</a>.</p>
<p>The matrices <span class="math inline">\(W\)</span> and <span class="math inline">\(H\)</span> are found by maximizing
<span class="math display" id="eq:nmf">\[\begin{equation}
\mathcal{L}(W, H) = \sum^N_{i=1}\sum^{p}_{j=1}[x_{ij} log(WH)_{ij} − (WH)_{ij} ] \tag{2.19}
\end{equation}\]</span></p>
<p>The log-likelihood from a model in which <span class="math inline">\(x_{ij}\)</span> is drawn from a pre-defined distribution (i.e., Poisson) with a mean <span class="math inline">\((WH)_{ij}\)</span>.</p>
<p>This formula can be maximized through minimization of divergence:</p>
<span class="math display" id="eq:divNMF">\[\begin{equation}
\underset{W,H}{min} f(W,H)  = \frac{1}{2}\rVert X-WH\rVert^2_F \tag{2.20}
\end{equation}\]</span>
<p>Where <span class="math inline">\(\rVert .\rVert_F\)</span> is Frobenius norm, which can be replaced by Kullback-Leibler divergence.</p>
<p>The optimization can be done employing different methods:</p>
<ul>
<li><strong>euclidean</strong> update with multiplicative update rules, it is the classic NMF <span class="citation">(Seung and Lee <a href="#ref-Seung1999">1999</a>)</span>
<span class="math display" id="eq:euclidNMF">\[
W \leftarrow W \frac{XH^T}{WHH^T}\\H \leftarrow H\frac{W^TX}{W^TWH}  \tag{2.21} 
\]</span></li>
<li><strong>alternating least squares</strong> <span class="citation">(Paatero and Tapper <a href="#ref-Paatero1994">1994</a>)</span> where the matrices W and H are fixed alternatively</li>
<li><strong>alternating non-negative least squares</strong> using projected gradients (<span class="citation">Lin (<a href="#ref-Lin2007">2007</a>)</span>)</li>
<li><strong>convex-NMF </strong><span class="citation">(Ding, Tao Li, and Jordan <a href="#ref-Ding2010">2010</a>)</span> imposes a constraint that the columns of <span class="math inline">\(W\)</span> must lie within the column space of <span class="math inline">\(X\)</span>, i.e. <span class="math inline">\(W = XA\)</span> (where <span class="math inline">\(A\)</span> is an auxiliary adaptative weight matrix that fully determines <span class="math inline">\(W\)</span>), so that <span class="math inline">\(X=XAH\)</span>. In this method only <span class="math inline">\(H\)</span> must be non-negative.</li>
</ul>
<p>The NMF algorithms can differ in initialization method as well and even in situations where <span class="math inline">\(X = WH\)</span> holds exactly, and the decomposition may not be unique. This implies that the solution found by NMF depends on the starting values. The performance of different combinations applied to MRS data from human brain tumors can be found in <span class="citation">Ortega-Martorell et al. (<a href="#ref-OrtegaMartorell2012">2012</a>)</span>.</p>
<p><span class="citation">Brunet et al. (<a href="#ref-Brunet2004">2004</a>)</span> created an NMF Matlab toolbox and demonstrated applicability of NMF (using Kullback-Leibler divergence and euclidean multiplicative update <span class="citation">(Lee and Seung <a href="#ref-Lee2000">2000</a>)</span> to cancer transcriptomes with focus on cancer subtyping (focusing on the <span class="math inline">\(H\)</span> matrix). <span class="citation">Brunet et al. (<a href="#ref-Brunet2004">2004</a>)</span> also proposed a way to evaluate the optimal number of factors (sources) to which matrix should be decomposed.</p>
<p>NMF as imposing non-negativity in the context of decomposition of transcriptomes seems as an attractive concept as both cell profiles and cell proportions should be non-negative. It is not surprising then that some authors used NMF to perform cell-type deconvolution.</p>
<p>To my knowledge, <em>deconf</em> <span class="citation">(Repsilber et al. <a href="#ref-Repsilber2010">2010</a>)</span> was the first tool proposing NMF cell-type deconvolution of PBMC transcriptome, of considerable dimensions, 80 samples (40 control and 40 cases) of Tuberculosis. <span class="citation">Repsilber et al. (<a href="#ref-Repsilber2010">2010</a>)</span> employed random initialization and alternating non-negative least squares to minimize the model divergence. The complete deconvolution of the transcriptome was used to perform DEG analysis on the deconvoluted profiles.</p>
<p><span class="citation">Shen-Orr and Gaujoux (<a href="#ref-Shen-Orr2013">2013</a>)</span>, not only presented exhaustive literature review through implementing cell-type deconvolution methods in an R package <em>CellMix</em> <span class="citation">(Gaujoux and Seoighe <a href="#ref-Gaujoux2013">2013</a>)</span> but also proposed a semi-supervised NMF for cell-type deconvolution and published an R package implementing different NMF methods <span class="citation">(Gaujoux and Seoighe <a href="#ref-Gaujoux2010">2010</a>)</span>. The semi-supervised version of NMF proposed by <span class="citation">Gaujoux and Seoighe (<a href="#ref-Gaujoux2013">2013</a>)</span>, need a set of specific marker genes for each desired cell type. Then at initialization and after each iteration of the chosen NMF algorithm (applies to some versions of NMF <span class="citation">(Seung and Lee <a href="#ref-Seung1999">1999</a>; Brunet et al. <a href="#ref-Brunet2004">2004</a>; Pascual-Montano et al. <a href="#ref-PascualMontano2006">2006</a>)</span>, “each cell type signature has the values corresponding to markers of other cell types set to zero. The values for its own markers are left free to be updated by the algorithm’s own iterative schema”. Applying their algorithm to <em>in vitro</em> controlled dataset [GSE11058 <span class="citation">(Abbas et al. <a href="#ref-Abbas2009">2009</a>)</span>, testing selected NMF implementations and a varying number of markers per cell, authors observed the best performance with guided version of <em>brunet</em> <span class="citation">(Brunet et al. <a href="#ref-Brunet2004">2004</a>)</span> implementation.</p>
<p><span class="citation">Moffitt et al. (<a href="#ref-Moffitt2015">2015</a>)</span> applied NMF to separate tumor from stroma in pancreatic ductal carcinoma (PDAC) using multiplicative update NMF. They scaled <span class="math inline">\(H\)</span> matrix rows to 1 so that the values correspond to the proportions. Authors tested different possibilities of the number of sources (<span class="math inline">\(k\)</span>), the final number of factors was defined through hierarchical clustering on gene-by-gene consensus matrix of top 50 genes of each component.</p>
<p>Finally <span class="citation">Liu et al. (<a href="#ref-Liu2017">2017</a>)</span> proposed post-modified NMF in order to separate <em>in vitro</em> mixtures of different tissues.</p>
<p>In brief, NMF is a popular, in biology, algorithm performing source separation with non-negativity constraint. It was applied to <em>in vitro</em> cell-mixtures and blood transcriptomes, showing a satisfying accuracy of cell-type <em>in silico</em> dissection and evaluating proportions. It was also applied in cancer context. However, it did not recover cell-type specific signals but rather groups of signals that could be associated with cancer or stroma.</p>
</div>
<div id="independent-components-analysis" class="section level4">
<h4><span class="header-section-number">2.3.6.3</span> Independent Components Analysis</h4>
<p>Independent Components Analysis is written as in Eq. <a href="methods.html#eq:linear">(2.1)</a> maximizing <em>independence</em> and <em>non-Gaussianity</em> of columns of <span class="math inline">\(S\)</span>: <span class="math inline">\(S_i\)</span>. It was first formulated by <span class="citation">Herault and Jutten (<a href="#ref-Herault1986">1986</a>)</span></p>
<p>The independence can be measured with entropy, kurtosis, mutual information or negentropy measure <span class="math inline">\(J(Y_j )\)</span> <span class="citation">(Hyvärinen and Oja <a href="#ref-Hyvarinen2000">2000</a>)</span> defined by</p>
<span class="math display" id="eq:negentropy">\[\begin{equation}
J(Y_j ) = H(Z_j ) - H(Y_j ) \tag{2.22}
\end{equation}\]</span>
<p>where <span class="math inline">\(H(Y_j )\)</span> is entropy, <span class="math inline">\(Z_j\)</span> is a Gaussian random variable with the same variance as <span class="math inline">\(Y_j\)</span>. Negentropy is non-negative and measures the deviation of <span class="math inline">\(Y_j\)</span> from Gaussianity. An approximation of negentropy is used in popular implementation of <strong>FastICA</strong> <span class="citation">(Hyvärinen and Oja <a href="#ref-Hyvarinen2000">2000</a>)</span>. Other existing implementations of ICA are</p>
<ul>
<li>Infomax <span class="citation">(Bell and Sejnowski <a href="#ref-Bell1995">1995</a>)</span> using Information-Maximization that maximizes the joint entropy</li>
<li>JADE (<span class="citation">Cardoso and Souloumiac (<a href="#ref-Cardoso1993">1993</a>)</span>) on the construction of a fourth-order cumulants array from the data</li>
</ul>
<p>However, they are usually a lot slower which limits their application to a large corpus of data and <span class="citation">Teschendorff et al. (<a href="#ref-Teschendorff2007">2007</a>)</span> demonstrated that <em>FastICA</em> gives the most interpretable results.</p>
<p>Therefore, I will focus on FastICA implementation as it will be extensively used in the <a href="results.html#results">Results</a> part.</p>
<p>FastICA requires <em>prewhitening</em> of the data (centering and whitening). Centering is removing mean from each row of <span class="math inline">\(X\)</span> (input data). Whitening is a linear transformation that columns are not correlated and have variance equal to 1.</p>
<p><strong>Prewhitenning </strong></p>
<ol style="list-style-type: decimal">
<li>Data centering
<span class="math display" id="eq:cent">\[\begin{equation}
  x_{ij} \leftarrow x_{ij} - {\frac {1}{M}}\sum_{j^{\prime }}x_{ij^{\prime }}  \tag{2.23}
  \end{equation}\]</span>
<span class="math inline">\(x_{ij}\)</span>: data point</li>
<li>Whitenning
<span class="math display" id="eq:whit">\[\begin{equation}
 \mathbf {X} \leftarrow \mathbf {E}\mathbf {D} ^{-\frac{1}{2}}\mathbf {E} ^{T}\mathbf {X} \tag{2.24}
 \end{equation}\]</span>
Where <span class="math inline">\(\mathbf {X}\)</span> - centered data, <span class="math inline">\(\mathbf {E}\)</span> is the matrix of eigenvectors, <span class="math inline">\(\mathbf{D}\)</span> is the diagonal matrix of eigenvalues</li>
</ol>
<div align="left">
<p><strong>Algorithm 1: FastICA multiple component extraction</strong></p>
<p><strong>INPUT</strong>: <span class="math inline">\(K\)</span> Number of desired components</p>
<p><strong>INPUT</strong>: <span class="math inline">\(X \in \mathbb{R}^{N \times M}\)</span> Prewhitened matrix, where each column represents an <span class="math inline">\(N\)</span>-dimensional sample, where <span class="math inline">\(K \leq N\)</span></p>
<p><strong>OUTPUT</strong>: <span class="math inline">\(A \in \mathbb{R}^{N \times K}\)</span> Un-mixing matrix where each column projects <span class="math inline">\(\mathbf {X}\)</span> onto independent component.</p>
<p><strong>OUTPUT</strong>: <span class="math inline">\(S \in \mathbb{R}^{K \times M}\)</span> Independent components matrix, with <span class="math inline">\(M\)</span> columns representing a sample with <span class="math inline">\(K\)</span> dimensions.</p>
<span class="math display" id="eq:fasticaMul">\[\begin{equation}
\tag{2.25}
\end{equation}\]</span>
<p>For <span class="math inline">\(p\gets [1, K]\)</span></p>
<p><span class="math inline">\(\mathbf{w}_{p} \gets\)</span> <em>Random vector of length</em> <span class="math inline">\(N\)</span></p>
<p><strong>WHILE</strong> <span class="math inline">\(\mathbf{w_{p}}\)</span> <em>changes</em></p>
<p>       <span class="math inline">\(\mathbf{w}_p \gets \frac{1}{M}Xg(\mathbf{w}_p^TX)^T - \frac{1}{M}g&#39;(\mathbf{w}_p^TX)1w_p\)</span></p>
<p>       <span class="math inline">\(\mathbf{w}_p \gets \mathbf{w}_p - (\sum_{j=1}^{p-1}\mathbf{w}_p^T\mathbf{w}_j\mathbf{w}_j^T)^T\)</span></p>
<p>       <span class="math inline">\(\mathbf{w}_p \gets \frac{\mathbf{w}_p}{\lVert \mathbf{w}_p \rVert}\)</span></p>
<p>where <span class="math inline">\(\mathbf {1}\)</span> is a column vector of 1’s of dimension <span class="math inline">\(M\)</span></p>
<p><strong>OUTPUT</strong>: <span class="math inline">\(A = [\mathbf{w}_1, ..., \mathbf{w}_K]\)</span></p>
<p><strong>OUTPUT</strong>: <span class="math inline">\(S = \mathbf{A}^T\mathbf{X}\)</span></p>
</div>
<p>However, the results of this algorithm (Alg. 1. <a href="methods.html#eq:fasticaMul">(2.25)</a>) are not deterministic, as the <span class="math inline">\(\mathbf{w}_{p}\)</span> initial vector of weights is generated at random in the iterations of fastICA. If ICA is run multiple times, one can measure <strong>stability</strong> of a component. Stability of an independent component, regarding varying the initial starts of the ICA algorithm, is a measure of internal compactness of a cluster of matched independent components produced in multiple ICA runs for the same dataset and with the same parameter set but with random initialization <span class="citation">(Himberg and Hyvarinen, n.d.)</span>.</p>
<p>The Icasso procedure can be summarized in a few steps :</p>
<ol style="list-style-type: decimal">
<li>applying multiple runs of ICA with different initializations</li>
<li>clustering the resulting components</li>
<li>defining the final result as cluster centroids</li>
<li>estimating the compactness of the clusters</li>
</ol>
<p>In brief, ICA looks for a sequence of orthogonal projections such that the projected data look as far from Gaussian as possible. ICA starts from a factor analysis solution and looks for rotations that lead to independent components.</p>
<p>So far, ICA was used to deconvolute transcriptomes into biological functions <span class="citation">(Biton et al. <a href="#ref-Biton2014">2014</a>; Engreitz et al. <a href="#ref-Engreitz2010">2010</a>; Gorban <a href="#ref-Gorban2007">2007</a>; Teschendorff et al. <a href="#ref-Teschendorff2007">2007</a>; Zinovyev et al. <a href="#ref-Zinovyev2013">2013</a>)</span>. However, it has never been used for cell-type deconvolution.</p>
<p>In theory, ICA outputs: <span class="math inline">\(S\)</span> could be interpreted as sources and <span class="math inline">\(A\)</span> as proportions in the cell-type deconvolution context. In practice, the fact that the ICA allows negative weights of projections, it makes the interpretation less trivial.</p>
<p>To my knowledge, my <a href="#DeconICA">DeconICA</a> R-package (that will be described in the results part) is the first method allowing interpretation of ICA-based signals as cell-type context-specific signatures and quantify their abundance in the transcriptome.</p>
<div class="figure" style="text-align: center"><span id="fig:matrixfact"></span>
<img src="figures-ext/bss.png" alt="Simple illustration of matrix factorisation methods. Adapted with permision from (Zinovyev et al. 2013)" width="80%" />
<p class="caption">
Figure 2.9: <strong>Simple illustration of matrix factorisation methods</strong>. Adapted with permision from <span class="citation">(Zinovyev et al. <a href="#ref-Zinovyev2013">2013</a>)</span>
</p>
</div>

<p>All in all, matrix factorization methods are able to decompose a gene expression matrix into a weighted set of genes (metagene)(<span class="math inline">\(S\)</span>) and weighted set of samples (metasample <span class="math inline">\(A\)</span>). Discussed here PCA, NMF and ICA differ in constraints and starting hypotheses. PCA components are ordered by variance and are orthogonal in the initial space of data (Fig. <a href="methods.html#fig:matrixfact">2.9</a> ). NMF impose non-negativity constraint and ICA independence of sources hypothesis. Components of NMF and ICA are not ordered. For all the matrix factorization methods number of components (or factors) (<span class="math inline">\(k\)</span>) needs to be given to the algorithm. Some authors propose a way to estimate the optimal number of components usually justified in a specific context. NMF and SVD were applied in the context of cell-type deconvolution while ICA, so far, was used to dissect transcriptome into factors related to signaling pathways, technical biases or clinical features. Also, ICA was proven to find reproducible signals between different datasets <span class="citation">(Cantini et al. <a href="#ref-Cantini2018">2018</a>; Teschendorff et al. <a href="#ref-Teschendorff2007">2007</a>)</span>. I am going to discuss this aspect of the <a href="results.html#results-1">Results</a> section.</p>
</div>
</div>
<div id="attractor-metagenes" class="section level3">
<h3><span class="header-section-number">2.3.7</span> Attractor metagenes</h3>
<p>A method proposed by <span class="citation">Cheng, Yang, and Anastassiou (<a href="#ref-Cheng2013">2013</a>)</span> that can be run in semi-supervised or unsupervised mode is called attractor metagenes. Authors describe their rationale as follows:</p>
<blockquote>
<p><em>We can first define a consensus metagene from the average expression levels of all genes in the cluster, and rank all the individual genes in terms of their association (defined numerically by some form of correlation) with that metagene. We can then replace the member genes of the cluster with an equal number of the top-ranked genes. Some of the original genes may naturally remain as members of the cluster, but some may be replaced, as this process will “attract” some other genes that are more strongly correlated with the cluster. We can now define a new metagene defined by the average expression levels of the genes in the newly defined cluster, and re-rank all the individual genes concerning their association with that new metagene; and so on. It is intuitively reasonable to expect that this iterative process will eventually converge to a cluster that contains precisely the genes that are most associated with the metagene of the same cluster so that any other individual genes will be less strongly associated with the metagene. We can think of this particular cluster defined by the convergence of this iterative process as an “attractor,” i.e., a module of co-expressed genes to which many other gene sets with close but not identical membership will converge using the same computational methodology.</em></p>
</blockquote>
<p>Which in pseudocode works as described in Algorithm 2 <a href="methods.html#eq:attr">(2.26)</a> and it is implemented in R code is available online in <a href="https://www.synapse.org/#!Synapse:syn1446295">Synapse portal</a>.</p>
<div align="left">
<p><strong>Algorithm 2: Attractor metagenes algorithm</strong></p>
<p><strong>INPUT</strong>: <span class="math inline">\(\alpha\)</span> shrinkage parameter</p>
<p><strong>INPUT</strong>: <span class="math inline">\(X \in \mathbb{R}^{N \times M}\)</span> gene expression matrix</p>
<p><strong>OUTPUT</strong>: <span class="math inline">\(m_j\)</span> metagene of <span class="math inline">\(g_{seed}\)</span></p>
<span class="math display" id="eq:attr">\[\begin{equation}
\tag{2.26}
\end{equation}\]</span>
<p><span class="math inline">\(g_{seed} \gets\)</span> <em>a gene from</em> <span class="math inline">\(1:N\)</span></p>
<p><span class="math inline">\(I^{\alpha}(g_{seed}; g_i)\)</span> <em># compute association beteen <span class="math inline">\(g_{seed}\)</span> and <span class="math inline">\(g_i\)</span></em></p>
<p><span class="math inline">\(w_i = f(I^{\alpha}(g_{seed}; g_i))\)</span> <em># compute weights for each gene</em></p>
<p><span class="math inline">\(m_0 = \frac{\sum^N-1_{i=1}(g_iw_i)}{\sum^N-1_{i=1}w_i}\)</span> <em># compute metagene as weighted average of all genes</em></p>
<p><span class="math inline">\(I^{\alpha}(m_0; g_i)\)</span> <em># compute association between metagene <span class="math inline">\(m_0\)</span> and each gene <span class="math inline">\(g_i\)</span></em></p>
<p><strong>REPEAT</strong></p>
<p>       <span class="math inline">\(w_i = f(I^{\alpha}(m_0; g_i))\)</span></p>
<p>       <span class="math inline">\(m_j = \frac{\sum^N-1_{i=1}(m_0w_i)}{\sum^N-1_{i=1}w_i}\)</span></p>
<p><strong>UNTIL</strong> <span class="math inline">\(m_{j+1} = m_j\)</span></p>
</div>
<p>The produced signatures’ weights are non-negative. In the original paper, the generation of tumor signatures leads to three reproducible signatures among different tumor types, including <em>leucocyte</em> metagene. Typically with the essential parameter <span class="math inline">\(\alpha=5\)</span>, they discovered typically approximately 50 to 150 resulting attractors.</p>
<p>This method was further to study breast cancer <span class="citation">(Al-Ejeh et al. <a href="#ref-AlEjeh2014">2014</a>)</span> and to SNP data <span class="citation">(Elmas et al. <a href="#ref-Elmas2016">2016</a>)</span>.</p>
<p>There is a possibility to tune the <span class="math inline">\(\alpha\)</span> parameter in order to obtain more or less metagenes that would be possibly interpretable as cell-type signatures.</p>
</div>
<div id="others-aspects" class="section level3">
<h3><span class="header-section-number">2.3.8</span> Others aspects</h3>
<p>Here I will discuss transversal aspects common to most deconvolution methods. They play the critical role in the final results and are often omitted while algorithms are published which impacts the reproducibility significantly.</p>
<div id="types-of-biological-reference" class="section level4">
<h4><span class="header-section-number">2.3.8.1</span> Types of biological reference</h4>
<p>Let us consider the case of the deconvolution where neither <span class="math inline">\(A\)</span> or <span class="math inline">\(S\)</span> are not known (Eq. <a href="methods.html#eq:linear">(2.1)</a>) (as in the case of cancer transcriptomes), and we would like to estimate cell proportions or both cell proportions and cell profiles. No matter if the method is supervised or unsupervised at some point of the protocol the biological knowledge about cell types is necessary in order to either derive the model or interpret the data. I discussed signatures from the biological perspective in Section <a href="intro.html#immune-signatures">1.3.3</a>. Here, I would like to stress the importance of the design of gene signatures which aim is to facilitate cell-type deconvolution.</p>
<p>Depending on chosen solution different type of reference can be used. In regression algorithms, a way to approximate the individual cell-type profiles is necessary to estimate proportions. However, the genes that are the most variant between cell types are enough for regression, and not all profiles are necessary. A matrix containing gene expression characteristic for a set of cell-types, often for selected, most discriminative, genes is called <strong>basis matrix</strong>. The choice of the genes and the number of the genes impact the outcome <span class="citation">(Vallania et al. <a href="#ref-Vallania2017">2017</a>)</span> significantly. Therefore, most of the regression methods come together with a new basis matrix, ranging from hundreds to tens of genes. Typically, genes selected for basis matrix should be cell-type specific in respect to other estimated cell types, validated across many biological conditions <span class="citation">(Hoffmann et al. <a href="#ref-Hoffmann2006">2006</a>)</span>. <span class="citation">Racle et al. (<a href="#ref-Racle2017">2017</a>)</span> adds a weight directly in the regression formula (see Eq. <a href="methods.html#eq:epic">(2.9)</a> ) that corresponds to the variability of a signature gene between independent measurements of the same cell type so that the least inter-cell type variable genes have more weight in the model. <em>CellMix</em> <span class="citation">(Gaujoux and Seoighe <a href="#ref-Gaujoux2013">2013</a>)</span> regroups different indexes to select the most specific genes based on signal-to-noise ratio. However, the most popular method is the selection of differentially expressed genes between pure populations. Often criteria for the optimal number of genes in the basis matrix are not knowledge-based but data-driven. <span class="citation">Abbas et al. (<a href="#ref-Abbas2009">2009</a>)</span> uses matrix mathematical property - condition number of basis matrix (<em>kappa</em>) in order to select the number of genes. CIBERSORT and many other regression methods follow the same approach. <span class="citation">Newman et al. (<a href="#ref-Newman2015">2015</a>)</span> also added another step while constructing the basis matrix, and it preselects reference profiles having maximal discriminatory power. Some authors <span class="citation">(Ju et al. <a href="#ref-Ju2013">2013</a>; Nelms et al. <a href="#ref-Nelms2016">2016</a>)</span> propose to find marker genes through correlation with a provided marker gene (a single one or a group of genes).</p>
<p>In enrichment methods, <strong>gene list</strong> can be enough to estimate cell abundance, sometimes (i.e., GSEA) ranked gene list is necessary. The choice of extremely specific markers is crucial for accurate call-type abundance estimation. The choice of markers can also be platform-dependent, this point is strongly underlined in <span class="citation">(Becht et al. <a href="#ref-Becht2016">2016</a>)</span>. An interesting possibility is the use of gene list of different <em>cell states</em> in order obtain coarse-grain resolution.</p>
<p>The impact of missing gene from a signature in the bulk dataset remains an unanswered question. It would be logical that shorter the gene list for a specific cell, a lack of a gene can have more impact on the result. There is a need for an accurate threshold between robustness and accuracy of the method.</p>
<p>In unsupervised methods, purified cell-profiles, signatures or list of genes can be used <em><strong>a posteriori</strong></em> to interpret the obtained sources. Even though the choice of reference does not affect the original deconvolution, it affects the interpretation. The advantage of <em>a posteriori</em> interpretation is a possibility to use different sources and types of signatures in order to provide the most plausible interpretation. It is common that the way of interpretation of components is not included in the deconvolution tool <span class="citation">(Wang et al. <a href="#ref-Wang2016">2016</a>, <span class="citation">Newberg et al. (<a href="#ref-Newberg2018">2018</a>)</span>, <span class="citation">Moffitt et al. (<a href="#ref-Moffitt2015">2015</a>)</span>)</span>, even though it is a crucial part of the analysis.</p>
<p>For the deconvolution of tumoral biopsies, most of the reference profiles, up to now, are coming from the blood, which is the most available resource. Therefore most of the methods make a hypothesis that blood cell-type profiles/signatures are a correct approximation of cancer infiltrating immune cells. Rare models like PERT <span class="citation">(Qiao et al. <a href="#ref-Qiao2012">2012</a>)</span> or ImmuneStates <span class="citation">(Vallania et al. <a href="#ref-Vallania2017">2017</a>)</span> discuss the perturbation of the blood-derived profiles in diseases.</p>
<p>With the availability of single-cell RNA-seq of human cancers <span class="citation">(Chung et al. <a href="#ref-Chung2017">2017</a>; Lavin et al. <a href="#ref-Lavin2017">2017</a>; Li, Liu, and Liu <a href="#ref-Li2017">2017</a>; Puram et al. <a href="#ref-Puram2017">2017</a>; Schelker et al. <a href="#ref-Schelker2017">2017</a>; Tirosh et al. <a href="#ref-Tirosh2016">2016</a>; Zheng et al. <a href="#ref-Zheng2017">2017</a>)</span>, we gain more knowledge on immune cells in TME, and there is growing evidence that they differ importantly from blood immune cells. <span class="citation">Racle et al. (<a href="#ref-Racle2017">2017</a>)</span> show that lymph node-resident immune cells have expression profile closer to blood immune cells than cancer immune cells. <span class="citation">Schelker et al. (<a href="#ref-Schelker2017">2017</a>)</span> shows, using a synthetic bulk dataset that using single cell profiles with existing regression methods (CIBERSORT) can improve their performance in the cancer context. However, availability of scRNA-seq remains succinct and probably do not embrace the patient heterogeneity that can be found in large bulk transcriptome cohorts.</p>
</div>
<div id="data-processing" class="section level4">
<h4><span class="header-section-number">2.3.8.2</span> Data processing</h4>
<p>Data pre- and post-processing can have a substantial impact on the deconvolution. Many authors apply strong filtering of genes <span class="citation">(Wang et al. <a href="#ref-Wang2016">2016</a>)</span>, removing probes with the lowest and the highest expression (potential outliers). In many cases, data preprocessing is not detailed and therefore impossible to reproduce.</p>
<p>There is also a debate on the data normalization.</p>
<p>In microarray analysis pipeline, data are transformed into log-space, usually <span class="math inline">\(log_2(x+1)\)</span> in order to make the distribution closer to Gaussian and therefore facilitate statistical hypothesis testing <span class="citation">(<span class="citeproc-not-found" data-reference-id="Hoyle2002"><strong>???</strong></span>)</span>. It also stabilizes the variance <span class="citation">(<span class="citeproc-not-found" data-reference-id="Tsai2003"><strong>???</strong></span>)</span>.</p>
<p>Most of the authors suggest to use counts (not transformed into log-space) for estimating cell abundance as log-transformed data violate the linearity assumption <span class="citation">(Zhong et al. <a href="#ref-Zhong2013">2013</a>)</span>, some opt against it <span class="citation">(Shannon et al. <a href="#ref-Shannon2017">2017</a>; Clarke, Seo, and Clarke <a href="#ref-Clarke2010">2010</a>)</span>, and some envisage both possibilities <span class="citation">(Erkkilä et al. <a href="#ref-Erkkila2010">2010</a>; Repsilber et al. <a href="#ref-Repsilber2010">2010</a>)</span>.</p>
<p>For the RNA-seq data TPM (transcripts per million) normalization is preferred or even required by most methods <span class="citation">(Chen et al. <a href="#ref-Chen2018">2018</a>; Finotello et al. <a href="#ref-Finotello2017">2017</a>; Racle et al. <a href="#ref-Racle2017">2017</a>)</span>. <span class="citation">(<span class="citeproc-not-found" data-reference-id="Jin2017"><strong>???</strong></span>)</span> performed an extensive analysis of RNA-seq data processing that preserves best the linearity for deconvolution suggesting that TPM normalization for deconvolution studies.</p>
<p>For matrix factorization: PCA, ICA the data are usually log-transformed and centred to reduce the influence of extreme values or outliers. NMF for cell-type deconvolution is usually applied in non-log space.</p>
</div>
<div id="Validation" class="section level4">
<h4><span class="header-section-number">2.3.8.3</span> Validation</h4>
<p>Most of algorithm validation starts with <em>in silico</em> mixtures (Fig. <a href="methods.html#fig:dataVal">2.10</a>). In published articles, the bulk transcriptome is simulated in two ways (1) mixing numerically simulated sources at defined proportions of given distribution (i.e. uniform) using linear model (for instance NMF) (2) using sampling (for instance Monte Carlo) to randomly select existing pure profiles and mixing them (additive model) at random proportions. To the obtained bulk samples, noise can be added at different steps of the simulation. Additional parameters can be defined in <em>in silico</em> mixtures, for instance, CellMix allows defining the number of marker genes (specific to only one source) for each cell type. The simulated benchmark based on single cell data was used in <span class="citation">Schelker et al. (<a href="#ref-Schelker2017">2017</a>)</span> and <span class="citation">Görtler et al. (<a href="#ref-Gortler2018">2018</a>)</span>. In this framework, simulated data was obtained through summing single cell profiles at known proportions. The main pitfall of those methods is that in the proposed simulations the gene covariance structure is not preserved. In reality, the proportions of cell types are usually not random, and some immune cell types can be correlated or anti-correlated. In addition, these simulations create a simple additive model which perfectly agrees with the linear deconvolution model. This is probably not the case of the real bulk data affected by different factors as cell cycle, technical biases, patients heterogeneity and especially cell-cell interactions.</p>
<div class="figure" style="text-align: center"><span id="fig:dataVal"></span>
<img src="figures-ext/dataVal.png" alt="From theory to practice: a simplified pipeline of model validation. The scheme reflects pipeline of data validation commonly used in transcriptome deconvolution methods validation. The project can be started from a biological problem (A) and then a way to solve the problem in mathematical model (C) is tested. Most commonly, the project starts with the model (C) then it is tested on simulated data (D). Next level of difficulty is testing the model with real data, so-called, benchmark data (B) that were generated in some biological context different from initial problem. They need to be usually normalized (E) before the model is challenged. B data are widely used as they are easily available and there is some validation available facilitating comparison. Lastly, it is assumed that if the method works fine in the context B, it will work as well in the context A, preferably accompanied by some partial validation. One can replace A by cancer transcriptomics, B by blood data or in vitro mixtures if the focus is TME bulk transcriptomics deconvolution." width="100%" />
<p class="caption">
Figure 2.10: <strong>From theory to practice: a simplified pipeline of model validation</strong>. The scheme reflects pipeline of data validation commonly used in transcriptome deconvolution methods validation. The project can be started from a biological problem (A) and then a way to solve the problem in mathematical model (C) is tested. Most commonly, the project starts with the model (C) then it is tested on simulated data (D). Next level of difficulty is testing the model with real data, so-called, benchmark data (B) that were generated in some biological context different from initial problem. They need to be usually normalized (E) before the model is challenged. B data are widely used as they are easily available and there is some validation available facilitating comparison. Lastly, it is assumed that if the method works fine in the context B, it will work as well in the context A, preferably accompanied by some partial validation. One can replace A by cancer transcriptomics, B by blood data or <em>in vitro</em> mixtures if the focus is TME bulk transcriptomics deconvolution.
</p>
</div>

<p>Naturally, algorithms validated with simulated mixtures are then validated with controlled <em>in vitro</em> mixtures of cell types or tissues mixed in known proportions. The most popular benchmark datasets are:</p>
<ul>
<li>mix of human cell lines Jurkat, THP-1, IM-9 and Raji in four different concentration in triplicates and the pure cell-line profiles (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE11058">GSE11058</a>) <span class="citation">(Abbas et al. <a href="#ref-Abbas2009">2009</a>)</span>;</li>
<li>mix of rat tissues: liver, brain, lung mixed in 11 different concentrations in triplicates and the pure tissues expression(<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE19830">GSE19830</a>) <span class="citation">(Shen-Orr et al. <a href="#ref-ShenOrr2010">2010</a>)</span></li>
</ul>
<p>Similar simple mixtures are also proposed by other authors <span class="citation">(Becht et al. <a href="#ref-Becht2016">2016</a>, <span class="citation">Kuhn et al. (<a href="#ref-Kuhn2011">2011</a>)</span>)</span>. This type of benchmark adds the complexity of possible data processing and experimental noise. However, it still follows an almost perfect additive model as the cell/tissues do not interact and they are only constituents of the mixture.</p>
<p>Several tools performed systematic benchmark using PBMC or whole blood datasets, where for a number of patients (that can be over one hundred) FACS measured proportions of selected cell types and bulk transcriptomes are available. Many such datasets can be found at <a href="http://www.immport.org/immport-open/public/home/home">IMMPORT</a> database. <span class="citation">Aran, Hu, and Butte (<a href="#ref-Aran2017">2017</a>)</span> kindly shared with scientific community two datasets with a considerable number of patients (<span class="math inline">\(\sim80\)</span> and <span class="math inline">\(\sim110\)</span>) and processed FACS data (actual proportions) on their <a href="https://github.com/dviraran/xCell/blob/master/Dev_scripts/xCell_ImmPort.zip">github repository</a>. It is still important to remember that liquid tissues are easier to deconvolute and for the tools using <em>a priori</em> reference, the reference profiles are obtained from the blood. Therefore the context remains consistent.</p>
<p>For the solid cancer tissues deconvolution, some of the tools were validated with the stained histopathology cuts using in situ-hybridisation (ISH) <span class="citation">(Kuhn et al. <a href="#ref-Kuhn2011">2011</a>)</span> or immunohistochemistry (IHC) (<span class="citation">Becht et al. (<a href="#ref-Becht2016">2016</a>)</span>). Often this method estimates a limited number of cell types and the measured abundance of pictures can also be biased by the technical issues (image/ staining quality).</p>
<p>Authors of EPIC validated their tool with paired RNAseq and Lymph node FACS-derived proportions in 4 patients (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE93722">GSE93722</a>). They also noticed that it is more straightforward to correctly evaluate lymph node immune cell types than cancer infiltrating cell types as lymph node-resident cells are more similar to the blood immune cells.</p>
<p>FACS and gene expression of blood cancer (Follicular lymphoma) were also used by <span class="citation">Newman et al. (<a href="#ref-Newman2015">2015</a>)</span> for 14 patients (unpublished data). For solid tissues, <span class="citation">Newman et al. (<a href="#ref-Newman2015">2015</a>)</span> used paired FACS and expression datasets of normal lung tissues for B-cell and CD8 and CD4 T cells of 11 patients (unpublished data).</p>
<p>Some authors proposed to cross-validate estimated proportions with estimated proportions based on a different data input (i.e., methylome) <span class="citation">(Li et al. <a href="#ref-Li2016">2016</a>, <span class="citation">Şenbabaoğlu et al. (<a href="#ref-Senbabaoglu2016">2016</a>)</span>)</span> or CNA (<span class="citation">Şenbabaoğlu et al. (<a href="#ref-Senbabaoglu2016">2016</a>)</span>). This type of validation is interesting, even though in many projects only one type of data types are available for the same samples. TCGA data is one of few exceptions.</p>
<p>Finally, a validation of deconvolution of solid cancer tissues remains incomplete as no paired expression and FACS data is available up to date.</p>
</div>
<div id="statistical-significance" class="section level4">
<h4><span class="header-section-number">2.3.8.4</span> Statistical significance</h4>
<p>A little number of tools propose a statistical significance assessment. CIBERSORT computes empirical p-values using Monte Carlo sampling. Infino authors <span class="citation">(Zaslavsky et al. <a href="#ref-Zaslavsky2017">2017</a>)</span> provide a confidence interval for the proportion estimations. This allows knowing which proportion estimation are more trustful than other.</p>
<p>Most tools compare themselves to others measuring the accuracy of the prediction, or Pearson correlation, on the benchmark datasets (described above). Often, in the idealized mixtures, methods perform well. Evaluation of their performance in cancer tissues remains unanswered without proper statistical evaluation.</p>
</div>
</div>
<div id="summary-3" class="section level3">
<h3><span class="header-section-number">2.3.9</span> Summary</h3>
<p>The field of computational transcriptome deconvolution is continuously growing. Initially used to solve simple in vitro or simulated mixtures of quite distinct ingredients, then to deconvolute blood expression data, finally applied to solid cancer tissues. In cancer research, digital quantification of cancer purity becomes a routine part of big cancer research projects <span class="citation">(Yoshihara et al. <a href="#ref-Yoshihara2013">2013</a>)</span>. Cell-type quantification, even though the validation framework and statistical significance of deconvolution tools can still be improved, seems to be considered as a popular part of an analytical pipeline of bulk tumor transcriptomes <span class="citation">(Cieślik and Chinnaiyan <a href="#ref-Cieslik2017">2017</a>)</span>. Different types of approaches try to solve the deconvolution problem, focusing on different aspects of the quantification, or proposing methodologically different approaches. Methods proposing an unsupervised solution to the deconvolution problem of transcriptomes are still underrepresented. All the tools assume a linear additive model without explicitly including the impact of possible interactions on the cell-type quantification. The tools that met the most prominent success were proven by the authors to be readily applicable to a variety of cancer datasets and reusable without an extra effort (through a programming library or web interface). The field is still waiting for a gold standard validation benchmark that would allow a fair comparison of all the tools in solid cancer tissues. It is also remarkable that the recent methods focus on quantification of the abundance of an average representation of cell-types without aspiring to deconvolute the cell-type context-specific profiles. Thanks to various cancer single-cell data and big-scale projects <span class="citation">(Regev et al. <a href="#ref-Regev2017">2017</a>)</span>, we will be able to improve the existing deconvolution approaches and finally replace the collection of bulk transcriptomes by a collection of scRNA-seq ones.</p>
</div>
</div>
<div id="otherDecon" class="section level2">
<h2><span class="header-section-number">2.4</span> Deconvolution of other data types</h2>
<p>The transcriptome data is not the unique omic data type that can be used to infer cell type proportions. Genomic and epigenomic data was used in numerous publications to perform cell-type deconvolution or estimate sample purity. I will present a general landscape of the tools and methods used for this purpose.</p>
<div id="dna-methylation-data" class="section level3">
<h3><span class="header-section-number">2.4.1</span> DNA methylation data</h3>
<p>Cell-type composition can be computed from DNA methylation data (described in <a href="intro.html#epi">Section X</a>). In EWAS (Epigenome Wide Association Studies) variation origination from cell types is considered as an important confounding factor that should be removed before comparing cases and controls and defining Differentially Methylated Positions (DMPs). <span class="citation">Teschendorff and Zheng (<a href="#ref-Teschendorff2017">2017</a>)</span> reviewed ten tools for epigenome deconvolution. Authors identify six of the described methods as reference-free (which I called in this Chapter <em>unsupervised</em>), three are regression-based, and one is semi-supervised. Another review on this topic was authored by <span class="citation">Titus et al. (<a href="#ref-Titus2017">2017</a>)</span>.</p>
<p>Unsupervised methods employed in methylome cell-type deconvolution are RefFreeEWAS <span class="citation">(Houseman, Molitor, and Marsit <a href="#ref-Houseman2014">2014</a>)</span>, SVA <span class="citation">(Leek and Storey <a href="#ref-Leek2007">2007</a>)</span> are based on SVD, ISVA based on ICA [<span class="citation">Teschendorff, Zhuang, and Widschwendter (<a href="#ref-Teschendorff2011">2011</a>)</span>) are more general methods that aim to detect and remove confounders from the data (that do not need to be necessary the cell types). RUVm <span class="citation">(Maksimovic et al. <a href="#ref-Maksimovic2015">2015</a>)</span> is a semi-supervised method using generalized least squares (GLS) regression with negative reference also used to remove <em>unwanted variation</em> from the data and could be potentially adapted to cell-type deconvolution. EWASher (<span class="citation">Zou et al. (<a href="#ref-Zou2014">2014</a>)</span>) is linear mixed model and PCA based method that corrects for cell-type composition. Similarly, ReFACTor <span class="citation">(Rahmani et al. <a href="#ref-Rahmani2016">2016</a>)</span> use sparse PCA to remove the variation due to cell-type proportions. <span class="citation">Houseman et al. (<a href="#ref-Houseman2016">2016</a>)</span> proposed RefFreeCellMix: an NMF model with convex constraints to estimate factors representing cell types and cell-type proportions and a likelihood-based method of estimating the underlying dimensionality (<span class="math inline">\(k\)</span> number of factors). A different tool MeDeCom <span class="citation">(Lutsik et al. <a href="#ref-Lutsik2017">2017</a>)</span> uses alternating non-negative least squares to fit a convex hull.</p>
<p>As far as supervised methods are concerned, EPiDISH (<em>E</em>pigenetic <em>D</em>issection of <em>I</em>ntra-<em>S</em>ample-<em>H</em>eterogeneity) R-package <span class="citation">(Teschendorff and Zheng <a href="#ref-Teschendorff2017">2017</a>)</span> includes previously published tools: Quadratic programming method using reference specific DNAse hypersensitive sites [Constrained Projection (CP) <span class="citation">(Houseman et al. <a href="#ref-Houseman2012">2012</a>)</span>), adapted to methylome deconvolution CIBERSORT algorithm (<span class="math inline">\(\nu\)</span>-SVR) and robust partial correlations (RPC) method (a form of linear regression). Reference cell-type specific profiles were obtained from the blood.</p>
<p>eFORGE <span class="citation">(Breeze et al. <a href="#ref-Breeze2016">2016</a>)</span> can detect in a list of DMPs if there is a significant cell-type effect.</p>
<p>EDec <span class="citation">(Onuchic et al. <a href="#ref-Onuchic2016">2016</a>)</span> uses DNAm to infer factors proportions using NMF and then derives factors profiles though linear regression of transcriptome data of cancer datasets. Authors identify the tumor and stroma compartments and profiles. However, they admit the error rate for profiles is quite high for most genes.</p>
<p><span class="citation">Wen et al. (<a href="#ref-Wen2016">2016</a>)</span> focused on intra-tumor heterogeneity (clonal evolution) based on DNAm data. Profiles obtained from cell lines were used in a regression model to identify the proportions of sub-clones in breast cancer data. InfiniumPurify <span class="citation">(Zheng et al. <a href="#ref-Zheng2017">2017</a>)</span> and LUMP <span class="citation">(Aran, Sirota, and Butte <a href="#ref-Aran2015">2015</a>)</span> uses DNAm to estimate sample purity.</p>
<p>The validation framework for methylation deconvolution is very similar to transcriptome ones: in silico mixtures and FACS-measured proportions of the blood. Most of the tools assume the cell composition is a factor the most contributing to the variability and therefore SVD/PCA based approaches are sufficient to correct for the variability. According to <span class="citation">Teschendorff and Zheng (<a href="#ref-Teschendorff2017">2017</a>)</span>, this assumption was not proven to hold true in solid tissues like cancer. Supervised methods have the same drawback as in the case of the transcriptome, and they use purified profiles from one context to derive cell proportions in a different context. In overall, it seems that no study proposed cell-type quantification based on methylome profiles in a pan-cancer manner.</p>
</div>
<div id="copy-number-aberrations-cna" class="section level3">
<h3><span class="header-section-number">2.4.2</span> Copy number aberrations (CNA)</h3>
<p>To my knowledge there is no method using CNA data in order to estimate cell-type composition, as CNA occur in tumor tissue and natural distinction can be made between tumor and normal cells and within tumor cells (intra-tumor).</p>
<p>Therefore, copy number aberrations can be used to estimate tumor purity and clonality. BACOM 2.0 <span class="citation">(Fu et al. <a href="#ref-Fu2015">2015</a>)</span>, ABSOLUTE <span class="citation">(Carter et al. <a href="#ref-Carter2012">2012</a>)</span>, CloneCNA <span class="citation">(Yu, Li, and Wang <a href="#ref-Yu2016">2016</a>)</span>, PureBayes <span class="citation">(Larson and Fridley <a href="#ref-Larson2013">2013</a>)</span>, CHAT <span class="citation">(Li and Li <a href="#ref-Li2014">2014</a>)</span>, ThetA <span class="citation">(Oesper, Mahmoody, and Raphael <a href="#ref-Oesper2013">2013</a>)</span>, SciClone <span class="citation">(Miller et al. <a href="#ref-Miller2014">2014</a>)</span>, Canopy <span class="citation">(Jiang et al. <a href="#ref-Jiang2016">2016</a>)</span>, PyClone <span class="citation">(Roth et al. <a href="#ref-Roth2014">2014</a>)</span>, EXPANDS <span class="citation">(Andor et al. <a href="#ref-Andor2014">2014</a>)</span> estimate tumor purity and quantify true copy numbers. <a href="https://omictools.com/tumor-purity-and-heterogeneity-category">OmicTools website</a> reports 70 tools serving this purpose and their review goes beyond the scope of my work. Most tools use tumor and normal samples, paired if possible. <a href="https://github.com/DeveauP/QuantumClone/">QuantumClone</a> seem to be the only tool that requires a few samples from the same tumor (in time or space dimension).</p>
<p><span class="citation">Aran, Sirota, and Butte (<a href="#ref-Aran2015">2015</a>)</span> published Consensus measurement of purity estimation that combines purity estimations based on different data types (available in <a href="http://www.cbioportal.org/">cBioportal</a>) using: ESTIMATE <span class="citation">(Yoshihara et al. <a href="#ref-Yoshihara2013">2013</a>)</span> (gene expression data), ABSOLUTE <span class="citation">(Carter et al. <a href="#ref-Carter2012">2012</a>)</span> (CNA), LUMP <span class="citation">(Aran, Sirota, and Butte <a href="#ref-Aran2015">2015</a>)</span> (DNAm and IHC of stained slides. Authors concluded that the estimation based on different data types highly correlate with each other, besides the IHC estimates, which suggests that IHC provides a qualitative estimation of purity.</p>
</div>
</div>
<div id="summary-of-the-chapter-1" class="section level2">
<h2><span class="header-section-number">2.5</span> Summary of the chapter</h2>
<p>A plethora of machine learning solutions has been developed to solve problems of different nature. Supervised and Unsupervised approaches can be distinguished depending if a model is provided a set of training data with known response or the algorithm works blindly trying to find patterns in the data. Some of the algorithms found an essential application in healthcare and are included in the clinical routine.</p>
<p>One of the critical problems that can, in theory, be solved with ML, is bulk omic data deconvolution. Different types of deconvolution of cancer samples can be distinguished: clonal, purity and cell-type deconvolution. Here I focused on cell-type deconvolution of transcriptome data. Through an extensive review, I presented 64 tools and divided them into categories depending on the adapted type of approach. I distinguished probabilistic, enrichment-based, regression-based, convex hull, matrix factorization and attractor metagene approaches that can be used for cell-type deconvolution. I detailed the basis of the different models and highlighted the most important features counting for cell-type deconvolution.</p>
<p>DNAm data were also used to estimate cell-type proportions. However, the heterogeneity found in methylome data resulting from the difference in cell type proportions is usually seen as a confounding factor to be removed. CNA data can be used for estimation of tumor purity and clonality.</p>
<p>In brief, for the transcriptome cell-type deconvolution, it can be observed that just a limited number of tools are usable in practice in order to deconvolute large cancer cohorts and without the need to provide hard to estimate parameters. Supervised methods applied to cancer use reference-profiles coming from a different context. Unsupervised tools, so far, are somewhat underrepresented in the field and do not offer a solution directly applicable to cancer transcriptomes of high dimensions. All of the presented methods are still waiting for consistent validation with the gold standard benchmark. This could be done if systematic data of bulk transcriptome paired with FACS-measured cell-type proportions information for many cells and in many samples were generated. Another unanswered question is the validity of the linear mixture model in the presence of cell-cell interactions.</p>
<p>There is still a room for improvement in the field in order to provide more user-friendly, accurate and precise cell-type abundance estimations.</p>
<p>A question can be asked, <strong>are cell-type proportions enough to understand tumor immune phenotypes?</strong> Can we extract more valuable information from the bulk omic data that would give useful insight to biological functions of the <em>in silico</em> dissected cells?</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Mitchell1997">
<p>Mitchell, Tom M. (Tom Michael). 1997. <em>Machine Learning</em>. McGraw-Hill.</p>
</div>
<div id="ref-Hastie2009">
<p>Hastie, Trevor, Robert Tibshirani, and Jerome Friedman. 2009. <em>The Elements of Statistical Learning</em>. Springer Series in Statistics. New York, NY: Springer New York. doi:<a href="https://doi.org/10.1007/978-0-387-84858-7">10.1007/978-0-387-84858-7</a>.</p>
</div>
<div id="ref-VanDerMaaten2008">
<p>Van Der Maaten, Laurens, and Geoffrey Hinton. 2008. “Visualizing Data using t-SNE.” <em>J. Mach. Learn. Res.</em> 9: 2579–2605. <a href="https://lvdmaaten.github.io/publications/papers/JMLR{\_}2008.pdf" class="uri">https://lvdmaaten.github.io/publications/papers/JMLR{\_}2008.pdf</a>.</p>
</div>
<div id="ref-Mcinnes2018">
<p>Mcinnes, Leland, and John Healy. 2018. “UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction.” <a href="https://arxiv.org/pdf/1802.03426.pdf" class="uri">https://arxiv.org/pdf/1802.03426.pdf</a>.</p>
</div>
<div id="ref-Becht2018">
<p>Becht, Etienne, Charles-Antoine Dutertre, Immanuel W.H. Kwok, Lai Guan Ng, Florent Ginhoux, and Evan W Newell. 2018. “Evaluation of UMAP as an alternative to t-SNE for single-cell data.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 298430. doi:<a href="https://doi.org/10.1101/298430">10.1101/298430</a>.</p>
</div>
<div id="ref-Cherry1953">
<p>Cherry, E. Colin. 1953. “Some Experiments on the Recognition of Speech, with One and with Two Ears.” <em>J. Acoust. Soc. Am.</em> 25 (5). Acoustical Society of America: 975–79. doi:<a href="https://doi.org/10.1121/1.1907229">10.1121/1.1907229</a>.</p>
</div>
<div id="ref-Shen-Orr2013">
<p>Shen-Orr, Shai S, and Renaud Gaujoux. 2013. “Computational deconvolution: extracting cell type-specific information from heterogeneous samples.” <em>Curr. Opin. Immunol.</em> 25 (5): 571–78. doi:<a href="https://doi.org/10.1016/j.coi.2013.09.015">10.1016/j.coi.2013.09.015</a>.</p>
</div>
<div id="ref-Schwartz2010">
<p>Schwartz, Russell, and Stanley E Shackney. 2010. “Applying unmixing to gene expression data for tumor phylogeny inference.” <em>BMC Bioinformatics</em> 11 (1). BioMed Central: 42. doi:<a href="https://doi.org/10.1186/1471-2105-11-42">10.1186/1471-2105-11-42</a>.</p>
</div>
<div id="ref-Venet2001">
<p>Venet, D, F Pecasse, C Maenhaut, and H Bersini. 2001. “Separation of samples into their constituents using gene expression data.” <em>Bioinformatics</em> 17 Suppl 1: S279–87. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11473019" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/11473019</a>.</p>
</div>
<div id="ref-Abbas2009">
<p>Abbas, Alexander R., Kristen Wolslegel, Dhaya Seshasayee, Zora Modrusan, and Hilary F. Clark. 2009. “Deconvolution of Blood Microarray Data Identifies Cellular Activation Patterns in Systemic Lupus Erythematosus.” Edited by Patrick Tan. <em>PLoS One</em> 4 (7). Public Library of Science: e6098. doi:<a href="https://doi.org/10.1371/journal.pone.0006098">10.1371/journal.pone.0006098</a>.</p>
</div>
<div id="ref-Newman2015">
<p>Newman, Aaron M, Chih Long Liu, Michael R Green, Andrew J Gentles, Weiguo Feng, Yue Xu, Chuong D Hoang, Maximilian Diehn, and Ash A Alizadeh. 2015. “Robust enumeration of cell subsets from tissue expression profiles.” <em>Nat. Methods</em> 12 (5). Nature Publishing Group: 453–57. doi:<a href="https://doi.org/10.1038/nmeth.3337">10.1038/nmeth.3337</a>.</p>
</div>
<div id="ref-Yoshihara2013">
<p>Yoshihara, Kosuke, Maria Shahmoradgoli, Emmanuel Martínez, Rahulsimham Vegesna, Hoon Kim, Wandaliz Torres-Garcia, Victor Treviño, et al. 2013. “Inferring tumour purity and stromal and immune cell admixture from expression data.” <em>Nat. Commun.</em> 4 (1). Nature Publishing Group: 2612. doi:<a href="https://doi.org/10.1038/ncomms3612">10.1038/ncomms3612</a>.</p>
</div>
<div id="ref-ShenOrr2010">
<p>Shen-Orr, Shai S, Robert Tibshirani, Purvesh Khatri, Dale L Bodian, Frank Staedtler, Nicholas M Perry, Trevor Hastie, Minnie M Sarwal, Mark M Davis, and Atul J Butte. 2010. “Cell type–specific gene expression differences in complex tissues.” <em>Nat. Methods</em> 7 (4). Nature Publishing Group: 287–89. doi:<a href="https://doi.org/10.1038/nmeth.1439">10.1038/nmeth.1439</a>.</p>
</div>
<div id="ref-Moffitt2015">
<p>Moffitt, Richard A, Raoud Marayati, Elizabeth L Flate, Keith E Volmar, S Gabriela Herrera Loeza, Katherine A Hoadley, Naim U Rashid, et al. 2015. “Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma.” <em>Nat. Genet.</em> 47 (10). Nature Publishing Group: 1168–78. doi:<a href="https://doi.org/10.1038/ng.3398">10.1038/ng.3398</a>.</p>
</div>
<div id="ref-Gaujoux2012">
<p>Gaujoux, Renaud, and Cathal Seoighe. 2012. “Semi-supervised Nonnegative Matrix Factorization for gene expression deconvolution: A case study.” <em>Infect. Genet. Evol.</em> 12 (5). Elsevier: 913–21. doi:<a href="https://doi.org/10.1016/J.MEEGID.2011.08.014">10.1016/J.MEEGID.2011.08.014</a>.</p>
</div>
<div id="ref-Thorsson2018">
<p>Thorsson, Vésteinn, David L Gibbs, Scott D Brown, Denise Wolf, Dante S Bortone, Tai-Hsien Ou Yang, Eduard Porta-Pardo, et al. 2018. “The Immune Landscape of Cancer.” <em>Immunity</em> 48 (4). Elsevier: 812–830.e14. doi:<a href="https://doi.org/10.1016/j.immuni.2018.03.023">10.1016/j.immuni.2018.03.023</a>.</p>
</div>
<div id="ref-Li2017">
<p>Li, Bo, Jun S. Liu, and X. Shirley Liu. 2017. “Revisit linear regression-based deconvolution methods for tumor gene expression data.” <em>Genome Biol.</em> 18 (1). BioMed Central: 127. doi:<a href="https://doi.org/10.1186/s13059-017-1256-5">10.1186/s13059-017-1256-5</a>.</p>
</div>
<div id="ref-Tamborero2018">
<p>Tamborero, David, Carlota Rubio-Perez, Ferran Muinos, Radhakrishnan Sabarinathan, Josep Maria Piulats, Aura Muntasell, Rodrigo Dienstmann, Nuria Lopez-Bigas, and Abel Gonzalez-Perez. 2018. “A pan-cancer landscape of interactions between solid tumors and infiltrating immune cell populations.” <em>Clin. Cancer Res.</em> American Association for Cancer Research, clincanres.3509.2017. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-17-3509">10.1158/1078-0432.CCR-17-3509</a>.</p>
</div>
<div id="ref-Racle2017">
<p>Racle, Julien, Kaat de Jonge, Petra Baumgaertner, Daniel E Speiser, and David Gfeller. 2017. “Simultaneous enumeration of cancer and immune cell types from bulk tumor gene expression data.” <em>Elife</em> 6. eLife Sciences Publications Limited: e26476. doi:<a href="https://doi.org/10.7554/eLife.26476">10.7554/eLife.26476</a>.</p>
</div>
<div id="ref-Finotello2017">
<p>Finotello, Francesca, Clemens Mayer, Christina Plattner, Gerhard Laschober, Dietmar Rieder, Hubert Hackl, Anne Krogsdam, et al. 2017. “quanTIseq: quantifying immune contexture of human tumors.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 223180. doi:<a href="https://doi.org/10.1101/223180">10.1101/223180</a>.</p>
</div>
<div id="ref-Zaslavsky2017">
<p>Zaslavsky, Maxim E, Jacqueline Buros Novik, Eliza J Chang, and Jeffrey E Hammerbacher. 2017. “Infino: a Bayesian hierarchical model improves estimates of immune infiltration into tumor microenvironment.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 221671. doi:<a href="https://doi.org/10.1101/221671">10.1101/221671</a>.</p>
</div>
<div id="ref-Aran2017">
<p>Aran, Dvir, Zicheng Hu, and Atul J. Butte. 2017. “xCell: digitally portraying the tissue cellular heterogeneity landscape.” <em>Genome Biol.</em> 18 (1). BioMed Central: 220. doi:<a href="https://doi.org/10.1186/s13059-017-1349-1">10.1186/s13059-017-1349-1</a>.</p>
</div>
<div id="ref-Onuchic2016">
<p>Onuchic, Vitor, Ryan J. Hartmaier, David N. Boone, Michael L. Samuels, Ronak Y. Patel, Wendy M. White, Vesna D. Garovic, et al. 2016. “Epigenomic Deconvolution of Breast Tumors Reveals Metabolic Coupling between Constituent Cell Types.” <em>Cell Rep.</em> 17 (8): 2075–86. doi:<a href="https://doi.org/10.1016/j.celrep.2016.10.057">10.1016/j.celrep.2016.10.057</a>.</p>
</div>
<div id="ref-Li2013">
<p>Li, Yi, and Xiaohui Xie. 2013. “A mixture model for expression deconvolution from RNA-seq in heterogeneous tissues.” <em>BMC Bioinforma. 2013 145</em> 14 (5). BioMed Central: S11. doi:<a href="https://doi.org/10.1186/1471-2105-14-s5-s11">10.1186/1471-2105-14-s5-s11</a>.</p>
</div>
<div id="ref-Steuerman2016">
<p>Steuerman, Yael, and Irit Gat-Viks. 2016. “Exploiting Gene-Expression Deconvolution to Probe the Genetics of the Immune System.” Edited by Gary A. Churchill. <em>PLOS Comput. Biol.</em> 12 (4). Public Library of Science: e1004856. doi:<a href="https://doi.org/10.1371/journal.pcbi.1004856">10.1371/journal.pcbi.1004856</a>.</p>
</div>
<div id="ref-Gosink2007">
<p>Gosink, Mark M., Howard T. Petrie, and Nicholas F. Tsinoremas. 2007. “Electronically subtracting expression patterns from a mixed cell population.” <em>Bioinformatics</em> 23 (24). Oxford University Press: 3328–34. doi:<a href="https://doi.org/10.1093/bioinformatics/btm508">10.1093/bioinformatics/btm508</a>.</p>
</div>
<div id="ref-drucker1997">
<p>Drucker, Harris, Christopher J C Burges, Linda Kaufman, Alex J Smola, and Vladimir Vapnik. 1997. “Support vector regression machines.” In <em>Adv. Neural Inf. Process. Syst.</em>, 155–61.</p>
</div>
<div id="ref-Scholkopf2000">
<p>Schölkopf, Bernhard, Alex J. Smola, Robert C. Williamson, and Peter L. Bartlett. 2000. “New Support Vector Algorithms.” <em>Neural Comput.</em> 12 (5). MIT Press: 1207–45. doi:<a href="https://doi.org/10.1162/089976600300015565">10.1162/089976600300015565</a>.</p>
</div>
<div id="ref-Ju2013">
<p>Ju, Wenjun, Casey S Greene, Felix Eichinger, Viji Nair, Jeffrey B Hodgin, Markus Bitzer, Young-Suk Lee, et al. 2013. “Defining cell-type specificity at the transcriptional level in human disease.” <em>Genome Res.</em> 23 (11). Cold Spring Harbor Laboratory Press: 1862–73. doi:<a href="https://doi.org/10.1101/gr.155697.113">10.1101/gr.155697.113</a>.</p>
</div>
<div id="ref-Repsilber2010">
<p>Repsilber, Dirk, Sabine Kern, Anna Telaar, Gerhard Walzl, Gillian F Black, Joachim Selbig, Shreemanta K Parida, Stefan HE Kaufmann, and Marc Jacobsen. 2010. “Biomarker discovery in heterogeneous tissue samples -taking the in-silico deconfounding approach.” <em>BMC Bioinformatics</em> 11 (1). BioMed Central: 27. doi:<a href="https://doi.org/10.1186/1471-2105-11-27">10.1186/1471-2105-11-27</a>.</p>
</div>
<div id="ref-Zuckerman2013">
<p>Zuckerman, Neta S., Yair Noam, Andrea J. Goldsmith, and Peter P. Lee. 2013. “A Self-Directed Method for Cell-Type Identification and Separation of Gene Expression Microarrays.” Edited by Teresa M. Przytycka. <em>PLoS Comput. Biol.</em> 9 (8). Public Library of Science: e1003189. doi:<a href="https://doi.org/10.1371/journal.pcbi.1003189">10.1371/journal.pcbi.1003189</a>.</p>
</div>
<div id="ref-Wang2016">
<p>Wang, Niya, Eric P. Hoffman, Lulu Chen, Li Chen, Zhen Zhang, Chunyu Liu, Guoqiang Yu, David M. Herrington, Robert Clarke, and Yue Wang. 2016. “Mathematical modelling of transcriptional heterogeneity identifies novel markers and subpopulations in complex tissues.” <em>Sci. Rep.</em> 6 (1). Nature Publishing Group: 18909. doi:<a href="https://doi.org/10.1038/srep18909">10.1038/srep18909</a>.</p>
</div>
<div id="ref-Lu2003">
<p>Lu, Peng, Aleksey Nakorchevskiy, and Edward M Marcotte. 2003. “Expression deconvolution: a reinterpretation of DNA microarray data reveals dynamic changes in cell populations.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 100 (18). National Academy of Sciences: 10370–5. doi:<a href="https://doi.org/10.1073/pnas.1832361100">10.1073/pnas.1832361100</a>.</p>
</div>
<div id="ref-Wang2006">
<p>Wang, Min, Stephen R Master, and Lewis A Chodosh. 2006. “Computational expression deconvolution in a complex mammalian organ.” <em>BMC Bioinformatics</em> 7 (1). BioMed Central: 328. doi:<a href="https://doi.org/10.1186/1471-2105-7-328">10.1186/1471-2105-7-328</a>.</p>
</div>
<div id="ref-Gong2011">
<p>Gong, Ting, Nicole Hartmann, Isaac S. Kohane, Volker Brinkmann, Frank Staedtler, Martin Letzkus, Sandrine Bongiovanni, and Joseph D. Szustakowski. 2011. “Optimal Deconvolution of Transcriptional Profiling Data Using Quadratic Programming with Application to Complex Clinical Blood Samples.” Edited by Magnus Rattray. <em>PLoS One</em> 6 (11). Public Library of Science: e27156. doi:<a href="https://doi.org/10.1371/journal.pone.0027156">10.1371/journal.pone.0027156</a>.</p>
</div>
<div id="ref-Mohammadi2017">
<p>Mohammadi, Shahin, Neta Zuckerman, Andrea Goldsmith, and Ananth Grama. 2017. “A Critical Survey of Deconvolution Methods for Separating Cell Types in Complex Tissues.” <em>Proc. IEEE</em> 105 (2): 340–66. doi:<a href="https://doi.org/10.1109/JPROC.2016.2607121">10.1109/JPROC.2016.2607121</a>.</p>
</div>
<div id="ref-Gortler2018">
<p>Görtler, Franziska, Stefan Solbrig, Tilo Wettig, Peter J. Oefner, Rainer Spang, and Michael Altenbuchinger. 2018. “Loss-function learning for digital tissue deconvolution.” <a href="http://arxiv.org/abs/1801.08447" class="uri">http://arxiv.org/abs/1801.08447</a>.</p>
</div>
<div id="ref-Chen2018">
<p>Chen, Shu-Hwa, Wen-Yu Kuo, Sheng-Yao Su, Wei-Chun Chung, Jen-Ming Ho, Henry Horng-Shing Lu, and Chung-Yen Lin. 2018. “A gene profiling deconvolution approach to estimating immune cell composition from complex tissues.” <em>BMC Bioinformatics</em> 19 (S4). BioMed Central: 154. doi:<a href="https://doi.org/10.1186/s12859-018-2069-6">10.1186/s12859-018-2069-6</a>.</p>
</div>
<div id="ref-Schelker2017">
<p>Schelker, Max, Sonia Feau, Jinyan Du, Nav Ranu, Edda Klipp, Birgit Schoeberl, Gavin MacBeath, et al. 2017. “Estimation of immune cell content in tumour tissue using single-cell RNA-seq data.” <em>Nat. Commun.</em> 8 (1). Nature Publishing Group: 2032. doi:<a href="https://doi.org/10.1038/s41467-017-02289-3">10.1038/s41467-017-02289-3</a>.</p>
</div>
<div id="ref-Liebner2014">
<p>Liebner, David A., Kun Huang, and Jeffrey D. Parvin. 2014. “MMAD: microarray microdissection with analysis of differences is a computational tool for deconvoluting cell type-specific contributions from tissue samples.” <em>Bioinformatics</em> 30 (5). Oxford University Press: 682–89. doi:<a href="https://doi.org/10.1093/bioinformatics/btt566">10.1093/bioinformatics/btt566</a>.</p>
</div>
<div id="ref-Newman">
<p>Newman, AM, CL Liu, and AA Alizadeh. 2018. “CIBERSORT- Absolute mode.” Accessed June 4. <a href="https://cibersort.stanford.edu/manual.php" class="uri">https://cibersort.stanford.edu/manual.php</a>.</p>
</div>
<div id="ref-Subramanian2005">
<p>Subramanian, Aravind, Pablo Tamayo, Vamsi K Mootha, Sayan Mukherjee, Benjamin L Ebert, Michael A Gillette, Amanda Paulovich, et al. 2005. “Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 102 (43). National Academy of Sciences: 15545–50. doi:<a href="https://doi.org/10.1073/pnas.0506580102">10.1073/pnas.0506580102</a>.</p>
</div>
<div id="ref-Barbie2009">
<p>Barbie, David A., Pablo Tamayo, Jesse S. Boehm, So Young Kim, Susan E. Moody, Ian F. Dunn, Anna C. Schinzel, et al. 2009. “Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.” <em>Nature</em> 462 (7269). Nature Publishing Group: 108–12. doi:<a href="https://doi.org/10.1038/nature08460">10.1038/nature08460</a>.</p>
</div>
<div id="ref-Senbabaoglu2016">
<p>Şenbabaoğlu, Yasin, Ron S. Gejman, Andrew G. Winer, Ming Liu, Eliezer M. Van Allen, Guillermo de Velasco, Diana Miao, et al. 2016. “Tumor immune microenvironment characterization in clear cell renal cell carcinoma identifies prognostic and immunotherapeutically relevant messenger RNA signatures.” <em>Genome Biol.</em> 17 (1). BioMed Central: 231. doi:<a href="https://doi.org/10.1186/s13059-016-1092-z">10.1186/s13059-016-1092-z</a>.</p>
</div>
<div id="ref-Hanzelmann2013">
<p>Hänzelmann, Sonja, Robert Castelo, and Justin Guinney. 2013. “GSVA: gene set variation analysis for microarray and RNA-Seq data.” <em>BMC Bioinformatics</em> 14 (1). BioMed Central: 7. doi:<a href="https://doi.org/10.1186/1471-2105-14-7">10.1186/1471-2105-14-7</a>.</p>
</div>
<div id="ref-Becht2016">
<p>Becht, Etienne, Aurélien de Reyniès, Nicolas A Giraldo, Camilla Pilati, Bénédicte Buttard, Laetitia Lacroix, Janick Selves, Catherine Sautès-Fridman, Pierre Laurent-Puig, and Wolf Herman Fridman. 2016. “Immune and Stromal Classification of Colorectal Cancer Is Associated with Molecular Subtypes and Relevant for Precision Immunotherapy.” <em>Clin. Cancer Res.</em> 22 (16). American Association for Cancer Research: 4057–66. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-15-2879">10.1158/1078-0432.CCR-15-2879</a>.</p>
</div>
<div id="ref-Erkkila2010">
<p>Erkkilä, Timo, Saara Lehmusvaara, Pekka Ruusuvuori, Tapio Visakorpi, Ilya Shmulevich, and Harri Lähdesmäki. 2010. “Probabilistic analysis of gene expression measurements from heterogeneous tissues.” <em>Bioinformatics</em> 26 (20). Oxford University Press: 2571–7. doi:<a href="https://doi.org/10.1093/bioinformatics/btq406">10.1093/bioinformatics/btq406</a>.</p>
</div>
<div id="ref-Ghosh2004">
<p>Ghosh, D. 2004. “Mixture models for assessing differential expression in complex tissues using microarray data.” <em>Bioinformatics</em> 20 (11). Oxford University Press: 1663–9. doi:<a href="https://doi.org/10.1093/bioinformatics/bth139">10.1093/bioinformatics/bth139</a>.</p>
</div>
<div id="ref-Lahdesmaki2005">
<p>Lähdesmäki, Harri, llya Shmulevich, Valerie Dunmire, Olli Yli-Harja, and Wei Zhang. 2005. “In silico microdissection of microarray data from heterogeneous cell populations.” <em>BMC Bioinformatics</em> 6 (1). BioMed Central: 54. doi:<a href="https://doi.org/10.1186/1471-2105-6-54">10.1186/1471-2105-6-54</a>.</p>
</div>
<div id="ref-Roy2006">
<p>Roy, Sushmita, Terran Lane, Chris Allen, Anthony D. Aragon, and Margaret Werner-Washburne. 2006. “A Hidden-State Markov Model for Cell Population Deconvolution.” <em>J. Comput. Biol.</em> 13 (10). Mary Ann Liebert, Inc. 2 Madison Avenue Larchmont, NY 10538 USA: 1749–74. doi:<a href="https://doi.org/10.1089/cmb.2006.13.1749">10.1089/cmb.2006.13.1749</a>.</p>
</div>
<div id="ref-Wang2017">
<p>Wang, Zeya, Shaolong Cao, Jeffrey S. Morris, Jaeil Ahn, Rongjie Liu, Svitlana Tyekucheva, Fan Gao, et al. 2017. “Transcriptome Deconvolution of Heterogeneous Tumor Samples with Immune Infiltration.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 146795. doi:<a href="https://doi.org/10.1101/146795">10.1101/146795</a>.</p>
</div>
<div id="ref-Yang2015">
<p>Yang, Zuyuan, Yong Xiang, Yue Rong, and Kan Xie. 2015. “A Convex Geometry-Based Blind Source Separation Method for Separating Nonnegative Sources.” <em>IEEE Trans. Neural Networks Learn. Syst.</em> 26 (8): 1635–44. doi:<a href="https://doi.org/10.1109/TNNLS.2014.2350026">10.1109/TNNLS.2014.2350026</a>.</p>
</div>
<div id="ref-Preparata1985">
<p>Preparata, Franco P., and Michael Ian Shamos. 1985. “Convex Hulls: Basic Algorithms.” In <em>Comput. Geom.</em>, 95–149. New York, NY: Springer New York. doi:<a href="https://doi.org/10.1007/978-1-4612-1098-6_3">10.1007/978-1-4612-1098-6_3</a>.</p>
</div>
<div id="ref-Erickson2018">
<p>Erickson, Jeff. 2018. “Jeff Erickson’s Algorithms, Etc.” Accessed June 5. <a href="http://jeffe.cs.illinois.edu/teaching/algorithms/" class="uri">http://jeffe.cs.illinois.edu/teaching/algorithms/</a>.</p>
</div>
<div id="ref-Shoval2012">
<p>Shoval, O., H. Sheftel, G. Shinar, Y. Hart, O. Ramote, A. Mayo, E. Dekel, K. Kavanagh, and U. Alon. 2012. “Evolutionary Trade-Offs, Pareto Optimality, and the Geometry of Phenotype Space.” <em>Science (80-. ).</em> 336 (6085): 1157–60. doi:<a href="https://doi.org/10.1126/science.1217405">10.1126/science.1217405</a>.</p>
</div>
<div id="ref-Wang2013">
<p>Wang, Niya, Fan Meng, Li Chen, Subha Madhavan, Robert Clarke, Eric P Hoffman, Jianhua Xuan, and Yue Wang. 2013. “The CAM Software for Nonnegative Blind Source Separation in R-Java.” <em>J. Mach. Learn. Res.</em> 14: 2899–2903. <a href="http://www.jmlr.org/papers/volume14/wang13d/wang13d.pdf" class="uri">http://www.jmlr.org/papers/volume14/wang13d/wang13d.pdf</a>.</p>
</div>
<div id="ref-Wang2010">
<p>Fa-Yu Wang, Chong-Yung Chi, Tsung-Han Chan, and Yue Wang. 2010. “Nonnegative Least-Correlated Component Analysis for Separation of Dependent Sources by Volume Maximization.” <em>IEEE Trans. Pattern Anal. Mach. Intell.</em> 32 (5): 875–88. doi:<a href="https://doi.org/10.1109/TPAMI.2009.72">10.1109/TPAMI.2009.72</a>.</p>
</div>
<div id="ref-Newberg2018">
<p>Newberg, Lee A., Xiaowei Chen, Chinnappa D. Kodira, and Maria I. Zavodszky. 2018. “Computational de novo discovery of distinguishing genes for biological processes and cell types in complex tissues.” Edited by Paolo Provero. <em>PLoS One</em> 13 (3). Public Library of Science: e0193067. doi:<a href="https://doi.org/10.1371/journal.pone.0193067">10.1371/journal.pone.0193067</a>.</p>
</div>
<div id="ref-Arora2013">
<p>Arora, Sanjeev, Rong Ge, Yoni Halpern, David Mimno, Ankur Moitra, David Sontag, Yichen Wu, and Michael Zhu. 2013. “A practical algorithm for topic modeling with provable guarantees.” JMLR.org. <a href="https://dl.acm.org/citation.cfm?id=3042925" class="uri">https://dl.acm.org/citation.cfm?id=3042925</a>.</p>
</div>
<div id="ref-Zinovyev2013">
<p>Zinovyev, Andrei, Ulykbek Kairov, Tatyana Karpenyuk, and Erlan Ramanculov. 2013. “Blind source separation methods for deconvolution of complex signals in cancer biology.” <em>Biochem. Biophys. Res. Commun.</em> 430 (3). Academic Press: 1182–7. doi:<a href="https://doi.org/10.1016/J.BBRC.2012.12.043">10.1016/J.BBRC.2012.12.043</a>.</p>
</div>
<div id="ref-Rutledge2013">
<p>Rutledge, D.N., and D. Jouan-Rimbaud Bouveresse. 2013. “Independent Components Analysis with the JADE algorithm.” <em>TrAC Trends Anal. Chem.</em> 50. Elsevier: 22–32. doi:<a href="https://doi.org/10.1016/J.TRAC.2013.03.013">10.1016/J.TRAC.2013.03.013</a>.</p>
</div>
<div id="ref-Wang2015">
<p>Wang, Niya, Ting Gong, Robert Clarke, Lulu Chen, Ie-Ming Shih, Zhen Zhang, Douglas A. Levine, Jianhua Xuan, and Yue Wang. 2015. “UNDO: a Bioconductor R package for unsupervised deconvolution of mixed gene expressions in tumor samples.” <em>Bioinformatics</em> 31 (1). Oxford University Press: 137–39. doi:<a href="https://doi.org/10.1093/bioinformatics/btu607">10.1093/bioinformatics/btu607</a>.</p>
</div>
<div id="ref-Nelms2016">
<p>Nelms, Bradlee D., Levi Waldron, Luis A. Barrera, Andrew W. Weflen, Jeremy A. Goettel, Guoji Guo, Robert K. Montgomery, et al. 2016. “CellMapper: rapid and accurate inference of gene expression in difficult-to-isolate cell types.” <em>Genome Biol.</em> 17 (1). BioMed Central: 201. doi:<a href="https://doi.org/10.1186/s13059-016-1062-5">10.1186/s13059-016-1062-5</a>.</p>
</div>
<div id="ref-Seung1999">
<p>Seung, H. Sebastian, and Daniel D. Lee. 1999. “Learning the parts of objects by non-negative matrix factorization.” <em>Nature</em> 401 (6755). Nature Publishing Group: 788–91. doi:<a href="https://doi.org/10.1038/44565">10.1038/44565</a>.</p>
</div>
<div id="ref-Paatero1994">
<p>Paatero, Pentti, and Unto Tapper. 1994. “Positive matrix factorization: A non-negative factor model with optimal utilization of error estimates of data values.” <em>Environmetrics</em> 5 (2). Wiley-Blackwell: 111–26. doi:<a href="https://doi.org/10.1002/env.3170050203">10.1002/env.3170050203</a>.</p>
</div>
<div id="ref-Lin2007">
<p>Lin, Chih-Jen. 2007. “Projected Gradient Methods for Nonnegative Matrix Factorization.” <em>Neural Comput.</em> 19 (10). MIT Press 238 Main St., Suite 500, Cambridge, MA 02142-1046 USA journals-info@mit.edu: 2756–79. doi:<a href="https://doi.org/10.1162/neco.2007.19.10.2756">10.1162/neco.2007.19.10.2756</a>.</p>
</div>
<div id="ref-Ding2010">
<p>Ding, C.H.Q., Tao Li, and M.I. Jordan. 2010. “Convex and Semi-Nonnegative Matrix Factorizations.” <em>IEEE Trans. Pattern Anal. Mach. Intell.</em> 32 (1): 45–55. doi:<a href="https://doi.org/10.1109/TPAMI.2008.277">10.1109/TPAMI.2008.277</a>.</p>
</div>
<div id="ref-OrtegaMartorell2012">
<p>Ortega-Martorell, Sandra, Paulo J.G. Lisboa, Alfredo Vellido, Margarida Julia-Sape, and Carles Arus. 2012. “Non-negative Matrix Factorisation methods for the spectral decomposition of MRS data from human brain tumours.” <em>BMC Bioinformatics</em> 13 (1). BioMed Central: 38. doi:<a href="https://doi.org/10.1186/1471-2105-13-38">10.1186/1471-2105-13-38</a>.</p>
</div>
<div id="ref-Brunet2004">
<p>Brunet, Jean-Philippe, Pablo Tamayo, Todd R Golub, and Jill P Mesirov. 2004. “Metagenes and molecular pattern discovery using matrix factorization.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 101 (12). National Academy of Sciences: 4164–9. doi:<a href="https://doi.org/10.1073/pnas.0308531101">10.1073/pnas.0308531101</a>.</p>
</div>
<div id="ref-Lee2000">
<p>Lee, Daniel D., and H. Sebastian Seung. 2000. “Algorithms for non-negative matrix factorization.” MIT Press. <a href="https://dl.acm.org/citation.cfm?id=3008829" class="uri">https://dl.acm.org/citation.cfm?id=3008829</a>.</p>
</div>
<div id="ref-Gaujoux2013">
<p>Gaujoux, R., and C. Seoighe. 2013. “CellMix: a comprehensive toolbox for gene expression deconvolution.” <em>Bioinformatics</em> 29 (17): 2211–2. doi:<a href="https://doi.org/10.1093/bioinformatics/btt351">10.1093/bioinformatics/btt351</a>.</p>
</div>
<div id="ref-Gaujoux2010">
<p>Gaujoux, Renaud, and Cathal Seoighe. 2010. “A flexible R package for nonnegative matrix factorization.” <em>BMC Bioinformatics</em> 11 (1). BioMed Central: 367. doi:<a href="https://doi.org/10.1186/1471-2105-11-367">10.1186/1471-2105-11-367</a>.</p>
</div>
<div id="ref-PascualMontano2006">
<p>Pascual-Montano, A., J.M. Carazo, K. Kochi, D. Lehmann, and R.D. Pascual-Marqui. 2006. “Nonsmooth nonnegative matrix factorization (nsNMF).” <em>IEEE Trans. Pattern Anal. Mach. Intell.</em> 28 (3): 403–15. doi:<a href="https://doi.org/10.1109/TPAMI.2006.60">10.1109/TPAMI.2006.60</a>.</p>
</div>
<div id="ref-Liu2017">
<p>Liu, Yuan, Yu Liang, Qifan Kuang, Fanfan Xie, Yingyi Hao, Zhining Wen, and Menglong Li. 2017. “Post-modified non-negative matrix factorization for deconvoluting the gene expression profiles of specific cell types from heterogeneous clinical samples based on RNA-sequencing data.” <em>J. Chemom.</em> Wiley-Blackwell, e2929. doi:<a href="https://doi.org/10.1002/cem.2929">10.1002/cem.2929</a>.</p>
</div>
<div id="ref-Herault1986">
<p>Herault, J., and C. Jutten. 1986. “Space or time adaptive signal processing by neural network models.” In <em>AIP Conf. Proc.</em>, 151:206–11. 1. AIP. doi:<a href="https://doi.org/10.1063/1.36258">10.1063/1.36258</a>.</p>
</div>
<div id="ref-Hyvarinen2000">
<p>Hyvärinen, Aapo, and Erkki Oja. 2000. “Independent Component Analysis: Algorithms and Applications.” <em>Neural Networks</em> 13 (45): 411–30. <a href="https://www.cs.helsinki.fi/u/ahyvarin/papers/NN00new.pdf" class="uri">https://www.cs.helsinki.fi/u/ahyvarin/papers/NN00new.pdf</a>.</p>
</div>
<div id="ref-Bell1995">
<p>Bell, Anthony J, and Terrence J Sejnowski. 1995. “An information-maximisation approach to blind separation and blind deconvolution.” <a href="http://www.inf.fu-berlin.de/lehre/WS05/Mustererkennung/infomax/infomax.pdf" class="uri">http://www.inf.fu-berlin.de/lehre/WS05/Mustererkennung/infomax/infomax.pdf</a>.</p>
</div>
<div id="ref-Cardoso1993">
<p>Cardoso, J.F., and A. Souloumiac. 1993. “Blind beamforming for non-gaussian signals.” <em>IEE Proc. F Radar Signal Process.</em> 140 (6): 362. doi:<a href="https://doi.org/10.1049/ip-f-2.1993.0054">10.1049/ip-f-2.1993.0054</a>.</p>
</div>
<div id="ref-Teschendorff2007">
<p>Teschendorff, Andrew E., Michel Journée, Pierre A. Absil, Rodolphe Sepulchre, and Carlos Caldas. 2007. “Elucidating the Altered Transcriptional Programs in Breast Cancer using Independent Component Analysis.” <em>PLoS Comput. Biol.</em> 3 (8). Public Library of Science: e161. doi:<a href="https://doi.org/10.1371/journal.pcbi.0030161">10.1371/journal.pcbi.0030161</a>.</p>
</div>
<div id="ref-Biton2014">
<p>Biton, Anne, Isabelle Bernard-Pierrot, Yinjun Lou, Clémentine Krucker, Elodie Chapeaublanc, Carlota Rubio-Pérez, Nuria López-Bigas, et al. 2014. “Independent Component Analysis Uncovers the Landscape of the Bladder Tumor Transcriptome and Reveals Insights into Luminal and Basal Subtypes.” <em>Cell Rep.</em> 9 (4): 1235–45. doi:<a href="https://doi.org/10.1016/j.celrep.2014.10.035">10.1016/j.celrep.2014.10.035</a>.</p>
</div>
<div id="ref-Engreitz2010">
<p>Engreitz, Jesse M, Bernie J Daigle, Jonathan J Marshall, Russ B Altman, and Russ B. Altman. 2010. “Independent component analysis: mining microarray data for fundamental human gene expression modules.” <em>J. Biomed. Inform.</em> 43 (6). NIH Public Access: 932–44. doi:<a href="https://doi.org/10.1016/j.jbi.2010.07.001">10.1016/j.jbi.2010.07.001</a>.</p>
</div>
<div id="ref-Gorban2007">
<p>Gorban, A. N. (Aleksandr Nikolaevich). 2007. <em>Principal manifolds for data visualization and dimension reduction</em>. Springer.</p>
</div>
<div id="ref-Cantini2018">
<p>Cantini, Laura, Ulykbek Kairov, Aurelien de Reynies, Emmanuel Barillot, Francois Radvanyi, and Andrei Zinovyev. 2018. “Stabilized Independent Component Analysis outperforms other methods in finding reproducible signals in tumoral transcriptomes.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 318154. doi:<a href="https://doi.org/10.1101/318154">10.1101/318154</a>.</p>
</div>
<div id="ref-Cheng2013">
<p>Cheng, Wei-Yi, Tai-Hsien Ou Yang, and Dimitris Anastassiou. 2013. “Biomolecular Events in Cancer Revealed by Attractor Metagenes.” Edited by Isidore Rigoutsos. <em>PLoS Comput. Biol.</em> 9 (2). Public Library of Science: e1002920. doi:<a href="https://doi.org/10.1371/journal.pcbi.1002920">10.1371/journal.pcbi.1002920</a>.</p>
</div>
<div id="ref-AlEjeh2014">
<p>Al-Ejeh, F, P T Simpson, J M Sanus, K Klein, M Kalimutho, W Shi, M Miranda, et al. 2014. “Meta-analysis of the global gene expression profile of triple-negative breast cancer identifies genes for the prognostication and treatment of aggressive breast cancer.” <em>Oncogenesis</em> 3 (4). Nature Publishing Group: e100–e100. doi:<a href="https://doi.org/10.1038/oncsis.2014.14">10.1038/oncsis.2014.14</a>.</p>
</div>
<div id="ref-Elmas2016">
<p>Elmas, Abdulkadir, Tai-Hsien Ou Yang, Xiaodong Wang, and Dimitris Anastassiou. 2016. “Discovering Genome-Wide Tag SNPs Based on the Mutual Information of the Variants.” Edited by Srinivas Mummidi. <em>PLoS One</em> 11 (12). Public Library of Science: e0167994. doi:<a href="https://doi.org/10.1371/journal.pone.0167994">10.1371/journal.pone.0167994</a>.</p>
</div>
<div id="ref-Vallania2017">
<p>Vallania, Francesco, Andrew Tam, Shane Lofgren, Steven Schaffert, Tej D. Azad, Erika Bongen, Meia Alsup, et al. 2017. “Leveraging heterogeneity across multiple data sets increases accuracy of cell-mixture deconvolution and reduces biological and technical biases.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 206466. doi:<a href="https://doi.org/10.1101/206466">10.1101/206466</a>.</p>
</div>
<div id="ref-Hoffmann2006">
<p>Hoffmann, Martin, Dirk Pohlers, Dirk Koczan, Hans-Jürgen Thiesen, Stefan Wölfl, and Raimund W Kinne. 2006. “Robust computational reconstitution – a new method for the comparative analysis of gene expression in tissues and isolated cell fractions.” <em>BMC Bioinformatics</em> 7 (1). BioMed Central: 369. doi:<a href="https://doi.org/10.1186/1471-2105-7-369">10.1186/1471-2105-7-369</a>.</p>
</div>
<div id="ref-Qiao2012">
<p>Qiao, Wenlian, Gerald Quon, Elizabeth Csaszar, Mei Yu, Quaid Morris, and Peter W. Zandstra. 2012. “PERT: A Method for Expression Deconvolution of Human Blood Samples from Varied Microenvironmental and Developmental Conditions.” Edited by Richard Bonneau. <em>PLoS Comput. Biol.</em> 8 (12). Public Library of Science: e1002838. doi:<a href="https://doi.org/10.1371/journal.pcbi.1002838">10.1371/journal.pcbi.1002838</a>.</p>
</div>
<div id="ref-Chung2017">
<p>Chung, Woosung, Hye Hyeon Eum, Hae-Ock Lee, Kyung-Min Lee, Han-Byoel Lee, Kyu-Tae Kim, Han Suk Ryu, et al. 2017. “Single-cell RNA-seq enables comprehensive tumour and immune cell profiling in primary breast cancer.” <em>Nat. Commun.</em> 8. Nature Publishing Group: 15081. doi:<a href="https://doi.org/10.1038/ncomms15081">10.1038/ncomms15081</a>.</p>
</div>
<div id="ref-Lavin2017">
<p>Lavin, Yonit, Soma Kobayashi, Andrew Leader, El-ad David Amir, Naama Elefant, Camille Bigenwald, Romain Remark, et al. 2017. “Innate Immune Landscape in Early Lung Adenocarcinoma by Paired Single-Cell Analyses.” <em>Cell</em> 169 (4). Cell Press: 750–765.e17. doi:<a href="https://doi.org/10.1016/J.CELL.2017.04.014">10.1016/J.CELL.2017.04.014</a>.</p>
</div>
<div id="ref-Puram2017">
<p>Puram, Sidharth V., Itay Tirosh, Anuraag S. Parikh, Anoop P. Patel, Keren Yizhak, Shawn Gillespie, Christopher Rodman, et al. 2017. “Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer.” <em>Cell</em> 171 (7). Cell Press: 1611–1624.e24. doi:<a href="https://doi.org/10.1016/J.CELL.2017.10.044">10.1016/J.CELL.2017.10.044</a>.</p>
</div>
<div id="ref-Tirosh2016">
<p>Tirosh, Itay, Benjamin Izar, Sanjay M. Prakadan, Marc H. Wadsworth, Daniel Treacy, John J. Trombetta, Asaf Rotem, et al. 2016. “Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq.” <em>Science (80-. ).</em> 352 (6282): 189–96. doi:<a href="https://doi.org/10.1126/science.aad0501">10.1126/science.aad0501</a>.</p>
</div>
<div id="ref-Zheng2017">
<p>Zheng, Chunhong, Liangtao Zheng, Jae-Kwang Yoo, Huahu Guo, Yuanyuan Zhang, Xinyi Guo, Boxi Kang, et al. 2017. “Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing.” <em>Cell</em> 169 (7). Cell Press: 1342–1356.e16. doi:<a href="https://doi.org/10.1016/J.CELL.2017.05.035">10.1016/J.CELL.2017.05.035</a>.</p>
</div>
<div id="ref-Zhong2013">
<p>Zhong, Yi, Ying-Wooi Wan, Kaifang Pang, Lionel ML Chow, and Zhandong Liu. 2013. “Digital sorting of complex tissues for cell type-specific gene expression profiles.” <em>BMC Bioinformatics</em> 14 (1). BioMed Central: 89. doi:<a href="https://doi.org/10.1186/1471-2105-14-89">10.1186/1471-2105-14-89</a>.</p>
</div>
<div id="ref-Shannon2017">
<p>Shannon, Casey P., Robert Balshaw, Virginia Chen, Zsuzsanna Hollander, Mustafa Toma, Bruce M. McManus, J. Mark FitzGerald, Don D. Sin, Raymond T. Ng, and Scott J. Tebbutt. 2017. “Enumerateblood – an R package to estimate the cellular composition of whole blood from Affymetrix Gene ST gene expression profiles.” <em>BMC Genomics</em> 18 (1). BioMed Central: 43. doi:<a href="https://doi.org/10.1186/s12864-016-3460-1">10.1186/s12864-016-3460-1</a>.</p>
</div>
<div id="ref-Clarke2010">
<p>Clarke, J., P. Seo, and B. Clarke. 2010. “Statistical expression deconvolution from mixed tissue samples.” <em>Bioinformatics</em> 26 (8). Oxford University Press: 1043–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btq097">10.1093/bioinformatics/btq097</a>.</p>
</div>
<div id="ref-Kuhn2011">
<p>Kuhn, Alexandre, Doris Thu, Henry J Waldvogel, Richard L M Faull, and Ruth Luthi-Carter. 2011. “Population-specific expression analysis (PSEA) reveals molecular changes in diseased brain.” <em>Nat. Methods</em> 8 (11). Nature Publishing Group: 945–47. doi:<a href="https://doi.org/10.1038/nmeth.1710">10.1038/nmeth.1710</a>.</p>
</div>
<div id="ref-Li2016">
<p>Li, Bo, Eric Severson, Jean-Christophe Pignon, Haoquan Zhao, Taiwen Li, Jesse Novak, Peng Jiang, et al. 2016. “Comprehensive analyses of tumor immunity: implications for cancer immunotherapy.” <em>Genome Biol.</em> 17 (1). BioMed Central: 174. doi:<a href="https://doi.org/10.1186/s13059-016-1028-7">10.1186/s13059-016-1028-7</a>.</p>
</div>
<div id="ref-Cieslik2017">
<p>Cieślik, Marcin, and Arul M. Chinnaiyan. 2017. “Cancer transcriptome profiling at the juncture of clinical translation.” <em>Nat. Rev. Genet.</em> 19 (2). Nature Publishing Group: 93–109. doi:<a href="https://doi.org/10.1038/nrg.2017.96">10.1038/nrg.2017.96</a>.</p>
</div>
<div id="ref-Regev2017">
<p>Regev, Aviv, Sarah A Teichmann, Eric S Lander, Ido Amit, Christophe Benoist, Ewan Birney, Bernd Bodenmiller, et al. 2017. “The Human Cell Atlas.” <em>Elife</em> 6. eLife Sciences Publications Limited: e27041. doi:<a href="https://doi.org/10.7554/eLife.27041">10.7554/eLife.27041</a>.</p>
</div>
<div id="ref-Teschendorff2017">
<p>Teschendorff, Andrew E, and Shijie C Zheng. 2017. “Cell-type deconvolution in epigenome-wide association studies: a review and recommendations.” <em>Epigenomics</em> 9 (5): 757–68. doi:<a href="https://doi.org/10.2217/epi-2016-0153">10.2217/epi-2016-0153</a>.</p>
</div>
<div id="ref-Titus2017">
<p>Titus, Alexander J., Rachel M. Gallimore, Lucas A. Salas, and Brock C. Christensen. 2017. “Cell-type deconvolution from DNA methylation: a review of recent applications.” <em>Hum. Mol. Genet.</em> 26 (R2): R216–R224. doi:<a href="https://doi.org/10.1093/hmg/ddx275">10.1093/hmg/ddx275</a>.</p>
</div>
<div id="ref-Houseman2014">
<p>Houseman, Eugene Andres, John Molitor, and Carmen J. Marsit. 2014. “Reference-free cell mixture adjustments in analysis of DNA methylation data.” <em>Bioinformatics</em> 30 (10). Oxford University Press: 1431–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btu029">10.1093/bioinformatics/btu029</a>.</p>
</div>
<div id="ref-Leek2007">
<p>Leek, Jeffrey T., and John D. Storey. 2007. “Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis.” <em>PLoS Genet.</em> 3 (9): e161. doi:<a href="https://doi.org/10.1371/journal.pgen.0030161">10.1371/journal.pgen.0030161</a>.</p>
</div>
<div id="ref-Teschendorff2011">
<p>Teschendorff, Andrew E., Joanna Zhuang, and Martin Widschwendter. 2011. “Independent surrogate variable analysis to deconvolve confounding factors in large-scale microarray profiling studies.” <em>Bioinformatics</em> 27 (11). Oxford University Press: 1496–1505. doi:<a href="https://doi.org/10.1093/bioinformatics/btr171">10.1093/bioinformatics/btr171</a>.</p>
</div>
<div id="ref-Maksimovic2015">
<p>Maksimovic, Jovana, Johann A. Gagnon-Bartsch, Terence P. Speed, and Alicia Oshlack. 2015. “Removing unwanted variation in a differential methylation analysis of Illumina HumanMethylation450 array data.” <em>Nucleic Acids Res.</em> 43 (16). Oxford University Press: e106–e106. doi:<a href="https://doi.org/10.1093/nar/gkv526">10.1093/nar/gkv526</a>.</p>
</div>
<div id="ref-Zou2014">
<p>Zou, James, Christoph Lippert, David Heckerman, Martin Aryee, and Jennifer Listgarten. 2014. “Epigenome-wide association studies without the need for cell-type composition.” <em>Nat. Methods</em> 11 (3). Nature Publishing Group: 309–11. doi:<a href="https://doi.org/10.1038/nmeth.2815">10.1038/nmeth.2815</a>.</p>
</div>
<div id="ref-Rahmani2016">
<p>Rahmani, Elior, Noah Zaitlen, Yael Baran, Celeste Eng, Donglei Hu, Joshua Galanter, Sam Oh, et al. 2016. “Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies.” <em>Nat. Methods</em> 13 (5). Nature Publishing Group: 443–45. doi:<a href="https://doi.org/10.1038/nmeth.3809">10.1038/nmeth.3809</a>.</p>
</div>
<div id="ref-Houseman2016">
<p>Houseman, E. Andres, Molly L. Kile, David C. Christiani, Tan A. Ince, Karl T. Kelsey, and Carmen J. Marsit. 2016. “Reference-free deconvolution of DNA methylation data and mediation by cell composition effects.” <em>BMC Bioinformatics</em> 17 (1). BioMed Central: 259. doi:<a href="https://doi.org/10.1186/s12859-016-1140-4">10.1186/s12859-016-1140-4</a>.</p>
</div>
<div id="ref-Lutsik2017">
<p>Lutsik, Pavlo, Martin Slawski, Gilles Gasparoni, Nikita Vedeneev, Matthias Hein, and Jörn Walter. 2017. “MeDeCom: discovery and quantification of latent components of heterogeneous methylomes.” <em>Genome Biol.</em> 18 (1). BioMed Central: 55. doi:<a href="https://doi.org/10.1186/s13059-017-1182-6">10.1186/s13059-017-1182-6</a>.</p>
</div>
<div id="ref-Houseman2012">
<p>Houseman, Eugene, William P Accomando, Devin C Koestler, Brock C Christensen, Carmen J Marsit, Heather H Nelson, John K Wiencke, and Karl T Kelsey. 2012. “DNA methylation arrays as surrogate measures of cell mixture distribution.” <em>BMC Bioinformatics</em> 13 (1). BioMed Central: 86. doi:<a href="https://doi.org/10.1186/1471-2105-13-86">10.1186/1471-2105-13-86</a>.</p>
</div>
<div id="ref-Breeze2016">
<p>Breeze, Charles E., Dirk S. Paul, Jenny van Dongen, Lee M. Butcher, John C. Ambrose, James E. Barrett, Robert Lowe, et al. 2016. “eFORGE: A Tool for Identifying Cell Type-Specific Signal in Epigenomic Data.” <em>Cell Rep.</em> 17 (8): 2137–50. doi:<a href="https://doi.org/10.1016/j.celrep.2016.10.059">10.1016/j.celrep.2016.10.059</a>.</p>
</div>
<div id="ref-Wen2016">
<p>Wen, Yanhua, Yanjun Wei, Shumei Zhang, Song Li, Hongbo Liu, Fang Wang, Yue Zhao, Dongwei Zhang, and Yan Zhang. 2016. “Cell subpopulation deconvolution reveals breast cancer heterogeneity based on DNA methylation signature.” <em>Brief. Bioinform.</em> 18 (3): bbw028. doi:<a href="https://doi.org/10.1093/bib/bbw028">10.1093/bib/bbw028</a>.</p>
</div>
<div id="ref-Aran2015">
<p>Aran, Dvir, Marina Sirota, and Atul J Butte. 2015. “Systematic pan-cancer analysis of tumour purity.” <em>Nat. Commun.</em> 6. Nature Publishing Group: 8971. doi:<a href="https://doi.org/10.1038/ncomms9971">10.1038/ncomms9971</a>.</p>
</div>
<div id="ref-Fu2015">
<p>Fu, Yi, Guoqiang Yu, Douglas A. Levine, Niya Wang, Ie-Ming Shih, Zhen Zhang, Robert Clarke, and Yue Wang. 2015. “BACOM2.0 facilitates absolute normalization and quantification of somatic copy number alterations in heterogeneous tumor.” <em>Sci. Rep.</em> 5 (1). Nature Publishing Group: 13955. doi:<a href="https://doi.org/10.1038/srep13955">10.1038/srep13955</a>.</p>
</div>
<div id="ref-Carter2012">
<p>Carter, Scott L, Kristian Cibulskis, Elena Helman, Aaron McKenna, Hui Shen, Travis Zack, Peter W Laird, et al. 2012. “Absolute quantification of somatic DNA alterations in human cancer.” <em>Nat. Biotechnol.</em> 30 (5). Nature Publishing Group: 413–21. doi:<a href="https://doi.org/10.1038/nbt.2203">10.1038/nbt.2203</a>.</p>
</div>
<div id="ref-Yu2016">
<p>Yu, Zhenhua, Ao Li, and Minghui Wang. 2016. “CloneCNA: detecting subclonal somatic copy number alterations in heterogeneous tumor samples from whole-exome sequencing data.” <em>BMC Bioinformatics</em> 17 (1): 310. doi:<a href="https://doi.org/10.1186/s12859-016-1174-7">10.1186/s12859-016-1174-7</a>.</p>
</div>
<div id="ref-Larson2013">
<p>Larson, Nicholas B, and Brooke L Fridley. 2013. “PurBayes: estimating tumor cellularity and subclonality in next-generation sequencing data.” <em>Bioinformatics</em> 29 (15). Oxford University Press: 1888–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btt293">10.1093/bioinformatics/btt293</a>.</p>
</div>
<div id="ref-Li2014">
<p>Li, Bo, and Jun Z Li. 2014. “A general framework for analyzing tumor subclonality using SNP array and DNA sequencing data.” <em>Genome Biol.</em> 15 (9). BioMed Central: 473. doi:<a href="https://doi.org/10.1186/s13059-014-0473-4">10.1186/s13059-014-0473-4</a>.</p>
</div>
<div id="ref-Oesper2013">
<p>Oesper, Layla, Ahmad Mahmoody, and Benjamin J Raphael. 2013. “THetA: inferring intra-tumor heterogeneity from high-throughput DNA sequencing data.” <em>Genome Biol.</em> 14 (7). BioMed Central: R80. doi:<a href="https://doi.org/10.1186/gb-2013-14-7-r80">10.1186/gb-2013-14-7-r80</a>.</p>
</div>
<div id="ref-Miller2014">
<p>Miller, Christopher A., Brian S. White, Nathan D. Dees, Malachi Griffith, John S. Welch, Obi L. Griffith, Ravi Vij, et al. 2014. “SciClone: Inferring Clonal Architecture and Tracking the Spatial and Temporal Patterns of Tumor Evolution.” Edited by Niko Beerenwinkel. <em>PLoS Comput. Biol.</em> 10 (8). Public Library of Science: e1003665. doi:<a href="https://doi.org/10.1371/journal.pcbi.1003665">10.1371/journal.pcbi.1003665</a>.</p>
</div>
<div id="ref-Jiang2016">
<p>Jiang, Yuchao, Yu Qiu, Andy J Minn, and Nancy R Zhang. 2016. “Assessing intratumor heterogeneity and tracking longitudinal and spatial clonal evolutionary history by next-generation sequencing.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 113 (37). National Academy of Sciences: E5528–37. doi:<a href="https://doi.org/10.1073/pnas.1522203113">10.1073/pnas.1522203113</a>.</p>
</div>
<div id="ref-Roth2014">
<p>Roth, Andrew, Jaswinder Khattra, Damian Yap, Adrian Wan, Emma Laks, Justina Biele, Gavin Ha, Samuel Aparicio, Alexandre Bouchard-Côté, and Sohrab P Shah. 2014. “PyClone: statistical inference of clonal population structure in cancer.” <em>Nat. Methods</em> 11 (4). Nature Publishing Group: 396–98. doi:<a href="https://doi.org/10.1038/nmeth.2883">10.1038/nmeth.2883</a>.</p>
</div>
<div id="ref-Andor2014">
<p>Andor, Noemi, Julie V. Harness, Sabine Müller, Hans W. Mewes, and Claudia Petritsch. 2014. “EXPANDS: expanding ploidy and allele frequency on nested subpopulations.” <em>Bioinformatics</em> 30 (1). Oxford University Press: 50–60. doi:<a href="https://doi.org/10.1093/bioinformatics/btt622">10.1093/bioinformatics/btt622</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="objectives.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/03-lntroductionMath.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
