<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="intro.html">
<link rel="next" href="sens.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; min-height: 1.25em; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; }
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
@media screen {
a.sourceLine::before { text-decoration: underline; color: initial; }
}
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.bn { color: #40a070; } /* BaseN */
code span.fl { color: #40a070; } /* Float */
code span.ch { color: #4070a0; } /* Char */
code span.st { color: #4070a0; } /* String */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.ot { color: #007020; } /* Other */
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.fu { color: #06287e; } /* Function */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code span.cn { color: #880000; } /* Constant */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.ss { color: #bb6688; } /* SpecialString */
code span.im { } /* Import */
code span.va { color: #19177c; } /* Variable */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.op { color: #666666; } /* Operator */
code span.bu { } /* BuiltIn */
code span.ex { } /* Extension */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.at { color: #7d9029; } /* Attribute */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-phd---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary PhD - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#general-context-of-the-thesis"><i class="fa fa-check"></i>General context of the thesis</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#organisation-of-the-dissertation"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#biological-dimension-of-the-thesis"><i class="fa fa-check"></i><b>1.4</b> Biological dimension of the thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#cell-type-deconvolution-models-of-bulk-transcriptome"><i class="fa fa-check"></i><b>2.2</b> Cell-type deconvolution models of bulk transcriptome</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.2.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.2.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.2.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.2.3" data-path="methods.html"><a href="methods.html#regression-algorithm"><i class="fa fa-check"></i><b>2.2.3</b> regression algorithm</a></li>
<li class="chapter" data-level="2.2.4" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.2.4</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.2.5" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.2.5</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.2.6" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.2.6</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.2.7" data-path="methods.html"><a href="methods.html#matrix-factorisation-methods"><i class="fa fa-check"></i><b>2.2.7</b> Matrix factorisation methods</a></li>
<li class="chapter" data-level="2.2.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.2.8</b> Others aspects</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#deconvolution-of-other-data-types"><i class="fa fa-check"></i><b>2.3</b> Deconvolution of other data types</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#methodological-dimension-of-the-thesis"><i class="fa fa-check"></i><b>2.4</b> Methodological dimension of the thesis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="sens.html"><a href="sens.html"><i class="fa fa-check"></i><b>3</b> Study of sensitivity and reproducibility of known methods</a><ul>
<li class="chapter" data-level="3.1" data-path="sens.html"><a href="sens.html#finding-optimal-number-of-components-and-over-decomposition-of-transcriptopmes"><i class="fa fa-check"></i><b>3.1</b> Finding optimal number of components and over-decomposition of transcriptopmes</a></li>
<li class="chapter" data-level="3.2" data-path="sens.html"><a href="sens.html#section"><i class="fa fa-check"></i><b>3.2</b> </a></li>
<li class="chapter" data-level="3.3" data-path="sens.html"><a href="sens.html#reproducibility-of-nmf-versus-ica-vs-cam"><i class="fa fa-check"></i><b>3.3</b> REPRODUCIBILITY OF NMF VERSUS ICA (VS CAM?)</a><ul>
<li class="chapter" data-level="3.3.1" data-path="sens.html"><a href="sens.html#comparing-metagenes-obtained-with-nmf-vs-ica."><i class="fa fa-check"></i><b>3.3.1</b> Comparing metagenes obtained with NMF vs ICA.</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="sens.html"><a href="sens.html#impact-of-modification-of-signatures-list-on-result-for-signature-based-deconvolution-methods"><i class="fa fa-check"></i><b>3.4</b> Impact of modification of signatures list on result for signature-based deconvolution methods</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>4</b> Deconvolution of transcriptomes and methylomes</a><ul>
<li class="chapter" data-level="4.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>4.1</b> From blind deconvolution to cell-type quantification: general overview</a><ul>
<li class="chapter" data-level="4.1.1" data-path="deconica.html"><a href="deconica.html#the-ica-based-deconvolution-of-transcriptomes"><i class="fa fa-check"></i><b>4.1.1</b> The ICA-based deconvolution of Transcriptomes</a></li>
<li class="chapter" data-level="4.1.2" data-path="deconica.html"><a href="deconica.html#interpretation-of-independent-components"><i class="fa fa-check"></i><b>4.1.2</b> Interpretation of Independent components</a></li>
<li class="chapter" data-level="4.1.3" data-path="deconica.html"><a href="deconica.html#transforming-metagenes-into-signature-matrix"><i class="fa fa-check"></i><b>4.1.3</b> Transforming metagenes into signature matrix</a></li>
<li class="chapter" data-level="4.1.4" data-path="deconica.html"><a href="deconica.html#regression-based-estimation-of-cell-type-proportions-solving-system-of-equations"><i class="fa fa-check"></i><b>4.1.4</b> Regression-based estimation of cell-type proportions : solving system of equations</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="deconica.html"><a href="deconica.html#deconica-r-package-for-ica-based-deconvolution"><i class="fa fa-check"></i><b>4.2</b> <em>DeconICA</em> R package for ICA-based deconvolution</a><ul>
<li class="chapter" data-level="4.2.1" data-path="deconica.html"><a href="deconica.html#demo"><i class="fa fa-check"></i><b>4.2.1</b> <em>Demo</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>5</b> Comparative analysis of cancer immune infiltration</a></li>
<li class="chapter" data-level="6" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>6</b> Heterogeneity of immune cell types</a></li>
<li class="chapter" data-level="7" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>7</b> Conclusions and perspectives</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#phd-timeline-for-defence-before-the-end-of-october-2018"><i class="fa fa-check"></i>PhD timeline for defence before the end of October 2018</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#thesis-writing"><i class="fa fa-check"></i>Thesis writing</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#activity-report-2017"><i class="fa fa-check"></i>Activity Report 2017</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="methods" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Mathematical foundation of cell-type deconvolution of biological data</h1>
<p>In this chapter, we will discuss how mathematical models can be used to extract information about specific cell-types from ‘bulk’ transcriptome data or how to de-mix mixed sources. It will introduce you to basic concepts of approaches adapted for cell-type deconvolution, show an overview of the evolution of the field as well as it will explicit most tools for estimating presence and proportion of immune cells within cancer biopsies.</p>
<p>Deconvolution - definition</p>
<p>Explain the principle</p>
<div class="figure" style="text-align: center"><span id="fig:deconvolution-cartoon"></span>
<img src="figures-ext/deconv.png" alt="Principle of the deconvolution applied to transcriptome Graphical illustration of the deconvolution of mixed samples. Starting from the left, gene expression of genes A B C is a sum of expression of cell types 1, 2, 3, 4. After deconvolution, cell types are separated and gene expression of each cell type is estimated taking into account cell type proportions." width="100%" />
<p class="caption">
Figure 2.1: <strong>Principle of the deconvolution applied to transcriptome</strong> Graphical illustration of the deconvolution of mixed samples. Starting from the left, gene expression of genes A B C is a sum of expression of cell types 1, 2, 3, 4. After deconvolution, cell types are separated and gene expression of each cell type is estimated taking into account cell type proportions.
</p>
</div>


<div id="introduction-to-supervised-and-unsupervised-learning" class="section level2">
<h2><span class="header-section-number">2.1</span> Introduction to supervised and unsupervised learning</h2>
<ul>
<li>ML</li>
<li>Explaining difference between supervised and unsupervised learning technically and in our context</li>
</ul>
<p>Unsupervised learning The computer is handed over some data to learn from and is the source for selecting an action, although no feedback is given so the computer has no clue of how well it did.</p>
<p>Supervised learning The computer is handed over some data to learn from and is also told what is the right decision in each case. This requires a lot of observing.</p>
<p>Reinforcement learning The computer is handed over some data to learn from and receives feedback on how well it did based on the result, but is not told which specific action was good or bad.</p>
<p>Clustering</p>
</div>
<div id="cell-type-deconvolution-models-of-bulk-transcriptome" class="section level2">
<h2><span class="header-section-number">2.2</span> Cell-type deconvolution models of bulk transcriptome</h2>
<div id="literature-overview" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Literature overview</h3>
<table>
<caption>
<span id="tab:mytab">Table 2.1: </span><b>Summary of methods for cell-type deconvolution of bulk transcriptome</b>
</caption>
</table>
<div id="htmlwidget-de10812a57f35338d397" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-de10812a57f35338d397">{"x":{"filter":"top","filterHTML":"<tr>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none; position: absolute; width: 200px;\">\n      <div data-min=\"2001\" data-max=\"2018\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none; position: absolute; width: 200px;\">\n      <div data-min=\"0\" data-max=\"343\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"number\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n    <div style=\"display: none; position: absolute; width: 200px;\">\n      <div data-min=\"0\" data-max=\"85.75\" data-scale=\"15\"><\/div>\n      <span style=\"float: left;\"><\/span>\n      <span style=\"float: right;\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n<\/tr>","data":[["CIBERSORT","ESTIMATE","csSAM","Virtual Microdissection","Abbas regression","MCPcounter","PSEA","Quadratic programming","Semi-supervised Nonnegative Matrix Factorization","DeconRNASeq","DSA","DECONVOLUTE","ISOpure","xCell","CellCODE","DCQ","DeMix","Statistical expression deconvolution","DSection","Nanodissection","TIMER","Direct method","SPEC","PERT","deconf","CTen","Mixture models","CAM","ISOLATE","UNDO","In silico microdissection","BioQC","Self-directed Method for Cell-Type Identification","Computational expression deconvolution","Electronical substraction","EPIC","MMAD","Immune  Quant","VoCAL","CellMapper","Estimation of immune cell content","contamDE","GSVA scores","Enumerateblood","CoD","ImmunoStates","quanTIseq","SMC"," modular discrimination index","Robust Computational Reconstitution","Statical mechanics approach","MHMM","MySort","ADVOCATE","DTD","CellDistinguisher ","dtangle","DeconICA","DemixT","Post‐modified non‐negative matrix factorization","Infino","ImSig","TEMT"],["22","2","user-defined","14","17","10","user-defined","6","user-defined","undefiend","user-defined","user-defined","2","64","user-defined","undefiend","2","2","user-defined","undefined","6","SM","user-defined","11","100","undefined","2","10","undefined","2","3","150","undefined","undefined","2","8","unknown","user-defined","user-defined","user-defined","8","2+","6","6","207","20","11","user-defined","5","SM","2","user-defined","22","2","user-defined","user-defined","user-defined","100","3","user-defined","user-defined","11","2"],["MA","MA + RNA-seq","MA","MA","MA","MA","MA","MA","MA","RNA-seq","MA","MA","MA","MA + RNA-seq","MA","RNA-seq","MA","MA","MA","MA","MA + RNA-seq","MA","MA","MA","MA","MA","MA","MA","MA","MA","MA","MA + RNA-seq","MA","MA","MA","RNA-seq","MA","undefined","MA, GWAS","MA","scRNA-seq","RNA-seq","RNA-seq","MA","RNA-seq","MA","RNA-seq + Images","MA","MA + RNA-seq","MA","undefined","MA","MA","RNA-seq","scRNA-seq","MA + RNA-seq","MA + RNA-seq","MA + RNA-seq","MA + RNA-seq","RNA-seq","RNA-seq","RNA-seq","RNAseq"],["supervised","supervised","supervised","supervised","supervised","supervised","supervised","supervised","semi-supervised","supervised","supervised","supervised","supervised","supervised","semi-supervised","supervised","supervised","supervised","supervised","supervised","supervised","unsupervised","supervised","supervised","unsupervised","supervised","supervised","unsupervised","supervised","unsupervised","unsupervised","supervised","unsupervised","supervised","supervised","supervised","BOTH","supervised","supervised","semi-supervised","supervised","supervised","supervised","supervised","supervised","supervised","supervised","unsupervised","supervised","supervised","unsupervised","unsupervised","supervised","supervised","supervised","unsupervised","supervised","unsupervised","supervised","unsupervised","supervised","supervised","supervised"],["Supported vector regression","ssGSEA","least-squares regression","multiplicative update NMF","linear least squares regression","means of marker genes","linear regression","linear latent variable model solved with quadratic programming","NMF minimizing the Kullback-Leibler divergence on pre-selected genes and with pure proportions","linear latent variable model solved with quadratic programming","linear model solved with quadratic programming.","system of linear equations","maximum a posteriori(MAP) estimation of multinomial distribution","ssGSEA+ spillover compensation","rebust latent variable"," regularized regression model ","Maximul likelihood estimate","linear equations","Bayesian MCMC-based model","iterative SVM","constrained linear regression","non negative least squares and decorellation","enrichment score","Non-negative maximum likelihood model  with adjustement for perturbations","Least squares non-negative matrix factorization algorithm","GSEA","methods of moments procedures and the expectation–maximization algorithm","convex analysis of mixtures ","Latent Dirichlet Allocation (LDA)","matrix inversion","Gaussian Mixture Model","Wilcoxon-Mann-Whitney test","non-negative least squares, Kullback-Leibler divergence","linear equations with simulated annealing","system of linear equations","weighted constrained least square optimization","constrained regression with corrected AIC parameter fit","DCQ algorithm adapted to human","linear regression","SVD","CIBERSORT using scRNA-seq basis matrix","empirical Bayes estimate of the negative binomial dispersion","GSVA"," multi-response Gaussian model trained","DCQ algorithm + random forest classifier","regression (previously published)","constrained least squares regression","Bayesian inference with sequential monte carlo samplers","correlation-basd score","trimmed least modulus (L1) regression.","Bayesian model with MCMC sampling assuming Gaussian distribution","Hidden state markov model"," v -Support Vector Regression","trained Gaussian-mixture mode","optimising loss-function of  penalized least-squares regression","convex hull + NMF","linear regression","ICA + post identification: enrichment","Maximul likelihood estimate","NMF using alternating least square","Bayesian inference with a generative model","correlation-based score","generative mixture model "],["<a href=\"https://doi.org/10.1038/nmeth.3337\">https://doi.org/10.1038/nmeth.3337<\/a>","<a href=\"https://doi.org/10.1038/ncomms3612\">https://doi.org/10.1038/ncomms3612<\/a>","<a href=\"10.1038/nmeth.1439\">10.1038/nmeth.1439<\/a>","<a href=\"https://doi.org/10.1038/ng.3398\">https://doi.org/10.1038/ng.3398<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0006098\">https://doi.org/10.1371/journal.pone.0006098<\/a>","<a href=\"https://doi.org/10.1186/s13059-016-1070-5\">https://doi.org/10.1186/s13059-016-1070-5<\/a>","<a href=\"https://doi.org/10.1038/nmeth.1710\">https://doi.org/10.1038/nmeth.1710<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0027156\">https://doi.org/10.1371/journal.pone.0027156<\/a>","<a href=\"https://doi.org/10.1016/j.meegid.2011.08.014\">https://doi.org/10.1016/j.meegid.2011.08.014<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btt090\">https://doi.org/10.1093/bioinformatics/btt090<\/a>","<a href=\"https://dx.doi.org/10.1186%2F1471-2105-14-89\">https://dx.doi.org/10.1186%2F1471-2105-14-89<\/a>","<a href=\"https://doi.org/10.1073/pnas.1832361100\">https://doi.org/10.1073/pnas.1832361100<\/a>","<a href=\"https://doi.org/10.1186/gm433\">https://doi.org/10.1186/gm433<\/a>","<a href=\"https://dx.doi.org/10.1186%2Fs13059-017-1349-1\">https://dx.doi.org/10.1186%2Fs13059-017-1349-1<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btv015\">https://doi.org/10.1093/bioinformatics/btv015<\/a>","<a href=\"https://dx.doi.org/10.1002%2Fmsb.134947\">https://dx.doi.org/10.1002%2Fmsb.134947<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btt301\">https://doi.org/10.1093/bioinformatics/btt301<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btq097\">https://doi.org/10.1093/bioinformatics/btq097<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btq406\">https://doi.org/10.1093/bioinformatics/btq406<\/a>","<a href=\"https://doi.org/10.1101/gr.155697.113\">https://doi.org/10.1101/gr.155697.113<\/a>","<a href=\"https://doi.org/10.1186/s13059-016-1028-7\">https://doi.org/10.1186/s13059-016-1028-7<\/a>","<a href=\"https://www.ncbi.nlm.nih.gov/pubmed/11473019\">https://www.ncbi.nlm.nih.gov/pubmed/11473019<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-12-258\">https://doi.org/10.1186/1471-2105-12-258<\/a>","<a href=\"https://doi.org/10.1371/journal.pcbi.1002838\">https://doi.org/10.1371/journal.pcbi.1002838<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-11-27\">https://doi.org/10.1186/1471-2105-11-27<\/a>","<a href=\"https://doi.org/10.1186/1471-2164-13-460\">https://doi.org/10.1186/1471-2164-13-460<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/bth139\">https://doi.org/10.1093/bioinformatics/bth139<\/a>","<a href=\"https://doi.org/10.1038/srep18909\">https://doi.org/10.1038/srep18909<\/a>","<a href=\"https://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp378\">https://dx.doi.org/10.1093%2Fbioinformatics%2Fbtp378<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btu607\">https://doi.org/10.1093/bioinformatics/btu607<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-6-54\">https://doi.org/10.1186/1471-2105-6-54<\/a>","<a href=\"https://doi.org/10.1186/s12864-017-3661-2\">https://doi.org/10.1186/s12864-017-3661-2<\/a>","<a href=\"https://doi.org/10.1371/journal.pcbi.1003189\">https://doi.org/10.1371/journal.pcbi.1003189<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-7-328\">https://doi.org/10.1186/1471-2105-7-328<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btm508\">https://doi.org/10.1093/bioinformatics/btm508<\/a>","<a href=\"https://dx.doi.org/10.7554%2FeLife.26476\">https://dx.doi.org/10.7554%2FeLife.26476<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btt566\">https://doi.org/10.1093/bioinformatics/btt566<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btw535\">https://doi.org/10.1093/bioinformatics/btw535<\/a>","<a href=\"https://doi.org/10.1371/journal.pcbi.1004856\">https://doi.org/10.1371/journal.pcbi.1004856<\/a>","<a href=\"https://doi.org/10.1186/s13059-016-1062-5\">https://doi.org/10.1186/s13059-016-1062-5<\/a>","<a href=\"https://doi.org/10.1038/s41467-017-02289-3\">https://doi.org/10.1038/s41467-017-02289-3<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btv657\">https://doi.org/10.1093/bioinformatics/btv657<\/a>","<a href=\"https://doi.org/10.1158/1078-0432.CCR-17-3509\">https://doi.org/10.1158/1078-0432.CCR-17-3509<\/a>","<a href=\"https://doi.org/10.1186/s12864-016-3460-1\">https://doi.org/10.1186/s12864-016-3460-1<\/a>","<a href=\"https://doi.org/10.1093/bioinformatics/btv498\">https://doi.org/10.1093/bioinformatics/btv498<\/a>","<a href=\"https://doi.org/10.1101/206466\">https://doi.org/10.1101/206466<\/a>","<a href=\"https://doi.org/10.1101/223180\">https://doi.org/10.1101/223180<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0186167\">https://doi.org/10.1371/journal.pone.0186167<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0169271\">https://doi.org/10.1371/journal.pone.0169271<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-7-369\">https://doi.org/10.1186/1471-2105-7-369<\/a>","<a href=\"https://arxiv.org/abs/1210.7508v1\">https://arxiv.org/abs/1210.7508v1<\/a>","<a href=\"https://doi.org/10.1089/cmb.2006.13.1749\">https://doi.org/10.1089/cmb.2006.13.1749<\/a>","<a href=\"https://doi.org/10.1186/s12859-018-2069-6\">https://doi.org/10.1186/s12859-018-2069-6<\/a>","<a href=\"https://doi.org/10.1101/288779\">https://doi.org/10.1101/288779<\/a>","<a href=\"https://arxiv.org/abs/1801.08447v1\">https://arxiv.org/abs/1801.08447v1<\/a>","<a href=\"https://doi.org/10.1371/journal.pone.0193067\">https://doi.org/10.1371/journal.pone.0193067<\/a>","<a href=\"https://doi.org/10.1101/290262\">https://doi.org/10.1101/290262<\/a>","<a href=\"https://doi.org/10.5281/zenodo.1250069\">https://doi.org/10.5281/zenodo.1250069<\/a>","<a href=\"https://doi.org/10.1101/146795\">https://doi.org/10.1101/146795<\/a>","<a href=\"https://doi.org/10.1002/cem.2929\">https://doi.org/10.1002/cem.2929<\/a>","<a href=\"https://doi.org/10.1101/221671\">https://doi.org/10.1101/221671<\/a>","<a href=\"https://doi.org/10.1101/077487\">https://doi.org/10.1101/077487<\/a>","<a href=\"https://doi.org/10.1186/1471-2105-14-S5-S11\">https://doi.org/10.1186/1471-2105-14-S5-S11<\/a>"],["Aaron M Newman","Kosuke Yoshihara","Shai S Shen-Orr","Richard A Moffitt","Alexander R. Abbas","Etienne Becht","Alexandre Kuhn","Ting Gong","Renaud Gaujoux","Ting Gong","Yi Zhong","Peng Lu","Gerald Quon","Dvir Aran","Maria Chikina","Zeev Altboum","Jaeil Ahn","Jennifer Clarke","Timo Erkkilä","Wenjun Ju","Bo Li","David Venet","Christopher R Bolen","Wenlian Qiao","Dirk Repsilber","Jason E Shoemaker","Debashis Ghosh","Niya Wang","Gerald Quon","Niya Wang","Harri Lähdesmäki","Jitao David Zhang","Neta S. Zuckerman","Min Wang","Mark M. Gosink","Julien Racle","David A. Liebner","Amit Frishberg","Yael Steuerman","Bradlee D. Nelms","Max Schelker","Qi Shen","David Tamborero","Casey P. Shannon","Amit Frishberg","Francesco Vallania","Francesca Finotello","Oyetunji E. Ogundijo","Gabriele Pollara","Martin Hoffmann","Nico Riedel","Sushmita Roy","Shu-Hwa Chen","Jing He","Franziska Görtler","Lee A. Newberg","Gregory J. Hunt","Urszula Czerwinska","Zeya Wang","Yuan Liu","Maxim E Zaslavsky","Ajit Johnson Nirmal","Yi Li "],[2015,2013,2010,2015,2009,2016,2011,2011,2012,2013,2013,2003,2013,2017,2015,2014,2013,2010,2010,2013,2013,2001,2011,2012,2010,2012,2004,2016,2009,2014,2005,2017,2013,2006,2007,2017,2013,2016,2016,2016,2017,2016,2018,2017,2015,2017,2017,2017,2017,2006,2013,2006,2018,2018,2018,2018,2018,2018,2017,2017,2017,2016,2013],["FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","FALSE","TRUE","TRUE","TRUE","FALSE","TRUE","FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE"],["FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE*","TRUE","TRUE","FALSE","TRUE*","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","FALSE","TRUE*","TRUE","FALSE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE*","FALSE","TRUE","TRUE","TRUE","TRUE*","FALSE","TRUE","FALSE","FALSE","FLASE","FALSE","FALSE","FLASE","FALSE","TRUE","FALSE","FLASE","FALSE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE"],["Cancer transcriptome","Cancer transcriptome","Blood","detection of cancer and stroma in PDAC (TCGA)","Blood","Cancer transcriptome","Brain tissue","Blood","Blood","Tissue mixtures","Cancer transcriptome","yeast cell cycle","Cancer transcriptome","Cancer transcriptome","Blood","Mice blood under flu infection","Cancer purity","Cancer xenografts","Tissue mixtures","Chronic kidney disease (Cell lineages)","Cancer transcriptome","cander and normal tissue","Blood","blood","Blood","Infected lung tissue","Cancer transcriptome","yeast cell cycle","Cancer transcriptome","Cancer transcriptome","In vitro tissue mixtures","Gene expression","Cancer transcriptome","Murine mammary gland","Infected macrophages","Cancer transcriptome","in vitro tissue mlxtures","Human tissues","Lung tissue","Brain tissue","Cancer transcriptome","Tumor purity","Cancer transcriptome","Blood gene expression","Mice diseased tissues","Blood, solid tissue, disease","Cancer transcriptome","Tissue mixtures","Skin tuberculosis","Synovial tissue (cell types in silico)","Udefined","Yeast cell cycle","Blood","Cancer transcriptome","Cancer transcriptome","yeast cell cycle","Blood","Cancer transcriptome","Cancer transcriptome","Cancer transcriptome","Cancer transcriptome","Cancer transcriptome","in vitro cell mixtures"],["<a href=\"http://cibersort.stanford.edu/\">http://cibersort.stanford.edu/<\/a>","<a href=\"https://sourceforge.net/projects/estimateproject/\">https://sourceforge.net/projects/estimateproject/<\/a>","<a href=\"https://github.com/shenorrLab/csSAM\">https://github.com/shenorrLab/csSAM<\/a>","NA","NA","<a href=\"https://github.com/ebecht/MCPcounter\">https://github.com/ebecht/MCPcounter<\/a>","<a href=\"https://bioconductor.org/packages/release/bioc/html/PSEA.html\">https://bioconductor.org/packages/release/bioc/html/PSEA.html<\/a>","NA","<a href=\"https://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix/\">https://web.cbio.uct.ac.za/~renaud/CRAN/web/CellMix/<\/a>","<a href=\"https://www.bioconductor.org/packages/release/bioc/html/DeconRNASeq.html\">https://www.bioconductor.org/packages/release/bioc/html/DeconRNASeq.html<\/a>","NA","broken link","<a href=\"https://qlab.faculty.ucdavis.edu/isopure/\">https://qlab.faculty.ucdavis.edu/isopure/<\/a>","<a href=\"http://xcell.ucsf.edu/; https://github.com/dviraran/xCell\">http://xcell.ucsf.edu/; https://github.com/dviraran/xCell<\/a>","<a href=\"http://www.pitt.edu/~mchikina/CellCODE/\">http://www.pitt.edu/~mchikina/CellCODE/<\/a>","<a href=\"http://www.dcq.tau.ac.il/\">http://www.dcq.tau.ac.il/<\/a>","<a href=\"http://odin.mdacc.tmc.edu/∼wwang7/DeMix.html.\">http://odin.mdacc.tmc.edu/∼wwang7/DeMix.html.<\/a>","NA","<a href=\"http://informatics.systemsbiology.net/DSection\">http://informatics.systemsbiology.net/DSection<\/a>","<a href=\"http://nano.princeton.edu/\">http://nano.princeton.edu/<\/a>","<a href=\"http://cistrome.org/TIMER/\">http://cistrome.org/TIMER/<\/a>","NA","<a href=\"http://clip.med.yale.edu/SPEC/\">http://clip.med.yale.edu/SPEC/<\/a>","<a href=\"https://github.com/gquon/PERT\">https://github.com/gquon/PERT<\/a>","<a href=\"https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-11-27/MediaObjects/12859_2009_3484_MOESM1_ESM.ZIP\">https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-11-27/MediaObjects/12859_2009_3484_MOESM1_ESM.ZIP<\/a>","<a href=\"http://www.influenza-x.org/~jshoemaker/cten/\">http://www.influenza-x.org/~jshoemaker/cten/<\/a>","broken link","<a href=\"http://mloss.org/software/view/437,\">http://mloss.org/software/view/437,<\/a>","<a href=\"https://qlab.faculty.ucdavis.edu/isolate/\">https://qlab.faculty.ucdavis.edu/isolate/<\/a>","<a href=\"https://www.bioconductor.org/packages/release/bioc/html/UNDO.html\">https://www.bioconductor.org/packages/release/bioc/html/UNDO.html<\/a>","NA","<a href=\"https://www.bioconductor.org/packages/release/bioc/html/BioQC.html\">https://www.bioconductor.org/packages/release/bioc/html/BioQC.html<\/a>","NA","NA","NA","<a href=\"https://github.com/GfellerLab/EPIC\">https://github.com/GfellerLab/EPIC<\/a>","<a href=\" http://sourceforge.net/projects/mmad/\"> http://sourceforge.net/projects/mmad/<\/a>","<a href=\"http://csgi.tau.ac.il/ImmQuant/\">http://csgi.tau.ac.il/ImmQuant/<\/a>","<a href=\"https://cran.r-project.org/web/packages/ComICS/index.html\">https://cran.r-project.org/web/packages/ComICS/index.html<\/a>","<a href=\"http://bioconductor.org/packages/release/bioc/html/CellMapper.html\">http://bioconductor.org/packages/release/bioc/html/CellMapper.html<\/a>","NA","<a href=\"https://github.com/zhanghfd/contamDE/\">https://github.com/zhanghfd/contamDE/<\/a>","NA","<a href=\"https://github.com/cashoes/enumerateblood\">https://github.com/cashoes/enumerateblood<\/a>","<a href=\"http://www.csgi.tau.ac.il/CoD/\">http://www.csgi.tau.ac.il/CoD/<\/a>","NA","<a href=\"http://icbi.at/software/quantiseq/doc/index.html\">http://icbi.at/software/quantiseq/doc/index.html<\/a>","<a href=\"https://github.com/moyanre/smcgenedeconv\">https://github.com/moyanre/smcgenedeconv<\/a>","<a href=\"https://github.com/MJMurray1/MDIScoring\">https://github.com/MJMurray1/MDIScoring<\/a>","NA","NA","NA","<a href=\"https://testtoolshed.g2.bx.psu.edu/repository?repository_id=6e9a9ab163e578e0&changeset_revision=e3afe097e80a\">https://testtoolshed.g2.bx.psu.edu/repository?repository_id=6e9a9ab163e578e0&changeset_revision=e3afe097e80a<\/a>","NA","NA","<a href=\"https:// github.com/GeneralElectric/CcellDdistinguisher\">https:// github.com/GeneralElectric/CcellDdistinguisher<\/a>","<a href=\"cran.r-project.org/package=dtangle\">cran.r-project.org/package=dtangle<\/a>","<a href=\"https://urszulaczerwinska.github.io/DeconICA/\">https://urszulaczerwinska.github.io/DeconICA/<\/a>","<a href=\"https://github.com/wwylab/DeMixT\">https://github.com/wwylab/DeMixT<\/a>","NA","<a href=\"https://github.com/hammerlab/infino\">https://github.com/hammerlab/infino<\/a>","NA","<a href=\"https://github.com/uci-cbcl/TEMT\">https://github.com/uci-cbcl/TEMT<\/a>"],["FALSE","FALSE","TRUE","TRUE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","FALSE","TRUE","FALSE","FALSE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","FALSE","FALSE"],["TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE"],[null,"purity estimation","computes DEG",null,null,null,null,null,null,null,null,null,"purity estimation","deep deconvolution","goal improve DEG, plugs marker-genes based covariated to the SVD","comparing conditions (normal vs infection)","on log transformed data",null,null,"identifies genes","made for TCGA, other application need adaptation",null,null,null,null,"first enrichment based","improve DEG","partial profiles","designed for detection of site of origin",null,null,"heterogeneity detection",null,null,"pays attention to rare cell types","absolute quantities, requires TPM normalized data",null,null,"retruns association with eQTLS","ranks genes, computes p-value of specificity",null,"computes differential gene expression staistics",null,"speciffic to Affymetrix Gene ST, pre-trained","distinguish cell types important in a disease",null,"absolute quantities from images, strating with raw data, returns cell densities","DEG analysis","optimisation of signature genes",null,"theoretical solution","deaing with missign values and time dependency","Galaxy platform pluggable tool",null,null,"partial profiles",null,"metagene profiles",null,"DEG analysis","specalised in deep deconvolution",null,"use reads"],["regression","enrichment","regression","matrix factorisation","regression","enrichment","regression","regression","matrix factorisation","regression","regression","regression","probabilistic","enrichment","matrix factorisation","regression","probabilistic","regression","probabilistic","regression","regression","matrix factorisation","enrichment","probabilistic","matrix factorisation","enrichment","probabilistic","convex hull","matrix factorisation","matrix factorisation","probabilistic","enrichment","matrix factorisation","regression","regression","regression","regression","regression","regression","matrix factorisation","regression","probabilistic","enrichment","probabilistic","regression","regression","regression","probabilistic","enrichment","regression","probabilistic","probabilistic","regression","probabilistic","regression","convex hull","regression","matrix factorisation","probabilistic","matrix factorisation","probabilistic","enrichment","probabilistic"],["R, web tool","R","R","matlab","R","R","R","unknown","R","R","unknown","Java 2","matlab, R","R, web tool","R- C-C++- Fortran","web tool","C, R","unknown","matlab","web tool","web tool","unknown","R","octave","R","web tool","R","R-java","matlab","R","unknown","R","matlab","unknown","unknown","R","matlab","web tool","R","R","unknown","R","unknown","R","web tool","R","web tool","matlab","R","unknown","unknown","unknown","R, web tool","R","unknown","R","R","R, matlab","R","matlab","Stan","unknown","python"],[343,266,286,86,207,42,96,76,61,52,52,135,44,14,28,32,38,53,52,33,33,96,39,33,41,31,66,12,37,18,45,6,18,36,30,4,11,5,5,5,3,4,1,2,4,1,1,1,1,6,2,4,0,0,0,0,0,0,0,0,0,0,0],[85.75,44.3333333333333,31.7777777777778,21.5,20.7,14,12,9.5,8.71428571428571,8.66666666666667,8.66666666666667,8.4375,7.33333333333333,7,7,6.4,6.33333333333333,5.88888888888889,5.77777777777778,5.5,5.5,5.33333333333333,4.875,4.71428571428571,4.55555555555556,4.42857142857143,4.4,4,3.7,3.6,3.21428571428571,3,3,2.76923076923077,2.5,2,1.83333333333333,1.66666666666667,1.66666666666667,1.66666666666667,1.5,1.33333333333333,1,1,1,0.5,0.5,0.5,0.5,0.461538461538462,0.333333333333333,0.307692307692308,0,0,0,0,0,0,0,0,0,0,0],["journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","journal","bioRxiv","bioRxiv","journal","journal","journal","arXiv","journal","journal","bioRxiv","bioRxiv","journal","bioRxiv","NA","bioRxiv","journal","bioRxiv","bioRxiv","journal"],["TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","TRUE","TRUE","TRUE","FALSE","TRUE","FALSE","TRUE","TRUE","FALSE","FALSE","FALSE","FALSE","TRUE","FALSE","TRUE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","FALSE","TRUE"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>name<\/th>\n      <th>number of sources<\/th>\n      <th>data<\/th>\n      <th>type<\/th>\n      <th>method<\/th>\n      <th>doi<\/th>\n      <th>author<\/th>\n      <th>year<\/th>\n      <th>proportions.in<\/th>\n      <th>profiles.in<\/th>\n      <th>application<\/th>\n      <th>availability<\/th>\n      <th>out.profiles<\/th>\n      <th>out.proportions<\/th>\n      <th>comments<\/th>\n      <th>category<\/th>\n      <th>language<\/th>\n      <th>citations<\/th>\n      <th>pop.index<\/th>\n      <th>published<\/th>\n      <th>previously.covered<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"width":"10em","targets":[1,2,14]},{"width":"30em","targets":[4]},{"width":"5em","targets":[5,7,8,9]},{"className":"dt-right","targets":[7,17,18]}],"pageLength":20,"autoWidth":true,"scrollX":true,"order":[],"orderClasses":false,"orderCellsTop":true,"lengthMenu":[10,20,25,50,100]}},"evals":[],"jsHooks":[]}</script>

<p>Tab. <a href="methods.html#tab:mytab">2.1</a></p>
<div class="figure" style="text-align: center"><span id="fig:pubyear"></span>
<div id="699740fea59b" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="699740fea59b">{"x":{"data":[{"orientation":"v","width":[0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5,0.5],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"x":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16],"y":[1,1,1,1,3,1,2,4,3,3,11,2,4,7,12,7],"text":["~year: 2001<br />~`nb. of papers`:  1","~year: 2003<br />~`nb. of papers`:  1","~year: 2004<br />~`nb. of papers`:  1","~year: 2005<br />~`nb. of papers`:  1","~year: 2006<br />~`nb. of papers`:  3","~year: 2007<br />~`nb. of papers`:  1","~year: 2009<br />~`nb. of papers`:  2","~year: 2010<br />~`nb. of papers`:  4","~year: 2011<br />~`nb. of papers`:  3","~year: 2012<br />~`nb. of papers`:  3","~year: 2013<br />~`nb. of papers`: 11","~year: 2014<br />~`nb. of papers`:  2","~year: 2015<br />~`nb. of papers`:  4","~year: 2016<br />~`nb. of papers`:  7","~year: 2017<br />~`nb. of papers`: 12","~year: 2018<br />~`nb. of papers`:  7"],"type":"bar","marker":{"autocolorscale":false,"color":"rgba(190,190,190,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[13],"y":[4.3],"text":"CIBERSORT","hovertext":"~x: 13<br />~y: 4.3","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[11],"y":[11.6],"text":"ESTIMATE<br /> CellMix toolbox","hovertext":"~x: 11<br />~y: 11.6","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[14],"y":[7.3],"text":"MCPcounter","hovertext":"~x: 14<br />~y: 7.3","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[15],"y":[12.6],"text":"xCell<br /> EPIC","hovertext":"~x: 15<br />~y: 12.6","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[7],"y":[2.3],"text":"Abbas et al.","hovertext":"~x: 7<br />~y: 2.3","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[8],"y":[4.3],"text":"csSAM","hovertext":"~x: 8<br />~y: 4.3","textfont":{"size":14.6645669291339,"color":"rgba(255,0,0,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":26.2283105022831,"r":7.30593607305936,"b":40.1826484018265,"l":37.2602739726027},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"type":"linear","autorange":false,"range":[0.4,16.6],"tickmode":"array","ticktext":["2001","2003","2004","2005","2006","2007","2009","2010","2011","2012","2013","2014","2015","2016","2017","2018"],"tickvals":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16],"categoryorder":"array","categoryarray":["2001","2003","2004","2005","2006","2007","2009","2010","2011","2012","2013","2014","2015","2016","2017","2018"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"year","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"type":"linear","autorange":false,"range":[-0.63,13.23],"tickmode":"array","ticktext":["0","5","10"],"tickvals":[0,5,10],"categoryorder":"array","categoryarray":["0","5","10"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"nb. of papers","titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"source":"A","attrs":{"69973ca7abde":{"x":{},"y":{},"width":{},"type":"bar"},"699724049aee":{"x":{},"y":{}},"69972a5791bb":{"x":{},"y":{}},"69975b20a1b4":{"x":{},"y":{}},"69973b585f29":{"x":{},"y":{}},"699726cf9b2f":{"x":{},"y":{}},"69977d53e91":{"x":{},"y":{}}},"cur_data":"69973ca7abde","visdat":{"69973ca7abde":["function (y) ","x"],"699724049aee":["function (y) ","x"],"69972a5791bb":["function (y) ","x"],"69975b20a1b4":["function (y) ","x"],"69973b585f29":["function (y) ","x"],"699726cf9b2f":["function (y) ","x"],"69977d53e91":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p class="caption">
Figure 2.2: <strong>Distibution of publications of cell-type deconvolution of bulk transcriptome over the years</strong>. In red: hallmark publications. Data gathered based on pubmed and google scholar search in May 2018.
</p>
</div>

<div class="figure" style="text-align: center"><span id="fig:languagesA"></span>
<div id="69973d48e59" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="69973d48e59">{"x":{"visdat":{"6997afd6c5a":["function () ","plotlyVisDat"]},"cur_data":"6997afd6c5a","attrs":{"6997afd6c5a":{"labels":{},"values":{},"textposition":"inside","textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"colors":["function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}"],"line":{"color":"#FFFFFF","width":1}},"alpha":1,"sizes":[10,100],"type":"pie"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"title":"a) Percentage of used approach type","xaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"yaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"data":[{"labels":["convex hull","enrichment","matrix factorisation","probabilistic","regression"],"values":[2,9,11,15,26],"textposition":["inside","inside","inside","inside","inside"],"textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","colors":["structure(function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}, apiSrc = TRUE)"],"line":{"color":"#FFFFFF","width":1}},"type":"pie","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p class="caption">
Figure 2.3: caption
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:languagesB"></span>
<div id="69971233b32e" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="69971233b32e">{"x":{"visdat":{"699748c550ad":["function () ","plotlyVisDat"]},"cur_data":"699748c550ad","attrs":{"699748c550ad":{"labels":{},"values":{},"textposition":"inside","textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"colors":["function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}"],"line":{"color":"#FFFFFF","width":1}},"alpha":1,"sizes":[10,100],"type":"pie"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"title":"b) Percentage of supervised/unsupervised tools","xaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"yaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"data":[{"labels":["BOTH","semi-supervised","supervised","unsupervised"],"values":[1,3,47,12],"textposition":["inside","inside","inside","inside"],"textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","colors":["structure(function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}, apiSrc = TRUE)"],"line":{"color":"#FFFFFF","width":1}},"type":"pie","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p class="caption">
Figure 2.4: caption
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:languagesC"></span>
<div id="69977e0970aa" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="69977e0970aa">{"x":{"visdat":{"699740d7686f":["function () ","plotlyVisDat"]},"cur_data":"699740d7686f","attrs":{"699740d7686f":{"labels":{},"values":{},"textposition":"inside","textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"colors":["function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}"],"line":{"color":"#FFFFFF","width":1}},"alpha":1,"sizes":[10,100],"type":"pie"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"title":"c) Percentage of the programming languages of implementation","xaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"yaxis":{"showgrid":false,"zeroline":false,"showticklabels":false},"hovermode":"closest","showlegend":false},"source":"A","config":{"modeBarButtonsToAdd":[{"name":"Collaborate","icon":{"width":1000,"ascent":500,"descent":-50,"path":"M487 375c7-10 9-23 5-36l-79-259c-3-12-11-23-22-31-11-8-22-12-35-12l-263 0c-15 0-29 5-43 15-13 10-23 23-28 37-5 13-5 25-1 37 0 0 0 3 1 7 1 5 1 8 1 11 0 2 0 4-1 6 0 3-1 5-1 6 1 2 2 4 3 6 1 2 2 4 4 6 2 3 4 5 5 7 5 7 9 16 13 26 4 10 7 19 9 26 0 2 0 5 0 9-1 4-1 6 0 8 0 2 2 5 4 8 3 3 5 5 5 7 4 6 8 15 12 26 4 11 7 19 7 26 1 1 0 4 0 9-1 4-1 7 0 8 1 2 3 5 6 8 4 4 6 6 6 7 4 5 8 13 13 24 4 11 7 20 7 28 1 1 0 4 0 7-1 3-1 6-1 7 0 2 1 4 3 6 1 1 3 4 5 6 2 3 3 5 5 6 1 2 3 5 4 9 2 3 3 7 5 10 1 3 2 6 4 10 2 4 4 7 6 9 2 3 4 5 7 7 3 2 7 3 11 3 3 0 8 0 13-1l0-1c7 2 12 2 14 2l218 0c14 0 25-5 32-16 8-10 10-23 6-37l-79-259c-7-22-13-37-20-43-7-7-19-10-37-10l-248 0c-5 0-9-2-11-5-2-3-2-7 0-12 4-13 18-20 41-20l264 0c5 0 10 2 16 5 5 3 8 6 10 11l85 282c2 5 2 10 2 17 7-3 13-7 17-13z m-304 0c-1-3-1-5 0-7 1-1 3-2 6-2l174 0c2 0 4 1 7 2 2 2 4 4 5 7l6 18c0 3 0 5-1 7-1 1-3 2-6 2l-173 0c-3 0-5-1-8-2-2-2-4-4-4-7z m-24-73c-1-3-1-5 0-7 2-2 3-2 6-2l174 0c2 0 5 0 7 2 3 2 4 4 5 7l6 18c1 2 0 5-1 6-1 2-3 3-5 3l-174 0c-3 0-5-1-7-3-3-1-4-4-5-6z"},"click":"function(gd) { \n        // is this being viewed in RStudio?\n        if (location.search == '?viewer_pane=1') {\n          alert('To learn about plotly for collaboration, visit:\\n https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html');\n        } else {\n          window.open('https://cpsievert.github.io/plotly_book/plot-ly-for-collaboration.html', '_blank');\n        }\n      }"}],"cloud":false},"data":[{"labels":["Java 2","matlab","octave","python","R","Stan","unknown","web tool"],"values":[1,7,1,1,31,1,14,7],"textposition":["inside","inside","inside","inside","inside","inside","inside","inside"],"textinfo":"label+percent","insidetextfont":{"color":"#FFFFFF","size":20},"marker":{"fillcolor":"rgba(31,119,180,1)","color":"rgba(31,119,180,1)","colors":["structure(function (distinct = FALSE) ","{","    c <- .Call(C_colors)","    if (distinct) ","        c[!duplicated(t(col2rgb(c)))]","    else c","}, apiSrc = TRUE)"],"line":{"color":"#FFFFFF","width":1}},"type":"pie","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1}},"base_url":"https://plot.ly"},"evals":["config.modeBarButtonsToAdd.0.click"],"jsHooks":{"render":[{"code":"function(el, x) { var ctConfig = crosstalk.var('plotlyCrosstalkOpts').set({\"on\":\"plotly_click\",\"persistent\":false,\"dynamic\":false,\"selectize\":false,\"opacityDim\":0.2,\"selected\":{\"opacity\":1}}); }","data":null}]}}</script>
<p class="caption">
Figure 2.5: <strong>Simple statistics illustrating characteristics of published cell-type deconvolution tools</strong>: <strong>a)</strong> Percentage of used approach type, <strong>b)</strong> Percentage of supervised/ unsupervised tools, <strong>c)</strong> Percentage of the programming languages of implementation. Data gathered based on pubmed and google scholar search in May 2018.
</p>
</div>

<!-- (ref:lang-caption) this is a captions are: \@ref(fig:languages1) some \@ref(fig:languages2) ererer -->
<ul>
<li>the table</li>
<li>graph : percentage of programming language</li>
<li>nb pub / year</li>
<li>top methods (nb of citation based)</li>
<li>published code</li>
</ul>
</div>
<div id="regression-based-methods" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Regression-based methods</h3>
<div id="basis-matrix" class="section level4">
<h4><span class="header-section-number">2.2.2.1</span> basis matrix</h4>
<ul>
<li>define what basis matrix is vs signatures vs marker genes</li>
<li>explain the differences between basis matrices in published algorithms</li>
</ul>
<p>3.4.7 Signature Matrix</p>
<p>CIBERSORT requires an input matrix of reference gene expression signa- tures called “signature matrix”, and this matrix has in earlier studies been called base or basis matrices according to Newman et al.[7]. The signature matrix possesses a small number of “trademark genes” for pure cell lines. Trademark genes are significantly differentially expressed genes among all the cell lines, meaning that genes that express a high score for one cell line, and a low score for the others, are relevant for the signature matrix. The authors stated that while developing their own signature matrix (LM22), expressed genes were found by using a two-sided unequal variance t-text, and genes with a false discovery rate below &lt;0.3 are considered signific- ant[7]. The significant genes were sorted by decreasing fold change com- pared to other cell lines, and top G were kept for signature matrices. After iterating G from 50 to 200, they kept the signature matrix which had the lowest condition number (11.4) where G was 102 and the final signature matrix consisted of 547 distinct genes[7].</p>
<p>The number of genes present in signature matrix vary based on a combin- ation of the number of pure cell lines used, and how many genes are re- quired to express a specific pure cell line. Genes from the signature matrix are used for recognizing cell subsets within a mixture. The signature mat- rix is an essential part of CIBERSORT, and is required to run CIBERSORT. Newman et al. (2015) state that using a signature matrix (1) speeds up the run time as it ignores genes which do not express any particular cell sub- set, and (2) achieves a better signal-to-noise ratio as it preselects reference profiles having maximal discriminatory power[7]. They also state that their approach is unique compared to others for developing signature matrices like LLSR[8]. The signature genes are central for SVR when deconvolving a mixture, and the result is an estimated mixture calculated by CIBERSORT. The result from CIBERSORT also provides a Pearson correlation which is calculated by comparing the signature genes from the original mixture to the equivalent genes in the estimated mixture.</p>
</div>
</div>
<div id="regression-algorithm" class="section level3">
<h3><span class="header-section-number">2.2.3</span> regression algorithm</h3>
<ul>
<li>explain what it is for</li>
<li>explain how it works</li>
<li>explain the different approaches</li>
</ul>
</div>
<div id="enrichment-based-methods" class="section level3">
<h3><span class="header-section-number">2.2.4</span> Enrichment-based methods</h3>
</div>
<div id="probabilistic-methods" class="section level3">
<h3><span class="header-section-number">2.2.5</span> Probabilistic methods</h3>
</div>
<div id="convex-hull-based-methods" class="section level3">
<h3><span class="header-section-number">2.2.6</span> Convex-hull based methods</h3>
</div>
<div id="matrix-factorisation-methods" class="section level3">
<h3><span class="header-section-number">2.2.7</span> Matrix factorisation methods</h3>
</div>
<div id="others-aspects" class="section level3">
<h3><span class="header-section-number">2.2.8</span> Others aspects</h3>
<ul>
<li><p>explain possible normalisation</p></li>
<li><p>explain additional possible features</p></li>
</ul>
</div>
</div>
<div id="deconvolution-of-other-data-types" class="section level2">
<h2><span class="header-section-number">2.3</span> Deconvolution of other data types</h2>
</div>
<div id="methodological-dimension-of-the-thesis" class="section level2">
<h2><span class="header-section-number">2.4</span> Methodological dimension of the thesis</h2>
<ul>
<li>expose state of art and compare existing tools</li>
<li>adjustment of existing methodology</li>
<li>develop a new user-friendly tool</li>
<li>discuss limitations of computational approaches in biology</li>
<li>data integration</li>
<li>exploration of data</li>
</ul>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="sens.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/03-lntroductionMath.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
