<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8.md</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8.md" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8.md" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="post-scriptum-thesis-writing.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; min-height: 1.25em; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; }
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
@media screen {
a.sourceLine::before { text-decoration: underline; color: initial; }
}
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.bn { color: #40a070; } /* BaseN */
code span.fl { color: #40a070; } /* Float */
code span.ch { color: #4070a0; } /* Char */
code span.st { color: #4070a0; } /* String */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.ot { color: #007020; } /* Other */
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.fu { color: #06287e; } /* Function */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code span.cn { color: #880000; } /* Constant */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.ss { color: #bb6688; } /* SpecialString */
code span.im { } /* Import */
code span.va { color: #19177c; } /* Variable */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.op { color: #666666; } /* Operator */
code span.bu { } /* BuiltIn */
code span.ex { } /* Extension */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.at { color: #7d9029; } /* Attribute */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Note</a></li>
<li class="chapter" data-level="" data-path="resume.html"><a href="resume.html"><i class="fa fa-check"></i>Resumé</a></li>
<li class="chapter" data-level="" data-path="summary.html"><a href="summary.html"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="abbreviations.html"><a href="abbreviations.html"><i class="fa fa-check"></i>Abbreviations</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-phd---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary PhD - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#the-origins-of-the-phd-topic"><i class="fa fa-check"></i>The origins of the PhD topic</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="organisation-of-the-dissertation.html"><a href="organisation-of-the-dissertation.html"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#summary-of-the-chapter"><i class="fa fa-check"></i><b>1.4</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a><ul>
<li class="chapter" data-level="2.1.1" data-path="methods.html"><a href="methods.html#supervised-learning"><i class="fa fa-check"></i><b>2.1.1</b> Supervised learning</a></li>
<li class="chapter" data-level="2.1.2" data-path="methods.html"><a href="methods.html#unsup"><i class="fa fa-check"></i><b>2.1.2</b> Unsupervised learning</a></li>
<li class="chapter" data-level="2.1.3" data-path="methods.html"><a href="methods.html#low-dimensional-embeddingfor-visualization"><i class="fa fa-check"></i><b>2.1.3</b> Low-dimensional embedding for visualization</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#types-of-deconvolution"><i class="fa fa-check"></i><b>2.2</b> Types of deconvolution</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#cellTypeTrans"><i class="fa fa-check"></i><b>2.3</b> Cell-type deconvolution of bulk transcriptomes</a><ul>
<li class="chapter" data-level="2.3.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.3.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.3.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.3.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.3.3" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.3.3</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.3.4" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.3.4</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.3.5" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.3.5</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.3.6" data-path="methods.html"><a href="methods.html#matrix-factorization-methods"><i class="fa fa-check"></i><b>2.3.6</b> Matrix factorization methods</a></li>
<li class="chapter" data-level="2.3.7" data-path="methods.html"><a href="methods.html#attractor-metagenes"><i class="fa fa-check"></i><b>2.3.7</b> Attractor metagenes</a></li>
<li class="chapter" data-level="2.3.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.3.8</b> Others aspects</a></li>
<li class="chapter" data-level="2.3.9" data-path="methods.html"><a href="methods.html#summary-3"><i class="fa fa-check"></i><b>2.3.9</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#otherDecon"><i class="fa fa-check"></i><b>2.4</b> Deconvolution of other data types</a><ul>
<li class="chapter" data-level="2.4.1" data-path="methods.html"><a href="methods.html#dna-methylation-data"><i class="fa fa-check"></i><b>2.4.1</b> DNA methylation data</a></li>
<li class="chapter" data-level="2.4.2" data-path="methods.html"><a href="methods.html#copy-number-aberrations-cna"><i class="fa fa-check"></i><b>2.4.2</b> Copy number aberrations (CNA)</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#summary-of-the-chapter-1"><i class="fa fa-check"></i><b>2.5</b> Summary of the chapter</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="objectives.html"><a href="objectives.html"><i class="fa fa-check"></i>Objectives</a></li>
<li class="part"><span><b>II Results</b></span></li>
<li class="chapter" data-level="3" data-path="mstd.html"><a href="mstd.html"><i class="fa fa-check"></i><b>3</b> Determining the optimal number of independent components for reproducible transcriptomic data analysis</a></li>
<li class="chapter" data-level="4" data-path="lva.html"><a href="lva.html"><i class="fa fa-check"></i><b>4</b> Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals</a></li>
<li class="chapter" data-level="5" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html"><i class="fa fa-check"></i><b>5</b> Comparison of reproducibility between NMF and ICA</a><ul>
<li class="chapter" data-level="5.0.1" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html#comparing-metagenes-obtained-with-nmf-vs-ica."><i class="fa fa-check"></i><b>5.0.1</b> Comparing metagenes obtained with NMF vs ICA.</a></li>
<li class="chapter" data-level="5.1" data-path="comparison-of-reproducibility-between-nmf-and-ica.html"><a href="comparison-of-reproducibility-between-nmf-and-ica.html#impact-of-modification-of-signatures-list-on-result-for-signature-based-deconvolution-methods"><i class="fa fa-check"></i><b>5.1</b> ? Impact of modification of signatures list on result for signature-based deconvolution methods</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>6</b> Deconvolution of transcriptomes and methylomes</a><ul>
<li class="chapter" data-level="6.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>6.1</b> From blind deconvolution to cell-type quantification: general overview</a><ul>
<li class="chapter" data-level="6.1.1" data-path="deconica.html"><a href="deconica.html#the-ica-based-deconvolution-of-transcriptomes"><i class="fa fa-check"></i><b>6.1.1</b> The ICA-based deconvolution of Transcriptomes</a></li>
<li class="chapter" data-level="6.1.2" data-path="deconica.html"><a href="deconica.html#interpretation-of-independent-components"><i class="fa fa-check"></i><b>6.1.2</b> Interpretation of Independent components</a></li>
<li class="chapter" data-level="6.1.3" data-path="deconica.html"><a href="deconica.html#transforming-metagenes-into-signature-matrix"><i class="fa fa-check"></i><b>6.1.3</b> Transforming metagenes into signature matrix</a></li>
<li class="chapter" data-level="6.1.4" data-path="deconica.html"><a href="deconica.html#regression-based-estimation-of-cell-type-proportions-solving-system-of-equations"><i class="fa fa-check"></i><b>6.1.4</b> Regression-based estimation of cell-type proportions : solving system of equations</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="deconica.html"><a href="deconica.html#deconica-r-package-for-ica-based-deconvolution"><i class="fa fa-check"></i><b>6.2</b> <em>DeconICA</em> R package for ICA-based deconvolution</a><ul>
<li class="chapter" data-level="6.2.1" data-path="deconica.html"><a href="deconica.html#demo"><i class="fa fa-check"></i><b>6.2.1</b> <em>Demo</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>7</b> Comparative analysis of cancer immune infiltration</a></li>
<li class="chapter" data-level="8" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>8</b> Heterogeneity of immune cell types</a></li>
<li class="part"><span><b>III Discussion</b></span></li>
<li class="chapter" data-level="9" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>9</b> Discussion</a></li>
<li class="chapter" data-level="10" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>10</b> Conclusions and perspectives</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#dc-subtypes"><i class="fa fa-check"></i>Dc subtypes</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#dreamidea-challenge"><i class="fa fa-check"></i>DreamIdea Challenge</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#full-list-of-publications"><i class="fa fa-check"></i>Full list of publications</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#cv"><i class="fa fa-check"></i>CV</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="post-scriptum-thesis-writing.html"><a href="post-scriptum-thesis-writing.html"><i class="fa fa-check"></i>Post Scriptum: Thesis writing</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>

<div id="refs" class="references">
<div>
<p>Abbas, Alexander R., Kristen Wolslegel, Dhaya Seshasayee, Zora Modrusan, and Hilary F. Clark. 2009. “Deconvolution of Blood Microarray Data Identifies Cellular Activation Patterns in Systemic Lupus Erythematosus.” Edited by Patrick Tan. <em>PLoS One</em> 4 (7). Public Library of Science: e6098. doi:<a href="https://doi.org/10.1371/journal.pone.0006098">10.1371/journal.pone.0006098</a>.</p>
</div>
<div>
<p>Al-Ejeh, F, P T Simpson, J M Sanus, K Klein, M Kalimutho, W Shi, M Miranda, et al. 2014. “Meta-analysis of the global gene expression profile of triple-negative breast cancer identifies genes for the prognostication and treatment of aggressive breast cancer.” <em>Oncogenesis</em> 3 (4). Nature Publishing Group: e100–e100. doi:<a href="https://doi.org/10.1038/oncsis.2014.14">10.1038/oncsis.2014.14</a>.</p>
</div>
<div>
<p>Andor, Noemi, Julie V. Harness, Sabine Müller, Hans W. Mewes, and Claudia Petritsch. 2014. “EXPANDS: expanding ploidy and allele frequency on nested subpopulations.” <em>Bioinformatics</em> 30 (1). Oxford University Press: 50–60. doi:<a href="https://doi.org/10.1093/bioinformatics/btt622">10.1093/bioinformatics/btt622</a>.</p>
</div>
<div>
<p>Angelova, Mihaela, Pornpimol Charoentong, Hubert Hackl, Maria L Fischer, Rene Snajder, Anne M Krogsdam, Maximilian J Waldner, et al. 2015. “Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.” <em>Genome Biol.</em> 16 (1). BioMed Central: 64. doi:<a href="https://doi.org/10.1186/s13059-015-0620-6">10.1186/s13059-015-0620-6</a>.</p>
</div>
<div>
<p>Anitei, M G, G Zeitoun, B Mlecnik, F Marliot, N Haicheur, A M Todosi, A Kirilovsky, et al. 2014. “Prognostic and predictive values of the immunoscore in patients with rectal cancer.” <em>Clin Cancer Res</em> 20 (7): 1891–9. doi:<a href="https://doi.org/10.1158/1078-0432.ccr-13-2830">10.1158/1078-0432.ccr-13-2830</a>.</p>
</div>
<div>
<p>Aran, Dvir, Zicheng Hu, and Atul J. Butte. 2017. “xCell: digitally portraying the tissue cellular heterogeneity landscape.” <em>Genome Biol.</em> 18 (1). BioMed Central: 220. doi:<a href="https://doi.org/10.1186/s13059-017-1349-1">10.1186/s13059-017-1349-1</a>.</p>
</div>
<div>
<p>Aran, Dvir, Marina Sirota, and Atul J Butte. 2015. “Systematic pan-cancer analysis of tumour purity.” <em>Nat. Commun.</em> 6. Nature Publishing Group: 8971. doi:<a href="https://doi.org/10.1038/ncomms9971">10.1038/ncomms9971</a>.</p>
</div>
<div>
<p>Aris, Mariana, María Marcela Barrio, and José Mordoh. 2012. “Lessons from Cancer Immunoediting in Cutaneous Melanoma.” <em>Clin. Dev. Immunol.</em> 2012: 1–14. doi:<a href="https://doi.org/10.1155/2012/192719">10.1155/2012/192719</a>.</p>
</div>
<div>
<p>Arora, Sanjeev, Rong Ge, Yoni Halpern, David Mimno, Ankur Moitra, David Sontag, Yichen Wu, and Michael Zhu. 2013. “A practical algorithm for topic modeling with provable guarantees.” JMLR.org. <a href="https://dl.acm.org/citation.cfm?id=3042925" class="uri">https://dl.acm.org/citation.cfm?id=3042925</a>.</p>
</div>
<div>
<p>Azencott, Chloé-Agathe, Tero Aittokallio, Sushmita Roy, Ankit Agrawal, Tero Aittokallio, Chloé-Agathe Azencott, Emmanuel Barillot, et al. 2017. “The inconvenience of data of convenience: computational research beyond post-mortem analyses.” <em>Nat. Methods</em> 14 (10). Nature Publishing Group: 937–38. doi:<a href="https://doi.org/10.1038/nmeth.4457">10.1038/nmeth.4457</a>.</p>
</div>
<div>
<p>Balkwill, Fran, and Alberto Mantovani. 2001. “Inflammation and cancer: Back to Virchow?” doi:<a href="https://doi.org/10.1016/S0140-6736(00)04046-0">10.1016/S0140-6736(00)04046-0</a>.</p>
</div>
<div>
<p>Barbie, David A., Pablo Tamayo, Jesse S. Boehm, So Young Kim, Susan E. Moody, Ian F. Dunn, Anna C. Schinzel, et al. 2009. “Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.” <em>Nature</em> 462 (7269). Nature Publishing Group: 108–12. doi:<a href="https://doi.org/10.1038/nature08460">10.1038/nature08460</a>.</p>
</div>
<div>
<p>Baxevanis, C N, G V Dedoussis, N G Papadopoulos, I Missitzis, G P Stathopoulos, and M Papamichail. 1994. “Tumor specific cytolysis by tumor infiltrating lymphocytes in breast cancer.” <em>Cancer</em> 74 (4): 1275–82. <a href="http://www.ncbi.nlm.nih.gov/pubmed/7914469" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/7914469</a>.</p>
</div>
<div>
<p>Becht, Etienne, Charles-Antoine Dutertre, Immanuel W.H. Kwok, Lai Guan Ng, Florent Ginhoux, and Evan W Newell. 2018. “Evaluation of UMAP as an alternative to t-SNE for single-cell data.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 298430. doi:<a href="https://doi.org/10.1101/298430">10.1101/298430</a>.</p>
</div>
<div>
<p>Becht, Etienne, Aurélien de Reyniès, Nicolas A Giraldo, Camilla Pilati, Bénédicte Buttard, Laetitia Lacroix, Janick Selves, Catherine Sautès-Fridman, Pierre Laurent-Puig, and Wolf Herman Fridman. 2016. “Immune and Stromal Classification of Colorectal Cancer Is Associated with Molecular Subtypes and Relevant for Precision Immunotherapy.” <em>Clin. Cancer Res.</em> 22 (16). American Association for Cancer Research: 4057–66. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-15-2879">10.1158/1078-0432.CCR-15-2879</a>.</p>
</div>
<div>
<p>Bell, Anthony J, and Terrence J Sejnowski. 1995. “An information-maximisation approach to blind separation and blind deconvolution.” <a href="http://www.inf.fu-berlin.de/lehre/WS05/Mustererkennung/infomax/infomax.pdf" class="uri">http://www.inf.fu-berlin.de/lehre/WS05/Mustererkennung/infomax/infomax.pdf</a>.</p>
</div>
<div>
<p>Berghoff, Anna, Jakob Kather, and Dirk Jäger. 2018. “Genetics and Immunology: Tumor-Specific Genetic Alterations as a Target for Immune Modulating Therapies.” In <em>Oncoimmunology : A Practical Guide for Cancer Immunotherapy</em>, edited by Laurence Zitvogel and Guido Kroemer, 231–46.</p>
</div>
<div>
<p>Bernstein, Bradley E, Alexander Meissner, and Eric S Lander. 2007. “The mammalian epigenome.” <em>Cell</em> 128 (4). Elsevier: 669–81. doi:<a href="https://doi.org/10.1016/j.cell.2007.01.033">10.1016/j.cell.2007.01.033</a>.</p>
</div>
<div>
<p>Bhatia, Alka, and Yashwant Kumar. 2011. “Cancer-immune equilibrium: questions unanswered.” <em>Cancer Microenviron.</em> 4 (2). Springer: 209–17. doi:<a href="https://doi.org/10.1007/s12307-011-0065-8">10.1007/s12307-011-0065-8</a>.</p>
</div>
<div>
<p>Bindea, Gabriela, Bernhard Mlecnik, Marie Tosolini, Amos Kirilovsky, Maximilian Waldner, Anna C. Obenauf, Helen Angell, et al. 2013. “Spatiotemporal Dynamics of Intratumoral Immune Cells Reveal the Immune Landscape in Human Cancer.” <em>Immunity</em> 39 (4): 782–95. doi:<a href="https://doi.org/10.1016/j.immuni.2013.10.003">10.1016/j.immuni.2013.10.003</a>.</p>
</div>
<div>
<p>Biton, Anne, Isabelle Bernard-Pierrot, Yinjun Lou, Clémentine Krucker, Elodie Chapeaublanc, Carlota Rubio-Pérez, Nuria López-Bigas, et al. 2014. “Independent Component Analysis Uncovers the Landscape of the Bladder Tumor Transcriptome and Reveals Insights into Luminal and Basal Subtypes.” <em>Cell Rep.</em> 9 (4): 1235–45. doi:<a href="https://doi.org/10.1016/j.celrep.2014.10.035">10.1016/j.celrep.2014.10.035</a>.</p>
</div>
<div>
<p>Blank, Christian U, John B Haanen, Antoni Ribas, and Ton N Schumacher. 2016. “CANCER IMMUNOLOGY. the ‘Cancer Immunogram’.” <em>Science</em> 352 (6286). American Association for the Advancement of Science: 658–60. doi:<a href="https://doi.org/10.1126/science.aaf2834">10.1126/science.aaf2834</a>.</p>
</div>
<div>
<p>Boal, David H. 2002. <em>Mechanics of the cell</em>. Cambridge University Press.</p>
</div>
<div>
<p>Breeze, Charles E., Dirk S. Paul, Jenny van Dongen, Lee M. Butcher, John C. Ambrose, James E. Barrett, Robert Lowe, et al. 2016. “eFORGE: A Tool for Identifying Cell Type-Specific Signal in Epigenomic Data.” <em>Cell Rep.</em> 17 (8): 2137–50. doi:<a href="https://doi.org/10.1016/j.celrep.2016.10.059">10.1016/j.celrep.2016.10.059</a>.</p>
</div>
<div>
<p>Brunet, Jean-Philippe, Pablo Tamayo, Todd R Golub, and Jill P Mesirov. 2004. “Metagenes and molecular pattern discovery using matrix factorization.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 101 (12). National Academy of Sciences: 4164–9. doi:<a href="https://doi.org/10.1073/pnas.0308531101">10.1073/pnas.0308531101</a>.</p>
</div>
<div>
<p>Burnet, F M. 1970. “The concept of immunological surveillance.” <em>Prog. Exp. Tumor Res.</em> 13: 1–27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4921480" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/4921480</a>.</p>
</div>
<div>
<p>Cantini, Laura, Ulykbek Kairov, Aurelien de Reynies, Emmanuel Barillot, Francois Radvanyi, and Andrei Zinovyev. 2018. “Stabilized Independent Component Analysis outperforms other methods in finding reproducible signals in tumoral transcriptomes.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 318154. doi:<a href="https://doi.org/10.1101/318154">10.1101/318154</a>.</p>
</div>
<div>
<p>Cardoso, J.F., and A. Souloumiac. 1993. “Blind beamforming for non-gaussian signals.” <em>IEE Proc. F Radar Signal Process.</em> 140 (6): 362. doi:<a href="https://doi.org/10.1049/ip-f-2.1993.0054">10.1049/ip-f-2.1993.0054</a>.</p>
</div>
<div>
<p>Carter, Scott L, Kristian Cibulskis, Elena Helman, Aaron McKenna, Hui Shen, Travis Zack, Peter W Laird, et al. 2012. “Absolute quantification of somatic DNA alterations in human cancer.” <em>Nat. Biotechnol.</em> 30 (5). Nature Publishing Group: 413–21. doi:<a href="https://doi.org/10.1038/nbt.2203">10.1038/nbt.2203</a>.</p>
</div>
<div>
<p>Charoentong, Pornpimol, Francesca Finotello, Mihaela Angelova, Clemens Mayer, Mirjana Efremova, Dietmar Rieder, Hubert Hackl, and Zlatko Trajanoski. 2017. “Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade.” <em>Cell Rep.</em> 18 (1). Elsevier: 248–62. doi:<a href="https://doi.org/10.1016/j.celrep.2016.12.019">10.1016/j.celrep.2016.12.019</a>.</p>
</div>
<div>
<p>Chen, Daniel S., and Ira Mellman. 2017. “Elements of cancer immunity and the cancer-immune set point.” doi:<a href="https://doi.org/10.1038/nature21349">10.1038/nature21349</a>.</p>
</div>
<div>
<p>Chen, Pei Ling, Whijae Roh, Alexandre Reuben, Zachary A. Cooper, Christine N. Spencer, Peter A. Prieto, John P. Miller, et al. 2016. “Analysis of immune signatures in longitudinal tumor samples yields insight into biomarkers of response and mechanisms of resistance to immune checkpoint blockade.” <em>Cancer Discov.</em> 6 (8): 827–37. doi:<a href="https://doi.org/10.1158/2159-8290.CD-15-1545">10.1158/2159-8290.CD-15-1545</a>.</p>
</div>
<div>
<p>Chen, Shu-Hwa, Wen-Yu Kuo, Sheng-Yao Su, Wei-Chun Chung, Jen-Ming Ho, Henry Horng-Shing Lu, and Chung-Yen Lin. 2018. “A gene profiling deconvolution approach to estimating immune cell composition from complex tissues.” <em>BMC Bioinformatics</em> 19 (S4). BioMed Central: 154. doi:<a href="https://doi.org/10.1186/s12859-018-2069-6">10.1186/s12859-018-2069-6</a>.</p>
</div>
<div>
<p>Cheng, Wei-Yi, Tai-Hsien Ou Yang, and Dimitris Anastassiou. 2013. “Biomolecular Events in Cancer Revealed by Attractor Metagenes.” Edited by Isidore Rigoutsos. <em>PLoS Comput. Biol.</em> 9 (2). Public Library of Science: e1002920. doi:<a href="https://doi.org/10.1371/journal.pcbi.1002920">10.1371/journal.pcbi.1002920</a>.</p>
</div>
<div>
<p>Cherry, E. Colin. 1953. “Some Experiments on the Recognition of Speech, with One and with Two Ears.” <em>J. Acoust. Soc. Am.</em> 25 (5). Acoustical Society of America: 975–79. doi:<a href="https://doi.org/10.1121/1.1907229">10.1121/1.1907229</a>.</p>
</div>
<div>
<p>Chifman, Julia, Ashok Pullikuth, Jeff W Chou, Davide Bedognetti, and Lance D Miller. 2016. “Conservation of immune gene signatures in solid tumors and prognostic implications.” <em>BMC Cancer</em> 16 (1): 911. doi:<a href="https://doi.org/10.1186/s12885-016-2948-z">10.1186/s12885-016-2948-z</a>.</p>
</div>
<div>
<p>Chung, Woosung, Hye Hyeon Eum, Hae-Ock Lee, Kyung-Min Lee, Han-Byoel Lee, Kyu-Tae Kim, Han Suk Ryu, et al. 2017. “Single-cell RNA-seq enables comprehensive tumour and immune cell profiling in primary breast cancer.” <em>Nat. Commun.</em> 8. Nature Publishing Group: 15081. doi:<a href="https://doi.org/10.1038/ncomms15081">10.1038/ncomms15081</a>.</p>
</div>
<div>
<p>Cieślik, Marcin, and Arul M. Chinnaiyan. 2017. “Cancer transcriptome profiling at the juncture of clinical translation.” <em>Nat. Rev. Genet.</em> 19 (2). Nature Publishing Group: 93–109. doi:<a href="https://doi.org/10.1038/nrg.2017.96">10.1038/nrg.2017.96</a>.</p>
</div>
<div>
<p>Clarke, J., P. Seo, and B. Clarke. 2010. “Statistical expression deconvolution from mixed tissue samples.” <em>Bioinformatics</em> 26 (8). Oxford University Press: 1043–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btq097">10.1093/bioinformatics/btq097</a>.</p>
</div>
<div>
<p>Core, Cedar-Sinals Genomics. 2016. “Single Cell Genomics.” <a href="https://www.cedars-sinai.edu/Research/Research-Cores/Genomics-Core/Documents/website_pricing_jan_2018_final.pdf" class="uri">https://www.cedars-sinai.edu/Research/Research-Cores/Genomics-Core/Documents/website_pricing_jan_2018_final.pdf</a>.</p>
</div>
<div>
<p>Corlan, Alexandru Dan. 2004. “Medline Trend: Automated Yearly Statistics of Pubmed Results for Any Query.” <a href="http://dan.corlan.net/medline-trend.html" class="uri">http://dan.corlan.net/medline-trend.html</a>.</p>
</div>
<div>
<p>Costa, Ana, Yann Kieffer, Alix Scholer-Dahirel, Floriane Pelon, Brigitte Bourachot, Melissa Cardon, Philemon Sirven, et al. 2018. “Fibroblast Heterogeneity and Immunosuppressive Environment in Human Breast Cancer.” <em>Cancer Cell</em> 33 (3). Elsevier: 463–479.e10. doi:<a href="https://doi.org/10.1016/j.ccell.2018.01.011">10.1016/j.ccell.2018.01.011</a>.</p>
</div>
<div>
<p>Csermely, Peter., Korado. Korlevic, and Katalin. Sulyok. 2007. <em>Science education : models and networking of student research training under 21</em>. IOS Press.</p>
</div>
<div>
<p>Czerwinska, Urszula, Laura Cantini, Ulykbek Kairov, Emmanuel Barillot, and Andrei Zinovyev. 2018. “Application of Independent Component Analysis to Tumor Transcriptomes Reveals Specific and Reproducible Immune-Related Signals.” In, 501–13. Springer, Cham. doi:<a href="https://doi.org/10.1007/978-3-319-93764-9_46">10.1007/978-3-319-93764-9_46</a>.</p>
</div>
<div>
<p>Davis, Sean. 2016. “Github: Awesome-Single-Cell.” <a href="https://github.com/seandavi/awesome-single-cell" class="uri">https://github.com/seandavi/awesome-single-cell</a>.</p>
</div>
<div>
<p>Ding, C.H.Q., Tao Li, and M.I. Jordan. 2010. “Convex and Semi-Nonnegative Matrix Factorizations.” <em>IEEE Trans. Pattern Anal. Mach. Intell.</em> 32 (1): 45–55. doi:<a href="https://doi.org/10.1109/TPAMI.2008.277">10.1109/TPAMI.2008.277</a>.</p>
</div>
<div>
<p>Drucker, Harris, Christopher J C Burges, Linda Kaufman, Alex J Smola, and Vladimir Vapnik. 1997. “Support vector regression machines.” In <em>Adv. Neural Inf. Process. Syst.</em>, 155–61.</p>
</div>
<div>
<p>Dumont, Nancy, Bob Liu, Rosa Anna DeFilippis, Hang Chang, Joseph T. Rabban, Anthony N. Karnezis, Judy A. Tjoe, James Marx, Bahram Parvin, and Thea D. Tlsty. 2013. “Breast Fibroblasts Modulate Early Dissemination, Tumorigenesis, and Metastasis through Alteration of Extracellular Matrix Characteristics.” <em>Neoplasia</em> 15 (3): 249–IN7. doi:<a href="https://doi.org/10.1593/neo.121950">10.1593/neo.121950</a>.</p>
</div>
<div>
<p>Dunn, Gavin P., Allen T. Bruce, Hiroaki Ikeda, Lloyd J. Old, and Robert D. Schreiber. 2002. “Cancer immunoediting: from immunosurveillance to tumor escape.” <em>Nat. Immunol.</em> 3 (11). Nature Publishing Group: 991–98. doi:<a href="https://doi.org/10.1038/ni1102-991">10.1038/ni1102-991</a>.</p>
</div>
<div>
<p>Ediorial Cell Systems. 2017. “What Is Your Conceptual Definition of ‘Cell Type’ in the Context of a Mature Organism?” <em>Cell Syst.</em> 4 (3). Elsevier: 255–59. doi:<a href="https://doi.org/10.1016/j.cels.2017.03.006">10.1016/j.cels.2017.03.006</a>.</p>
</div>
<div>
<p>Ehrlich, P. 1909. “Über den jetzigen Stand der Chemotherapie.” <em>Berichte Der Dtsch. Chem. Gesellschaft</em> 42 (1). Wiley-Blackwell: 17–47. doi:<a href="https://doi.org/10.1002/cber.19090420105">10.1002/cber.19090420105</a>.</p>
</div>
<div>
<p>Elmas, Abdulkadir, Tai-Hsien Ou Yang, Xiaodong Wang, and Dimitris Anastassiou. 2016. “Discovering Genome-Wide Tag SNPs Based on the Mutual Information of the Variants.” Edited by Srinivas Mummidi. <em>PLoS One</em> 11 (12). Public Library of Science: e0167994. doi:<a href="https://doi.org/10.1371/journal.pone.0167994">10.1371/journal.pone.0167994</a>.</p>
</div>
<div>
<p>Engreitz, Jesse M, Bernie J Daigle, Jonathan J Marshall, Russ B Altman, and Russ B. Altman. 2010. “Independent component analysis: mining microarray data for fundamental human gene expression modules.” <em>J. Biomed. Inform.</em> 43 (6). NIH Public Access: 932–44. doi:<a href="https://doi.org/10.1016/j.jbi.2010.07.001">10.1016/j.jbi.2010.07.001</a>.</p>
</div>
<div>
<p>Erickson, Jeff. 2018. “Jeff Erickson’s Algorithms, Etc.” Accessed June 5. <a href="http://jeffe.cs.illinois.edu/teaching/algorithms/" class="uri">http://jeffe.cs.illinois.edu/teaching/algorithms/</a>.</p>
</div>
<div>
<p>Erkkilä, Timo, Saara Lehmusvaara, Pekka Ruusuvuori, Tapio Visakorpi, Ilya Shmulevich, and Harri Lähdesmäki. 2010. “Probabilistic analysis of gene expression measurements from heterogeneous tissues.” <em>Bioinformatics</em> 26 (20). Oxford University Press: 2571–7. doi:<a href="https://doi.org/10.1093/bioinformatics/btq406">10.1093/bioinformatics/btq406</a>.</p>
</div>
<div>
<p>Fa-Yu Wang, Chong-Yung Chi, Tsung-Han Chan, and Yue Wang. 2010. “Nonnegative Least-Correlated Component Analysis for Separation of Dependent Sources by Volume Maximization.” <em>IEEE Trans. Pattern Anal. Mach. Intell.</em> 32 (5): 875–88. doi:<a href="https://doi.org/10.1109/TPAMI.2009.72">10.1109/TPAMI.2009.72</a>.</p>
</div>
<div>
<p><em>Facilitating Interdisciplinary Research</em>. 2004. Washington, D.C.: National Academies Press. doi:<a href="https://doi.org/10.17226/11153">10.17226/11153</a>.</p>
</div>
<div>
<p>Feinberg, Andrew P., and Bert Vogelstein. 1983. “Hypomethylation distinguishes genes of some human cancers from their normal counterparts.” <em>Nature</em> 301 (5895). Nature Publishing Group: 89–92. doi:<a href="https://doi.org/10.1038/301089a0">10.1038/301089a0</a>.</p>
</div>
<div>
<p>Finotello, Francesca, Clemens Mayer, Christina Plattner, Gerhard Laschober, Dietmar Rieder, Hubert Hackl, Anne Krogsdam, et al. 2017. “quanTIseq: quantifying immune contexture of human tumors.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 223180. doi:<a href="https://doi.org/10.1101/223180">10.1101/223180</a>.</p>
</div>
<div>
<p>Folkman, J, E Merler, C Abernathy, and G Williams. 1971. “Isolation of a tumor factor responsible for angiogenesis.” <em>J. Exp. Med.</em> 133 (2). Rockefeller University Press: 275–88. doi:<a href="https://doi.org/10.1084/JEM.133.2.275">10.1084/JEM.133.2.275</a>.</p>
</div>
<div>
<p>Frey, Daniel M., Raoul A. Droeser, Carsten T. Viehl, Inti Zlobec, Alessandro Lugli, Urs Zingg, Daniel Oertli, Christoph Kettelhack, Luigi Terracciano, and Luigi Tornillo. 2010. “High frequency of tumor-infiltrating FOXP3+ regulatory T cells predicts improved survival in mismatch repair-proficient colorectal cancer patients.” <em>Int. J. Cancer</em> 126 (11): 2635–43. doi:<a href="https://doi.org/10.1002/ijc.24989">10.1002/ijc.24989</a>.</p>
</div>
<div>
<p>Fu, Yi, Guoqiang Yu, Douglas A. Levine, Niya Wang, Ie-Ming Shih, Zhen Zhang, Robert Clarke, and Yue Wang. 2015. “BACOM2.0 facilitates absolute normalization and quantification of somatic copy number alterations in heterogeneous tumor.” <em>Sci. Rep.</em> 5 (1). Nature Publishing Group: 13955. doi:<a href="https://doi.org/10.1038/srep13955">10.1038/srep13955</a>.</p>
</div>
<div>
<p>Gabrilovich, Dmitry I., Suzanne Ostrand-Rosenberg, and Vincenzo Bronte. 2012. “Coordinated regulation of myeloid cells by tumours.” doi:<a href="https://doi.org/10.1038/nri3175">10.1038/nri3175</a>.</p>
</div>
<div>
<p>Gajewski, Thomas F., Yuru Meng, Christian Blank, Ian Brown, Aalok Kacha, Justin Kline, and Helena Harlin. 2006. “Immune resistance orchestrated by the tumor microenvironment.” <em>Immunol. Rev.</em> 213 (1). Blackwell Publishing Ltd: 131–45. doi:<a href="https://doi.org/10.1111/J.1600-065X.2006.00442.X">10.1111/J.1600-065X.2006.00442.X</a>.</p>
</div>
<div>
<p>Galon, Jérôme, Bernhard Mlecnik, Gabriela Bindea, Helen K Angell, Anne Berger, Christine Lagorce, Alessandro Lugli, et al. 2014. “Towards the introduction of the ‘Immunoscore’ in the classification of malignant tumours.” <em>J. Pathol.</em> 232 (2): 199–209. doi:<a href="https://doi.org/10.1002/path.4287">10.1002/path.4287</a>.</p>
</div>
<div>
<p>Galon, Jérôme, Franck Pagès, Francesco M Marincola, Magdalena Thurin, Giorgio Trinchieri, Bernard A Fox, Thomas F Gajewski, and Paolo A Ascierto. 2012. “The immune score as a new possible approach for the classification of cancer.” <em>J. Transl. Med.</em> 10 (1). BioMed Central: 1. doi:<a href="https://doi.org/10.1186/1479-5876-10-1">10.1186/1479-5876-10-1</a>.</p>
</div>
<div>
<p>Gaujoux, R., and C. Seoighe. 2013. “CellMix: a comprehensive toolbox for gene expression deconvolution.” <em>Bioinformatics</em> 29 (17): 2211–2. doi:<a href="https://doi.org/10.1093/bioinformatics/btt351">10.1093/bioinformatics/btt351</a>.</p>
</div>
<div>
<p>Gaujoux, Renaud, and Cathal Seoighe. 2010. “A flexible R package for nonnegative matrix factorization.” <em>BMC Bioinformatics</em> 11 (1). BioMed Central: 367. doi:<a href="https://doi.org/10.1186/1471-2105-11-367">10.1186/1471-2105-11-367</a>.</p>
</div>
<div>
<p>———. 2012. “Semi-supervised Nonnegative Matrix Factorization for gene expression deconvolution: A case study.” <em>Infect. Genet. Evol.</em> 12 (5). Elsevier: 913–21. doi:<a href="https://doi.org/10.1016/J.MEEGID.2011.08.014">10.1016/J.MEEGID.2011.08.014</a>.</p>
</div>
<div>
<p>Ghosh, D. 2004. “Mixture models for assessing differential expression in complex tissues using microarray data.” <em>Bioinformatics</em> 20 (11). Oxford University Press: 1663–9. doi:<a href="https://doi.org/10.1093/bioinformatics/bth139">10.1093/bioinformatics/bth139</a>.</p>
</div>
<div>
<p>Giesen, Charlotte, Hao A O Wang, Denis Schapiro, Nevena Zivanovic, Andrea Jacobs, Bodo Hattendorf, Peter J Schüffler, et al. 2014. “Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry.” <em>Nat. Methods</em> 11 (4). Nature Publishing Group: 417–22. doi:<a href="https://doi.org/10.1038/nmeth.2869">10.1038/nmeth.2869</a>.</p>
</div>
<div>
<p>Gnjatic, Sacha, Vincenzo Bronte, Laura Rosa Brunet, Marcus O Butler, Mary L Disis, Jérôme Galon, Leif G Hakansson, et al. 2017. “Identifying baseline immune-related biomarkers to predict clinical outcome of immunotherapy.” <em>J. Immunother. Cancer</em> 5. BioMed Central: 44. doi:<a href="https://doi.org/10.1186/s40425-017-0243-4">10.1186/s40425-017-0243-4</a>.</p>
</div>
<div>
<p>Godec, Jernej, Yan Tan, Arthur Liberzon, Pablo Tamayo, Sanchita Bhattacharya, Atul J. Butte, Jill P. Mesirov, and W. Nicholas Haining. 2016. “Compendium of Immune Signatures Identifies Conserved and Species-Specific Biology in Response to Inflammation.” <em>Immunity</em> 44 (1): 194–206. doi:<a href="https://doi.org/10.1016/j.immuni.2015.12.006">10.1016/j.immuni.2015.12.006</a>.</p>
</div>
<div>
<p>Golub, T R, D K Slonim, P Tamayo, C Huard, M Gaasenbeek, J P Mesirov, H Coller, et al. 1999. “Molecular classification of cancer: class discovery and class prediction by gene expression monitoring.” <em>Science</em> 286 (5439). American Association for the Advancement of Science: 531–7. doi:<a href="https://doi.org/10.1126/SCIENCE.286.5439.531">10.1126/SCIENCE.286.5439.531</a>.</p>
</div>
<div>
<p>Gong, Ting, Nicole Hartmann, Isaac S. Kohane, Volker Brinkmann, Frank Staedtler, Martin Letzkus, Sandrine Bongiovanni, and Joseph D. Szustakowski. 2011. “Optimal Deconvolution of Transcriptional Profiling Data Using Quadratic Programming with Application to Complex Clinical Blood Samples.” Edited by Magnus Rattray. <em>PLoS One</em> 6 (11). Public Library of Science: e27156. doi:<a href="https://doi.org/10.1371/journal.pone.0027156">10.1371/journal.pone.0027156</a>.</p>
</div>
<div>
<p>Gorban, A. N. (Aleksandr Nikolaevich). 2007. <em>Principal manifolds for data visualization and dimension reduction</em>. Springer.</p>
</div>
<div>
<p>Gosink, Mark M., Howard T. Petrie, and Nicholas F. Tsinoremas. 2007. “Electronically subtracting expression patterns from a mixed cell population.” <em>Bioinformatics</em> 23 (24). Oxford University Press: 3328–34. doi:<a href="https://doi.org/10.1093/bioinformatics/btm508">10.1093/bioinformatics/btm508</a>.</p>
</div>
<div>
<p>Görtler, Franziska, Stefan Solbrig, Tilo Wettig, Peter J. Oefner, Rainer Spang, and Michael Altenbuchinger. 2018. “Loss-function learning for digital tissue deconvolution.” <a href="http://arxiv.org/abs/1801.08447" class="uri">http://arxiv.org/abs/1801.08447</a>.</p>
</div>
<div>
<p>Greenblatt, M, and P Shubi. 1968. “Tumor angiogenesis: transfilter diffusion studies in the hamster by the transparent chamber technique.” <em>J. Natl. Cancer Inst.</em> 41 (1): 111–24. <a href="http://www.ncbi.nlm.nih.gov/pubmed/5662020" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/5662020</a>.</p>
</div>
<div>
<p>Greger, Valerie, Eberhard Passarge, Wolfgang H�pping, Elmar Messmer, and Bernhard Horsthemke. 1989. “Epigenetic changes may contribute to the formation and spontaneous regression of retinoblastoma.” <em>Hum. Genet.</em> 83 (2). Springer-Verlag: 155–58. doi:<a href="https://doi.org/10.1007/BF00286709">10.1007/BF00286709</a>.</p>
</div>
<div>
<p>Győrffy, Balázs, Christos Hatzis, Tara Sanft, Erin Hofstatter, Bilge Aktas, and Lajos Pusztai. 2015. “Multigene prognostic tests in breast cancer: past, present, future.” <em>Breast Cancer Res.</em> 17 (1). BioMed Central: 11. doi:<a href="https://doi.org/10.1186/s13058-015-0514-2">10.1186/s13058-015-0514-2</a>.</p>
</div>
<div>
<p>Hanahan, D, and R A Weinberg. 2000. “The hallmarks of cancer.” <em>Cell</em> 100 (1). Elsevier: 57–70. doi:<a href="https://doi.org/10.1016/S0092-8674(00)81683-9">10.1016/S0092-8674(00)81683-9</a>.</p>
</div>
<div>
<p>Hanahan, Douglas, and Lisa M. Coussens. 2012. “Accessories to the Crime: Functions of Cells Recruited to the Tumor Microenvironment.” <em>Cancer Cell</em> 21 (3): 309–22. doi:<a href="https://doi.org/10.1016/j.ccr.2012.02.022">10.1016/j.ccr.2012.02.022</a>.</p>
</div>
<div>
<p>Harris, Lyndsay N., Nofisat Ismaila, Lisa M. McShane, Fabrice Andre, Deborah E. Collyar, Ana M. Gonzalez-Angulo, Elizabeth H. Hammond, et al. 2016. “Use of Biomarkers to Guide Decisions on Adjuvant Systemic Therapy for Women With Early-Stage Invasive Breast Cancer: American Society of Clinical Oncology Clinical Practice Guideline.” <em>J. Clin. Oncol.</em> 34 (10): 1134–50. doi:<a href="https://doi.org/10.1200/JCO.2015.65.2289">10.1200/JCO.2015.65.2289</a>.</p>
</div>
<div>
<p>Hastie, Trevor, Robert Tibshirani, and Jerome Friedman. 2009. <em>The Elements of Statistical Learning</em>. Springer Series in Statistics. New York, NY: Springer New York. doi:<a href="https://doi.org/10.1007/978-0-387-84858-7">10.1007/978-0-387-84858-7</a>.</p>
</div>
<div>
<p>Hänzelmann, Sonja, Robert Castelo, and Justin Guinney. 2013. “GSVA: gene set variation analysis for microarray and RNA-Seq data.” <em>BMC Bioinformatics</em> 14 (1). BioMed Central: 7. doi:<a href="https://doi.org/10.1186/1471-2105-14-7">10.1186/1471-2105-14-7</a>.</p>
</div>
<div>
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p>
</div>
<div>
<p>———. 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p>
</div>
<div>
<p>———. 2016b. “Pembrolizumab (Keytruda) Checkpoint Inhibitor.” <a href="https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm" class="uri">https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm</a>.</p>
</div>
<div>
<p>———. 2017a. “FDA Approval Brings First Gene Therapy to the United States.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm</a>.</p>
</div>
<div>
<p>———. 2017b. “FDA Approves Car-T Cell Therapy to Treat Adults with Certain Types of Large B-Cell Lymphoma.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm581216.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm581216.htm</a>.</p>
</div>
<div>
<p>Heng, Tracy S P, Michio W Painter, Kutlu Elpek, Veronika Lukacs-Kornek, Nora Mauermann, Shannon J Turley, Daphne Koller, et al. 2008. “The Immunological Genome Project: networks of gene expression in immune cells.” <em>Nat. Immunol.</em> 9 (10): 1091–4. doi:<a href="https://doi.org/10.1038/ni1008-1091">10.1038/ni1008-1091</a>.</p>
</div>
<div>
<p>Herault, J., and C. Jutten. 1986. “Space or time adaptive signal processing by neural network models.” In <em>AIP Conf. Proc.</em>, 151:206–11. 1. AIP. doi:<a href="https://doi.org/10.1063/1.36258">10.1063/1.36258</a>.</p>
</div>
<div>
<p>Herbst, Roy S., Jean-Charles Soria, Marcin Kowanetz, Gregg D. Fine, Omid Hamid, Michael S. Gordon, Jeffery A. Sosman, et al. 2014. “Predictive correlates of response to the anti-PD-L1 antibody MPDL3280A in cancer patients.” <em>Nature</em> 515 (7528): 563–67. doi:<a href="https://doi.org/10.1038/nature14011">10.1038/nature14011</a>.</p>
</div>
<div>
<p>Himberg, J., and A. Hyvarinen. n.d. “Icasso: software for investigating the reliability of ICA estimates by clustering and visualization.” In <em>2003 Ieee Xiii Work. Neural Networks Signal Process. (Ieee Cat. No.03TH8718)</em>, 259–68. IEEE. doi:<a href="https://doi.org/10.1109/NNSP.2003.1318025">10.1109/NNSP.2003.1318025</a>.</p>
</div>
<div>
<p>Hoffmann, Martin, Dirk Pohlers, Dirk Koczan, Hans-Jürgen Thiesen, Stefan Wölfl, and Raimund W Kinne. 2006. “Robust computational reconstitution – a new method for the comparative analysis of gene expression in tissues and isolated cell fractions.” <em>BMC Bioinformatics</em> 7 (1). BioMed Central: 369. doi:<a href="https://doi.org/10.1186/1471-2105-7-369">10.1186/1471-2105-7-369</a>.</p>
</div>
<div>
<p>Houseman, E. Andres, Molly L. Kile, David C. Christiani, Tan A. Ince, Karl T. Kelsey, and Carmen J. Marsit. 2016. “Reference-free deconvolution of DNA methylation data and mediation by cell composition effects.” <em>BMC Bioinformatics</em> 17 (1). BioMed Central: 259. doi:<a href="https://doi.org/10.1186/s12859-016-1140-4">10.1186/s12859-016-1140-4</a>.</p>
</div>
<div>
<p>Houseman, Eugene Andres, John Molitor, and Carmen J. Marsit. 2014. “Reference-free cell mixture adjustments in analysis of DNA methylation data.” <em>Bioinformatics</em> 30 (10). Oxford University Press: 1431–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btu029">10.1093/bioinformatics/btu029</a>.</p>
</div>
<div>
<p>Houseman, Eugene, William P Accomando, Devin C Koestler, Brock C Christensen, Carmen J Marsit, Heather H Nelson, John K Wiencke, and Karl T Kelsey. 2012. “DNA methylation arrays as surrogate measures of cell mixture distribution.” <em>BMC Bioinformatics</em> 13 (1). BioMed Central: 86. doi:<a href="https://doi.org/10.1186/1471-2105-13-86">10.1186/1471-2105-13-86</a>.</p>
</div>
<div>
<p>Hyvärinen, Aapo, and Erkki Oja. 2000. “Independent Component Analysis: Algorithms and Applications.” <em>Neural Networks</em> 13 (45): 411–30. <a href="https://www.cs.helsinki.fi/u/ahyvarin/papers/NN00new.pdf" class="uri">https://www.cs.helsinki.fi/u/ahyvarin/papers/NN00new.pdf</a>.</p>
</div>
<div>
<p>Institute, National Cancer. 2017. “Types of Cancer Treatment.” <a href="https://www.cancer.gov/about-cancer/treatment/types" class="uri">https://www.cancer.gov/about-cancer/treatment/types</a>.</p>
</div>
<div>
<p>Itadani, Hiraku, Shinji Mizuarai, and Hidehito Kotani. 2008. “Can Systems Biology Understand Pathway Activation? Gene Expression Signatures as Surrogate Markers for Understanding the Complexity of Pathway Activation.” <em>Curr. Genomics</em> 9 (5): 349–60. doi:<a href="https://doi.org/10.2174/138920208785133235">10.2174/138920208785133235</a>.</p>
</div>
<div>
<p>Jackson, Hollie J., Sarwish Rafiq, and Renier J. Brentjens. 2016. “Driving CAR T-cells forward.” doi:<a href="https://doi.org/10.1038/nrclinonc.2016.36">10.1038/nrclinonc.2016.36</a>.</p>
</div>
<div>
<p>Jeschke, Jana, Martin Bizet, Christine Desmedt, Emilie Calonne, Sarah Dedeurwaerder, Soizic Garaud, Alexander Koch, et al. 2017. “DNA methylation–based immune response signature improves patient diagnosis in multiple cancers.” <em>J. Clin. Invest.</em> 127 (8): 3090–3102. doi:<a href="https://doi.org/10.1172/JCI91095">10.1172/JCI91095</a>.</p>
</div>
<div>
<p>Jiang, Yuchao, Yu Qiu, Andy J Minn, and Nancy R Zhang. 2016. “Assessing intratumor heterogeneity and tracking longitudinal and spatial clonal evolutionary history by next-generation sequencing.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 113 (37). National Academy of Sciences: E5528–37. doi:<a href="https://doi.org/10.1073/pnas.1522203113">10.1073/pnas.1522203113</a>.</p>
</div>
<div>
<p>Ju, Wenjun, Casey S Greene, Felix Eichinger, Viji Nair, Jeffrey B Hodgin, Markus Bitzer, Young-Suk Lee, et al. 2013. “Defining cell-type specificity at the transcriptional level in human disease.” <em>Genome Res.</em> 23 (11). Cold Spring Harbor Laboratory Press: 1862–73. doi:<a href="https://doi.org/10.1101/gr.155697.113">10.1101/gr.155697.113</a>.</p>
</div>
<div>
<p>Kairov, Ulykbek, Laura Cantini, Alessandro Greco, Askhat Molkenov, Urszula Czerwinska, Emmanuel Barillot, and Andrei Zinovyev. 2017. “Determining the optimal number of independent components for reproducible transcriptomic data analysis.” <em>BMC Genomics</em> 18 (1). BioMed Central: 712. doi:<a href="https://doi.org/10.1186/s12864-017-4112-9">10.1186/s12864-017-4112-9</a>.</p>
</div>
<div>
<p>Kornstein, M J, J S Brooks, and D E Elder. 1983. “Immunoperoxidase localization of lymphocyte subsets in the host response to melanoma and nevi.” <em>Cancer Res.</em> 43 (6): 2749–53. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6342758" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/6342758</a>.</p>
</div>
<div>
<p>Kuhn, Alexandre, Doris Thu, Henry J Waldvogel, Richard L M Faull, and Ruth Luthi-Carter. 2011. “Population-specific expression analysis (PSEA) reveals molecular changes in diseased brain.” <em>Nat. Methods</em> 8 (11). Nature Publishing Group: 945–47. doi:<a href="https://doi.org/10.1038/nmeth.1710">10.1038/nmeth.1710</a>.</p>
</div>
<div>
<p>Kuperstein, I, E Bonnet, H-A Nguyen, D Cohen, E Viara, L Grieco, S Fourquet, et al. 2015. “Atlas of Cancer Signalling Network: a systems biology resource for integrative analysis of cancer data with Google Maps.” <em>Oncogenesis</em> 4 (7): e160–e160. doi:<a href="https://doi.org/10.1038/oncsis.2015.19">10.1038/oncsis.2015.19</a>.</p>
</div>
<div>
<p>Kuperstein, Inna, David PA Cohen, Stuart Pook, Eric Viara, Laurence Calzone, Emmanuel Barillot, and Andrei Zinovyev. 2013. “NaviCell: a web-based environment for navigation, curation and maintenance of large molecular interaction maps.” <em>BMC Syst. Biol.</em> 7 (1). BioMed Central: 100. doi:<a href="https://doi.org/10.1186/1752-0509-7-100">10.1186/1752-0509-7-100</a>.</p>
</div>
<div>
<p>Larson, Nicholas B, and Brooke L Fridley. 2013. “PurBayes: estimating tumor cellularity and subclonality in next-generation sequencing data.” <em>Bioinformatics</em> 29 (15). Oxford University Press: 1888–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btt293">10.1093/bioinformatics/btt293</a>.</p>
</div>
<div>
<p>Lavin, Yonit, Soma Kobayashi, Andrew Leader, El-ad David Amir, Naama Elefant, Camille Bigenwald, Romain Remark, et al. 2017. “Innate Immune Landscape in Early Lung Adenocarcinoma by Paired Single-Cell Analyses.” <em>Cell</em> 169 (4). Cell Press: 750–765.e17. doi:<a href="https://doi.org/10.1016/J.CELL.2017.04.014">10.1016/J.CELL.2017.04.014</a>.</p>
</div>
<div>
<p>Lähdesmäki, Harri, llya Shmulevich, Valerie Dunmire, Olli Yli-Harja, and Wei Zhang. 2005. “In silico microdissection of microarray data from heterogeneous cell populations.” <em>BMC Bioinformatics</em> 6 (1). BioMed Central: 54. doi:<a href="https://doi.org/10.1186/1471-2105-6-54">10.1186/1471-2105-6-54</a>.</p>
</div>
<div>
<p>Ledford, Heidi. 2015. “How to solve the world’s biggest problems.” <em>Nature</em> 525 (7569): 308–11. doi:<a href="https://doi.org/10.1038/525308a">10.1038/525308a</a>.</p>
</div>
<div>
<p>Lee, Daniel D., and H. Sebastian Seung. 2000. “Algorithms for non-negative matrix factorization.” MIT Press. <a href="https://dl.acm.org/citation.cfm?id=3008829" class="uri">https://dl.acm.org/citation.cfm?id=3008829</a>.</p>
</div>
<div>
<p>Leek, Jeffrey T., and John D. Storey. 2007. “Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis.” <em>PLoS Genet.</em> 3 (9): e161. doi:<a href="https://doi.org/10.1371/journal.pgen.0030161">10.1371/journal.pgen.0030161</a>.</p>
</div>
<div>
<p>Leung, D., G Cachianes, W. Kuang, D. Goeddel, and N Ferrara. 1989. “Vascular endothelial growth factor is a secreted angiogenic mitogen.” <em>Science (80-. ).</em> 246 (4935): 1306–9. doi:<a href="https://doi.org/10.1126/science.2479986">10.1126/science.2479986</a>.</p>
</div>
<div>
<p>Li, Bo, and Jun Z Li. 2014. “A general framework for analyzing tumor subclonality using SNP array and DNA sequencing data.” <em>Genome Biol.</em> 15 (9). BioMed Central: 473. doi:<a href="https://doi.org/10.1186/s13059-014-0473-4">10.1186/s13059-014-0473-4</a>.</p>
</div>
<div>
<p>Li, Bo, Jun S. Liu, and X. Shirley Liu. 2017. “Revisit linear regression-based deconvolution methods for tumor gene expression data.” <em>Genome Biol.</em> 18 (1). BioMed Central: 127. doi:<a href="https://doi.org/10.1186/s13059-017-1256-5">10.1186/s13059-017-1256-5</a>.</p>
</div>
<div>
<p>Li, Bo, Eric Severson, Jean-Christophe Pignon, Haoquan Zhao, Taiwen Li, Jesse Novak, Peng Jiang, et al. 2016. “Comprehensive analyses of tumor immunity: implications for cancer immunotherapy.” <em>Genome Biol.</em> 17 (1). BioMed Central: 174. doi:<a href="https://doi.org/10.1186/s13059-016-1028-7">10.1186/s13059-016-1028-7</a>.</p>
</div>
<div>
<p>Li, Yi, and Xiaohui Xie. 2013. “A mixture model for expression deconvolution from RNA-seq in heterogeneous tissues.” <em>BMC Bioinforma. 2013 145</em> 14 (5). BioMed Central: S11. doi:<a href="https://doi.org/10.1186/1471-2105-14-s5-s11">10.1186/1471-2105-14-s5-s11</a>.</p>
</div>
<div>
<p>Liebner, David A., Kun Huang, and Jeffrey D. Parvin. 2014. “MMAD: microarray microdissection with analysis of differences is a computational tool for deconvoluting cell type-specific contributions from tissue samples.” <em>Bioinformatics</em> 30 (5). Oxford University Press: 682–89. doi:<a href="https://doi.org/10.1093/bioinformatics/btt566">10.1093/bioinformatics/btt566</a>.</p>
</div>
<div>
<p>Lin, Chih-Jen. 2007. “Projected Gradient Methods for Nonnegative Matrix Factorization.” <em>Neural Comput.</em> 19 (10). MIT Press 238 Main St., Suite 500, Cambridge, MA 02142-1046 USA journals-info@mit.edu: 2756–79. doi:<a href="https://doi.org/10.1162/neco.2007.19.10.2756">10.1162/neco.2007.19.10.2756</a>.</p>
</div>
<div>
<p>Liu, Jiangang, Andrew Campen, Shuguang Huang, Sheng-Bin Peng, Xiang Ye, Mathew Palakal, A Keith Dunker, Yuni Xia, and Shuyu Li. 2008. “Identification of a gene signature in cell cycle pathway for breast cancer prognosis using gene expression profiling data.” <em>BMC Med. Genomics</em> 1 (1): 39. doi:<a href="https://doi.org/10.1186/1755-8794-1-39">10.1186/1755-8794-1-39</a>.</p>
</div>
<div>
<p>Liu, Yuan, Yu Liang, Qifan Kuang, Fanfan Xie, Yingyi Hao, Zhining Wen, and Menglong Li. 2017. “Post-modified non-negative matrix factorization for deconvoluting the gene expression profiles of specific cell types from heterogeneous clinical samples based on RNA-sequencing data.” <em>J. Chemom.</em> Wiley-Blackwell, e2929. doi:<a href="https://doi.org/10.1002/cem.2929">10.1002/cem.2929</a>.</p>
</div>
<div>
<p>Lu, Peng, Aleksey Nakorchevskiy, and Edward M Marcotte. 2003. “Expression deconvolution: a reinterpretation of DNA microarray data reveals dynamic changes in cell populations.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 100 (18). National Academy of Sciences: 10370–5. doi:<a href="https://doi.org/10.1073/pnas.1832361100">10.1073/pnas.1832361100</a>.</p>
</div>
<div>
<p>Lutsik, Pavlo, Martin Slawski, Gilles Gasparoni, Nikita Vedeneev, Matthias Hein, and Jörn Walter. 2017. “MeDeCom: discovery and quantification of latent components of heterogeneous methylomes.” <em>Genome Biol.</em> 18 (1). BioMed Central: 55. doi:<a href="https://doi.org/10.1186/s13059-017-1182-6">10.1186/s13059-017-1182-6</a>.</p>
</div>
<div>
<p>Lyons, Yasmin A, Sherry Y Wu, Willem W Overwijk, Keith A Baggerly, and Anil K Sood. 2017. “Immune cell profiling in cancer: molecular approaches to cell- specific identification.” <em>Npj Precis. Oncol.</em> 1. doi:<a href="https://doi.org/10.1038/s41698-017-0031-0">10.1038/s41698-017-0031-0</a>.</p>
</div>
<div>
<p>Maksimovic, Jovana, Johann A. Gagnon-Bartsch, Terence P. Speed, and Alicia Oshlack. 2015. “Removing unwanted variation in a differential methylation analysis of Illumina HumanMethylation450 array data.” <em>Nucleic Acids Res.</em> 43 (16). Oxford University Press: e106–e106. doi:<a href="https://doi.org/10.1093/nar/gkv526">10.1093/nar/gkv526</a>.</p>
</div>
<div>
<p>Marquez-Medina, Diego, Joel Salla-Fortuny, and Antonieta Salud-Salvia. 2012. “Role of gamma-delta T-cells in cancer. Another opening door to immunotherapy.” <em>Clin. Transl. Oncol.</em> 14 (12): 891–95. doi:<a href="https://doi.org/10.1007/s12094-012-0935-7">10.1007/s12094-012-0935-7</a>.</p>
</div>
<div>
<p>McCarroll, Steven A, and David M Altshuler. 2007. “Copy-number variation and association studies of human disease.” <em>Nat. Genet.</em> 39 (7s): S37–S42. doi:<a href="https://doi.org/10.1038/ng2080">10.1038/ng2080</a>.</p>
</div>
<div>
<p>Mcinnes, Leland, and John Healy. 2018. “UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction.” <a href="https://arxiv.org/pdf/1802.03426.pdf" class="uri">https://arxiv.org/pdf/1802.03426.pdf</a>.</p>
</div>
<div>
<p>Michiels, S., N. Ternès, and F. Rotolo. 2016. “Statistical controversies in clinical research: prognostic gene signatures are not (yet) useful in clinical practice.” <em>Ann. Oncol.</em> 27 (12): 2160–7. doi:<a href="https://doi.org/10.1093/annonc/mdw307">10.1093/annonc/mdw307</a>.</p>
</div>
<div>
<p>Miller, Christopher A., Brian S. White, Nathan D. Dees, Malachi Griffith, John S. Welch, Obi L. Griffith, Ravi Vij, et al. 2014. “SciClone: Inferring Clonal Architecture and Tracking the Spatial and Temporal Patterns of Tumor Evolution.” Edited by Niko Beerenwinkel. <em>PLoS Comput. Biol.</em> 10 (8). Public Library of Science: e1003665. doi:<a href="https://doi.org/10.1371/journal.pcbi.1003665">10.1371/journal.pcbi.1003665</a>.</p>
</div>
<div>
<p>Mitchell, Tom M. (Tom Michael). 1997. <em>Machine Learning</em>. McGraw-Hill.</p>
</div>
<div>
<p>Mody, Rajen J., Yi-Mi Wu, Robert J. Lonigro, Xuhong Cao, Sameek Roychowdhury, Pankaj Vats, Kevin M. Frank, et al. 2015. “Integrative Clinical Sequencing in the Management of Refractory or Relapsed Cancer in Youth.” <em>JAMA</em> 314 (9). American Medical Association: 913. doi:<a href="https://doi.org/10.1001/jama.2015.10080">10.1001/jama.2015.10080</a>.</p>
</div>
<div>
<p>Moffitt, Richard A, Raoud Marayati, Elizabeth L Flate, Keith E Volmar, S Gabriela Herrera Loeza, Katherine A Hoadley, Naim U Rashid, et al. 2015. “Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma.” <em>Nat. Genet.</em> 47 (10). Nature Publishing Group: 1168–78. doi:<a href="https://doi.org/10.1038/ng.3398">10.1038/ng.3398</a>.</p>
</div>
<div>
<p>Mohammadi, Shahin, Neta Zuckerman, Andrea Goldsmith, and Ananth Grama. 2017. “A Critical Survey of Deconvolution Methods for Separating Cell Types in Complex Tissues.” <em>Proc. IEEE</em> 105 (2): 340–66. doi:<a href="https://doi.org/10.1109/JPROC.2016.2607121">10.1109/JPROC.2016.2607121</a>.</p>
</div>
<div>
<p>Moncada, Reuben, Florian Wagner, Marta Chiodin, Joseph C. Devlin, Maayan Baron, Cristina H. Hajdu, Diane Simeone, and Itai Yanai. 2018. “Building a tumor atlas: integrating single-cell RNA-Seq data with spatial transcriptomics in pancreatic ductal adenocarcinoma.” <em>bioRxiv</em>, March. Cold Spring Harbor Laboratory, 254375. doi:<a href="https://doi.org/10.1101/254375">10.1101/254375</a>.</p>
</div>
<div>
<p>Naito, Y, K Saito, K Shiiba, A Ohuchi, K Saigenji, H Nagura, and H Ohtani. 1998. “CD8+ T cells infiltrated within cancer cell nests as a prognostic factor in human colorectal cancer.” <em>Cancer Res.</em> 58 (16): 3491–4. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9721846" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/9721846</a>.</p>
</div>
<div>
<p>Nelms, Bradlee D., Levi Waldron, Luis A. Barrera, Andrew W. Weflen, Jeremy A. Goettel, Guoji Guo, Robert K. Montgomery, et al. 2016. “CellMapper: rapid and accurate inference of gene expression in difficult-to-isolate cell types.” <em>Genome Biol.</em> 17 (1). BioMed Central: 201. doi:<a href="https://doi.org/10.1186/s13059-016-1062-5">10.1186/s13059-016-1062-5</a>.</p>
</div>
<div>
<p>Newberg, Lee A., Xiaowei Chen, Chinnappa D. Kodira, and Maria I. Zavodszky. 2018. “Computational de novo discovery of distinguishing genes for biological processes and cell types in complex tissues.” Edited by Paolo Provero. <em>PLoS One</em> 13 (3). Public Library of Science: e0193067. doi:<a href="https://doi.org/10.1371/journal.pone.0193067">10.1371/journal.pone.0193067</a>.</p>
</div>
<div>
<p>Newman, Aaron M, Chih Long Liu, Michael R Green, Andrew J Gentles, Weiguo Feng, Yue Xu, Chuong D Hoang, Maximilian Diehn, and Ash A Alizadeh. 2015. “Robust enumeration of cell subsets from tissue expression profiles.” <em>Nat. Methods</em> 12 (5). Nature Publishing Group: 453–57. doi:<a href="https://doi.org/10.1038/nmeth.3337">10.1038/nmeth.3337</a>.</p>
</div>
<div>
<p>Newman, AM, CL Liu, and AA Alizadeh. 2018. “CIBERSORT- Absolute mode.” Accessed June 4. <a href="https://cibersort.stanford.edu/manual.php" class="uri">https://cibersort.stanford.edu/manual.php</a>.</p>
</div>
<div>
<p>NPR. 2010a. “Science Diction: The Origin of the Word ’Cancer’.” <a href="https://www.npr.org/templates/story/story.php?storyId=130754101" class="uri">https://www.npr.org/templates/story/story.php?storyId=130754101</a>.</p>
</div>
<div>
<p>———. 2010b. “Science Diction: The Origin of the Word ’Cancer’ : NPR.” <a href="https://www.npr.org/templates/story/story.php?storyId=130754101" class="uri">https://www.npr.org/templates/story/story.php?storyId=130754101</a>.</p>
</div>
<div>
<p>Oberg, Jennifer A., Julia L. Glade Bender, Maria Luisa Sulis, Danielle Pendrick, Anthony N. Sireci, Susan J. Hsiao, Andrew T. Turk, et al. 2016. “Implementation of next generation sequencing into pediatric hematology-oncology practice: moving beyond actionable alterations.” <em>Genome Med.</em> 8 (1). BioMed Central: 133. doi:<a href="https://doi.org/10.1186/s13073-016-0389-6">10.1186/s13073-016-0389-6</a>.</p>
</div>
<div>
<p>Oesper, Layla, Ahmad Mahmoody, and Benjamin J Raphael. 2013. “THetA: inferring intra-tumor heterogeneity from high-throughput DNA sequencing data.” <em>Genome Biol.</em> 14 (7). BioMed Central: R80. doi:<a href="https://doi.org/10.1186/gb-2013-14-7-r80">10.1186/gb-2013-14-7-r80</a>.</p>
</div>
<div>
<p>Onuchic, Vitor, Ryan J. Hartmaier, David N. Boone, Michael L. Samuels, Ronak Y. Patel, Wendy M. White, Vesna D. Garovic, et al. 2016. “Epigenomic Deconvolution of Breast Tumors Reveals Metabolic Coupling between Constituent Cell Types.” <em>Cell Rep.</em> 17 (8): 2075–86. doi:<a href="https://doi.org/10.1016/j.celrep.2016.10.057">10.1016/j.celrep.2016.10.057</a>.</p>
</div>
<div>
<p>Ortega-Martorell, Sandra, Paulo J.G. Lisboa, Alfredo Vellido, Margarida Julia-Sape, and Carles Arus. 2012. “Non-negative Matrix Factorisation methods for the spectral decomposition of MRS data from human brain tumours.” <em>BMC Bioinformatics</em> 13 (1). BioMed Central: 38. doi:<a href="https://doi.org/10.1186/1471-2105-13-38">10.1186/1471-2105-13-38</a>.</p>
</div>
<div>
<p>Paatero, Pentti, and Unto Tapper. 1994. “Positive matrix factorization: A non-negative factor model with optimal utilization of error estimates of data values.” <em>Environmetrics</em> 5 (2). Wiley-Blackwell: 111–26. doi:<a href="https://doi.org/10.1002/env.3170050203">10.1002/env.3170050203</a>.</p>
</div>
<div>
<p>Paget, Stephen. 1889. “THE DISTRIBUTION OF SECONDARY GROWTHS IN CANCER OF THE BREAST.” <em>Lancet</em> 133 (3421). Elsevier: 571–73. doi:<a href="https://doi.org/10.1016/S0140-6736(00)49915-0">10.1016/S0140-6736(00)49915-0</a>.</p>
</div>
<div>
<p>Pagès, Franck, Anne Berger, Matthieu Camus, Fatima Sanchez-Cabo, Anne Costes, Robert Molidor, Bernhard Mlecnik, et al. 2005. “Effector Memory T Cells, Early Metastasis, and Survival in Colorectal Cancer.” <em>N. Engl. J. Med.</em> 353 (25): 2654–66. doi:<a href="https://doi.org/10.1056/NEJMoa051424">10.1056/NEJMoa051424</a>.</p>
</div>
<div>
<p>Pagès, Franck, Bernhard Mlecnik, Florence Marliot, Gabriela Bindea, Fang-Shu Ou, Carlo Bifulco, Alessandro Lugli, et al. 2018. “International validation of the consensus Immunoscore for the classification of colon cancer: a prognostic and accuracy study.” <em>Lancet (London, England)</em> 391 (10135). Elsevier: 2128–39. doi:<a href="https://doi.org/10.1016/S0140-6736(18)30789-X">10.1016/S0140-6736(18)30789-X</a>.</p>
</div>
<div>
<p>Palucka, Karolina, and Jacques Banchereau. 2013. “Dendritic-Cell-Based Therapeutic Cancer Vaccines.” doi:<a href="https://doi.org/10.1016/j.immuni.2013.07.004">10.1016/j.immuni.2013.07.004</a>.</p>
</div>
<div>
<p>Papalexi, Efthymia, and Rahul Satija. 2017. “Single-cell RNA sequencing to explore immune cell heterogeneity.” <em>Nat. Rev. Immunol.</em> 18 (1): 35–45. doi:<a href="https://doi.org/10.1038/nri.2017.76">10.1038/nri.2017.76</a>.</p>
</div>
<div>
<p>Parker, Joel S, Michael Mullins, Maggie C U Cheang, Samuel Leung, David Voduc, Tammi Vickery, Sherri Davies, et al. 2009. “Supervised risk predictor of breast cancer based on intrinsic subtypes.” <em>J. Clin. Oncol.</em> 27 (8). American Society of Clinical Oncology: 1160–7. doi:<a href="https://doi.org/10.1200/JCO.2008.18.1370">10.1200/JCO.2008.18.1370</a>.</p>
</div>
<div>
<p>Pascual-Montano, A., J.M. Carazo, K. Kochi, D. Lehmann, and R.D. Pascual-Marqui. 2006. “Nonsmooth nonnegative matrix factorization (nsNMF).” <em>IEEE Trans. Pattern Anal. Mach. Intell.</em> 28 (3): 403–15. doi:<a href="https://doi.org/10.1109/TPAMI.2006.60">10.1109/TPAMI.2006.60</a>.</p>
</div>
<div>
<p>Perkel, Jeffrey M. 2017. “Single-cell sequencing made simple.” <em>Nature</em> 547 (7661): 125–26. doi:<a href="https://doi.org/10.1038/547125a">10.1038/547125a</a>.</p>
</div>
<div>
<p>Piegorsch, Walter W. n.d. <em>Statistical data analytics : foundations for data mining, informatics, and knowledge discovery</em>. <a href="https://www.wiley.com/en-fr/Statistical+Data+Analytics:+Foundations+for+Data+Mining,+Informatics,+and+Knowledge+Discovery-p-9781118619650" class="uri">https://www.wiley.com/en-fr/Statistical+Data+Analytics:+Foundations+for+Data+Mining,+Informatics,+and+Knowledge+Discovery-p-9781118619650</a>.</p>
</div>
<div>
<p>Pitt, Jonathan M, Marie Vétizou, Romain Daillère, María Paula Roberti, Takahiro Yamazaki, Bertrand Routy, Patricia Lepage, et al. 2016. “Resistance Mechanisms to Immune-Checkpoint Blockade in Cancer: Tumor-Intrinsic and -Extrinsic Factors.” <em>Immunity</em> 44 (6). Elsevier: 1255–69. doi:<a href="https://doi.org/10.1016/j.immuni.2016.06.001">10.1016/j.immuni.2016.06.001</a>.</p>
</div>
<div>
<p>Ponting, Chris P. 2017. “Big knowledge from big data in functional genomics.” <em>Emerg. Top. Life Sci.</em> 1 (3). Portland Press Journals portal: 245–48. doi:<a href="https://doi.org/10.1042/ETLS20170129">10.1042/ETLS20170129</a>.</p>
</div>
<div>
<p>Predina, J., E. Eruslanov, B. Judy, V. Kapoor, G. Cheng, L.-C. Wang, J. Sun, et al. 2013. “Changes in the local tumor microenvironment in recurrent cancers may explain the failure of vaccines after surgery.” <em>Proc. Natl. Acad. Sci.</em> 110 (5): E415–E424. doi:<a href="https://doi.org/10.1073/pnas.1211850110">10.1073/pnas.1211850110</a>.</p>
</div>
<div>
<p>Preparata, Franco P., and Michael Ian Shamos. 1985. “Convex Hulls: Basic Algorithms.” In <em>Comput. Geom.</em>, 95–149. New York, NY: Springer New York. doi:<a href="https://doi.org/10.1007/978-1-4612-1098-6_3">10.1007/978-1-4612-1098-6_3</a>.</p>
</div>
<div>
<p>Puram, Sidharth V., Itay Tirosh, Anuraag S. Parikh, Anoop P. Patel, Keren Yizhak, Shawn Gillespie, Christopher Rodman, et al. 2017. “Single-Cell Transcriptomic Analysis of Primary and Metastatic Tumor Ecosystems in Head and Neck Cancer.” <em>Cell</em> 171 (7). Cell Press: 1611–1624.e24. doi:<a href="https://doi.org/10.1016/J.CELL.2017.10.044">10.1016/J.CELL.2017.10.044</a>.</p>
</div>
<div>
<p>Qian, Bin Z., and Jeffrey W. Pollard. 2010. “Macrophage Diversity Enhances Tumor Progression and Metastasis.” doi:<a href="https://doi.org/10.1016/j.cell.2010.03.014">10.1016/j.cell.2010.03.014</a>.</p>
</div>
<div>
<p>Qiao, Wenlian, Gerald Quon, Elizabeth Csaszar, Mei Yu, Quaid Morris, and Peter W. Zandstra. 2012. “PERT: A Method for Expression Deconvolution of Human Blood Samples from Varied Microenvironmental and Developmental Conditions.” Edited by Richard Bonneau. <em>PLoS Comput. Biol.</em> 8 (12). Public Library of Science: e1002838. doi:<a href="https://doi.org/10.1371/journal.pcbi.1002838">10.1371/journal.pcbi.1002838</a>.</p>
</div>
<div>
<p>Quail, D F, and J A Joyce. 2013. “Microenvironmental regulation of tumor progression and metastasis.” <em>Nat Med</em> 19 (11): 1423–37. doi:<a href="https://doi.org/10.1038/nm.3394">10.1038/nm.3394</a>.</p>
</div>
<div>
<p>Racle, Julien, Kaat de Jonge, Petra Baumgaertner, Daniel E Speiser, and David Gfeller. 2017. “Simultaneous enumeration of cancer and immune cell types from bulk tumor gene expression data.” <em>Elife</em> 6. eLife Sciences Publications Limited: e26476. doi:<a href="https://doi.org/10.7554/eLife.26476">10.7554/eLife.26476</a>.</p>
</div>
<div>
<p>Rahmani, Elior, Noah Zaitlen, Yael Baran, Celeste Eng, Donglei Hu, Joshua Galanter, Sam Oh, et al. 2016. “Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies.” <em>Nat. Methods</em> 13 (5). Nature Publishing Group: 443–45. doi:<a href="https://doi.org/10.1038/nmeth.3809">10.1038/nmeth.3809</a>.</p>
</div>
<div>
<p>Regev, Aviv, Sarah A Teichmann, Eric S Lander, Ido Amit, Christophe Benoist, Ewan Birney, Bernd Bodenmiller, et al. 2017. “The Human Cell Atlas.” <em>Elife</em> 6. eLife Sciences Publications Limited: e27041. doi:<a href="https://doi.org/10.7554/eLife.27041">10.7554/eLife.27041</a>.</p>
</div>
<div>
<p>Repsilber, Dirk, Sabine Kern, Anna Telaar, Gerhard Walzl, Gillian F Black, Joachim Selbig, Shreemanta K Parida, Stefan HE Kaufmann, and Marc Jacobsen. 2010. “Biomarker discovery in heterogeneous tissue samples -taking the in-silico deconfounding approach.” <em>BMC Bioinformatics</em> 11 (1). BioMed Central: 27. doi:<a href="https://doi.org/10.1186/1471-2105-11-27">10.1186/1471-2105-11-27</a>.</p>
</div>
<div>
<p>Research in Cancer, International Agency for. 2018. “GLOBOCAN - Facts Sheets by Cancer.” <a href="http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx" class="uri">http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx</a>.</p>
</div>
<div>
<p>Rizvi, Naiyer A., Matthew D. Hellmann, Alexandra Snyder, Pia Kvistborg, Vladimir Makarov, Jonathan J. Havel, William Lee, et al. 2015. “Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer.” <em>Science (80-. ).</em> 348 (6230): 124–28. doi:<a href="https://doi.org/10.1126/science.aaa1348">10.1126/science.aaa1348</a>.</p>
</div>
<div>
<p>Robinson, Dan R., Yi-Mi Wu, Robert J. Lonigro, Pankaj Vats, Erin Cobain, Jessica Everett, Xuhong Cao, et al. 2017. “Integrative clinical genomics of metastatic cancer.” <em>Nature</em> 548 (7667). Nature Publishing Group: 297–303. doi:<a href="https://doi.org/10.1038/nature23306">10.1038/nature23306</a>.</p>
</div>
<div>
<p>Ross, J. S., C. Hatzis, W. F. Symmans, L. Pusztai, and G. N. Hortobagyi. 2008. “Commercialized Multigene Predictors of Clinical Outcome for Breast Cancer.” <em>Oncologist</em> 13 (5): 477–93. doi:<a href="https://doi.org/10.1634/theoncologist.2007-0248">10.1634/theoncologist.2007-0248</a>.</p>
</div>
<div>
<p>Roth, Andrew, Jaswinder Khattra, Damian Yap, Adrian Wan, Emma Laks, Justina Biele, Gavin Ha, Samuel Aparicio, Alexandre Bouchard-Côté, and Sohrab P Shah. 2014. “PyClone: statistical inference of clonal population structure in cancer.” <em>Nat. Methods</em> 11 (4). Nature Publishing Group: 396–98. doi:<a href="https://doi.org/10.1038/nmeth.2883">10.1038/nmeth.2883</a>.</p>
</div>
<div>
<p>Roy, Sushmita, Terran Lane, Chris Allen, Anthony D. Aragon, and Margaret Werner-Washburne. 2006. “A Hidden-State Markov Model for Cell Population Deconvolution.” <em>J. Comput. Biol.</em> 13 (10). Mary Ann Liebert, Inc. 2 Madison Avenue Larchmont, NY 10538 USA: 1749–74. doi:<a href="https://doi.org/10.1089/cmb.2006.13.1749">10.1089/cmb.2006.13.1749</a>.</p>
</div>
<div>
<p>Rutledge, D.N., and D. Jouan-Rimbaud Bouveresse. 2013. “Independent Components Analysis with the JADE algorithm.” <em>TrAC Trends Anal. Chem.</em> 50. Elsevier: 22–32. doi:<a href="https://doi.org/10.1016/J.TRAC.2013.03.013">10.1016/J.TRAC.2013.03.013</a>.</p>
</div>
<div>
<p>Sagiv-Barfi, Idit, Debra K. Czerwinski, Shoshana Levy, Israt S. Alam, Aaron T. Mayer, Sanjiv S. Gambhir, and Ronald Levy. 2018. “Eradication of spontaneous malignancy by local immunotherapy.” <em>Sci. Transl. Med.</em> 10 (426). American Association for the Advancement of Science: eaan4488. doi:<a href="https://doi.org/10.1126/SCITRANSLMED.AAN4488">10.1126/SCITRANSLMED.AAN4488</a>.</p>
</div>
<div>
<p>Satija, Rahul, and Alex K. Shalek. 2014. “Heterogeneity in immune responses: from populations to single cells.” <em>Trends Immunol.</em> 35 (5): 219–29. doi:<a href="https://doi.org/10.1016/j.it.2014.03.004">10.1016/j.it.2014.03.004</a>.</p>
</div>
<div>
<p>Schelker, Max, Sonia Feau, Jinyan Du, Nav Ranu, Edda Klipp, Birgit Schoeberl, Gavin MacBeath, et al. 2017. “Estimation of immune cell content in tumour tissue using single-cell RNA-seq data.” <em>Nat. Commun.</em> 8 (1). Nature Publishing Group: 2032. doi:<a href="https://doi.org/10.1038/s41467-017-02289-3">10.1038/s41467-017-02289-3</a>.</p>
</div>
<div>
<p>Schölkopf, Bernhard, Alex J. Smola, Robert C. Williamson, and Peter L. Bartlett. 2000. “New Support Vector Algorithms.” <em>Neural Comput.</em> 12 (5). MIT Press: 1207–45. doi:<a href="https://doi.org/10.1162/089976600300015565">10.1162/089976600300015565</a>.</p>
</div>
<div>
<p>Schwartz, Russell, and Stanley E Shackney. 2010. “Applying unmixing to gene expression data for tumor phylogeny inference.” <em>BMC Bioinformatics</em> 11 (1). BioMed Central: 42. doi:<a href="https://doi.org/10.1186/1471-2105-11-42">10.1186/1471-2105-11-42</a>.</p>
</div>
<div>
<p>Sebeok, Thomas A. 1976. “DISCUSSION.” <em>Ann. N. Y. Acad. Sci.</em> 280 (1 Origins and E). Blackwell Publishing Ltd: 481. doi:<a href="https://doi.org/10.1111/j.1749-6632.1976.tb25511.x">10.1111/j.1749-6632.1976.tb25511.x</a>.</p>
</div>
<div>
<p>Seung, H. Sebastian, and Daniel D. Lee. 1999. “Learning the parts of objects by non-negative matrix factorization.” <em>Nature</em> 401 (6755). Nature Publishing Group: 788–91. doi:<a href="https://doi.org/10.1038/44565">10.1038/44565</a>.</p>
</div>
<div>
<p>Shannon, Casey P., Robert Balshaw, Virginia Chen, Zsuzsanna Hollander, Mustafa Toma, Bruce M. McManus, J. Mark FitzGerald, Don D. Sin, Raymond T. Ng, and Scott J. Tebbutt. 2017. “Enumerateblood – an R package to estimate the cellular composition of whole blood from Affymetrix Gene ST gene expression profiles.” <em>BMC Genomics</em> 18 (1). BioMed Central: 43. doi:<a href="https://doi.org/10.1186/s12864-016-3460-1">10.1186/s12864-016-3460-1</a>.</p>
</div>
<div>
<p>Shen-Orr, Shai S, and Renaud Gaujoux. 2013. “Computational deconvolution: extracting cell type-specific information from heterogeneous samples.” <em>Curr. Opin. Immunol.</em> 25 (5): 571–78. doi:<a href="https://doi.org/10.1016/j.coi.2013.09.015">10.1016/j.coi.2013.09.015</a>.</p>
</div>
<div>
<p>Shen-Orr, Shai S, Robert Tibshirani, Purvesh Khatri, Dale L Bodian, Frank Staedtler, Nicholas M Perry, Trevor Hastie, Minnie M Sarwal, Mark M Davis, and Atul J Butte. 2010. “Cell type–specific gene expression differences in complex tissues.” <em>Nat. Methods</em> 7 (4). Nature Publishing Group: 287–89. doi:<a href="https://doi.org/10.1038/nmeth.1439">10.1038/nmeth.1439</a>.</p>
</div>
<div>
<p>Shoval, O., H. Sheftel, G. Shinar, Y. Hart, O. Ramote, A. Mayo, E. Dekel, K. Kavanagh, and U. Alon. 2012. “Evolutionary Trade-Offs, Pareto Optimality, and the Geometry of Phenotype Space.” <em>Science (80-. ).</em> 336 (6085): 1157–60. doi:<a href="https://doi.org/10.1126/science.1217405">10.1126/science.1217405</a>.</p>
</div>
<div>
<p>Slater, Matthew H. 2013. “Cell Types as Natural Kinds.” <em>Biol. Theory</em> 7 (2). Springer Netherlands: 170–79. doi:<a href="https://doi.org/10.1007/s13752-012-0084-9">10.1007/s13752-012-0084-9</a>.</p>
</div>
<div>
<p>Slavicek, Gregor. 2012. “Interdisciplinary -A Historical Reflection.” <em>Int. J. Humanit. Soc. Sci.</em> 2 (20). <a href="http://www.ijhssnet.com/journals/Vol{\_}2{\_}No{\_}20{\_}Special{\_}Issue{\_}October{\_}2012/10.pdf" class="uri">http://www.ijhssnet.com/journals/Vol{\_}2{\_}No{\_}20{\_}Special{\_}Issue{\_}October{\_}2012/10.pdf</a>.</p>
</div>
<div>
<p>Ståhl, Patrik L, Fredrik Salmén, Sanja Vickovic, Anna Lundmark, José Fernández Navarro, Jens Magnusson, Stefania Giacomello, et al. 2016. “Visualization and analysis of gene expression in tissue sections by spatial transcriptomics.” <em>Science</em> 353 (6294). American Association for the Advancement of Science: 78–82. doi:<a href="https://doi.org/10.1126/science.aaf2403">10.1126/science.aaf2403</a>.</p>
</div>
<div>
<p>Steuerman, Yael, and Irit Gat-Viks. 2016. “Exploiting Gene-Expression Deconvolution to Probe the Genetics of the Immune System.” Edited by Gary A. Churchill. <em>PLOS Comput. Biol.</em> 12 (4). Public Library of Science: e1004856. doi:<a href="https://doi.org/10.1371/journal.pcbi.1004856">10.1371/journal.pcbi.1004856</a>.</p>
</div>
<div>
<p>Stewart, T., S. CJ Tsai, H. Grayson, R. Henderson, and G. Opelz. 1995. “Incidence of de-novo breast cancer in women chronically immunosuppressed after organ transplantation.” <em>Lancet</em> 346 (8978): 796–98. doi:<a href="https://doi.org/10.1016/S0140-6736(95)91618-0">10.1016/S0140-6736(95)91618-0</a>.</p>
</div>
<div>
<p>Subramanian, Aravind, Pablo Tamayo, Vamsi K Mootha, Sayan Mukherjee, Benjamin L Ebert, Michael A Gillette, Amanda Paulovich, et al. 2005. “Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 102 (43). National Academy of Sciences: 15545–50. doi:<a href="https://doi.org/10.1073/pnas.0506580102">10.1073/pnas.0506580102</a>.</p>
</div>
<div>
<p>Sudhakar, Akulapalli. 2009. “History of Cancer, Ancient and Modern Treatment Methods.” <em>J. Cancer Sci. Ther.</em> 1 (2). NIH Public Access: 1–4. doi:<a href="https://doi.org/10.4172/1948-5956.100000e2">10.4172/1948-5956.100000e2</a>.</p>
</div>
<div>
<p>Svensson, Valentine, Sarah A Teichmann, and Oliver Stegle. 2018. “SpatialDE: identification of spatially variable genes.” <em>Nat. Methods</em> 15 (5). Nature Publishing Group: 343–46. doi:<a href="https://doi.org/10.1038/nmeth.4636">10.1038/nmeth.4636</a>.</p>
</div>
<div>
<p>Şenbabaoğlu, Yasin, Ron S. Gejman, Andrew G. Winer, Ming Liu, Eliezer M. Van Allen, Guillermo de Velasco, Diana Miao, et al. 2016. “Tumor immune microenvironment characterization in clear cell renal cell carcinoma identifies prognostic and immunotherapeutically relevant messenger RNA signatures.” <em>Genome Biol.</em> 17 (1). BioMed Central: 231. doi:<a href="https://doi.org/10.1186/s13059-016-1092-z">10.1186/s13059-016-1092-z</a>.</p>
</div>
<div>
<p>Talmadge, J E, and D I Gabrilovich. 2013. “History of myeloid-derived suppressor cells.” <em>Nat Rev Cancer</em> 13 (10): 739–52. doi:<a href="https://doi.org/10.1038/nrc3581">10.1038/nrc3581</a>.</p>
</div>
<div>
<p>Tamborero, David, Carlota Rubio-Perez, Ferran Muinos, Radhakrishnan Sabarinathan, Josep Maria Piulats, Aura Muntasell, Rodrigo Dienstmann, Nuria Lopez-Bigas, and Abel Gonzalez-Perez. 2018. “A pan-cancer landscape of interactions between solid tumors and infiltrating immune cell populations.” <em>Clin. Cancer Res.</em> American Association for Cancer Research, clincanres.3509.2017. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-17-3509">10.1158/1078-0432.CCR-17-3509</a>.</p>
</div>
<div>
<p>Taube, Janis M, Jérôme Galon, Lynette M Sholl, Scott J Rodig, Tricia R Cottrell, Nicolas A Giraldo, Alexander S Baras, et al. 2017. “Implications of the tumor immune microenvironment for staging and therapeutics.” <em>Mod. Pathol.</em> doi:<a href="https://doi.org/10.1038/modpathol.2017.156">10.1038/modpathol.2017.156</a>.</p>
</div>
<div>
<p>Teng, Michele W. L., Jeremy B. Swann, Catherine M. Koebel, Robert D. Schreiber, and Mark J. Smyth. 2008. “Immune-mediated dormancy: an equilibrium with cancer.” <em>J. Leukoc. Biol.</em> 84 (4): 988–93. doi:<a href="https://doi.org/10.1189/jlb.1107774">10.1189/jlb.1107774</a>.</p>
</div>
<div>
<p>Teschendorff, Andrew E, and Shijie C Zheng. 2017. “Cell-type deconvolution in epigenome-wide association studies: a review and recommendations.” <em>Epigenomics</em> 9 (5): 757–68. doi:<a href="https://doi.org/10.2217/epi-2016-0153">10.2217/epi-2016-0153</a>.</p>
</div>
<div>
<p>Teschendorff, Andrew E., Michel Journée, Pierre A. Absil, Rodolphe Sepulchre, and Carlos Caldas. 2007. “Elucidating the Altered Transcriptional Programs in Breast Cancer using Independent Component Analysis.” <em>PLoS Comput. Biol.</em> 3 (8). Public Library of Science: e161. doi:<a href="https://doi.org/10.1371/journal.pcbi.0030161">10.1371/journal.pcbi.0030161</a>.</p>
</div>
<div>
<p>Teschendorff, Andrew E., Joanna Zhuang, and Martin Widschwendter. 2011. “Independent surrogate variable analysis to deconvolve confounding factors in large-scale microarray profiling studies.” <em>Bioinformatics</em> 27 (11). Oxford University Press: 1496–1505. doi:<a href="https://doi.org/10.1093/bioinformatics/btr171">10.1093/bioinformatics/btr171</a>.</p>
</div>
<div>
<p>“The CRI | Centre for Research and Interdisciplinarity.” 2018. Accessed April 30. <a href="https://cri-paris.org/the-cri/" class="uri">https://cri-paris.org/the-cri/</a>.</p>
</div>
<div>
<p>Thorsson, Vésteinn, David L Gibbs, Scott D Brown, Denise Wolf, Dante S Bortone, Tai-Hsien Ou Yang, Eduard Porta-Pardo, et al. 2018. “The Immune Landscape of Cancer.” <em>Immunity</em> 48 (4). Elsevier: 812–830.e14. doi:<a href="https://doi.org/10.1016/j.immuni.2018.03.023">10.1016/j.immuni.2018.03.023</a>.</p>
</div>
<div>
<p>Tirosh, Itay, Benjamin Izar, Sanjay M. Prakadan, Marc H. Wadsworth, Daniel Treacy, John J. Trombetta, Asaf Rotem, et al. 2016. “Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq.” <em>Science (80-. ).</em> 352 (6282): 189–96. doi:<a href="https://doi.org/10.1126/science.aad0501">10.1126/science.aad0501</a>.</p>
</div>
<div>
<p>Titus, Alexander J., Rachel M. Gallimore, Lucas A. Salas, and Brock C. Christensen. 2017. “Cell-type deconvolution from DNA methylation: a review of recent applications.” <em>Hum. Mol. Genet.</em> 26 (R2): R216–R224. doi:<a href="https://doi.org/10.1093/hmg/ddx275">10.1093/hmg/ddx275</a>.</p>
</div>
<div>
<p>Vallania, Francesco, Andrew Tam, Shane Lofgren, Steven Schaffert, Tej D. Azad, Erika Bongen, Meia Alsup, et al. 2017. “Leveraging heterogeneity across multiple data sets increases accuracy of cell-mixture deconvolution and reduces biological and technical biases.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 206466. doi:<a href="https://doi.org/10.1101/206466">10.1101/206466</a>.</p>
</div>
<div>
<p>Van Der Maaten, Laurens, and Geoffrey Hinton. 2008. “Visualizing Data using t-SNE.” <em>J. Mach. Learn. Res.</em> 9: 2579–2605. <a href="https://lvdmaaten.github.io/publications/papers/JMLR{\_}2008.pdf" class="uri">https://lvdmaaten.github.io/publications/papers/JMLR{\_}2008.pdf</a>.</p>
</div>
<div>
<p>Van Noorden, Richard. 2015. “Interdisciplinary research by the numbers.” <em>Nature</em> 525 (7569): 306–7. doi:<a href="https://doi.org/10.1038/525306a">10.1038/525306a</a>.</p>
</div>
<div>
<p>Van Pel, A, and T Boon. 1982. “Protection against a nonimmunogenic mouse leukemia by an immunogenic variant obtained by mutagenesis.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 79 (15): 4718–22. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6981814 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC346748" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/6981814 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC346748</a>.</p>
</div>
<div>
<p>Vandenbon, Alexis, Viet H Dinh, Norihisa Mikami, Yohko Kitagawa, Shunsuke Teraguchi, Naganari Ohkura, and Shimon Sakaguchi. 2016. “Immuno-Navigator, a batch-corrected coexpression database, reveals cell type-specific gene networks in the immune system.” <em>Proc. Natl. Acad. Sci. U. S. A.</em> 113 (17). National Academy of Sciences: E2393–402. doi:<a href="https://doi.org/10.1073/pnas.1604351113">10.1073/pnas.1604351113</a>.</p>
</div>
<div>
<p>Velculescu, V E, L Zhang, W Zhou, J Vogelstein, M A Basrai, D E Bassett, P Hieter, B Vogelstein, and K W Kinzler. 1997. “Characterization of the yeast transcriptome.” <em>Cell</em> 88 (2): 243–51. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9008165" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/9008165</a>.</p>
</div>
<div>
<p>Venet, D, F Pecasse, C Maenhaut, and H Bersini. 2001. “Separation of samples into their constituents using gene expression data.” <em>Bioinformatics</em> 17 Suppl 1: S279–87. <a href="http://www.ncbi.nlm.nih.gov/pubmed/11473019" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/11473019</a>.</p>
</div>
<div>
<p>Vesely, Matthew D., Michael H. Kershaw, Robert D. Schreiber, and Mark J. Smyth. 2011. “Natural Innate and Adaptive Immunity to Cancer.” <em>Annu. Rev. Immunol.</em> 29 (1): 235–71. doi:<a href="https://doi.org/10.1146/annurev-immunol-031210-101324">10.1146/annurev-immunol-031210-101324</a>.</p>
</div>
<div>
<p>Virchow Rudolf. 1847. “Archiv für pathologische Anatomie und Physiologie und für klinische Medizin.” Berlin ; New York: Springer. <a href="file://catalog.hathitrust.org/Record/000493993 http://hdl.handle.net/2027/njp.32101076036878 (Bd.64 (1875)) http://hdl.handle.net/2027/uc1.31175008645106 (v.170 {\&amp;}index v.161-170) http://hdl.handle.net/2027/hvd.32044093331494 (Bd.111 (1888)) http://hdl.han" class="uri">file://catalog.hathitrust.org/Record/000493993 http://hdl.handle.net/2027/njp.32101076036878 (Bd.64 (1875)) http://hdl.handle.net/2027/uc1.31175008645106 (v.170 {\&amp;}index v.161-170) http://hdl.handle.net/2027/hvd.32044093331494 (Bd.111 (1888)) http://hdl.han</a>.</p>
</div>
<div>
<p>Wang, Min, Stephen R Master, and Lewis A Chodosh. 2006. “Computational expression deconvolution in a complex mammalian organ.” <em>BMC Bioinformatics</em> 7 (1). BioMed Central: 328. doi:<a href="https://doi.org/10.1186/1471-2105-7-328">10.1186/1471-2105-7-328</a>.</p>
</div>
<div>
<p>Wang, Niya, Ting Gong, Robert Clarke, Lulu Chen, Ie-Ming Shih, Zhen Zhang, Douglas A. Levine, Jianhua Xuan, and Yue Wang. 2015. “UNDO: a Bioconductor R package for unsupervised deconvolution of mixed gene expressions in tumor samples.” <em>Bioinformatics</em> 31 (1). Oxford University Press: 137–39. doi:<a href="https://doi.org/10.1093/bioinformatics/btu607">10.1093/bioinformatics/btu607</a>.</p>
</div>
<div>
<p>Wang, Niya, Eric P. Hoffman, Lulu Chen, Li Chen, Zhen Zhang, Chunyu Liu, Guoqiang Yu, David M. Herrington, Robert Clarke, and Yue Wang. 2016. “Mathematical modelling of transcriptional heterogeneity identifies novel markers and subpopulations in complex tissues.” <em>Sci. Rep.</em> 6 (1). Nature Publishing Group: 18909. doi:<a href="https://doi.org/10.1038/srep18909">10.1038/srep18909</a>.</p>
</div>
<div>
<p>Wang, Niya, Fan Meng, Li Chen, Subha Madhavan, Robert Clarke, Eric P Hoffman, Jianhua Xuan, and Yue Wang. 2013. “The CAM Software for Nonnegative Blind Source Separation in R-Java.” <em>J. Mach. Learn. Res.</em> 14: 2899–2903. <a href="http://www.jmlr.org/papers/volume14/wang13d/wang13d.pdf" class="uri">http://www.jmlr.org/papers/volume14/wang13d/wang13d.pdf</a>.</p>
</div>
<div>
<p>Wang, Zeya, Shaolong Cao, Jeffrey S. Morris, Jaeil Ahn, Rongjie Liu, Svitlana Tyekucheva, Fan Gao, et al. 2017. “Transcriptome Deconvolution of Heterogeneous Tumor Samples with Immune Infiltration.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 146795. doi:<a href="https://doi.org/10.1101/146795">10.1101/146795</a>.</p>
</div>
<div>
<p>Wen, Yanhua, Yanjun Wei, Shumei Zhang, Song Li, Hongbo Liu, Fang Wang, Yue Zhao, Dongwei Zhang, and Yan Zhang. 2016. “Cell subpopulation deconvolution reveals breast cancer heterogeneity based on DNA methylation signature.” <em>Brief. Bioinform.</em> 18 (3): bbw028. doi:<a href="https://doi.org/10.1093/bib/bbw028">10.1093/bib/bbw028</a>.</p>
</div>
<div>
<p>Wilczynski, Sharon P. 2009. “Molecular Biology.” In <em>Mod. Surg. Pathol.</em>, 85–120. Elsevier. doi:<a href="https://doi.org/10.1016/B978-1-4160-3966-2.00006-0">10.1016/B978-1-4160-3966-2.00006-0</a>.</p>
</div>
<div>
<p>Wolchok, Jedd D. 2015. “PD-1 Blockers.” doi:<a href="https://doi.org/10.1016/j.cell.2015.07.045">10.1016/j.cell.2015.07.045</a>.</p>
</div>
<div>
<p>Yamamoto, Toshiyuki, Keiko Shimojima, Yumiko Ondo, Katsumi Imai, Pin Fee Chong, Ryutaro Kira, Mitsuhiro Amemiya, Akira Saito, and Nobuhiko Okamoto. 2016. “Challenges in detecting genomic copy number aberrations using next-generation sequencing data and the eXome Hidden Markov Model: a clinical exome-first diagnostic approach.” <em>Hum. Genome Var.</em> 3 (1). Nature Publishing Group: 16025. doi:<a href="https://doi.org/10.1038/hgv.2016.25">10.1038/hgv.2016.25</a>.</p>
</div>
<div>
<p>Yang, Zuyuan, Yong Xiang, Yue Rong, and Kan Xie. 2015. “A Convex Geometry-Based Blind Source Separation Method for Separating Nonnegative Sources.” <em>IEEE Trans. Neural Networks Learn. Syst.</em> 26 (8): 1635–44. doi:<a href="https://doi.org/10.1109/TNNLS.2014.2350026">10.1109/TNNLS.2014.2350026</a>.</p>
</div>
<div>
<p>Yoshihara, Kosuke, Maria Shahmoradgoli, Emmanuel Martínez, Rahulsimham Vegesna, Hoon Kim, Wandaliz Torres-Garcia, Victor Treviño, et al. 2013. “Inferring tumour purity and stromal and immune cell admixture from expression data.” <em>Nat. Commun.</em> 4 (1). Nature Publishing Group: 2612. doi:<a href="https://doi.org/10.1038/ncomms3612">10.1038/ncomms3612</a>.</p>
</div>
<div>
<p>Yu, Zhenhua, Ao Li, and Minghui Wang. 2016. “CloneCNA: detecting subclonal somatic copy number alterations in heterogeneous tumor samples from whole-exome sequencing data.” <em>BMC Bioinformatics</em> 17 (1): 310. doi:<a href="https://doi.org/10.1186/s12859-016-1174-7">10.1186/s12859-016-1174-7</a>.</p>
</div>
<div>
<p>Zarrei, Mehdi, Jeffrey R. MacDonald, Daniele Merico, and Stephen W. Scherer. 2015. “A copy number variation map of the human genome.” <em>Nat. Rev. Genet.</em> 16 (3): 172–83. doi:<a href="https://doi.org/10.1038/nrg3871">10.1038/nrg3871</a>.</p>
</div>
<div>
<p>Zaslavsky, Maxim E, Jacqueline Buros Novik, Eliza J Chang, and Jeffrey E Hammerbacher. 2017. “Infino: a Bayesian hierarchical model improves estimates of immune infiltration into tumor microenvironment.” <em>bioRxiv</em>. Cold Spring Harbor Laboratory, 221671. doi:<a href="https://doi.org/10.1101/221671">10.1101/221671</a>.</p>
</div>
<div>
<p>Zheng, Chunhong, Liangtao Zheng, Jae-Kwang Yoo, Huahu Guo, Yuanyuan Zhang, Xinyi Guo, Boxi Kang, et al. 2017. “Landscape of Infiltrating T Cells in Liver Cancer Revealed by Single-Cell Sequencing.” <em>Cell</em> 169 (7). Cell Press: 1342–1356.e16. doi:<a href="https://doi.org/10.1016/J.CELL.2017.05.035">10.1016/J.CELL.2017.05.035</a>.</p>
</div>
<div>
<p>Zhong, Yi, Ying-Wooi Wan, Kaifang Pang, Lionel ML Chow, and Zhandong Liu. 2013. “Digital sorting of complex tissues for cell type-specific gene expression profiles.” <em>BMC Bioinformatics</em> 14 (1). BioMed Central: 89. doi:<a href="https://doi.org/10.1186/1471-2105-14-89">10.1186/1471-2105-14-89</a>.</p>
</div>
<div>
<p>Zinovyev, Andrei, Ulykbek Kairov, Tatyana Karpenyuk, and Erlan Ramanculov. 2013. “Blind source separation methods for deconvolution of complex signals in cancer biology.” <em>Biochem. Biophys. Res. Commun.</em> 430 (3). Academic Press: 1182–7. doi:<a href="https://doi.org/10.1016/J.BBRC.2012.12.043">10.1016/J.BBRC.2012.12.043</a>.</p>
</div>
<div>
<p>Zou, James, Christoph Lippert, David Heckerman, Martin Aryee, and Jennifer Listgarten. 2014. “Epigenome-wide association studies without the need for cell-type composition.” <em>Nat. Methods</em> 11 (3). Nature Publishing Group: 309–11. doi:<a href="https://doi.org/10.1038/nmeth.2815">10.1038/nmeth.2815</a>.</p>
</div>
<div>
<p>Zou, Weiping, Jedd D. Wolchok, and Lieping Chen. 2016. “PD-L1 (B7-H1) and PD-1 pathway blockade for cancer therapy: Mechanisms, response biomarkers, and combinations.” <em>Sci. Transl. Med.</em> 8 (328). doi:<a href="https://doi.org/10.1126/scitranslmed.aad7118">10.1126/scitranslmed.aad7118</a>.</p>
</div>
<div>
<p>Zuckerman, Neta S., Yair Noam, Andrea J. Goldsmith, and Peter P. Lee. 2013. “A Self-Directed Method for Cell-Type Identification and Separation of Gene Expression Microarrays.” Edited by Teresa M. Przytycka. <em>PLoS Comput. Biol.</em> 9 (8). Public Library of Science: e1003189. doi:<a href="https://doi.org/10.1371/journal.pcbi.1003189">10.1371/journal.pcbi.1003189</a>.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="post-scriptum-thesis-writing.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/12-references.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
