<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>UCzPhDThesis.utf8</title>
  <meta name="description" content="PhDThesis">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="UCzPhDThesis.utf8" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="PhDThesis" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="UCzPhDThesis.utf8" />
  
  <meta name="twitter:description" content="PhDThesis" />
  

<meta name="author" content="">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="annexes.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<script src="libs/plotly-binding-4.7.1/plotly.js"></script>
<script src="libs/typedarray-0.1/typedarray.min.js"></script>
<link href="libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="libs/plotlyjs-1.29.2/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="libs/plotlyjs-1.29.2/plotly-latest.min.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.4.11/datatables.js"></script>
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="libs/selectize-0.12.0/selectize.min.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; min-height: 1.25em; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; }
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: absolute; left: -5em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
@media screen {
a.sourceLine::before { text-decoration: underline; color: initial; }
}
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.bn { color: #40a070; } /* BaseN */
code span.fl { color: #40a070; } /* Float */
code span.ch { color: #4070a0; } /* Char */
code span.st { color: #4070a0; } /* String */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.ot { color: #007020; } /* Other */
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.fu { color: #06287e; } /* Function */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code span.cn { color: #880000; } /* Constant */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.ss { color: #bb6688; } /* SpecialString */
code span.im { } /* Import */
code span.va { color: #19177c; } /* Variable */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.op { color: #666666; } /* Operator */
code span.bu { } /* BuiltIn */
code span.ex { } /* Extension */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.at { color: #7d9029; } /* Attribute */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD Thesis Urszula Czerwinska</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html"><i class="fa fa-check"></i>Preamble about Interdisciplinary Research</a><ul>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#what-does-interdisciplinarity-in-science-mean-in-xxi-century"><i class="fa fa-check"></i>What does interdisciplinarity in science mean in XXI century?</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#strengths-weaknesses-opportunities-threats-swot-of-an-interdisciplinary-phd---personal-perspective"><i class="fa fa-check"></i>Strengths, Weaknesses Opportunities, Threats (SWOT) of an interdisciplinary PhD - personal perspective</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#general-context-of-the-thesis"><i class="fa fa-check"></i>General context of the thesis</a></li>
<li class="chapter" data-level="" data-path="preamble-about-interdisciplinary-research.html"><a href="preamble-about-interdisciplinary-research.html#organisation-of-the-dissertation"><i class="fa fa-check"></i>Organisation of the dissertation</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Immuno-biology of cancer</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#cancer-disease"><i class="fa fa-check"></i><b>1.1</b> Cancer disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="intro.html"><a href="intro.html#hist"><i class="fa fa-check"></i><b>1.1.1</b> Historical understanding of cancer</a></li>
<li class="chapter" data-level="1.1.2" data-path="intro.html"><a href="intro.html#tumor-microenvironment-as-a-complex-system"><i class="fa fa-check"></i><b>1.1.2</b> Tumor Microenvironment as a complex system</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#quantifying-and-qualifying-immune-infiltration-data"><i class="fa fa-check"></i><b>1.2</b> Quantifying and qualifying immune infiltration (data)</a><ul>
<li class="chapter" data-level="1.2.1" data-path="intro.html"><a href="intro.html#facs"><i class="fa fa-check"></i><b>1.2.1</b> Cell sorting</a></li>
<li class="chapter" data-level="1.2.2" data-path="intro.html"><a href="intro.html#staining"><i class="fa fa-check"></i><b>1.2.2</b> Microscope Staining</a></li>
<li class="chapter" data-level="1.2.3" data-path="intro.html"><a href="intro.html#omics"><i class="fa fa-check"></i><b>1.2.3</b> omics</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#immunotherapies"><i class="fa fa-check"></i><b>1.3</b> From cancer phenotyping to immune therapies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="intro.html"><a href="intro.html#cancer-immune-phenotypes"><i class="fa fa-check"></i><b>1.3.1</b> Cancer immune phenotypes</a></li>
<li class="chapter" data-level="1.3.2" data-path="intro.html"><a href="intro.html#scoring-the-immune-infiltration"><i class="fa fa-check"></i><b>1.3.2</b> Scoring the immune infiltration</a></li>
<li class="chapter" data-level="1.3.3" data-path="intro.html"><a href="intro.html#immune-signatures"><i class="fa fa-check"></i><b>1.3.3</b> Immune signatures - biological perspective</a></li>
<li class="chapter" data-level="1.3.4" data-path="intro.html"><a href="intro.html#cancer_Therapies"><i class="fa fa-check"></i><b>1.3.4</b> Cancer therapies</a></li>
<li class="chapter" data-level="1.3.5" data-path="intro.html"><a href="intro.html#recent-progress-in-immuno-therapies"><i class="fa fa-check"></i><b>1.3.5</b> Recent progress in immuno-therapies</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#biological-dimension-of-the-thesis"><i class="fa fa-check"></i><b>1.4</b> Biological dimension of the thesis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Mathematical foundation of cell-type deconvolution of biological data</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#introduction-to-supervised-and-unsupervised-learning"><i class="fa fa-check"></i><b>2.1</b> Introduction to supervised and unsupervised learning</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#cell-type-deconvolution-models-of-bulk-transcriptome"><i class="fa fa-check"></i><b>2.2</b> Cell-type deconvolution models of bulk transcriptome</a><ul>
<li class="chapter" data-level="2.2.1" data-path="methods.html"><a href="methods.html#literature-overview"><i class="fa fa-check"></i><b>2.2.1</b> Literature overview</a></li>
<li class="chapter" data-level="2.2.2" data-path="methods.html"><a href="methods.html#regression-based-methods"><i class="fa fa-check"></i><b>2.2.2</b> Regression-based methods</a></li>
<li class="chapter" data-level="2.2.3" data-path="methods.html"><a href="methods.html#regression-algorithm"><i class="fa fa-check"></i><b>2.2.3</b> regression algorithm</a></li>
<li class="chapter" data-level="2.2.4" data-path="methods.html"><a href="methods.html#enrichment-based-methods"><i class="fa fa-check"></i><b>2.2.4</b> Enrichment-based methods</a></li>
<li class="chapter" data-level="2.2.5" data-path="methods.html"><a href="methods.html#probabilistic-methods"><i class="fa fa-check"></i><b>2.2.5</b> Probabilistic methods</a></li>
<li class="chapter" data-level="2.2.6" data-path="methods.html"><a href="methods.html#convex-hull-based-methods"><i class="fa fa-check"></i><b>2.2.6</b> Convex-hull based methods</a></li>
<li class="chapter" data-level="2.2.7" data-path="methods.html"><a href="methods.html#matrix-factorisation-methods"><i class="fa fa-check"></i><b>2.2.7</b> Matrix factorisation methods</a></li>
<li class="chapter" data-level="2.2.8" data-path="methods.html"><a href="methods.html#others-aspects"><i class="fa fa-check"></i><b>2.2.8</b> Others aspects</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#deconvolution-of-other-data-types"><i class="fa fa-check"></i><b>2.3</b> Deconvolution of other data types</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#methodological-dimension-of-the-thesis"><i class="fa fa-check"></i><b>2.4</b> Methodological dimension of the thesis</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="sens.html"><a href="sens.html"><i class="fa fa-check"></i><b>3</b> Study of sensitivity and reproducibility of known methods</a><ul>
<li class="chapter" data-level="3.1" data-path="sens.html"><a href="sens.html#finding-optimal-number-of-components-and-over-decomposition-of-transcriptopmes"><i class="fa fa-check"></i><b>3.1</b> Finding optimal number of components and over-decomposition of transcriptopmes</a></li>
<li class="chapter" data-level="3.2" data-path="sens.html"><a href="sens.html#section"><i class="fa fa-check"></i><b>3.2</b> </a></li>
<li class="chapter" data-level="3.3" data-path="sens.html"><a href="sens.html#reproducibility-of-nmf-versus-ica-vs-cam"><i class="fa fa-check"></i><b>3.3</b> REPRODUCIBILITY OF NMF VERSUS ICA (VS CAM?)</a><ul>
<li class="chapter" data-level="3.3.1" data-path="sens.html"><a href="sens.html#comparing-metagenes-obtained-with-nmf-vs-ica."><i class="fa fa-check"></i><b>3.3.1</b> Comparing metagenes obtained with NMF vs ICA.</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="sens.html"><a href="sens.html#impact-of-modification-of-signatures-list-on-result-for-signature-based-deconvolution-methods"><i class="fa fa-check"></i><b>3.4</b> Impact of modification of signatures list on result for signature-based deconvolution methods</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="deconica.html"><a href="deconica.html"><i class="fa fa-check"></i><b>4</b> Deconvolution of transcriptomes and methylomes</a><ul>
<li class="chapter" data-level="4.1" data-path="deconica.html"><a href="deconica.html#from-blind-deconvolution-to-cell-type-quantification-general-overview"><i class="fa fa-check"></i><b>4.1</b> From blind deconvolution to cell-type quantification: general overview</a><ul>
<li class="chapter" data-level="4.1.1" data-path="deconica.html"><a href="deconica.html#the-ica-based-deconvolution-of-transcriptomes"><i class="fa fa-check"></i><b>4.1.1</b> The ICA-based deconvolution of Transcriptomes</a></li>
<li class="chapter" data-level="4.1.2" data-path="deconica.html"><a href="deconica.html#interpretation-of-independent-components"><i class="fa fa-check"></i><b>4.1.2</b> Interpretation of Independent components</a></li>
<li class="chapter" data-level="4.1.3" data-path="deconica.html"><a href="deconica.html#transforming-metagenes-into-signature-matrix"><i class="fa fa-check"></i><b>4.1.3</b> Transforming metagenes into signature matrix</a></li>
<li class="chapter" data-level="4.1.4" data-path="deconica.html"><a href="deconica.html#regression-based-estimation-of-cell-type-proportions-solving-system-of-equations"><i class="fa fa-check"></i><b>4.1.4</b> Regression-based estimation of cell-type proportions : solving system of equations</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="deconica.html"><a href="deconica.html#deconica-r-package-for-ica-based-deconvolution"><i class="fa fa-check"></i><b>4.2</b> <em>DeconICA</em> R package for ICA-based deconvolution</a><ul>
<li class="chapter" data-level="4.2.1" data-path="deconica.html"><a href="deconica.html#demo"><i class="fa fa-check"></i><b>4.2.1</b> <em>Demo</em></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>5</b> Comparative analysis of cancer immune infiltration</a></li>
<li class="chapter" data-level="6" data-path="map.html"><a href="map.html"><i class="fa fa-check"></i><b>6</b> Heterogeneity of immune cell types</a></li>
<li class="chapter" data-level="7" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>7</b> Conclusions and perspectives</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html"><i class="fa fa-check"></i>Annexes</a><ul>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#phd-timeline-for-defence-before-the-end-of-october-2018"><i class="fa fa-check"></i>PhD timeline for defence before the end of October 2018</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#thesis-writing"><i class="fa fa-check"></i>Thesis writing</a></li>
<li class="chapter" data-level="" data-path="annexes.html"><a href="annexes.html#activity-report-2017"><i class="fa fa-check"></i>Activity Report 2017</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>

<div id="refs" class="references">
<div>
<p>Angelova, Mihaela, Pornpimol Charoentong, Hubert Hackl, Maria L Fischer, Rene Snajder, Anne M Krogsdam, Maximilian J Waldner, et al. 2015. “Characterization of the immunophenotypes and antigenomes of colorectal cancers reveals distinct tumor escape mechanisms and novel targets for immunotherapy.” <em>Genome Biology</em> 16 (1). BioMed Central: 64. doi:<a href="https://doi.org/10.1186/s13059-015-0620-6">10.1186/s13059-015-0620-6</a>.</p>
</div>
<div>
<p>Anitei, M G, G Zeitoun, B Mlecnik, F Marliot, N Haicheur, A M Todosi, A Kirilovsky, et al. 2014. “Prognostic and predictive values of the immunoscore in patients with rectal cancer.” <em>Clin Cancer Res</em> 20 (7): 1891–9. doi:<a href="https://doi.org/10.1158/1078-0432.ccr-13-2830">10.1158/1078-0432.ccr-13-2830</a>.</p>
</div>
<div>
<p>Aris, Mariana, María Marcela Barrio, and José Mordoh. 2012. “Lessons from Cancer Immunoediting in Cutaneous Melanoma.” <em>Clinical and Developmental Immunology</em> 2012: 1–14. doi:<a href="https://doi.org/10.1155/2012/192719">10.1155/2012/192719</a>.</p>
</div>
<div>
<p>Balkwill, Fran, and Alberto Mantovani. 2001. “Inflammation and cancer: Back to Virchow?” doi:<a href="https://doi.org/10.1016/S0140-6736(00)04046-0">10.1016/S0140-6736(00)04046-0</a>.</p>
</div>
<div>
<p>Baxevanis, C N, G V Dedoussis, N G Papadopoulos, I Missitzis, G P Stathopoulos, and M Papamichail. 1994. “Tumor specific cytolysis by tumor infiltrating lymphocytes in breast cancer.” <em>Cancer</em> 74 (4): 1275–82. <a href="http://www.ncbi.nlm.nih.gov/pubmed/7914469" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/7914469</a>.</p>
</div>
<div>
<p>Becht, Etienne, Aurélien de Reyniès, Nicolas A Giraldo, Camilla Pilati, Bénédicte Buttard, Laetitia Lacroix, Janick Selves, Catherine Sautès-Fridman, Pierre Laurent-Puig, and Wolf Herman Fridman. 2016. “Immune and Stromal Classification of Colorectal Cancer Is Associated with Molecular Subtypes and Relevant for Precision Immunotherapy.” <em>Clinical Cancer Research : An Official Journal of the American Association for Cancer Research</em> 22 (16). American Association for Cancer Research: 4057–66. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-15-2879">10.1158/1078-0432.CCR-15-2879</a>.</p>
</div>
<div>
<p>Berghoff, Anna, Jakob Kather, and Dirk Jäger. 2018. “Genetics and Immunology: Tumor-Specific Genetic Alterations as a Target for Immune Modulating Therapies.” In <em>Oncoimmunology : A Practical Guide for Cancer Immunotherapy</em>, edited by Laurence Zitvogel and Guido Kroemer, 231–46.</p>
</div>
<div>
<p>Bernstein, Bradley E, Alexander Meissner, and Eric S Lander. 2007. “The mammalian epigenome.” <em>Cell</em> 128 (4). Elsevier: 669–81. doi:<a href="https://doi.org/10.1016/j.cell.2007.01.033">10.1016/j.cell.2007.01.033</a>.</p>
</div>
<div>
<p>Bhatia, Alka, and Yashwant Kumar. 2011. “Cancer-immune equilibrium: questions unanswered.” <em>Cancer Microenvironment : Official Journal of the International Cancer Microenvironment Society</em> 4 (2). Springer: 209–17. doi:<a href="https://doi.org/10.1007/s12307-011-0065-8">10.1007/s12307-011-0065-8</a>.</p>
</div>
<div>
<p>Bindea, Gabriela, Bernhard Mlecnik, Marie Tosolini, Amos Kirilovsky, Maximilian Waldner, Anna C. Obenauf, Helen Angell, et al. 2013. “Spatiotemporal Dynamics of Intratumoral Immune Cells Reveal the Immune Landscape in Human Cancer.” <em>Immunity</em> 39 (4): 782–95. doi:<a href="https://doi.org/10.1016/j.immuni.2013.10.003">10.1016/j.immuni.2013.10.003</a>.</p>
</div>
<div>
<p>Blank, Christian U, John B Haanen, Antoni Ribas, and Ton N Schumacher. 2016. “CANCER IMMUNOLOGY. the ‘Cancer Immunogram’.” <em>Science (New York, N.Y.)</em> 352 (6286). American Association for the Advancement of Science: 658–60. doi:<a href="https://doi.org/10.1126/science.aaf2834">10.1126/science.aaf2834</a>.</p>
</div>
<div>
<p>Boal, David H. 2002. <em>Mechanics of the cell</em>. Cambridge University Press.</p>
</div>
<div>
<p>Burnet, F M. 1970. “The concept of immunological surveillance.” <em>Progress in Experimental Tumor Research</em> 13: 1–27. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4921480" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/4921480</a>.</p>
</div>
<div>
<p>Cavallo, Federica, Carla De Giovanni, Patrizia Nanni, Guido Forni, and Pier-Luigi Lollini. 2011. “No Title” 60 (3). Springer-Verlag. doi:<a href="https://doi.org/10.1007/s00262-010-0968-0">10.1007/s00262-010-0968-0</a>.</p>
</div>
<div>
<p>Charoentong, Pornpimol, Francesca Finotello, Mihaela Angelova, Clemens Mayer, Mirjana Efremova, Dietmar Rieder, Hubert Hackl, and Zlatko Trajanoski. 2017. “Pan-cancer Immunogenomic Analyses Reveal Genotype-Immunophenotype Relationships and Predictors of Response to Checkpoint Blockade.” <em>Cell Reports</em> 18 (1). Elsevier: 248–62. doi:<a href="https://doi.org/10.1016/j.celrep.2016.12.019">10.1016/j.celrep.2016.12.019</a>.</p>
</div>
<div>
<p>Chen, Daniel S., and Ira Mellman. 2017. “Elements of cancer immunity and the cancer-immune set point.” doi:<a href="https://doi.org/10.1038/nature21349">10.1038/nature21349</a>.</p>
</div>
<div>
<p>Chen, Pei Ling, Whijae Roh, Alexandre Reuben, Zachary A. Cooper, Christine N. Spencer, Peter A. Prieto, John P. Miller, et al. 2016. “Analysis of immune signatures in longitudinal tumor samples yields insight into biomarkers of response and mechanisms of resistance to immune checkpoint blockade.” <em>Cancer Discovery</em> 6 (8): 827–37. doi:<a href="https://doi.org/10.1158/2159-8290.CD-15-1545">10.1158/2159-8290.CD-15-1545</a>.</p>
</div>
<div>
<p>Chifman, Julia, Ashok Pullikuth, Jeff W Chou, Davide Bedognetti, and Lance D Miller. 2016. “Conservation of immune gene signatures in solid tumors and prognostic implications.” <em>BMC Cancer</em> 16 (1): 911. doi:<a href="https://doi.org/10.1186/s12885-016-2948-z">10.1186/s12885-016-2948-z</a>.</p>
</div>
<div>
<p>Cieślik, Marcin, and Arul M. Chinnaiyan. 2017. “Cancer transcriptome profiling at the juncture of clinical translation.” <em>Nature Reviews Genetics</em> 19 (2). Nature Publishing Group: 93–109. doi:<a href="https://doi.org/10.1038/nrg.2017.96">10.1038/nrg.2017.96</a>.</p>
</div>
<div>
<p>Core, Cedar-Sinals Genomics. 2016. “Single Cell Genomics.” <a href="https://www.cedars-sinai.edu/Research/Research-Cores/Genomics-Core/Documents/website_pricing_jan_2018_final.pdf" class="uri">https://www.cedars-sinai.edu/Research/Research-Cores/Genomics-Core/Documents/website_pricing_jan_2018_final.pdf</a>.</p>
</div>
<div>
<p>Corlan, Alexandru Dan. 2004. “Medline Trend: Automated Yearly Statistics of Pubmed Results for Any Query.” <a href="http://dan.corlan.net/medline-trend.html" class="uri">http://dan.corlan.net/medline-trend.html</a>.</p>
</div>
<div>
<p>Costa, Ana, Yann Kieffer, Alix Scholer-Dahirel, Floriane Pelon, Brigitte Bourachot, Melissa Cardon, Philemon Sirven, et al. 2018. “Fibroblast Heterogeneity and Immunosuppressive Environment in Human Breast Cancer.” <em>Cancer Cell</em> 33 (3). Elsevier: 463–479.e10. doi:<a href="https://doi.org/10.1016/j.ccell.2018.01.011">10.1016/j.ccell.2018.01.011</a>.</p>
</div>
<div>
<p>Csermely, Peter., Korado. Korlevic, and Katalin. Sulyok. 2007. <em>Science education : models and networking of student research training under 21</em>. IOS Press.</p>
</div>
<div>
<p>Davis, Sean. 2016. “Github: Awesome-Single-Cell.” <a href="https://github.com/seandavi/awesome-single-cell" class="uri">https://github.com/seandavi/awesome-single-cell</a>.</p>
</div>
<div>
<p>Dumont, Nancy, Bob Liu, Rosa Anna DeFilippis, Hang Chang, Joseph T. Rabban, Anthony N. Karnezis, Judy A. Tjoe, James Marx, Bahram Parvin, and Thea D. Tlsty. 2013. “Breast Fibroblasts Modulate Early Dissemination, Tumorigenesis, and Metastasis through Alteration of Extracellular Matrix Characteristics.” <em>Neoplasia</em> 15 (3): 249–IN7. doi:<a href="https://doi.org/10.1593/neo.121950">10.1593/neo.121950</a>.</p>
</div>
<div>
<p>Dunn, Gavin P., Allen T. Bruce, Hiroaki Ikeda, Lloyd J. Old, and Robert D. Schreiber. 2002. “Cancer immunoediting: from immunosurveillance to tumor escape.” <em>Nature Immunology</em> 3 (11). Nature Publishing Group: 991–98. doi:<a href="https://doi.org/10.1038/ni1102-991">10.1038/ni1102-991</a>.</p>
</div>
<div>
<p>Ediorial Cell Systems. 2017. “What Is Your Conceptual Definition of ‘Cell Type’ in the Context of a Mature Organism?” <em>Cell Systems</em> 4 (3). Elsevier: 255–59. doi:<a href="https://doi.org/10.1016/j.cels.2017.03.006">10.1016/j.cels.2017.03.006</a>.</p>
</div>
<div>
<p>Ehrlich, P. 1909. “Über den jetzigen Stand der Chemotherapie.” <em>Berichte Der Deutschen Chemischen Gesellschaft</em> 42 (1). Wiley-Blackwell: 17–47. doi:<a href="https://doi.org/10.1002/cber.19090420105">10.1002/cber.19090420105</a>.</p>
</div>
<div>
<p><em>Facilitating Interdisciplinary Research</em>. 2004. Washington, D.C.: National Academies Press. doi:<a href="https://doi.org/10.17226/11153">10.17226/11153</a>.</p>
</div>
<div>
<p>Feinberg, Andrew P., and Bert Vogelstein. 1983. “Hypomethylation distinguishes genes of some human cancers from their normal counterparts.” <em>Nature</em> 301 (5895). Nature Publishing Group: 89–92. doi:<a href="https://doi.org/10.1038/301089a0">10.1038/301089a0</a>.</p>
</div>
<div>
<p>Folkman, J, E Merler, C Abernathy, and G Williams. 1971. “Isolation of a tumor factor responsible for angiogenesis.” <em>The Journal of Experimental Medicine</em> 133 (2). Rockefeller University Press: 275–88. doi:<a href="https://doi.org/10.1084/JEM.133.2.275">10.1084/JEM.133.2.275</a>.</p>
</div>
<div>
<p>Frey, Daniel M., Raoul A. Droeser, Carsten T. Viehl, Inti Zlobec, Alessandro Lugli, Urs Zingg, Daniel Oertli, Christoph Kettelhack, Luigi Terracciano, and Luigi Tornillo. 2010. “High frequency of tumor-infiltrating FOXP3+ regulatory T cells predicts improved survival in mismatch repair-proficient colorectal cancer patients.” <em>International Journal of Cancer</em> 126 (11): 2635–43. doi:<a href="https://doi.org/10.1002/ijc.24989">10.1002/ijc.24989</a>.</p>
</div>
<div>
<p>Gabrilovich, Dmitry I., Suzanne Ostrand-Rosenberg, and Vincenzo Bronte. 2012. “Coordinated regulation of myeloid cells by tumours.” doi:<a href="https://doi.org/10.1038/nri3175">10.1038/nri3175</a>.</p>
</div>
<div>
<p>Gajewski, Thomas F., Yuru Meng, Christian Blank, Ian Brown, Aalok Kacha, Justin Kline, and Helena Harlin. 2006. “Immune resistance orchestrated by the tumor microenvironment.” <em>Immunological Reviews</em> 213 (1). Blackwell Publishing Ltd: 131–45. doi:<a href="https://doi.org/10.1111/J.1600-065X.2006.00442.X">10.1111/J.1600-065X.2006.00442.X</a>.</p>
</div>
<div>
<p>Galon, J., B. A. Fox, C. B. Bifulco, G. Masucci, T. Rau, G. Botti, F. M. Marincola, et al. 2016. “Immunoscore and Immunoprofiling in cancer: An update from the melanoma and immunotherapy bridge 2015.” <em>Journal of Translational Medicine</em> 14 (1). doi:<a href="https://doi.org/10.1186/s12967-016-1029-z">10.1186/s12967-016-1029-z</a>.</p>
</div>
<div>
<p>Galon, Jérôme, Bernhard Mlecnik, Gabriela Bindea, Helen K Angell, Anne Berger, Christine Lagorce, Alessandro Lugli, et al. 2014. “Towards the introduction of the ‘Immunoscore’ in the classification of malignant tumours.” <em>The Journal of Pathology</em> 232 (2): 199–209. doi:<a href="https://doi.org/10.1002/path.4287">10.1002/path.4287</a>.</p>
</div>
<div>
<p>Galon, Jérôme, Franck Pagès, Francesco M Marincola, Magdalena Thurin, Giorgio Trinchieri, Bernard A Fox, Thomas F Gajewski, and Paolo A Ascierto. 2012. “The immune score as a new possible approach for the classification of cancer.” <em>Journal of Translational Medicine</em> 10 (1). BioMed Central: 1. doi:<a href="https://doi.org/10.1186/1479-5876-10-1">10.1186/1479-5876-10-1</a>.</p>
</div>
<div>
<p>Giesen, Charlotte, Hao A O Wang, Denis Schapiro, Nevena Zivanovic, Andrea Jacobs, Bodo Hattendorf, Peter J Schüffler, et al. 2014. “Highly multiplexed imaging of tumor tissues with subcellular resolution by mass cytometry.” <em>Nature Methods</em> 11 (4). Nature Publishing Group: 417–22. doi:<a href="https://doi.org/10.1038/nmeth.2869">10.1038/nmeth.2869</a>.</p>
</div>
<div>
<p>Gnjatic, Sacha, Vincenzo Bronte, Laura Rosa Brunet, Marcus O Butler, Mary L Disis, Jérôme Galon, Leif G Hakansson, et al. 2017. “Identifying baseline immune-related biomarkers to predict clinical outcome of immunotherapy.” <em>Journal for Immunotherapy of Cancer</em> 5. BioMed Central: 44. doi:<a href="https://doi.org/10.1186/s40425-017-0243-4">10.1186/s40425-017-0243-4</a>.</p>
</div>
<div>
<p>Godec, Jernej, Yan Tan, Arthur Liberzon, Pablo Tamayo, Sanchita Bhattacharya, Atul J. Butte, Jill P. Mesirov, and W. Nicholas Haining. 2016. “Compendium of Immune Signatures Identifies Conserved and Species-Specific Biology in Response to Inflammation.” <em>Immunity</em> 44 (1): 194–206. doi:<a href="https://doi.org/10.1016/j.immuni.2015.12.006">10.1016/j.immuni.2015.12.006</a>.</p>
</div>
<div>
<p>Golub, T R, D K Slonim, P Tamayo, C Huard, M Gaasenbeek, J P Mesirov, H Coller, et al. 1999. “Molecular classification of cancer: class discovery and class prediction by gene expression monitoring.” <em>Science (New York, N.Y.)</em> 286 (5439). American Association for the Advancement of Science: 531–7. doi:<a href="https://doi.org/10.1126/SCIENCE.286.5439.531">10.1126/SCIENCE.286.5439.531</a>.</p>
</div>
<div>
<p>Greenblatt, M, and P Shubi. 1968. “Tumor angiogenesis: transfilter diffusion studies in the hamster by the transparent chamber technique.” <em>Journal of the National Cancer Institute</em> 41 (1): 111–24. <a href="http://www.ncbi.nlm.nih.gov/pubmed/5662020" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/5662020</a>.</p>
</div>
<div>
<p>Greger, Valerie, Eberhard Passarge, Wolfgang H�pping, Elmar Messmer, and Bernhard Horsthemke. 1989. “Epigenetic changes may contribute to the formation and spontaneous regression of retinoblastoma.” <em>Human Genetics</em> 83 (2). Springer-Verlag: 155–58. doi:<a href="https://doi.org/10.1007/BF00286709">10.1007/BF00286709</a>.</p>
</div>
<div>
<p>Győrffy, Balázs, Christos Hatzis, Tara Sanft, Erin Hofstatter, Bilge Aktas, and Lajos Pusztai. 2015. “Multigene prognostic tests in breast cancer: past, present, future.” <em>Breast Cancer Research : BCR</em> 17 (1). BioMed Central: 11. doi:<a href="https://doi.org/10.1186/s13058-015-0514-2">10.1186/s13058-015-0514-2</a>.</p>
</div>
<div>
<p>Hanahan, D, and R A Weinberg. 2000. “The hallmarks of cancer.” <em>Cell</em> 100 (1). Elsevier: 57–70. doi:<a href="https://doi.org/10.1016/S0092-8674(00)81683-9">10.1016/S0092-8674(00)81683-9</a>.</p>
</div>
<div>
<p>Hanahan, Douglas, and Lisa M. Coussens. 2012. “Accessories to the Crime: Functions of Cells Recruited to the Tumor Microenvironment.” <em>Cancer Cell</em> 21 (3): 309–22. doi:<a href="https://doi.org/10.1016/j.ccr.2012.02.022">10.1016/j.ccr.2012.02.022</a>.</p>
</div>
<div>
<p>Harris, Lyndsay N., Nofisat Ismaila, Lisa M. McShane, Fabrice Andre, Deborah E. Collyar, Ana M. Gonzalez-Angulo, Elizabeth H. Hammond, et al. 2016. “Use of Biomarkers to Guide Decisions on Adjuvant Systemic Therapy for Women With Early-Stage Invasive Breast Cancer: American Society of Clinical Oncology Clinical Practice Guideline.” <em>Journal of Clinical Oncology</em> 34 (10): 1134–50. doi:<a href="https://doi.org/10.1200/JCO.2015.65.2289">10.1200/JCO.2015.65.2289</a>.</p>
</div>
<div>
<p>Health, U.S. Department of, and Human Services. 2015. “FDA Approves Yervoy to Reduce the Risk of Melanoma Returning After Surgery.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm469944.htm</a>.</p>
</div>
<div>
<p>———. 2016a. “FDA Approves New, Targeted Treatment for Bladder Cancer.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm501762.htm</a>.</p>
</div>
<div>
<p>———. 2016b. “Pembrolizumab (Keytruda) Checkpoint Inhibitor.” <a href="https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm" class="uri">https://www.fda.gov/drugs/informationondrugs/approveddrugs/ucm526430.htm</a>.</p>
</div>
<div>
<p>———. 2017a. “FDA Approval Brings First Gene Therapy to the United States.” <a href="https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm" class="uri">https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm574058.htm</a>.</p>
</div>
<div>
<p>———. 2017b. “FDA Approves Car-T Cell Therapy to Treat Adults with Certain Types of Large B-Cell Lymphoma.” <a href="https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm581216.htm" class="uri">https://www.fda.gov/newsevents/newsroom/pressannouncements/ucm581216.htm</a>.</p>
</div>
<div>
<p>Heng, Tracy S P, Michio W Painter, Kutlu Elpek, Veronika Lukacs-Kornek, Nora Mauermann, Shannon J Turley, Daphne Koller, et al. 2008. “The Immunological Genome Project: networks of gene expression in immune cells.” <em>Nature Immunology</em> 9 (10): 1091–4. doi:<a href="https://doi.org/10.1038/ni1008-1091">10.1038/ni1008-1091</a>.</p>
</div>
<div>
<p>Herbst, Roy S., Jean-Charles Soria, Marcin Kowanetz, Gregg D. Fine, Omid Hamid, Michael S. Gordon, Jeffery A. Sosman, et al. 2014. “Predictive correlates of response to the anti-PD-L1 antibody MPDL3280A in cancer patients.” <em>Nature</em> 515 (7528): 563–67. doi:<a href="https://doi.org/10.1038/nature14011">10.1038/nature14011</a>.</p>
</div>
<div>
<p>Institute, National Cancer. 2017. “Types of Cancer Treatment.” <a href="https://www.cancer.gov/about-cancer/treatment/types" class="uri">https://www.cancer.gov/about-cancer/treatment/types</a>.</p>
</div>
<div>
<p>Itadani, Hiraku, Shinji Mizuarai, and Hidehito Kotani. 2008. “Can Systems Biology Understand Pathway Activation? Gene Expression Signatures as Surrogate Markers for Understanding the Complexity of Pathway Activation.” <em>Current Genomics</em> 9 (5): 349–60. doi:<a href="https://doi.org/10.2174/138920208785133235">10.2174/138920208785133235</a>.</p>
</div>
<div>
<p>Jackson, Hollie J., Sarwish Rafiq, and Renier J. Brentjens. 2016. “Driving CAR T-cells forward.” doi:<a href="https://doi.org/10.1038/nrclinonc.2016.36">10.1038/nrclinonc.2016.36</a>.</p>
</div>
<div>
<p>Jeschke, Jana, Martin Bizet, Christine Desmedt, Emilie Calonne, Sarah Dedeurwaerder, Soizic Garaud, Alexander Koch, et al. 2017. “DNA methylation–based immune response signature improves patient diagnosis in multiple cancers.” <em>Journal of Clinical Investigation</em> 127 (8): 3090–3102. doi:<a href="https://doi.org/10.1172/JCI91095">10.1172/JCI91095</a>.</p>
</div>
<div>
<p>Kornstein, M J, J S Brooks, and D E Elder. 1983. “Immunoperoxidase localization of lymphocyte subsets in the host response to melanoma and nevi.” <em>Cancer Research</em> 43 (6): 2749–53. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6342758" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/6342758</a>.</p>
</div>
<div>
<p>Kuperstein, I, E Bonnet, H-A Nguyen, D Cohen, E Viara, L Grieco, S Fourquet, et al. 2015. “Atlas of Cancer Signalling Network: a systems biology resource for integrative analysis of cancer data with Google Maps.” <em>Oncogenesis</em> 4 (7): e160–e160. doi:<a href="https://doi.org/10.1038/oncsis.2015.19">10.1038/oncsis.2015.19</a>.</p>
</div>
<div>
<p>Kuperstein, Inna, David PA Cohen, Stuart Pook, Eric Viara, Laurence Calzone, Emmanuel Barillot, and Andrei Zinovyev. 2013. “NaviCell: a web-based environment for navigation, curation and maintenance of large molecular interaction maps.” <em>BMC Systems Biology</em> 7 (1). BioMed Central: 100. doi:<a href="https://doi.org/10.1186/1752-0509-7-100">10.1186/1752-0509-7-100</a>.</p>
</div>
<div>
<p>Ledford, Heidi. 2015. “How to solve the world’s biggest problems.” <em>Nature</em> 525 (7569): 308–11. doi:<a href="https://doi.org/10.1038/525308a">10.1038/525308a</a>.</p>
</div>
<div>
<p>Leung, D., G Cachianes, W. Kuang, D. Goeddel, and N Ferrara. 1989. “Vascular endothelial growth factor is a secreted angiogenic mitogen.” <em>Science</em> 246 (4935): 1306–9. doi:<a href="https://doi.org/10.1126/science.2479986">10.1126/science.2479986</a>.</p>
</div>
<div>
<p>Liu, Jiangang, Andrew Campen, Shuguang Huang, Sheng-Bin Peng, Xiang Ye, Mathew Palakal, A Keith Dunker, Yuni Xia, and Shuyu Li. 2008. “Identification of a gene signature in cell cycle pathway for breast cancer prognosis using gene expression profiling data.” <em>BMC Medical Genomics</em> 1 (1): 39. doi:<a href="https://doi.org/10.1186/1755-8794-1-39">10.1186/1755-8794-1-39</a>.</p>
</div>
<div>
<p>Lyons, Yasmin A, Sherry Y Wu, Willem W Overwijk, Keith A Baggerly, and Anil K Sood. 2017. “Immune cell profiling in cancer: molecular approaches to cell- specific identification.” <em>Npj Precision Oncology</em> 1. doi:<a href="https://doi.org/10.1038/s41698-017-0031-0">10.1038/s41698-017-0031-0</a>.</p>
</div>
<div>
<p>Marquez-Medina, Diego, Joel Salla-Fortuny, and Antonieta Salud-Salvia. 2012. “Role of gamma-delta T-cells in cancer. Another opening door to immunotherapy.” <em>Clinical and Translational Oncology</em> 14 (12): 891–95. doi:<a href="https://doi.org/10.1007/s12094-012-0935-7">10.1007/s12094-012-0935-7</a>.</p>
</div>
<div>
<p>Michiels, S., N. Ternès, and F. Rotolo. 2016. “Statistical controversies in clinical research: prognostic gene signatures are not (yet) useful in clinical practice.” <em>Annals of Oncology</em> 27 (12): 2160–7. doi:<a href="https://doi.org/10.1093/annonc/mdw307">10.1093/annonc/mdw307</a>.</p>
</div>
<div>
<p>Mody, Rajen J., Yi-Mi Wu, Robert J. Lonigro, Xuhong Cao, Sameek Roychowdhury, Pankaj Vats, Kevin M. Frank, et al. 2015. “Integrative Clinical Sequencing in the Management of Refractory or Relapsed Cancer in Youth.” <em>JAMA</em> 314 (9). American Medical Association: 913. doi:<a href="https://doi.org/10.1001/jama.2015.10080">10.1001/jama.2015.10080</a>.</p>
</div>
<div>
<p>Naito, Y, K Saito, K Shiiba, A Ohuchi, K Saigenji, H Nagura, and H Ohtani. 1998. “CD8+ T cells infiltrated within cancer cell nests as a prognostic factor in human colorectal cancer.” <em>Cancer Research</em> 58 (16): 3491–4. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9721846" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/9721846</a>.</p>
</div>
<div>
<p>NPR. 2010a. “Science Diction: The Origin of the Word ’Cancer’.” <a href="https://www.npr.org/templates/story/story.php?storyId=130754101" class="uri">https://www.npr.org/templates/story/story.php?storyId=130754101</a>.</p>
</div>
<div>
<p>———. 2010b. “Science Diction: The Origin of the Word ’Cancer’ : NPR.” <a href="https://www.npr.org/templates/story/story.php?storyId=130754101" class="uri">https://www.npr.org/templates/story/story.php?storyId=130754101</a>.</p>
</div>
<div>
<p>Oberg, Jennifer A., Julia L. Glade Bender, Maria Luisa Sulis, Danielle Pendrick, Anthony N. Sireci, Susan J. Hsiao, Andrew T. Turk, et al. 2016. “Implementation of next generation sequencing into pediatric hematology-oncology practice: moving beyond actionable alterations.” <em>Genome Medicine</em> 8 (1). BioMed Central: 133. doi:<a href="https://doi.org/10.1186/s13073-016-0389-6">10.1186/s13073-016-0389-6</a>.</p>
</div>
<div>
<p>Paget, Stephen. 1889. “THE DISTRIBUTION OF SECONDARY GROWTHS IN CANCER OF THE BREAST.” <em>The Lancet</em> 133 (3421). Elsevier: 571–73. doi:<a href="https://doi.org/10.1016/S0140-6736(00)49915-0">10.1016/S0140-6736(00)49915-0</a>.</p>
</div>
<div>
<p>Pagès, Franck, Anne Berger, Matthieu Camus, Fatima Sanchez-Cabo, Anne Costes, Robert Molidor, Bernhard Mlecnik, et al. 2005. “Effector Memory T Cells, Early Metastasis, and Survival in Colorectal Cancer.” <em>New England Journal of Medicine</em> 353 (25): 2654–66. doi:<a href="https://doi.org/10.1056/NEJMoa051424">10.1056/NEJMoa051424</a>.</p>
</div>
<div>
<p>Palucka, Karolina, and Jacques Banchereau. 2013. “Dendritic-Cell-Based Therapeutic Cancer Vaccines.” doi:<a href="https://doi.org/10.1016/j.immuni.2013.07.004">10.1016/j.immuni.2013.07.004</a>.</p>
</div>
<div>
<p>Papalexi, Efthymia, and Rahul Satija. 2017. “Single-cell RNA sequencing to explore immune cell heterogeneity.” <em>Nature Reviews Immunology</em> 18 (1): 35–45. doi:<a href="https://doi.org/10.1038/nri.2017.76">10.1038/nri.2017.76</a>.</p>
</div>
<div>
<p>Parker, Joel S, Michael Mullins, Maggie C U Cheang, Samuel Leung, David Voduc, Tammi Vickery, Sherri Davies, et al. 2009. “Supervised risk predictor of breast cancer based on intrinsic subtypes.” <em>Journal of Clinical Oncology : Official Journal of the American Society of Clinical Oncology</em> 27 (8). American Society of Clinical Oncology: 1160–7. doi:<a href="https://doi.org/10.1200/JCO.2008.18.1370">10.1200/JCO.2008.18.1370</a>.</p>
</div>
<div>
<p>Perkel, Jeffrey M. 2017. “Single-cell sequencing made simple.” <em>Nature</em> 547 (7661): 125–26. doi:<a href="https://doi.org/10.1038/547125a">10.1038/547125a</a>.</p>
</div>
<div>
<p>Pitt, Jonathan M, Marie Vétizou, Romain Daillère, María Paula Roberti, Takahiro Yamazaki, Bertrand Routy, Patricia Lepage, et al. 2016. “Resistance Mechanisms to Immune-Checkpoint Blockade in Cancer: Tumor-Intrinsic and -Extrinsic Factors.” <em>Immunity</em> 44 (6). Elsevier: 1255–69. doi:<a href="https://doi.org/10.1016/j.immuni.2016.06.001">10.1016/j.immuni.2016.06.001</a>.</p>
</div>
<div>
<p>Ponting, Chris P. 2017. “Big knowledge from big data in functional genomics.” <em>Emerging Topics in Life Sciences</em> 1 (3). Portland Press Journals portal: 245–48. doi:<a href="https://doi.org/10.1042/ETLS20170129">10.1042/ETLS20170129</a>.</p>
</div>
<div>
<p>Predina, J., E. Eruslanov, B. Judy, V. Kapoor, G. Cheng, L.-C. Wang, J. Sun, et al. 2013. “Changes in the local tumor microenvironment in recurrent cancers may explain the failure of vaccines after surgery.” <em>Proceedings of the National Academy of Sciences</em> 110 (5): E415–E424. doi:<a href="https://doi.org/10.1073/pnas.1211850110">10.1073/pnas.1211850110</a>.</p>
</div>
<div>
<p>Qian, Bin Z., and Jeffrey W. Pollard. 2010. “Macrophage Diversity Enhances Tumor Progression and Metastasis.” doi:<a href="https://doi.org/10.1016/j.cell.2010.03.014">10.1016/j.cell.2010.03.014</a>.</p>
</div>
<div>
<p>Quail, D F, and J A Joyce. 2013. “Microenvironmental regulation of tumor progression and metastasis.” <em>Nat Med</em> 19 (11): 1423–37. doi:<a href="https://doi.org/10.1038/nm.3394">10.1038/nm.3394</a>.</p>
</div>
<div>
<p>Research in Cancer, International Agency for. 2018. “GLOBOCAN - Facts Sheets by Cancer.” <a href="http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx" class="uri">http://globocan.iarc.fr/Pages/fact_sheets_cancer.aspx</a>.</p>
</div>
<div>
<p>Rizvi, Naiyer A., Matthew D. Hellmann, Alexandra Snyder, Pia Kvistborg, Vladimir Makarov, Jonathan J. Havel, William Lee, et al. 2015. “Mutational landscape determines sensitivity to PD-1 blockade in non-small cell lung cancer.” <em>Science</em> 348 (6230): 124–28. doi:<a href="https://doi.org/10.1126/science.aaa1348">10.1126/science.aaa1348</a>.</p>
</div>
<div>
<p>Robinson, Dan R., Yi-Mi Wu, Robert J. Lonigro, Pankaj Vats, Erin Cobain, Jessica Everett, Xuhong Cao, et al. 2017. “Integrative clinical genomics of metastatic cancer.” <em>Nature</em> 548 (7667). Nature Publishing Group: 297–303. doi:<a href="https://doi.org/10.1038/nature23306">10.1038/nature23306</a>.</p>
</div>
<div>
<p>Ross, J. S., C. Hatzis, W. F. Symmans, L. Pusztai, and G. N. Hortobagyi. 2008. “Commercialized Multigene Predictors of Clinical Outcome for Breast Cancer.” <em>The Oncologist</em> 13 (5): 477–93. doi:<a href="https://doi.org/10.1634/theoncologist.2007-0248">10.1634/theoncologist.2007-0248</a>.</p>
</div>
<div>
<p>Rygaard, Josrgen, and Carl O. Povlsen. 1976. “The Nude Mouse vs. The Hypothesis of Immunological Surveillance.” <em>Immunological Reviews</em> 28 (1). Wiley/Blackwell (10.1111): 43–61. doi:<a href="https://doi.org/10.1111/j.1600-065X.1976.tb00192.x">10.1111/j.1600-065X.1976.tb00192.x</a>.</p>
</div>
<div>
<p>Sagiv-Barfi, Idit, Debra K. Czerwinski, Shoshana Levy, Israt S. Alam, Aaron T. Mayer, Sanjiv S. Gambhir, and Ronald Levy. 2018. “Eradication of spontaneous malignancy by local immunotherapy.” <em>Science Translational Medicine</em> 10 (426). American Association for the Advancement of Science: eaan4488. doi:<a href="https://doi.org/10.1126/SCITRANSLMED.AAN4488">10.1126/SCITRANSLMED.AAN4488</a>.</p>
</div>
<div>
<p>Satija, Rahul, and Alex K. Shalek. 2014. “Heterogeneity in immune responses: from populations to single cells.” <em>Trends in Immunology</em> 35 (5): 219–29. doi:<a href="https://doi.org/10.1016/j.it.2014.03.004">10.1016/j.it.2014.03.004</a>.</p>
</div>
<div>
<p>Schelker, Max, Sonia Feau, Jinyan Du, Nav Ranu, Edda Klipp, Birgit Schoeberl, Gavin MacBeath, et al. 2017. “Estimation of immune cell content in tumour tissue using single-cell RNA-seq data.” <em>Nature Communications</em> 8 (1). Nature Publishing Group: 2032. doi:<a href="https://doi.org/10.1038/s41467-017-02289-3">10.1038/s41467-017-02289-3</a>.</p>
</div>
<div>
<p>Sebeok, Thomas A. 1976. “DISCUSSION.” <em>Annals of the New York Academy of Sciences</em> 280 (1 Origins and E). Blackwell Publishing Ltd: 481. doi:<a href="https://doi.org/10.1111/j.1749-6632.1976.tb25511.x">10.1111/j.1749-6632.1976.tb25511.x</a>.</p>
</div>
<div>
<p>Slater, Matthew H. 2013. “Cell Types as Natural Kinds.” <em>Biological Theory</em> 7 (2). Springer Netherlands: 170–79. doi:<a href="https://doi.org/10.1007/s13752-012-0084-9">10.1007/s13752-012-0084-9</a>.</p>
</div>
<div>
<p>Slavicek, Gregor. 2012. “Interdisciplinary -A Historical Reflection.” <em>International Journal of Humanities and Social Science</em> 2 (20). <a href="http://www.ijhssnet.com/journals/Vol{\_}2{\_}No{\_}20{\_}Special{\_}Issue{\_}October{\_}2012/10.pdf" class="uri">http://www.ijhssnet.com/journals/Vol{\_}2{\_}No{\_}20{\_}Special{\_}Issue{\_}October{\_}2012/10.pdf</a>.</p>
</div>
<div>
<p>Sonachalam, Madhankumar, Jeffrey Shen, Hui Huang, and Xiaogang Wu. 2012. “Systems biology approach to identify gene network signatures for colorectal cancer.” <em>Frontiers in Genetics</em> 3. Frontiers Media SA: 80. doi:<a href="https://doi.org/10.3389/fgene.2012.00080">10.3389/fgene.2012.00080</a>.</p>
</div>
<div>
<p>Stewart, T., S. CJ Tsai, H. Grayson, R. Henderson, and G. Opelz. 1995. “Incidence of de-novo breast cancer in women chronically immunosuppressed after organ transplantation.” <em>The Lancet</em> 346 (8978): 796–98. doi:<a href="https://doi.org/10.1016/S0140-6736(95)91618-0">10.1016/S0140-6736(95)91618-0</a>.</p>
</div>
<div>
<p>Stutman, O. 1974. “Tumor development after 3-methylcholanthrene in immunologically deficient athymic-nude mice.” <em>Science (New York, N.Y.)</em> 183 (4124): 534–6. <a href="http://www.ncbi.nlm.nih.gov/pubmed/4588620" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/4588620</a>.</p>
</div>
<div>
<p>Subramanian, Aravind, Pablo Tamayo, Vamsi K Mootha, Sayan Mukherjee, Benjamin L Ebert, Michael A Gillette, Amanda Paulovich, et al. 2005. “Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 102 (43). National Academy of Sciences: 15545–50. doi:<a href="https://doi.org/10.1073/pnas.0506580102">10.1073/pnas.0506580102</a>.</p>
</div>
<div>
<p>Sudhakar, Akulapalli. 2009. “History of Cancer, Ancient and Modern Treatment Methods.” <em>Journal of Cancer Science &amp; Therapy</em> 1 (2). NIH Public Access: 1–4. doi:<a href="https://doi.org/10.4172/1948-5956.100000e2">10.4172/1948-5956.100000e2</a>.</p>
</div>
<div>
<p>Talmadge, J E, and D I Gabrilovich. 2013. “History of myeloid-derived suppressor cells.” <em>Nat Rev Cancer</em> 13 (10): 739–52. doi:<a href="https://doi.org/10.1038/nrc3581">10.1038/nrc3581</a>.</p>
</div>
<div>
<p>Tamborero, David, Carlota Rubio-Perez, Ferran Muiños, Radhakrishnan Sabarinathan, Josep Maria Piulats, Aura Muntasell, Rodrigo Dienstmann, Nuria Lopez-Bigas, and Abel Gonzalez-Perez. 2018. “A pan-cancer landscape of interactions between solid tumors and infiltrating immune cell populations.” <em>Clinical Cancer Research : An Official Journal of the American Association for Cancer Research</em>, April. American Association for Cancer Research, clincanres.3509.2017. doi:<a href="https://doi.org/10.1158/1078-0432.CCR-17-3509">10.1158/1078-0432.CCR-17-3509</a>.</p>
</div>
<div>
<p>Taube, Janis M, Jérôme Galon, Lynette M Sholl, Scott J Rodig, Tricia R Cottrell, Nicolas A Giraldo, Alexander S Baras, et al. 2017. “Implications of the tumor immune microenvironment for staging and therapeutics.” <em>Modern Pathology</em>. doi:<a href="https://doi.org/10.1038/modpathol.2017.156">10.1038/modpathol.2017.156</a>.</p>
</div>
<div>
<p>Teng, Michele W. L., Jeremy B. Swann, Catherine M. Koebel, Robert D. Schreiber, and Mark J. Smyth. 2008. “Immune-mediated dormancy: an equilibrium with cancer.” <em>Journal of Leukocyte Biology</em> 84 (4): 988–93. doi:<a href="https://doi.org/10.1189/jlb.1107774">10.1189/jlb.1107774</a>.</p>
</div>
<div>
<p>“The CRI | Centre for Research and Interdisciplinarity.” 2018. Accessed April 30. <a href="https://cri-paris.org/the-cri/" class="uri">https://cri-paris.org/the-cri/</a>.</p>
</div>
<div>
<p>Thorsson, Vésteinn, David L Gibbs, Scott D Brown, Denise Wolf, Dante S Bortone, Tai-Hsien Ou Yang, Eduard Porta-Pardo, et al. 2018. “The Immune Landscape of Cancer.” <em>Immunity</em> 48 (4). Elsevier: 812–830.e14. doi:<a href="https://doi.org/10.1016/j.immuni.2018.03.023">10.1016/j.immuni.2018.03.023</a>.</p>
</div>
<div>
<p>Tirosh, Itay, Benjamin Izar, Sanjay M. Prakadan, Marc H. Wadsworth, Daniel Treacy, John J. Trombetta, Asaf Rotem, et al. 2016. “Dissecting the multicellular ecosystem of metastatic melanoma by single-cell RNA-seq.” <em>Science</em> 352 (6282): 189–96. doi:<a href="https://doi.org/10.1126/science.aad0501">10.1126/science.aad0501</a>.</p>
</div>
<div>
<p>Van Noorden, Richard. 2015. “Interdisciplinary research by the numbers.” <em>Nature</em> 525 (7569): 306–7. doi:<a href="https://doi.org/10.1038/525306a">10.1038/525306a</a>.</p>
</div>
<div>
<p>Van Pel, A, and T Boon. 1982. “Protection against a nonimmunogenic mouse leukemia by an immunogenic variant obtained by mutagenesis.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 79 (15): 4718–22. <a href="http://www.ncbi.nlm.nih.gov/pubmed/6981814 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC346748" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/6981814 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC346748</a>.</p>
</div>
<div>
<p>Vandenbon, Alexis, Viet H Dinh, Norihisa Mikami, Yohko Kitagawa, Shunsuke Teraguchi, Naganari Ohkura, and Shimon Sakaguchi. 2016. “Immuno-Navigator, a batch-corrected coexpression database, reveals cell type-specific gene networks in the immune system.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 113 (17). National Academy of Sciences: E2393–402. doi:<a href="https://doi.org/10.1073/pnas.1604351113">10.1073/pnas.1604351113</a>.</p>
</div>
<div>
<p>Velculescu, V E, L Zhang, W Zhou, J Vogelstein, M A Basrai, D E Bassett, P Hieter, B Vogelstein, and K W Kinzler. 1997. “Characterization of the yeast transcriptome.” <em>Cell</em> 88 (2): 243–51. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9008165" class="uri">http://www.ncbi.nlm.nih.gov/pubmed/9008165</a>.</p>
</div>
<div>
<p>Vesely, Matthew D., Michael H. Kershaw, Robert D. Schreiber, and Mark J. Smyth. 2011. “Natural Innate and Adaptive Immunity to Cancer.” <em>Annual Review of Immunology</em> 29 (1): 235–71. doi:<a href="https://doi.org/10.1146/annurev-immunol-031210-101324">10.1146/annurev-immunol-031210-101324</a>.</p>
</div>
<div>
<p>Virchow Rudolf. 1847. “Archiv für pathologische Anatomie und Physiologie und für klinische Medizin.” Berlin ; New York: Springer. <a href="file://catalog.hathitrust.org/Record/000493993 http://hdl.handle.net/2027/njp.32101076036878 (Bd.64 (1875)) http://hdl.handle.net/2027/uc1.31175008645106 (v.170 {\&amp;}index v.161-170) http://hdl.handle.net/2027/hvd.32044093331494 (Bd.111 (1888)) http://hdl.han" class="uri">file://catalog.hathitrust.org/Record/000493993 http://hdl.handle.net/2027/njp.32101076036878 (Bd.64 (1875)) http://hdl.handle.net/2027/uc1.31175008645106 (v.170 {\&amp;}index v.161-170) http://hdl.handle.net/2027/hvd.32044093331494 (Bd.111 (1888)) http://hdl.han</a>.</p>
</div>
<div>
<p>Wolchok, Jedd D. 2015. “PD-1 Blockers.” doi:<a href="https://doi.org/10.1016/j.cell.2015.07.045">10.1016/j.cell.2015.07.045</a>.</p>
</div>
<div>
<p>Zou, Weiping, Jedd D. Wolchok, and Lieping Chen. 2016. “PD-L1 (B7-H1) and PD-1 pathway blockade for cancer therapy: Mechanisms, response biomarkers, and combinations.” <em>Science Translational Medicine</em> 8 (328). doi:<a href="https://doi.org/10.1126/scitranslmed.aad7118">10.1126/scitranslmed.aad7118</a>.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="annexes.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/UrszulaCzerwinska/UCzPhDThesis/blob/master/10-references.Rmd",
"text": "Edit"
},
"download": ["UCzPhDThesis.pdf", "UCzPhDThesis.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
